<!DOCTYPE html>
<html>
<head>
<title>PubMed IDs SCR_001847</title> 
<!-- need to fill in SCR_???? here -->
<style>
table {
    font-family: arial, sans-serif;
    border-collapse: collapse;
    width: 100%;
}

td, th {
    border: 1px solid #dddddd;
    text-align: left;
    padding: 8px;
}

tr:nth-child(1) {
    background-color: #cccccc;
}

tr:nth-child(even) {
    background-color: #dddddd;
}
</style>


</head>

<body>
<h1>PubMed IDs SCR_001847 mentioned</h1>
<table>
  <tr>  <!-- These are the field names from resource_mentions_10142016.txt -->
    <!-- only 5 fields are used here -->
    <!-- id	uid -->
    <th>rid</th>
    <th>mentionid</th>
    <!-- rating	timestamp	mentionid_int -->
    <th>input_source</th>
    <th>confidence</th>
    <!-- vote_sum -->
    <th> snippet </th>
  </tr>
  <tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26221954">PMID:26221954</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each participant received a brain MRI using a 1.5T Philips Intera scanner (TR/TE 20/2.1 ms/ Flip angle 20 deg/ 256 x 256 matrix / acquisition time 8 ’  09 ” / 1.3 mm slice thickness/ 105 slices). FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ), the MRI software package comprising a suite of automated tools for segmentation, reconstruction, and derivation of regional volumes and surface-based rendering, was used for derivation of regions-of-interest (ROI). In total, 95 ROIs masks (35x2 cortical, 23 subcortical, and cerebellar gray matter and white matter) were extracted from the structural T1 image. Four set of non-overlapping ROIs were selected: FRC; TMP; PAR; and CG for the statistical analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26217172">PMID:26217172</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The datasets were analyzed with FreeSurfer 4.50 ( http://surfer.nmr.mgh.harvard.edu/ ) for automatic segmentation and SPM8 (rel. 5236) ( www.fil.ion.ucl.ac.uk/spm ) for ICV estimation. The volumes of 18 neuroanatomical structures (accumbens, amygdala, brainstem, caudate, cerebellar cortex, cerebellar white matter, cerebral cortex, cerebral white matter, cerebrospinal fluid (CSF), hippocampus, inferior lateral ventricle, lateral ventricle, pallidum, putamen, thalamus, total brain volume, 3rd ventricle, 4th ventricle) were segmented using an automated procedure described previously (Fischl et al.,  2002 ) (Figure  1 ). In structures that come in pairs, the sum of left and right hemisphere was used. Cortical gray matter was defined as the volume of the cerebral and cerebellar cortex; white matter was defined as the volume of the cerebral and cerebellar white matter; subcortical gray matter was defined as the volume of the accumbens, amygdala, caudate, hippocampus, pallidum, putamen, and thalamus; the ventricles were defined as the volume of the 3rd and 4th ventricle, the inferior lateral and lateral ventricle and the CSF. To avoid bias, all Freesurfer results were visually inspected by a blinded colleague at the Multimodal imaging lab, UCSD, where the data were analyzed, and all subpar datasets removed. ICV was estimated using an automated version of the reverse brain mask method (RBM) (Keihaninejad et al.,  2010 ) called the  “ automatic reverse brain mask method ”  by using the  “ new segment ”  approach of the SPM8 toolbox, full description in Hansen et al. ( 2015 ). This method was recently shown to have improved accuracy compared to the ICV estimate generated by Freesurfer (Hansen et al.,  2015 ) and comparable to the SPM12 results obtained in Malone et al. ( 2015 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26215471">PMID:26215471</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Finally, we utilized the three main tissue volumes measurement to evaluate our normalization method. To achieve this goal, we used the FreeSurfer tool ( http://surfer.nmr.mgh.harvard.edu/ ) to count the volumes of WM, GM and CSF for each subject. And we compared the volume difference of all the three main tissues with tissue volumes of high-quality images for the same subject before and after normalization.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26211895">PMID:26211895</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The functional imaging data were preprocessed and analyzed using Nipype 21 , which is a pipeline platform that joins software packages, including SPM8 (Welcome Department of Cognitive Neurology, London, United Kingdom,  http://www.fil.ion.ucl.ac.uk/spm ), Freesurfer ( http://surfer.nmr. mgh.harvard.edu ), and FSL ( http://www.fmrib.ox.ac.uk/fsl/index.html ) into a single workflow. In the preprocessing stage, a rigid-body transformation was used to realign the functional images to the mean EPI image, correcting for subject head movement and for slice timing. Outliers with movements  > 1 ? mm or with an intensity Z-threshold  > 3 standard deviations (SDs) from the mean were removed from the data using an artifact detection algorithm ( http://www.nitrc.org/projects/artifact_detect ). Surface-based analysis can provide more specific results for cortical structures based on the individually-reconstructed anatomical structure of cortical surfaces 22 . Moreover, this analysis is potentially more sensitive because its search domain is small (no white matter and no cerebrospinal fluid) 23 . Cortical surface models can also simplify data visualization by revealing the pattern of activation throughout the whole cortex in one view. For the surfaced-based analysis, Freesurfer was used to segment each anatomical volume into gray and white matter structures and to perform cortical surface reconstruction. The mean functional image generated by realignment was registered to each subject ’ s reconstructed structural MRI data 24 . The functional images were smoothed on the cortical surface using a Gaussian filter with a full-width at half-maximum (FWHM) of 4 ? mm. In addition, a whole-brain analysis was conducted that also encompassed subcortical regions and the cerebellum using AFNI software (NIMH, USA); the detailed methods and the resulting activation patterns are reported in  Supplementary Table S1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26207191">PMID:26207191</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each subject ’ s anatomical image is first corrected for intensity non-uniformity using the Non-parametric Non-uniform intensity Normalization (N3) ( Sled  &  Zijdenbos, 1998 ). Next, the volume is registered in the MNI305 atlas through an affine registration. Intensity normalization and skull stripping are then performed to improve further processing. Data is non-linearly registered to an average brain, and the brain is parcellated into cortical and subcortical structures according to an atlas. The parcellated regions-of-interest (ROIs) are then mapped to the subject ’ s native space. Data finally follow a pipeline to derive the cortical thickness (CT), surface area (SA) and gray matter volume (GMV) measures for the subject cortical ROIs and also the volume of subcortical structures. All of the above processes were implemented using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). The generated measures are then loaded and converted into a MATLAB (.mat) file. Finally, anatomical connectivity matrices (a-CM) are computed from ratios of these measures between each pair of ROIs (CT, SA and GMV connectivity matrices).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26190993">PMID:26190993</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>3 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26175908">PMID:26175908</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging data were converted from DICOM to NIfTI-1 format ( http://nifti.nimh.nih.gov/ ) using mri_convert from FreeSurfer v4.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ). The de-identification of the high-resolution anatomical images was completed through the mask_face software 37 , which  ‘ blurs ’  facial anatomy. Facial blurring was selected for data anonymization as traditional skull stripping algorithms may lead to the failure of automated pre-processing pipelines and/or remove anatomical features necessary for the calculation of intracranial volume and cerebrospinal fluid volume 37 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26175549">PMID:26175549</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Lateral temporal cortex volumes were extracted in the left and the right hemisphere according to the following subregions: superior temporal gyrus, inferior temporal region — including middle and inferior temporal gyrus — and temporal pole (see  Figure 1(a) ). This cortical parcellation was done for each patient using a multiatlas label propagation approach. The image processing pipeline consisted of the following steps: (1) a skull-stripping step [ 62 ] was performed with the Freesurfer image analysis suite, freely available for download online ( http://surfer.nmr.mgh.harvard.edu ); (2) each MR image was given B0 nonuniformity correction (using Freesurfer) and ITK-based histogram matching using an arbitrarily chosen patient image as a reference histogram; (3) a set of 16 anatomically annotated images (obtained from the NIREP project,  http://www.nirep.org ) was registered in a nonlinear way on each patient image [ 63 ] using the ANTs software ( http://www.picsl.upenn.edu/ANTS ); (4) cortical labels were fused [ 64 ] using the simultaneous truth and performance level estimation (STAPLE); (5) final cortical parcellation was obtained by fusing logical and estimated labels using STAPLE and the gray matter mask obtained with the Freesurfer image analysis suite.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26161075">PMID:26161075</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26147749">PMID:26147749</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used subject-specific striatal VOIs derived from FreeSurfer 5.1 (Massachusetts General Hospital, Harvard Medical School;  http://surfer.nmr.mgh.harvard.edu ) as a gold standard for comparing striatal [ 18 F]-FP-CIT binding values derived from five different spatial normalization methods. T1-weighted MR images were resliced to 1 mm isovoxel space, corrected for inhomogeneity, skull-stripped, and segmented into gray and white matter. And then, subcortical structures were segmented and labelled by a probabilistic registration technique [ 17 ]. We chose five segmented subcortical segments of the caudate and putamen for each side, as well as that of the cerebellum. These subcortical segments were coregistered to original PET images by using the parameter coregistering inhomogeneity-corrected MR images created by FreeSurfer to CT images in native space, and finally, FreeSurfer-generated specific VOIs (FSVOI) for the caudate, putamen and cerebellum were created. For the striatal subregional analysis, a striatal subregional VOI template was non-linearly transformed to FSVOI by applying the inverse of the normalization parameter transforming the MR images to the MR template [ 18 ,  19 ]. Thereby, FSVOI was further parcellated into five striatal subregions (anterior and posterior caudate, anterior and posterior putamen, and ventral striatum).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26146534">PMID:26146534</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI images were processed with FreeSurfer ( http://surfer.nmr.mgh.harvard.edu , version 5.3) in each individual ’ s native space. Prior to cortical reconstruction, all images were resampled to a common isotropic voxel size of 1 ? × ? 1 ? × ? 1   mm. For each participant, cortical reconstructions were carefully inspected on a slice-by-slice basis and corrected by a single experienced FreeSurfer user (MS). Following thorough quality control, 15   % of the acquisitions from the original ABIDE sample had to be excluded, mainly due to excessive head motion. Among the remaining good quality acquisitions ( n ? = ? 945), the samples were selected via the matching process detailed above. Cerebral volumes were extracted using previously described procedures [ 49 ]. Briefly, the processing steps involved (1) removing non-brain tissue, (2) executing automatic segmentation of the subcortical gray matter structures, and (3) extracting cortical surfaces [ 50 ,  51 ]. Both intensity and continuity data from the entire three-dimensional MR volume were used in the segmentation procedures, thus producing accurate representations of cortical thickness and volume. These procedures have been validated against histological studies [ 52 ] and have been shown to be reliable across scanner models and field strengths [ 53 ]. The reconstruction process resulted in measurements for cortical volume, cerebral white matter volume, and subcortical volume. Supratentorial volume was also computed as the sum of cortical, cerebral white matter, and subcortical volumes. Intracranial volume was not extracted, as some sites did not include the entire cerebellum in their field of view.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26125043">PMID:26125043</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Diffusion data were processed using The Oxford Centre for Functional Magnetic Resonance Imaging of the Brain (FMRIB) software library (FSL version 4.1;  www.fmrib.ox.ac.uk/fsl ) 28  and the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ). The dt_recon command, as implemented in FreeSurfer, was used for eddy current and motion correction. A 12-parameter affine transformation process with FMRIB ’ s Linear Image Registration Tool (FLIRT) was used for within-subject image registration (e.g., the low  b  volume was collected using identical parameters as the diffusion-weighted volumes and thus registered with the diffusion maps, within-subject registration of the MP-RAGE and diffusion images). Diffusion tensor reconstruction (including the six components, three eigenvectors and three eigenvalues) were calculated for each voxel using a least-squares fit to the diffusion signal. 29  To remove non-brain voxels from the analysis, the low  b  volume was skull stripped using FSL ’ s brain extraction tool (BET), and the resulting brain mask applied to the fractional anisotropy (FA) maps. FA, the most common metric reported in diffusion studies, measures the primary direction of diffusion of water molecules, and is considered to reflect myelination, fiber density, and axonal diameter. FA values range between 0 and 1, with higher values assumed to reflect greater white matter integrity. Voxelwise statistical analysis of the FA data was carried out using Tract-Based Spatial Statistics (TBSS 30 ) as implemented in FSL. All subjects ’  FA data were aligned into MNI152 space using   the nonlinear registration tool FNIRT, which uses a b-spline representation of the registration warp field. Next, the mean FA image was created and thinned to create a mean FA skeleton that represents the centers of all tracts common to the group. Maps were threshold to exclude voxels with FA values  < 0.2 that may reflect multiple tissue types or crossing fibers. Each subject ’ s aligned FA data was then projected onto this skeleton, manually inspected to confirm accurate registration, and the resulting data fed into voxelwise cross-subject statistics.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26110108">PMID:26110108</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based methods work on the  surface model  ( Fischl et al., 2001 ). In this model, cortical thickness is calculated as the average of the distance from the WM surface (white-gray interface) to the closest possible point on the pial surface (gray-CSF interface), then from that point back to the closest point on the WM surface again ( Fischl and Dale, 2000 ;  Han et al., 2006 ). This is visualized in  Fig. 1 . Thus, the precision of cortical thickness is determined by the brain tissue (WM and pial) boundaries, but will be affected by voxel resolution, scanner and workstation type ( Gronenschild et al., 2012 ;  Han et al., 2006 ;  Lusebrink et al., 2013 ). To achieve an accurate approximation of boundaries, many topologic methods such as level sets were utilized to improve the triangulated mesh used to segment WM and GM ( Han et al., 2001a ;  Lohmann et al., 2003 ). Further improvement in surface topology correction occurred after the WM and GM boundaries were ascertained ( Han et al., 2001b ), improving the accuracy of segmentation ( Dale et al., 1999 ;  Han et al., 2001b ). Smoothing and constraints are the normal ways to reduce bias caused by incorrect tissue segmentation and subsequent inaccuracy of topology surface estimation. This topological mesh model makes them very time-consuming to process, and may lack the ability to describe the deep sulci and tightly folded areas ( Im et al., 2006 ;  Lohmann et al., 2003 ;  Zeng et al., 1998 ). Despite these operational disadvantages, surface-based methods, especially FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ), are popular, and their performance has been demonstrated to be robust in datasets with different field strengths, scanner upgrades and scanner types ( Han et al., 2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106565">PMID:26106565</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical images were processed in FreeSurfer v5.0 ( Dale et al., 1999 ). We focused on volumes of the amygdala and hippocampus. Each participant's anatomical image was processed using an automated segmentation and probabilistic region-of-interest (ROI) labeling technique (FreeSurfer,  http://surfer.nmr.mgh.harvard.edu ). Relative amygdala volume and hippocampal volume were calculated by dividing raw amygdala or hippocampal volume by total cranial volume in each participant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106557">PMID:26106557</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer to obtain surface meshes of the boundary between grey matter and white matter from T1 anatomical brain images ( http://surfer.nmr.mgh.harvard.edu , cf.  Fig. 1 ; Image processing on the Left Side). After registering surface meshes into the diffusion space, we generated volume regions of interest (ROIs), which are voxels in the grey matter. FreeSurfer provides parcellation of anatomical regions of cortices (34 for each hemisphere) based on the Desikan atlas ( Fischl et al., 2004 ;  Desikan et al., 2006 ) and subcortical regions ( Fischl et al., 2004, 2002 ) of which seven for each hemisphere (nucleus accumbens, amygdala, caudate, hippocampus, pallidum, putamen, and thalamus) were included (see  Fig. 2 a for names of ROIs). Thus, our structural brain networks, observing both hemispheres, consisted of 82 cortical and subcortical regions in total. Parcellation and registration were manually checked for errors by visual inspection.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106549">PMID:26106549</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI-based quantification of cortical thickness, SA and lGI was performed using FreeSurfer 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ). This method has been described in detail in a previous study ( Fischl, 2012 ). Briefly, the procedure involves segmentation of white matter, tessellation of the gray-white matter junction, inflation of the folded surfaces and automatic correction of topological defects in the resulting manifolds to construct representations of the gray/white matter boundary and the cortical surface. This approach uses both intensity and continuity information from the entire 3D MRI volume in segmentation and deformation procedures, and employs spatial intensity gradients across tissue classes instead of relying on absolute signal intensity. Successively, each individual's entire cortex was visually inspected and, if needed, manually edited by one of the authors (N.T.), who was blind to participant group status. This involved: (i) realignment of each subject's image to the Montreal Neurological Institute (MNI) template; (ii) setting intensity normalization control points where brain matter was erroneously skull-stripped; and (iii) adjustment of the watershed parameters of the skull strip. Cortical thickness measurements were obtained by reconstructing representations of the gray/white matter boundary and the cortical surface (approximately 160,000 vertices arranged in a triangular grid), where the distance between these two surfaces was calculated individually at each point/vertex across the cortical mantle.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106543">PMID:26106543</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was performed with the FreeSurfer 5.3.0 image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu / ). The technical details of these procedures are described in other publications ( Dale et al., 1999 ;  Dale and Sereno, 1993 ;  Fischl et al., 2004a ;  Fischl and Dale, 2000 ;  Fischl et al., 2001 ). Briefly, this includes motion correction and averaging ( Reuter et al., 2010 ) of multiple volumetric T1 weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure ( S é gonne et al., 2004 ), automated Talairach transformation, intensity normalization ( Sled and Pike, 1998 ), tessellation of the gray and white matter boundary, automated topology correction ( Fischl et al., 2001 ;  S é gonne et al., 2007 ), and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid (CSF) borders at the location where the greatest shift in intensity defines the transition to the other tissue class ( Dale et al., 1999 ;  Dale and Sereno, 1993 ;  Fischl and Dale, 2000 ). Once the cortical models are complete, a number of deformable procedures can be performed for further data processing and analysis including surface inflation ( Fischl et al., 1999 ), registration to a spherical atlas which is based on individual cortical folding patterns to match cortical geometry across subjects ( Fischl et al., 1999 ), parcellation of the cerebral cortex into units with respect to the gyral and sulcal structures ( Desikan et al., 2006 ;  Fischl et al., 2004b ), and creation of a variety of surface based data. This method uses both intensity and continuity information from the entire three-dimensional MR volume in the segmentation and deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/CSF boundary at each vertex on the tessellated surface ( Fischl and Dale, 2000 ). The maps are created using spatial intensity gradients across tissue classes and are therefore not simply reliant on absolute signal intensity.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106539">PMID:26106539</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The data were analyzed using a combination of the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ) and The Oxford Centre for Functional Magnetic Resonance Imaging of the Brain (FMRIB) FSL software package ( http://www.fmrib.ox.ac.uk/fsl ). Images were corrected for motion and eddy currents using Freesurfer. FA images were created by fitting a tensor model using linear least squares to the raw diffusion data using the Freesurfer command  dt_recon . Images were brain-extracted using BET ( Smith, 2002 ) to remove non-brain voxels from the analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106533">PMID:26106533</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction, volumetric segmentation, and diffusion processing for the generation of connectomes were executed ( Pannek et al., 2014 ). In brief, freesurfer ( http://surfer.nmr.mgh.harvard.edu , version 5.0) was used to parcellate the brain into 68 cortical regions, left and right thalami (2 regions), left and right cerebellum (2 regions), and brain stem (1 region). Posterior limb of internal capsule (PLIC) was not included in the connectome because it is not a region where connections should terminate. A total of 73 regions were thus included in the connectome construction.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106315">PMID:26106315</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing of the fMRI data was performed using the mrVista software package (available at  http://vistalab.stanford.edu/ ), modified for use in our own lab. The first 15 s of data were discarded to reduce transient effects. Within-scan motion was then estimated using a robust intensity-based scheme (Nestares and Heeger,  2000 ). Between-run motion was corrected using the same scheme, this time applied to the temporal average intensity of the entire scan. The first run of the session was used as the reference. Because the goal is to learn associations between patterns of activation in the brain and stimulus presentation, it is important that the activation is temporally aligned with the stimulus. Therefore, a Wiener filter deconvolution (Poor,  1980 ) was applied using a generic difference-of-gamma hemodynamic response function (Glover,  1999 ) as the kernel to the recorded BOLD signal. Mostly, the deconvolution served to shift the peak response in time so that it was aligned with its associated stimulus, but it also provided some amount of noise reduction. The high-resolution reference anatomies were segmented using the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) to create approximate parcellations of the gray matter in each subject, as well as a surface model for visualization of mapping results.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106307">PMID:26106307</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26090872">PMID:26090872</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The best corrected visual acuity was measured using decimal Monoyer charts and converted into logMAR units for statistical analysis purposes. After fundoscopy, images of the retinal fundus were systematically taken via non-mydriatic fundus photography and reexamined post-hoc by an experienced ophthalmologist. Evaluation of comorbidities (i.e., diseases lasting at least 3 months and running a course with minimal change, whatever the etiology) was based on self-report and medical record. All participants in the study had a cognitive assessment at the time of inclusion. Dementia was diagnosed using the consensus criteria of the Diagnostic and Statistical Manual of Mental Disorders, fourth edition [ 24 ]. Functional autonomy was assessed using the Instrumental Activities of Daily Living (IADL) score [ 25 ], with a score ranging from 0 to 4, best. The body mass index (BMI) was calculated as: [weight (kg) / height 2  (m 2 )]. Weight was measured with a beam balance scale, and height with a height gauge. The intracranial volume, which was reported to possibly influence the OCT measures [ 26 ], was measured in cm 3  to account for interindividual differences in the size of cranial cavity that could explain differences in intracranial su-volumes and thicknesses independent of vitamin D status. The intracranial volume was obtained from 3D T 1 -weighted MP-RAGE images obtained from 1.5 Tesla MRI scanner (Magnetom Avanto, Siemens Medical Solutions, Erlangen, Germany) (acquisition matrix = 256x256x144, FOV = 240mm x 240mm x 187mm, TE/TR/TI = 4.07ms/2170ms/1100ms) and analyzed using FreeSurfer (v5.1.0), a set of tools that automatically segments, labels and quantifies brain tissue volumes ( http://surfer.nmr.mgh.harvard.edu/ ) [ 27 ]. The season of evaluation was recorded as follows: spring from March 21 to June 20, summer from June 21 to September 20, fall from September 21 to December 20, winter from December 21 to March 20. Finally, the serum concentration of calcium was measured using automated standard laboratory methods at the University Hospital of Angers, France. Because of the high prevalence of hypoalbuminemia in older adults, calcium values were corrected according to the formula: [corrected calcium value = Ca + 0.02 (46-albumin)] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26089775">PMID:26089775</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To infer the cortico-cortical connectivity of the patient we first, using the T1 image, segmented white matter and gray matter areas, then performed parcellation of the gray matter into 66 regions of interest. These regions of interest correspond to major gyral-based anatomical areas which have been shown to be highly consistent between subjects (Desikan et al.,  2006 ). These gray matter volume ROIs generated using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) were then imported into DSI studio (Yeh et al.,  2010 ) along with the motion corrected diffusion images. Whole brain seeding was then used and tractography was performed. Only tracts with both ends terminating in the gray matter were retained. When a total of 5,000,000 tracts were found tractography was terminated. With the tracts and the ROIs registered to the same space the mean fractional anisotropy along tracts connecting two ROIs was then taken as a connectivity weight. This weighted structural connectivity matrix ( M ) is then used in the model to directly represent cortical connectivity of the patient. Figure  1  summarizes the image processing. A full list of ROI names can be found in Table  S1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26086820">PMID:26086820</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI scans were processed and analysed using the freely available software FreeSurfer (version 5.1.0;  http://surfer.nmr.mgh.harvard.edu/ ), run on Ubuntu with the Linux 2.6.28-11-generic kernel. Further technical details of FreeSurfer can be found in the literature [ 48 – 52 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26080319">PMID:26080319</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumes of the hippocampi and entorhinal cortices in the ANM cohort were obtained using FreeSurfer 5.1.0 (freely available from  http://surfer.nmr.mgh.harvard.edu/ ) from 276 ANM subjects who had undergone structural MRI. These regions were selected as they are known to be related to early AD pathology, and were normalised by intracranial volume. 29  Detailed information about data acquisition, pre-processing and quality control assessment have been described for this cohort in detail elsewhere. 29 ,  30 ,  31 ,  32 ,  33</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26075614">PMID:26075614</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Both regions of interest (ROIs) and whole brain analyses were then carried. First, two ROIs were defined in the left and right amygdalae of each participant using FreeSurfer v5.0.0 64-bit ( http://surfer.nmr.mgh.harvard.edu/ ), automated segmentation of subcortical structures ( http://surfer.nmr.mgh.harvard.edu/fswiki/SubcorticalSegmentation/ ). Anatomical brain FreeSurfer's processing includes removal of non-brain tissue, automated Talairach transformation, and segmentation of the subcortical white matter and deep grey matter volumetric structures. Amygdala ROIs in nifti (.nii) format where imported to BrainVoyager (plugin: nifticonverter_v106.dll) and transformed into Talairach space (mean[SD] coordinates, x, y, z, and number of voxels: left amygdala, -21.27[1.24] , -3.23[1.33], -13.27[0.92] , N = 3055 voxels; and right amygdala: 22.06[1.17], -3.90[1.50] , -13.46[0.95], N = 3091voxels) (see  S1 File .).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26075097">PMID:26075097</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All patients underwent a 1.5 ? T Philips Achieva magnetic resonance imaging (MRI) using an eight-channel head coil. The structural images were acquired using a sagittal 3D T1 weighted magnetization-prepared rapid gradient-echo (MPRAGE) sequence (repetition time = 8.7 ? ms, echo time = 4.0 ? ms, inversion time = 1000 ? ms, and flip angle = 8 ° ). Possible head movements were minimized by placing foam pads inside the head coil. The volumetric segmentation and measurement were performed with the FreeSurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26074781">PMID:26074781</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>HARDI preprocessing included motion correction, eddy current and correction of the vectors using the Artifact correction in diffusion MRI (ACID) toolbox (Mohammadi et al.,  2010 ) into the batch system of Statistical Parametric Mapping 8 (SPM8). We used FSL and FMRIB's Diffusion Toolbox (FDT) ( www.fmrib.ox.ac.uk/fsl ) to perform segregation of brain tissue from non-brain tissue using the Brain Extraction Tool, local fitting of diffusion tensors and construction of individual FA maps using DTIFIT, and tensor estimation with BEDPOSTX routine with the following options: Fibers (3), Weight (1), Burn In (1000). A segregation of brain tissue from non-brain tissue using the Brain Extraction Tool (BET) was performed on the structural images. The left and right NAcc were segmented using FIRST from FSL on structural images. After the segmentation, we applied boundaries correction using first_boundary_corr that is used for the classification of the boundary voxels in the volumetric output for a single structure. We applied  fast  boundary correction method that used FSL's FAST-based tissue classification for correcting boundary voxels. The results were visualized and checked using Freesurfer (Freeview) image analysis suite (Version 5.1.0) ( http://surfer.nmr.mgh.harvard.edu/ ). The same method has been applied to extract sub-cortical target masks. The other target masks were extracted with Individual Brain Atlases using SPM (IBMASPM) (Aleman-Gomez et al.,  2006 ). The masks have been extracted using ITK-SNAP ( http://www.itksnap.org ). All the masks have been binarized. Masks have been taken by manually drawing the region of interest, when automatic extraction results did not reach quality expectations or when region masks were not available within the previously mentioned tools. This was the case for subthalamic nucleus, substantia nigra, ventral tegmental area, hypothalamus, habenula and subcalosal cingulate (Chowdhury et al.,  2013 ). Registration from structural to diffusion space was performed using FMRIB's Linear Image Registration Tool (FLIRT) in FSL. We performed a 3D-to-3D registration between diffusion and structural image with the affine model, 12 °  of Freedom and the use of the Tri-linear final interpolation method. The resulting structural to diffusion registration was manually checked to ensure satisfactory alignment, with particular attention paid to the regional borders of the BG in general and NAcc, in particular. We run tractography analysis with probabilistic tracking (probtrackx) in FSL using the segmented left and right NAcc seeds, the target masks and the output matrix from the registration (structural to diffusion). We used the following parameters: curvature threshold of 0.2 corresponding to a minimum angle of approximately 80 ° , number of samples 5000 and, loopcheck option. We developed a Matlab script in order to extract from the probabilistic tractography outputs the number of tracts that leave a voxel from NAcc to reach a given target. A Python script has been developed to calculate the number of voxels within the NAcc that have a probability greater than 1% to be connected to a specific target. Voxels with a connection probability of at least 0.01 were included as voxels containing anatomically valid pathway. The 1% threshold is a typical threshold used in probabilistic tractography (Lambert et al.,  2012 ; Li et al.,  2013 ). Threshold was set to 1% (out of the 5000 generated from each seed voxel) to reject voxels with low probability. This means that at least 1% of the identified fibers intersect the voxels in the valid pathway. This gave us a matrix of  “ strengths ”  of interconnection on a scale of 0 – 100%.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26072517">PMID:26072517</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated reconstruction and segmentation of individual subjects ’  T 1  scans into surface mesh representations were carried out using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). The surface mesh of each cerebral hemisphere was inflated to a sphere while minimizing distortions to facilitate registration and maintain individual differences in cortical topology ( Dale  et al. , 1999 ). The body-part-specific statistical parametric maps were registered and projected to the individual sphere mesh. To allow us to specifically focus on topographic shifts in body-part representation, individual maps were masked by FreeSurfer's pre- and postcentral gyrus labels. Maps were thresholded by a false discovery rate (FDR) of q  <  0.05 and a minimal use area of 100 mm 2 . The resulting maps were visually inspected to verify that the clusters were located along the sensorimotor strip. To minimize the potential contribution of neighbouring sensorimotor representations beyond the homunculus, clusters centred in the secondary somatosensory cortex and in the posterior parietal cortex surviving the contrasting and masking procedures were discarded from this analysis (see  Makin  et al. , 2015  for a study of whole-brain reorganization in amputees). As the aim of this analysis was simply to locate the spatial position of activation centres, the threshold criteria were waived when clusters did not survive the thresholding criteria (feet 2/3; lips 2/1; hands 1/0; arms 2/1 for amputees/controls, respectively). A CoG approach was applied to identify the location of each cluster. This approach was previously proven to be reliable across sessions, independently of functional MRI procedures (e.g. thresholding, clustering) for hand and lip movement representations ( Fesl  et al. , 2008 ). The CoG of a surface cluster was defined as the average position of all its vertices on the sphere. Therefore, the distance between the CoG was defined as the shortest distance between the points along the surface of the sphere (i.e. the great circle distance). Distances between the feet CoG and each of the body part representations were measured for each participant bilaterally. In addition, distances were measured for each hemisphere between the centres of gravity of hand and lip clusters, and between lip clusters CoGs and the intercept of the central sulcus with the medial wall (as specified manually by the experimenter). These measurements of lip distances from these various anchors strongly correlated with each other (all r ’ s  >  0.74, all  P -values  < 0.001), even when accounting for differences in brain size (by controlling for hand-to-feet distances). The clusters from the surface analysis were further projected back to a folded brain, and the CoG of a 3D cluster was defined as the average position of the voxels in each axis. The 3D distance was defined as the Euclidian distance across the three axes ( Lotze  et al. , 2001 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26066863">PMID:26066863</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The methods for our ROI-based analyses are described in Eisenstein  et al. 10 33 . The neuroimaging software FreeSurfer (http://surfer.nmr.mgh.harvard.edu) was used for segmentation of striatal regions 34 . To limit multiple comparisons, D2R specific binding (BP ND ) for each ROI was averaged across left and right hemispheres. Putamen and caudate D2R BP ND s were averaged to obtain a composite dorsal striatal BP ND  and ventral striatal BP ND  included average nucleus accumbens D2R BP ND.  The midbrain region was traced on each individual ’ s MPRAGE as previously described 33 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26066658">PMID:26066658</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical thickness and subcortical structure volume were estimated using FreeSurfer v5.1.0 ( http://www.nmr.mgh.harvard.edu/freesurfer/ ). The overall process and analysis pipeline have been described elsewhere ( http://surfer.nmr.mgh.harvard.edu ). First, the high-resolution 3D T 1 TFE volumes were converted to FreeSurfer format (mgh file), normalized for intensity and resampled to isotropic voxels of 1mm 3 . Next, the skull was removed using a skull-stripping algorithm and segmented into three tissue types (white matter, grey matter and CSF). Third, cortical thickness measurements were obtained by reconstructing representations of the grey/white matter interface and the cortical surface and then by calculating the distance between these surfaces at each point across the curtail mantle. For this cortical thickness measurements purpose, a warping of subject cortical surfaces to a reference surface with known localization of cortical structures is required. An FS standard averaged cortical surface template (FSaverage template) and nonlinear procedure were used[ 18 ]. In order to obtain difference maps of cortical thickness, the data were smoothed on the level of the sphere using Gaussian smoothing kernel with a full-width half maximum of 10mm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26065881">PMID:26065881</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the Freesurfer image analysis suite, freely available online ( http://surfer.nmr.mgh.harvard.edu/ ), the technical details of which have been described previously [ 32 ]. Briefly, this method uses both intensity and continuity information from the entire three dimensional MR volumes in segmentation and deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/CSF boundary at each vertex on the tessellated surface. After a careful visual inspection for any gross anatomical abnormalities, all the scans were run as a batch on FreeSurfer v.5.1. Once the entire subject pool finished running, we made sure the reconstruction was successful by checking for Talairach registration. The adequacy of skull stripping, generation of white and pial surfaces, and segmentations were evaluated by the checking the images using TKMEDIT. Freesurfer completed the processing of the images without any need for manual interventions. The details of integration of surface- and volume-based representations are given in Makris et al. [ 60 ] and Desikan et al. [ 61 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26061655">PMID:26061655</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Estimates of CTh were performed from cortical surface reconstructions computed from T1 weighted images using FreeSurfer (v. 5.1,  http://surfer.nmr.mgh.harvard.edu/ ). The technical aspects of these methods have been described,in detail, elsewhere[ 23 ,  24 ]. In brief, the processing stream involved intensity non-uniformity correction, Talairach registration, removal of non-brain tissue (skull stripping), white matter (WM) and subcortical grey matter (GM) segmentation, tessellation of the GM-WM boundary then surface deformation following GM-CSF intensity gradients to optimally place GM-WM and GM-CSF borders[ 23 ,  24 ]. Once cortical models were generated, surface inflation, transformation to a spherical atlas and parcellation of the cerebral cortex into regions based on gyraland sulcal structure were carried out[ 25 ]. This technique used both intensity and continuity information from the entire 3D MR volume in the segmentation and deformation procedures to produce representations of CTh, calculated as the closest distance from the GM-WM to GM-CSF boundaries at each vertex on the tessellated surface[ 26 ].CTh measures were mapped to the inflated surface. All images were then aligned to a common surface template and smoothed with a 20mm full width at half maximum (FWHM) surface based Gaussian kernel.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26060504">PMID:26060504</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In this study, we propose a fully automated method for segmentation of the cerebellum that combined tissue classification, a template-based approach, and morphological operations sequentially. The method was validated by comparing the results with the manual segmentation results of the LONI Probabilistic Brain Atlas (LPBA40) dataset [ 18 ] using the dice similarity index and recall and precision measures. The method was also compared with other popular packages such as FreeSurfer ([ 24 ],  http://surfer.nmr.mgh.harvard.edu/ ) and ITK-SNAP ([ 35 ],  http://www.itksnap.org ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26029102">PMID:26029102</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The software package FreeSurfer version 4.3.1 ( http://surfer.nmr.mgh.harvard.edu ) was used for structural image analysis at all three waves. The two 3D T1 weighted volumes recorded in succession were first coregistered and averaged to improve signal to noise ratio (SNR). The following semi-automated FreeSurfer processing pipeline included Talairach registration (Talairach and Tournoux,  1988 ), skull stripping (S é gonne et al.,  2004 ), intensity normalization, tissue segmentation, and cortical reconstruction. Each analyzing step, including automatic anatomical labeling, was carefully inspected, and manual edits were preformed if needed. The FreeSurfer parcellated and labeled WM regions and subcortical nuclei were used as a basis for defining regions of interests (ROIs) in the subsequent DWI-based fiber tracking/WM analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26017384">PMID:26017384</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional data were analyzed using AFNI ([ 27 ], program names appearing in parentheses below are part of the AFNI suite). Cortical surface models were created with FreeSurfer (available at  http://surfer.nmr.mgh.harvard.edu/ ), and functional data projected into anatomical space using SUMA ([ 28 \n 29 ], AFNI and SUMA are available at  http://afni.nimh.nih.gov/afni ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26000319">PMID:26000319</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness was measured using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). 28 , 29  Each scan was coregistered to a surface-based atlas. 30  Standard vertex-wise statistical analysis of cortical thickness using a general linear model was performed to test for mean differences between groups. Age, sex, and estimated total intracranial volume (TIV) were used as covariates. Significance was tested using a vertex-wise false discovery rate corrected threshold of  P   <   0.05. A structural brain network was then constructed from the data. The native Freesurfer parcellation scheme was extended by subdividing each region into subregions of approximately equal area (mean 124   mm 2 , SD 30   mm 2 ) using  k -means clustering on the surface-based Euclidean distance, resulting in a total of 915 subregions over both hemispheres. Each subregion was treated as a node in the subsequent analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25984446">PMID:25984446</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and automated thickness measures were performed using the Freesurfer software, version 5.1 ( Fischl and Dale, 2000 ), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu ). Only images that passed a quality review were included in the present analyses. Cortical thickness was measured in subjects' native space. The average CT for each of the 68 cortical regions of interest (ROIs) as provided by Freesurfer (34 regions per hemisphere,  Desikan et al., 2006 ) was used to construct the brain graphs.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25984353">PMID:25984353</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Morphometric analysis of brain structure was completed with FreeSurfer Version 5.1 ( http://surfer.nmr.mgh.harvard.edu ). Detailed methodology for regional and total volume derivation has been described in detail previously [ 25 - 27 ]. Briefly, FreeSurfer was used to preprocess images (e.g. intensity normalization, skull stripping) then provide an automated parcellation of cortical and subcortical structures via an automated processing stream. FreeSurfer performs parcellation by registering images to a probabilistic brain atlas, built from a manually labeled training set, and uses this probabilistic atlas to assign a neuroanatomical label to each voxel in an MRI volume. Total brain volume, total gray matter, subcortical gray matter, cortical white matter volume, and intracranial volume measurements are derived automatically.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25977808">PMID:25977808</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Some of the most common software packages for analysing these data are freely available, and include SPM ( http://www.fil.ion.ucl.ac.uk/spm/ ), FieldTrip ( http://fieldtrip.fcdonders.nl/ ), FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ), FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ), MNE ( http://martinos.org/mne/ ), Brainstorm ( http://neuroimage.usc.edu/brainstorm/ ), EEGLAB ( http://sccn.ucsd.edu/eeglab/ ), NIPY ( http://www.nipy.org ) and AFNI ( http://afni.nimh.nih.gov/afni/ ). There are also several others (see  http://en.wikipedia.org/wiki/List_of_neuroimaging_software  for a more complete list). The Matlab scripts used for the Technical Validation in this paper can be provided by the authors.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25977792">PMID:25977792</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All 3D volumes were processed using FreeSurfer 6 – 8  (freely available online at  http://surfer.nmr.mgh.harvard.edu/ ), which provides quantitative volume data for a range of brain structures. We ran FreeSurfer without user intervention ( ‘ recon-all  – i data1.dcm  – all ’ ), since this is the mode of operation that would be used in an automated pipeline processing patient data. The computing hardware, operating system (Ubuntu 12.04.1) and FreeSurfer version (v5.1) were kept constant for the duration of this study, in line with recommendations made by Gronenschild  et al. \n 20 , who showed that these factors can affect results. We chose to restrict our analysis to the lateral ventricle volumes as well as a select group of subcortical structures of particular interest to us, including the hippocampus, amygdala, putamen, pallidum, caudate and thalamus. In all cases, volumes from the left and right hemispheres were summed together to generate a single value for each structure.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25969982">PMID:25969982</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI was performed using a 3.0-Tesla instrument (Siemens TIM Trio) with a gradient amplitude of 45 mT/m and a slew ratio of 150 mT/m/s. A reception head coil containing 12 elements was used for image acquisition. Both anatomical and functional scans were obtained. Volumetric T1-weighting images (MPRAGE, TR/TE/TI/FA/VOXEL = 2,500 ms/3.45 ms/1,100 ms/7 ° /1 × 1 × 1 mm 3 ) were automatically processed using the Freesurfer image analysis suite, version 5.1.0, which is well-documented and freely available online ( http://surfer.nmr.mgh.harvard.edu/ ). ROIs were determined anatomically via automatic segmentation using the Aparc.a2009s atlas [ 26 ] and used in the fMRI analysis. The ROIs used were the occipital pole and the superior and inferior calcarine regions of each hemisphere ( Fig 1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25962164">PMID:25962164</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used high-resolution T1-weighted images to measure regional cortical thicknesses in HF and control subjects using FreeSurfer (v 5.3.0) [ 9 ]. Both high-resolution T1-weighted image volumes, collected from each subject, were realigned and averaged to increase signal-to-noise ratio, and averaged brain volumes from each subject were converted into FreeSurfer data format ( http://surfer.nmr.mgh.harvard.edu/ ). Data processing included removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, intensity normalization, segmentation of sub-cortical white and deep gray matter tissue types, tessellation of the gray and white matter boundaries, automated topology correction, and surface registration to the FreeSurfer atlas [ 9 ]. All subjects ’  processed data were manually-evaluated by an investigator to ensure no brain areas were excluded. Similarly, gray, white, and pial boundaries were visually-assessed, and if needed, edits were made to correct misidentified regions. Such edits were required in a few HF and control subjects, which were performed by the same investigator who manually-evaluated all subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25909080">PMID:25909080</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were analyzed retrospectively after patient visits to Massachusetts General Hospital (MGH) between December 2009 and April 2013. All patients suffered from refractory TLE and were referred for presurgical imaging work-up. Inclusion criteria were age over 18   years and suspected TLE as determined by a specialized team of experienced epileptologists (N. T., A. J. C., C. R., S. M. S.) based on ictal video-EEG in accordance with the epilepsy classification by the International League Against Epilepsy (ILAE). Exclusion criteria were previous surgical intervention and neurological or psychiatric comorbidity. In addition to high-resolution imaging, clinical information was collected by medical chart review. Seizure frequency per month was extrapolated from patients' medical charts, and was only included when reported within 3   months of scanning date. Twenty-three healthy participants were also included, exclusion criteria being psychiatric or neurological disease and use of medication influencing the central nervous system. Hippocampal volumes were determined using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) automatic segmentation procedures. 25</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25904851">PMID:25904851</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25883546">PMID:25883546</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For M1, nine EPI runs (180 time points each) were acquired over six sessions. For M2, seven runs were acquired over four sessions. All data analyses were performed using AFNI (Cox,  1996 ) ( http://afni.nimh.nih.gov/afni ), FreeSurfer (Dale et al.,  1999 ; Fischl et al.,  1999 ) ( http://surfer.nmr.mgh.harvard.edu/ ), SUMA ( http://afni.nimh.nih.gov/ ) and custom code written in Matlab (MathWorks). Preprocessing involved slice timing correction, motion correction (relative to the run-specific mean GE-EPI), spatial smoothing with a 3.0 mm full width at half-maximum Gaussian kernel, and normalization of the time series at each voxel by its mean. All volumes that had motion values with shifts  > 0.5 mm and/or rotations  > 0.5 °  were excluded from further analyses. Lastly, we performed linear least-squares detrending to remove non-specific variations (i.e., scanner drift). Following preprocessing, data were submitted to generalized linear model analyses. The model included three stimulus-specific regressors and six estimated motion regressors of no interest. For each stimulus category (Env, MC, SMC) we estimated a regressor by convolving a one-parameter gamma distribution estimate of the hemodynamic response function with the square-wave stimulus function. We performed  t -tests contrasting all sounds vs. baseline ( “ silence ”  trials), MC vs. Env, and MC vs. SMC. Finally we co-registered and normalized our functional data to the population-average MRI-based template for rhesus monkeys 112RM-SL (McLaren et al.,  2009 ) and then displayed the results on a semi-inflated cortical surface of the template extracted with Freesurfer and displayed with SUMA to facilitate visualization and identification of cortical activations. The anatomical boundaries described here are based on the macaque brain atlas of Saleem and Logothetis ( 2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25867953">PMID:25867953</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We conducted cortical shape analysis using QDEC tool box provided by FreeSurfer ( www.surfer.nmr.mgh.harvard.edu ), in which general linear models were applied on a vertex basis. The results were projected onto the template, which were then corrected for multiple comparisons using FDR at a level of 0.05.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25852541">PMID:25852541</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25844307">PMID:25844307</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To determine whether normalizing the children's cerebella to an adult template would lead to excessively small effective regions of interest (ROIs), cerebellar volumes generated by FreeSurfer (version 5.1.0,  http://surfer.nmr.mgh.harvard.edu ) were compared to values reported in adult studies ( Luft et al., 1999 ;  Woodruff-Pak et al., 2001 ).  Woodruff-Pak et al. (2001)  calculated cerebellar volumes ranging from 122.73 to 142.37   ml in eight adults (age 21 – 35   years) and  Luft et al. (1999)  found a range of 99.86 – 170.6   ml in 48 adults (age 19.8 – 73.1   years). The children included in our functional study had an average cerebellar volume of 130.85   ±   13.03   ml (range 107.18 – 170.11   ml), which is within the limits of the aforementioned studies. The children from all diagnostic groups were included in this analysis, as previous studies have shown that children prenatally exposed to alcohol have reduced cerebellar volumes ( Archibald et al., 2001 ;  Mattson et al., 1994 ). It was, therefore, necessary to establish whether the volumes in these children were also comparable to the cerebellar volumes of adults. The overall effect of normalization to an adult template was, therefore, deemed negligible.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25816070">PMID:25816070</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with FreeSurfer (version 5.3.0,  http://surfer.nmr.mgh.harvard.edu ). The automated processing stream mainly included removal of non-brain tissue [ 22 ], Talairach transformation, segmentation of gray/white matter tissue [ 23 ], intensity normalization, topological correction of the cortical surface [ 24 ] and surface deformation to optimally place the tissue borders [ 25 ]. After creation of the cortical representations, the visual areas were parcellated based on existing atlases. The delineation of the V1 label was based on the study by Hinds et al. [ 26 ] and corresponded to Brodmann area (BA) 17 ( Fig. 1B ), and the V2 label was described by Fischl et al. [ 27 ] and corresponded to BA 18 ( Fig. 1E ). The delineation of the V5/MT+ label was based on the work of Malikovic et al. [ 28 ]. This V5/MT+ area was located close to the intersection of the anterior occipital and the inferior lateral occipital sulci in the region of the temporo-occipital junction ( Fig. 1N ). As the V3 and V4 templates are not available in FreeSurfer, we extracted the ventral V3 (V3v) and V4 using the Juelich histological atlas implemented in FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Atlases/Juelich ). The templates were transformed into surface labels and manually corrected according to Rottschy ’ s description [ 29 ]. The V3v is buried deep in the collateral sulcus ( Fig. 1H ) and V4 is located on the lateral bank of this sulcus but also reaches the fusiform gyrus in the occipital section ( Fig. 1K ). The dorsal V3 was not delineated here because there are differing views regarding its anatomical location and additional subdivisions [ 30 ]. The labels were then registered to the surface of each subject using the spherical parameters of the cortical surface [ 31 ]. For each subject, the five visual areas were visually inspected for any inaccuracies in parcellation and manually corrected if necessary.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25805989">PMID:25805989</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Secondly, an analysis of cortical thickness was performed using the longitudinal pipeline of FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). It creates a within-subject template through a variety of steps including image registration, normalization, skull-stripping, segmentation of gray and white matter, and delineation between the inner gray-white matter and outer pial surface. The resulting surface maps where then used to assess the longitudinal training effect, at both a whole-brain (QDEC) and regional level (parcellation), with changes in cortical thickness indicative of structural neuroplasticity (Reuter et al.,  2012 ). Vertex-based analysis was conducted on the outputs from the FreeSurfer longitudinal pipeline, using the FreeSurfer QDEC toolbox. After 20 mm smoothing, three measures of cortical thickness change were used to assess for plasticity, namely: (1) annualized rate of change ( ± ) in mm/year; (2) symmetrized percent change (SPC) with respect to the temporal average; and (3) percentage change with respect to baseline (PCL). In the post-processing model, SPC and PCL were set as dependent variable, while group was defined as the main factor. Multiple comparisons were corrected using two methods, namely whole-brain vertex based false detection rate (FDR) correction, and exploratory small-volume based FDR correction.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25802806">PMID:25802806</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray matter (GM), white matter (WM) and visual cortex volumes of interest (VOIs), excluding CSF regions, were defined from each subject ’ s high-resolution MP-RAGE image by FreeSurfer (version 5.3,  http://surfer.nmr.mgh.harvard.edu ) ( Desikan et al., 2006 ). VOIs were limited to voxels that were represented in every image in every subject; this step excluded much of the inferior occipital cortex in the visual cortex VOI ( Black et al., 2010b ). We also used the thalamic VOIs previously defined in our drug study ( Black et al., 2010b ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25793287">PMID:25793287</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package was used to process the images (version 5.3.0,  http://surfer.nmr.mgh.harvard.edu/ ). The procedures were based on previous reports [ 38 ]. Briefly, it included removal of non-brain voxels from each subject ’ s T1-weighted magnetic resonance imaging scan, transformation to standard space, and segmentation of the grey-white matter tissues. The grey-white boundary was represented with a polygonal tessellation, and topological correction was performed. This surface was then deformed outwards to locate the pial surface (i.e., boundary between cerebrospinal fluid (CSF) and grey matter). We reviewed all obtained cortical surfaces, and minimal manual editing was performed at inaccurate segmentations. The generated cortical surfaces were validated by comparing them to the manual measures on MRI data [ 38 – 40 ]. Cortical thickness was computed at each vertex as the average distance between the grey-white surface and pial surface. Surface area for each vertex was calculated on the pial surface, representing the area of the tessellated triangles linked to the vertex. The local cortical folding for each vertex was measured by means of local gyrification index (LGI), which took into account the ratio of local surface area to the outer hull layer that tightly wraps the pial surface [ 41 ], and was extended from the two-dimensional gyrification measurement [ 42 ]. The LGI value is an indication of sulcal cortex buried in its locality and thus denotes the extent of cortical folding [ 40 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25783540">PMID:25783540</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical volume and thickness measures were estimated using the openware, Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Our MR data were of high quality, and appropriate checks and edits were performed throughout the entire Freesurfer workflow (both authors evaluated/checked all the steps especially, brain segmentation, registration to Talaraich space, pial surface and GM-WM boundaries extraction results in each subject). Standard image-processing steps were adopted including: 1) correct for motion artifacts and strip skull based on a hybrid watershed/surface deformation procedure [ 17 ], 2) register images to a Talairach brain template and segment for subcortical WM and GM structures [ 18 ], 3) estimate the GM-WM boundary via a tessellation step, and subsequently perform automated topology correction, 4) optimally place GM-WM and GM-cerebrospinal fluid (GM-CSF) boundaries using surface normalization and intensity gradients, 6) after cortical models are complete, perform deformable procedures for further processing and analysis, such as surface inflation and registration to a spherical atlas [ 19 ]. Both intensity and continuity information from the entire 3D  MR volume are used to produce representations of cortical thickness, where thickness is measured as the closest distance from GM-WM to GM-CSF boundary at each vertex on the tessellated surface [ 20 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25781178">PMID:25781178</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Segmentation and parcellation of the T1 weighted anatomical structural images was performed with Freesurfer (available for download at  http://surfer.nmr.mgh.harvard.edu/ ) using the Desikan-Kyliany Atlas, in order to obtain the total intracranial volume (TICV), subcortical GM, cortical GM, total WM, thalamus, cerebellar and brainstem volumes [ 35 – 37 ]. On the MPRAGE, resampled to be in axial planes correctly oriented, we measured the area of the foramen magnum. The foramen magnum perimeter was manually delineated on the most inferior slice in which the bone was completely visible.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25761059">PMID:25761059</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FSL ( http://www.fmrib.ox.ac.uk/fsl ) routines-FLIRT and-BETSURF were used to register T1w and T2w MRIs, and to distinguish between scalp (which includes all extracranial tissues), skull and brain. The results of the automatic segmentation algorithm were afterwards inspected and manually corrected. FSL-FAST [ 50 ] and Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) were later used to distinguish CSF, white and gray matter. The skull spongiosa-compacta classification was implemented by eroding the skull estimate and performing a threshold based segmentation on the T2w MRI (limited to the skull estimate).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25749359">PMID:25749359</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Then, additional preprocessing was performed using the ANATICOR method [ 30 ] to remove further physiological and hardware related confounds. Individual masks of large ventricles and white matter to be used in this approach were obtained from the structural scans segmentation using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). The white matter mask was eroded slightly (one functional voxel) to prevent partial volume effects that might include signal from gray matter voxels in the mask. This step was not performed for the CSF mask, since with the functional voxel size used in this study the eroded CSF mask would not contain enough voxels in most subjects. Then, for each run, a global nuisance regressor was obtained extracting the EPI average time course within the ventricle mask and local nuisance regressors were obtained calculating for each gray matter voxel the average signal time course for all white matter voxels within a 3 cm radius [ 30 ]. These nuisance regressors and the six regressors derived from motion parameters were removed from the EPI timeseries using AFNI ’ s @ANATICOR.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25745134">PMID:25745134</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Grey-matter volumes, surface area and average cortical thickness were measured using Freesurfer Pipeline version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ). The analysis involved removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of\n Table 1 Demographic features of the participants</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25714434">PMID:25714434</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Region of Interest independent localization.  Regions of interest were localized using first session (10 blocks of faces and 10 blocks of cups). Face identity and cup type main discrimination analysis was done using all other sessions (see below). For Region of Interest (ROI) localization we estimated GLM model (HRF boxcar function) with two regressors: faces and cups. For defining face-selective regions the faces  >  cups contrast was used. The face-selective FFA, pSTS, prefrontal cortex were defined using p < 0.001, uncorrected threshold and ATL face-area and amygdala were defined using p < 0.01, uncorrected threshold. Non-face selective region in the collateral sulcus was defined using cups  >  faces contrast (p < 0.001, uncorrected) [ 45 ]. ROIs definition was done using the MarsBaR region of interest toolbox for SPM [ 46 ]. Cortical reconstruction of representative participant (inflated cortex map in  Fig. 1 ) was created using Freesurfer image analysis 4.5 ( http://surfer.nmr.mgh.harvard.edu/ ) [ 47 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25713519">PMID:25713519</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical data from the subjects were processed using the FMRIB Software Library (FSL) toolkit ( http://www.fmrib.ox.ac.uk/fsl/ ) to correct for spatial inhomogeneity and to perform non-linear noise-reduction. Brain surfaces were reconstructed and inflated from the MPRAGE images using the FreeSurfer (v5.1) toolkit ( http://surfer.nmr.mgh.harvard.edu/ ) as described previously (Dale et al.,  1999 ; Fischl and Dale,  2000 ; Salat et al.,  2004 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25706380">PMID:25706380</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed by Freesurfer 5.3 image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications, we made no changes to this pipeline. Briefly, image processing includes motion correction [ 28 ], removal of non-brain tissue using a hybrid watershed/surface deformation procedure [ 29 ], automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles) [ 30 ] intensity normalization, tessellation of the gray matter white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class [ 31 , 32 ]. Once the cortical models were completed, Freesurfer performed a number of deformable procedures for in further data processing and analysis. Steps included surface inflation [ 33 ], registration to a spherical atlas which utilized individual cortical folding patterns to match cortical geometry across subjects [ 34 ], parcellation of the cerebral cortex into units based on gyral and sulcal structure [ 30 ], and creation of a variety of surface based data including maps of curvature and sulcal depth. Segmentation and cortical models were checked and corrected manually on each subject, however correction showed no significant changes to the results.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25704244">PMID:25704244</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We analysed differences in regional cortical thickness between participant groups using Freesurfer software (v.5.3.0;  http://surfer.nmr.mgh.harvard.edu/ ). This approach has been previously described in detail [Dale et al.,  1999 ; Fischl and Dale,  2000 ] and applied in many quantitative MRI studies [Fischl,  2012 ]. All images were processed using the standard Freesurfer  “ recon-all ”  processing stream, which supplies the surfaces and morphometry data for each subject. Regional cortical thickness measures are expressed as vertices that represent distances between the reconstructed gray-white matter boundary and pial surface across the cerebral cortex. After cortical reconstruction and registration to standard space, each dataset was smoothed using a circularly symmetric Gausssian kernel across the cortical surface with a full width at half maximum of 10 mm. The analysis of cortical thickness maps was performed using the query design estimate contrast tool incorporated into Freesurfer software. Group differences were investigated by computing a general linear model of cortical thickness at each vertex of the cortical surface. For group comparisons, the model included cortical thickness as a dependant factor, diagnosis (left mTLE/right mTLE or ILAE I/ILAE II-VI, and controls) as an independent factor, and participant age as a nuisance variable. We also investigated the relationship between clinical variables and cortical thickness in patients using a regression analysis, including cortical thickness as a dependant factor, clinical data (preoperative duration of mTLE, age of onset of mTLE and preoperative frequency of seizures) as independent factors, and, where specified, patient age as a nuisance variable. Results were reported as statistically significant if they survived correction for multiple comparisons ( P ? < ? 0.05, False Discovery Rate).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25698916">PMID:25698916</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To minimize postural differences during image acquisition, the T 1 -weighted and FLASH images in each subject were co-registered using an 6-parameter affine transformation in Amira (Visage Imaging) with Lanczos resampling. The two registered FLASH images were summed then smoothed twice using a 3D median filter of kernel size 3 voxels to reduce noise. The filter was applied twice since the kernel size in 3D could not be increased beyond 3 voxels in Amira. The sum of the two co-registered T 1 -weighted images was divided by the filtered, average FLASH image. The T 1 -weighted images and the FLASH images have similar receive field (B ? ) profiles; hence their division produces a ratio image with greatly reduced B ?  inhomogeneity. The transmit field inhomogeneity (B + ) is also reduced in the ratio image (Van de Moortele et al.,  2009 ; Marques et al.,  2010 ). Each ratio image was then resampled to 1 mm isotropic resolution and Free Surfer ( http://surfer.nmr.mgh.harvard.edu/ ) (Fischl et al.,  2002 ) was used to extract labels for the cerebrum masked to the pial/CSF boundary. The labels were resampled back to 0.7 mm isotropic resolution to mask the cerebral hemispheres in the original 0.7 mm isotropic resolution ratio image. The subcortical gray matter structures including the basal ganglia and thalamus were manually removed from the masked ratio image, leaving only the cortex and underlying white matter tracts. The ratio image was then segmented into four brain- tissue classes, GM, mGM, WM and CSF (Figure  1 ). This tissue segmentation was performed with a fuzzy c-means clustering algorithm (Pham and Prince,  1999 ) in the software package: MIPAV (version 7.0.1,  http://mipav.cit.nih.gov/ ). After segmentation, the two hemispheres were separated for subsequent processing. The output of the fuzzy c-means segmentation was used to produce tissue labels for GM, mGM, and CSF using a fuzzy membership value of 0.5 for those tissue classes. The WM labels were produced using a fuzzy membership level of 0.1. This membership level was used to account for the very thin WM blades in the crowns of the gyrii where WM intensity is reduced due to partial voluming. Finally, in each tissue class, islands unconnected in 3D from the main cluster were removed. The post processing was performed in Amira, unless otherwise stated.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25695154">PMID:25695154</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were analyzed using AFNI ( Cox, 1996 ) ( http://afni.nimh.nih.gov/afni/ ), SUMA ( http://afni.nimh.nih.gov/afni/suma/ ), MATLAB (The MathWorks Inc., Natick, MA), and FreeSurfer ( Dale et al., 1999 ;  Fischl et al., 1999a ) ( http://surfer.nmr.mgh.harvard.edu/ ). Functional data were slice-time and motion corrected. Motion distance (estimated by AFNI's 3dvolreg) did not exceed 1.0 mm (relative to starting head position) in any of the six motion parameter estimates (three translation and three rotation) during any run for any subject. In preparation for correlation analyses, several additional steps were performed on the data: (1) removal of signal deviation  > 2.5 SDs from the mean (AFNI's 3dDespike); (2) temporal filtering retaining frequencies in the 0.01 – 0.1 Hz band; (3) linear and quadratic detrending; and (4) removal by linear regression of several sources of variance: (i) the six motion parameter estimates (three translation and three rotation) and their temporal derivatives, (ii) the signal from a ventricular region, and (iii) the signal from a white matter region. Removal of ventricular and white matter signal resulted in a general, broad decrease in the raw correlation values by about 0.2, though subsequent eccentricity-specific effects were slightly increased. These are standard preprocessing steps for resting-state correlation analyses (e.g.,  Vincent et al., 2007 ;  Yeo et al., 2011 ), though our results were not dependent on these preprocessing steps, and correlation analyses on the raw data yielded qualitatively and statistically similar results. Global mean signal (GMS) removal was not included in the analysis reported here given concerns about negative correlations ( Fox et al., 2009 ;  Murphy et al., 2009 ;  Saad et al., 2012 ), though inclusion of GMS removal yielded statistically similar results. To minimize the effect of any evoked response due to the scanner onset, the initial 21.6 s and 19.5 s were removed from each rest and movie scan, respectively. All voxels that fell between the gray and white matter boundaries were mapped to surface model units (nodes). Only for figure illustrations from single seed correlation analyses, data were spatially filtered using a Gaussian filter to a maximum smoothness of 4 mm full-width at half-max (FWHM) (by estimating the FWHM before spatial filtering), ensuring uniformity across the surface and maintaining spatial specificity while increasing the signal-to-noise ratio (SNR) ( Chung et al., 2005 ). No such spatial filtering was applied on data used for eccentricity bin or topographic model regression analyses. The timeseries from all surface nodes spanning early visual and extrastriate areas V1, V2, V3, hV4, VO1 – 2 (combined), V3A – B (combined), were extracted into MATLAB for correlation analyses. Eccentricity measurements were coarse for the surrounding visual cortex, so no additional extrastriate visual areas (e.g., LO1/2, TO1/2, PHC1/2, IPS0-5) were included in the analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25685711">PMID:25685711</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To ensure that volumetric differences between MC and PD participants were not impacting our results, the following analyses were conducted. MP-RAGE images were processed using FreeSurfer (version 5.3.0) ( http://surfer.nmr.mgh.harvard.edu ). Standard preprocessing of structural volumes produced reconstructions that were used to determine if there were any areas of cortical thinning or subcortical atrophy in the PD group compared to MC using the QDEC utility. We also compared striatal volume between MC and PD groups to determine if there were any volumetric differences in the seed region.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25679805">PMID:25679805</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 MP-RAGE sequence image processing was performed using the FreeSurfer image analysis suite (version 5.3,  http://surfer.nmr.mgh.harvard.edu/ ). To avoid postprocessing irregularities between workstations, all image post-processing was conducted using a single Mac workstation running OS X Lion 10.7.5 software. The methodology for this procedure is described in detail in prior papers [ 18 ,  19 ,  20 ,  21 ,  22 ,  23 ,  24 ,  25 ,  26 ,  27 ,  28 ,  29 ]. In short, processing includes skull stripping [ 29 ], automated Talairach transformation, segmentation of subcortical white matter and gray matter structures [ 22 ,  26 ] intensity normalization [ 30 ], tessellation of brain boundaries, automated topology correction [ 29 ], and surface deformation [ 18 ,  19 ,  20 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25670955">PMID:25670955</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the data obtained from the ADNI DB ( http://www.loni.usc.edu/ADNI ). The ADNI MRIs analyzed here were the baseline, screening, or follow-up scans at the entry to the study. They were acquired at multiple sites using the GE Health Care (Buckinghamshire, England), Siemens Medical Solutions USA (Atlanta, Georgia), or Philips Electronics 1.5 T (Philips Electronics North America; Sunnyvale, California) system. 22  Two high-resolution T1-weighted volumetric-magnetization-prepared 180 °  radiofrequency pulses and rapid gradient-echo scans were collected from each study participant and normalized for intensity inhomogeneities, non-brain tissue was removed, and subcortical white matter and deep gray matter volumetric structures were segmented. 23 , 24  The raw Digital Imaging and Communications in Medicine images were downloaded from the public ADNI site ( http://www.loni.usc.edu/ADNI/Data/index.shtml ). All the MRIs were processed according to previously published methods 25  using the FreeSurfer version 4.1.0 software package ( http://surfer.nmr.mgh.harvard.edu ). The intensity gradients were followed outward from the white matter surface to find the gray matter surface (gray-cerebrospinal fluid boundary). 26 , 27</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25670951">PMID:25670951</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Whole-brain structural analyses of cerebral cortical thickness and subcortical volume based on T1WI were performed using FreeSurfer v.5.1 ( http://surfer.nmr.mgh.harvard.edu/ ). 30 , 31  As a fully automated image-processing tool for macroscopically visible brain structures, FreeSurfer constructs models of brain tissue surfaces and subcortical areas, and is useful to measure cerebral cortical thickness and subcortical volumes. Briefly, the images are processed with a head motion correction, a non-uniform intensity correction for MR acquisition artifacts, affine transformation to the MNI 305 template, further intensity normalization across brain tissues, and stripping of non-brain areas. The preprocessed brain image was non-linearly transformed to align with Gaussian classifier atlas model, which was designated with the voxel labels based on neuroanatomical probabilistic information estimated from a large training dataset. Images were segmented into cortical and subcortical structures using iterative methods to compute anatomical probabilities by combining intensity distribution with spatial relationships between subject and probabilistic atlas for each voxel until no any changes are detected in two consecutive iterations. Brain volume per hemisphere was initially covered with triangular tessellation to construct surface models across the different tissue types and to define the surface vertices. The surface defects from previous segmentation steps are topologically refined for accurate boundary connectivity, and then the intensity gradient determined WM surfaces as well as pial surfaces. Local cortical thickness was measured as the average distance between vertices corresponding to the brain anatomy of these two surface models. The cortical thickness and subcortical volume normalized with intracranial volume were compared using multivariate analysis of covariance (MANCOVA) with age entered as a covariate followed by Bonferroni post hoc test for feature selection of the classification.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25667864">PMID:25667864</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The structures in the medial temporal region of the right side appeared normal on the MRI, and indeed, measurements of right hippocampal volume, mean diffusivity, and fractional anisotrophy of the uncinate and stria of the fornix (using Freesurfer software;  http://surfer.nmr.mgh.harvard.edu/ )  [20]  showed results within plus or minus one standard deviation of healthy subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25615578">PMID:25615578</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical images were preprocessed using the FreeSurfer imaging analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ; version 5.3). FreeSurfer is a semi-automatic software package that performs volumetric segmentation of subcortical structures [ 19 – 21 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25613079">PMID:25613079</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>With the animal ’ s own anatomical scans serving as intermediates, the results of these analyses were registered to a standard macaque template brain 60 , which is registered to a macaque atlas in stereotactic coordinates 24  as well as a digital atlas developed from it 61 . This allowed us to determine the anatomical areas within which the significant activity clusters occurred. The results were also registered to a FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) surface-based representation of the standard template monkey brain 60 . Anatomically defined ROIs in the anterior auditory cortex (rostral core/belt fields) or OFC were defined in reference to the macaque brain atlas 24 61 . For statistically testing the voxel-based responses in the ROIs, the ROIs were registered back to the animals ’  own functional imaging data. This helped avoid overinflating the number of voxels used for analysis (which would happen if instead we had registered the functional scans to the higher resolution anatomical space and used this for the ROI analyses). These ROI data were also analysed with the animals ’  data combined using a mixed-effects ANOVA with a random between-subjects factor of  ‘ monkey ’  and a fixed within-subjects factor of 'stimulation site ’ . Hemisphere was also added to these models as needed (see Results for further details).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610793">PMID:25610793</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each T 1  weighted image was processed using the surface-based registration pipeline in the FreeSurfer (version 5.3) image analysis suite ( http://surfer.nmr.mgh.harvard.edu /). This fully automated process has been previously described ( Dale et al., 1999 ;  Fischl and Dale, 2000 ), and briefly includes: Talairach transformation, motion correction, intensity normalization, removal of non-brain tissue, segmentation and tessellation of the grey and white matter boundary, automatic topology correction, and surface deformation. Given that there is a one-to-one correspondence of points on the white matter surface to the pial matter surface, the grey matter – white matter boundary was then used as an initial contour that deformed to detect the grey matter-pial matter boundary with subvoxel precision ( Fischl and Dale, 2000 ). This procedure has been validated against histological analysis ( Rosas et al., 2002 ) and manual measurements ( Salat et al., 2004 ). Reconstructed surfaces were visually evaluated for errors in Talairach registration, brain extraction, and segmentation, and minimal manual editing was performed when necessary.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610790">PMID:25610790</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumes of interest (VOI) for the thalamus, hippocampus, amygdala, and entorhinal cortex were created using the Freesurfer (5.1;  http://surfer.nmr.mgh.harvard.edu ) recon-all function. In this procedure image volumes are resampled from native T1 image space to 1   mm isotropic space; segmentations are generated in 1   mm space then mapped back onto native image space. These tools have been validated by reference to manual thalamus ( Keller et al., 2012 ) and hippocampus ( Pardoe et al., 2009 ) labeling in healthy subjects and epilepsy patients. Anterior and posterior hippocampal VOIs were created in individual patients by dividing the hippocampal VOI (described above) by a coronal plane at its anterior – posterior center. These anterior and posterior divisions were created to, in part, replicate the  Bettus et al. (2010)  study ( Fig.   1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610783">PMID:25610783</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>BOLD data preprocessing and analysis were performed using tools from the FMRIB Software Library (FSL) ( http://www.fmrib.ox.ac.uk/fsl ) ( Smith et al., 2004 ;  Woolrich et al., 2009 ), the FreeSurfer (v. 5.2) suite ( http://surfer.nmr.mgh.harvard.edu /) ( Dale et al., 1999 ;  Fischl et al., 1999 ;  Fischl et al., 2002 ;  Fischl et al., 2004 ), AFNI ( Cox, 1996 ;  Cox and Hyde, 1997 ), and RETROICOR ( Glover et al., 2000 ) for retrospective correction of physiological motion artifacts using our peripheral measures of cardiac and respiratory activities as independent assessments of physiological noise in the BOLD signal.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610779">PMID:25610779</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer (5.1.0) software package ( http://surfer.nmr.mgh.harvard.edu ) was used to reconstruct cortical surfaces of each participant from the MRI scans. Main steps included (1) Talairach registration, (2) intensity normalization, (3) skull stripping, (4) white matter segmentation, (5) generation, refinement and tessellation of the white matter surface (i.e., the boundary between gray and white matter), (6) deformation of the white matter surface into the pial surface (i.e., the boundary between the gray matter and the cerebrospinal fluid) and (7) automatic correction of topological defects. Details of these steps are described elsewhere ( Dale et al., 1999 ;  Fischl et al., 1999 ,  2001 ). To ensure accuracy of the reconstruction, we also inspected and manually edited the reconstructed surfaces whenever necessary during the process. All inspection and editing were performed by one trained operator to avoid variability introduced by multiple raters.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610761">PMID:25610761</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All 3D MP-RAGE images were also transferred to the EVAL and were corrected for lack of homogeneity in intensity using in-house software. All images were automatically processed by software scripts in exactly the same way on an Apple ®  Mac Pro (OS X 10.7, 32   GB, 2   ×   2.93   GHz 6-Core Intel Xeon, Apple ®  Inc., Cupertino, CA, USA) running FreeSurfer software (Version 5.1;  http://surfer.nmr.mgh.harvard.edu/ ). For details of this method see  Dale et al. (1999) ,  Fischl (2012) ,  Fischl and Dale (2000)  and  Fischl et al. (1999) . Briefly, cerebral WM was initially segmented, divided into two hemispheres, and the brainstem and cerebellum were removed. A triangle-based mesh of the WM surface is produced through tessellation, which is subsequently deformed outward to generate the pial surface before automatic correction of topological defects in the resulting main fold. Cortical thickness mean and Gaussian intrinsic curvature measurements were obtained at each vertex by reconstructing representations of the GM – WM boundary and the pial surface. The entire cortex of each patient and control was visually inspected, but no  relevant  errors in cortical labeling could be detected that had to be manually corrected. Thus the whole quantitative brain analysis was completely objective and could not be biased by any subjective factor. Local cortical thickness, rectified extrinsic and Gaussian curvature had been averaged over both cerebral hemispheres using software tools incorporated into FreeSurfer software for each examined individual. Additionally, we calculated the individual total cortical surface area. From the segmented data, we used the cortical WMV and the cortex volume as calculated by FreeSurfer. By summarizing the volumes of WM hypointensities as detected by FreeSurfer in the 3D T1w images we assessed the individual WM  “ lesion load ” .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25584461">PMID:25584461</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The anterior HF (HF head), posterior HF (HF body and tail), PRC, and PHC were manually segmented on the MPRAGE coronal plane according to previously published guidelines ( Insausti et al., 1998 ;  Franko et al., 2014 ). In brief, the most posterior slice of the anterior HF was defined as the last slice containing the gyrus intralimbicus; the posterior HF immediately followed. The anterior extent of the PRC was defined as 2 mm anterior to the limen insula or the most anterior slice in which the collateral sulcus was visible, whichever was more anterior. The most posterior slice of the PRC was defined as 4 mm posterior to the anterior/posterior HF transition. The PHC immediately followed the PRC, and the posterior extent was defined as 2 mm posterior to the appearance of the posterior crus of the fornix. The PRC segmentation included the entire lateral bank and dorsal half of the medial bank of the collateral sulcus. The PHC segmentation included the medial bank of the collateral sulcus, extending to the most medial aspect of the parahippocampal gyrus. Some analyses also included a set of regions outside of the medial temporal lobes, including the retrosplenial cortex, posterior cingulate, precuneus, angular gyrus, and medial prefrontal cortex. These ROIs were labeled with FreeSurfer cortical parcellation tools ( http://surfer.nmr.mgh.harvard.edu/ ) using the Destrieux atlas ( Destrieux et al., 2010 ) for the following labels: G_cingul-Post-ventral, G_cingul-Post-dorsal, G_precuneus, G_pariet_inf-Angular, and S_suborbital. Segmented brains were co-registered to the mean functional image and split into masks for each ROI (see  Figure 1B  for an example set of MTL ROIs). Masks were filtered to exclude voxels with low signal, defined as having mean temporal SNR (calculated across all functional runs) more than 1 standard deviation below the ROI mean. For visualization in standard space, ROI masks were warped to MNI space and combined across subjects into probabilistic maps.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25566178">PMID:25566178</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The processes of coregistration of the electrode positions to the MRI voxels started with a thin section head CT following implantation of electrodes. Then using in-house techniques, all electrode contact positions were identified and recorded ( 13 ). The CT scan was then registered to the anatomical scan (T1-weighted MPRAGE) obtained prior to surgery, and the positions of all electrodes translated into the MRI-space. All image registration was affine and performed using FSL FLIRT ( 14 ,  15 ). Segmentation was performed on the T1 MPRAGE images using the Freesurfer package ( http://surfer.nmr.mgh.harvard.edu/ ), whose parcelation maps were interpolated into the dMRI space.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25551572">PMID:25551572</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer ( http://surfer.nmr.mgh.harvard.edu ), a semiautomatic segmentation method, was used to extract bilateral rACC and bilateral mOFC regions. Then, FNIRT - FMRIB's Non-linear Image Registration Tool ( http://www.fmrib.ox.ac.uk/fsl/fnirt ) was used to co-register anatomical scans along with corresponding ROIs to DTI space. Additionally, the mOFC was divided into anterior (ant mOFC) and posterior (post mOFC) regions  [34] . The anterior border of the post mOFC was defined as the most anterior coronal slice that contained rACC.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25549088">PMID:25549088</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Motion and eddy current correction of diffusion weighted images (DWI) was performed using the Functional Magnetic Resonance Imaging of the Brain FMRIB software library version 4.1 (FSL, [ http://www.fmrib.ox.ac.uk/fsl ], Smith et al.  [29] ). The automated parcellation of T1-weighted images was performed in FreeSurfer (Athinoula A. Martinos Center for Biomedical Imaging, Harvard-MIT, Boston [ http://surfer.nmr.mgh.harvard.edu ]). Subsequently, T1-weighted images were co-registered to the first b0 and the T1-b0 transformation was also applied to atlas image using nearest neighbor interpolation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25514366">PMID:25514366</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was measured automatically using FreeSurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ,  [41] ,  [42] . Details of the surface-based cortical reconstruction and subcortical volumetric segmentation procedures have been extensively documented previously  [41] – [44] . Briefly, the FreeSurfer pipeline performs motion correction on the T1-images, automatically removes non-brain tissues  [44] , transforms volumetric data to a common atlas, performs intensity normalization and topology correction  [43] ,  [45]  and defines the boundaries of the gray/white matter and pial surface  [41] ,  [42] . For the purposes of the current study, automated image surfaces and segmentations were inspected and screened for quality control but were not manually edited, in order to maintain the objectivity of results. No scans were excluded due to poor quality or poor segmentations. Total intracranial volume (TIV) was determined by a validated automated method known to be equivalent to manual intracranial volume estimation  [46] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25505869">PMID:25505869</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Evaluation of structural volumes . Segmentation generated with FreeSurfer was used to derive volumes of the total intracranial volume and the volumes of five subcortical structures: the amygdala, hippocampus, caudate, putamen, and thalamus [FreeSurfer software ( www.surfer.nmr.mgh.harvard.edu ), according to standard procedure, Fischl,  2012 ]. Data on cortical thickness, which were also generated by the FreeSurfer program, partly from these populations, has been presented in a separate manuscript. The method for generation of subcortical volumes has been described in detail previously (Fischl et al.,  2002 ,  2004 ; Fischl,  2012 ; Savic and Arver,  2013 ). The segmentation masks were first registered to original grayscale images. When necessary, manual correction was carried out by a person educated in neuroanatomy (who had no information about the identity of the subjects).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25505670">PMID:25505670</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness and volumetric measurements were performed using the FreeSurfer software package version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ), and technical details of these procedures have been previously described (Dale et al.  1999 ; Fischl and Dale  2000 ; Fischl et al.  1999a ,  b ,  2001 ,  2002 ,  2004a ,  b ; Han et al.  2006 ; Jovicich et al.  2006 ; Segonne et al.  2004 ). In brief, this process involved the following: motion correction and averaging of two volumetric T1-weighted images (Reuter et al.  2010 ), removal of non-brain tissue (Segonne et al.  2004 ), alignment of scans to the standard Talairach space, segmentation of the deep grey matter volumetric structures (Fischl et al.  2002 ,  2004a ), intensity normalization (Sled et al.  1998 ), tessellation of the grey matter/white matter boundary, topology correction (Fischl et al.  2001 ; Segonne et al.  2007 ) and surface deformation to optimally place the grey/white and grey/cerebrospinal fluid borders (Dale et al.  1999 ; Dale and Sereno  1993 ; Fischl and Dale  2000 ). The subsequent cortical representations underwent surface inflation (Fischl et al.  1999a ), registration to a spherical atlas to align individual cortical folding patterns with group cortical geometry (Fischl et al.  1999b ), parcellation of the cerebral cortex into gyral and sulcal structures (Desikan et al.  2006 ; Fischl et al.  2004b ) and creation of cortical thickness statistical maps, calculated as the closest distance from the grey/white boundary to the grey/CSF boundary at each vertex on the tessellated surface (Fischl and Dale  2000 ). FreeSurfer's procedure for automated measurement of cortical thickness has been validated against histological analysis (Rosas et al.  2002 ) and manual measurements (Salat et al.  2004 ) including in schizophrenic patients (Kuperberg et al.  2003 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25482267">PMID:25482267</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To further characterize the time – frequency properties of the relationship between RT and oscillatory power, and to test for a relationship with phase, we reconstructed the MEG signal in multiple locations within the dorsal network. We wanted to avoid the circularity of defining virtual electrodes from the correlational beamformer analysis. We first tried to define functional localizers by contrasting post-stimulus and pre-stimulus time windows to reveal areas visually responsive or involved in saccade planning. However, probably because our task involved reflexive saccades to small peripheral visual targets, frontal and occipital contrasts were weak and not reliably present across individuals. In contrast, parietal activations were extended, making the selection of one peak for each individual difficult. We therefore used anatomically-driven locations. For primary visual cortex (V1), cortical reconstruction and volumetric segmentation for each participant were performed with the Freesurfer image analysis suite, which is documented and freely available ( http://surfer.nmr.mgh.harvard.edu/ ). The automatic extraction and quantification of V1 were performed using a previously validated morphometric-based heuristic approach ( Hinds et al., 2008 ). These mesh-based definitions of both left and right V1 were then split into 10 areas of equal length along an axis from central to peripheral vision in order to check for spatially selective effects ( Fig.   4 A). Parietal and frontal virtual electrodes were localized from the MNI coordinates provided in the literature ( Fig.   4 B): SEF [0  ?   9 70.5] ( Amiez and Petrides, 2009 ), left FEF [ ?   40 0 44], right FEF [28  ?   6 50], left IPS [ ?   30  ?   54 52] and right IPS [36  ?   58 58] ( Mort et al., 2003 ) and projected onto each individual. The center of each area was used as a virtual electrode (0 – 100   Hz bandpass) for the time – frequency ( Figs.   5 – 8 ) and phase ( Fig.   9 ) analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25469101">PMID:25469101</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used individual specific VOI derived from FreeSurfer 5.1 (Massachusetts General Hospital, Harvard Medical School;  http://surfer.nmr.mgh.harvard.edu ) as gold standard method to compare regional standardized uptake value (SUV) of each spatial normalization method. The T1-weighted MR images were resliced to 1 mm isovoxel space within 256  ×  256  ×  256 matrix. They were corrected for inhomogeneity, processed for skull-stripping, and segmented into gray and white matter. Cortical gray matter was parcellated with a probabilistic labelling algorithm by inflating the boundary between the gray and the white matter with overlaying curvature information on the inflated surface ( 11 ,  12 ). Subcortical structures were segmented and labeled by probabilistic registration technique ( 13 ). We obtained whole brain mask images with 112 parcellated regions after merging anatomically related regions to create VOI mask (FreeSurfer-generated VOI; FSVOI) which contained 7 cortical (frontal, parietal, occipital, temporal, medial temporal, cingulate, and insula cortices) and 4 subcortical regions (thalamus, caudate, putamen, and cerebellar cortex). Steps for applying FSVOI were illustrated in  Figure 2 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25463618">PMID:25463618</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical atrophy is frequently regionally specific in disease and the progress of atrophy can therefore reveal much about the evolution and causative factors of a disease. Few studies have used surface-based methods to assess cortical morphology in Parkinsonian syndromes, those that have, have focused on PD and identified widespread regions of cortical thinning, [15] , [16]  however these studies are not consistent with earlier reports of frontal atrophy. To our knowledge there have been no previous studies comparing patterns of cortical change in PD, MSA and PSP. Thus in this study we have chosen to measure cortical thickness, surface area and cortical volume using the automated surface-based analysis package FreeSurfer (Massachusetts General Hospital, Harvard Medical School;  http://surfer.nmr.mgh.harvard.edu).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25453257">PMID:25453257</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation of the hippocampus was performed using the  freesurfer  software package, version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ) (Fischl  et   al .,  2002 ). The procedure automatically assigns a neuroanatomical label to each voxel in an MRI volume based on probabilistic information automatically estimated from a manually labelled training set. All hippocampal volumes (left and right side averaged) from each subject were normalized by the subject ’ s intracranial volume (Westman  et   al .,  2013 ), which is estimated based on an affine transform in FreeSurfer. This segmentation approach has been used for multivariate classification of Alzheimer ’ s disease and healthy controls (Westman  et   al .,  2011 ) and biomarker discovery (Thambisetty  et   al .,  2010 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25437873">PMID:25437873</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical and subcortical parcellations provide the ROIs that are used as network nodes in CVU. In CVU, parcellations are displayed as nodes located on or near the cortical surface. It is possible to specify parcellations in three ways: as parcels of surface vertices, collections of voxels, or ROI coordinate centers described in a stereotaxic space. Parcellations are specified by specifying a  surface  file and several  parcellation  files depending on the method. Surface files store geometric information about the morphometry of the cortex, can be created using FreeSurfer, an open source imaging analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ), and can be specified in either the native FreeSurfer format or the more general GIFTI format. In order to display the parcellation on the surface, it is necessary (but not required to run the program) to additionally provide a 4 × 4 registration matrix in any matrix format between the parcellation volume and the anatomical surface space.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25432022">PMID:25432022</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Standardized uptake values (SUV; i.e., mean radioactivity/injected dose/weight) were computed voxelwise from the emission data collected 40 – 60 min post-injection. In order to quantitatively assess metabolic changes, cerebral metabolic rate of glucose (CMRglc) were computed from the dynamic PET frames with the non-invasive method proposed by  Wu (2008) , using the whole-brain as our reference region. SUV and CMRglc maps were then coregistered with the high resolution MRI scan using BBREGISTER tool ( Greve and Fischl, 2009 ) from the FreeSurfer suite ( http://surfer.nmr.mgh.harvard.edu/ ), normalized to the Montreal Neurological Institute (MNI) space using nonlinear registration (FNIRT, from the FSL suite; FMRIB's Software Library,  www.fmrib.ox.ac.uk/fsl/ ) ( Smith et al., 2004 ) and then smoothed with an 8 mm FWHM kernel. A paired, whole-brain, voxel wise, mixed effects analysis was conducted to compare SUV and CMRglc across visits (n = 10). Statistical parametric maps were thresholded using clusters determined by a voxel-wise threshold (Z  >  2.3) and a (corrected) cluster significance threshold of p = 0.05 ( Worsley, 2001 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25429257">PMID:25429257</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We estimate the location of cortical sources with the anatomically constrained minimum norm estimate (MNE; H ä m ä l ä inen and Ilmoniemi,  1994 ). MR structural images were obtained using a GRAPPA 3D MPRAGE sequence ( TR  = 2250 ms;  TE  = 2.99 ms; flip-angle = 9 ° ; acceleration factor = 2) on a 3 T Trio (Siemens, Erlangen, Germany) with 1 mm isotropic voxels. From the MRI data, a representation of each participant's cerebral cortex was constructed using the FreeSurfer program ( http://surfer.nmr.mgh.harvard.edu/ ). The forward model was calculated with a three-layer Boundary Element Model (BEM) using the outer surface of the scalp as well as the outer and inner surfaces of the skull identified in the anatomical MRI. This combination of MRI, MEG, and EEG data provides better source localization than MEG or EEG alone. The constructed cortical surface was decimated to yield approximately 12,000 vertices that were used as the locations of the dipoles. To perform group analysis, the cortical surfaces of individual subjects were inflated and aligned using a spherical morphing technique implemented by MNE (Gramfort et al.,  2014 ). Sensitivity to neural sources was improved by calculating a noise covariance matrix based on the 100 ms pre-stimulus period. The activations at each location of the cortical surface were estimated over 1 ms windows.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25428654">PMID:25428654</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Study MRIs were analyzed centrally. Lacunae, cerebral microbleeds (CMBs), and white matter hyperintensity of presumed vascular origin (WMH) were defined according to consensus recommendations. 13  The presence, number, and location of brain infarcts — including lacunar infarcts, large subcortical infarcts, and cortical infarcts — and CMBs were determined by either a neurologist (E.E.S.) or a radiologist (J.M.). An independent re-review of 30 cases by both raters showed good inter-rater reliability (kappa = 0.79). The volume of WMH was determined using Quantomo software (Cybertrials, Calgary, Canada). In this semiautomated threshold-based seed-growing technique, a user places a seed in each WMH and the boundary between WMH and normal tissue is determined by a computer algorithm. In separate validation work on 30 stroke patients, Quantomo-generated volumes closely approximated manual tracings with high inter-rater reliability (intraclass correlation coefficient = 0.98). Regional brain volume measurements, cortical thicknesses, and intracranial volumes were calculated using FreeSurfer v5.0 ( http://surfer.nmr.mgh.harvard.edu/ ). 14 , 15  FreeSurfer-generated output was reviewed for accuracy by the consensus of 2 raters, who excluded 25 of 803 cases (3.1%) because of inaccurate surface labels. To account for normal variation in head size, all volumes were normalized to the subject average intracranial volume (1,448.9cm 3 ). In the 25 cases with inaccurate FreeSurfer processing, the WMH volume was normalized using the sex-specific average intracranial volume. Prefrontal cortex volume was calculated as the sum of all frontal lobe regions parcellated by FreeSurfer, excluding the primary motor cortex. Basal ganglia volume was the sum of the thalamus, putamen, and caudate volumes. Average cortical thickness was calculated as the average of all 68 regions of interest generated by FreeSurfer, weighted by the amount of surface area occupied by each region.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25426935">PMID:25426935</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with the Freesurfer v. 5.1 image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). This processing was used to generate Estimated Total Intracranial Volume (eTIV) used for normalization. The technical details of these procedures are described in prior publications  [29] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25426062">PMID:25426062</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Still, reliance on newer investigative techniques will require an accumulation of many large studies in order to find certitude in our conclusions. In terms of DTI investigations, further knowledge of the anatomy of multi-orientational fiber populations may be necessary to increase accuracy; when axons are not oriented in a coherent fashion, so that then the voxel-averaged estimate of orientation cannot accurately summarize the orientation of the underlying fibers, continuity may be assumed between the fibers where there is none (Fernandez-Miranda et al.,  2008a ). Modern,  in vivo , human imaging studies of the claustrum still suffer from low statistical power due to small sample sizes. Number of diffusion directions, voxel size, magnetic field strength, and eddy current correction will all need to be carefully controlled to obtain accurate results in a structure so small. In the long-term, parcellation algorithms (like FreeSurfer reconstruction;  http://surfer.nmr.mgh.harvard.edu/ ) will need to become more precise if we hope to perform population-level analyses of small structures.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25426059">PMID:25426059</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The automated procedure for measuring TBV was performed with the volumetric and surface-based processing streams of the FreeSurfer software suite (version 5.0.0), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The procedure automatically assigns a neuroanatomical label to each voxel in a T1-weighted MRI scan, a label that is based on probabilistic information automatically estimated from a manually labeled training set (Fischl et al.,  2002 ). The technique has previously been shown to be comparable in accuracy to manual labeling (Fischl et al.,  2002 ,  2004 ). TBV is simply the sum of the total cortical white matter volume and the total cortical and subcortical gray matter volume (including cerebellar gray and white matter volumes) as computed by FreeSurfer per default.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25422929">PMID:25422929</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) was used for cortical reconstruction and volumetric segmentation. Processing of the T1-weighted images in Freesurfer included motion correction [ 51 ], removal of non-brain tissue using a hybrid watershe d/surface demonstration procedure [ 52 ], automated Talairach transformation, white matter segmentation of subcortical volumetric structures [ 53 , 54 ], intensity normalization [ 55 ], tessellation of the gray matter/white matter boundary, automated topology correction [ 56 , 57 ], and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class [ 58 , 59 , 60 ]. The cortex was parcellated into units based on gyral and sulcal structure [ 61 , 62 ], and estimates of gray and white matter volume were derived for the pre-specified regions of interest (ROIs) of the right and left transverse temporal gyri, superior temporal gyri, and pars orbitalis (see  Figure 1 ). Freesurfer morphometric procedures have shown good test-retest reliability across scanner manufacturers and across field strengths [ 63 , 64 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25422659">PMID:25422659</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI scans were processed using the FreeSurfer software package, available at  http://www.surfer.nmr.mgh.harvard.edu . Multiple MP-RAGE MRI acquisitions for each participant were motion corrected, averaged and normalized to create a single image volume with relatively high contrast to noise. This averaged volume was used to locate the grey/white matter boundary (white matter surface) and this, in turn, was then used to locate the grey/CSF boundary (grey matter surface).[ 36 ]</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25421402">PMID:25421402</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface extraction was performed on the averaged multi-echo T1w images using the default settings of the Freesurfer software (Freesurfer, Boston, MA, USA) 24  ( http://surfer.nmr.mgh.harvard.edu/ ). Individual images were examined for potential defects of surface reconstruction. Mean cortical thickness and cortex surface area were then computed for each subject.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25414852">PMID:25414852</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the 3D high-resolution anatomical MPRAGE data of each subject to obtain structural small-world networks based on cortical thickness measurements using Freesurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). Cortical parcellation was performed using the fully automated Freesurfer tool, and data was ascertained from the Destrieux cortical atlas [ 36 ]. Each brain was segmented to derive the boundary between the gray and the white matters, and the outer surface of the cortex (the pial surface) to represent the local fine curvatures. Bayesian maximum  a posteriori  algorithm with high flexibility was used to incorporate the observed individual surface geometry and the atlas function [ 37 ]. The thickness vector of total of 148 brain regions of interest (74 ROIs on each brain hemisphere from Freesurfer standardized landmarks) was exported from Freesurfer to derive the Pearson correlation matrix, which was used to generate small-world graphic topology. Small-world properties such as absolute clustering coefficient ( C p ), absolute shortest-path length ( L p ), relative clustering coefficient ( ? ), and relative path length ( ? ) were then computed and analyzed with in house MATLAB ( http://www.mathworks.com/ ) scripts and programs from matlab_bgl ( http://www.stanford.edu/~dgleich/programs/matlab_bgl ) in each group. These particular graph metrics were chosen because they have been used to study a variety of clinically important neurodegenerative and psychiatric disorders affecting the central nervous system [ 5 ,  30 ,  38 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25414635">PMID:25414635</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging analysis was performed using FreeSurfer (version 5.0,  http://surfer.nmr.mgh.harvard.edu ), an automated surface reconstruction and tissue classification and segmentation software. Each participant's MRI scan was analyzed in original space using the standard FreeSurfer analysis pipeline. Briefly, processing included removal of non-brain tissue by a hybrid watershed/surface deformation procedure (S é gonne et al.,  2004 ), subcortical structures were segmented (Fischl et al.,  2002 ), and further intensity normalization was conducted. This was followed by white-matter segmentation, tessellation of the gray-white matter boundary, and automated topology correction (Fischl et al.,  2001 ). Then surface deformation following intensity gradients optimally places the gray/white and gray/cerebrospinal fluid (CSF) borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Fischl et al.,  2001 ). Once the cortical models were complete, additional data processing and analysis were performed with deformable procedures, including parcellation of the cerebral cortex into 34 (Desikan et al.,  2006 ) or 74 (Destrieux et al.,  2010 ) conventional gyral- and sulcal-based neuroanatomical regions in each hemisphere. In automatic subcortical segmentation, each voxel in the normalized brain volume was assigned one of 37 labels. The  ICV  was estimated using the method described by Buckner et al. ( 2004 ). Previous studies have shown that FreeSurfer-derived volumetric measures of  ICV  (Buckner et al.,  2004 ; Jovicich et al.,  2009 ), cortical structures (Wonderlick et al.,  2009 ) and subcortical structures (Jovicich et al.,  2009 ; Wonderlick et al.,  2009 ) all have high reliabilities.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25407566">PMID:25407566</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Retinotopic mapping was acquired using gradient echo EPI sequences (TR = 2 sec, TE = 30 msec, Flip Angle = 90 deg) for measurement of BOLD contrast. Thirty-three contiguous slices (3  ×  3  ×  3.5 mm 3 ) oriented parallel to the AC-PC plane were acquired to cover the entire brain. The slices were positioned using an automatic slice positioning technique 59 . Data were analyzed with the FS-FAST and FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). All functional images were motion corrected 60 , spatially smoothed with a Gaussian kernel of 5.0 mm (FWHM), and normalized individually across scans.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25405994">PMID:25405994</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To create seeding targets for reconstructing sAF, cortical gyri demarcated by the Desikan – Killiany atlas  [34]  were extracted using the FreeSurfer software package version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) as previously described  [35] . These seeding regions were linearly registered to the IIT/ICBM standard space using FLIRT  [36] ,  [37]  with 12 DOF, mutual information cost function and spline interpolation. This process was then followed by a non-linear registration using FNIRT using the T1_2_MNI152_2mm configurations provided within FSL 5.0, but using the IIT Human Brain Atlas as the reference image and mask.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25401177">PMID:25401177</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI images were segmented into  n  = 194 cortical and subcortical regions, including white matter cerebrospinal fluid (CSF) compartments, using F reesurfer  ( http://surfer.nmr.mgh.harvard.edu/ ), according to the Destrieux anatomical atlas [ 57 ]. In order to extract mean-functional time courses from the BOLD fMRI, segmented T1 images were registered to the middle volume of the motion-corrected fMRI data, using boundary-based registration [ 58 ], once in functional space mean time-courses were extracted for each of the  n  = 194 regions in native fMRI space.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25386919">PMID:25386919</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>First, the Markov atlas was transformed from the PALS surface to the fsaverage surface (FreeSurfer average) via the deformation maps provided by the Surface Management System Database and WebCaret Online Visualization ( http://sumsdb.wustl.edu/sums/index.jsp ). Then, each subject ’ s T1-weighted MP-RAGE underwent the  ‘ recon-all ’  segmentation procedure from the Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ). One of the many outputs is a gray matter surface and gray matter volume, specific to each subjects ’  anatomy. Using Freesurfer surface to surface registration tools, each segmented grey matter volume was converted to a 3D surface and registered to the fsaverage surface. These mappings were then used to register the Markov parcellation atlas to each individual ’ s surface anatomy. The resulting subject defined regions were transformed from each subject ’ s individual gray matter surface to 1   volume space using a gray matter mask to assign the voxels. The individualized regions were then transformed from 1  native space to 3   Talaraich space to match the atlas registered volume data for each individual. Regions on the order of 1   or less could not be accurately accounted for in a 3   isotropic volume in all subjects, likely due to individual variability in cortical folding and gray matter thickness. Thus, a total of seven subjects were excluded as they had one or more regions fail this transformation step. The resulting regions, individualized to each subject ’ s gray matter, were used to create time courses from the processed BOLD data (see  Figure 1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25383319">PMID:25383319</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) software package was used to segment subcortical structures and generate volumes for these structures. Each voxel is assigned a neuroanatomical label based on probabilistic information from an atlas ( Fischl et al., 2002 ). FreeSurfer measures of right and left thalamic volumes were collected from each subject, as was intracranial volume (ICV) to control for differences in head size. The statistical software package SPSS was used to compare volumetric differences between people with IGE and normal controls. Thalamic volume was modeled as the dependent variable using multiple linear regression and ICV and subject status (i.e., IGE or normal control) were used as independent variables.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25379443">PMID:25379443</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness analysis was performed using FreeSurfer software package (v5.10) ( http://surfer.nmr.mgh.harvard.edu ). Standard preprocessing of the structural images was performed including intensity normalization, Talairach transformation, skull stripping, white matter segmentation, and tessellation and inflation of the surface ( Dale et al., 1999 ). The pial surface of each hemisphere was computed by using a deformable surface algorithm to extract white matter surface outward toward the gray matter boundary. After automatic correction of topological defects ( Fischl et al., 2001 ), each individual image was visually checked for labeling inaccuracies in white matter and pial surface. In case of defective labels they were manually corrected and reexamined before surface generation. The distance between the white matter surface and pial surface yields an estimate of cortical thickness at each vertex ( Fischl and Dale, 2000 ). For further analysis the data was transformed into a common template with a smoothing kernel of 10   mm which is in line with most cortical thickness studies. Higher smoothing kernels (20   mm and above) as suggested by other groups ( Lerch and Evans, 2005 ) were not used to minimize artificial extension of cortical thinning patterns ( Bernal-Rusiel et al., 2010 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25379423">PMID:25379423</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Next, the raw 3D T1 MRI data underwent processing for surface-based cortex reconstruction and volumetric segmentation using the Freesurfer image analysis software ( http://surfer.nmr.mgh.harvard.edu/ ) version 5.1 installed on a CentOS4 x86_64 cluster. There are several rationales for using Freesurfer in our study. Firstly, the surface-based registration approach incorporated into this software has been shown to have better reproducibility compared to Laplacian- or Registration-based methods for cortical thickness estimation ( Clarkson et al., 2011 ). Secondly, this framework provides a range of different kinds of surface-based and volumetric measurements, as well as different parcellation atlases for extracting averaged morphometric data. The steps of this processing are described in detail elsewhere ( S é gonne et al., 2007 ;  S é gonne et al., 2004 ;  Fischl and Dale, 2000 ;  Fischl et al., 1999 ;  Dale et al., 1999 ;  Sled et al., 1998 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25375933">PMID:25375933</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution T1-weighted and diffusion-weighted images (DWIs) were obtained with a whole-body 3-Tesla MR system (Achieva 3.0 TX; Philips Medical Systems, Best, The Netherlands) with an eight-channel phased-array head coil. T1-weighted images were used to define anatomically correct regions of interests based on cortical and subcortical structures as described below. A combination of T1-weighted images and DWIs is therefore advantageous to delineate specific fibers within a neural circuitry. The scanning parameters of the T1-weighted three-dimensional magnetization-prepared rapid gradient-echo (3D-MPRAGE) sequences were as follows: flip angle, 10 degrees; acquisition matrix, 256 × 256 × 170; field of view, 25.6 cm; section thickness, 1.0 mm; voxel size, 1.0 mm × 1.0 mm × 1.0 mm; TR, 7.1 ms; and TE, 3.3 ms. The 3D-MPRAGE images were preprocessed using the FreeSurfer software package version 5.2.0 ( http://surfer.nmr.mgh.harvard.edu ).  [34] ,  [35]  In brief, the processing stream included affine transformation of each subject ’ s native brain to the MNI305 atlas, removal of non-brain tissue, volumetric subcortical labeling  [36] , and surface based segmentation of grey matter and white matter tissue  [37] , [38] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25368547">PMID:25368547</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Using the anatomical MR scans, a cortical source space was constructed using dipoles with 7 mm spacing, yielding roughly 3500 dipoles per hemisphere normal to the cortical surface located along the gray/white matter boundary segmented from the MRI (Dale et al.,  1999 ) using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Note that this is fewer than the 20,484 dipoles on the average brain template, ensuring that upsampling to the common template would not lose spatial information. Combining this structural information with subject colocation information, a three-compartment boundary element model (BEM) was used to provide an accurate calculation of the forward solution (Mosher et al.,  1999 ) mapping dipole currents in the brain (source space) to observable M/EEG signals (sensor space).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25365802">PMID:25365802</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Using FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu ) (RRID:nif-0000-00304) and the Lausanne 2008 parcellation provided by the Connectome Mapper ( www.connectomics.org ) (RRID:nlx_153920), cortical surface representations of MPRAGE volumes were divided into 1000 regions of interest (ROIs) that spanned the entire cortex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25360100">PMID:25360100</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25343503">PMID:25343503</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For building the cortically constrained source space, we used FreeSurfer 5.3  [50]  ( http://surfer.nmr.mgh.harvard.edu/ ) to determine the shapes of the surfaces separating the scalp, skull, and brain compartments, and to build the triangular cortical surface mesh from T1-weighted anatomical 3D-volume MRI data by exploiting the gray-white matter boundary generated by the software. The FreeSurfer software was used due to the following advantages  [50] – [52] : first, in addition to folded surfaces, it also computes inflated and flattened representations of the cortex, and can thus expose the parts of the cortex embedded in the sulci. Further, it provides an automated parcellation of the cortex that can be used in inquiring source waveforms in specific ROI. Finally, an additional benefit of the surface-based analysis is that cortical surfaces can be aligned across individuals for the computation of group statistics using a morphing procedure in a spherical coordinate system  [50]  and several recent studies have shown that the method has the highest stability in gray matter volume estimate compared to two other volume-based methods  [53] , and better alignments of cortical landmarks than volume-based registrations  [54] . To achieve sufficient anatomical detail for the visualization of the folded cortical mantle, the triangular tessellations of each hemisphere contain roughly 100,000 potential vertices, spaced at  ? 1 mm. For inverse computations, the dense triangulations were decimated to  ? 1,000 vertices per hemisphere for computational efficiency.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25309433">PMID:25309433</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu , version 5.3.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25285560">PMID:25285560</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 weighted structural MRI scans were acquired on a separate day using an 8-channel phase array head coil in a 3T scanner (Siemens-Trio, Erlagen, Germany). Parameters of the sequence were: distance factor: 50%; slices per slab: 128; FOV: 256; FOV phase: 100; slice thickness: 1.33 mm, TR: 2530 ms, TE: 3.39 ms. Freesurfer software ( http://surfer.nmr.mgh.harvard.edu )  [32] – [35]  was used for cortical reconstruction and volumetric segmentation of the T1 weighed whole brain images for each subject. The individual brain scans were motion corrected, spatially co-registered by morphing into the Freesurfer average brain through spherical surface mapping  [36]  and spatially smoothed with a 5 mm FWHM (Full with at Half Maximum) kernel.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25275322">PMID:25275322</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and volumetric segmentation was performed using the FreeSurfer software, version 5.1.0 (Athinoula A. Martinos Center for Biomedical Imaging;  http://surfer.nmr.mgh.harvard.edu/ ). The technical details of this procedure have been described in previous publications  [15] ,  [16] ,  [17] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25273998">PMID:25273998</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Manual tracing of hippocampus was performed on the raw DICOM images of patients and controls ( n  = 17) using the software PMOD V2.7 (PMOD Technologies Ltd,  http://www.pmod.com ). The structural magnetic resonance images had a voxel size of 1 mm 3 . Two raters delineated the border of the hippocampus manually on each coronal 1 mm slice. Anatomical criteria were based on  Konrad  et al.  (2009) . Left and right hippocampal volumes were highly correlated between raters ( r left  = 0.897,  r right  = 0.929) and did not significantly differ in size [volume left :  t (16) = 1.002,  P  = 0.331; volume right :  t (16) = 1.110,  P  = 0.283]. Rater-averaged left and right hippocampal volumes were corrected for total intracranial volume using the covariance approach ( Free  et al. , 1995 ;  Buckner  et al. , 2004 ). Total intracranial volume was computed based on participants ’  structural images using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25258037">PMID:25258037</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated cortical surface reconstruction was performed on the T1 MPRAGE scans in FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu ) using previously described methods 53 54 55 . Freesurfer's Longitudinal Processing Stream 56  was used to analyze the cortical changes between groups. The longitudinal processing stream creates an unbiased within-subject template space and image 57  using robust, inverse consistent registration 58 . The template is used as an estimate to initialize subsequent segmentation processes for each time point, providing within-subject common information regarding anatomical structures. Several processing steps, such as skull stripping, Talaraich transforms, atlas registration as well as spherical surface maps and parcellations are then initialized with common information from this template, something which significantly increases reliability and statistical power 56 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25251441">PMID:25251441</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and thickness measurements were performed using the FreeSurfer toolkit ( http://surfer.nmr.mgh.harvard.edu/ ). Briefly, the structural image data of each participant were motion-corrected and the volume data were obtained. Cerebral GM/WM was then segmented and the GM/WM boundaries were estimated  [21] . Topological defects at the GM/WM boundaries were corrected  [22] . The boundaries were then used in a deformable surface algorithm designed to find the pial surface with submillimeter precision  [23] . Cortical thickness measurements were obtained by calculating the distance between the GM/WM boundary and pial surface at each of approximately 160,000 points (per hemisphere) across the cortical mantle  [23] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25226172">PMID:25226172</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the FreeSurfer 5.3 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described elsewhere  [31] - [33] . Briefly, this processing includes motion correction  [34] , removal of non-brain tissue  [32] ,  [35] , automated Talairach transformation, segmentation of the subcortical white matter and deep grey matter volumetric structures  [33] ,  [35] , intensity normalization  [36] , automated topology correction  [37] . Parcellation of the cerebral cortex into units was performed based on gyral and sulcal structure  [38] . FreeSurfer morphometric procedures have been demonstrated to show good test-retest reliability across scanner manufacturers and across field strengths  [39] . All FreeSurfer outputs were manually reviewed by a trained researcher who was blinded to the patient data (A. L.) and manual edits were performed where needed to improve the white and pial surface reconstruction.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25224588">PMID:25224588</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T 1 -weighted MPRAGE scans were collected on a Siemens 3T TIM TRIO imaging system (Siemens Medical Systems, Malvern, Pennsylvania) with 1 mm  ×  1 mm  ×  1 mm resolution. The MRI scans were processed with the Freesurfer (FS) image analysis suite release 4.1.0 69  ( http://surfer.nmr.mgh.harvard.edu ), which produced surface tessellations at the gray/white matter interface and the gray/CSF interface. FS provided 72 cortical regions of interest (ROI) based on a standard parcellation atlas 70 , along with measures of cortical gray matter thickness (mm) for each of these regions using embedded FS algorithms.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25188448">PMID:25188448</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer: The T1 MPRAGE structural MR images were analyzed using the FreeSurfer image analysis suite, which is documented and freely available for download online at:  http://surfer.nmr.mgh.harvard.edu . FreeSurfer is a surface-based structural MRI analysis tool that segments white matter and tessellates both gray and white matter surfaces  [22] . Previous publications have provided detailed descriptions and validation of the software  [23] ,  [24] . After initial automated segmentation through FreeSurfer version 5.1, each subject ’ s image was individually checked for quality and accuracy of segmentation. Inaccuracies in grey and white matter segmentation, as well as inaccurate pial boundaries and residual skull fragments were manually corrected with FreeSurfer ’ s built-in editing software. These cases were then reprocessed to take edits into account and final volumes were recalculated. Intracranial volume (ICV) was calculated based on FreeSurfer ’ s own eTIV (estimated total intracranial volume) metric, which uses atlas normalization as well as the relationship between the linear transform to MNI305 space and ICV.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25166875">PMID:25166875</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To identify activations at the group-level related to attention strength and subjective-sensation for somatosensory experiences, a subjective-sensation analysis using FLAME (FMRIB's Local Analysis of Mixed Effects) was conducted using subject's subjective sensations scores as covariates to model both attention and somatosensory experience regressors (see the Quantitative evaluation of the phenomenology questionnaire and rows  A  and   of  Table 1 ). All group Z statistical images were thresholded at Z > 2.3 (p < 0 .05) to define contiguous voxel clusters. The FSL cluster correction for multiple comparisons (Gaussian-random field theory based) was set at p < 0.05, whole brain correction ( http://www.fmrib.ox.ac.uk/fsl )  [29] . Because we did not find any frontal activation at this threshold as previously hypothesized (see  Introduction ), we additionally performed an exploratory whole-brain group-level analyses using an uncorrected p-value of p < 0.001 with a minimum cluster size threshold (k) of 15 voxels  [32] ,  [33] . This statistical threshold is in line with the recommendations for such complex and subtle cognitive processes, as used in previous social and affective neuroscience studies  [33] . Subsequently, except where indicated, and due to the documented importance of the frontopolar prefrontal cortex in the integration of multiple separate cognitive processes in the service of higher-order behavioral goals like self referential processes (i.e. mentalizing) and attention  [11] , we specifically explored this region using a small-volume-correction through a region of interest (ROI) approach. The frontopolar prefrontal ROIs were based on the peak activation of this exploratory whole-brain group-level analyses and the results reported in the meta-analysis in Gilbert et all 2006 that specifically relate to left frontopolar cortex activation either during attention  [34] – [37]  or during self referential processes (i.e. metalizing)  [11] – [13] . The ROIs were defined by merging individually created ROIs of 5 voxel (10 mm) diameter spheres ( ? 131 mm 3 ) around each of the documented peak coordinates and our own results in order to obtain oblong ROI volumes for a) Attention of k ? = ? 725 voxels (1450 mm 3 ) and b) Subjective Sensation of k ? = ? 500 voxels (1000 mm 3 ) covering the left frontopolar prefrontal cortex associated with these processes (ROIs were constructed in the 2 mm MNI-152 template). The statistical significance for the ROI analysis were corrected for multiple comparisons using the false discovery rate (FDR) correction as implemented in FSL  [38] . The FDR procedure ensures that on average no more than 5% of activated voxels for each contrast are expected to be false positives. The resulting peak voxel activation for either regressor was used to calculate the percent changes of BOLD signal in each subject using Featquery (part of FSL 4.1.5). These signal changes were then correlated with the subject's individual specific subjective attention strength or mean somoatosensory qualia scores of the Lickert scale using Spearman's rank correlation coefficient (see rows  A  and   of  Table 1 ). Results were projected onto the surface representation of the MNI-152 template with the Freesurfer suite ( http://surfer.nmr.mgh.harvard.edu/ )  [39]  for visualization purposes.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25165824">PMID:25165824</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Three Tesla T 1  images were processed using FreeSurfer version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ), which produced surface-based data for each subject  [26] . FreeSurfer pipeline includes data resampling, standardized stereotaxic space normalization, intensity non-uniformity correction using N3, skull stripping, tessellation of the gray/white matter boundary, automated topology correction, and surface deformation to optimally delineate the gray/white matter and gray matter/cerebrospinal fluid boundaries  [27] ,  [28] . As previously reported, masks of white matter hyperintensities were obtained from FLAIR sequences and registered to 3D T1 images  [9] ,  [29] . Reconstructed datasets were always visually inspected for accuracy.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25141298">PMID:25141298</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For both studies, the imaging protocol included a high-resolution sagittal 3D T1-weighted Magnetization Prepared RApid Gradient Echo (MPRAGE) volume. Volumetric segmentation, cortical surface reconstruction and cortical parcellation, based on the FreeSurfer package (4.5.0,  http://surfer.nmr.mgh.harvard.edu/ ), were used to quantify baseline thicknesses and volumes of brain regions, as detailed previously  [21] . Sixty-eight cortical thickness measures (34 from each hemisphere) and 50 regional volumes were generated. Volumes of white matter hypointensities, optic chiasm, right and left vessel, and right and left choroid plexus were excluded. White matter hypointensities were excluded since most subjects were characterized by zero values. Volumetric measures were normalized by their intracranial volume while cortical thickness measures were not normalized  [22] . Right and left measures were averaged  [23] . In total this results in 57 measures to be used as input features for ordinal regression; 34 regional cortical thickness measures and 23 regional volumes ( Table S3 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25140149">PMID:25140149</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used all FreeSurfer data from 1.5 Tesla scans across ADNI-1, ADNI-GO, and ADNI-2 in our analyses by merging the publicly available FreeSurfer data for the two cohorts. Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite version 4.3 ( http://surfer.nmr.mgh.harvard.edu/ ; Dale et al.,  1999 ; Fischl et al.,  1999a , b ). FreeSurfer processing in ADNI has been described in detail elsewhere (Mormino et al.,  2009 ). An early version of the longitudinal image processing framework was used to process the sequential scans (Reuter et al.,  2012 ). We used the change in volume of the left inferior lateral ventricle as our primary outcome measurement and included a measurement of intracranial volume (ICV) as a covariate in all volumetric analyses; both of which were defined in FreeSurfer (Desikan et al.,  2006 ). Slopes of change in left ventricular volume over time were calculated in SAS 9.3 (SAS Institute Inc., Cary, NC) using mixed model regression (PROC MIXED) to leverage the longitudinal data available. A conventional mixed model was used that included the fixed effect of time (fraction of years from baseline) and the intercept, as well as a random effect for time and the intercept. In such a model, the assumption is that individual slopes are normally distributed with the fixed effect of time representing the mean, and the variance represented in the random effect. On average we had four MRI scans for each subject. More details on the longitudinal data are presented in Table  1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25126064">PMID:25126064</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In this study, we are especially interested in three early visual areas: V1, V2d, and V3d; we used retinotopic mapping techniques to identify their borders and make region of interest (ROI) masks to cover them. The scan for retinotopic mapping consisted of two 8 min runs, one with a clockwise and the other an anti-clockwise stimulus direction. Individual retinotopic maps were obtained from all nine subjects because of individual differences in the position and borders of early visual areas (Stensaas et al.,  1974 ). Retinotopic mapping stimuli were generated with scripts written by Schwarzkopf ( http://www.fil.ion.ucl.ac.uk/~sschwarz/retinotopy.html ) for MATLAB and Psychtoolbox-3 ( http://psychtoolbox.org/HomePage ) and using the same projector as the functional stimuli. A rotating wedge-shaped stimulus containing an expanding color ripple pattern was used (Figure  2A ). To make ROI masks for areas V1, V2d, and V3d in each subject, EPI images from the retinotopic sessions were transformed to phase space (phase maps) using phase-encoded retinotopic mapping techniques (Sereno et al.,  1995 ) and averaged with scripts written by Schwarzkopf ( http://www.fil.ion.ucl.ac.uk/~sschwarz/retinotopy.html ). These phase maps were then overlaid onto an inflated anatomical brain image which had been created individually for each subject using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). The boundaries of V1, V2d, and V3d were then manually delineated by identifying the representation of the meridians from the mirror reversals in the phase map (Figure  2B ). As our interest was focused on V1, V2d, and V3d we did not extend our retinotopic mapping to areas V3A and V3B.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25120447">PMID:25120447</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject, functional data from the mapping experiments were transformed to phase space, averaged across directions, and overlaid onto the inflated anatomical brain image, generated using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). The boundaries of the IPS areas, V1, V4, and V5 were delineated by identifying the representation of the vertical and horizontal meridians from the mirror reversals in the phase map. Mask images for these areas were made from these boundaries and average intensities from the concept experiment were evaluated using SPM-8. Retinotopic maps of V1, V4, V5, and IPS areas were calculated for four subjects. Analysis of concept selectivity across IPS areas was performed for all four subjects separately using their individual retinotopic maps. Analysis of concept selectivity within V4 and V5 was also calculated using the normalized and averaged retinotopic maps of the same four subjects. Although this method of normalization leads to a loss of precision, our intention was to define in approximate terms the locations of the V4 and V5, enabling use of these retinotopically defined masks on the normalized group data ( n  = 16). Three subjects were excluded because of poor alignment between functional data and mask images, leading to a total of thirteen subjects for the retinotopic analysis of V4 and V5.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25120423">PMID:25120423</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For analysis across participants (third level analysis), the high-resolution anatomical images were normalized in spherical standard space using FreeSurfer (release 5.1.0,  http://surfer.nmr.mgh.harvard.edu ). The anatomically normalized surfaces were rotated and projected to a two dimensional (2D) space separately for each hemisphere using equal area Mollweide projection (Python libraries matplotlib and basemap,  http://matplotlib.sourceforge.net ). This procedure was then applied separately for each subject to transform the results of the 3D second-level statistical analysis to 2D. Finally, the group analysis (FMRIB's local analysis of mixed effects,  N  = 17) was run on the flattened data. Z-statistic images were thresholded using clusters determined by  Z   >  2.3 and a (corrected) cluster significance threshold of  P   <  0.05 (using Gaussian random field theory).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25110675">PMID:25110675</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness estimation from 74 brain regions per hemisphere was performed for each participant using the Freesurfer 5.1 analysis suite with Destrieux et al.'s Atlas [ 24 ] (documented and freely available for download online,  http://surfer.nmr.mgh.harvard.edu/ ). The technical details of the procedures were described previously [ 30 – 32 ]. For processing, we used optimized intensity nonuniformity correction for 3 Tesla MRI scanners [ 33 ] and a process that included visual inspections and the manual correction of topological defects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25092428">PMID:25092428</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In terms of translating neuroimaging findings from the research arena to the clinic, it is important to consider the cost, image acquisition time, availability of imaging techniques in hospitals, and the skills needed to process and interpret data. For example PET imaging has been used with specific ligands to measure dopamine receptor availability in patients with schizophrenia ( Howes et al., 2011 ) and amyloid plaques in patients with Alzheimer ? s disease ( Jack et al., 2009 ). However these types of PET scans cost several thousand of Euros per patient, which may make it prohibitively expensive to use as a clinical tool. Diffusion tensor imaging has identified abnormalities in white matter tracts in patients with schizophrenia ( Kanaan et al., 2009 ). However, at present the acquisition time and MRI hardware required mean that it is not easy for this technique to be adopted in some clinical centres. Structural MRI has an advantage as these types of images are routinely collected in hospitals. The post-processing of the images is complex but it is conceivable that an automated analysis pipeline can be developed and with the continuing rise in computing power, results would be available in a comparable time to having the image examined by a radiologist, but with the advantage that the results would be quantitative, standardised and not susceptible to inter-rater variability. If research demonstrated a robust neuroimaging biomarker for diagnosis or treatment response for patients with psychosis, a number of regulatory approvals would be needed before utilising the biomarker in clinical trials and in clinical diagnosis. No neuroimaging biomarker has reached this stage in psychosis research, but in Alzheimer ? s disease, automatic quantification of hippocampal volume is currently being considered for clinical trials ( Jack et al., 2011 ). Approval by regulatory bodies such as the US Food and Drug Administration (FDA) and European Medicines Agency (EMA) would require comprehensive step-by-step validation and documentation of the technique (including the scanning protocol, data preprocessing and statistical analysis). Currently popular neuroimaging packages expressly advise against clinical applications in their software licence agreements (see  http://surfer.nmr.mgh.harvard.edu ). In terms of looking forward, developers of neuroimaging software should consider that their algorithms may need to be adapted in the future for clinical applications and to document and validate processing steps in preparation for this. As both clinical trials and fulfilling regulatory approval are costly, it is important to foster links with industrial partners who would be able invest in the development of neuroimaging biomarkers (e.g. IXICO;  www.ixico.com ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25086839">PMID:25086839</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements were made using FreeSurfer version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ). The detailed procedure has been described and validated in previous publications ( Dale et   al., 1999; Fischl and Dale, 2000 ). Two modifications to the standard FreeSurfer processing stream were made: a locally generated brain mask was used to improve skull stripping, and FreeSurfer ventricular segmentations were added to the white matter mask to improve cortical segmentation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25083902">PMID:25083902</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual white matter lesion masks were generated manually on the 3D T1-MPRAGE sequence using ITK-SNAP and the whole-brain lesion volume was obtained for each patient. The volumetric segmentation of supratentorial tissue and brainstem was performed on the T1-MPRAGE data sets using the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). In patients, lesion refilling was used to avoid misclassification of white matter lesions and all the images were checked for errors and edited manually when necessary after the automatic segmentation processes. The complete topography of supratentorial tissue included all brain except cerebellum and brainstem, and the volume of each region was calculated and normalized for the intracranial volume. The supratentorial brain volume was obtained separately for each hemisphere, and we also obtained the whole brainstem volume. Subsequently, the volume of lesions in each side of the supratentorial tissue and in the brainstem was defined incorporating the lesion masks.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25077698">PMID:25077698</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Clinical and demographic variables were compared using parametric (ANOVA or  t  tests) or non-parametric ( ? 2 ) tests where appropriate. Between-group differences in cortical thickness were estimated using the FreeSurfer query design estimate contrast (QDEC;  http://surfer.nmr.mgh.harvard.edu ) interface. General linear models were used to estimate thickness differences at each vertex of both hemispheres with a threshold of  p   <   0.001 to derive spatially precise and statistically robust clusters of significant vertices. For each cluster, the family-wise error was then corrected using the FreeSurfer Monte-Carlo approach as described previously (Hagler  et al. \n 2006 ). This procedure estimates the probability of observing a cluster of equal or greater spatial extent from a null distribution of cluster sizes generated from 10   000 simulations of random data. Statistical significance was then declared if the cluster probability was  p   <   0.05. The cortical label, Talairach coordinates, cluster area (mm 2 ), effect size and cluster  p  value were then tabulated.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25071515">PMID:25071515</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25071484">PMID:25071484</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25068110">PMID:25068110</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The processing/analysis procedures are depicted as a flow chart in  Supplementary Fig.   1 . The T1 images were processed in Freesurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu ). The labels cerebellum gray and white, brainstem, left and right diencephalon, left and right thalamus that are produced in the Freesurfer subcortical segmentation stream ( Fischl et al., 2002 ) were used to generate a brainstem mask for each subject that encompassed the brainstem, the cerebellum and the diencephalon including the thalami. This mask was used to generate a T1 brainstem image by extracting the corresponding regions from each subject ’ s gray scale image. The template building routine from DARTEL toolbox ( Ashburner et al., 2007 ) as implemented in SPM8 ( http://www.fil.ion.ucl.ac.uk/spm/ ) and running Matlab (version 8.1.0.604) was used to generate a brainstem template from the T1 brainstem images of the control group. Each subjects T1 brainstem image was warped onto this template using the high dimensional warping algorithm of the DARTEL toolbox and the Jacobian determinants calculated from the resulting transformation matrices. The resulting Jacobian determinant maps (JDM) were masked to suppress the background and corrected for differences of head size using the intracranial volume that had been calculated from the skull-stripped T2 images.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25068109">PMID:25068109</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Images in DICOM format were transferred to a Linux workstation for processing.   A custom, semi-automated processing stream combining MATLAB and C++ was used. Cerebral parcellations, subcortical volume segmentation, cortical thickness and gray – white matter contrast (GWMC) estimates were accomplished using FreeSurfer v5.1.0 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). Details of the reconstruction, which involves removal of non-brain tissue, Talairach transformation, sub-cortical and cortical segmentation, intensity normalization and topology correction to estimate gray – white matter and gray matter – CSF boundaries, are described in detail in past publications ( Dale et al., 1999; Fischl and Dale, 2000; Fischl et al., 1999a, 1999b, 2001, 2002, 2004a, 2004b; Han et al., 2006; Jovicich et al., 2006; Segonne et al., 2004 ). The cerebral cortex was parcellated into surface ROIs based on sulcal and gyral structure ( Desikan et al., 2006 ). Cortical thickness was defined as the shortest distance from the gray/white boundary to the gray/CSF boundary at each vertex on the tessellated surface ( Fischl and Dale, 2000 ). For GWMC, the cortical matter was sampled at 0.2   mm from the gray – white matter boundary in both directions and gray – white matter contrast was calculated as difference between T1 white matter intensity and gray matter intensity divided by their averaged intensity at each voxel. Consequently, the range for these values was zero to one, with larger numbers representing increased (i.e., better) GWMC. Frontal and temporal lobe ROIs were included in the statistical analysis (see  Fig.   1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25057467">PMID:25057467</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Prior to applying TBM, non-brain tissue (scalp and meninges) was removed from each image volume using brain masks created with FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). Any brain segmentation errors were corrected on a slice-by-slice basis. Image volumes were corrected for signal intensity and magnetic field inhomogeneity artifacts ( Sled and Pike, 1998 ). TBM workflows including procedures similar to those detailed in prior studies ( Chiang et al., 2007 ;  Gogtay et al., 2008 ;  Ho et al., 2010 ;  Hua et al., 2008 ;  Leow et al., 2006 ;  Lepore et al., 2010 ) were implemented in the LONI Pipeline environment ( Dinov et al., 2009 ;  Dinov et al., 2010 ). First, each preprocessed image volume was registered to a single template image using an affine 9-parameter registration to adjust for global brain scale and alignment. Transformation matrices were retained for subsequent processing steps allowing the data to be assessed without global scaling. Next, images from the control group were used to create a minimal deformation target (MDT) or average anatomical template. For this processing step, each individual volume was aligned to all other volumes using a mutual information (MI)-based inverse-consistent algorithm, followed by applying the inverse of the mean displacement field from all subjects to the MDT. Finally, image volumes from all subjects were each separately and similarly aligned to the MDT by nonlinearly deforming the anatomy of each individual image to the anatomical template while optimizing the regularity of the deformation by quantifying the inverse consistent symmetric Kullback – Leibler (KL)-distance between the MDT and the resulting deformation to allow for unbiased registration ( Hua et al., 2011 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25057466">PMID:25057466</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>fMRI analyses were conducted on each participant's inflated cortical surfaces reconstructed from the MP-RAGE scan using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) segmentation, surface reconstruction, and inflation algorithms ( Dale et al., 1999; Fischl et al., 1999 ). Functional and structural scans were spatially normalized to a template brain consisting of the averaged cortical surface of an independent sample of 40 adults (Buckner Laboratory, Washington University, St. Louis, MO) using FreeSurfer's surface-based spherical coordinate system that employs a non-rigid alignment algorithm that explicitly aligns cortical folding patterns and is relatively robust to inter-individual differences in the gyral and sulcal anatomy of the cingulate cortex. Cortical activation was localized using automated surface-based parcellation software ( Fischl et al., 2004 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25057465">PMID:25057465</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Prior to data preprocessing, anatomical MRI and DTI data were reviewed for image quality by an expert user with extensive neuroimaging knowledge and experience (JJ) who was blind to age, gender, and neuropsychological performance outcomes. Morphometric analyses were conducted in FreeSurfer v4.0.5 ( http://www.surfer.nmr.mgh.harvard.edu ) ( Fischl, 2012 ) using a 64-bit Linux computer. Skull-stripping and regional segmentation were executed using a fully automated command (e.g., recon-all  – all) for each T 1 -weighted image, yielding 3 classes of voxels: gray matter (GM), white matter (WM), and cerebrospinal fluid (CSF) ( Fischl et al., 2002 ;  Fischl et al., 2004a ;  Fischl et al., 2004b ). Subsequently, within FreeSurfer, a fully automated routine determined delimiting boundaries of deep GM structures (i.e., caudate, putamen, pallidum, thalamus, hippocampus, amygdala). The brains of people with SBM are dysmorphic and boundaries are not always reliably indicated by the FreeSurfer output. The output images from FreeSurfer's segmentation and classification algorithms were visually inspected and extensively manually edited by a single rater (JJ) with substantial expertise using FreeSurfer's software as well its predecessor, Cardviews ( Filipek et al., 1989 ). An example of finalized segmentation after manual editing is shown in  Fig.   1 . At the time of MRI acquisitions (collected between 2005 and 2010), FreeSurfer v4.0.5 (released June 2008) was the most recent version available for analyses. Although subsequent revisions of FreeSurfer were released, FreeSurfer v5.0.0 (released in 2010) was the first version that retained manual edits (i.e., persistent edits) and the older version of FreeSurfer was maintained to ensure consistency across study subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25045379">PMID:25045379</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Two distinct approaches were adopted for cortical parcellation in order to cluster the large CC fiber bundles. The first used a BA template, which is a cytoarchitectonic atlas-based cortical map that separates the cerebral cortex according to the cellular composition, whereby each BA is associated with a particular neuropsychological function. 23  The second approach is based on a sulcal-gyral segmentation of the cortical gray matter produced by a program called FreeSurfer (version 5.0,  http://surfer.nmr.mgh.harvard.edu/ ). 24  For the BA ROI approach, the BA template was projected onto the canonical 3D T1-weighted image of all participants, as in our previous study. 18  Our FreeSurfer ROI approach was similar to that of Park et al., 17  in which the FreeSurfer parcellation with the sulcus/gyrus distinction (aparc2009+aseg segmentation in FreeSurfer software) provides a very fine subdivision of the cortical gray matter. However, the resulting small area might be too sensitive, and possibly cause spatial mismatch between the structural image (T1 spoiled gradient-recalled echo) and the DTI. Therefore, we used another version of FreeSurfer parcellation that does not distinguish between sulci and gyri (aparc+ aseg segmentation in FreeSurfer software).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25018716">PMID:25018716</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated estimation of voxel-based cortical thickness uses gray matter, white matter and CSF tissue partitions created in the classification step of VBM to extract cortical gray matter boundaries (Hutton et al.,  2009 ). Subsequently, the voxel-based cortical thickness maps are registered to a common standardized space using the deformation fields applied for gray matter warping. The surface-based method for cortical thickness estimation implemented in Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) relies on atlas information to detect the boundaries of gray matter followed by projection of the thickness values on a surface mesh (Dale et al.,  1999 ; Fischl et al.,  1999 ). Estimation of cortical thickness is assumed to provide more straightforward interpretation of computational anatomy results but is strongly affected by spatial and temporal changes in the brain tissue properties underlying MRI contrast changes (Salat et al.,  2009 ; Lutti et al.,  2014 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25012224">PMID:25012224</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Details of the MRI methodology have previously been described ( Jack  et al. , 2008 ). Cross-sectional regional measures of brain volume for the hippocampus, entorhinal cortex, middle temporal gyrus, fusiform, ventricles and whole brain, as well as total intracranial volume, were calculated at baseline using FreeSurfer Version 4.3, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). All regional volumes were normalized by dividing by total intracranial volume for each subject.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25010186">PMID:25010186</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Analyses were conducted on each participant ’ s inflated cortical surfaces reconstructed from the MP-RAGE scan using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) segmentation, surface reconstruction, and inflation algorithms  [43] ,  [44] . Functional and structural scans were spatially normalized to a template brain consisting of the averaged cortical surface of an independent sample of 40 adults (Buckner laboratory, Washington University, St. Louis, MO) using Freesurfer ’ s surface-based spherical coordinate system, which employs a non-rigid alignment algorithm that explicitly aligns cortical folding patterns and is relatively robust to inter-individual differences in the gyral and sulcal anatomy of the cingulate cortex. Cortical activation was localized using automated surface-based parcellation software  [45] . To facilitate comparison with other studies, approximate Talairach coordinates were derived by mapping surface-based coordinates back to the original structural volume for each of the individuals whose brains were used to create the template brain, registering the volumes to the Montreal Neurological Institute (MNI305) atlas  [46]  and averaging the corresponding MNI305 coordinates. These coordinates were transformed to standard Talairach space ( http://imaging.mrc-cbu.cam.ac.uk/imaging/MniTalairach ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25003031">PMID:25003031</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed using the MPRAGE images with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ). Briefly, non-brain tissue was removed using a hybrid watershed/surface deformation procedure ( S é gonne et al., 2004 ), subcortical white matter and deep grey matter volumetric structures were segmented automatically ( Fischl et al., 2002, 2004b ), and intensity inhomogeneity of the images was corrected ( Sled et al., 1998 ). The cerebral cortex was parcellated into 34 units per hemisphere based on gyral and sulcal structures ( Desikan et al., 2006; Fischl et al., 2004a ). In addition to cortical regions, the left and right thalami, left and right cerebella, and brain stem were extracted. The accuracy of the cortical and subcortical parcellation was assessed visually. An axial slice was manually defined below the inferiormost slice on which the pons was visible to include only the most caudal part of the brain stem. Extracted cortical, subcortical and cerebellar structures are shown in  Fig.   1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24968356">PMID:24968356</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>EPI fMRI data were first pre-processed for motion correction, slice timing correction, and spatial smoothing (12 mm 3D full-width-half-maximum Gaussian filter) by using the FreeSufer software ( http://surfer.nmr.mgh.harvard.edu ). We used GLM to identify visual and motor cortices with the FIR bases described above. Estimated hemodynamic activity from each subject was spatially registered to a common surface coordinate system using FreeSufer. The BOLD signals were then averaged across subjects and the dynamic significance of the BOLD signal was calculated using the dSPM procedure described above.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24967157">PMID:24967157</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surfaces for CVR registration and CThk measurements were generated for each participant using a freely available automated procedure in Freesurfer (V 5.1.0,  http://surfer.nmr.mgh.harvard.edu/ ), described elsewhere ( Fischl and Dale, 2000 ). Briefly: 1) high-resolution T1-weighted images were used to remove non-brain structures ( S é gonne et al., 2004 ) and white matter was segmented, 2) gray matter/white matter boundary was covered with triangular tessellation and a deformation algorithm was applied to produce a final representation of the boundary between gray and white matter as well as the gray matter/CSF boundary, and 3) CThk was computed as the closest distance between gray/white and gray/CSF surfaces ( Fischl and Dale, 2000 ). Cortical thickness maps and surface-registered CVR maps were computed for each participant separately and later registered to a spherical atlas, applying a 10   mm FWHM smoothing kernel, to facilitate inter-subject comparison ( Fischl et al., 1999 ). FLAIR images were used in conjunction with in-house software, a fuzzy lesion extractor (FLEX) ( Gibson et al., 2010 ) that is designed to segment white matter hyperintensities (WMH).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24959141">PMID:24959141</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and analysis of participants' structural MRIs was run with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications (Fischl and Dale,  2000 ). Briefly, this processing includes motion correction and averaging of multiple volumetric T1 weighted images, removal of non-brain tissue, intensity normalization (Sled et al.,  1998 ), tessellation of the gray matter white matter boundary, automated topology correction (S é gonne et al.,  2007 ), and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Fischl and Dale,  2000 ). Once the cortical models were complete, reconstructed and registered individual cortical thickness maps were smoothed using a Gaussian kernel of 10 mm full-width half maximum and subsequently used in statistical analyses. In order to assess if cortical thickness could explain the differences in plasticity measured by MEPs we performed a correlation analysis between subjects' MEPs and the values of cortical thickness of a region of interest (ROI) (Figure  1B ). This particular ROI was drawn to encompass the hot spots of all subjects. The ROI was done firstly in standard space-brain and then was mapped for each subject in order to measure this region-specific for each individual.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24947491">PMID:24947491</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Pre-processing of both rest and task state data was performed using Freesurfer 4.5 ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in several publications [ 19 – 32 ]. Briefly, the first 4 volumes of each scan were discarded to allow for T1-equilibration effects. Pre-processing included motion correction, slice time correction, volume based intensity normalization and removal of non-brain tissue using a hybrid watershed/surface deformation procedure, segmentation of the subcortical white matter and deep gray matter volumetric structures along with automated topology correction. Normalization to the MNI 305 space was achieved by first registering the functional runs with a 6 DOF linear (affine) transformation matrix to map between the individual ’ s functional data and the Freesurfer structural/anatomical data. The functional data was then resampled to the MNI 305 brain with 2mm thickness and registered to the MNI 305 space using a 12 DOF transformation matrix. No additional spatial smoothing was applied in order to minimize artificial local spatial correlations in the whole-brain analysis [ 33 – 37 ]. Both rest and task state data sets were upsampled to a TR of 1 and band-pass filtered to allow for the timecourses to be compared directly. We were interested in comparing the resting state frequencies in both the data sets. Since there is higher frequency information available in the task signal compared to the rest-state signal, we applied the same filtering to both the upsampled rest-state and task signals, thus ensuring the same frequency content in both the data. For the resting state and gap-closure fMRI data, we used a band-pass temporal filtering to remove constant offsets and linear trends over each run while retaining frequencies between 0.009 Hz <  f  < 0.08 Hz. There is broad agreement that low frequencies are typical for revealing connectivity in resting state fMRI data [ 11 ]. However, since the temporal characteristics of event-related data may change the frequency profile of the BOLD signal (likely increasing the high frequency content), we investigated the implications of filtering fcMRI timecourses. Overall, connectivity in the DMN and in whole brain networks was stronger in the fullband (unfiltered) condition compared to the bandpass version of the analysis (see  Supplement Tables 2  and  3 ). For consistency, and for using a conservative method that prevents any increase in task connectivity to be attributed to less restrictive temporal filtering, in all subsequent analyses we used the band-pass filtered fcMRI data, identical to the filter used with the resting state data. This also ensured that any interpolated data due to the upsampling step was accounted for.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24944871">PMID:24944871</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High resolution 3D brain images were acquired on a Philips 1.5 T MR system (Philips Medical System, Andover, MA). Images were collected by means of an axial three-dimensional, T1-weighted, fast field echo sequence (field of view 256 mm; view matrix 256  ·  256; repetition time 24 ms; echo time 5 ms; flip angle 40 degrees, slice thickness 1 mm). For the present study, volumetric measurements were extracted through a standard procedure using Freesurfer software (Greve and Fischl  2009 ; Postelnicu et al.  2009 ; Fischl  2012 ) version 4.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ). Specifically, the  ‘ recon-all ’  command embedded within Freesurfer was executed for all T 1 -weighted scan data and resulting anatomical volumes used for subsequent statistical analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24939237">PMID:24939237</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted structural images with 1 × 1 × 1   mm voxel size were acquired for each participant using Magnetisation Prepared Rapid Gradient Echo (MPRAGE) on a 3T TIM Trio system (Siemens, Erlangen, Germany). These images were processed in FreeSurfer (FS) (Version FreeSurfer-Linux-centos6_ × 86_64-stable-pub-v5.3.0,  http://surfer.nmr.mgh.harvard.edu/ ). The automated procedure for volumetric measures of subcortical regions have been previously described by  Fischl et al. (2002) . This procedure segments and labels unique structures based on probabilistic information were estimated from a manually labelled training set. In brief, FreeSurfer uses a Bayesian segmentation procedure that takes into account the prior probability of a given tissue class occurring in a specific atlas location, the likelihood of the image given the tissue class and the probability of the local spatial configuration of labels given the tissue class. Total intracranial volumes (TIV) were calculated to ensure that regional brain volume analyses are not biased by potential changes in TIV.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24936410">PMID:24936410</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A dedicated segmentation pipeline was developed by combining   VBM8, SPM8 ( http://www.fil.ion.ucl.ac.uk/spm/software/spm8/ ), Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) and BrainVISA ( http://brainvisa.info/index.html ) in order to cumulate their strength. It results in a set of sulcal pieces, each sulcal part being associated with a set of shape descriptors. The 4 steps are detailed in the text and supplementary materials and the sulcal attributes are illustrated in   panel 4: sulcal pieces that are colored following a) anatomical denomination, b) mean depth, c) maximum depth and d) area of the folds.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24932546">PMID:24932546</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For cortical reconstruction and segmentation, T1-weighted MP-RAGE sequences were processed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). All image post-processing was performed on a single Mac workstation running OS X Lion 10.7.5 software by a single technician. Processing utilized methods that have been described in detail in prior publications, including skull stripping, automated Talairach transformation, segmentation of subcortical white matter and gray matter structures, intensity normalization, tessellation of brain boundaries, automated topology correction, and surface deformation.  [27] ,  [28] ,  [29] ,  [30] ,  [31] ,  [32] ,  [33] ,  [34] ,  [35] ,  [36] ,  [37] ,  [38] ,  [39]  FreeSurfer output was visually inspected for errors prior to data being included for further analyses. Cortical thickness was defined as the distance from the boundary of the gray matter with white matter to the boundary of the gray matter with cerebrospinal fluid at each vertex along the brain surface.  [29]  For data interpretation and analysis, a surface-based toolbox within FreeSurfer was used to interrogate cortical thickness to pain threshold correlations in migraineurs and healthy controls. Cortical thickness output maps were smoothed by applying a smoothing kernel of 20 mm FWHM (full width at half maximum).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24918069">PMID:24918069</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were processed with FreeSurfer software (v5.1,  http://surfer.nmr.mgh.harvard.edu/ ). FreeSurfer allows for semi-automatic reconstruction of the cortical surface using T1-weighted MRI images. Major steps during image analyses include motion correction, removal of non-brain tissue, automated Talairach transformation, subcortical and cortical matter segmentation, intensity correction and delineation of gray/white/pial boundaries ( Dale et al., 1999 ;  Fischl et al., 1999 ). After the formation of cortical models, deformable procedures are applied including cortical inflation, registration to a spherical atlas and parcellation of the cerebral cortex into gyral and sulcal units ( Desikan et al., 2006 ). For the participants with data at two time points, the structural scans were additionally processed through the longitudinal stream ( Reuter et al., 2010 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24907483">PMID:24907483</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was completed by using FreeSurfer 5.0.0 ( http://surfer.nmr.mgh.harvard.edu/ ). Comprehensive details of these procedures are provided in previous publications ( Dale et al., 1999; Fischl et al., 1999a,b; Fischl and Dale, 2000; Fischl et al., 2001; Fischl et al., 2002; Fischl et al., 2004; Han et al., 2006; Jovich et al., 2006; Segonne et al., 2004 ). Briefly, brightness and contrast normalisation is performed on the images, and then all non-brain tissues are removed with a hybrid watershed/surface deformation procedure ( Segonne et al., 2004 ). Images then undergo Talairach transformation, subcortical white matter and deep grey matter structures are segmented ( Fischl et al., 2004 ), the grey white matter boundary is tessellated, topology automatically corrected ( Fischl et al., 2001; Segonne et al., 2007 ), and surface deformation is performed by using intensity gradients to optimally place the grey/white and grey/CSF borders where the greatest change in intensity signifies transition to the other tissue class ( Dale et al., 1999 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24904351">PMID:24904351</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation of structural MRI data were performed using an automated processing stream in Freesurfer v5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ; e.g., Dale et al.,  1999 ; Fischl et al.,  2001 ,  2002 ; Segonne et al.,  2004 ; Han et al.,  2006 ; Reuter et al.,  2012 ). This stream includes motion correction and averaging, automated transformation, spatial smoothing, removal of non-brain tissue, segmentation of white and gray matter, and intensity normalization. We defined the white matter tract of interest (the uncinate fasciculus) using FreeSurfer's TRActs Constrained by UnderLying Anatomy (TRACULA; Yendiki et al.,  2011 ), an automated method that reconstructs probabilistic distributions of major white matter tracts from each participant's diffusion images. This method has been shown to accurately reconstruct tracts in individual subjects using anatomical priors. TRACULA is made up of three processing steps: preprocessing, fitting of a ball-and-stick model of diffusion to the diffusion-weighted data, and reconstruction of the pathways of interest. TRACULA preprocessing includes eddy-current compensation, motion correction, intra-subject registration (to subject's T1), inter-subject registration (to MNI template), creation of cortical and white-matter masks, tensor fitting, and computation of anatomical prior for white-matter pathways. The ball-and-stick model is applied to each subject's preprocessed diffusion data, estimating parameters of the model at every voxel. These results, along with prior knowledge of pathway anatomy, are then used to fit the pathway of interest (here, the uncinate fasciculus, see Figure  1A  for depiction of a representative pathway) for each subject. Measures of fractional anisotropy (FA; i.e., a ratio of radial and axial diffusivity) are then extracted from the estimated pathway.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24904268">PMID:24904268</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Source space modeling and HPI-MRI alignment . Individuals' anatomical surfaces, to which MEG data were co-registered, were created from the aMRI data with Freesurfer software  recon-all  process (default parameters) ( http://surfer.nmr.mgh.harvard.edu ; Dale et al.,  1999 ; Fischl et al.,  1999 ). Correct segmentation of white/gray matter for cortical surfaces was confirmed by eye. FreeSurfer's  watershed  algorithm was used to reconstruct the inner skull, outer skull and outer skin surfaces from the individuals' aMRI data and to estimate the boundary element model (BEM) compartments. BEM compartments are used to specify the model for the electrical conductivity geometry of the head. A  “ single shell ”  forward model based upon this BEM was used in all source reconstruction methods.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24891029">PMID:24891029</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the FreeSurfer toolkit ( http://surfer.nmr.mgh.harvard.edu ) to measure cortical thickness from the MPRAGE images. Using the standard options, the MPRAGE image was subjected to intensity normalization, skull-stripping and an affine transformation to Talairach space. A tissue segmentation algorithm was used to separate WM from non-WM, and the results were further optimized using a tesellation process followed by topological correction and smoothing ( 29 ). The cortex was inflated, flattened and registered to a spherical atlas using individual cortical folding patterns point-to-point ( 30 ). A cortical parceling algorithm assigns an anatomical label to each vertex on the cortical sheet ( 31 ). We computed GM volumes for the OFC (lateral and medial combined) and ACC (rostral and caudal combined). To account for interhemispheric differences in surface area, we averaged the cortical thickness of the OFC and ACC subregions between hemispheres weighted by their corresponding surface areas.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24865593">PMID:24865593</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analyses were performed using a modified Freesurfer software suite ( http://surfer.nmr.mgh.harvard.edu/ ) that is based on routine three-dimensional T1-weighted MRI volumes. FreeSurfer provides a method for automatic parcellation of the cortical surface, by which each location on the surface is assigned a neuroanatomical label on the basis of probabilistic information estimated from a manually labeled training set. Details on surface-based cortical reconstruction and subcortical volumetric segmentation were reported previously. 34  Delineation and labeling of the white matter tracts were performed with the atlas-based method, AtlasTrack, using diffusion tensor images (DTIs), T1-weighted images and a probabilistic atlas of fiber tract locations and orientations. 42  Validated for use in pediatric populations, AtlasTrack provides estimates for each tract the average mean diffusion, transverse and longitudinal diffusivity (TD and LD, respectively) and fractional anisotropy (FA).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24855156">PMID:24855156</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical and volumetric segmentation were performed with FreeSurfer 4.4 ( http://surfer.nmr.mgh.harvard.edu ) using the T1-weighted images. Volumes were determined for both cortical and subcortical regions ( 12 ). Parcellated brain volumes were combined into larger regions matching the regions for diffusion analysis. These included frontal, temporal, and parietal regional volumes ( Supplementary Table 1 ) and were expressed as a proportion of the total brain volume (TBV) to normalize for age-related variations in brain size.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24847245">PMID:24847245</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>One of the most common label sets used in studies of both functional and structural connectivity is the AAL label set (Tzourio-Mazoyer et al.,  2002 ) which is a template based label set. An existing multivariate template had been created from the Kirby dataset using the  antsMultivariateTemplateConstruction.sh  tool, part of the Advanced Normalization Tools (ANTs) software package (Avants et al.,  2009 ). The  antsRegistration  tool was used to find a deformable mapping between the T1 template image distributed with the AAL label and the population specific template created from the Kirby data. In order to transform these labels into each subject's DTI space, it was necessary to find a transform from the template to each subject's T1 and from T1 to DTI within each subject. For the template-to-T1 transform, the  antsCorticalThickness.sh  tool was used. This software first applied a bias correction using the N4 algorithm (Tustison et al.,  2010 ). Next a registration based skull stripping was performed to provide a cerebrum mask of the T1 image. This was followed by a final cerebrum-only registration to the template. These transforms were composed with the T1-to-DTI transforms, providing a single transform that was used to warp the the AAL labels into DTI space using nearest neighbor interpolation. Labels of structures outside of the cerebrum were removed. Many AAL labels include both gray and white matter, here the labels were masked to only include voxels that were identified as cortical gray matter by the DKT31 labels described in the previous section. The AAL labels for deep gray structures (e.g., thalamus) were not masked but used in their entirety. Both label sets are illustrated in Figure  1 , while the entire processing scheme is illustrated in Figure  2 . The availability of the processing scripts is intended to provide a framework that allows for convenient exploration of alternate anatomical labels, such as the anatomical parcellations that may be obtained via FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) or the UCLA Multimodal Connectivity Package ( http://ccn.ucla.edu/wiki/index.php/UCLA_Multimodal_Connectivity_Package ), both of which have been used in previous graph-theory based examinations of structural connectivity based on diffusion-weighted imaging.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24818077">PMID:24818077</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All image pre-processing was performed in   Statistical Parametric Mapping version 8 (SPM8;  http://www.fil.ion.ucl.ac.uk/spm ). Reference regions were defined in native MRI space for each subject using subcortical parcellations from FreeSurfer 4.5 ( http://surfer.nmr.mgh.harvard.edu ). FDG-PET frames were summed and   standardized uptake value ratios (SUVR) were calculated by normalizing the summed FDG image to mean activity in the pons for each subject. For PiB, voxel-wise   distribution volume ratios (DVRs) were calculated using   Logan graphical analysis ( Logan et al., 1996 ) with the gray matter cerebellum time – activity curve used as a reference tissue input function ( t   =   35 – 90   min) ( Price et al., 2005 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24816735">PMID:24816735</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>ASL data analyses were performed using a combination of FSL (FMRIB ’ s Software Library,  www.fmrib.ox.ac.uk/fsl )  [38]  and FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) tools. The  ‘ tag ’ ,  ‘ control ’ , and M 0  scans were first motion-corrected using MCFLIRT  [39]  and registered to the high resolution anatomical images using FreeSurfer ’ s bbregsiter tool  [40] . Then tag and control scans were surround subtracted (i.e. given each tag X , [(control X ? 1 +control X+1 )/2 ? tag X ]) to generate perfusion-weighted images. All of the perfusion-weighted maps were then averaged and scaled by a factor proportional to the M 0  scan to obtain rCBF maps in absolute values (ml/100 g of tissue/min)  [36] . Group-level analyses were conducted in the volume, and group rCBF maps were smoothed with a 3D isotropic, 7.02 mm full-width half-maximum (FWHM) Gaussian kernel (set to 2*voxel size). Regional CBF values were extracted from the BLA ROIs mapped into each subject ’ s individual anatomical scan. Additionally, whole-brain voxel-wise analyses using STAI state anxiety score as a regressor were conducted, in order to determine whether there were significant correlations between anxiety and rCBF of regions other than the amygdala. In this analysis, the Gaussian Random Fields (GRF) method was used to correct for multiple comparisons  [41] , with a voxel-level height threshold of p < .0005. Clusters that showed cluster-wise significance (cluster-level threshold of p < .05, GRF whole brain corrected) are reported in the text and in  Table 2 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24789257">PMID:24789257</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain cortical lesions were identified and scored on DIR sequences, blinded to clinical data, in accordance to consensus recommendation,  [12]  and accurately controlled for artifacts. Lesion volume was quantified using the MIPAV software. The intrarater reliability between lesion quantification was assessed using intra-class correlation (with three sets of data points; ICC ? = ? 0.896). Brain cortical volume evaluation was performed on volumetric T1 gradient echo images by means of the FreeSurfer v5.1 image analysis suite, available online ( http://surfer.nmr.mgh.harvard.edu/ ) (left + right hemispheres volumes), as described elsewhere. [13] – [15]  All images were systematically controlled for errors and artifacts.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24761228">PMID:24761228</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All analyses were performed with FreeSurfer, version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ), following the methods detailed by Bigler  et al.  [ 88 ], and included automated volume calculations of the following temporal lobe regions of interest (ROI): fusiform gyrus, amygdala and hippocampus along with total intracranial volume (TICV). It should be noted that hippocampus segmentations are more reliable than amygdala segmentations [ 89 ]. TICV was used as a matching variable and covariate.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24734233">PMID:24734233</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness analyses were carried out using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). Detailed procedures on cortical thickness analyses using FreeSurfer have been published [ 30 – 33 ]. In brief, T1-weighted images were intensity-normalized (correcting for magnetic field inconsistencies) and then a skull-stripping procedure was applied to remove extracerebral voxels. A researcher (F.M.-blind to identity/diagnoses) then carried out manual edits to the skull-stripped images. Scans subsequently underwent a segmentation procedure using an estimation of the structure of the grey-white interface. In order to create a smooth spherical representation of the grey-white interface and pial surface, each scan was covered with a triangular tessellation and inflated. Inflated scans were then aligned to FreeSurfer's default reference template via a 2D warp based on cortical folding patterns. Once smoothed using a circularly symmetric Gaussian kernel, sulci and gyri curvature patterns were aligned and the average cortical thickness was measured at each surface point. A uniform surface-based spherical coordinate system was created by transforming the reconstructed surfaces into parameterizable surfaces. An averaging procedure (50 iterations) was applied to smooth the surface and the reconstructed pial surface refined with a deformable surface algorithm. Data was again aligned on a common spherical coordinate system. Cortical thickness was determined by measuring and averaging the distance between the grey-white and pial surfaces [ 30 – 33 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24723876">PMID:24723876</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The multifocal tubers were identified by an experienced pediatric neuroradiologist (PEG) and labeled in FreeView ( http://surfer.nmr.mgh.harvard.edu/ ). The tuber closest to the ECDs was then manually segmented in FreeView with margins confirmed by an experienced pediatric neuroradiologist (PEG). The tuber margins were used to determine the distance between the ECDs and the tuber (Section  MEG and EEG Data Analysis ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24718105">PMID:24718105</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ) served to assess cortical thickness (thk)  [21] . In short, each MPRAGE image was registered to Talairach space, and subsequently grey matter (GM), white matter (WM), and pial surface were segmented. Cortical-thk was calculated as shortest distance between the boundary between WM and GM and pial surface. To realize change measurements, we used the longitudinal stream implemented in FreeSurfer  [22] . An unbiased within-subject template was created for BL and FU for each participant and each measurement was registered to this template to increase reliability of thk-measurements compared with a cross-sectional design  [23] . For statistical analysis, a general linear model (GLM) approach was used (Free Surfer ’ s QDEC, version 1.4), focusing on group differences (HC, MS) at each time point. Changes of cortical-thk (mm/year) were computed in a vertex-wise manner and smoothed using a Gaussian kernel (FWHM ? = ? 20 mm). A Monte-Carlo simulation with 500 iterations served to correct for Type 2 errors due to multiple comparisons. The significance threshold was set to p ? = ? 0.05.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24715957">PMID:24715957</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Whole-brain volume for each subject, excluding the ventricles, was obtained from FreeSurfer (\n http://surfer.nmr.mgh.harvard.edu/ )\n 39 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24713859">PMID:24713859</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition to those factors, cortical gray matter volume measures combine influences of cortical thickness and surface area, features which are believed to be influenced by different genetic factors regulating sulcal patterning and the thickness of the cortical mantle itself. 20  Previous studies 21 ,  22 ,  23  have found that cortical volume is driven mostly by cortical surface area rather than cortical thickness. Advanced automated analytic techniques such as FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) now make possible the examination of cortical thickness and area  in vivo  using high spatial resolution structural MRI. Independent measurements of these aspects of cortical anatomy are important as cortical thickness reflects the size, density and arrangement of neurons, neuroglia and nerve fibers, 24  and thus its measurement could provide important and relatively unique information about disease-specific neuroanatomical changes. For instance, the regional thinning of cortex can reflect reduced dendritic arborization or changing myelination at the gray/white matter interface 25  within specific brain systems. Postmortem studies in MDD have reported modest thinning of the cortical ribbon and/or layer-specific decreases in cell density due to cell loss and cell atrophy in localized regions, including the rostral and caudal orbitofrontal and DLPFC. 26 , 27  However, postmortem cytoarchitectural measurements have limitations, including tissue fixation artifacts and limited antemortem characterization, and they typically are done with chronic patients typically with a protracted course of illness.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24713364">PMID:24713364</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To parcellate the brain into cortical and subcortical tissue classes and derive quantitative estimates of brain volume, we used an automated, non-biased atlas-based Bayesian segmentation procedure, applied in FreeSurfer v.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ) [ 27 ]. FreeSurfer preprocessing for volumetric T1-weighted images included: brain extraction and removal of non-brain tissue using a hybrid watershed/surface deformation procedure [ 28 ]; automated spatial transformation and white matter segmentation of subcortical volumetric structures [ 29 ]; intensity normalization, tessellation of gray matter/white matter boundary and automated topology correction [ 30 ]; and surface deformation following intensity gradients to optimally place gray matter/white matter and gray matter/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class [ 27 ]. Image outputs from each stage of FreeSurfer processing were visually inspected independently by three imaging analysts (CME, MEK). Only images that passed quality control by both analysts were used; seven DS, five WS, and five TD adults were not included in the analysis because of failure to pass quality control, leaving a total of 14, 58 and 81 adults in each category, respectively. Quantitative estimates of volume were derived in a large set of spatially distinct region of interests (ROIs) that covered the entire brain, as specified in the Desikan atlas [ 31 ]. This atlas includes parcellations of gray and white matter and segmentations of subcortical gray matter, and also includes summary volumes (that is total cortex volume). Parcellations of the gray and white matter and segmentations of subcortical gray matter were included, along with two summary measurements of total gray and total white matter (see Additional file  1 : Tables S1a and b for complete list of ROIs included and those excluded, respectively). Total intracranial volume (ICV) was also estimated in FreeSurfer, and all ROI measures were normalized to ICV for subsequent analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24709970">PMID:24709970</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI images were acquired by a 3-Tesla scanner (Trio; Siemens, Erlangen, Germany) in the Bioimaging Research Center at the Institute of Physiology and Pathology of Hearing in Kajetany, Poland. A High-resolution T 1 -weighted MR volume images were acquired with the Magnetization Prepared RApid Gradient-Echo (MP-RAGE) imaging sequence with the following settings: TR ? = ? 2.3 s; TE ? = ? 3.0 ms, slice thickness 1.2 mm, 256 × 240 matrix, FOV ? = ? 23.8 × 25.5 cm, 176 sagittal slices. The original imaging data were fetched from the scanner in the DICOM format and anonymized. The introductory subject-specific analysis comprised of estimation of cortical thickness with the FreeSurfer (FS v5.1.0) image analysis environment ( http://surfer.nmr.mgh.harvard.edu/ )  [29] – [31] . First, the high resolution T1 MPRAGE volumes were converted to FreeSurfer format, normalized for intensity  [32]  and resampled to isotropic voxels of 1 × 1 × 1 mm. Next, the skull was removed using a skull-stripping algorithm and segmented into three tissue types (white matter, grey matter and CSF)  [29] ,  [31] ,  [33] – [36] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24683508">PMID:24683508</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In this study, we investigated age-related changes in functional connectivity of the DMN by analyzing the resting-state BOLD fMRI data in subjects' native space instead of standardized atlas space. The native space approach substitutes for spatial normalization. Spatial normalization is the conventional method for warping all subjects in a study into a standard space and facilitates the use of predefine regions-of-interest (ROI) mask from the utilized brain atlas instead of the subjects own regional mask. It is implemented by registering each subject's brain to a canonical template brain. Spatial normalization is a key step for studies doing voxel-wise, across-subject comparisons. A long-standing problem in functional neuroimaging studies of aging is that the large age-related changes in brain morphology make it difficult to coregister brains (Yassa and Stark  2009 ; Seibert and Brewer  2011 ). Although spatial normalization is intended to fit all brain images to a standardized space, the assumption that any voxel represents the same brain location for every subject is typically untrue. To illustrate the extent of the problem caused by spatial normalization, we used FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) to extract hippocampus and precuneus binary masks for the 51 subjects included in this study. We then used statistical parametric mapping (SPM8, Wellcome Department of Cognitive Neurology) to perform spatial normalization. The 51 spatially normalized masks were summed and overlaid on top of the MNI152 atlas. Figure   1 A and C display these summed mask images, which we call an  “ overlay map ”  of the regions. The voxel values inside these  overlay maps  show how many subjects have their hippocampus/precuneus regions in that particular location. For instance, if the voxel value in the  overlay map  of the hippocampus is 51, then this voxel belongs to hippocampus in all the subjects after spatial normalization. Lower values indicate that fewer subjects have the same region in that voxel. As shown in Figure   1 A and C, the  overlay maps  of the aforementioned two regions extend far beyond their borders (the red curve shows the border of two regions in the atlas) even after spatial normalization. This simple example clearly demonstrates the poor performance of the prevailing spatial normalization method for fMRI data analysis in aging research.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24671208">PMID:24671208</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In a first step, the T1w-MRI was resampled to obtain an isotropic resolution of 1 mm. The T2w-MRI was registered onto the T1w-MRI using a rigid registration approach and mutual information as a cost-function as implemented in FSL ( http://www.fmrib.ox.ac.uk/fsl ). Then, brain, inner skull, outer skull and extracranial tissue (summarized in the following as the  skin  compartment) masks were obtained from the T1w and T2w images following  [44] . In a next step, the T1w image served for the segmentation of gray and white matter and the T2w image for the segmentation of the CSF using a hidden Markov random field model  [45] . All these steps were realized using FSL software. The gray matter mask was then further improved using Freesurfer ( http://surfer.nmr.mgh.harvard.edu ). The skull segmentation was visually inspected and manually corrected using CURRY ( http://www.neuroscan.com/curry.cfm ). The skull spongiosa was segmented from the T2w-MRI using a threshold based segmentation restricted within the skull compartment. The resulting segmentation is shown in  Figure 1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24667541">PMID:24667541</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess the potential impact of platelet V max  on coherent functional networks, we calculated functional connectivity, which provides a robust measure to assess temporal correlations in the fMRI signal across functionally linked areas. Specifically, functional connectivity analysis explores which parts of the brain are coupled with a given  “ seed ”  region by cross-correlating the time series of the seed with all other voxels. Briefly, preprocessing of functional connectivity data consisted of despiking, motion correction, and alignment of functional data to the respective anatomical dataset including template alignment (TT_N27 Talairach template). Beyond the widely used removal of motion parameters derived from volume registration, we utilized the ANATICOR method  [67]  to eliminate nuisance signals estimated from eroded white matter and cerebrospinal fluid masks provided by FreeSurfer anatomical segmentation (FreeSurfer version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ))  [68] . For temporal filtering, a broad frequency band (0.008 – 0.15 Hz) has been applied, which has previously been found to yield the highest test-retest reliability  [69] . Finally, data have been blurred with a spatial Gaussian filter of 8 mm FWHM. To avoid any impact of task-related activations on connectivity estimates, correlation analysis was restricted to the control task blocks  [70] . The selection of seeds was based on the whole-brain correlation analysis with platelet V max  including the largest positively and negatively correlated clusters as seeds for extraction of mean time series located in the medial prefrontal cortex (555 voxels, z ? = ? 3.756) and motor cortex (242 voxels, z ? = ? 6.942). Within first level analysis mean time series were regressed against the time series of the remaining voxels. The resulting single subject statistical seed to voxel correlation maps were then converted to z-scores using Fisher ’ s transformation formula. Single subject z-score maps were included in a second level voxel-wise regression analysis to test for possible correlation with platelet V max  while controlling for age, gender, and 5-HTTLPR genotype. Monte Carlo simulation (10,000 iterations, dimensions: 74 × 87 × 69 grid, 2.19 × 2.19 × 2.19 mm, 11.0 × 11.3 × 11.5 mm smoothness) indicated that an initial voxel-wise threshold of p < 0.005 and a minimum cluster size of 182 voxels yielded a corrected p-value of 0.05.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24663006">PMID:24663006</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation based on the T1-weighted data was performed with the FreeSurfer image analysis suite, version 5.0 ( http://surfer.nmr.mgh.harvard.edu/ )  [33] . The cerebral cortex was subdivided into gyral-based regions of interest  [34] . During the processing pipeline, data were visually inspected and manually edited as required, by an operator blind to clinical history. Estimates of cortical volume, surface area and thickness  [35]  were extracted from the left and right hemisphere pericalcarine cortices, which are the anatomical locations of V1.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24661349">PMID:24661349</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automatic subcortical volumetric segmentation and cortical thickness evaluation was performed using the FreeSurfer ®  (FS) image analysis suite (v.5.0.0 Martinos Center for Biomedical Imaging, Harvard-MIT, Boston, MA), which is documented and freely available for download ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures were described previously [ 18 ]. Briefly, this fully automated process includes motion correction, removal of non-brain tissue, automated Talairach transformation, segmentation of the subcortical white matter and deep grey-matter volumetric structures (including hippocampus, amygdale, caudate, putamen, and ventricles), intensity normalization, and cortical reconstruction. This segmentation procedure assigns a neuroanatomical label to every voxel in the MR image volume. The method is based on probabilistic information estimated from a manually labeled training set. The Markov random field theory is applied, where the probability of a label at a given voxel is computed not just in terms of the grey-scale intensities and prior probabilities at that voxel, but also as a function of the labels in a neighborhood around the voxel in question. This is important for correct differentiation between the hippocampus and amygdala, which have similar grey-scale values. The results of the automatic segmentations were reviewed and any errors were corrected. Correction for the intracranial volume ( ICV), including white matter, grey matter, and cerebrospinal fluid, was also estimated for each subject using FS. Individuals with a large intracranial volume tend to have larger subcortical structures; therefore, correction by the intracranial volume is crucial in volume quantification of the subcortical structures. This correction can be simply done by dividing numerically the volume of the subcortical structure through the ICV.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24658422">PMID:24658422</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing was performed and structural volumetric measures were obtained using reconstruction and volumetric segmentation procedures from the freely available Freesurfer stable release version v5.0.0 ( http://surfer.nmr.mgh.harvard.edu/ ), published in detail previously  [11]  and running on a Macintosh with OSX 10.6.4. In short, volumetric assessments of the left and right hippocampus were obtained for all participants using automatic classification procedures and labeling  [35] . The FreeSurfer processing pipeline automatically assigns a neuroanatomical label, roughly corresponding to each voxel in an MRI volume (after partial volume correction), based on probabilistic information estimated from a manually labeled training set. The accurate labeling of subcortical structures is achieved through the use of both global and local information. The global information is based on an atlas that makes the labeling robust to contrast properties of the anatomical structures. Modeling the classification as a non-stationary anisotropic Markov random field incorporates local information. The introduction of non-stationary and anisotropy into the classical Markov random field model allows spatial relationships of anatomical classes to enter into the segmentation procedure. For instance, the probability that a voxel labeled  “ hippocampus ”  will have its inferior neighbor labeled as amygdala provides a strong set of spatial constraints. The output that FreeSurfer yields is rounded off to an integer number and therefore reflects the number of voxels in an area. The technique has previously been shown to be comparable in accuracy to manual labeling  [35] . The segmentations were visually inspected for accuracy. A measure of intracranial volume (all voxels in a brain) was also generated, by adding up voxel counts for each area.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24653952">PMID:24653952</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was performed using the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ), which is an automated tissue classification and segmentation software that exhibits good test – retest reliability across scanner manufactures and field strengths (Han et   al.  2006 ). Each subject's MRI was initially analyzed in original space using the following analysis pipeline. Processing included removal of nonbrain tissue by a hybrid watershed/surface deformation procedure, subcortical structures were segmented (Fischl et   al.  2002 ), and further intensity normalization was conducted. This was followed by white matter segmentation, tessellation of the gray – white matter boundary, and automated topology correction (Fischl et   al.  2001 ). Then surface deformation following intensity gradients optimally placed the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Fischl et   al.  2001 ). Once the cortical models were complete, deformable procedures performed additional data processing and analysis, including parcellation of the cerebral cortex into 34 conventional gyral-and sulcal-based neuroanatomical regions in each hemisphere (Desikan et   al.  2006 ). This parcellation method demonstrates diagnostic sensitivity in other diseases (Desikan et   al.  2009 ). Intensity and continuity information from the segmentation and deformation procedures produced representations of cortical thickness, which were calculated as the closest distance from the gray – white matter boundary to the gray – CSF boundary at each vertex on the tessellated surface (Fischl and Dale  2000 ). Cortical thickness was used in this study as it accounts for most volumetric changes in prHD (Nopoulos et   al.  2010 ) and is influenced by genetic factors (Winkler et   al.  2010 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24645731">PMID:24645731</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each MRI dataset underwent cortical surface reconstruction via the FreeSurfer automated cortical measurement technique. FreeSurfer is a free software program that can be downloaded ( http://surfer.nmr.mgh.harvard.edu ). Procedures for using the FreeSurfer program have been outlined in various sources [ 36 - 38 ]. First, the cortical surface undergoes reconstruction involving the correction of intensity variations that exist between MRI data sets due to variations in the magnetic field during the scanning procedure [ 36 ]. This reconstruction also involves the removal of any voxels containing the skull or other extra-cerebral entities [ 36 ]. After the intensity corrections and skull stripping, a researcher blind to the condition of each subject manually corrected the images. Subsequently, the individual data sets underwent a segmentation procedure based on a structural estimation of the gray-white matter interface [ 36 ]. In order to generate a smooth and accurate representation of both the pial surface and the gray-white interface, a triangular tessellation cover was applied to each scan, which was then inflated [ 36 ]. Inflation allows for easier visualization by reducing any interference by folding so that the cortical area within sulci can be visualized [ 38 ]. Scans were then aligned with the reference brain template based on the positioning of the gyri and sulci [ 38 ]. It is generally difficult to establish a coordinate system in which the cortical surfaces of different volumes can be accurately matched to provide consistent measurements [ 38 ]. Therefore, the reconstructed cortical surfaces were transformed into parameterizable surfaces. Parameterization provides a reproducible and consistent coordinate system [ 38 ]. Finally, the average cortical thickness measurements were taken at every point on the smoothed and aligned images [ 37 ]. Cortical thickness is calculated by averaging the distance between the pial surface and the gray-white interface [ 37 ]. It is important to note that this technique for taking automated cortical thickness measurements is not restricted by the resolution of the voxels in the original MRI data and can provide measurements accurate to a sub-millimeter scale [ 37 ]. For this study we restricted our focus to MFG and caudal ACC.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24633348">PMID:24633348</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness was estimated from the structural magnetic resonance images using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ,  [30] ), a set of automated tools for reconstruction of brain cortical surface  [31] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24625888">PMID:24625888</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each subject ’ s structural T1 scans were reconstructed using FreeSurfer  [30]  ( http://surfer.nmr.mgh.harvard.edu/ ). The accuracy of FreeSurfer ’ s subcortical segmentation and cortical parcellation  [31] ,  [32]  has been reported to be comparable to manual labeling. Each subject ’ s white and gray matter boundaries as well as gray matter and cerebrospinal fluid boundaries were visually inspected slice by slice by an experienced user (DOS), manual control points were added in the case of any visible discrepancy, and reconstruction was repeated until we reached satisfactory results within every subject. The subcortical structure borders were plotted by Freeview visualization tools and compared against the actual brain regions. In case of discrepancy, they were corrected manually. The regions of interest used in this analysis are listed in  Table S1 in File S1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24624328">PMID:24624328</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The two T1-weighted 3-D inversion recovery-spoiled images (40 and 8 ° ) were used to calculate the T1 relaxometry maps (T1R). FreeSurfer Image Analysis Suite 4.5 version ( http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al., 1999; Fischl et al., 1999 ) was used for cortical reconstruction, volumetric segmentation, subcortical ( Fischl et al., 2004 ) and cortical parcellation (Desikan-Kyliany Atlas: 40 VOIs per hemisphere with Freesurfer Atlas), as well as determining cortical thickness in each VOI and deep nuclei volumes. Volumes were normalized for intracranial volume ( Desikan et al., 2006 ). Segmentation results were assessed by an experienced operator to ensure accuracy.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24624052">PMID:24624052</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>8 surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24611042">PMID:24611042</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image segmentation, cortical surface reconstruction and surface-based coregistration were performed on the high-resolution T1-weighted image using  FreeSurfer  5.0.0 ( http://surfer.nmr.mgh.harvard.edu/ ; Fischl,  2012 ). Each cortical hemisphere was then parcellated into 63 ROIs according to the  SpeechLabel  cortical labeling system. This new parcellation extends the cortical parcellation paradigm of Nieto-Castanon et al. ( 2003 ) by including finer-scale subdivisions of the cortical regions involved in speech compared to the default protocols offered by FreeSurfer (see Figure  2 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24604691">PMID:24604691</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical and subcortical brain regions were selected by parcellating the cerebrum and the brain stem into distinct, anatomically separated brain regions on the basis of the T1-weighted image, using the well-validated Freesurfer suite (V4.5,  http://surfer.nmr.mgh.harvard.edu/ ). This included the automatic segmentation of gray and white matter tissue, followed by parcellation of the segmented gray matter mask into distinct brain regions, based on a normalized template, parcellating the brain into a number of brain regions, including left and right caudate nucleus, globus pallidum, nucleus accumbens, thalamus, amygdala, hippocampus, brain stem, and 68 cortical brain regions. In total, 83 distinct brain regions were parcellated [Fischl et al.,  , ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24602322">PMID:24602322</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1.5-T MRI and 3-T non-accelerated sagittal volumetric 3D MPRAGE MRI images were acquired at each performance site for the ADNI 1 and ADNI-GO/2, respectively ( http://adni.loni.ucla.edu ). Only MRIs which passed the quality control evaluations were included. To estimate hippocampal volumes (HV) measures, cortical grey matter (GM) volumes were processed using Free-surfer software package version 4.4 and 5.1 image processing framework for the 1.5 and 3-T MRI images, respectively ( http://surfer.nmr.mgh.harvard.edu/ ) [ 25 , 26 ]. We estimated in an independent dataset a method to obtain the adjusted HV (aHV; adjusted for intracranial volume (ICV)) for the MRIs (Additional file  1 : supplementary material) (Figure   2 a). The SPARE-AD (Spatial Pattern of Abnormality for Recognition of Early Alzheimer ’ s disease) is an index that captures brain atrophy related to AD [ 27 , 28 ]. FDG-PET data were acquired and reconstructed with the use of measured-attenuation correction and the specified reconstruction algorithm for each scanner type according to a standardized protocol ( http://adni.loni.ucla.edu/ ). Images were downloaded and pre-processing using SPM5 by investigators at Banner Alzheimer ’ s Institute ( http://www.fil.ion.ucl.ac.uk/spm ). We calculated a pattern based summary score, the hypometabolic convergence index (HCI) [ 29 ] and an anatomically defined ROI, the posterior cingulate (PC-FDG-PET with FDG-images using pons as reference region) CMRgl (cerebral metabolic rate for glucose).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24592234">PMID:24592234</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To generate tracts containing the PC efferent fibers from the ICA+BSM tractography of individual subjects, the current study defined seeding points at the posterior-lateral region of the cerebellar cortex (e.g., cerebellum crus 1 and 2). The conventional FreeSurfer process ( http://surfer.nmr.mgh.harvard.edu ) was applied to the high resolution FSPGR images in order to segment the cerebellar cortex in each hemisphere. The resulting cerebellar cortex was then masked by the standard templates of cerebellum crus 1 and 2 (available at  http://www.cyceron.fr/index.php/fr/plateforme/freeware ). To seed the streamlines containing the PC efferent fibers, the masked region was finally registered to the b0 image via rigid body transformation using SPM 8 ( http://www.fil.ion.ucl.ac.uk/spm/ ). A total of 2000 seeding points were uniformly distributed over all the voxels of the registered seed region.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24586801">PMID:24586801</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer  FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) is a UNIX-based software for registration (often atlas based) and analysis of medical imaging data. It is often discussed as having excellent registration and visualization tools.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24586770">PMID:24586770</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural imaging was obtained on a 3 Tesla Siemens scanner using a 32-channel head coil to obtain a T1 5 echo sagittal MPRAGE sequence [TE ? = ? 1.64 ms; 3.5 ms; 5.36 ms; 7.22 ms; 9.08 ms; TR ? = ? 2530 ms; voxel size  ? = ? 1.0 × 1.0 × 1.0 mm 3 ; FOV ? = ? 256 mm; slices  ? = ? 192; acquisition time  ? = ? 6:03]. Scans were reviewed for image quality. Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) described in several papers  [23] – [25] . For this study, we focused on the sub-cortical volume segmentation results including the caudate, putamen, globus pallidus, thalamus, nucleus accumbens, hippocampus, amygdala, and corpus callosum ( Figure 1 ). Measures of the corpus callosum were automatically segmented into five regions including: anterior, mid-anterior, central, mid-posterior, and posterior.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24578695">PMID:24578695</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were preprocessed and analyzed using AFNI ( 74 ), FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) and SPSS version 20. Following image reconstruction, resting data were submitted to slice-timing correction, motion correction, and quadratic detrending of time series data. A low band pass temporal filter was applied to restrict signal variation to frequencies between 0.01 and 0.1 ? Hz ( 54 ). Resting data images were then registered to standard (Talairach) space with a resampled resolution of 3.25 ? mm ? × ? 3.25 ? mm ? × ? 3 ? mm and smoothed with an isotropic 6 ? mm full-width half-maximum (FWHM) Gaussian kernel. To facilitate group analysis, an unbiased groupwise non-linear registration method was used to generate an implicit group reference image ( 75 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24578688">PMID:24578688</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess the size of the hippocampal sclerosis and surrounding cortex, volumetric measures of hippocampal, entorhinal, and parahippocampal gyrus were obtained for all participants with the Freesurfer image analysis suite (Fischl,  2012 ; Reuter et al.,  2012 ), which is documented and freely available for downloading online ( http://surfer.nmr.mgh.harvard.edu/ ). Non-parametric tests (Kruskal – Wallis, SPSS 18.0) were used to compare these measures between the patient and controls groups. To control global differences, intracranial volume was included in the analysis as a covariate, which was not found to be significant. The percentage of reduction of each structure was calculated for each patient group in comparison to the control group and is reported in Table  2 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24561387">PMID:24561387</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>DT-MRI data were pre-processed using FSL ( http://www.fmrib.ox.ac.uk/fsl ), to extract brain ( Smith, 2002 ), remove bulk subject motion and eddy current induced distortions by registering all diffusion-weighted volumes to the first undistorted baseline ( b   =   0   sec/mm 2 ) volume ( Jenkinson  &  Smith, 2001 ), estimate the water DT and calculate parametric maps of FA and MD from its eigenvalues using DTIFIT ( Behrens et   al., 2003 ). To extract FA and MD in the hippocampus the high-resolution T1-weighted volume scan was brain extracted using Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) and then transformed to DT-MRI space using FLIRT ( Jenkinson  &  Smith, 2001 ). The transformation matrix computed was applied to the hippocampal masks and the resulting masks in DT-MRI space were then applied to the FA and MD parametric maps. The grey matter mask previously segmented was also applied to the FA and MD hippocampal mask producing pure grey matter segmentations, and the average FA and MD values were computed. Finally, the hippocampal masks in the T1, MTR, FA and MD maps were visually checked by an image analyst (NAR) before computation of average values was performed ( Fig.   1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24550794">PMID:24550794</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For source localization and spatial normalization, two T1-weighted high-resolution structural magnetic resonance images (MRIs) were acquired during a separate session using a 3.0 T Siemens (Erlangen, Germany) Trio whole body high-speed imaging device equipped for echo planar imaging (EPI). We used a 3D multi-echo magnetization-prepared rapid gradient echo (ME-MPRAGE) sequence repetition time or (TR, 2530 ms; echo spacing, 7.25 ms) echo time, or (TE, 3 ms; flip angle 7 ° ; voxel size, 1.3 mm  ×  1.3 mm  ×  1 mm). A 3D structural image was created for each participant by averaging the two MPRAGE scans after correcting for motion. The geometry of each participant ’ s cortical surface was reconstructed from the 3D structural MRI data using FreeSurfer software (http://surfer.nmr.mgh.harvard.edu). The segmented cortical surface was registered to an average cortical representation by optimally aligning individual sulcal-gyral patterns ( Fischl et al., 1999 ). We employed a surface-based registration technique based on folding patterns because it provides more accurate inter-subject alignment of cortical regions than volume-based approaches ( Fischl et al., 1999 ;  Van Essen and Dierker, 2007 ). The cortical surface was decimated to a grid of 4098 dipoles per hemisphere, corresponding to a spacing of approximately 5 mm between adjacent source locations on the cortical surface. The MEG forward solution was computed using a single-compartment boundary-element model (BEM;  H ä m ä l ä inen and Sarvas, 1989 ). The head-position information from the first run was used to estimate the sensor location relative to the source space. Sensor data from subsequent runs were transformed to correspond to the head-position of the first run using the signal-space separation method ( Taulu et al., 2005 ). The cortical current distribution was estimated using minimum-norm estimate (MNE) software ( http://www.martinos.org/mne ); in this solution, we assumed that the orientation of the source was fixed and perpendicular to the cortical mesh. Cross-channel correlations in the recording noise used to calculate the inverse operator were estimated from data collected without a subject present (empty-room data). To reduce the bias of the MNEs toward superficial source distributions, we used a noise-normalization procedure to obtain dynamic statistical parametric maps (dSPMs) as  z -scores ( Dale et al., 2000 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24550710">PMID:24550710</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer pipeline version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ), widely used in AD research [ 12 – 15 ,  20 ], was applied to all the MRI scans to produce 55 volumetric variables, including 45 volumetric measures of subcortical parcellation and 10 morphometric statistics. For cortical thickness, 34 regional variables were determined for each hemisphere, resulting in 68 variables for cortical thickness measures. Also, surface area was estimated from 35 regions of the brain for each hemisphere resulting in 70 measures for the entire brain.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24548877">PMID:24548877</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All participants underwent the same imaging protocol, which included standard clinical sequences (Fluid Attenuated Inversion Recovery (FLAIR), Proton Density-T2-weighted) and a whole-brain high-resolution T1-weighted sequence obtained in the sagittal plane using a modified driven equilibrium Fourier transform sequence (TE/TR=2.4/7.92 ? ms, flip angle: 15 ° , voxel size: 1 ? mm 3 ) using a 3 ? T Allegra MR imager (Siemens, Erlangen, Germany) with a standard quadrature head coil. All planar sequence acquisitions were obtained in the plane of the anterior commissure – posterior commissure line. Particular care was taken to center the subject in the head coil and to restrain the subject's movements with cushions and adhesive medical tape. MRI-based quantification of cerebral volumes was performed using Freesurfer (v.4.05) software package ( http://surfer.nmr.mgh.harvard.edu ). Freesurfer includes a sophisticated automated segmentation algorithm, which delineates gross brain anatomy into a series of cortical and subcortical labels. The stream consists of five different stages, fully described elsewhere. 13 , 14  Initially, the MRI volumes were registered to the Talairach space and the output images were intensity normalized. At the next stage, the skull was automatically stripped off the three-dimensional anatomical data set by employing a hybrid method that uses both watershed algorithms and deformable surface models. At this stage, manual intervention is needed to visualize and edit areas of skull and the areas of cortex or cerebellum that should be corrected. After skull stripping, the output brain mask was labeled using a probabilistic atlas 15  and a complex algorithm combining information on image intensity, probabilistic atlas location and the local spatial relationships between structures. 16 ,  17 ,  18  For the purpose of this study, calculated volumes (in mm 3 , subsequently converted to ml) for these labels were summed up to derive estimates of total gray and white matter volume as well as total brain volume. The FreeSurfer software and its documentation can be downloaded from  http://surfer.nmr.mgh.harvard.edu .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24505468">PMID:24505468</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>First, a whole-brain, high-resolution T1-weighted magnetization-prepared gradient-echo image (MP-RAGE) was acquired with the following parameters: repetition time (TR) ? = ? 2,300 ms, inversion time (TI) ? = ? 900 ms, echo time (TE) ? = ? 3.58 ms, flip angle (FA) ? = ? 10 ° , 1 mm 3  voxels, 160 slices, FOV ? = ? 240 × 256 mm). Tissue segmentation into white and gray matter was performed on the T1 image using  Freesurfer  software ( http://surfer.nmr.mgh.harvard.edu ). Total brain volumes were quantified separately for children and adults and showed no significant group differences (1136+/ ? 119 cm ? 3 and 1215+/ ? 144 cm ? 3, respectively (p > .1)). Freesurfer was also used to parcellate the cortical gray matter into 68 regional labels in native space. In a second step,  Connectome Mapper  ( http://www.connectomics.org/connectomemapper/ ) was used to further subdivide these regions into 219 cortical ROIs of roughly equivalent size and covering the entire brain ( Figure 1 ;  Table S1 ). Whereas previous studies on rich club phenomena used parcellations at either a very coarse resolution (82 regions) or a very dense resolution (1170 regions), here we examined rich club organization in a medium-density resolution of 219 cortical parcels. We considered this a useful strategy for validating past results, as prior work has demonstrated that network-based analyses of brain connectivity can vary substantially depending upon factors such as region selection and network size  [20] ,  [21] . In addition, these structurally based region sets have been used for functional connectivity analyses by other groups as well  [18] ,  [22] . Importantly, regions of interest were applied to both structural and functional data in each participant after surface registration, which is required for proper tractography and for assuring comparability of structural and functional data types. We note that while we chose this anatomically-based parcellation in part to maintain consistency with the initial reports of rich-clubness in structural brain networks  [5] ,  [6] , other factors were considered as well. For example, biases can occur with tractography if region sizes are largely discrepant  [1] , and thus using parcellations based on functional networks (e.g.  [10] ,  [11]  has its own limitations. Nonetheless, future work should consider functionally based parcellation sets as well.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24501701">PMID:24501701</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were only available on 17 subjects from the 1.5   T MRI study ( Mathew et al., 2008 ). MRI images were collected using a Sagittal T1-gradient echo volumetric acquisition protocol (TE/TR   =   2/9   ms, voxel size   =   0.9   ×   0.9   ×   1.5   mm, flip angel   =   7 ° , FOV   =   240   mm, 1.5-mm thickness with no gaps, totalizing 256 slices per slab, matrix size 256   ×   256, NEX   =   1). Images were converted to ANALYZE format using MRIcro. Hippocampal segmentation was performed with the fully automated Freesurfer image analysis package ( http://surfer.nmr.mgh.harvard.edu/ ). This processing includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including the hippocampus), intensity normalization, tessellation of the gray matter – white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (for additional details, please see  Fischl and Dale, 2000 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24501700">PMID:24501700</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). Freesurfer morphometric procedures have been demonstrated to show good test – retest reliability across scanner manufacturers and across field strengths and were preferred in our analysis for this reason ( Han et al., 2006 ). The technical details of these procedures have been previously described elsewhere ( Grieve et al., 2011b ). Briefly, for each participant, the boundary between the GM and WM and the outer surface of the cortex (the pial surface) was segmented. Cortical thickness measurements at each point across the cortical mantle were calculated as the closest distance from the GM/WM boundary to the GM/cerebrospinal fluid boundary. Each participant dataset was then normalized to an average reference surface template in standard space  –  provided with Freesurfer  –  to allow for cortical thickness evaluations at every surface point across participants. Registration was performed in a spherical surface-based coordinate system that is adapted to the folding pattern of each individual dataset, allowing a much higher localization accuracy of structural features of the brain across participants ( Fischl et al., 1999 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24498354">PMID:24498354</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The MRI scans were analyzed using the FreeSurfer software package (version 5.0  http://surfer.nmr.mgh.harvard.edu ) based on established processing steps (recon-all pipeline). The cortical volumes and thickness information of the brain regions were estimated using the automated parcellation of the Desikan-Killiany Atlas  [40]  and exported to SPSS for further analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24498184">PMID:24498184</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Frontal regional cortical and limbic system volumes were determined using Freesurfer image-analysis software ver. 4.0 ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of those procedures have been described previously  [33] . Cortical volume combined both thickness and area information, and subcortical volume was based on voxel numbers. Parcellation of the cerebral cortex into units was based on gyral and sulcal structures  [34] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24490167">PMID:24490167</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The structural MRI images were converted from the scanned images into the Analyze format, which were further processed using the Free Surfer software package (Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Harvard University,  http://surfer.nmr.mgh.harvard.edu/ ). The software reformats each brain volume image into a volume image with 1 ? mm 3  isovoxels, from which relevant brain structures can be delineated [ 21 – 26 ]. This automated method has been shown to be statistically indistinguishable from manual raters and reduces random errors, rater error, and intersubject variability typical of manual techniques [ 24 ]. Fractional Anisotropy (FA) maps were acquired from the DTI images from the soft ware DTI Studio [ 27 ] and were then coregistered automatically to the MP-RAGE images using a mutual information cost function and a 12 parameter affine transformation. Eddy current correction was performed prior to registration. As the DTI images are co-registered to the subjects' structural images, FA images were also automatically delineated for the separate brain structures using the same delineation parameters in the structural images and FA parameters of relevant brain structures were obtained.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24489639">PMID:24489639</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>HWA in FreeSurfer (Version 4.5.0;  http://surfer.nmr.mgh.harvard.edu/ ) was used in this evaluation. In accordance with  [3] ,  [27] , we investigated:  “ default ” ,  “ -less ”  (shrink the surface),  “ -more ”  (expand the surface),  “ -atlas ”  (use basic atlas information to correct the result),  “ -less -atlas ”  and  “ -more -atlas ” .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24478691">PMID:24478691</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>21 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24478669">PMID:24478669</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We concentrated  a priori  on the left and right amygdalae as ROIs. Volumes of these structures were acquired with an automatic subcortical segmentation procedure (Fischl et al.,  2002 ) in FreeSurfer v5.0 (Massachusetts General Hospital, Boston, MA;  http://surfer.nmr.mgh.harvard.edu ; Dale et al.,  1999 ; Fischl et al.,  1999 ). This procedure uses signal intensities and an  a priori  probabilistic atlas as well as information from neighboring voxels to establish an accurate subcortical segmentation (Fischl et al.,  2002 ). This segmentation method has been shown to perform comparably to manual segmentations (Fischl et al.,  2002 ) and has been used in previous studies in AD (Desikan et al.,  2006 ,  2009 ; Morey et al.,  2009 ). One-way ANOVAs were used to test for differences in the mean volumes of the left and right amygdalae and hippocampi in the three diagnostic groups using age as a covariate.  Post-hoc  comparisons were performed using one-tailed independent samples'  t -tests.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24475088">PMID:24475088</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The regions-of-interest (ROIs), left and right hippocampus, were automatically segmented using FreeSurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu ) which procedure is based on an atlas of probabilistic information computed from a manually labelled dataset  [20] ,  [21]  based on the Duvernoy atlas  [22] . The hippocampi of all subjects were transformed into the Talairach space and were combined to perform the RFX analysis in BrainVoyager QX 2.3 (Brain Innovation, Maastricht, The Netherlands).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24474988">PMID:24474988</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MR imaging was performed on a 1.5 Tesla Espree MRI scanner (SIEMENS, Erlangen, Germany). 3D T1-weighted magnetization-prepared rapid gradient echo (MPRAGE) data were collected with the following parameters: TR=1500 ms; TE=3.00 ms; Inversion time=1100 ms; FOV=256 × 256 mm; Flip angle=15 ° ; 128 slices; 1.0 × 1.0 × 1.33 mm voxel size. Cortical thickness was measured using FreeSurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu ). The standard protocol of the package uses intensity normalization, white matter segmentation, tessellation of the gray/white matter junction, inflation of the folded surface with correction of topological defect, and the measurement of the cortical thickness as defined by the distance between the grey-white matter junction and the pial surface. All images were aligned to 1-mm isotropic stereotaxic standard space (Montreal Neurological Institute, MNI 305 template) and smoothed with a Gaussian kernel of 15 mm full-width at half maximum (FWHM).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24473557">PMID:24473557</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer package is documented and freely available for download online ( https://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferWiki ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24466177">PMID:24466177</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness on T 1 -weighted image was estimated at each vertex across the brain surface using Freesurfer software package ( http://surfer.nmr.mgh.harvard.edu ). This method has been previously described in detail by Fischl and Dale  [21] . Specifically, cortical reconstruction process included magnetic field inhomogeneity correction, affine-registration to Talairach-atlas and skull-strip, segmentation of the subcortical white matter and deep grey matter volumetric structures, intensity normalization, tessellation of the grey and white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the grey and white matter and grey/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class. Once the cortical reconstruction process is complete, registration to a spherical atlas to match cortical geometry across subjects, and subsequent parcellation of the cerebral cortex into units based on gyral and sulcal structure were performed. Cortical thickness measurements were obtained by reconstructing representations of the gray/white matter boundary and the cortical surface. The distance between these two surfaces was calculated individually at each point across the cortical mantle: for each point on the white matter surface, the shortest distance to the pial surface was first computed; next, for each point on the pial surface, the shortest distance to the white matter was found, and the cortical thickness at that location was set to the average of these two values. All subjects were aligned to a common surface template using a high-resolution surface-based averaging technique that aligns cortical folding patterns. The spatial cortical thickness distribution was smoothed with a circularly symmetric Gaussian kernel of 10 mm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24465557">PMID:24465557</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness, defined as the shortest distance between gray/white matter boundary and the spiral surface at each point across the cortical mantle  [26] , was automatically estimated by FreeSurfer (software version 5.0,  http://surfer.nmr.mgh.harvard.edu ), which is the most widely used method  [27] . The technical details of these procedures have been described elsewhere  [26] ,  [28] . Briefly, the process consists of correction for motion artifact, averaging over multiple T1 images, removal of non-brain tissue, Talairach transformation, white matter and subcortical gray matter segmentation, tessellation of the gray matter-white matter boundary, automatic correction for topological defects, intensity normalization, surface deformation, and parcellation of the cerebral cortex into anatomical regions. Total intracranial volume (TIV) was also automatically calculated by FreeSurfer software  [29] . Through careful inspection of all raw images at segmented and inflated stages, we confirmed that no images had substantial defects. For further analysis, cortical maps were smoothed using a Gaussian kernel with a full width at half maximum of 10 mm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24454765">PMID:24454765</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI-data were processed with the surface and volume pipeline of the FreeSurfer-software version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu )  [25] ,  [26]  and estimates of cortical thickness, GM- and WM- volume, cortical surface area, the 3-D local gyrification Index (lGI) and estimated intracranial volume (eTIV) were obtained. The standard FreeSurfer pipeline was followed and automatically reconstructed surfaces were inspected for accuracy and if necessary, manual interventions using FreeSurfer correction tools were used.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24432000">PMID:24432000</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24427112">PMID:24427112</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We performed subject-specific cortical segmentation, skull stripping and extraction of 3D pial surfaces for further analysis using the Freesurfer image analysis suite, which is fully automatic and freely available for Unix and Macintosh platforms at ( http://surfer.nmr.mgh.harvard.edu/ ). The entire process took about 20 h per patient in a quad-core processor, 2.0 Mhz Intel i5 PC, with 4 Gb of RAM memory.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24416245">PMID:24416245</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We employed the LONI pipeline for image preprocessing, cortical surface modeling and gray matter thickness analysis  [19] ,  [20] . Following a de-identification step, the structural neuroimaging data were converted from Digital Imaging and Communications in Medicine to ANALYZE 7.5 format, skull-stripped using the LONI Skull-Stripping Meta Algorithm pipeline workflow  [28]  and cortical surface models were generated using FreeSurfer 4.0  [29]  ( http://surfer.nmr.mgh.harvard.edu/fswiki  and  http://ucla.in/xSQPqT ). Cortical grey matter thickness was computed at each point of the surface using the distance from the pial surface to the nearest point on the white matter surface. For numerical implementation, we first built the signed distance function  [30] ,  [31]  of the white matter surface in 3D space and then computed the CT as the value on the signed distance function at those locations. The cortical surfaces and the corresponding CT maps were registered to the International Consortium for Brain Mapping (ICBM) brain surface  [32]  and then vertex-wise correspondences were established between all cortical surface models using a Conformal Metric Optimization method  [33] . An experienced human brain researcher rated each brain surface reconstruction by visually inspecting the surfaces using LONI ShapeViewer ( http://www.loni.ucla.edu/Software/ShapeViewer ). The quality of surface reconstruction and accuracy of vertex labeling were assessed on the scale of 0 to 1 (0 ? = ? completely unacceptable; 1 ? = ? perfectly reconstructed and labeled). A threshold of 0.7 was selected as the criterion to reconstruct a subject's surface data to be included in the final analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24416184">PMID:24416184</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was measured automatically using FreeSurfer5.0 ( http://surfer.nmr.mgh.harvard.edu/ ,  [31] ,  [32] . Details of the surface-based cortical reconstruction and subcortical volumetric segmentation procedures have been extensively documented previously  [31] – [34] . Briefly, the FreeSurfer pipeline performs motion correction on the T1-images, automatically removes non-brain tissues  [34] , transforms volumetric data to a common atlas, performs intensity normalization and topology correction  [33] ,  [35]  and defines the boundaries of the gray/white matter and pial surface  [31] ,  [32] . Volumetric subcortical segmentation and measurement was performed using automated procedures that have been validated as comparable in accuracy to much slower, labor-intensive manual tracing and labeling methods  [33] ,  [36] . This procedure automatically classifies brain tissue into multiple distinct structures such as cerebral and cerebellar gray and white matter, cerebrospinal fluid (CSF), basal ganglia, and other subcortical structures. Using probabilistic information derived from a manually labeled training data set, this approach automatically assigns a neuroanatomical label to each voxel in the MRI volume. For the purposes of the current study, automated image surfaces and segmentations were inspected and screened for quality control but were not manually edited, in order to maintain the objectivity of results. Of note, large deformities such as failure to segment or include the entire brain were excluded from the study (N ? = ? 20, not included in  Table 1 ). Intracranial volume was determined by a validated automated method known to be equivalent to manual intracranial volume estimation  [37] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24416143">PMID:24416143</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In this study, the MRI features, denoted as  ‘ M ’ , were based on the imaging data from ADNI database processed by the UCSF team. They performed cortical reconstruction and volumetric segmentations with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The processed MRI features can be grouped into 5 categories: average cortical thickness (CTA), standard deviation in cortical thickness (CTStd), the volumes of cortical parcellations (Vol.Cort), the volumes of specific white matter parcellations (Vol.WM), and the total surface area of the cortex (Surf.A). There are 305 MRI features in total. We also use four kinds of features, including demographic and genetic information, baseline cognitive scores, and lab tests. We use  ‘ META ’  to denote all these four types of features. There are 52 META features in total (shown in  Table 1 ). The baseline cognitive scores are obtained when the patient performs the screening in the hospital for the first time. Here we also consider the performance by using the remaining 22 META features without the baseline cognitive scores, denoted as  ‘ META-22 ’ .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24416009">PMID:24416009</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural analysis based on segmentation of brain structures from T1 high-resolution anatomical data was performed using the freely available Freesurfer toolkit version 5.0 ( http://surfer.nmr.mgh.harvard.edu ). Intracranial volume (ICV) was used to correct the volumes and the processing pipeline was the same as previously described (Soares et al.,  2012 ). DMN was defined by the summed volume of the angular gyrus of inferior parietal lobe, the posterior cingulate, the precuneus, and the frontopolar region (Raichle et al.,  2001 ; Buckner et al.,  2008 ). The summed volume of the middle frontal gyrus (dorsolateral and prefrontal region) and the posterior parietal region constituted the DAN (Seeley et al.,  2007 ; Sridharan et al.,  2008 ). VAN was constituted by the sum of the temporal-parietal junction and the ventral frontal cortex volumes (Fox et al.,  2006 ). SMN was defined by the summed volume of the paracentral, precentral postcentral, and the cerebellum (Shirer et al.,  2012 ). The summed volume of the cuneus, pericalcarine, and the lingual region constituted the primary VN (Shirer et al.,  2012 ). Auditory network (AN) was defined by the summed volume of the temporal transverse and the temporal superior (Shirer et al.,  2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24409139">PMID:24409139</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural connectomes were generated using a pipeline which integrates multiple image analysis techniques and has been reported elsewhere (GadElkarim et al.,  2012 ; Leow et al.,  2012 ). In brief, DW images were eddy current corrected using the automatic image registration (AIR) tool embedded in DTIStudio software ( http://www.mristudio.org ) by registering all DW images to their corresponding b0 images with 12-parameter affine transformations. This was followed by the computation of diffusion tensors and deterministic tractography using the DTIStudio program. T1-weighted images were used to generate label maps using the Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ). Brain networks formed by the 82 cortical/subcortical gray matter regions were generated using an in-house program in Matlab by counting the number of fibers connecting each pair of nodes.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24407503">PMID:24407503</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Realistic TD-NIRS datasets were generated using GPU-based Monte Carlo simulations on adult heads whose structures were obtained from MRI anatomical scans. Three subjects were selected from a library of 32 head volumes, previously used by Cooper et al. to validate the use of atlas-based image reconstructions of functional data. 40  We chose three subjects which resulted in poor, average, and good performance in the functional NIRS study. The head volumes were segmented using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) 41 , 42  into four tissue types: extracerebral tissue (skin and skull), cerebrospinal fluid (CSF), gray matter, and white matter. CSF, gray, and white matters were subsequently combined into a single brain tissue type. The details of the MRI scan preprocessing steps and of the segmentation procedure have been previously described. 40</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24392272">PMID:24392272</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). Briefly, this processing includes removal of nonbrain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white-matter and deep gray-matter volumetric structures, and intensity normalization for the TOF contrast. Mean intensities and standard deviations were calculated for each volume in both participants and used for nonparametric testing (Wilcoxon rank-sum test) followed by a correction for multiple comparisons (Holm  1979 ), adjusting  P -values for testing multiple hypotheses on effects pertaining to the 14 selected subcortical structures.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24392009">PMID:24392009</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated cortical and subcortical volume measures were performed with FreeSurfer software package, version 4.4 ( http://surfer.nmr.mgh.harvard.edu/fswiki ) [ 25 , 26 ]. To reduce the confounding effect of intra-subject morphological variability, each subject ’ s longitudinal data series was processed by FreeSurfer longitudinal workflow. For a full description of the FreeSurfer processing steps, see  25 , 26 , and for a full description of the longitudinal workflow, see  http://surfer.nmr.mgh.harvard.edu/fswiki/LongitudinalProcessing . All images underwent standardized quality control procedures. Subjects with complete segmentation failure or gross errors throughout all brain regions were rated as complete failure and the ones with gross errors in one or more specific brain regions (i.e., temporal lobe regions, superior regions, occipital regions, and insula) were given partial pass rating. All the subjects with passing quality control rating were included in analyses presented in this work. Details of our quality control procedures are posted online at  http://www.loni.ucla.edu/twiki/pub/ADNI/ADNIPostProc/UCS-FFreeSurferMethodsSummary.pdf .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24386081">PMID:24386081</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were processed using the FreeSurfer package (version 4.5.1,  http://surfer.nmr.mgh.harvard.edu ) to estimate cortical gray matter volume per ROI. Brain mask images obtained from the VBM8 toolbox were introduced into the default FreeSurfer processing pipeline because they provide more accurate skull stripping.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24376742">PMID:24376742</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The whole schematic flowchart of cortical structural network construction is shown in  Figure 1 . For each subject, freesurfer toolkit ( http://surfer.nmr.mgh.harvard.edu ) was used to analyze the anatomical MPRAGE images. The freesurfer procedure involving white matter segmentation  [18] , subcortical mass labeling and filling  [19] , tessellation and topology fixing, topological defects correction  [20] , white and pial cortical surface generation  [21] , and cortical thickness measurement  [21] . The cortical thickness at each vertex was measured as the shortest distance to the white surface from each point on the pial surface averaged with the shortest distance to the pial surface from each point on the white surface.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24376719">PMID:24376719</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1-weighted structural image was parcellated into 35 cortical regions per hemisphere using the Freesurfer software package as available at  http://surfer.nmr.mgh.harvard.edu/ . The steps involved are brain extraction, tissue segmentation, and cortical parcellation based on image intensity (gradients) in a template-driven, fully automated and highly robust way  [19] ,  [20] ,  [21] . The resulting cortical parcellation of in total 70 regions is gyral pattern based and includes, among others, the left and right pre- and postcental gyrus (henceforth referred to as the 4 rolandic regions).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24349280">PMID:24349280</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measures : We measured the thickness of the neocortex throughout the whole brain on 3D MPRAGE images using the automated reconstruction algorithm of the software FreeSurfer, version 4.5 ( http://surfer.nmr.mgh.harvard.edu )  [36] ,  [37] ,  [38] . The entire cortex of the subjects was visually inspected and in the MS group errors due to misclassifications of WM lesions located close to the cortex were manually edited and automatically corrected. In the MS group, the relationship between performances in each mentalization tests and cortical thickness was computed at each vertex point using general linear models corrected for age  [39] . Cortical regions showing correlation at p < 0.001 were displayed as significance maps on the FreeSurfer averaged brain surface. Regions covering surface area  ? 25 mm 2  were manually outlined (region of interest [ROI]), and automatically mapped back on the brain surface of each patient with MS and participant of the healthy control group recruited for MRI examination. The average thickness of ROIs was calculated for each participant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24343892">PMID:24343892</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used Freesurfer to obtain surface meshes of the boundary between GM and WM from  T 1  anatomical brain images ( http://surfer.nmr.mgh.harvard.edu ) (Fig.   1 ). After registering surface meshes into the DTI space, we generated volume regions of interest (ROIs) based on GM voxels. Freesurfer provides parcellation of 34 anatomical regions of cortices based on the Deskian atlas ( Fischl et al. 2004 ;  Desikan et al. 2006 ) and 7 subcortical regions (Nucleus accumbens, Amygdala, Caudate, Hippocampus, Pallidum, Putamen, and Thalamus) ( Fischl et al. 2002 ,  2004 ) for each hemisphere, thus leading to 82 ROIs in total (See  Supplementary Table S5  for full and abbreviated names of ROIs).\n Figure   1. Overall procedure. From  T 1 -weighted images, we generated 82 regions of interests (ROIs, 34 cortical areas and 7 subcortical areas per hemisphere, on the left). From diffusion tensor images (DTI), we reconstructed streamlines using deterministic tracking (on the right). Combining 2 preprocessing steps, we constructed weighted networks, where the number of streamlines between any pair of ROIs formed the weight of an edge (fiber tract).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24332820">PMID:24332820</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were processed by a single operator, blinded to group status, on Linux workstations using the automated FreeSurfer image processing suite (stable release v. 5.0,  http://surfer.nmr.mgh.harvard.edu ). 19,20  Images were excluded if they were considered of unacceptably poor quality because of artifact and those with significant structural abnormalities that precluded FreeSurfer registration. The magnetic resonance images were visually inspected and manually edited as required during the automated processing pipeline.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24319655">PMID:24319655</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All scans were acquired on a Siemens MAGNETOM Trio 3   T MRI scanner with a 32-channel head coil at the Sherman Health Sciences Research Centre, York University. A T 1  weighted three-dimensional MPRAGE scan of the entire head was acquired sagitally with the following parameters: rapid gradient echo, 1   mm 3  isotropic voxels, TR   =   1900   ms, TE   =   2.52   ms, 256   ×   256   matrix, and flip angle   =   9 ° . All T 1  images were re-oriented on the MRI console (Syngo; Siemens, Munich, Germany) to ensure the optic nerves, optic chiasm, and optic tracts were observed in the same plane and a reformatted image with a 1   mm slice thickness was obtained parallel to the optic chiasm. Processing of these reformatted images was conducted using tools from the freely available FMRIB's Software Library (FSL; version 4.1.8) ( http://www.fmrib.ox.ac.uk/fsl ). Chiasm region-of-interest (ROI) masks were manually traced by three independent raters blind to group membership in FSLview. Chiasm masks from each rater were merged using fslmerge and a median mask was created using fslmaths that was comprised of voxels chosen in at least 2 of the 3 original masks. From the median mask, chiasm volumes were calculated using the fslstats tool. Several chiasm measurements ( Fig.   2 ) were also taken by four independent raters blind to group membership using the OsiriX ®  length measurement tool, which computes selected distances using bicubic interpolation ( Rosset et al., 2004 ): optic nerve diameter (a) ipsilateral and (b) contralateral to the dominant eye for controls and the remaining eye for ME participants, optic tract diameter (c) ipsilateral and (d) contralateral to the dominant/remaining eye, and optic chiasm width in the (x) X and (y) Y planes. Each rater took three measurements and the average of these measurements across raters was calculated per structure. To ensure whole brain volume was not a factor affecting our results, we also calculated whole brain volumes excluding the cerebellum and brain stem using volumetric segmentation performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24278382">PMID:24278382</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to enable regional extraction using T1-weighted anatomical information and to perform whole-brain analysis of QSM data, rigid registration and nonlinear warping to a standard space were performed using SPM8 ( http://fil.ion.ac.uk/spm ). Prior to this, datasets were preprocessed as follows: first, the origin for each dataset's coordinate system was set to the anterior commissure. Then, MPRAGE volumes were skull-stripped using the hybrid watershed algorithm (HWA) with atlas information  [80]  (available from FreeSurfer v5.1.0,  http://surfer.nmr.mgh.harvard.edu ); this was followed by RF-bias correction using N4-ITK  [64] , which performs best in the absence of spurious signals from skin, fat and venous sinuses. Note that RF-bias inhomogeneity correction is known to improve spatial normalisation in the context of voxel-based morphometry  [81] . In this study, however, N4-corrected signal intensities were substituted into the original magnitude image to reconstitute a bias-corrected whole-head dataset, as it was found experimentally that the inclusion of cranial information improved the performance of rigid coregistrations. RF-bias corrected T2*-weighted magnitude information was also inserted into the original image for coregistration. In addition, it was found experimentally that the  “ clean-up partition ”  feature in SPM8's unified segmentation ( “ Segment ” ) implementation was successful at eliminating spurious mis-segmentation of venous sinuses into grey matter partitions, resulting in visually improved tissue maps and spatial warping; this option was therefore enabled in the present study.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24278317">PMID:24278317</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>CTh analysis was conducted with the freely available FreeSurfer software package (version 5.1.0), a set of automated tools for reconstruction of the brain ’ s cortical surface from MRI data ( http://surfer.nmr.mgh.harvard.edu )  [9] . The processing procedures require 3D T1 weighted MRI data. After skull stripping, segmentation into white and grey matter and tessellation, CTh measurements were obtained by reconstructing representations of the grey/white matter interface and the cortical surface and then by calculating the distance between these surfaces at each point across the cortical mantle  [24] .  Figure 1  shows the boundaries of cerebral cortex, i.e. the white matter surface and the pial surface, as defined by the software. By inflating the cortical sulci, three-dimensional inflated and spherical white matter models were created for spatial normalization and vertex-based statistical analysis  [25] . The cortex was parcellated in 34 different regions for each hemisphere according to the brain atlas included in Freesurfer (Desikan/Killiany Atlas) and average measures of CTh were produced at each point on the reconstructed surface (vertex)  [26] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24273724">PMID:24273724</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer image analysis suite (version 5.0,  http://surfer.nmr.mgh.harvard.edu ) surface-based processing pipeline was used to derive measures of volume, CT, and SA. FreeSurfer is a well-validated processing protocol that has been previously described in detail ( Dale et al., 1999; Fischl et al., 1999 ). In short, the following steps were taken in the processing stream: motion correction, transformation of images to standard Talairach space, intensity normalization, removal of non-brain tissue, segmentation of white matter and subcortical structures, and final segmentation of cortical surfaces. Final segmentation is based on both a subject-independent probabilistic atlas and subject-specific measured values. Raters (MJ, AP, RJ) blind to diagnosis visually inspected the scans at several points along the processing pipeline and any errors were manually edited. Details regarding the manual editing procedures can be found in the  Supplementary text . Using an automated computer algorithm, CT estimates were derived by taking the distance between the gray – white matter border and the pial surface at each vertex ( Fischl and Dale, 2000 ). As in prior publications ( Bhojraj et al., 2011; Gutierrez-Galve et al., 2010; Meda et al., 2012; Venkatasubramanian et al., 2012 ), surface area was calculated by taking the sum of the area of the vertices in each parcellation. Volume was then calculated as the product of the SA and CT for each region.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24273717">PMID:24273717</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and volumetric segmentation was performed in an automated manner using the FreeSurfer image analysis suite (version 4.3,  http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures have been previously described elsewhere ( Grieve et al., 2011 ). Briefly, for each participant, the boundary between the GM and white matter and the outer surface of the cortex (the pial surface) was segmented. Cortical thickness measurements at each point across the cortical mantle were calculated as the closest distance from the GM/white matter boundary to the GM/cerebrospinal fluid boundary ( Fischl and Dale, 2000 ). This method uses both intensity and continuity information from the entire three-dimensional magnetic resonance volume in segmentation and deformation procedures to produce representations of cortical thickness. The surface representations are created using spatial intensity gradients across tissue classes and are therefore not simply reliant on absolute signal intensity. The surfaces produced are not restricted to the voxel resolution of the original data and thus are capable of detecting sub-millimeter differences between groups. Cortical segmentation and labels were manually inspected for accuracy. Each participant dataset was then normalized to an average reference surface template in standard space — provided with FreeSurfer — to allow for cortical thickness evaluations at every surface point across participants. Registration was performed in a spherical surface-based coordinate system that is adapted to the folding pattern of each individual dataset, allowing a much higher localization accuracy of structural features of the brain across participants ( Fischl et al., 1999 ). These images were then entered for statistical evaluation of MDD vs. control groups as described below.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24273708">PMID:24273708</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>White matter seed and target regions of interest (ROIs) for tractography were defined in the T1 MPRAGE structural volumes using a two-step process. First, cortical ROIs were identified according to individual anatomical landmarks using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) automatic image segmentation and cortical reconstruction and labeling. Cortical ROIs were labeled on the FreeSurfer-generated white matter surface using a parcellation system tailored for studies of speech networks ( Tourville, 2003 ). Automated parcellation was followed by review and edits by a neuroanatomical expert (J.A.T.) who was blind to group membership. Second, for each cortical ROI defined in step one, a corresponding WM ROI comprised of the WM voxels lying up to 2   mm below the WM surface adjacent to the cortical ROI was identified. The ROI labels defined in the WM were subsequently transformed into the coordinate space of the diffusion data using FLIRT ( Jenkinson and Smith, 2001 ) for use in tractography analyses. The a priori ROIs, six in each hemisphere, were the ventral premotor cortex (vPMC), supplementary motor area (SMA), ventral motor cortex (vMC), posterior superior temporal gyrus (pSTg), anterior supramarginal gyrus (aSMg), and posterior middle frontal gyrus (pMFg). t-tests were used to detect significant group differences in ROI size; familywise error correction (FWE) for multiple comparisons was computed using permutation tests to insure a false discovery rate of 5% over the entire family of 12 ROI size comparisons.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24273707">PMID:24273707</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and parcellation were performed with the FreeSurfer V4 package (v4.0.2) image analysis suite ( Fischl and Dale, 2000 ), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). This method is fully described ( Du et al., 2007 ) and has been validated via both histological and manual measurements ( Bischoff-Grethe et al., 2007; Pantazis et al., 2010 ). For cortical analysis, the gray matter – white matter interface of each hemisphere was segmented, tessellated, corrected for topological errors and inflated in order to unfold the cortical surface. Each individual hemispheric surface was then registered to a surface-based template using its sulco-gyral pattern to allow inter-subject analysis ( Fischl and Dale, 2000 ). Cortical thickness was calculated from both intensity gradients across tissue class and continuity information, as the closest distance between the gray/white matter boundary and the gray/cerebro spinal fluid interface at each vertex of the tessellated surface ( Fischl and Dale, 2000 ). Automated topology correction was performed as necessary ( Fischl et al., 2001 ). We inspected each dataset at multiple points within the processing stream. We verified the Talairach transform, the accuracy of the skull strip, the accuracy of the white matter and pial surface segmentation, and finally inspected the cortical surface reconstruction for topological defects. Error correction was performed as necessary. Cortical thickness was measured as the distance between the gray/white matter boundary and the pial surface at each point on the cortical mantle. The data for each participant was resampled to an average participant and surface smoothing was performed using the  ‘ qcache ’  command and a 10   mm full-width half-maximum Gaussian kernel, prior to statistical analysis. In this study, we get the cortical thickness values from the statistical output of FreeSurfer cortical parcellation, called  “ aparc.a2005s.stats ” , which uses the Desikan – Killiany atlas, and has 75 regions.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24273501">PMID:24273501</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was performed with the FreeSurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). (Dale et al.,  1999 ; Fischl et al.,  1999 ; Fischl and Dale,  2000 ) Briefly, following skull-stripping and correction of inhomogeneity artifact, constrained region growing was used to create a unitary white matter volume for each hemisphere. The gray-matter/white-matter boundary for each cortical hemisphere was determined using tissue intensity and neighborhood constraints. The white matter surface was tessellated by assigning two triangles to the square face of each surface voxel. This process yielded approximately 160000 vertices per hemisphere. The white matter surfaces were deformed toward the gray matter/pial boundary, with a point to point correspondence at each vertex. Cortical thickness was computed as the distance between the white and the pial surfaces at each vertex. Cross-subject registration of hemispheric cortical surfaces was performed by projecting them onto the spherical representations. The maps produced are not restricted to the voxel resolution of the original images and are thus capable of detecting sub-millimeter differences between groups. The parcellations were obtained using the Desikan sulcogyral-based atlas, which follows the anatomical conventions of Duvernoy. The FS image-processing pipeline was visually inspected and corrected at critical points in order to avoid errors permeating through the subsequent analyses. Procedures for the measurement of cortical thickness have been validated against histological analysis and manual measurements. The Desikan Killiany atlas produces 68 parcellations based on gyri and sulci (Desikan et al.,  2006 ). In addition to the Desikan Killiany atlas parcellation scheme, we also used fine-grained parcellation schemes based on anatomical sulcogyral boundaries including the Destrieux atlas, (148 parcellations) and fine-grained parcellation schemes (200, and 1000 parcellations) that did not follow anatomical conventions described in Echtermeyer et al. (Destrieux et al.,  2010 ; Echtermeyer et al.,  2011 ). The pipeline of the analysis and the parcellation are shown in Figure  2 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24266007">PMID:24266007</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer is a freely available image analysis package ( http://surfer.nmr.mgh.harvard.edu ) that has both cortical surface reconstruction ( Dale et al., 1999 ) and volumetric segmentation ( Fischl et al., 2002, 2004a,b ). In this study, the surface-based cortical thickness is used to measure cortical gray matter atrophy. The images are analyzed first cross-sectionally; then the unbiased longitudinal scheme is applied to improve the consistency ( Reuter et al., 2012 ). XPP or LPP is not applied for FreeSurfer because FreeSurfer has its own longitudinal pipeline. The FreeSurfer version is 5.1.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24265612">PMID:24265612</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24205009">PMID:24205009</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to acquire the lead field matrix for the forward modeling, a source model and a volume conductor model were prepared for each participant. For the volume conductor model, we chose a realistically shaped single-compartment  boundary element model  (BEM), which was constructed from individual anatomical MRI data. The segmentation of the MRI-data and the triangulation of the relevant surfaces were calculated using the watershed algorithm  [105]  in FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) and the MNE software  [106] . We used 5120 triangles per surface. The grid spacing for the source space on the white matter surface was 5 mm. Dipole were oriented perpendicularly to the cortical surface. In the time range of the N100m (70 – 130 ms; see  Fig. 4  for representative spatial distributions) a large proportion of the MEG can be explained by one dipolar source per hemisphere. Therefore, a representative source for the N100m generator (time windows: 70 – 130 ms post stimulus) in the right temporal lobe was localized using a subset of the MEG-channels on the right hemisphere (102 planar gradiometers and 51 magnetometers), by exhaustive search of a equivalent current dipole. The search area was constrained to the right Heschl ’ s gyrus (rHG). Note that we did not intend to find the  “ true ”  N100m generator position, but wanted to acquire a representative dipole, which captured the main features of the habituation of the N100m peak. We will discuss this issue in the  Discussion  section later. The goodness-of-fit (GoF) expressed the proportion of variance of the MEG data explained by the source, and for subjects 1 through 6 amounted to 95%, 93%, 96%, 98%, 97% and 97% at the latency of the peak of the N100m (59%, 76%, 62%, 94%, 81% and 62%, for subjects 1 – 6 respectively, for the entire time window). The lead field matrix of the single source for each participant was calculated using the MNE software  [106] . For the Bayesian inversion, the source activity time courses were downsampled to 125 Hz and the amplitudes were normalized to their maximum.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179837">PMID:24179837</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based cortical analyses were performed using Freesurfer 5.0.0 ( Fischl and Dale, 2000 ) ( http://surfer.nmr.mgh.harvard.edu ). Cortical thickness measures by Freesurfer have been validated against autopsy findings ( Rosas et al., 2002 ) and manual measurements ( Salat et al., 2004 ). Using a surface-based protocol, region-specific atrophy has been shown in various patient populations including small-vessel disease ( de Laat et al., 2012 ), multiple sclerosis ( Sailer et al., 2003 ) and neurodegenerative disease ( Dickerson et al., 2009; Du et al., 2007; Rosas et al., 2002 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179830">PMID:24179830</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural images of each patient were processed for surface-based analysis using the standard recon-all pipeline of the Freesurfer 5.0 suite (see  http://surfer.nmr.mgh.harvard.edu  for details). The pipeline is based on cortical surface modeling, spherical coordinate space morphing, multi-subject registration based on surface curvature, and automated segmentation of cortical regions. Note that these steps are only possible in the absence of lesions. For this reason the Freesurfer analysis was confined to non-lesioned hemisphere. Primary analysis of the non-lesioned hemisphere was performed on the estimated cortical volume at each vertex of the mesh. Region of interest (ROI) analysis was then undertaken using cortical volume within the pre- and postcentral gyri. The boundaries of these ROIs were based on the Desikan atlas ( Desikan et al., 2006 ). Detailed information about recon-all pipeline theory, algorithms and implementation is available at  Fischl and Dale (2000) ,  Fischl et al. (1999a) , and  Fischl et al. (1999b) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179815">PMID:24179815</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the method similar to that of  Leritz et al. (2011) , who examined the association between cerebrovascular risk factors to CT in a normative sample of community dwelling adults ( Leritz et al., 2011 ). Surface extraction, cortical parcellations and thickness computation were performed with the FreeSurfer image analysis suite. The pre-processing was carried out according to documentation, which has been validated and the description is available at ( http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al., 1999; Fischl and Dale, 2000; Fischl et al., 1999 ). Briefly, following skull-stripping and correction of inhomogeneity artefact, constrained region growing was used to create a unitary white matter volume for each hemisphere. The grey-matter/white-matter boundary for each cortical hemisphere was determined using tissue intensity and neighbourhood constraints. The white matter surface was tessellated by assigning 2 triangles to the square face of each surface voxel. This process yielded approximately 160,000 vertices per hemisphere. The white matter surfaces were deformed towards the grey matter/pial boundary, with a point-to-point correspondence at each vertex. CT was computed as the distance between the white and the pial surfaces at each vertex. Cross-subject registration of hemispheric cortical surfaces was performed by projecting them onto the spherical representations. The maps produced are not restricted to the voxel resolution of the original images and are thus capable of detecting sub-millimetre differences between groups. Procedures for the measurement of CT have been validated against histological analysis and manual measurements. In addition, FreeSurfer morphometric procedures have demonstrated good test – retest reliability across scanner manufacturers and across field strengths and across various sequence parameters.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179811">PMID:24179811</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI images were processed to create individualized models of each participant's cortical mantle through the cortical reconstruction procedure in the FreeSurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications ( Brands et al., 2006; Dale et al., 1999; Fischl and Dale, 2000; Fischl et al., 1999a, 2001, 2002, 2004a, 2004b ). Briefly, this processing first involves motion correction and averaging of the two volumetric T1-weighted images, removal of non-brain tissue using a hybrid watershed/surface deformations procedure ( Segonne et al., 2004 ), automated Talairach transformation, intensity normalization ( Sled et al., 1998 ), tessellation of the gray matter white matter boundary, automated topology correction ( Fischl et al., 2001; Segonne et al., 2007 ), and surface deformation following intensity gradients to optimally place the gray/white and gray/CSF borders at the location where the greatest shift in intensity defines the transition to the other tissue class ( Dale et al., 1999; Fischl and Dale, 2000 ). Surface inflation and spherical registration procedures based on the matching of cortical folding patterns across individuals were used for group-based analyses ( Fischl et al., 1999a, 1999b ). Cortical thickness was computed as the closest distance from the gray matter/white matter boundary to the gray matter/cerebrospinal fluid (CSF) boundary at each vertex on the tessellated surface ( Fischl and Dale, 2000 ). In FreeSurfer the cortical surface boundaries are occasionally slightly misplaced with regard to the boundary of the gray matter/white matter and/or gray matter/CSF border and occasionally need to be manually edited for very small adjustments. Due to this, cortical models are checked slice-by-slice throughout the entire brain image to assure that surfaces are appropriately placed at the gray/white and gray/CSF border (EL). Thickness data was smoothed using a circularly symmetric Gaussian kernel across the surface with a standard deviation of 20   mm. The reliability of obtaining cortical thickness measurements from MRI scans has been well documented in several instances ( Dickerson et al., 2008; Han et al., 2006; Jovicich et al., 2009 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179799">PMID:24179799</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To enable surface-based analysis of fMRI data, anatomical data were processed using FreeSurfer ( https://surfer.nmr.mgh.harvard.edu ) to construct models of the cortical surfaces for each subject ( Dale et al., 1999; Fischl et al., 1999a ). These were then registered to a surface-based group template ( Fischl et al., 1999b ), and aligned with the MNI305 brain ( Collins et al., 1995 ). fMRI data was analyzed with FS-FAST ( https://surfer.nmr.mgh.harvard.edu/fswiki/FsFast ). Preprocessing involved motion correction using the AFNI (afni.nimh.nih.gov/afni) 3dvolreg program ( Cox and Jesmanowicz, 1999 ), and slice-timing using the FSL ( www.fmrib.ox.ac.uk/fsl ) slicetimer program. Non-brain voxels were masked out using the FSL Brain Extraction Tool ( Smith, 2002 ). The middle timepoint was registered to the subject's anatomical image using the FreeSurfer bbregister program ( Greve and Fischl, 2009 ). Using this registration, the raw fMRI time series was mapped to the three group analysis spaces: (1) left and (2) right hemisphere group surface space, and (3) MNI305 space within a subcortical mask of gray matter structures. Data was spatially smoothed on the surface by 5   mm full-width/half-max (FWHM) using an iterative technique ( Hagler et al., 2006 ), and in the volume using a 3D smoothing kernel of 5   mm FWHM.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179782">PMID:24179782</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Eddy current correction of the DTI data was performed using the eddy correct tool in FSL (version 4.0.1) ( Smith et al., 2004 ) and the Camino toolkit (version 2, release 767) was used to calculate fractional anisotropy (FA) images for each subject. Automated segmentation of the T1-weighted volumetric scans using Freesurfer was used to identify the hippocampi ( http://surfer.nmr.mgh.harvard.edu/ ). The b   =   0 image from the DTI scan was affinely registered to the T1 volumetric scan using FLIRT ( http://www.fmrib.ox.ac.uk/fsl/flirt/index.html ). The inverse transformation was applied to the segmentation to give the hippocampal seeds in native diffusion space. Tractography was performed from the hippocampal seeds employing a multi-tensor model with 1000 iterations from each voxel within the seed region as previously described ( Winston et al., 2012 ). The connectivity distributions generated for each subject were thresholded at 0.05 and the mean FA of the areas to which each hippocampus was connected was determined to give a global measure of hippocampal structural connectivity used for subsequent analyses ( Fig.   1 ). Greater mean FA was interpreted as a measure of greater structural connectivity of the hippocampus.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179779">PMID:24179779</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer software version 4.5.0 ( Dale et al., 1999; Fischl et al., 1999 ) ( http://surfer.nmr.mgh.harvard.edu/ ) was utilized. Briefly, all images underwent skull-stripping and transformations into Talaraich space, followed by subcortical segmentation, identification of the gray/white boundary, automated topology correction, and surface deformation ( Dale et al., 1999; Fischl and Dale, 2000 ). The resulting cortical models were registered to a spherical atlas ( Fischl et al., 1999 ). The cerebral cortex was parcellated into regions based on gyral and sulcal structure. The atlas used for parcellation was created using 40 normal and AD participants. FreeSurfer outputted volume of hippocampus and amygdala, and cortical thickness for all other ROIs. Vertex-level comparisons of cortical thickness were also performed across groups using two sample t-tests after the spherical data was smoothed at 10   mm full-width at half-maximum (FWHM). A very lenient Statistical threshold of p   <   0.005 uncorrected for multiple comparisons was applied.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179768">PMID:24179768</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures have been described and validated in prior publications ( Dale et al., 1999; Fischl and Dale, 2000; Fischl et al., 1999a, 1999b ). Briefly, the processing included skull stripping, Talairach transformation, optimization of the grey matter-white matter and grey matter – CSF boundaries, segmentation, and tessellation ( Dale et al., 1999; Fischl and Dale, 2000 ). The tessellated surfaces were then inflated and registered to a spherical atlas which allowed for parcellation of the cerebral cortex into units based on gyral and sulcal structure, and creation of surface based data, including maps of curvature and sulcal depth ( Fischl et al., 1999a, 1999b ). Cortical thickness was calculated as the closest distance from the grey/white boundary to the grey/CSF boundary at each vertex on the tessellated surface ( Fischl and Dale, 2000 ). Cortical gyri were parcellated as described by  Desikan et al. (2006)  for region-of-interest analyses. To measure changes in cortical thickness and volume between the initial and follow-up scans, images were processed using the longitudinal stream in FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/fswiki/LongitudinalProcessing ) ( Reuter et al., 2012 ). Briefly, this processing steam creates an unbiased within-subject template space and image using a robust inverse consistent registration ( Reuter et al., 2010 ). Skull stripping, Talairach transformation, spherical maps, and parcellation are then initialized from the within-subject template for each subject's time point, utilizing common information and thereby increasing reliability and statistical power ( Reuter et al., 2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179750">PMID:24179750</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A post-hoc analysis was carried out using baseline and one-year bilateral cerebral white matter volumes (cc) based on the imaging data from the ADNI database processed by the UCSF team, who performed volumetric segmentations with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). Volumes for 355 of the 359 Caucasian subjects at baseline and 303 of the 307 Caucasian subjects at one-year were reported to have passed quality control and were therefore used in the analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24171759">PMID:24171759</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Morphometric analysis of brain structure was completed with FreeSurfer Version 5.1 ( http://surfer.nmr.mgh.harvard.edu ). Detailed methodology for regional and total volume derivation has been described in detail previously [ 28 - 30 ]. FreeSurfer was used to perform image preprocessing (e.g. intensity normalization, skull stripping), then to provide both cortical and subcortical volume measures using the surface stream and the subcortical segmentation stream respectively. FreeSurfer performs such parcellations by registering images to a probabilistic brain atlas, built from a manually labeled training set, and then using this probabilistic atlas to assign a neuroanatomical label to each voxel in an MRI volume. Total brain volume, total gray matter volume, and volume of the thalamus and brain stem were all automatically derived with the subcortical processing stream (i.e.,  “ aseg.stats ” ). Intracranial volume was also automatically derived and served as a covariate in analyses examining MRI indices in order to control for interindividual differences in head size. A summary composite was computed for the left and right hemisphere volumes of the thalamus.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24155730">PMID:24155730</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Available T1-weighted MRI images of all participants, obtained with a 3-T Magnetom Trio scanner (Siemens), formed the basis for source reconstruction. The freely available software Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) was used to construct individual cortical representations. First, non-cerebral tissue was stripped off (S é gonne et al.,  2004 ). Secondly, the white matter was segmented and the boundary between gray and white matter was estimated. Finally, individual topological representations of the cortical surface were obtained (Dale et al.,  1999 ). Co-registration of MRI and MEG coordinate systems was accomplished using the HPIs and about 100 digitized points on the head surface (acquired by a Polhemus FASTRAK 3D digitizer).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24146656">PMID:24146656</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical source estimates were computed using the Minimum Norm Estimates (MNE) (Hamalainen et al.,  1993 ) in MNE Suite software ( http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/sofMNE.php ). Source dipoles were evenly distributed (5 mm separation between neighboring sources) with orientations fixed normally to the cortical surface. Surface brain models for each individual participant were constructed by Freesurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) from structural magnetic resonance imaging scans acquired at the Scientific Imaging and Brain Research Center at Carnegie Mellon University (Siemens Verio 3T, T1-weighted MPRAGE sequence, 1  ×  1  ×  1 mm, 176 sagittal slices, TR = 1870 ms, TI = 1100 ms, FA = 8 degrees, GRAPPA = 2). Based on the neural anatomy of each individual participant, 24 ventral visual and prefrontal cortical regions containing multiple source dipoles were identified from Freesurfer segmentation using the Desikan-Killiany Atlas (Desikan et al.,  2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24144356">PMID:24144356</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Five children with PWS were excluded from the MRI analysis due to motion artifacts, and failure of proper pial and white surface reconstruction by FreeSurfer (Athinoula A Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Boston, MA, USA;  http://surfer.nmr.mgh.harvard.edu ), leaving 20 eligible patients for analysis (11 children with DEL, 9 children with mUPD).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24141314">PMID:24141314</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data analysis was performed using FREESURFER and FS-FAST software ( http://surfer.nmr.mgh.harvard.edu/ ), the custom  “ jip ”  tookit provided by Joseph Mandeville ( http://www.nitrc.org/projects/jip/ ), and custom scripts written in Matlab. The surfaces of the high-resolution structural volumes were reconstructed and inflated using FREESURFER; functional data were motion corrected with the AFNI motion correction algorithm 57 , spatially smoothed with a Gaussian kernel (fwhm = 2 mm; except for  Supplementary Figure 5  which used no spatial smoothing), and registered to each animal ’ s own anatomical volume using jip. Following standard alert monkey fMRI processing techniques, images were first normalized to correct for signal intensity changes and temporal drift, and t-tests uncorrected for multiple comparisons were performed to construct statistical activation maps based on a general linear model 2 , 7 , 8 , 13 , 14 , 31 . Activation was thresholded at significance levels indicated in the figures by a color scale bar (values show the common logarithm of the probability of error, and are uncorrected). Activation maps were then projected on high resolution anatomical volumes and surfaces. Time courses were calculated by first detrending the fMRI response. The temporal drift often associated with fMRI signals was modeled by a second order polynomial: \n x ( t ) = s ( t ) + a t 2 + b t + c ,  where  x ( t ) was the raw fMRI signal, and  s ( t ) was the detrended signal. The coefficients  a ,  b  and  c  were calculated using the Matlab function  polyfit . The percentage deviation of the fMRI signal,  s ? ( t ), reported as the y-axis values in the time-course traces, was calculated by: \n s ? ( t ) = 100 × ( s ( t ) - s ¯ ) / s ¯ ,  where  s ( t ),  t  = 1, 2,  … N ,  s ?  is the mean of  s ( t ), and  N  is the number of TRs in the experiment.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24133436">PMID:24133436</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structurally defined right and left anterior insular cortex ROIs were used to investigate BOLD signal changes within the greater anterior insular cortex volumes during neurofeedback and to assess for laterality effects during neurofeedback training. These ROIs were defined using the anterior short insular gyrus as derived from probabilistic labeling of the SPM ( ) single subject average image based on the Destrieux atlas in Freesurfer ( ). Both ROIs were smoothed using an 8 mm FWHM kernel analogous to that used for the rtfMRI data analysis and then intensity filtered to limit the overall size of the ROI, approximate the structures in Talairach space, and minimize artifactual increases in statistical thresholds due to large surface areas relative to volumes (see  Figure  1E ). A lateralization index (LI) was calculated for each subject using a normalized difference between percent signal change extracted from the target (%RIC) and contralateral ROI (%LIC) using the equation: LI = (%RIC - %LIC)/(%RIC+%LIC), as has been applied elsewhere ( Caria et al., 2007 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24130887">PMID:24130887</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were processed by a single operator, blinded to group status, on Linux workstations using the automated FreeSurfer image processing suite (stable release version 5.0,  http://surfer.nmr.mgh.harvard.edu ) [ 16 , 17 ]. Images were visually inspected for artifacts and abnormalities that could compromise automated processing. Images that were considered of unacceptably poor quality due to artifact or major cerebral structural abnormalities that affected registration were excluded (EP n=14, control n=10). During the automated image processing pipeline, the MR images were visually inspected and manually edited as required.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24130724">PMID:24130724</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We performed shape analysis of subcortical structures by measuring relative deformation of subcortical surface meshes against the template mesh. The shape analysis process consists of four steps: volume parcellation, surface extraction, registration, and surface deformity computation (see  Figure 1[a] ). The first step obtains the anatomical parcellations of human subcortical structures from the T1 images of each patient using the FreeSurfer software package (Version 5.0, Athinoula A. Martinos Center at the Massachusetts General Hospital, Harvard Medical School;  http://www.surfer.nmr.mgh.harvard.edu/ ). The parcellated images were then transformed to native anatomical space for surface extraction. The second step extracts surface meshes of the subcortical structures for each subject by deforming the template surface models. Specifically, we utilized the subcortical shape atlas models  [20]  as a template surface and used the Laplacian-based surface deformation method  [21] ,  [22]  in order to extract the subcortical surfaces of each subject. The third step involved surface registration in order to establish the vertex correspondence of subcortical surface meshes across the sample. We used the surface registration method developed by Cho et al. (2012) for accurate registration. Finally, surface deformity of each subcortical structure was measured against the template surface model by calculating the vertex-wise spatial displacement along its outward normal direction. In order to correct the brain size effect when computing the shape deformity data, each subcortical surface mesh was transformed from native anatomical space to the template space using the inverse of the registration mapping.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24121634">PMID:24121634</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each subject ’ s anatomical MRIs were segmented and co-registered to volume-based and surface-based standard brain representations using FreeSurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu ). The TMS target locations were selected based on a sample of previously published maximally-activated fMRI  10 ,  11 ,  13 – 16  and positron emission tomography  51  voxels during audiospatial or sound-object identity processing. More specifically, the posterior AC targets were defined by averaging the peak-voxel Talairach coordinates of contrasts reflecting sound movement vs. rest  11 , sound direction changes vs. constant stimulation  13 , sound distance vs. intensity changes  14 , and spatially shifting vs. stationary sounds  10 . When needed, MNI coordinates were converted to Talairach coordinates. The anterior AC targets were defined similarly, based on peak voxels in contrasts reflecting varying vs. fixed pitch  10 ,  15 ,  16 , fixed pitch  vs.  noise  15 , fixed Huggins pitch vs. noise  16 , fixed binaural band pitch vs. noise  16 , and pitch-strength-by-melody interactions  51 . For both targets, average loci were calculated first within, then across studies. The resulting Talairach coordinates [mm] of initial targets were { x,y,z }={ ? 52, ? 30,12} for the left posterior, { x,y,z }={ ? 54,0, ? 5} for the left anterior, { x,y,z }={60, ? 30,11} for the right posterior, and { x,y,z }={55, ? 4, ? 6} for the right anterior AC ( Fig. 1b ). These locations were then transformed to each individual subject ’ s brain representations. Finally, before the TMS experiment, an optimized entry point closer to the inner skull was selected, as guided by the E-field estimates produced by the navigation system, to allow the TMS effects to be maximally focused to the target area.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24121203">PMID:24121203</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Subcortical volume estimation was performed with the Freesurfer 5.1 image analysis suite using the programme's automated segmentation procedure ( http://surfer.nmr.mgh.harvard.edu/ ). This procedure has been described in detail previously ( Fischl et al., 2002 ), and is summarized here. An optimal linear transform is computed that maximises the likelihood of the input image, given an atlas constructed from manually labelled images. A nonlinear transform is then initialized with the linear one, and the image is allowed to further deform to better match the atlas. Finally, a Bayesian segmentation procedure is carried out, and the maximum a posteriori estimate of the labelling is computed. The segmentation uses three pieces of information to disambiguate labels: (1) the prior probability of a given tissue class occurring at a specific atlas location; (2) the likelihood of the image given that tissue class; and (3) the probability of the local spatial configuration of labels given the tissue class. The automated segmentations have been found to be statistically indistinguishable from manual labelling ( Fischl et al., 2002 ), and correlations between Freesurfer segmentation and manual labelling of hippocampal volume by trained researchers reached 0.85 in a study by  Tae et al. (2008) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24099844">PMID:24099844</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>SPM8, Freesurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) and FSL 4.1.9 (FMRIB's software library,  www.fmrib.ox.ac.uk/fsl ) were used for image analysis. The Longitudinal VBM analysis performed in this study consists of the following main steps ( Fig.   1 ): 1. Lesion mask creation and lesion filling:</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24098564">PMID:24098564</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Entire cortex vertex-wise analyses of cortical thickness were performed contrasting rs12541 and rs12807809 C allele carriers vs. TT homozygotes. Briefly, spherical registered cortical thickness data from all subjects were mapped to an average subject ( http://surfer.nmr.mgh.harvard.edu/fswiki/FsAverage ). Cortical thickness maps were smoothed using a 10mm full-width-at-half-maximum Gaussian kernel. Finally, univariate general linear models were run for each SNP separately (model 3 and 4) at all vertices (n=163,842) per hemisphere. We included age, gender, scanner field strength and diagnostic group into the models as covariates and also tested for the diagnosis by SNP interaction effect. All cortical thickness results were corrected for multiple comparisons using a Monte-Carlo simulation with 10,000 repeats. Vertex-wise threshold and cluster-wise probability (CWP) were set to 0.05. For details, see SI 1.6 in  File S1 . Final statistical maps are shown on the inflated surface of the standard average subject, allowing visualization of data across the entire cortical surface without interference from cortical folding.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24086243">PMID:24086243</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional MRI data set courtesy Govinda Poudel; cortical surface extraction by FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24073391">PMID:24073391</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Medial temporal ROIs were defined based on segmentation of each participant's high-resolution T1-weighted MRI scan. Automated volumetric segmentation was performed using the Freesurfer image analysis suite (version 4.5.0,  https://surfer.nmr.mgh.harvard.edu ) and yielded individualized volumes corresponding to given anatomic regions of interest, including hippocampus, parahippocampal gyrus, fusiform gyrus, inferior temporal gyrus, middle temporal gyrus, superior temporal gyrus, insula and amygdala bilaterally. This individualized method of ROI definition was chosen to minimize the contribution of non-tissue (e.g. CSF) to the regions of interest of participants with atrophy. Furthermore, it more reliably accounted for individual variability in the anatomic position and extent of these regions. The accuracy of the segmentation was verified visually for each subject, as well as quantitatively for hippocampal regions by quantifying hippocampal volume and by comparing the Freesurfer segmentation to a more accurate hippocampal shape assessment method via a high-dimensional large deformation algorithm ( Hogan et al., 2003 ). Participant-specific anatomic regions defined in this fashion were used to extract BOLD functional data for use in subsequent analyses. Functional MRI BOLD time-series for the purpose of estimating functional connectivity strength of ROI pairs were extracted using ROIs in subject space (i.e. without transformation to atlas space). Images were resliced and transformed into a standard atlas space ( Talairach and Tournoux, 1988 ) only when generating group connectivity maps.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24073270">PMID:24073270</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and volumetric segmentation was performed in an automated manner on the 3D T1 weighted structural images using the Freesurfer image analysis suite (v4.3) ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described elsewhere [ 16 ]. Briefly, for each subject, the boundary between the grey and white matter and the outer pial surface of the cortex were segmented. Cortical thickness measurements at each point across the cortical mantle were calculated as the closest distance from the gray/white boundary to the gray/CSF boundary [ 23 ]. The cortical surface and subcortical regions were automatically labeled [ 24 ]. For this study, the rostral and caudal anterior cingulate cortex (ACC) and amygdala and hippocampus limbic regions were analyzed. The individual cortical regions were also grouped into frontal, occipital, parietal and temporal lobes for lobe-wise analyses.  Figure 1  shows the different brain lobes and the cingulate sub-regions parcelled using freesurfer for a representative dataset.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24073235">PMID:24073235</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The aim of this step is to partition the cortical surface into ROIs that are compact and of similar size. Ideally, each ROI should be placed in the same anatomical location for each subject, thus allowing inter-subject comparison of the connection matrices. The proposed procedure relies on an atlas-based cortical registration method using the curvature information, i.e. sulcus and gyrus  [29] ,  [30] . This method has been directly implemented in the Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ), providing an automatic labeling of the cortex into 66 gyral-based parcels, which are defined using curvature-based information on 40 manually labeled brains  [29] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24068765">PMID:24068765</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The major cortical areas (M1, SMA and PMdv) known to contribute to the descending motor tracks were defined using 16 healthy participants. The volunteers ’  T 1 -weighted image were co-registered into standard space as defined by the Montreal Neurological Institute (MNI) template. The co-registration process was carried out using FSL linear registration tool ( http://www.fmrib.ox.ac.uk/fsl ). The M1 and SMA for both left and right side were defined using the BrainMap database in MNI space. We used Freesurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu/fswiki ) to perform parcellation to determine the location of the premotor areas and the M1. Dorsal promotor area (PMd) was identified as superior part of precentral sulcus and ventral premotor area (PMv) was identified as inferior part of precentral sulcus. In this study, PMd and PMv were combined together as premotor area (PMdv). Streamline track algorithm was used to trace the connection from these motor areas to pontine nuclei. The probabilistic maps of descending motor corticofugal tracks from these participants were transformed to standard space ( figure 1 ). Masks of the corticofugal fibres were created from these maps. Involvement of the corticofugal fibres by stroke was determined by multiplying the corticofugal fibre masks and the infarct. The volumes of overlap between the infarct and the fibre masks were determined by voxel counting method.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24058401">PMID:24058401</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements were conducted using the FreeSurfer Tools ( http://surfer.nmr.mgh.harvard.edu/ ) and the method implemented in this study has been previously described in detail elsewhere  [24] ,  [26] – [28] . In brief, initial surface was obtained for further reconstructing the pial surface using deformable surface algorithm  [29]  after the segmentation of white matter  [30] , tessellation of gray/white matter boundaries  [30] , and automatic correction of topological defects  [31] . All surfaces of each individual image data were visually inspected for accuracy of spatial registration and gray/white matter segmentation and were manually corrected, as recommended by the guideline. Cortical thickness at each vertex was defined as the shortest distances from the pial to the corresponding gray/white surface. This measurement for the thickness of the cerebral cortex has been validated using histological  [32]  and manual  [32] ,  [33]  estimations with sub-millimeter precision. Reliability for this measurement has also been established across field strength, scanner upgrade, and manufacturer  [34] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24040262">PMID:24040262</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>K-space density weighted and Cartesian EPI were further processed and statistically analysed using FSL 5.0 ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ )  [16] ,  [17] . Unfiltered time-series were corrected for motion using mcflirt (part of FSL;  [18] ). The rigid-body (6 degrees of freedom) inter-volume registration matrices obtained for correction were then applied to the volumes of the filtered time-series. Subsequently, brain extraction (using BET, also part of FSL;  [19] ) and high-pass temporal filtering (cutoff at 70 s, slightly above the block design ’ s on-off cycle time) were performed. Time-series statistical analysis was carried out according to the General Linear Model (GLM) within FEAT using FILM prewhitening (both part of FSL) with local autocorrelation correction  [20] . The time-series model was set up by convolving the block design of the finger tapping (see above) with FSL ’ s canonical hemodynamic response function (HRF) using the default gamma function and including a temporal derivative (in FSL ’ s GLM GUI). Because the mean 3D signal differs between density weighted and Cartesian volumes, the joint model for their interleaved recording was then separated for the two types of EPI acquisitions (by extracting the volume-wise entries from the relevant design.mat files and re-entering these as explanatory variables into FEAT without re-convolution). Thereby, grand-mean 4D intensity scaling by a single multiplicative factor was performed separately for the density weighted and Cartesian EPI time-series at the first level while 3D intensity normalization to a preset constant was avoided to ensure that the analyses are valid at the second level. Upon boundary-based within-subject registration of the functional and anatomical scans  [21]  and subsequent non-linear registration of the structural scans to the MNI152 template using FNIRT (part of FSL;  [22] ), group-level analysis across the n ? = ? 5 subjects was carried out by a Fixed-Effects (FE) model at the higher (i.e. second) level using FEAT  [23] . FE error variances are the variances from the first level, and weighting is introduced into a standard weighted FE model by allowing these to be heteroscedastic. Statistical inference from FE modeling is very sensitive to detect activations at the higher level, yet the reported results are with respect to the sample of subjects studied and are not generalized to the wider population from which these are drawn. Z- (i.e. Gaussianised T-) statistic images were thresholded using clusters determined by Z > 2.3 and a family-wise error rate (FWER)-corrected cluster significance threshold of p ? 0.05  [24] , both for exemplary first-level data as well as the second level FE analysis. For the latter, pre-threshold masking was performed in order to limit the number of multiple comparisons. The binary pre-threshold mask included pre- and postcentral gyrus and supplementary motor area (SMA) as derived from the Havard-Oxford Cortical Structural Atlas in MNI152 space (part of FSL; each of the three structures thresholded at 25% probability). Group-level mean FE fMRI results for k-space density weighted and Cartesian EPI were separately projected to and visualized on the MNI152 pial surface using mri_vol2surf and tksurfer, both part of FreeSurfer 5.2.0 ( http://surfer.nmr.mgh.harvard.edu/fswiki ;  [25] ,  [26] ). The differential effect of density weighted vs. Cartesian EPI is rendered in MNI152 volume space. All sectional brain images are displayed in radiological convention, with the left side of the brain shown on the right side in the figures.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24040166">PMID:24040166</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were further processed using Freesurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) to produce segmented maps with anatomical labels of 52 brain regions, yielding the volume (in mm 3 ) of each region along with cortical thickness (in mm 2 ) of 33 regions. The segmented maps were visually rated for accuracy by experienced quality control staff and excluded from the analysis if quality criteria were not met. A full description of the FreeSurfer processing steps can be found elsewhere  [39] ,  [40] . Finally, a subset of 5 of these regional MRI-derived volume and thickness measures were selected for primary analysis based on a-priori knowledge available in the MCI literature and included the following regions: hippocampus, entorhinal cortex, parahippocampal cortex, precuneus, and posterior cingulate cortex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24040118">PMID:24040118</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All structural scans were first anonymized, converted from DICOM to analyzed format and then skull-stripped. Gray matter thickness was estimated by using a well validated method  [29]  implemented in FreeSurfer 4.0  [29]  (available free to public,  http://surfer.nmr.mgh.harvard.edu/fswiki  and  http://ucla.in/xSQPqT ). The obtained CT maps were registered to ICBM brain surface and then vertex-wise correspondences were established between all cortical surface models using a Conformal Metric Optimization method  [30] . An 8 mm heat kernel  [31]  was applied to smooth the realigned thickness maps to increase statistical power. Preprocessing and CT estimation were carried out using the LONI Pipeline  [17]  environment. A separate tissue-classification pipeline workflow ( http://ucla.in/wxqlcJ ) was employed to obtain total gray matter volume (TGMV), which was used as a covariate in the post hoc mapping and quantitative analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24039777">PMID:24039777</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Bias-field correction and tissue classification were performed on the MPRAGE volumes using an in-house segmentation tool based on a Variational Expectation-Maximization algorithm  [21] . The following 6 regions-of-interest (ROI) were extracted in a fully automated fashion: global  white  and  cortical gray matter (WM and cGM) ,  thalamus ,  caudate ,  globus pallidus  and  putamen  ( Figure 1 D ). Subsequently, T1, MTR and T2* maps were aligned to the MPRAGE volume by a rigid-body registration with 6 degrees of freedom and mutual information cost function using ELASTIX  [22] . Rigid and non-rigid registration  [23] ,  [24]  were performed to align an in-house template (obtained by manual correction of a segmentation provided by Freesurfer software  www.surfer.nmr.mgh.harvard.edu  in a single healthy subject) to the MPRAGEs volumes. Average MTR, T1 and T2* values were obtained for all ROIs.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24025315">PMID:24025315</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject the following  ‘ Regions Of Interest ’  (ROIs) were defined from native space (T1-image) using  FreeSurfer  ( http://surfer.nmr.mgh.harvard.edu/ ): pallidum, putamen, caudate, nucleus accumbens and (entire) thalamus. Further subdivision of these structures was beyond the scope of this study. The thalamus was included since it constitutes a major input/output region of the BG and thus is of interest in the pathophysiology of PD. For each ROI we computed its volume, and mean R2* and MT value ( Table   S1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24024963">PMID:24024963</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>PING imaging techniques, data acquisition and analyses are discussed in depth elsewhere and briefly below (Brown  et al. \n 2012 ; Fjell  et al. \n 2012 ; Walhovd  et al. \n 2012 ). Across the 10 sites and 12 scanners, a standardized multiple modality high-resolution structural MRI protocol was implemented, involving 3D T1- and T2-weighted volumes and a set of diffusion-weighted scans. At the University of California at San Diego, data were obtained on a GE 3T SignaHD ×  scanner and a 3T Discovery 750 ×  scanner (GE Healthcare, Waukesha, WI, USA) using eight-channel phased array head coils. The protocol included a conventional three-plane localizer, a sagittal 3D inversion recovery spoiled gradient echo T1-weighted volume optimized for maximum gray/white matter contrast (echo time = 3.5 milliseconds, repetition time = 8.1 milliseconds, inversion time = 640 milliseconds, flip angle = 8 ° , receiver bandwidth =  ±  31.25 kHz, FOV = 24 cm, frequency = 256, phase = 192, slice thickness = 1.2 mm), and two axial 2D diffusion tensor imaging (DTI) pepolar scans (30-directions  b  value = 1000, TE = 83 milliseconds, TR = 13 600 milliseconds, frequency = 96, phase = 96, slice thickness = 2.5 mm). Acquisition protocols with pulse sequence parameters identical or near identical to those protocols used at the University of California at San Diego were installed on scanners at the other nine sites. Data were acquired on all scanners to estimate relaxation rates and measure and correct for scanner-specific gradient coil nonlinear warping. Image files in DICOM format were processed with an automated processing stream written in MATLAB (Natick, MA, USA) and C++ by the UCSD Multimodal Imaging Laboratory. T1-weighted structural images were corrected for distortions caused by gradient nonlinearities, coregistered, averaged and rigidly resampled into alignment with an atlas brain. Image postprocessing and analysis were performed using a fully automated set of tools available in the  Free Surfer  software suite ( http://surfer.nmr.mgh.harvard.edu/ ) as well as an atlas-based method for delineating and labeling WM fiber tracts (Fischl  2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24014446">PMID:24014446</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated cortical segmentation of the anatomical images and reconstruction of the white/gray and gray/pial cortical surfaces were performed using Freesurfer [ http://surfer.nmr.mgh.harvard.edu/ ; Dale et al.  1999 ]. The mrFlatMesh algorithm (VISTA software,  http://white.stanford.edu/software/ ) was used to create flattened representations of the cortical regions surrounding the central sulcus and post-central gyrus of the right hemisphere.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24009767">PMID:24009767</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All structural image analysis was conducted using the Connectome Computation System (CCS:  http://lfcd.psych.ac.cn/ccs.html ) pipeline  [40] . Specifically, each participant's MR images were first denoised through a spatially adaptive non-local means filter  [41] ,  [42] . The two denoised MRI scans for each participant were averaged to create a single high signal-to-noise average volume  [31] . To determine the amygdala volume, we performed quantitative morphometry analysis on the averaged T1-weighted MRI data using an automated segmentation and probabilistic region of interest (ROI) labeling technique  [43] . These images were first corrected for intensity variations due to MR inhomogeneities  [44] . As described in the Freesurfer Wiki document ( http://surfer.nmr.mgh.harvard.edu/fswiki ), a hybrid watershed/surface deformation procedure  [45]  was first employed to extract brain tissues that were then automatically segmented into the cerebrospinal fluid (CSF), white matter (WM) and deep gray matter (GM) volumetric structures  [44] . To explore the relationship between fearful face recognition performance and other brain areas, we further conducted individual cortical surface reconstructions to measure the volumes of these regions. Two researchers (Y.M and L.Y.), blind to the hypotheses, manually inspected the results of the automated brain tissue segmentation. The criteria used for quality assurance on brain extraction, surface reconstruction and anatomical image registration are as follows: 1) the quality of the brain extraction and intensity bias correction must be visually assessed and manually corrected if the procedure failed, and 2) the brain tissue segmentation was also visually checked to ensure good quality. A detailed description of the criteria can be found on the following website:  http://lfcd.psych.ac.cn/ccs/QC.html , and in our previous work  [40] . After the visual inspection, there were 6 participants whose brainmask datasets were manually edited to achieve better estimates of pial surfaces. The results of the automated segmentation were verified as accurate without the need for any correction.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23982596">PMID:23982596</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess the thickness of cortical gray matter, we used the FreeSurfer automated tool ( http://surfer.nmr.mgh.harvard.edu ) and voxel-based morphometry (VBM) conducted using SPM8 calculated deviations of the brain volume of a patient and from 11 age- and gender-matched controls. The total acquisition time for DTI was 10 min. DTI fiber tract reconstruction was performed using the DTI Studio software package. Deterministic tractography was performed using the fuzzy art with add clustering technique (FACT) algorithm ( 49 ). All tracts were visualized and subsequently visually inspected for directionality and location. The regions of interest (balls of 3 mm diameter) were designated in the ascending fibers of the pons to visualize the corticospinal tract (CST). The purpose of this analysis was to probe alterations in diffusion-based tractography, and consequently, to demonstrate changes in structural plasticity in addition to the functional changes we observed in the brains of the chronic stroke patients as a result of hand training. We reconstructed the CST tract, selecting as seeding areas the pons, the posterior limb of the internal capsule and the motor cortex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23965184">PMID:23965184</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated cortical surface reconstruction was performed on the T1 MPRAGE scans in Free Surfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu ) using previously described methods [ 31 - 33 ]. Any inaccuracies in the reconstruction of white and pial surfaces of individual subjects were manually corrected before calculating cortical thickness following instructions provided by the software developers. The cortical thickness measure was computed as the distance between the pial and white matter surfaces at each point across the cortical mantle.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23964245">PMID:23964245</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23956957">PMID:23956957</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A high-resolution structural scan was acquired to position the voxel during data acquisition and to determine the tissue composition of the voxel through segmentation. The  T 1 -weighted scan was processed using the standard FreeSurfer pipeline ( http://surfer.nmr.mgh.harvard.edu ) for tissue segmentation and anatomical parcellation ( Fischl et al., 2002 ). Further details related to the FreeSurfer processing can be found online, and in one of our previous publications ( Porter et al., 2011 ). In-house software was used to compute the transformation matrix from the scanner coordinates to the FreeSurfer-processed  T 1 -weighted image. A mask representing the spectroscopy voxel in the anatomical image space was then created using tools from the FMRIB Software Library ( Smith et al., 2004 ), which was subsequently segmented and parcellated using the FreeSurfer anatomical information. Thus, each  T 1 -weighted voxel (1   mm isotropic resolution) within the spectroscopy volume (2   cm   ×   2   cm   ×   2   cm), was classified as either white matter, gray matter, cerebrospinal fluid (CSF), or non-brain, and was further parcellated into subcortical and cortical structures. This was done to confirm a consistent voxel placement across all subjects (i.e., the majority of the spectroscopy voxel contents were within the basal ganglia in all subjects) and to determine the basic tissue composition within the voxel (i.e., gray matter, white matter, or CSF). Further details of the voxel composition can be found in the results section below.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23919088">PMID:23919088</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The brain surface used to produce simulations is from a healthy male subject, age 23 at time of collection. Structural MRI scans were acquired using an 8-channel phase-array head coil in a 3T scanner (Siemens-Trio, Erlagen, Germany). Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ) (Fischl et al.,  2002 ,  2004 ) was used for cortical surface reconstruction and parcellation. ROIs (Figure  1 ) were chosen based on clusters of significant activation during a visual search task with a facilitatory auditory cue adapted from (Vaina et al.,  2010 ; Calabro et al.,  2011 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23888133">PMID:23888133</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Our source estimation procedure followed the standard procedure described for the MNE software ( http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/sofMNE.php ). Minimum norm estimates (H ä m ä l ä inen and Ilmoniemi,  1994 ; Hauk,  2004 ) were computed on individually reconstructed cortical surfaces using boundary element models of the head geometry derived from structural MRI images. MEG sensor configurations and MRI images were co-registered based on the matching of about 50 – 100 digitized additional points on the scalp surface with the reconstructed scalp surface from the FreeSurfer software (Version 4.3;  http://surfer.nmr.mgh.harvard.edu/ ). Structural MRI images were processed using the automated segmentation algorithms of FreeSurfer (Dale et al.,  1999 ; Fischl et al.,  1999 ). The noise covariance matrices for each data set were computed for baseline intervals of 200 ms duration before the onset of each stimulus presented during the whole experiment. For regularization, the default signal-to-noise ratio in the MNE software was used ( SNR  = 3).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23880882">PMID:23880882</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI data of this sample were part of the ENIGMA sample. 1  The volumetric segmentation of the left and right hippocampus was performed with the Freesurfer image analysis suite (version 5.0;  http://surfer.nmr.mgh.harvard.edu/ ). The three mprage-images were fed into the volumetric stream which included motion correction, averaging, removal of non-brain tissue, automated Talairach transformation, intensity nomalization and segmentation of the subcortical white matter and deep gray matter volumetric structures (including the hippocampus) with the help of a probabilistic atlas. 7 ,  8  Quality control included image inspection, outlier detection via boxplot and subsequent manual editing of the brainmask, details are given in Stein  et al.  2012. 2</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23878736">PMID:23878736</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Atrophy in multiple sclerosis as measured using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). (a) Cortical and subcortical segmentation in a healthy control patient. (b) Cortical and subcortical segmentation in an age-matched patient with longstanding RRMS. Note the prominent subcortical, white matter, and neocortical atrophy in MS compared to healthy control.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874936">PMID:23874936</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The quality of the anatomical information present in high resolution R 1  maps was evaluated by observing the contrast and anatomical detail observed in structures such as the hippocampus. The individual R 1  maps were processed with freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) to create cortical surface models  [31] . Five equally spaced surfaces were generated in between the white matter and pial surfaces in order to study cyto-architectonic cortical variations in the mid layer throughout the brain. A 2 mm smoothing along the surface was applied and the R 1  surface maps were then plotted using MATLAB (The MathWorks Inc.).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874696">PMID:23874696</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The anisotropic images were resampled into volumes with isotropic voxel dimensions of 1.02 × 1.02 × 1.02 mm 3  to facilitate the computation of cortical complexity. We located the anterior commissure and posterior commissure using the Automatic Registration Toolbox  [31]  to determine the midsagittal plane, which was then aligned parallel to the YZ volume plane. Skull-stripping and intensity non-uniformity correction were performed using the hybrid watershed algorithm  [32]  and non-parametric, non-uniform intensity normalization  [33] , respectively, in Freesurfer (available online at  http://surfer.nmr.mgh.harvard.edu/ ). We classified the brain tissue into gray matter (GM), white matter (WM), and cerebrospinal fluid (CSF) using SPM8 (Statistical Parametric Mapping, version 8; available online at  http://www .fil.ion.ucl.ac.uk/spm/). The outer surface of cortex was delineated as the GM/CSF boundary. The extracted outer surface was validated by two experienced neuroradiologists (PS Wang and TP Su) and was manually adjusted if necessary. In the subsequent analysis, we only computed the shape index and curvedness on the outer surface to observe the changes of cortical surface morphology since we hypothesized that the impairment of target of serotonin (serotonin receptors in the GM) caused the therapeutic differences between remitted and non-remitted MDD patients. To exclude the pathological changes in WM irrelevant to our serotonin hypothesis, such as the degeneration or demyelination of axons that may affect the morphology of inner surface (GM/WM surface), the morphological changes of inner surface were not considered. Ninety gross cortical structures on the cerebral gray matter were extracted and labeled based on Anatomical Automatic Labeling (AAL)  [34]  by the Individual Brain Atlases using Statistical Parametric Mapping (IBASPM) software  [35] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874692">PMID:23874692</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements were performed using the software FreeSurfer version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ). The cortical surface was reconstructed using a semiautomated approach  [40] . All registered MRIs were segmented to identify GM/WM boundaries. Thickness measurements were obtained by reconstructing representations of the GM/WM boundary and the white boundary to the GM/cerebrospinal fluid boundary and then calculating the closest distance from those surfaces at each vertex on the tessellated surfaces. All subjects ’  data were resampled to the freeSurfer default common surface template using a high-resolution surface-based averaging technique that aligned cortical folding patterns. Finally, the surface data were spatial smoothed using a Gaussian kernel of 15 mm full-width at half-maximum. All images were carefully visually inspected to ensure accurate identification of the gray/white matter boundary and the pial surface. Regional cortical thickness variations were compared using general linear model at each vertex across the cortical surface, with cortical thickness as dependent variable and age, education, and total intracranial volume as covariates. The statistical parametric map was generated at p < 0.05 and p < 0.001 (uncorrected for multiple comparisons), respectively.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874403">PMID:23874403</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Analysis of cortical parameters was performed on 3D images. We used two different approaches to quantify cortical pathology. The first approach was to assess cortical thickness using Freesurfer software, version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ). This included skull stripping, automated Talairach transformation, segmentation of the subcortical white/gray matter structures, intensity normalization, automated topology correction, and registration to a spherical atlas. This was followed by parcellation of the cerebral cortex into units based on gyral and sulcal structure [ 33 , 34 ]. The pipeline generated cortical thickness (mean, in mm, of each area) and cortical volumes (mm 3  of each volume). All of the images were visually inspected after each step of processing. The anatomical parcellation according to Desikan Killiany map [ 33 ] divides the ACC into a rostral anterior division and a caudal anterior division ( Figure 2A) , and for the present study a mean between these two divisions was used, with the purpose of including the entire ACC. Results after parcellation were accepted if they included the ACC, and in the cases of a present paracingulate gyrus, the ACC only or ACC and the paracingulate. To partly avoid the problem that the parcellation scheme does not always discriminate between the cingulate and paracingulate cortex, we used only cortical thickness for further analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874398">PMID:23874398</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness and volumes of left and right precentral cortices (cortical regions holding central motoneurons) were obtained for each participant using Freesurfer image analysis suite (version 5.1,  http://surfer.nmr.mgh.harvard.edu/ );  [33] ,  [34] . Briefly, the processing includes removal of non-brain tissue  [35] , transformation to Talairach-like space, and segmentation of gray/white matter tissue  [36] ,  [37] . The entire cortex of each subject was then visually inspected; no subject was discarded for poor segmentation reason. After creation of the cortical representations, the cerebral cortex was parcellated into anatomical structures  [38] ,  [39] .The anatomical labels were mapped to all individual brains for the quantification of average cortical thickness and average volume using automated FreeSurfer tools. In particular, cortical thickness was computed by finding the shortest distance between a given point on the estimated pial surface and the gray/white matter boundary and vice versa and averaging these two values  [40] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23866851">PMID:23866851</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data are processed using a custom-built automated pipeline run via scripts in the MATLAB 7.5 software (MathWorks, Natick, MA, USA). Processing tasks are performed by a cluster of 18 custom-built computer nodes running Linux (CENTOS 5.0 distribution); tasks are assigned, queued and coordinated using a central computer. Specific details of the analysis procedures are provided below. In summary, the raw DICOM data from the magnetic resonance scanner are imported using a script and converted to NIfTI format, then is run through sequential routines drawn from the Functional MRI of the Brain (FMRIB) Software Library (FSL) release 4.1.3 ( http://www.fmrib.ox.ac.uk/fsl ) [ 26 ], the Freesurfer image analysis suite (version 4.3) (documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ) [ 27 ], SPM8 software [ 28 ] and custom-written routines that have been described in previous publications [ 29 - 32 ]. Anonymized data are transferred between the two imaging sites via a secure ftp connection.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23861951">PMID:23861951</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We defined four types of import files for BrainNet Viewer, namely, brain surface, node, edge and volume files. 1) Brain surface file. The brain surface file is an ASCII text file, with the suffix  ‘ nv ’ , containing four fields: the number of vertices, the coordinates of each vertex, the number of triangle faces and the index of the vertices comprising the triangles. Currently, the  ‘ .pial ’  file of hemisphere mesh, generated using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ )  [63] , and the  ‘ .mesh ’  files, generated using BrainVISA ( http://brainvisa.info/ )  [64] , are supported for direct loading and visualization. 2) Node file. The node file is an ASCII text file with the suffix  ‘ node ’ . Nodal information is arranged in 6 columns in the node file: columns 1 – 3 represent the x, y and z coordinates, respectively, of the nodes; column 4 represents the index for node color; column 5 represents the node size; and column 6 represents the node label. A  ‘ ? ’  symbol (no  ‘ ’ ) in column 6 indicates no label for the corresponding node. The values for this file are easily arranged depending on the aspects of the network shown. For example, the modular information for the nodes can be assigned to column 4 to use color to distinguish nodes belonging to different modules. Column 5 could be set as nodal degree, centrality and T-value to emphasize nodal differences according to size. 3) Edge file. The brain edge file is an ASCII text file with the suffix  ‘ edge ’ , representing an association (e.g., correlations) matrix among the nodes, which can be weighted or binarized, and therefore, the size of the matrix must correspond to the number of nodes. Both the node and edge files can be generated or edited using text editors or spreadsheet software. 4) Volume file. BrainNet Viewer facilitates the mapping of volume data to the brain surface, which can be a functional connectivity map, gray matter density map, statistical parametric map or a brain atlas. The volume file should be in the NIfTI or Analyze format, and either a single or paired nii files are acceptable. A text file containing an n × 1 vector is also acceptable, in which n equals the vertex number of the brain surface (e.g., 81,924 vertices in the International Consortium for Brain Mapping (ICBM) whole brain surface).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23860484">PMID:23860484</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To contrast against the results from the manual tracing protocol, we also computed amygdala volumes with the fully automated subcortical segmentation component of Freesurfer's ( http://surfer.nmr.mgh.harvard.edu/ ) cortical reconstruction process, which has been shown to be more highly correlated with hand tracing than other automated protocols. 48  This also provided a measure of intracranial volume, which was used as a covariate in all analyses, because of the influence of intracranial volume on amygdala volumetric measurements. 55</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23840493">PMID:23840493</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural analysis based on segmentation of brain structures from T1 high-resolution anatomical data was performed using the freely available Freesurfer toolkit version 5.0 ( http://surfer.nmr.mgh.harvard.edu ). Intracranial volume (ICV) was used to correct the volumes and the processing pipeline was the same as previously described  [7] . DMN was defined by the summed volume of the angular gyrus of inferior parietal lobe, the posterior cingulate, the precuneus and the frontopolar region  [14] ,  [15] . The summed volume of the middle frontal gyrus (dorsolateral and prefrontal region) and the posterior parietal region constituted the DAN  [46] ,  [47] . VAN was constituted by the sum of the temporal-parietal junction and the ventral frontal cortex volumes  [25] . SMN was defined by the summed volume of the paracentral, precentral postcentral and the cerebellum  [45] . The summed volume of the cuneus, pericalcarine and the lingual region constituted the primary VN  [45] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23840184">PMID:23840184</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23825453">PMID:23825453</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical data from each patient were analyzed individually to measure cortical thickness using Freesurfer v4.5 (MGH, USA,  http://surfer.nmr.mgh.harvard.edu/ ). The followed workflow includes motion correction, skull stripping, automated Talairach transformation, white and gray matter segmentation, and intensity normalization. Further technical details of the automated procedures have been described (Dale et al.,  1999 ; Fischl et al.,  1999a , b ,  2001 ,  2002 ,  2004a , b ; Fischl and Dale,  2000 ; Segonne et al.,  2004 ; Jovicich et al.,  2006 ; Han et al.,  2006 ). Thickness maps were rendered onto an inflated brain template to reveal measurements of cortex in the sulci. A color spectrum was applied with yellow (5 mm) and red (greater than 2 mm) indicating areas of thicker cortex and gray indicating thin cortex (less than 2 mm).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23823184">PMID:23823184</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To examine cortical GM, cortical thickness analysis (CTA) was performed using FreeSurfer software v. 4.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ); methods have been described in detail elsewhere  [20] . Briefly, T1-weighted scans were registered to the Talairach atlas  [21] . Images underwent intensity normalization to identify tissue types, and the skulls were removed. White matter (WM) voxels were used to estimate the GM/WM boundary, which was deformed outward to the GM/cerebrospinal fluid (CSF) boundary. Thickness was computed as the distance measured between these boundaries at every point in each hemisphere. A general linear model (GLM) was used to evaluate group differences, with age included as a variable of no interest. We restricted analyses to a mask including our hypothesized regions of interest (ROI) ( Fig. 1A – D ) created with a predefined cortical parcellation scheme (aparc2005) implemented in FreeSurfer  [22] . Data were smoothed with a 6 mm full-width half-maximum (FWHM) spatial smoothing kernel, and thresholded at a corrected  p < 0.05 (from a combination of an uncorrected image-wide  p -value of 0.01, and a cluster threshold of 183 and 188 contiguous vertices for left and right hemispheres, respectively, based on Monte Carlo simulations with 5000 iterations (AlphaSim, implemented in Analysis of Functional NeuroImages (AFNI) ( http://afni.nimh.nih.gov/afni ), as done previously  [10] ,  [12] ,  [23] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23809257">PMID:23809257</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>White matter fibre orientation distributions will be estimated using constrained spherical deconvolution [ 52 ] using MRtrix software [ 53 ], which allows the resolution of crossing fibres which is critically important for fibre-tracking [ 54 ]. Whole brain probabilistic tractography will be carried out using MRtrix. White matter pathways will be extracted using cortical and subcortical target regions identified in an automated fashion from high-resolution structural images using freesurfer (http://surfer.nmr.mgh.harvard.edu/). Alterations in white matter microstructure within these pathways will be assessed using traditional diffusion tensor imaging (DTI) metrics - including fractional anisotropy (FA) and mean diffusivity (MD) - as well as the novel apparent fibre density (AFD) measure ? [ 55 ] which takes into account the presence of crossing fibres. Tractography will be used to delineate individual pathways as described previously [ 56 - 58 ], and for connectivity-based parcellation [ 59 , 60 ]. Changes in white matter microstructure within delineated pathways following therapy will be assessed.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23805179">PMID:23805179</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MCIC structural MRI data from three consecutive volumes were registered, motion corrected, averaged and analyzed in an automated manner with atlas-based FreeSurfer software suite ( http://surfer.nmr.mgh.harvard.edu , Version 4.0.1). This process included volumetric segmentation, cortical surface reconstruction  [34] – [37]  and the estimation of total intracranial volume (ICV)  [38] . Hippocampal volume is a standard output of the FreeSurfer volumetric segmentation  [35] . Previous imaging genetics studies have shown the same genetic effects for the left and right hippocampus  [39] ,  [40] . Therefore we used mean hippocampal volume (averaged across the right and left hemisphere) as the primary parameter for analysis. Segmentation and surface reconstruction quality were assured by manual inspection of all raw MRI volumes, segmented volumes in three planes and pial as well as inflated volumes. Five participants' MRI data failed the aforementioned quality assurance. The data of these subjects were then recovered with minor manual intervention following the FreeSurfer user guidelines.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23805098">PMID:23805098</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23805096">PMID:23805096</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). Briefly, the processing includes motion correction and averaging (Reuter et al.,  2010 ) of the two volumetric T1-weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure (Segonne et al.,  2004 ), automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (Fischl et al.,  2002 ,  2004 ), intensity normalization (Sled et al.,  1998 ), tessellation of the gray matter-white matter boundary, automated topology correction (Fischl et al.,  2001 ; Segonne et al.,  2007 ), and surface deformation. The last procedure is accomplished by following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl and Dale,  2000 ). Once the cortical models are complete, a number of deformable procedures can be performed for further data processing and analyses including surface inflation (Fischl et al.,  1999a ), registration to a spherical atlas which utilizes individual cortical folding patterns to match cortical geometry across subjects (Fischl et al.,  1999b ), parcellation of the cerebral cortex into units based on gyral and sulcal structure (Fischl et al.,  2004 ; Desikan et al.,  2006 ), and creation of a variety of surface based data including maps of curvature and sulcal depth. This method uses both intensity and continuity information from the entire three-dimensional MR volume in segmentation and deformation procedures to generate representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/cerebrospinal fluid boundary at each vertex on the tessellated surface. Freesurfer morphometric procedures have been demonstrated to show good test-retest reliability across scanner manufacturers and across field strengths (Han et al.,  2006 ). Volume measures may be mapped on the inflated surface of each participant ’ s reconstructed brain. Maps are smoothed using a circularly symmetric Gaussian kernel across the surface with a standard deviation of 10 ? mm and averaged across participants using a non-rigid high-dimensional spherical averaging method to align cortical folding patterns. This procedure provides accurate matching of morphologically homologous cortical locations among participants, resulting in a mean measure of cortical thickness for each group at each point on the reconstructed surface. The entire cortex in each participant was visually inspected and any inaccuracies in segmentation were manually corrected by experts in brain anatomy who were blind to group membership.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23762308">PMID:23762308</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>PiB-PET data was preprocessed using the Statistical Parametric Mapping software package (SPM8;  http://www.fil.ion.ucl.ac.uk/spm ). Region of Interest (ROI) labeling, including the cerebellum gray matter masked, was implemented using the FreeSurfer v5.1 software package ( http://surfer.nmr.mgh.harvard.edu/ ). Distribution volume ratio (DVR) images were created using Logan graphical analysis with PiB frames corresponding to 35 – 90 minutes post-injection and a gray matter cerebellar mask as the reference region  [38] ,  [39] . The grey matter cerebellum was chosen as the reference region because it is known to be relatively unaffected by fibrillar amyloid in AD and is thus valuable to estimate non-specific PiB binding  [39] ,  [40] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23762193">PMID:23762193</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For retinotopic mapping analysis, FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) was used to reconstruct a T1-weighted anatomical image. Then, the realigned retinotopic mapping images were registered to the anatomical image to obtain the registration file. The following retinotopic analysis was consistent with [ 26 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23761780">PMID:23761780</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Slice time correction, isotropic reslicing of voxels, and initial motion correction was performed with SPM99. We then used custom-built motion-correction software that is particularly accurate for the larger head motion seen in older and PD subjects (Liao et al.,  2005 ,  2006 ). Low frequency drifts were removed with a discrete cosine transformation, with cutoff period of 128 ? s. We did not spatially normalize each subject ’ s data to a common space, as we have demonstrated that this will incur excessive error (Nieto-Castanon et al.,  2003 ; Chen et al.,  2009 ; Ng et al.,  2009 ). Fifty two regions of interest (ROI) were extracted using a combined FreeSurfer (Harvard, MA, USA;  http://surfer.nmr.mgh.harvard.edu/ ) and Large Deformation Diffeomorphic Metric Mapping (LDDMM) method (Khan et al.,  2008 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23760360">PMID:23760360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI scans were performed on a 1.5 Tesla General Electric MRI scanner (GE, Waukesha, WI). High-resolution T 1 -weighted anatomical data was obtained for all subjects using a 3D magnetization-prepared rapid acquisition gradient-echo (MPRAGE) sequence with the following parameters: TE = 2.8 ms, TR = 6.3 ms, preparation time = 1000 ms, flip angle 8 ° , field-of-view 24 cm  ×  24 cm, 160 sagittal slices, 1 mm slice thickness, no gap, 224  ×  192 image matrix reconstructed to 256  ×  256, scan time = 10 min and 56 s. Two copies of the T 1 -weighted data were acquired on each subject. The two T 1 -weighted datasets from each subject were co-registered using rigid-body registration and averaged. The average T 1 -weighted dataset of each subject was then segmented using FreeSurfer Version 5 ( http://surfer.nmr.mgh.harvard.edu ; Fischl et al.,  2002 ; Xiao and Fischl,  2007 ). The results were reviewed and any errors were manually corrected. The volume of the segmented gray matter regions, including the hippocampus, and the total intracranial volume (ICV) were estimated for each subject. The volumes of homologous regions in contralateral hemispheres were averaged. Finally, hippocampal and all other volumes were normalized by dividing by the total ICV.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23755254">PMID:23755254</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were acquired on a 1.5-T MRI scanner (Siemens Magnetom Sonata, Erlangen, Germany) with a standard head coil for whole brain MRI data. Two sagittally oriented T1-weighted volumes (TE: 3.56 ms; TR: 12.24 ms; flip angle: 23 ° ; matrix: 256 × 256; voxel size: 1 × 1 × 1 mm) were acquired and used for further processing by the freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The freesurfer tool allows quantitative assessment of structural brain data without rater bias.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23734202">PMID:23734202</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Activations seen in the group analysis were labeled with the Automated Anatomical Labeling (AAL) tool  [26] . Mricron ( http://www.cabiatl.com/mricro/ ) and FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) were used for visualization.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23717572">PMID:23717572</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness automatic parcellation:  The cortical thickness parcellation was carried out using Freesurfer toolkit ( http://surfer.nmr.mgh.harvard.edu/ )  [28]  with a surface – based approach performing the reconstruction of the brain surface from structural data  [29] . The detailed procedure for the surface construction has been described and validated in previous publications  [29] – [31] . Briefly the processing involved intensity normalisation, skull stripping, segmentation of white matter, tessellation of the grey/white matter boundary and automatic topology correction. This surface was then used as the starting point for a deformable surface algorithm to find the grey/white and grey/cerebrospinal fluid (CSF) surfaces. This method used both intensity and continuity information from the surfaces in the deformation procedures to produce representations of cortical thickness, calculated as the average of the closest distance from the grey/white boundary to the grey/CSF boundary and the grey/CSF boundary to the grey/white boundary at each vertex on the tessellated surface After registration to a common surface template resulting from the surface-based averaging technique. The cortical thickness values were extracted from 74 labelled regions per hemisphere  [32] . The automated anatomical labelling volume of interests (VOIs) corresponding to the Desikan atlas regions as defined in Destrieux et al.  [33]  were created in the MNI standard space using the WFU-pickatlas 3.04 SPM8 tool. The 50 maps of cortical thickness were then created by an automated process obtained with a FSL 4.9 pipeline multiplying every Automated Anatomical Labelling (AAL) VOI by the corresponding cortical thickness value [modified method from Li G. et al.  [34] ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23684858">PMID:23684858</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To define the ventral striatum (nucleus accumbens) in our older cohort of subjects, we made a subject-derived mask for this region. We used Freesurfer's (version 4.5.0,  http://surfer.nmr.mgh.harvard.edu/ ) automated recon-all pipeline to parcellate cortical and subcortical regions ( Fischl et al., 2004 ). Each subjects' ventral striatum mask was visually inspected to ensure accurate segmentation. Subjects were excluded due to preprocessing errors (n   =   4) or inaccurate segmentation after visual inspection (n   =   3). For the remaining 23 participants, their ventral striatum masks were warped to MNI space using DARTEL in SPM8 ( Ashburner, 2007 ) and then group-averaged and binarised. This average mask was then normalised to each individuals' native space (n   =   30) using the inverse of the normalisation parameters. The aim of this approach was to obtain a target ventral striatum mask that was accurate and representative of our older cohort but the same size for each individual, hence we averaged the mask across the group. Thus we report DTI data for all 30 participants.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23675456">PMID:23675456</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing and cortical reconstruction were performed with the Freesurfer image analysis suite (version 5.1.0), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). Recent studies have evaluated the performance of Freesurfer with images acquired from different sequences and scanners, and have concluded that reliable volumetric and surface measurements can be achieved although a minor bias may exist  [26] ,  [27] . As demonstrated in  Fig. 1 , all images were resampled into isotropic voxel of 1 × 1 × 1 mm 3 , intensity inhomogeneity corrected, and skull stripped. Cerebral white matter was segmented using a hybrid watershed classifier and the gray – white matter interface was identified. A deformable surface algorithm was then employed to search for the pial surface (the boundary between gray matter and CSF) using the gray – white matter interface as starting point. The whole cortex of each individual subject was visually inspected for segmentation inaccuracies and manually corrected if necessary. Local cortical thickness was measured as the distance between the inner and the outer cortical surfaces at each vertex  [28] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23648965">PMID:23648965</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data from the retinotopic mapping scans were used to define the borders between areas V1, V2, V3/Vp, and V3a/V4v in each hemisphere ( Engel et al., 1994; Sereno et al., 1995 ). Data from the localizer scans were then used to define the segments of each of these areas that responded most strongly to stimuli in the location of the targets or distractors. To define regions of interest (ROIs) in the early visual cortex, the functional data were projected onto inflated representations of each participant's brain. Cortical reconstruction and volumetric segmentation were performed with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). These ROIs were imported into SPM8, and the GLM parameter estimates corresponding to signals elicited by the stimuli appearing in the contralateral visual field for each of the previously described events were extracted from each ROI, allowing for analysis of the response to the different distractor types under the different load conditions. The ROI data were extracted and analyzed separately for each participant, then averaged.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23616742">PMID:23616742</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23593231">PMID:23593231</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For the segmentation of the ventricular system, high-resolution T1-weighted images were processed using a rater-independent, automated atlas-based tissue segmentation method implemented in the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu )  [19] . In brief, after the processing of motion correction, intensity inhomogeneity correction, and the removal of non-brain tissue, each ventricular component was segmented by assigning each voxel of the preprocessed volume to the corresponding ventricle labels. Information about voxel intensity relative to probabilistic training atlas was utilized for these assignments and subsequent comparisons to neighboring voxel label obtained from atlas were performed  [19] . Final segmented images were visually inspected by an expert who judged the appropriateness of images in further shape analysis. Intracranial volume (ICV) and volumes of each ventricular component were measured. As exploratory analyses, cortical regions were segmented based on the Desikan-Killiany atlas  [20]  and volumes of each cortical region were measured using the FreesSurfer software.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23578686">PMID:23578686</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To improve our understanding of cortical plasticity post-training, cortical reorganisation will be investigated using a combined fMRI-structural connectivity analysis strategy. In this approach, regions of corticomomtor activation derived from the fMRI analysis (generated post-therapy) will be used as target masks for extracting white matter motor pathways. This will enable the identification of all corticomotor networks exhibiting plasticity as a result of the motor training paradigms. Plasticity within these neural circuits will be measured by comparing apparent fibre density, 43  a quantitative measure of the organisation of WM fibres, derived over the entire pathway. This strategy enables both an anatomical view of cortical reorganisation and quantitative measures of altered connectivity induced by therapy. We also propose to measure plasticity based on an analysis of structural connectivity. In this approach, connectivity matrices will be generated based on parcellation of cortical and subcortical using Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ) and the whole brain tractograms, as outlined above. Hit-testing every streamline with every cortical parcellation will generate the connectivity matrices. Diffusivity indices (Fractional Anisotropy (FA) and Mean Diffusivity (MD)), quantitative markers of the integrity of white matter, will be encoded within the connectome to enable assessment of motor task generated reorganisation. 44 \n 45  Network-based statistics 46  will be performed between the FA and MD connectomes for the control (CP without intervention) and intervention groups to identify statistically significant cortical networks that are associated with neural reorganisation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23555895">PMID:23555895</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to use PASL-MRI and PET methodology to assess if regional changes in rCBF occur in a-priori regions previously reported to have increased synaptic activity during hypoglycemia, each subject ’ s MPRAGE was segmented using FreeSurfer ’ s segmentation and cortical reconstruction analysis software package (Freesurfer Version 5.1 developed at the Martinos Center, Harvard University, Boston, MA;  http://surfer.nmr.mgh.harvard.edu )  [30] ,  [31] ,  [32] . Our regional analysis primarily focused on a network of brain areas [including the thalamus (thalamus proper), medial prefrontal cortex (rostral anterior cingulate), right orbital prefrontal cortex and globus pallidum] ( Figure 2 ) that have been shown to have increased blood flow during hypoglycemia with PET  [4] . Mean rCBF of each of these regions was obtained for both glycemic conditions.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23551922">PMID:23551922</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A  T 1 -weighed image (3D FFE using parallel imaging; TR/TE 10 ms/4.6 ms; FOV 240  ×  240 mm, 200 slices, 0.75 mm isotropic voxelsize) was acquired. Gray/white matter tissue classification and automatic cortical segmentation was performed using the Freesurfer suite (V5,  http://surfer.nmr.mgh.harvard.edu/ ) ( Fischl et al. 2004 ), parcellating the cortex into 68 distinct brain regions (34 regions per hemisphere). For each of the 68 brain regions, the average level of cortical thickness (CT) — an estimate of the thickness of the cortical sheet — was computed, together with their level of cortical volume (CV).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23533586">PMID:23533586</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The reconstructions of the cortical surfaces are obtained from the T1-weighted image using the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ). Semi-automated processing allows the reconstruction of accurate three-dimensional mesh models  [27] ,  [28]  and subcortical regions  [29] . The cortical surfaces are subdivided into 66 gyral cortical regions using a validated atlas-based segmentation  [30] . FreeSurfer surface reconstruction algorithms have been previously validated against manual delineation on MR images  [31]  and postmortem brains  [32] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23508528">PMID:23508528</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23505502">PMID:23505502</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A three-dimensional realistically shaped volume source space with 7 mm spacing was generated using our MNE toolbox ( http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/sofMNE.php ). Inner skull surface was extracted from a T1-weighted MRI of each subject segmented with FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ), and used as the boundary surface for the source space. The source localization was done on the whole brain volume and the 7 mm spacing denotes the distance between grid points of a regular 3D grid defined in the brain volume. The time window for source analysis was from 0 ms to 300 ms, which covered the main response components. Source analysis was done directly on the average response.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23486659">PMID:23486659</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23463013">PMID:23463013</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To compute parameter estimates, we applied a voxel-based general linear model (GLM), as implemented in FEAT. For the task-based scans, the block design paradigm was convolved with a gamma function 19 , and its temporal derivative was used to model the activation time course. Three main contrasts were defined between different task movement types: (i) intact (or dominant) hand versus feet; (ii) affected (or non-dominant) hand versus feet; and (iii) lips versus feet. Individual maps for contrast (ii) are presented in  Fig. 1 . For presentation purposes, the activation maps were smoothed using a FWHM of 2.5 ? mm, and thresholds were adjusted using a false discovery rate of  P < 0.05 (with the exception of amputees A06, A10 and A16, and one-handers C1, C6 and C11, which were threholded at  Z > 2). For resting state scans, individual time series from the intact/dominant cortex were extracted using a group ROI (see below) and used as individual  ‘ seeds' to model the activation time course for a further first-level FEAT analysis (with white matter and cerebrospinal fluid time series as nuisance regressors). Head motion parameters were included as nuisance regressors in individual scans. For presentation purposes, individual, statistical parametric activation maps with the first contrast were projected on individuals' inflated brains, using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23437228">PMID:23437228</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measures were utilized as covariates in the cortical-surface analysis to control for any potential partial-volume contamination in CBF data. Thickness assessments were performed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ), whereby cortical thickness was calculated as the closest distance from the grey ? /white-matter boundary to the grey/CSF (cerebrospinal fluid) boundary at each vertex on the tessellated surface, as described previously  [63] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23406822">PMID:23406822</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite (FreeSurfer v5.1.0,  http://surfer.nmr.mgh.harvard.edu ), as described in previous publications [ 39 - 41 ]. Briefly the processing included, motion correction, averaging of the two T 1 w images, registration to Talairach space, segmentation of the subcortical WM and deep GM volumetric structures [ 41 , 42 ], intensity normalization, tessellation of the GM/WM boundary, automated topology correction [ 43 , 44 ], and surface deformation following intensity gradients to optimally place the GM/WM and GM/cerebrospinal fluid (CSF) borders [ 39 ]. Image outputs were visually inspected and inaccuracies corrected when required. FreeSurfer provided correct segmentation and classification of subcortical structures in spite of the presence of UBOs in patients with NF1. UBOs were included as belonging to the structure they appeared on, as demonstrated in Additional file  1 : Figure S1. Registration was performed to a spherical atlas [ 45 ]. Cortical thickness was calculated by computing the average shortest distance between the GM/WM and the GM/CSF surface [ 40 ]. Surface area maps of the GM/WM boundary were computed for each subject by calculating the area of every triangle in a cortical surface tessellation, as implemented in FreeSurfer. In anatomical regions of interest, surface area values were calculated by summing the triangular area tessellations included in the region. Local gyrification indices (LGIs) were computed using local measurements of gyrification over the whole cortical surface using the method developed by Schaer  et al . [ 46 ]. The LGI reflects at each vertex the amount of cortical surface area than can be packed under a sphere with a fixed size.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23401776">PMID:23401776</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>WML progression generally begins in the periventricular region and subsequently expands radially to more peripheral locations [ 37 ]. The skewness of the distribution of WML, with a low fraction of high-burden WML subjects also in the dementia part of the sample, can be expected to exist also for clinical samples in general. The statistical atlas used in FreeSurfer was generated from a small sample ( http://surfer.nmr.mgh.harvard.edu/fswiki/Buckner40Testing ), where high-burden WML cases might be rare or lacking. An atlas generated with a low frequency of high-burden exposure may contribute to the FreeSurfer inability to find deep WML.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23359616">PMID:23359616</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were analyzed and the surfaces reconstructed using Freesurfer v4.3.1 ( http://surfer.nmr.mgh.harvard.edu ), as published  [30] . All subjects were affine registered to the Talairach atlas  [31] , image intensity variations due to magnetic field inhomogeneities were normalized, and a skull stripping algorithm was applied  [32] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23326379">PMID:23326379</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Before the cortical thickness analysis, we had visually checked the quality of raw data for subsequent pipeline. The images with distortion and artifact were excluded. Fortunately, no subject was removed according to the criteria. FreeSurfer 5.0 ( http://surfer.nmr.mgh.harvard.edu/ ) was employed to calculate the cortical thickness from the structural magnetic resonance images. Local cortical thickness was measured on the basis of the difference between the position of equivalent vertices in the pial and gray-white matter surfaces. In brief, cerebral white matter was segmented from the T1-weighted images and the gray-white matter interface was estimated. Topographical defects in the gray-white estimate were fixed, which were then used as the starting point for the deformable surface algorithm search for the pial surface. The surface of the gray-white matter border was inflated, and differences between subjects in the depth of the gyri and sulci were normalized. The reconstructed brain of each subject was deformed and registered to an average spherical surface. To obtain cortical thickness difference maps, the data were smoothed on the surface with a Gaussian smoothing kernel with a full-width half maximum of 10 mm. Due to the fact that the BDI score was significantly different between the two groups, the comparison of regional cortical thickness variations between groups was tested by vertex-by-vertex analysis of covariance (ANCOVA) including BDI as the covariate. To correct for multiple comparisons,  p  maps were thresholded to yield an expected false discovery rate (FDR) of 0.05. Cluster comprising of the vertexes showing significantly different cortical thicknesses between online gaming addiction group and control groups were defined. The average thicknesses of the cluster were extracted and used to calculate the % difference to indicate the magnitude of the effect. To investigate the relationship between cortical thickness findings and online gaming addiction, whole brain correlation analysis between cortical thickness and behavioral assessments (i.e. duration and Stroop task response errors respectively) was introduced in the current study. The peak values of the cluster showing significant correlation with the behavioral information (FDR,  p < 0.05) were extracted and used to calculate the correlation coefficients. In the current study, we focused on the brain regions with significantly different cortical thicknesses between online gaming addiction and control groups.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23308283">PMID:23308283</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness was calculated using Freesurfer V.5.1.0  [47] ,  [48]  ( http://surfer.nmr.mgh.harvard.edu/ ). All procedures were performed using the automated surface-based pipeline with default parameters of the Freesurfer package, which mainly included segmentation, surface reconstruction, and surface-based spatial registration. First, the 3D T1 structural images were registered with the Talairach atlas  [49] , and the intensity variation in the white matter was removed by intensity normalization. The skull was stripped using a deformable template model  [50] . White matter was then segmented based on intensity and neighbor constraints. Thereafter, the gray/white surface was obtained by tessellation of the gray/white matter boundary and topology correction. The pial surface was generated by nudging the gray/white matter surface along the T1 intensity gradients to reach the gray matter/cerebrospinal fluid boundary. Both surfaces were represented by vertices. The distance between the gray/white matter surface and its corresponding pial surface was defined as the cortical thickness  [51] . To compare cortical thickness between groups, the cortical surface of each subject was transformed into an average surface space (fsaverage template, provided in Freesurfer package) using a spherical registration method  [52] . Finally, the cortical thickness maps were smoothed with a Gaussian kernel of 10 mm full width at half maximum (FWHM).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23296187">PMID:23296187</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Retinotopic ROIs were identified using standard phase-encoded retinotopic mapping procedures ( Sereno et al., 1995 ). We extracted and normalised the time series for each voxel and applied a fast Fourier transformation to it. Visually responsive voxels were identified by peaks in their power spectra that corresponded to our stimulus frequencies. The preferred polar angle and eccentricity of each voxel was then identified as the phase lag of the signal at the corresponding stimulus frequency (wedge and ring, respectively). The phase lags for each voxel were stored in a  ‘ polar ’  and an  ‘ eccentricity ’  volume and then projected onto the reconstructed, inflated cortical surface (surface based analysis was performed using FreeSurfer:  http://surfer.nmr.mgh.harvard.edu ). The resulting maps allowed us to identify meridian polar angle reversals and thus to delineate the borders of visual areas V1-3 on the cortical surface. These labels were then exported as three-dimensional masks into NIfTI space and served as ROIs.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23284904">PMID:23284904</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The  high-res  template was then processed using a modified version of the automatic anatomical surface reconstruction pipeline of the FreeSurfer toolkit ( http://surfer.nmr.mgh.harvard.edu/ )  [31] ,  [32] . After automatic tissue segmentation, the images were manually inspected to identify errors in the gray / white matter boundary definition. The gray / white matter intensity differences were very large for the majority of the areas in the canine brain and the corresponding tissue segmentation was generally accurate. However, some regions contained partial volume effects and required manual intervention to demarcate the boundary. Control points for white matter voxels were manually defined in areas of problematic tissue segmentation, and the FreeSurfer pipeline re-invoked to estimate the gray white matter boundary. This was performed iteratively until the segmentation was judged sufficiently accurate. Topological holes in the white matter segmentation produced by the ventricles and hippocampi were manually filled. The two hemispheres of the filled white matter volume were then separated. Separate three-dimensional rendering of each white matter hemispheric volume was created. The surface of the three dimensional rendering is the gray matter / white matter boundary which was then smoothed with a surface smoothing kernel to create a smoothed white matter surface. The white matter surface was then inflated using the standard tools available in FreeSurfer.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23284783">PMID:23284783</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To carry out cortical thickness analysis we used Freesurfer version 5.0 software ( http://surfer.nmr.mgh.harvard.edu/ ). Technical details of the procedures to obtain the CTh maps for posterior analysis have been previously and fully described elsewhere  [36] . The procedures for the measurement of CTh have been validated against histological analyses  [37]  and manual measurements  [38] . Freesurfer morphometric procedures show good test-retest reliability across scanner manufacturers and field strengths  [39] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23267320">PMID:23267320</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data preprocessing and analysis were performed using SPM8 ( http://www.fil.ion.ucl.ac.uk/spm/software/spm8 ) and the ART toolbox ( http://www.nitrc.org/projects/artifact_detect/ ). Additionally, the AFNI program 3dClustSim was used to obtain threshold information ( http://afni.nimh.nih.gov/pub/dist/doc/program_help/3dClustSim.html ). Images were created using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23267318">PMID:23267318</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23263472">PMID:23263472</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23251669">PMID:23251669</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 images were processed using Freesurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ) and voxel-based morphometry toolbox (VBM8) implemented using SPM8 ( http://www.fil.ion.ucl.ac.uk/spm ). To measure GM volume in Broca ’ s area, a cortical segmentation was performed on the T1 image from each subject using the Freesurfer. Broca ’ s area traditionally comprises the  pars opercularis  and  pars triangularis . Thus, GM volume in Broca ’ s area was calculated by summing the volumes of these two regions. The statistical significances in volume differences among groups were determined using corrected (relative) volumes by intracranial volume. In addition, to obtain gray matter tissue probability maps for atrophy correction in functional imaging analyses, we conducted an optimized VBM protocol  [27] . First, a study-specific template and priors were created for all participants to minimize errors of spatial normalization and segmentation. Using this template, we spatially normalized and segmented all images and then applied these deformation parameters to the original images. GM voxel values were modulated by multiplying them by the Jacobian determinants derived from the spatial normalization step. These modulated images were smoothed with a 10-mm isotropic Gaussian kernel. To identify any GM regions that were atrophied across groups despite the small sample size, the GM volume maps were compared among groups using an ANOVA. We applied a threshold of  p < 0.005, uncorrected, with an extent of 120 voxels across the whole brain.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23239964">PMID:23239964</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The analyses were performed in Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) implemented in the PICNIC platform ( http://neuroweb.pic.es/ ), using a standard cortical thickness approach. Cortical thickness analyses have previously been validated using histological measures in human brains  [36] , and a good test-retest reliability has been demonstrated across different field strengths, scanner upgrades and manufacturers  [37] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23227001">PMID:23227001</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ’ s DK classifier atlas assigned the initial labels for 54 of the brains in the Mindboggle-101 data set (OASIS-TRT-20, HLN-12, MMRR-21, and MMRR-3T7T-2). These were then manually edited by GM and JT to conform to the DKT protocol as described above. We selected the first 40 brains that we labeled (20 male, 20 female, 26 ? ± ? 7 ? years of age, from the MMRR-21, OASIS-TRT-20, and HLN-12 data) to train a new FreeSurfer cortical parcellation atlas representing the DKT protocol (see  http://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/GcaFormat ; S ’ egonne et al.,  2004 ; Desikan et al.,  2006  for details regarding the algorithm that generates the atlas and how it is implemented). The resulting  “ DKT40 classifier atlas ”  then automatically generated the initial set of cortical labels for the remaining 47 brains in the data set (see  http://surfer.nmr.mgh.harvard.edu/fswiki/mris_ca_label ). To our knowledge, the DKT40 atlas was generated in the same manner as the DK atlas except for differences in the labeling protocol and training set.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23226269">PMID:23226269</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>3D T1-weighted brain MRI scans were acquired using a sagittal 3D MP-RAGE sequence following the ADNI MRI protocol  [66] . Baseline 1.5T MRI scans from 204 participants were downloaded from the ADNI public website ( http://www.loni.ucla.edu/ADNI/ ) onto local servers at Indiana University School of Medicine. As detailed in previous studies  [67] , FreeSurfer V4 software ( http://surfer.nmr.mgh.harvard.edu/ ), a widely employed brain segmentation and cortical parcellation tool, was used to label cortical and subcortical tissue classes using an atlas-based Bayesian segmentation procedure and to extract the measure of brain volume.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23226127">PMID:23226127</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to obtain connection strengths between brain regions, the regions must first be defined. This task typically takes one of several routes. Structural parcellation takes a structural image and parcellates it with an algorithm that uses anatomical information in the image and prior models to determine cortical and subcortical regional boundaries. Common analysis packages for performing parcellation are Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) and Automated Anatomical Labeling (AAL) (Tzourio-Mazoyer et al.,  2002 ). Functional parcellation can be performed on fMRI data based on a search for regions whose functional connectivity patterns are statistically similar in time and/or space (Craddock et al.,  2012 ). A third strategy is to use predefined subregions of cortex/subcortex from a predefined atlas such as the Harvard-Oxford cortical/subcortical probabilistic atlas distributed with the FMRIB Software Library (FSL). The set of ROIs are normally spatially registered to the subject's image space where connectivity is to be estimated. A fourth strategy is to use a set of meta-analytically defined coordinates in a standard stereotactic space (e.g., MNI152) based on sites of peak activation during behavioral tasks. Small regions of interest, typically spheres of 5 – 10 mm radius, are created around each coordinate and used as seeds to calculate connectivity strength with the remaining spheres (Power et al.,  2011 ). For any parcellation scheme, different tissue types may be included or excluded. fMRI-based analyses are typically uninterested in WM signal and may use gray matter ROIs. Conversely, dwMRI studies are more focused on water diffusion in WM and often use the gray/WM interface as a starting point for tractography. In this case, ROIs may include portions of both gray and WM. For atlas-based ROIs, region boundaries are commonly defined using probabilistic estimates rather than hard cutoffs. In this case, the user must decide a probability threshold above which to assign regional labels.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23226114">PMID:23226114</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23222445">PMID:23222445</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were analysed using SPM5 ( http://www.fil.ion.ucl.ac.uk/spm/ ). We discarded the first five images of each scanning run to allow for magnetic saturation. After this, images were realigned and coregistered to the individual structural scans for each participant, and data were spatially smoothed using a 4 mm full width at half maximum (FWHM) kernel. For the second-level group analysis, images were spatially normalized to the MNI template. For the univariate analysis and the localizer sessions, a general linear model was fitted to the data using regressors for each of the experimental conditions and covariates of no interest for the motion parameters. Regions of interest were delineated manually using software Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) by first segmenting the structural scan for each participant and reconstructing an inflated mesh of the boundary between grey and white matter to project the activations from the localizer runs onto this surface. For retinotopic mapping, we contrasted the response to vertical and horizontal meridian stimulation, and drew the boundaries of areas V1 – V3 along the peaks of the positive and negative activations. We defined V5/hMT+ by contrasting the response to moving and static stimuli and selecting the cluster of significant voxels in lateral occipital cortex. A control region, where above-chance decoding would not be expected, was defined in frontal cortex for each participant in an area that showed no stimulus-specific activity, as defined by contrasting the response to all conditions with fixation rest blocks. Binary volume masks for each region of interest (ROI) were then generated by projecting the grey matter voxels that fell within a region back from the surface into the native volume for each participant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23189084">PMID:23189084</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Flow chart representing the selection of the current MRI study sample from the initial DemWest study cohort. WML = White matter lesion.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23185621">PMID:23185621</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each subject underwent MRI on a 3T Siemens scanner, using a protocol that included a T1-weighted 3D MPRAGE sequence (TR/TE/TI 2300/3/900 ms; flip angle 9 ° ; sagittal acquisition with FOV 256 × 240 mm 2  and 1 mm thick slices; matrix 256 × 240 with 160 slices yielding 1 mm 3  isotropic voxels). Total intracranial volume (TIV) was calculated using the BET and FAST tools from the FSL 4.1 software package  [39]  ( http://www.fmrib.ox.ac.uk/fsl/ ) to segment the brain into gray matter, white matter, and CSF; TIV was calculated as the sum of the three tissue types. Because not all subjects had a fluid attenuation inverse recovery (FLAIR) or T2 sequence, the T1 scan was also used to derive white matter hypointensity volume, calculated from an automated subcortical segmentation routine using Freesurfer  [40]  ( http://surfer.nmr.mgh.harvard.edu/ ). Gray and white matter voxel-based morphometry (VBM) were carried out using SPM8  [41]  ( http://www.fil.ion.ucl.ac.uk/spm/ ), following the standard processing stream, using the Segment routine to generate gray and white matter partial volume maps, then DARTEL for spatial normalization; default parameters were used throughout.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23185334">PMID:23185334</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI data were processed using FSL 4.1 ( http://www.fmrib.ox.ac.uk/fsl/ ;  [43] ,  [44] ) and FreeSurfer v4.5.0 ( http://surfer.nmr.mgh.harvard.edu/ \n [45] ,  [46] ). First-level fMRI and DWI data were motion- and eddy current-corrected (using MCFLIRT  [47]  and eddy_correct, respectively), unwarped (using PRELUDE/FUGUE) and brain-extracted (using BET  [48] ; all part of FSL). First-level fMRI analysis was carried out by applying the General Linear Model (GLM) within FEAT using FILM prewhitening  [49] , with motion outliers (detected by fsl_motion_outliers) being added as confound regressors. High-pass temporal filtering of the data and the model was set to 100 secs based on the power spectra of the design matrices (estimated by cutoffcalc; all part of FSL). Three main explanatory variables were modeled and controlled: Optical flow, age and gender change. Parametric intensity modulation of their graded stimulus strength is described below, linear modeling of age and gender changes did not explain a significant amount of variance on top of this. Button press responses to anticipated target face appearance were also modeled but of no further interest. In order to capture slight deviations from the model, temporal derivatives of all explanatory variables convolved with FEAT's gamma hemodynamic response function (HRF) were included.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23162453">PMID:23162453</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed on the 3D T1-weighted structural images with the FreeSurfer image analysis suite (version 4.3) ( http://surfer.nmr.mgh.harvard.edu/ ), which has been shown to be comparable in accuracy to manual labeling (Fischl et al.,  2002 ). The technical details of these procedures are described in detail elsewhere (Dale et al.,  1999 ; Fischl et al.,  1999 ; Fisher and Defries,  2002 ; Grieve et al.,  2011 ). Volumetric measurements of the hippocampus and the amygdala were generated with FreeSurfer's automatic quantification of subcortical structures, which assigns a neuroanatomical label to each voxel in an MRI volume based on probabilistic information estimated from a manually labeled training set (see Figure  1 ). The classification technique employs a non-linear registration procedure that is robust to anatomical variability, as described in detail elsewhere (Fischl et al.,  2002 ). Anatomically labeled structures were manually inspected for accuracy, and corrected if necessary. Total brain volumes, which were used as a covariate in the statistical analyses, were also generated automatically with the FreeSurfer processing stream.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23152828">PMID:23152828</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) is a set of automated tools for reconstruction of the brain's cortical surface from structural MRI data  [36] [37] [38] [39] [40] . Briefly, this processing includes motion correction, removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white matter and deep grey matter volumetric structures, intensity normalization, tessellation of the grey matter/white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally segment borders at the location where the greatest shift in intensity defines the transition to the other tissue class . In its subcortical segmentation procedure, Freesurfer includes a label for WMH. T1 images were processed with Freesurfer and WMH segmentations were obtained in T1 space. They were then registered to FLAIR space using the transformation computed in the preprocessing step 2, in order to compare the results with the references.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23133666">PMID:23133666</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 images of all subjects were preprocessed with the volume and surface pipeline of FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ )  [17] . The cortical thickness was obtained for each subject with the following steps: Firstly, images were corrected for intensity nonuniformity using N3  [21] . Secondly, images were registered via affine transformation to Montreal Neurological Institute space to strip the skull. Thirdly, binary white matter masks were generated using the segmentation information and corrected in areas that commonly produce topological defects. Fourthly, a triangle-based mesh of the white matter surface was then produced and expanded to get the pial surface. Fifthly, cortical thickness data was then determined for each subject by computing the mean of the two shortest distances between the white matter and pial surfaces. However, to compare the cortical thickness between two different groups, an average surface template was built using the curvature-based registration. Finally, all of the individual surfaces were aligned to this template and the distribution of original cortical thickness on the template was obtained using reparameterization.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23125828">PMID:23125828</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Our approach is based on a region-of-interest (ROI) analysis (Nieto-Castanon et al.,  2003 ; Poldrack,  2007 ). Two anatomically defined ROIs, the left and right amygdalae, were automatically segmented in each subject's individual anatomical brain scan with the FreeSurfer (v. 5.1) image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ) (Fischl et al.,  2002 ,  2004 ). This segmentation is shown on one of our subjects in Figure  1 . One advantage of using anatomically defined ROIs is that it eliminates the potential problem of circular analysis that exists for functionally defined ROIs (Poldrack,  2007 ; Kriegeskorte et al.,  2009 ; Poldrack and Mumford,  2009 ; Vul et al.,  2009 ). There are other advantages of using an ROI-based approach. First, it removes some variability due to noise by averaging over all voxels in the ROI. Second, it is more statistically powerful than other methods, since it controls for Type I errors by limiting the number of statistical tests. Third, it enables precise spatial correspondence of the region-of-interest across subjects, since it does not involve normalizing different brains to a common atlas (Poldrack,  2007 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23071654">PMID:23071654</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MNE data analysis was conducted using MNE suite (v2.7.0,  http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/sofMNE.php ) for a subgroup of N ? = ? 10 subjects for which MRI images were available, and thus in whom surface reconstructions were possible. The inflated surface reconstructions  [21] ,  [22]  were done using FreeSurfer (v4.5.0,  http://surfer.nmr.mgh.harvard.edu/ ) software.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23071638">PMID:23071638</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All quantitative MRI analyses were performed on T1-weighted MPRAGE images using an Apple ®  Mac Pro (OS X 10.7, 32 GB, 2 × 2.93 GHz 6-Core Intel Xeon (HT), Apple ®  Inc., Cupertino, CA, USA). FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) was used to obtain subcortical and cortex volumes for all patients and controls using an observer-independent approach. The volumes analysed were the left and right hippocampus, thalamus, putamen, temporal horn of the lateral ventricle and entire hemispheric cortex. We also analysed the areas of the (i) anterior, (ii) mid-anterior, (iii) central, (iv) mid-posterior and (v) posterior corpus callosum. These volumetric ROIs were selected for investigation in order to maximise correspondence with subcortical DTI ROIs. All segmentations were based on the assignment of neuroanatomical labels to each voxel in an MR image based on the probabilistic information automatically estimated from a manually labelled training set. The methods of the automated volumetric approach have been described in detail previously  [31] , and the accuracy of automated labelling and volumetry of subcortical structures have been independently validated with respect to  ‘ gold standard ’  manual volumetric techniques, predominantly for the hippocampus  [32] ,  [33] ,  [34] , and also of the amygdala  [33] ,  [34] , striatum  [33]  and thalamus  [35] . The FreeSurfer  ‘ recon-all ’  function (for cortical reconstruction and brain segmentation) segmented and labelled 23 MRIs in approximately 17 hours using the Mac Pro. After the  ‘ recon-all ’  function, the neuroanatomical labels were inspected for accuracy in all patients and controls. Despite that FreeSurfer permits manual editing to improve subcortical segmentation, no obvious errors in the automatic labelling were observed for any subject, and so all data obtained from FreeSurfer analyses were 100% automated.  Figure 1B  shows a randomly selected patient with TLEhs for whom volumes have been automatically labelled and extracted.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23056945">PMID:23056945</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>(c) Cortical Thickness Evaluation Global cortical thickness (CTh) (mean of right and left hemispheres) was performed, as previously described [ 20 ] on the volumetric FFE data sets by means of Freesurfer image analysis [ 20 ] suite (release v5.1.0), available online ( http://surfer.nmr.mgh.harvard.edu/ ). All images were accurately controlled for errors/artifacts by an experienced neurologist (MC). A first visual ? ? inspection was necessary after the skull stripping to visualize areas not completely removed, while a second visual inspection was needed after WM/GM segmentation, to visualize possible misclassifications (especially due to MS lesions).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23049695">PMID:23049695</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Because we were mainly interested in activity in the thalamus region and in the subcortical areas known to be involved in tremor, in the  “ Tremor ”  protocol we restricted the analysis to regions of interest (ROIs) by applying two anatomy-based masks to the analysis. The first mask included only Thalamus, Putamen, Caudate, Cerebellum and Brainstem, excluding cortical areas (Circuit mask); the second mask was a thalamus-only mask. Anatomical regions were identified on each patient ’ s own T1 MRI scan using Freesurfer imaging analysis suite ( http://surfer.nmr.mgh.harvard.edu ,  Figure S3 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22988433">PMID:22988433</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>5 surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22973250">PMID:22973250</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22969710">PMID:22969710</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface is reconstructed by FreeSurfer (v5.0.0,  surfer.nmr.mgh.harvard.edu/ ) using the methods proposed by Dale et al. ( 1999 ) and Fischl et al. ( 1999 ), and CT on each vertex is measured. First, all T1-weighted images are corrected for intensity non-uniformity and registered into a template. Then the gray matter (GM) and white matter (WM) and subcortical tissues are segmented. A smoothly tessellated cortical surface is constructed for each hemisphere and automatic correction is preformed to remove topological defects. After that, the pial surface and GM/WM surface are built and the shortest distances from pial surface to GM/WM surface and from GM/WM surface to pial surface are calculated, respectively, on each vertex. Finally, CT on each vertex is obtained as the mean value of these two distances and a surface-based diffusion smoothing with 20 mm full-width-at-half-maximum (FWHM) is conducted to enhance the statistical performance.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22958316">PMID:22958316</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing and definition of cortical and subcortical gray matter regions on structural images were conducted in the UCLA Laboratory of Neuro Imaging (LONI) Pipeline Processing Environment\n[ 33 - 35 ] and using FreeSurfer ’ s automated brain segmentation software (FreeSurfer 4.0.5,\n http://surfer.nmr.mgh.harvard.edu ), as described in the work of Fischl and Dale\n[ 36 - 38 ]. We obtained volume measurements of seven subcortical brain regions (thalamus, caudate, putamen, pallidum, hippocampus, amygdala, and ventral diencephalon) as well as six frontal cortical regions (caudal middle frontal cortex, rostral middle frontal cortex, lateral orbitofrontal cortex, medial orbitofrontal cortex, superior frontal cortex, and frontal poles). During preprocessing, high-resolution T1-weighted image acquisitions for each participant were visually inspected for motion artifacts by a trained rater based on a 5-point Likert scale illustrating the severity of motion effects. No participants were rejected due to motion artifacts; however, one subject was rejected due to poor gray -white matter contrast, and another subject was rejected because the raw image data files appeared corrupted. In all remaining participants ( n ? = ? 40), the T1-weighted images were brain extracted, and gray-white matter boundaries were automatically delineated. All brain extractions were inspected visually and corrected manually as needed.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22952849">PMID:22952849</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data processing was conducted using FS-FAST and FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). All functional images were motion corrected  [34]  and registered to the individual anatomically reconstructed brain.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22937122">PMID:22937122</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surfaces were reconstructed from the high resolution anatomical 3D – MPRAGE data sets of each subject using FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu ). The automated processing stream included white matter segmentation and tessellation to identify grey/white matter and grey matter/pial surface interfaces. An experienced neuroradiologist visually inspected segmentation results and inaccuracies were manually corrected prior to subsequent processing. Thus, the produced maps are not restricted to the voxel resolution of the original data and allow detection of sub-millimeter differences. Summary tables of the cortical parcellations (156 anatomical structures for each hemisphere) were used for further statistical analysis. After the scans from the two time points were processed, cross-sectionally, with the default FreeSurfer workflow, an unbiased template from both time points was created for each subject and processed cross-sectionally. This unbiased template reduces the random variation in the processing procedure and improves the strength and sensitivity of the overall longitudinal analysis. One subject from both groups was excluded from further analysis due to errors encountered in the automated process which could not be corrected manually. Vertex-based difference maps between the two time points were then created, re-sampled to a common map created from all included scans and smoothed with a Gaussian kernel (full-width of half-maximum of 15 mm). A two-way two time-point longitudinal analysis, F-test (unsigned), was performed to model vertex-wise difference maps with group as two separate planes (two levels of a single discrete factor) with one continuous factor as covariate (age). Areas included in this evaluation were limited to the DMN and DAN and therefore no statistical correction was applied.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22920586">PMID:22920586</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Frontal regional cortical volumes were based on the Freesurfer image analysis software 4.0 version ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publication [ 20 ]. Cortical volume combined both thickness and area information and parcellation of the cerebral cortex into units based on gyral and sulcal structure [ 21 ] were done to get regional frontal volumes.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22880116">PMID:22880116</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI analysis was performed by an experienced observer, unaware of subjects' identity. Axial DE images were analyzed to assess the presence and location of areas with increased corticospinal tract signal intensity. Cortical reconstruction and thickness estimation was performed on the MP-RAGE images using the Freesurfer image analysis suite, version 4.5 ( http://surfer.nmr.mgh.harvard.edu/ )  [12]  on a 64-bit Linux CentOS 4. Briefly, after registration to the Talairach space and intensity normalization, the process involves an automatic skull stripping by using an hybrid method combining watershed algorithms and deformable surface models. Then, the white matter (WM)/GM boundary was tessellated and the surface was deformed following GM/cerebrospinal fluid (CSF) intensity gradients to optimally place WM/GM and GM/CSF borders. The results of these segmentations were inspected visually, and if needed, edited manually by adding control points. Finally, an automatic reconstruction of the cortex was produced and cortical thickness estimated by computing the average shortest distance between the WM boundary and the GM/CSF surface. Surface maps were generated following registration of all subjects' cortical reconstructions to a common average surface. Finally, surface maps were smoothed using a surface-based Gaussian kernel of 15 mm full width half-maximum. This smoothing kernel is slightly larger than the default kernel of Freesurfer (i.e., 10 mm) in order to ensure a better signal-to-noise ratio and a more precise alignment between input surfaces  [39] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22880000">PMID:22880000</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface-based reconstruction and volume segmentation were performed using the Freesurfer image analysis suite (version 4.5.0,  http://surfer.nmr.mgh.harvard.edu/ ), which has been described and validated in previous papers  [33] ,  [34] . In brief, non-brain tissue is removed using a hybrid watershed/surface deformation procedure, after which the segmentation of the subcortical white matter and deep gray matter structures is performed. After intensity normalisation, the gray/white matter boundary is identified and tessellated. This surface is further used by a surface deformation algorithm aimed at optimally placing the gray/white and gray/CSF borders with sub-millimeter precision. Thereby, topologically correct polygonal mesh models of the cortical surfaces are created. The cortical models for all subjects are registered to a common surface template through a high-resolution surface-based averaging technique based on cortical folding patterns  [35] . Parcellations grounded on gyral and sulcal structure (74 per hemisphere) are mapped from the template to each subject ’ s native space using a high-dimensional spherical morphing procedure  [36] . Finally, a number of surface-based measures are calculated for each parcellation unit, such as gray matter volume, cortical thickness, cortical surface area and gyrification  [37] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22867031">PMID:22867031</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The main interest in our Dissociative Identity Disorder study is volumetric and cortical thickness comparison of different psychiatric groups, so in this study we concentrated on the results from two fully automated brain analysis segmentation tools: FreeSurfer version 4.5 (http://surfer.nmr.mgh.harvard.edu) and Voxel Based Morphometry (VBM) (as implemented within SPM5 (www.fil.ion.ucl.ac.uk/spm)). A brief description of segmentation procedures follows, with further details provided in previous publications [ 36 , 37 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22860067">PMID:22860067</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>An automated cortical reconstruction of the T1-weighted images was performed by using FreeSurfer (version 4.3.1) software ( http://surfer.nmr.mgh.harvard.edu ). This method was used to create a three-dimensional cortical surface model of CTh using intensity and continuity information previously described in detail  [26] . Processing of T1 high-resolution images included several procedures: removal of non-brain tissue  [27] , automated Talairach transformation, intensity normalization  [28] , tessellation of the gray matter/white matter boundary, automated topology correction,  [29] ,  [30]  and surface deformation to detect gray matter/white matter and gray matter/cerebrospinal fluid boundaries  [26] . Moreover, the cerebral cortex was divided into different regions according to gyral and sulcal structure information  [31] . The resulting representation of cortical thickness was calculated as the distance between tissue boundaries (gray matter/white matter and gray matter/cerebrospinal fluid)  [26] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22860044">PMID:22860044</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The fMRI data were preprocessed using SPM8 software ( http://www.fil.ion.ucl.ac.uk/spm/software/spm8/ ), including realignment, co-registration, normalization into MNI space, and smoothing with a 6-mm (full-width-at-half-maximum) Gaussian filter. Before normalization, the images were skull-stripped using the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22832956">PMID:22832956</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The image files in DICOM format were transferred to a Linux workstation for morphometric analysis. Images were corrected for non-linear warping caused by gradient coil non-linearities, using tools developed through the Morphometry Biomedical Informatics Research Network, 31  and the two T1-weighted images were rigid body registered to each other (motion corrected). The FreeSurfer 3.0.2 software package ( http://surfer.nmr.mgh.harvard.edu ) was used to obtain measures of subcortical volumes 32  and cortical area and thickness. 33 ,  34  Summary measures were generated and used as structural phenotypes. 35  For more details, see  Supplementary Information  online.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22802944">PMID:22802944</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 weighted structural images of all participants were processed using automated cortical and subcortical segmentation (aseg.volume.stats and bilateral aparc.volume.stats files, see Information S1) within the recon-all pipeline of the Freesurfer package ( http://surfer.nmr.mgh.harvard.edu ). This procedure includes cortical surface modelling, spherical coordinate transformation, nonlinear curvature registration, and automated segmentation of cortical and subcortical structures. The estimated volume of each segmented region obtained using this routine was then used as the input to the classifiers. Further details about recon-all pipeline can be found at  [14] ,  [15] ,  [16] .  Note : Freesurfer software labels the basal putamen as  ‘ vessel ’ , since it is an area with prominent vascular space.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22731740">PMID:22731740</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All participants received 1.5 Tesla (T) structural MRI. The analyses in this study were based on the imaging data from the ADNI database processed by the team at the University of California at San Francisco, which performed cortical reconstruction and volumetric segmentation with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The detailed procedure is available at  http://adni.loni.ucla.edu/research/imaging-analysis/ . A list of 237 MRI features used in this study is provided at the Additional file  1 : Supplemental document. More details on ADNI neuroimaging instrumentation and procedures [ 14 ] can be found at  http://www.loni.ucla.edu . About 50% of MCI subjects (74 MCI Converts, 86 MCI Non-converters) have a complete set of CSF measurements and MRI scans at the baseline. CSF was analyzed at the ADNI Biomarker Core laboratory at the University of Pennsylvania Medical Center.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22721958">PMID:22721958</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition, to establish which cortical regions were most likely to be deafferented, diffusion-weighted data from 12 healthy aged-matched male subjects following the algorithm of  Draganski et   al. (2008) . After automated cortical and subcortical parcellation using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) we performed probabilistic diffusion tractography in subject-specific native space using a probabilistic index of connectivity (PICo) algorithm ( Parker and Alexander, 2003 ,  2005 ) implemented in Camino software ( http://www.cs.ucl.ac.uk/research/medic/camino/ ). To delineate the projection sites of specific cortical areas on the pallidum ( Fig.   2 A) we implemented a two stage probabilistic tractography approach: (i) probabilistic tractography from caudate to cortical targets as defined in FreeSurfer (LOFC  –  lateral orbitofrontal cortex, M1  –  precentral and paracentral gyrus) and (ii) probabilistic tractography from pallidum to caudate after definition of the specific cortical projection sites. We calculated voxel-based PICo maps for the pallidum seed structures to each target area and transformed the individual maps to standard MNI space using parameter estimates from each individual's T1-weighted data.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22719849">PMID:22719849</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer 5.0 ( http://surfer.nmr.mgh.harvard.edu/ ) was employed to calculate the cortical thickness from the structural MRI according to the detailed procedures in previous studies  [42] ,  [43] ,  [44] . Local cortical thickness was measured on the basis of the difference between the position of equivalent vertices in the pial and gray-white matter surfaces. In brief, cerebral white matter was segmented from the T1-weighted images and the gray-white matter interface was estimated  [43] . Topographical defects in the gray-white estimate were fixed, which were then used as the starting point for the deformable surface algorithm search for the pial surface. The surface of the gray-white matter border was inflated, and differences between subjects in the depth of the gyri and sulci were normalized. The reconstructed brain of each subject was deformed and registered to an average spherical surface  [44] . To obtain cortical thickness difference maps, the data were smoothed on the surface with a Gaussian smoothing kernel with a full-width half maximum of 10 mm. Statistical thickness difference maps were constructed with  t  statistics (with the inclusion of age and gender as covariates) with a threshold of P < 0.05 ((false discovery rate (FDR) corrected)).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22693642">PMID:22693642</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Modulations of cortical oscillatory activity were studied using an MRI-constrained MEG source modeling approach  [50] ,  [51] . The information from structural segmentation of the individual MRIs and the MEG sensor locations were used to compute the forward solutions for all source locations using a boundary element model (BEM)  [52] ,  [53] . For source estimation from MEG raw data, cortical surfaces extracted  [54]  with the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) were decimated to  ? 1,000 vertices per hemisphere. The individual forward solutions for current dipoles placed at these vertices comprised the columns of the gain matrix ( A ). A noise covariance matrix ( C ) was estimated from the raw MEG data. These two matrices, along with the source covariance matrix  R , were used to calculate the depth-weighted minimum-norm estimate (MNE) inverse operator  W   ? = ?   RA T  ( ARA T   +   C ) ? 1 . To estimate cortical oscillatory activity in the cortical sources, the recorded raw MEG time series at the sensors  x(t)  were multiplied by the inverse operator  W  to yield the estimated source activity, as a function of time, on the cortical surface:  s ( t )  ? = ?   Wx ( t ) ( e.g. ,  [55] ,  [56] ). For whole-brain cortical power estimates, source activities were estimated for all cortical vertices using a loose orientation constraint  [55] . Additionally, 16 regions of interest (ROI), selected from areas where crossmodal modulations of posterior alpha activity were hypothesized to be largest, were identified from each subject/hemisphere based on the standard anatomical parcellation of FreeSurfer 5.0  [57]  shown in  Figure \n 1 . For the TFR analyses, to reduce the computational load, the source component normal to the cortical surface was employed and an average raw data time course was obtained from each ROI, with the signs of the source waveforms aligned on the basis of surface-normal orientations within each ROI to avoid phase cancellations.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22675527">PMID:22675527</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer (Athinoula A. Martinos Center for Biomedical Imaging, Harvard-MIT, Boston) comprises a popular and freely available set of tools for deriving neuroanatomical volume and cortical thickness measurements from automated brain segmentation ( http://surfer.nmr.mgh.harvard.edu ), recently summarised by Fischl  [1] . A number of reported studies discussed the accuracy of the technique by comparing the volume of specific brain structures, such as the hippocampus or amygdala, with manually derived volumes  [2] – [5] . The measurement of cortical thickness was validated against histological analysis  [6]  and manual measurements  [7] ,  [8] . Also the reliability of the measurements was subject of a number of investigations. Some of these studies addressed the effect of scanner-specific parameters, including field strength, pulse sequence, scanner upgrade, and vendor (cortical thickness:  [9] ,  [10] ; volume:  [11] ). In addition, the scan-rescan variability of a number of subcortical brain volumes was assessed  [12] – [14] . Finally, it has been shown that Freesurfer is capable of reliably capturing (subtle) morphological and pathological changes in the brain (e.g.,  [5] ,  [13] ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22675310">PMID:22675310</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22664957">PMID:22664957</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI data were processed at the Psychiatry Neuroimaging Laboratory using 3D Slicer to visualize and realign the data ( http://www.slicer.org ). The FMRIB Software Library (FSL v.4.1;  http://www.fmrib.ox.ac.uk ) was used for skull stripping ( 15 ) and to estimate the intracranial volume to control for head size, and the FreeSurfer software package (version 4.5.0;  http://surfer.nmr.mgh.harvard.edu/ ) was used for automated segmentation of the hippocampus ( 16 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22658599">PMID:22658599</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>(B) Observers binocularly viewed a wide-field, flickering, colored wedge stimulus that rotated clockwise or counterclockwise (in separate runs). To ensure that subjects attended to the rotating wedge while maintaining central fixation, a small gray disc appeared at random intervals within the rotating wedge stimulus, and the subject had to report the number of times this occurred in each scan run. The resulting phase map was displayed on a reconstructed inflated surface of the individual's structural scan with FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). The boundaries of the visual areas were defined manually by identifying phase reversals in the phase map that corresponded to representation of the vertical and horizontal meridians. Mask volume images were created for ventral V1, V2, V3, and V4 in the left hemisphere for all participants.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22654746">PMID:22654746</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We examined regions included in the DMN as defined by Fox et al. ( 2005 ), and regions included in the FP, CO, and CER networks as defined by Dosenbach et al. ( 2007 ). To control for individual anatomical variability, ROI were defined for each individual in two steps. First, we created spherical ROIs in standard Talairach space centered on the reported coordinates for each region (Figure  1 ; Table S1 in Supplementary Material) and 15 ? mm in diameter. Second, we masked the resulting group ROIs with the individual FreeSurfer ( http://surfer.nmr.mgh.harvard.edu , version 4.1.0) segmentation of a high-resolution structural image that was previously registered to standard Talairach space, excluding any voxels within the group defined ROIs that did not represent the relevant gray matter (cerebral cortex, cerebellar cortex, hippocampus, thalamus) in the specific individual, as defined by FreeSurfer (Fischl et al.,  2002 ). Given that we used  a priori  ROIs, we conducted a number of analyses in the control subjects using these ROIs to validate the expected pattern of connectivity within and between these four networks. The results of these analyses are presented in the supplement.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22615955">PMID:22615955</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Total brain and cerebellar gray and white matter volumes were measured automatically using the software FreeSurfer version 5.0 ( http://surfer.nmr.mgh.harvard.edu )  [19] – [21] . Briefly, processing consisted of removal of non-brain tissue  [22] , and automatic segmentation of (sub)cortical gray matter structures  [23] . This procedure automatically assigns a neuroanatomical label to each voxel in an MRI volume, based on probabilistic information automatically estimated from a manually labeled training set. First, the image is rigid body registered to a probabilistic brain atlas followed by non-linear morphing to the atlas. Manually segmented images were previously used to create statistics about how likely a particular label is at any given location in the brain. This serves as a Bayesian prior for estimating the label of a given voxel in a given patient's brain image based on the maximum a posteriori probability. The segmentation uses three pieces of information to disambiguate labels: (1) the prior probability of a given tissue class occurring at a specific atlas location, (2) the likelihood of the image intensity given that tissue class, and (3) the probability of the local spatial configuration of labels given the tissue class. The technique has previously been shown to be comparable in accuracy to manual labeling  [23] . Finally, all cerebellar segmentations were visually inspected for accuracy prior to inclusion in the group analysis.  Figure 1  depicts an example of a representative anatomical T1-weighted scan that was used for automated volume extraction.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22615886">PMID:22615886</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Lesion-TOADS has two major advantages over methods used in previous studies to perform segmentation and classification of brain structures in MS. First, unlike tissue classification tools such as SPM ( http://fil.ion.ucl.ac.uk/spm ) and FSL's FAST, SIENA, and SIENAX, which only yield CSF, GM, and WM volumes, Lesion-TOADS provides a more detailed subsegmentation for each tissue. Second, neither these methods nor Freesurfer ( http://surfer.nmr.mgh.harvard.edu ), which is commonly used for brain substructure and cortical segmentation, nor FSL's FIRST (FSL's tool for segmentation of brain substructures), delineates MS lesions as part of the segmentation process. This commonly results in misclassifications, since lesions are often segmented as GM. In all these methods, a separate lesion delineation step, often performed manually or semi-automatically, is required to correct the problem  [17] ,  [46] . Lesion-TOADS addresses this issue without any manual interaction.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22615876">PMID:22615876</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was performed with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications  [17] – [21] . Briefly, processing includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure  [21] , automated Talairach transformation, intensity normalization in order to correct for MR intensity nonuniformity mainly arising from variations in the sensitivity of the reception coil and from gradient-driven eddy currents  [22] , tessellation of the gray matter white matter boundary, automated topology correction  [23] ,  [24] , and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class  [17] ,  [20] . Once the cortical models are complete, a number of deformable procedures can be performed for in further data processing and analysis including surface inflation  [18] , registration to a spherical atlas which utilized individual cortical folding patterns to match cortical geometry across subjects  [19] , parcellation of the cerebral cortex into units based on gyral and sulcal structure, and creation of a variety of surface based data including maps of curvature and sulcal depth. This method uses both intensity and continuity information from the entire three dimensional MR volume in segmentation and deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/cerebrospinal fluid boundary at each vertex on the tessellated surface  [20] . The maps are created using spatial intensity gradients across tissue classes and are therefore not simply reliant on absolute signal intensity. The maps produced are not restricted to the voxel resolution of the original data and thus are capable of detecting submillimeter differences between groups. Cortical thickness measurements as implemented in FreeSurfer have been validated against manual measurements  [3] ,  [25]  and histological analysis  [26] . Further, FreeSurfer morphometric procedures have been demonstrated to show sufficient test-retest reliability across scanner manufacturers, field strengths, and other imaging parameters  [27] – [29] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22607202">PMID:22607202</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with the FreeSurfer application version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) and technical details of these procedures have been previously described [ 68 - 78 ]. Briefly, this process involves the motion correction and averaging of two volumetric T1-weighted images [ 79 ], removal of non-brain tissue [ 78 ], transformation of the scans to the standard Talairach space, segmentation of the subcortical white matter and deep gray matter volumetric structures [ 71 , 72 ], intensity normalization [ 80 ], tessellation of the gray matter/white matter boundary, topology correction [ 70 , 81 ], surface deformation along intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders [ 68 , 69 , 82 ], and finally cortical representations were parcellated into anatomical structures. Results were visually inspected and any inaccuracies were manually edited. FreeSurfer ’ s procedure for measuring cortical thickness have been validated against histological analysis [ 83 ] and manual measurements [ 84 , 85 ], and morphometric procedures have been demonstrated to show good test-retest reliability across scanner and field strengths [ 76 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22593738">PMID:22593738</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>On the one hand, no informed analysis of the visual cortex could be done without knowledge of its retinotopic and functional organization; on the other hand, no retinotopic mapping or visual localizers are possible in the blind, so it was a challenge to localize any specific visual area. To resolve this issue and determine the borders of area V1 in blind participants, we took a three-pronged approach. First, we used the Freesurfer probability map atlas (see  http://surfer.nmr.mgh.harvard.edu/fswiki/BrodmannAreaMaps ), to transform the primary visual area map back to the blind subject's brain through the Freesurfer spherical surface registration procedure. To verify the process, we first ran this procedure in the brains of sighted subjects, for which we already had individual retinotopic maps; the borders of the retinotopically defined V1 aligned fairly accurately with those from the Freesurfer. Second, we verified the location of the V1 ROI by intersecting with its anatomical marker (the calcarine sulcus). And third, we used an innovative 14-step procedure (Likova,  2010a , b ,  2012 ) that allows us to warp the brains of sighted and blind subjects to the same MNI brain. This innovative three-way comparison enabled us to estimate the corresponding topographic regions in the blind brain. All methods converged very well to the definition of the V1 ROI.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22551361">PMID:22551361</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A 3-D T1-weighted gradient-echo sequence (MPRAGE, Siemens) was used with the FreeSurfer image analysis suite [ 26 ]. Up to eight simultaneous analyses were performed on an eight-core Mac Pro with 16 GB RAM (Apple Inc., Cupertino, CA) with an average  recon-all  time of 19 hours per subject  http://surfer.nmr.mgh.harvard.edu .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22536413">PMID:22536413</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Subcortical target regions were defined using the Harvard-Oxford subcortical probabilistic atlas ( http://www.cma.mgh.harvard.edu/fsl_atlas.html ) with the probability volumes of the putamen and thalamus thresholded at  p > 0.9 so as to exclude neighbouring gray and white matter. Cortical target regions were defined using Freesurfer's ( http://surfer.nmr.mgh.harvard.edu ) automated gray matter parcellation (with the aparc2005 atlas)  [30]  and were transformed to MNI 152 standard space. The IFG was divided into the IFGo and IFGt as these subregions have different cytoarchitecture and different white matter connectivity profiles  [19] ,  [31] ,  [32] . The insula ROI encompassed the short gyrus, which includes regions that are classified as anterior and middle portions of the insula  [33] ,  [34] ,  [35] . The more posterior portions of the insula were excluded from the analysis because the close proximity to the TPJ may preclude accurate tractography. Two subregions of the cingulate cortex were selected, the MCC and pregenual ACC (pgACC), because they exhibit different cytoarchitecture, anatomical connectivity, and functionality  [36] . Prior to probabilistic tractography, all defined seeds and targets were converted from standard to individual subject space.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22535907">PMID:22535907</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For the main experiment, single-subject parameter maps were imported into SPM2's analysis of variance (ANOVA) model to evaluate group statistics (random effects) for the following contrasts: 25 Hz movies with ordered frame  >  static face, increase proportional to frame rate (each condition was weighted by the frame rate: 1, 5, 12.5, and 25; ordered and scrambled conditions were weighted similarly, and the contrast was mean-centered), all ordered  >  all frame-scrambled stimuli (averaging over frame rates), and the interaction between frame rate and frame order testing for regions with a greater scrambling effect at higher frame rates. SPM2 uses the Greenhouse – Geisser correction for nonsphericity in the data. The results were thresholded so that all clusters survived multiple corrections across the whole brain at  P <  0.05 and all voxels therein survived an uncorrected threshold of  P <  0.001 ( Friston et al. 1994 ). Results are shown in Figure   3 , rendered on an inflated template brain from the Freesurfer toolbox ( http://surfer.nmr.mgh.harvard.edu ) using the spm_surfrend toolbox ( http://spmsurfrend.sourceforge.net ) and displayed using the Neurolens software ( http://www.neurolens.org ).\n Figure   2. Participants' ratings on the fluidity (left) and meaning (right) of the face motion as a function of the frame rate of the video stimuli (3, 4, 5, 6, 7, 9, 12.5, and 25 Hz) and the frame order (ordered or scrambled). Error bars represent standard errors of the mean across participants.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22493571">PMID:22493571</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Morphometric techniques that rely on tissue segmentation, like VBM, may not be optimal for assessing subcortical structures containing multiple types of tissue (i.e., both white and gray matter) (Ashburner and Friston,  2000 ). Therefore, additional volumetric analyses of subcortical structures were performed using Freesurfer software version 1.313.2.6 ( www.surfer.nmr.mgh.harvard.edu ). Images were first corrected for intensity bias and normalized into MNI space. Subcortical structures in these images were identified with an automated procedure that estimates the probability of structure-classification based on prior templates in which those structures were manually identified (Fischl et al.,  2002 ). Subcortical structures identified using this method include the thalamus, caudate, putamen, globus pallidus, nucleus accumbens, amygdala, and hippocampus. Once segmented, the total volume of these structures was calculated in each hemisphere. Because results were similar in each hemisphere, volumes for both hemispheres were summed for statistical analyses reported here. Between-groups differences in total volume were assessed for each structure with Student's  t -tests assuming unequal variance, using the percentage of total subcortical volume as the dependent measure. Because no tests survived a Bonferroni-correction for the number of  t -tests performed (7), uncorrected  p -values are reported.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22479566">PMID:22479566</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu ) was used to perform gray and white matter segmentation and a mid-gray cortical surface extraction. This cortical surface had 20,484 isotropically spaced vertices and was used both as a source constraint and for defining the visual areas. The FreeSurfer package extracts both gray/white and gray/cerebrospinal fluid (CSF) boundaries, but these surfaces can have different surface orientations. In particular, the gray/white boundary has sharp gyri (the curvature changes rapidly) and smooth sulci (slowly changing surface curvature), while the gray/CSF boundary is the inverse, with smooth gyri and sharp sulci. In order to avoid these discontinuities, we generated a surface partway between these two boundaries that has gyri and sulci with approximately equal curvature.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22479458">PMID:22479458</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess whether sex differences in areas with high densities of TES receptors and regions known to be sexually dimorphic during adolescence or adulthood were driven by TES-related change during puberty, we tested for interactions between sex and the effects of TES on gray matter thickness, independent of age using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). We predicted either a less extensive thinning, or even increases in cortical thickness would be associated with increased levels of TES in boys. In girls, conversely, we predicted that more pronounced cortical thinning would generally be associated with increased TES, given findings in longitudinal studies showing a delay in brain maturation (e.g. thinning of frontal cortices) in boys relative to girls  [25] ,  [35] , and given our previous results of global gray matter volume reductions being more prominent in girls than boys  [48] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22473874">PMID:22473874</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Analyses were performed with an automatic brain segmentation tool for surface-based cortical thickness ( http://surfer.nmr.mgh.harvard.edu ).  T 1 -weighted images underwent corrections for intensity inhomogeneity, skull strip and registration into Talairach space followed by segmentation into grey matter, white matter and various brain areas. Regional differences of cortical thickness between patient groups were investigated using Qdec (FreeSurfer for multiple comparisons and voxel-based morphometry). Statistics relied on  p   ?  0.05 (false discovery rate corrected for multiple comparisons). Visualization employed an inflated pial surface model.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22470360">PMID:22470360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22470336">PMID:22470336</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>4 http://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/MghFormat</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22393318">PMID:22393318</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22373370">PMID:22373370</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI-based quantification of cortical thickness was performed using the Freesurfer (v. 4.05) software package  http://surfer.nmr.mgh.harvard.edu . This method has already been described in detail [ 16 , 17 ]. Images were first corrected for intensity of non-uniformity and registered via affine transformation (12 parameters) to Montreal Neurological Institute (MNI) space [ 18 ]. Then, images underwent a further intensity normalization using a different automated algorithm and were automatically skull stripped [ 16 ]. Next, the entire cortex was visually inspected prior to analysis. The data from 70 subjects were deemed to require manual correction, which included: a) setting intensity normalization control points where brain matter was erroneously skull stripped, b) adjusting watershed parameters of the skull strip, and c) visual inspecting and correcting of the automatic subcortical segmentation. All processes (i.e. skull stripping and segmentation) were inspected by an expert neuroradiologist who was blinded to the aim of the study.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22366791">PMID:22366791</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements were made using FreeSurfer v5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) following previously described methods ( Dale  et al. , 1999 ;  Fischl and Dale, 2000 ). A surface-based Gaussian smoothing kernel of 20 ? mm full width at half-maximum was applied to reduce local variations in cortical thickness. Skull stripping used a locally generated brain mask, and FreeSurfer ventricular segmentations were added to the white matter mask to improve cortical segmentation. Between-group variations in cortical thickness were assessed using a vertex-by-vertex general linear model performed with SurfStat ( http://www.math.mcgill.ca/keith/surfstat/ ). Regional differences in grey matter between control and disease groups were assessed using the same covariates as the voxel-based morphometry analysis. A significance threshold of  P ? < ? 0.05 with false discovery rate correction was applied. Where this threshold was not met, percentage difference maps were used to illustrate trends in atrophy patterns.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22363554">PMID:22363554</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to help understand the basis of any group differences in cortical gyrification or sulcal spans, we also calculated a measure of cortical thickness. This was done using FreeSurfer 3.0.5 ( http://surfer.nmr.mgh.harvard.edu/ ) in a process that involved automated Talairach transformation, segmentation of the subcortical WM and deep GM volumetric structures  [43] , intensity normalization, tessellation of the GM/WM boundary, automated correction of topology defects, surface deformation to form the GM/WM boundary and GM/CSF boundary  [44] . Cortical thickness was calculated at each vertex  [43] , and averaged over the whole cortex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22355403">PMID:22355403</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High resolution 3D T1-weighted MRI scans were obtained on a 3T Philips Achieva scanner housed at the Vanderbilt University Institute of Imaging Science (Nashville, TN) with the following parameters: field of view ? = ? 256 × 256 mm 2 ; in plane voxel resolution ? = ? 1 × 1 mm 2 ; TR ? = ? 8.9 ms; TE ? = ? 4.6 ms; flip angle ? = ? 8 ° ; slice thickness ? = ? 1 mm and 170 slices with no slice gap. TD control subjects and caregivers of individuals with WS gave informed consent, while participants with WS gave assent. The study was approved by the Vanderbilt University Institutional Review Board. We used an automated, non-biased atlas-based Bayesian segmentation procedure, applied in Freesurfer v.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ), to derive quantitative estimates of brain structure and to label cortical and subcortical tissue classes  [37] ,  [38] ,  [39] . Freesurfer processing for volumetric T1-weighted images included: motion correction, brain extraction and removal of non-brain tissue using a hybrid watershed/surface deformation procedure  [40] ; automated spatial transformation and WM segmentation of subcortical volumetric structures  [41] ; intensity normalization, tessellation of GM/WM boundary and automated topology correction  [42] ; and surface deformation following intensity gradients to optimally place GM/WM and GM/CSF borders at the location where the greatest shift in intensity defines the transition to the other tissue class  [37] . Image outputs from each stage of Freesurfer processing were visually inspected and edited by an experienced imaging analyst (S.M.).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22347490">PMID:22347490</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to minimize the contribution of brain morphology in estimating the similarity of activation maps in twin pairs, a cortex-based inter-individual registration technique was used by incorporating the FreeSurfer 4.5 (Martinos Center for Biomedical Imaging,  http://surfer.nmr.mgh.harvard.edu ) automated reconstruction stream. First, each participant's T1 anatomical image was coregistered with the mean functional image. Then, this image underwent a series of reconstruction streams in FreeSurfer, which resulted in the identification of gray/white matter boundaries and gyral/sulcal folding patterns. Inter-individual registration was performed using this surface-based atlas by mapping individual cortical folding patterns to the FreeSurfer average curvature map. This procedure allows direct alignment of the anatomy instead of image intensities. The resulting surface map consisted of 163,842 vertices on each hemisphere.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22303478">PMID:22303478</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The first step was to perform cortical segmentation and labelling of the left BA4. In order to take into account variations in cortical folding across subjects, cortical segmentation of the T1 sequence was carried out in a standard spherical surface space, performed with the Freesurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ). The cortical segmentation was visually inspected for each subject by overlapping the grey matter (GM) - white mater (WM) boundary over the T1 image. No large misclassification of white matter or cerebral spinal fluid (CSF) voxels as grey matter voxels was found in any of the individual T1 images. After segmentation completion, Freesurfer generates surface labels for some cytoarchitectonic brain regions including BA4a and BA4p. The generated surface labels left-BA4a and left-BA4p were merged into a single label BA4. Afterwards, the surface label BA4 was transformed back to the original T1 space, visually inspected and manually corrected, if necessary ( figure 1 ). For visualization purposes, we show the approximate location of the tDCS electrode over left BA4 in  figure S1 . The next step was pre-processing of the functional datasets.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22281671">PMID:22281671</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the MNE software package (Version 2.6;  http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/sofMNE.php ) in combination with FreeSurfer (Version 4.3.0;  http://surfer.nmr.mgh.harvard.edu/ ) in order to apply minimum norm source estimation to our combined EEG and MEG data. The noise covariance matrices for each data set were computed for baseline intervals of 200   ms duration before the experimental stimuli. In the standard MNE procedure, it is used to transform the measurements into signal-to-noise ratios (SNRs), such that data from different sensor types (magnetometers, gradiometers and electrodes) can be combined for source estimation. The default regularization procedure implemented in MNE was used. MEG sensor configurations and MRI images were coregistered based on the matching of about 50 – 100 digitized locations on the scalp surface with the reconstructed scalp surface from the FreeSurfer software (see below).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22216343">PMID:22216343</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each scan was processed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/  ) using the volume and surface pipeline  [41] ,  [42] . Starting from the segmentation of the white matter and the tessellation of the grey/white matter boundary, an initial surface was obtained after automated topological correction. This surface was used as the initial shape for the deformable model that was used to reconstruct the pial surface. When all of the surfaces had been reconstructed, the cortical thickness was computed. The thickness was measured in the native space of each subject. The thickness was defined at each point on the pial surface (as well as its counterpart on the grey/white matter surface because of the one-to-one correspondence) as the mean of the two shortest distances  [43]  ; one was from the point on the pial surface to the grey/white surface, and the other was from the point on the grey/white matter surface to the pial surface. To compare cortical thicknesses point by point and to visualize the statistical results, the establishment of point correspondence across subjects in a standard surface-based coordinate system was required. Surface-based registration was used to build an average template, and all of the individual reconstructed cortical surfaces were aligned to this template  [44] . Then, the cortical thickness data were resampled for each subject. Prior to statistical analysis, a heat kernel with a 30-mm width was used to smooth the cortical thickness maps to increase the signal-to-noise ratio and to improve the ability to detect morphometric variations  [45] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22173294">PMID:22173294</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The major cortical and subcortical structures were initially parcellated using the Freesurfer recon-all pipeline ( Fischl et al., 2004 )( http://surfer.nmr.mgh.harvard.edu/ ). This produced 75 cortical regions per hemisphere that were divided according to the Destrieux 2009 atlas ( Destrieux et al., 2010 ). Using this pipeline, six subcortical regions per hemisphere were also defined (nucleus accumbens, caudate, putamen, pallidum, thalamus, and amygdala). The pallidal regions were then subdivided using ITK-SNAP ( http://www.itksnap.org ) ( Yushkevich et al., 2006 )  “ draw over label ”  function, into the internal (GPi) and external globus pallidus (GPe) using the medial medullary lamina as the boundary between the two. This was clearly demarcated on the MT images. The substantia nigra and red nucleus were defined by generating population tissue probability maps for six tissue classes using a novel segmentation algorithm developed in-house. For this, the group MT and R2* images were normalised using DARTEL in SPM ( Ashburner, 2007 ) to MNI space at 1   mm 3  resolution and the corresponding Jacobian determinants calculated. A modified version of a mixture of Gaussians model was then fitted to the collection of spatially normalised images, accounting for regional expansion or contraction by incorporating the Jacobian determinants into the computations. Because the images were quantitative, the intensity distributions of each tissue class were assumed to be identical for all subjects and modelled as a bivariate Gaussian. In addition, the model assumed that at any location in all of the spatially normalised images, there were the same prior probabilities in observing the various tissue types. These prior probabilities were estimated from the model and served as population tissue probability maps. For each region of interest (ROI), the regional probability maps covering our areas of interest were extracted using previously defined spatial priors from n   =   10 subjects which were dilated using 5   mm Gaussian smoothing and binarised at a threshold of 0.01. For each subject, the labels were assigned based on the maximum joint probability between the population spatial tissue probability maps, and individually calculated voxel intensity probability density functions (for white matter and ROI).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22136635">PMID:22136635</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Three of B.W.'s MRI studies of the brain were reviewed in detail and form the basis for our description. Two 1.5 Tesla (T) MRI examinations were done for clinical indications at ages four and eleven years, and a 3.0T MRI scan was done at age thirteen years for research purposes. The main finding was persistent and stable increased T2-signal intensity in the subcortical white matter of the left gyrus rectus in the vmPFC. This abnormal signal was best seen on thin section coronal T2-weighted and FLAIR sequences, and did not enhance following contrast administration (Figure  1A ). Linear extension and tapering of the T2-hyperintense abnormality towards the frontal horn of the left lateral ventricle presumably reflects a radial neuronal migration line. Associated focal abnormal thickening of the cortex and blurring of the gray-white matter junction along the gyrus rectus were also evident (Figure  1B ). These findings were stable between the patient's three available MRI studies. This constellation of MRI findings was thought to be compatible with Taylor type focal cortical dysplasia (FCD) with balloon cells, though confirmation of this subtype of FCD was not confirmed on histological analysis (see below) [ 33 , 34 ]. No other abnormalities of the brain were identified. The region of affected cortex is displayed in Figure  1C  using thickness data generated from FreeSurfer software  http://surfer.nmr.mgh.harvard.edu/ . The lesion appears to involve a portion of Brodmann areas 11, 12, 25, and 32. See appendix for details regarding the generation of Figure  1C .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22021878">PMID:22021878</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation of the SPGR and MPRAGE images was performed by the FreeSurfer image analysis suite, V.4.0.2 ( http://surfer.nmr.mgh.harvard.edu ). FreeSurfer is fully automated and so a manually drawn region of interest (ROI) was not used, which eliminates any human bias. FreeSurfer divides the brain into the temporal lobe, frontal lobe, OFC, parietal lobe, occipital lobe and cingulate gyrus. In each of these areas FreeSurfer measures multiple subdivisions. The development of this software and the details of how the specific regions are identified are found in a paper by Desikan  et al. 27  The orbitofrontal ROI was selected a priori based on our very specific hypotheses and its importance in response inhibition and palatability of food (through olfaction) and as an important part of the reward system. Given our use of FreeSurfer, we felt that it was extremely important to be very targeted in our approach and variables selected for analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22016733">PMID:22016733</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21949507">PMID:21949507</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution images of the brain were acquired using a T1-weighted MP – RAGE sequence. There were 192 contiguous sagittal slices with the following scanning parameters: TR ? = ? 2530 ? ms, TI ? = ? 1200 ? ms, flip angle ? = ? 7 ° , FOV 256 ? mm ? × ? 256 ? mm, 256 ? × ? 256 matrix, isotropic voxel dimensions of 1.0 ? mm, 6 ? min 3 ? s acquisition time. Automated measurements of brain volumes were performed using FreeSurfer 4.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ). In the present work, we report total cerebral volume (TCV; excluding brain stem and cerebellum), hippocampal volume, and ventricular volume. All of these measures were corrected for estimated total intracranial volume (eTIV) before statistical analysis. 3D-FLAIR images obtained in the sagittal plane with 176 slices (TR ? = ? 6000 ? ms, TI ? = ? 2100 ? ms, TE ? = ? 414 ? ms, isotropic voxel dimensions of 1.0 ? mm, 7 ? min 44 ? s acquisition time) were used to measure the volume of white matter hyperintensities.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21940605">PMID:21940605</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For group analysis, MRI and SAM images for each condition and frequency band were spatially normalized to the MNI template brain with FMRIB's Linear Image Registration Tool (FLIRT) ( Jenkinson et al. 2002 ). Statistical analysis was performed by nonparametric permutation testing ( Nichols and Holmes 2002 ;  Singh et al. 2003 ), and a 5-mm 3  Gaussian smoothing kernel was applied to the variance maps to generate pseudo-t images. These images were created for each condition and frequency band and thresholded using the omnibus test statistic value at  P   <  0.05 (2-tailed) to correct for multiple comparisons. To statistically compare the active versus baseline power obtained from the SAM analysis across the three conditions, pseudo-t images of contrasts between conditions were created with cluster-based thresholding with an initial cluster mass threshold of t = 2.5. Only clusters significant at  P   <  0.05 (2-tailed) were accepted. Subsequently, the contrast images were masked using the statistical maps of each condition included in the contrast, so that significant effects in the contrast image could only be found for voxels that were significantly activated in one of the conditions. 3  Cortical surface meshes were extracted from an average template MRI with FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). SAM source-reconstruction images were then superimposed on these meshes and visualized with mri3dX ( https://www.jiscmail.ac.uk/lists/MRI3DX.html ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21936901">PMID:21936901</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Subcortical volumetric segmentation was performed with the Freesurfer image analysis software ( http://surfer.nmr.mgh.harvard.edu/ ). The initial processing steps included (i) Motion correction and averaging of the two volumetric T 1 -weighted MPRAGE images, (ii) Removal of non-brain tissue using a hybrid watershed/surface deformation procedure [ 82 ], (iii) Automated Talairach transformation, (iv) Segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles) [ 83 , 84 ], (v) Intensity normalization [ 85 ]. Subsequent to these processing steps, the volumes were labeled based on both subject-independent probabilistic atlas and subject-specific measured values [ 83 , 84 ]. These labels were then mapped into Talairach space to achieve point-to-point correspondence for all subjects. This method uses both intensity and continuity information from the entire 3-dimensional high resolution structural volume in segmentation. A univariate analysis of variance for each of the segmented volumes was performed separately using IBM SPSS 19.0 statistics package to assess the differences between the two groups of migraine patients while accounting for the differences in the cranium size [ 86 ] and age as additional regressors.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21931650">PMID:21931650</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI scans were visually inspected by a radiologist for structural abnormalities (FB). Subsequently, all T1 weighted scans were analysed using Freesurfer (version 4.2,  http://surfer.nmr.mgh.harvard.edu/fswiki ,  Figure S1 ). In summary, this analysis included the reconstruction and parcellation of the cortical sheet of each hemisphere into 34 regions, used for the measurement of cerebral and cerebellar grey and white matter volume, cortical surface area and cortical thickness. For each individual dataset grey and white matter tissue and cerebrospinal fluid were classified, after intensity normalization and  ‘ skull stripping ’ . Next, using the grey/white matter segmentation, a surface tessellation was generated for the boundary between grey and white matter and for the boundary between grey matter and cerebrospinal fluid, for each hemisphere separately. Subsequently, cerebral cortical thickness of each point along the cortical mantle was computed by measuring the distance between the white and grey matter surface reconstructions. Automated parcellation of each individual cortical hemispheric sheet and subcortical structures resulted in the automatic segmentation of the cerebral and cerebellar cortex and subcortical structures (covering left and right hippocampus, amygdala, caudate nucleus, globus pallidum, putamen and thalamus). To ensure accurate automated segmentation using Freesurfer, each segmented brain was visually assessed by an experienced rater (MPvdH). Average volume of each of these parcellated brain regions and cerebral cortical surface were computed. Finally, cerebral and cerebellar grey and white matter volume and total cerebral volume were computed as the summation of the brain parameters of these regions. These steps are similar to the methods of Martinussen et al., examining T1 images of SGA +  adolescents, to increase comparability of our findings in young SGA children with brain development at 15 years of age  [11] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21912680">PMID:21912680</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical and sub-cortical brain regions were selected by parcellating the cerebrum into distinct, anatomically separated brain regions on the basis of the T1-weighted image, using the well-validated Freesurfer suite (V4.5,  http://surfer.nmr.mgh.harvard.edu/ ). In short, this included the automatic segmentation of grey and white matter tissue, followed by parcellation of the segmented grey matter mask into distinct brain regions, based on a normalized template, parcellating the brain into a number of brain regions, including left and right caudate nucleus, globus pallidum, nucleus accumbens, thalamus, amygdala, hippocampus and 70 cortical brain regions. In total, 82 distinct brain regions were parcellated  [37] ,  [38] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21908872">PMID:21908872</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Forty-two patients [26 males, mean age at scan 60.0 (SD 8.8) years] had volumetric magnetic resonance brain imaging performed on a 1.5T GE Signa scanner (General Electric). Magnetic resonance brain images were also acquired for 35 age-matched cognitively normal control subjects [16 males, mean age at scan 63.7 (SD 7.0) years] . Whole brain and cerebral hemisphere volumes were calculated in all individuals as previously described ( Rohrer  et al.,  2010 e ), and an index of cerebral asymmetry was derived [(absolute hemispheric volume difference/whole brain volume  ×  100)]. Volumetric analysis of grey matter cortical and subcortical (hippocampus, amygdala, caudate, putamen and brainstem) areas was performed using the Freesurfer image analysis suite (version 4.0.3;  http://surfer.nmr.mgh.harvard.edu/ ), on a 64-bit Linux CentOS 4 Sun Grid Engine Cluster. These brain regions were selected on the basis they were likely  a priori  to be targeted by pathologies in the FTLD spectrum.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21894151">PMID:21894151</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional and structural MRI data were analyzed using Freesurfer and FS-fast (available at  http://surfer.nmr.mgh.harvard.edu ), utilizing previously described techniques 1 ,  21 . The two 3D MPRAGE structural scans from each subject were averaged, after motion correction, to create a single high signal-to-noise average volume. Functional data were motion corrected using AFNI ( http://afni.nimh.nih.gov/afni/index.shtml ) 22 ,  23  and spatially smoothed (fwhm=5mm) using a 3D Gaussian filter ( www.fmrib.ox.ac.uk/fsl ). The spatially smoothed, normalized, motion-corrected functional images were aligned to a 3D structural image created by motion correcting and averaging the high-resolution 3D sagittal images. As part of the alignment procedure, the raw functional data from each subject were visualized over the high-resolution 3D anatomical from that individual to ensure that the BOLD signal in the amygdala was not obscured by susceptibility artifact. Individual subject functional data were spatially normalized using an optimal linear transformation method 24  that maximizes the likelihood that anatomic structures of individual subjects will overlap with each other across subjects. It is based on a previously described group atlas that retains the most common anatomic features in the majority of subjects 25 - 28 . Talairach transformation using the Montreal Neurological Institute automated registration algorithm was also performed for comparison (available at  ftp://ftp.bic.mni.mcgill.ca/pub/mni_autoreg ) 29 , but a better registration between anatomical structures and the coordinates in the Talairach atlas 30  was obtained with the optimal linear transformation method 24 . To facilitate comparisons across studies, we report Talairach coordinates based on registration of the images from the optimal linear transformation with the Talairach atlas 30 . After spatial normalization, functional data were averaged for each subject and then across subjects. Paradigm files were constructed to allow the separate averaging of the images acquired during fixation blocks, the novel face presentations, and the familiar faces. A group statistical map was computed using a random-effects model for the contrast novel and familiar faces (i.e. collapsed across condition) vs. the fixation cross (software available at  http://surfer.nmr.mgh.harvard.edu/docs/index.html ). For this group average we examined the responses to faces collapsed across all subjects. This analytic strategy assesses the role of temperament in a manner that was unbiased with respect to between group differences, and avoids circularity in the data analysis 1 ,  21 . A functionally constrained ROI was used because different regions within the amygdala, possibly representing subnuclei respond differently to facial stimuli 31 - 34 . For example, Leonard 33  found no responses to faces in the lateral nucleus of the amygdala in primates. Therefore, a ROI based on anatomical constraints alone might include portions of the amygdala that do not respond similarly to faces. A 43 voxel ROI were identified in the right amygdala, our a priori region of interest, with a statistical threshold of 10 -10 . Sixty-one of the 135 subjects had no voxels in the left amygdala exceeding this high threshold. However, at a reduced threshold of 10 -6 , a 41 voxel region was identified in left amygdala. Labels derived from the coordinates of these ROI's were used to extract percent BOLD signal change from baseline (the fixation cross) during Novel or Familiar face presentations in the functional data of each subject ( “ ROI analysis ” ) (software at  http://surfer.nmr.mgh.harvard.edu/docs/index.html ). A repeated-measures general linear model (PROC GLM with LSMEANS/tdiff) (SAS v9.2), with infant phenotype (high-reactive, low-reactive) and gender (female, male) as the between-group factors and time block (familiarization 1,2,3,4) as within-group factors, was performed on functional data from the familiarization phase of the protocol ( Figure 1 ). A second repeated-measures GLM with infant phenotype (high-reactive, low-reactive) and gender (female, male) as the between-group factors and face type (novel, familiar), and time block (1,2,3,4) as within- group factors, was performed on functional data from the second phase of the protocol in which alternating Novel (Blocks N1-N4) and Familiar (Blocks F1-F4) faces were presented. Covariance matrices for the time blocks in each GLM were tested for type-H structure 35  with a sphericity test 36 ; if the assumption of H-type covariance was not met, the more conservative p value based on the Greenhouse-Geisser Epsilon adjustment was reported 37 . All t-tests were two-tailed. A repeated-measures ANOVA with left and right hemisphere as an additional between-group factor was used to test for laterality effects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21860791">PMID:21860791</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Measurement of cortical thickness, based on the perpendicular distance from the pial surface to the gray/white matter juncture, is a validated measure of cerebral atrophy. FreeSurfer [ 10 – 15 ] is an image analysis software package available in the public domain ( http://surfer.nmr.mgh.harvard.edu ) that provides automated global and regional measures of cortical thickness. FreeSurfer performs gyral-based cortical parcellation using an algorithm that incorporates probable locations of regions of interest and the potential interparticipant variance based on the sample used. The atlas-generated ROIs were highly accurate compared to manual ROIs using intraclass correlation and mean distance maps [ 11 ]. Cortical thickness measurements have been shown to be reliable across different MRI platforms [ 15 ], and correlations between cortical thickness and cognition were reliable across different scanner platforms and different field strengths [ 16 ]. FreeSurfer (version 4.5.0) provided average cortical thickness values for the left and right hemispheres by automatically calculating the average of the values at each vertex across the hemisphere. FreeSurfer's reconstruction was checked for all participants, and edits were made where needed. The values of the two hemispheres were averaged to provide a measure of TCortTh across the entire cortex for final analyses. Middle temporal thickness (MTempTh) was also measured because prior research has suggested that the middle temporal lobe is relatively resistant to aging [ 17 ] but sensitive to AD [ 18 ,  19 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21833280">PMID:21833280</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All acquisitions were performed with a Siemens 3-T Trio magnet using a 12 channel head coil and T-1 weighted 3D MPRAGE sequence: field of view 256 ? mm, TR 2000 ? ms, TE 3.38 ? ms, flip angle 8 ° , 1 ? mm slice thickness. High-resolution T1 MPRAGE volumes in DICOM format were anonymized, imported into the FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) image analysis environment, and transformed into the MGZ format recognized by the analysis package. Standard semi-automated workflow procedures employing both surface-based and volume-based pipelines were used. The surface-based analysis involved registration with the Talairach and Tournoux atlas, intensity normalization, and skull stripping, followed by white matter labeling (Dale et al.,  1999 ). Cortical white matter surfaces were used in a deformation procedure that assigns gray and white matter borders by following intensity gradients to the position at which the maximum shift in intensity designated the transition to the other tissue class (Fischl and Dale,  2000 ). An automated registration procedure was used to label each voxel in an MRI volume based on probabilities estimated from a manually labeled training set (Fischl et al.,  2002 ). Standard FreeSurfer predefined region of interest (ROI) labels for moPFC and lateral orbital prefrontal cortex (loPFC) were used in the final statistical analysis. Robust correlations between the predefined ROIs and manual tracing procedures have been demonstrated in both of these regions (Desikan et al.,  2006 ). After the analyses were completed, volume data for ROIs in moPFC and loPFC were extracted from the imaging analysis environment and submitted to statistical testing.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21829681">PMID:21829681</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A – B.  MRI Acquisition: (A) high-resolution T1-weighted image and (B) diffusion images. The T1 is registered on the diffusion images. In every imaged voxel the Orientation Density Function (ODF) is extracted from the diffusion images.  C.  Whole brain tractography provides an estimate of axonal trajectories across the WM.  D.  Cortex partitioning into 83 gyral-based parcels using the Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ).  E.  Creation of the low-resolution structural connection matrix, representing the fiber density between every pair of the 83 parcels (upper left and lower right blocks: connections in the right, respectively left hemisphere; off-diagonal blocks: inter-hemispheric connections).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21814561">PMID:21814561</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Advances in MR image analysis algorithms have led to the development of automated parcellation tools which can segment the whole brain into anatomic regions and quantify the features of each region  [29] . The widely used FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ) was applied to each participant's pre-processed scan. Processing results using FreeSurfer Version 4.3.0 have been published on the website:  www.loni.ucla.edu/ADNI . Briefly, the processing included automated Talairach space transformation, intensity inhomogeneity correction, removal of non-brain tissues, intensity normalization, tissue segmentation (the subcortical structures, brain stem, cerebellum, and cerebral cortex)  [30] ,  [31] , automated correction of topology defects, surface deformation to form the gray/white matter boundary and gray matter/CSF boundary  [32] , and parcellation of the cerebral cortex  [33] . The atlas used, detailed in  [33] , included 34 cortical ROIs per hemisphere. For each ROI, the cortical thickness average (TA), standard deviation of thickness (TS), surface area (SA) and cortical volume (CV) were calculated as features. SA was calculated as the area of the surface layer equidistant between the gray/white matter and gray matter/CSF surfaces. CV at each vertex over the whole cortex was computed by the product of the SA and thickness at each surface vertex. Left and right hemisphere SA and total intracranial volume (ICV) were also included. For each subcortical structure, the subcortical volume (SV) was extracted. This yielded a total of 323 MRI features including 279 cortical and 44 subcortical features (see  Table S1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21811454">PMID:21811454</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21773026">PMID:21773026</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis was performed on a workstation with Linux Ubuntu 9.10, running SPM8 (Wellcome Department of Cognitive Neurology,  http://www.fil.ion.ucl.ac.uk/ ) on MATLAB 7.9 (The MathWorks, Natick, Mass, USA  http://www.mathworks.com/ ), FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ), and FSL (FMRIB Software Library,  http://www.fmrib.ox.ac.uk/fsl/ ) software ( Figure 1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21749494">PMID:21749494</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The Freesurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ) was used to segment the gray/white matter surface from the anatomical image ( Dale  et al. , 1999 ). Peak locations for all time windows were overlaid in a single figure on a corresponding anatomical  ‘ slice ’  that consisted of all points from the segmented surface falling within 2 mm of the mean peak location along the direction perpendicular to the  ‘ slice ’ . All figures are plotted in coordinates of the participants ’  anatomical MRI.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21712987">PMID:21712987</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the Freesurfer image analysis suite (version 4.5), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). This processing includes motion correction and averaging of multiple volumetric T1-weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure  [34] , automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus and amygdala)  [35] ,  [36] , intensity normalization  [37] , tessellation of the gray matter/white matter boundary, automated topology correction  [38] ,  [39] , and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid (CSF) borders at the location where the greatest shift in intensity defines the transition to the other tissue class  [19] ,  [40] ,  [41] . Once the cortical models are completed, a number of deformable procedures can be performed for further data processing and analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21629791">PMID:21629791</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the BrainVoyager QX analysis package, version 1.10.4.1250 (Brain Innovations, Maastricht, The Netherlands) for most functional data analyses as well as for the creation of inflated and flattened cortical representations. The freely available Freesurfer analysis package, version v4.5.0 ( http://surfer.nmr.mgh.harvard.edu/ ), was found to be better for retinotopic mapping data analysis on the reconstructed inflated brain, and the identified retinotopic areas were transferred to BrainVoyager using anatomical landmarks.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21584256">PMID:21584256</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21547716">PMID:21547716</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain MRIs were acquired using a GE Signa 1.5   T neuro-optimised MR system (General Electric, Milwaukee WI, USA).  “ Freesurfer ”  freeware ( http://surfer.nmr.mgh.harvard.edu/fswiki ) was used to derive models of the cortical sheet in each T1-weighted image and parcellate out the cortex into 33 regions. These well-validated [ 35 ,  36 ] and fully automated procedures have been extensively described elsewhere [ 37 ,  38 ], and we will only provide a brief description here. Firstly, a surface reconstruction was made for each scan [ 39 ]. This involved intensity normalisation, the removal of extra-cerebral tissues using a  “ skull stripping ”  algorithm, and image segmentation using a connected components algorithm. The output of this stage is a single filled white-matter volume for each cerebral hemisphere. A surface tessellation was then generated for each white- matter volume by fitting a deformable template. The grey matter/CSF surface was also modelled using a similar process. Given explicit models for the white/grey and grey/CSF surfaces, the measure of absolute cortical thickness at any given point on the white/grey matter surface is then taken to be the shortest distance between that point and the grey/CSF surface. This measurement is made at approximately 150,000 points across each hemisphere for each scan.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21437232">PMID:21437232</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>a  .w  file which contains the location of all dipoles estimated at all time points and can be visualized in FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) [ 10 ,  11 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21423423">PMID:21423423</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). This processing includes motion correction, removal of non-brain tissue using a hybrid watershed/surface deformation procedure (Segonne et al.,  2004 ), automated Talairach transformation, and segmentation of the sub-cortical white matter and deep gray matter volumetric structures (Fischl et al.,  2002 ,  2004 ). For the purpose of the present study only a subset of relevant segmented structures were selected for analysis and included the hippocampus, amygdala, thalamus, caudate, putamen, pallidum, ventricles, cerebellum, and total gray and white matter for each hemisphere (see Figure  1 ). In addition, to assess the role of differences in left and right brain structures an asymmetry index was computed for each left-right pairs using the equation: [(L ? R)/(L+R)] × 100 which produces an index ranging from  ? 100 (larger right volume) to +100 (larger left volume). The scans of nineteen participants were excluded from the sample due to poor scan quality, low signal-to-noise ratio, or movement artefacts which did not allow for normal processing with the standard Freesurfer pipeline leaving 398 participants for analysis. Each segmented volume was inspected slice by slice and reprocessed with additional parameters if errors were detected.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21373360">PMID:21373360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21364772">PMID:21364772</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess group differences in striatal, cerebral, and cerebellar volume, automated cortical reconstruction and volumetric segmentation of T1-weighted images was performed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ), which is a vertex-based approach. Processing included removal of non-brain tissue using a hybrid watershed/surface deformation procedure  [ 31 ]  and transformation to Talairach space. This was followed by segmentation of subcortical white- and deep gray-matter volumetric structures  [ 32 ] , intensity normalization  [ 33 ] , tessellation of the gray-white matter boundary, automated topology correction  [ 34 ] ,  [ 35 ] , and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class  [ 34 ] . Transformation and segmentations were manually verified. This approach provides anatomically accurate renderings of regional volumes, without potential rater bias  [ 32 ] . To account for differences in head size, volumes were divided by estimated total intracranial volume (eTIV).  Table 1  shows that striatum and cerebral/cerebellar gray-matter volumes did not differ between groups (p values  > .05).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21359177">PMID:21359177</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each scan was processed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) using the volume and surface pipeline  [ 50 ] ,  [ 51 ] . Starting from the segmentation of the white matter and the tessellation of the grey/white matter boundary, an initial surface was obtained after automated topological correction. This surface was used as the initial shape for the deformable model that was used to reconstruct the pial surface. When all of the surfaces had been reconstructed, the cortical thickness was computed. The thickness was measured in the native space of each subject. The thickness was defined at each point on the pial surface (as well as its counterpart on the grey/white matter surface because of the one-to-one correspondence) as the mean of the two shortest distances  [ 52 ] ; one was from the point on the pial surface to the grey/white surface, and the other was from the point on the grey/white matter surface to the pial surface. To compare cortical thicknesses point by point and to visualise the statistical results, the establishment of point correspondence across subjects in a standard surface-based coordinate system was required. Surface-based registration was used to build an average template, and all of the individual reconstructed cortical surfaces were aligned to this template  [ 53 ] . Then, the cortical thickness data were resampled for each subject. Prior to statistical analysis, a heat kernel with a 30-mm width was used to smooth the cortical thickness maps to increase the signal-to-noise ratio and to improve the ability to detect morphometric variations  [ 54 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21186358">PMID:21186358</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess amygdala (and, as a control region, hippocampal) volume, we performed quantitative morphometric analysis of T1 – weighted MRI data using an automated segmentation and probabilistic region – of – interest (ROI) labeling technique (FreeSurfer,  http://surfer.nmr.mgh.harvard.edu ). For methodological details, see  Supplementary Methods online . To adjust for differences in head size, amygdala and hippocampal volumes were divided by total intracranial volume as performed previously 10 ,  11 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21152081">PMID:21152081</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing and univariate analyses of the fMRI data were conducted with the Freesurfer functional analysis stream (FS-FAST,  http://surfer.nmr.mgh.harvard.edu ). Preprocessing of the fMRI data involved motion correction and grand-mean intensity normalization.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21124764">PMID:21124764</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All data processing was performed using Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ) and FSL ( http://www.fmrib.ox.ac.uk/fsl ) processing streams. Detailed methods for preprocessing DTI data are described in Salat et al.  [ 63 ] . Briefly, each tensor volume from the DTI dataset was resampled to the T2 image to correct for remaining eddy current distortion, and to correct for head motion. The fractional anisotropy (FA) metric was derived from the diffusion tensor as previously described  [ 64 ] . All resulting maps were resampled to 1 mm 3  resolution. Images were inspected for residual motion and susceptibility artifacts; subjects with significant artifacts were excluded.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21060689">PMID:21060689</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measures were performed using the validated and freely available Freesurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ). First, for each individual dataset (both ALS and healthy controls), the anatomical T1-weighted scan was placed into standard space and grey and white matter were segmented. Secondly, for each individual dataset, cortical thickness at every small region of the cortical surfaces was determined by computing the distance between the computed white matter and grey matter surface reconstructions  [ 48 ] . Thirdly, a group average anatomical image and surface rendering were constructed by normalising all anatomical images to standard space using spherical normalisation. All individual datasets (both of patients with ALS and healthy controls) were normalised to the computed group average anatomical surface, allowing for group comparison between patients with ALS and group matched healthy controls at each small sliver of cortical surface (vertex), covering whole brain.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21048942">PMID:21048942</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The preprocessed images from the retinotopic mapping runs were entered into a general linear model specific to each participant with two regressors of interest corresponding to the vertical and horizontal stimulus conditions. Blocks were convolved with a canonical hemodynamic response function to generate regressors. From the anatomical images we reconstructed, inflated and flattened  [ 23 ] ,  [ 24 ]  the grey-white matter boundary as a surface mesh for each cortical hemisphere using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/fswiki ). Linear contrasts between the two regressors were plotted on this surface. The boundaries of visual areas V1-3 were delineated by the activations corresponding to stimulation of the vertical and horizontal meridians. The surface vertices belonging to each visual area were projected back into volume space and the grey matter voxels falling in the space between these vertices on the grey-white matter surface and the pial surface were saved as binary masks.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21048895">PMID:21048895</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20954428">PMID:20954428</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Acquisition and post-processing techniques have also continued to advance and it is now possible to segment the brain, automatically, into gray matter, white matter, and CSF, as well as to delineate small brain regions of interest. The segmentation of brain into tissue classes seems a trivial task today, but it required more than 15 years of computer vision research to make this a reality. We tend to take for granted many of the postprocessing tools that we have available today, including software packages that enable registration, segmentation, tractography, and region of interest delineations, eg, Slicer (http://www.slicer.org), Brain Voyager (http://www.brainvoyager.com/), Brains2 (http://www.psychiatry.uiowa.edu/mhcrc/IPLpages/BRAI NS.htm), SPM (http://www.fil.ion.ucl.ac.uk/spm/), and FreeSurfer (http://surfer.nmr.mgh.harvard.edu/). These tools, in fact, were many years in the making and have significantly improved our ability to segment the brain and to investigate more precisely brain abnormalities in schizophrenia and other disorders, compared with healthy controls. Furthermore, the fact that these tools are more automated means that a larger number of subjects can be evaluated in a shorter amount of time than was possible using only manually driven methods to characterize brain regions of interest. These advances also make it possible to register brains at follow-up to baseline, in order to compare differences over time, and we are now able to use multiple imaging techniques in the same subject, which maximizes our opportunity to combine information from functional and structural measures of the brain.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20942940">PMID:20942940</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The computerised neuropsychological test battery CANTAB (Cambridge Cognition Ltd.) are employed for a precise estimation of drug effects on accuracy and reaction times during the above mentioned tasks probing executive function, attention and memory. The effects of Epo on the neural basis of executive function, memory and emotion processing as well as potential structural changes are investigated using the in vivo non-invasive techniques of fMRI, structural T1 weighted MRI, DWI, and ASL-based measurements of cerebral blood flow. Imaging data will be collected using a Siemens Trio scanner operating at 3.0 T, at the Danish Research Centre for Magnetic Resonance (DRCMR), Copenhagen University Hospital, Hvidovre. fMRI data will pre-processed and analysed using FEAT (FMRIB Expert Analysis Tool) version 5.43, part of FSL (FMRIB Software Library version 3.3)  http://www.fmrib.ox.ac.uk/fsl . Structural T1-weighted MRI data is analysed with FAST (FMRIB's Automated Segmentation Tool), FIRST (FMRIB's Integrated Registration and Segmentation Tool)  http://www.fmrib.ox.ac.uk/fsl  and FreeSurfer  http://surfer.nmr.mgh.harvard.edu/ . DWI data is analysed with FDT (FMRIB's Diffusion Toolbox;  http://www.fmrib.ox.ac.uk/fsl/fdt/ ). Finally analysis of ASL data is performed with SPM software  http://www.fil.ion.ucl.ac.uk/spm/software/spm8/  and FABBER  http://www.fmrib.ox.ac.uk/fsl/fabber .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20886094">PMID:20886094</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI scans were processed, with little to no manual intervention using the FreeSurfer software package (version 4.5), freely available at  http://surfer.nmr.mgh.harvard.edu . A single MPRAGE MRI acquisition for each participant was normalized for intensity inhomogeneities to create an image volume with high contrast-to-noise  [ 34 ] . The intensity-normalized volume was used to automatically locate the gray/white matter boundary (white matter surface)  [ 35 ]  and in turn, the gray/CSF boundary (gray matter surface)  [ 36 ] . Cortical thickness measurements were then obtained by calculating the distance between these two boundaries across the entire cortical mantle of each hemisphere  [ 36 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20884360">PMID:20884360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution structural T1-weighted MRI images were acquired in a 3   T Siemens Tim Trio scanner at the CBU using a 3D MPRAGE sequence, field-of-view 256   mm   ×   240   mm   ×   160   mm, 1   mm isotropic resolution, TR   =   2250   ms, TI   =   900   ms, TE   =   2.99   ms, flip angle 9 degrees. Structural MRI images were processed using automated segmentation algorithms of the FreeSurfer software (Version 4.3;  http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al., 1999; Fischl et al., 2001, 1999 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20699229">PMID:20699229</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data from the retinotopic mapping scans were used to define the borders between areas V1, V2, V3/Vp, and V3a/V4v in each hemisphere ( Engel et al. 1994 ;  Sereno et al. 1995 ) (see  Fig. 1 ). Data from the localizer scans were then used to define the segments of each of these areas that responded most strongly to stimuli in the location of the distractors. To define regions of interest (ROIs) in early visual cortex, the functional data were projected onto inflated representations of each participant's cortex. Cortical reconstruction and volumetric segmentation were performed with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). These ROIs were imported into SPM5, and the beta weights corresponding to stimuli appearing in the contralateral visual field for each of the previously described events were extracted from each ROI, allowing for analysis of the response to the different distractor types under the different load conditions.  Table 1  shows the mean location and volume for each of the imported ROIs, averaged across all subjects showing for which that region could be identified.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20660419">PMID:20660419</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To identify the boundaries of retinotopic visual cortices V1 – V3, standard retinotopic mapping procedures were used ( Sereno et al. 1995 ;  Teo et al. 1997 ;  Wandell et al. 2000 ). SPM5 was used to generate activation maps for the horizontal and vertical meridians. Mask volumes for each ROI were obtained by delineating the borders between visual areas using activation patterns from the meridian localizers. We followed standard definitions of V1 – V3 together with segmentation and cortical flattening using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Note that for all ROI analyses, only 11 (of a total of 12) participants were included as 1 participant had moved between the main functional experiment and the localizer experiment such that their occipital pole was mislocated and lay outside the range of the main scanned volume.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20589235">PMID:20589235</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20580129">PMID:20580129</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We investigated changes in imaging patterns with severity using cortical reconstruction and thickness estimation methods with the Freesurfer image analysis suite ( surfer.nmr.mgh.harvard.edu/ ) as previously described ( Rohrer et al., 2009 ). We used performance on the Graded Naming Test ( McKenna and Warrington, 1980 , total number of items equals 30) (i.e., degree of anomia) as a measure of disease severity, splitting the group according to their score: group 1 (less severe: 9 patients) scored  >  0 (mean 7.7, standard deviation 9.2) and group 2 (more severe: 4 patients) were unable to score. One case (AD-PPA6) with greater right than left hemisphere atrophy was not included in this analysis; this atrophy profile might reflect either a different disease phenotype or reversed hemisphere language dominance, however inclusion of this case could potentially bias any group-level correlations between cortical thickness and disease severity.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20577631">PMID:20577631</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20522270">PMID:20522270</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>While the subject was lying comfortably in the scanner with eyes closed, head fixed with noise-protecting ear-pads, and asked not to fall asleep, we used an EPI gradient echo pulse sequence (TR / TE / FA = 2 s / 50 ms / 90 0 ); 64  ×  64 acquisition matrix, 25 slices, 256 time frames, with voxelsize = 3.75  ×  3.75  ×  5.5 mm 3  and time resolution of 2 s to obtain "resting state" data. In addition, the imaging protocol consisted of an alternating left and right hand finger-tapping fMRI paradigm, two anatomical 3D T1-weighted scans for brain segmentation with  FreeSurfer  ( http://surfer.nmr.mgh.harvard.edu ), and DTI acquisitions (TR / TE / FA = 7900 / 104.8 / 90; 25 b=1000, 5 b=0; 25 slices; voxelsize = 1.88  ×  1.88  ×  4.0 mm 3 ) for assessment of white matter integrity and fibertracking with  TrackVis  ( http://trackvis.org ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20487539">PMID:20487539</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Scan processing and statistical analysis was performed using the FreeSurfer image analysis suite, version 4. FreeSurfer is documented and freely available for download online  http://surfer.nmr.mgh.harvard.edu/ . All non-automated image processing was performed by a single, experienced image analyst. Technical details of the procedures are described in prior publications, including [ 40 - 44 ]. Briefly, the processing includes motion correction and averaging of multiple volumetric T1 weighted images. In 9 cases only one image series was used in subsequent analysis, either because one series was corrupted by head movement or artifacts, or because only one series was successfully retrieved from the scan. Motion correction was followed by removal of non-brain tissue using a hybrid watershed/surface deformation procedure [ 44 ], automated Talairach transformation, intensity normalization [ 45 ], tessellation of the grey mat ter white matter boundary, automated topology correction [ 43 , 46 ], and surface deformation following intensity gradients to optimally place the grey/white and grey/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class [ 40 , 47 , 48 ]. This method used both intensity and continuity information from the entire three dimensional MR volume to produce representations of cortical thickness, calculated as the closest distance from the grey/white boundary to the grey/CSF boundary at each vertex on the tessellated surface [ 48 ]. The maps produced were not restricted to the voxel resolution of the original data and thus are capable of detecting sub-millimeter differences between groups. Procedures for the measurement of cortical thickness have been validated against histological analysis [ 49 ] and manual measurements [ 50 , 51 ]. FreeSurfer morphometric procedures have been demonstrated to show good test-retest reliability across scanner manufacturers and across field strengths [ 52 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20385243">PMID:20385243</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>From the anatomical images we reconstructed and inflated the surface of each cortical hemisphere using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/fswiki ) ( Dale et al., 1999 ). Polar maps of the visual cortex were calculated using phase-encoded retinotopic mapping techniques ( Sereno et al., 1995 ) and retinotopic visual areas were delineated manually. The boundaries of V1 – V3 were delineated by identifying the representation of the vertical and horizontal meridians from the mirror reversals in the phase map, separating the ventral and dorsal subregions of these areas. V4 and V3A were defined as maps of a full visual hemifield, anterior to V3v and V3d, respectively. We also mapped regions of interest (ROI) for the activations in the element localiser in left dorsal V1 (lower-right element) and right ventral V1 (upper-left element). We identified the motion-selective V5/MT complex by the linear contrast between transparent motion stimuli and the static random dot stimuli. Further, we mapped a region of interest anterior to area V3A in the lateral occipital cortex that was activated preferentially by kinetic boundaries relative to the transparent motion. We refer to this region as KO ( ‘ kinetic occipital ’ ) ( Dupont et al., 1997 ), although this encompasses several retinotopic areas and parts of the lateral occipital complex ( Larsson and Heeger, 2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20375074">PMID:20375074</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each individual brain was inflated using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). Statistical analysis of the functional data was performed with FS-FAST. All functional images were motion corrected, spatially smoothed with a 5-mm Gaussian kernel, and normalized across sessions individually. Average signal intensity maps were calculated for each condition for each subject. Voxel-by-voxel statistical tests were conducted by computing contrasts based on a General Linear Model (GLM). For averaging across subjects, each subject ’ s functional and anatomical data were spatially normalized by using the spherical transformation ( Fischl et al. 1999 ). Data from both hemispheres were averaged together unless otherwise noted.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20352085">PMID:20352085</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MR examinations were performed on a 1.5T Siemens Sonata scanner using an 8-channel phased array head coil. Structural imaging was obtained using a T1 coronal gradient echo sequence  [ TE ? = ? 4.76 ms; TR ? = ? 12 ms; Voxel Size  ? = ? 1 × 1 × 1.5 mm; acquisition time  ? = ? 7:15 ] . Subjects' heads were stabilized with tape across the forehead and padding around the sides. For all scans, each T1 was reviewed for image quality. Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The methodology for FreeSurfer is described in full in several papers  [ 36 ] ,  [ 37 ] ,  [ 63 ] ,  [ 64 ] ,  [ 65 ] ,  [ 66 ] ,  [ 67 ] ,  [ 68 ] ,  [ 69 ] ,  [ 70 ] ,  [ 71 ] . For this paper, we are focused on the cortical thickness results and volumes of select subcortical structures. Procedures for the measurement of cortical thickness have been validated against histological analysis  [ 72 ]  and manual measurements  [ 73 ] ,  [ 74 ] . The results of the automatic segmentations were reviewed and any errors were corrected.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20215478">PMID:20215478</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Recently, evidence has been reported for extensive and clinically relevant GM damage in MS. 10 , 11  The calculation of the density (or volume) of a cortical region obtained with a voxel-based morphometry (VBM) approach is most commonly performed in clinical studies, and has been interpreted as reflecting cortical atrophy. 10  However, a reduction in GM volume can be due to thinning of the cortex, reduction in the cortical area, change of its folding, or to a combination of these processes. The FreeSurfer cortical surface-based methodology ( http://surfer.nmr.mgh.harvard.edu/ ) allows us to estimate all these GM measures within specific brain regions in an automated way, leading to a comprehensive  in vivo  assessment of the cortical GM damage. Among these cortical measures, thickness has been investigated in a few studies in MS, 12 – 14  while cortical surface area of each hemisphere of MS patients has been reported in one study only, 15  and cortical curvature has not been investigated in MS so far.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20056156">PMID:20056156</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Lead fields were computed using a single-shell head model ( Nolte 2003 ) based on an inner skull mesh derived by inverse-normalizing a canonical mesh to the subject's individual structural image ( Mattout et al., 2007 ). Coregistration between the MRI and MEG coordinate systems used 3 fiducial points: nasion, left and right pre-auricular points. Since the structural scan had been acquired in the clinical setting before the patient consented to take part in the study, these points were not marked with vitamin-E capsules as usually done. We, therefore, marked them by hand on head surfaces extracted from the structural image using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Subsequently, we checked these points against the original structural image and made slight correction where necessary. In order to further reduce possible coregistration error, we followed the initial rigid registration of the fiducials by an additional step, where the nasion fiducials from the two coordinate systems were brought into exact correspondence on the antero-posterior axis and the averages of the preauricular points were brought into correspondence on the superior – inferior axis. The rationale for this was that the marked nasion point was more reliable in the antero-posterior direction, as the nasion must lie on the surface of the nose, whereas the marking of pre-auricular points was more reliable in the superior – inferior direction due to the presence of clear anatomical landmarks on the ear.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20043859">PMID:20043859</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical surface was reconstructed from high-resolution T1 images using the fully automated analysis protocol implemented in FreeSurfer 4.0  http://surfer.nmr.mgh.harvard.edu . First, the cortical surface was automatically segmented from the anatomical images (Figure  2A-C ). Gyral anatomy was aligned to a standard spherical template using surface convexity and curvature measures (Figure  2D ) [ 20 , 49 , 50 ]. Identical procedures were used to analyze control data to permit statistical comparisons of homologous cortical regions between the patient and controls. The frontal, temporal, parietal and occipital cortices delineated by the surface parcellation provided by FreeSurfer [ 51 ] were used to analyze cortical tissue properties at the lobar level.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19876406">PMID:19876406</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19826494">PMID:19826494</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19787067">PMID:19787067</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Tourville and Guenther  [ 47 ]  have developed a landmark-based protocol for parcellation that builds upon a system used at the Center for Morphometric Analysis (CMA)  [ 18 ]  and that is focused in particular on cortical areas involved in speech processing. This system, therefore, incorporates a large number of auditory, motor, and premotor cortical areas. The FreeSurfer software program ( http://surfer.nmr.mgh.harvard.edu ) was trained to perform automatic cortical parcellation  [ 22 ]  based on a training set of manually labeled scans using this protocol. The test brain was then parcellated using FreeSurfer, and manually edited by an expert anatomist to correct any apparent gross mislabelings. Freesurfer's default subcortical parcellation  [ 48 ]  was also performed, and the sum results of both cortical and subcortical labeling were projected back into the original volume-space, resulting in a parcellation ( T & G ) that could be readily compared to those described above.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19707517">PMID:19707517</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19679189">PMID:19679189</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis was performed using the MIDAS software package ( Freeborough et al., 1997 ). Scans were outlined using a rapid semi-automated technique which involves interactive selection of thresholds, followed by a series of erosions and dilations. This yields a region which separates brain from surrounding CSF, skull and dura. Scans and associated brain regions were subsequently transformed into standard space by registration to the Montreal Neurological Institute (MNI) Template ( Mazziotta et al., 1995 ). The left and right hemispheric regions were defined using the MNI average brain which was split by dividing the whole volume along a line coincident with the interhemispheric fissure. An intersection of each individual's brain region and the hemispheric regions defined on the MNI template was generated to provide a measure of brain volume in left and right hemispheres ( Chan et al., 2001; Beck et al., 2008 ). Left/right volume ratios were subsequently calculated. Volumetric analysis of specific subcortical structures (hippocampus, amygdala, caudate and brainstem) was performed using the Freesurfer image analysis suite version 4.0.3 ( http://surfer.nmr.mgh.harvard.edu/ ) ( Fischl et al., 2002 ), on a 64-bit Linux CentOS 4 Sun Grid Engine Cluster. Initially, the four groups (SD, PNFA, LPA and controls) were compared statistically by looking at the two-tailed contrasts between the group means using a linear regression model (STATA8 © , Stata Corp, College Station, TX). Subgroup analyses with SD, PNFA,  GRN -PPA only, AD-PPA only and controls were also performed.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19636426">PMID:19636426</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition to the whole-brain analyses used to evaluate the main effects of stimulus type, region of interest (ROI) analyses were used to evaluate responses in less-powered comparisons, namely the effect of eating conditions, in the insula, hypothalamus, and inferior visual cortex. ROIs were defined anatomically and used for small volume corrections (SVC). Two ROI, the anterior short insular gyrus and fusiform gyrus, were derived from probabilistic labeling of the SPM single subject average image using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). The probabilistic labeling was based on the Destrieux atlas of the Freesurfer package  [ 23 ] , and these two ROI included only the gray matter portion of the gyri. The ROI for the hypothalamus was generated from the Wake Forest University PickAtlas  [ 24 ] . All ROIs were smoothed with a 4 mm FWHM kernel and intensity filtered at 0.5 to minimize artifactual increases in statistical thresholds due to large surface areas relative to volumes  [ 25 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19592570">PMID:19592570</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The observation of an interaction between lexical information processing and expertise deserves particular consideration: The significant interactions  suprasegmental   ×   expertise  uncovered a left temporal cluster which is located on the transition strip between the lower bank of the posterior STS and the superior bank of the MTG [( ? 54,  ? 37, 6),  F  = 9.28] — as this cluster is located considerably inside the sulcus we henceforth use the term STS. Additionally we found a more anterior located cluster on the MTG [( ? 51,  ? 39,  ? 6),  F  = 9.28]; both clusters are characterized by the same effect size of interaction ( Fig. 4 A ). The precise anatomical location was evaluated by applying the  Harvard-Oxford cortical structural atlas  (available at:  http://www.cma.mgh.harvard.edu/ ) and the  Destrieux-Atlas  ( Fischl et al. 2004 ), which has been implemented in  FreeSurfer  software (available at:  http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19559801">PMID:19559801</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Estimation of cortical thickness based on T1-weighted images represents a viable methodological alternative to volumetric measurements for assessment of subtle cortical changes in the human brain. The initial image processing steps required to calculate cortical thickness typically involve segmentation of the images into grey matter, white matter and CSF, as with VBM. However, the subsequent steps may require substantially different processing in order to provide an absolute measure of thickness across the cortical surface. Typically cortical thickness measurements involve identification of the inner and outer cortical boundaries or surfaces. This may be achieved using image information and surface geometry to construct or fit a representation of the grey and white matter surfaces ( Fischl and Dale, 2000; Jones et al., 2000; MacDonald et al., 2000; Miller et al., 2000; Zeng et al., 1999 ). The thickness at each point on the grey matter surface is then given by a distance measure between corresponding points on the two surfaces. Investigating surface-based cortical thickness measures across different subjects requires methods to match corresponding anatomical regions of the cortical surfaces. For example, the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ , Center for Biomedical Imaging, Charlestown, MA) which has been widely used for measuring cortical thickness in MRI, averages across subjects using non-rigid high-dimensional spherical averaging to align cortical folding patterns ( Fischl et al., 1999 ). Other methods for measuring cortical thickness define the grey and white matter boundaries on the basis of voxel information and the thickness is based on the length of the trajectory from one boundary to another. A distinct difference between these methods and surface-based cortical thickness measures is that thickness is calculated for every volumetric point within the cortex rather than on the surface ( Hutton et al., 2008; Jones et al., 2000; Yezzi and Prince, 2003 ). Differences in cortical thickness have been identified in normal aging ( Salat et al., 2004 ), human intelligence ( Choi et al., 2008 ), cognitive performance ( Dickerson et al., 2008 ), and in neurological disorders including Gilles de la Tourette syndrome in children ( Sowell et al., 2008 ) and Huntington disease ( Rosas et al., 2008 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19460794">PMID:19460794</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI scans were processed using the FreeSurfer software package, freely available at  http://surfer.nmr.mgh.harvard.edu . Multiple MPRAGE MRI acquisitions for each participant were motion corrected, averaged and normalized for intensity inhomogeneities to create a single image volume with relatively high contrast to noise (Dale  et al .,  1999 ). This averaged volume was used to locate the grey/white matter boundary (white matter surface) and this, in turn, was then used to locate the grey/CSF boundary (grey matter surface) (Fischl  et al .,  1999a ;  2000 ). Cortical thickness measurements were then obtained by calculating the distance between the grey and the white matter surfaces at each point (per hemisphere) across the entire cortical mantle (Fischl  et al .,  2000 ). This cortical thickness measurement technique has been validated via histological (Rosas  et al .,  2002 ) as well as manual measurements (Salat  et al .,  2004 ; Dickerson  et al .,  2009 ). The reliability of the cortical thickness measures as well as the other image analysis procedures presented here has been demonstrated across different manufacturer types, scanner upgrades, varying contrast-to-noise ratio, and the number of MPRAGE MRI acquisitions used (Han  et al .,  2006 ; Fennema-Notestine  et al .,  2007 ; Jovicich  et al .,  2009 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19420007">PMID:19420007</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The structural  T 1 -weighted MRI images for each subject were brought into alignment (minctracc in MINC, Collins,  http://www.bic.mni.mcgill.ca ) then averaged. This produces a high-quality, low noise image. The cortical surface was then extracted from the intensity-normalized, segmented 3D image as a tessellated triangular mesh, then inflated to a sphere and registered to a spherical atlas (recon-all in FreeSurfer, Fischl,  http://surfer.nmr.mgh.harvard.edu ). Extraction of the cortical surface begins with automated alignment of the 3D brain image to the Talairach atlas ( Talairarch and Tournoux 1988 ) using affine transformations. The initial sphere, therefore, is a Talairach-normalized representation of cortical anatomy. The registered spherical meshes provide a 2D (spherical) coordinate system for comparing cortices across subjects at about 1-mm resolution (about 160 ? 000 nodes per hemisphere). To further simplify comparison between subjects and reduce the number of nodes to about 40 ? 000 per hemisphere, the meshes were resampled using standard numbering, uniformly on the sphere, at 2-mm resolution (MapIcosahedon in SUMA, Saad,  http://afni.nimh.nih.gov/afni/suma ). The new meshes provide a standard ordering for comparing cortices across subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19352444">PMID:19352444</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19289173">PMID:19289173</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess how closely activation from individuals matched the group results, their peak responses for words and objects were compared to the group results. For words, all 45 participants showed a peak response within ventral OTC with a  Z -score of at least 3.5, although the specific location varied considerably ( Table 2 ). The left panel of  Fig. 2 C illustrates the spatial distribution of peaks within ventral OTC. Individual subject peaks are shown as orange dots. Each peak has been projected onto a single brain that has been inflated to show not only the crests of the gyri (light grey) but also the depths of the sulci (dark grey) using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Note that the sharp demarcations between gyri and sulci do not accurately reflect the anatomical variability present in the group; instead the figure illustrates the spatial distribution of peaks relative to a single  “ average ”  brain. Consequently, the specific anatomical location of each peak was assessed relative to that individual's structural scan in standard space. The greatest consistency is in the medial-lateral direction, with the majority of peaks ( n   =   20) falling within the occipito-temporal sulcus. Another 18 were located on the crest of the posterior fusiform gyrus and 7 were in on the crest of the inferior temporal gyrus. In contrast, the largest variation was in the rostro-caudal direction while the variation in  z -axis is mostly due to the depth of the OTS. On average, the Euclidean distance from an individual subject's peak to the group peak was 15   mm ( ±   5   mm).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19205678">PMID:19205678</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Figure   2  (activations rendered on inflated brain) was created using the spm_surfrend toolbox ( http://spmsurfrend.sourceforge.net/ ) and displayed using Neurolens software ( http://www.neurolens.org ) on the inflated template brain from the Freesurfer toolbox ( http://surfer.nmr.mgh.harvard.edu ). Fig.   2 Results of the whole-brain ANOVA group statistics projected on the surface of an inflated standard structural scan.  a  Shows clusters responding more to static faces than static scrambled.  b  Shows clusters responding more to dynamic faces than dynamic scrambled.  c  Shows clusters responding more to dynamic faces than static faces.  d  Shows clusters with a significant interaction effect: (dynamic faces   >   dynamic scrambled)   >   (static faces   >   static scrambled). Insets in ( d ) show per cent signal change from fixation (mean and SEM over subjects) for static faces (SF), static scrambled (SS), dynamic faces (DF) and dynamic scrambled (DS) in left and right STS clusters ( left  and  right  insets, respectively). Maps are thresholded at  p   <   0.0001 uncorrected, but all activations survive whole-brain correction at  p   <   0.05.  Gradient bar  shows  T  values</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18978780">PMID:18978780</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Scanning was done on a 3T Siemens Trio scanner at the Athinoula A. Martinos Imaging Centre at the McGovern Institute for Brain Research at MIT. FMRI data analysis was conducted using FreeSurfer Functional Analysis STream (FS-FAST;  http://surfer.nmr.mgh.harvard.edu/ ) and region of interest analysis was conducted using FS-Fast region of interest toolbox (FROI;  http://froi.sourceforge.net/ ). Functional MRI runs were acquired using the standard 12-channel head matrix coil and a gradient-echo echo-planar sequence (TR=2s, TE=40ms, 1.4 * 1.4 * 2.0 mm + 20% spacing**maybe included matrix size and field of view**). The processing steps for both the localiser and experimental runs included motion correction and spatial smoothing with a 3mm kernel and linear trend removal. Note that only the localiser task was smoothed (correlation analysis was performed on unsmoothed data).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18958219">PMID:18958219</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Eventually, electrode contacts and experimental effects have been visualized on individual 3D rendering of isolated temporal cortices to facilitate the identification of auditory brain structures and the comparison of activated sites across subjects. The cortical surface was individually segmented by using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18632739">PMID:18632739</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Methods for our surface-based analyses and estimation of cortical thickness have been previously described in detail ( Dale et al. 1999 ;  Fischl, Sereno, and Dale 1999 ;  Fischl and Dale 2000 ;  Rosas et al. 2002 ;  Kuperberg et al. 2003 ;  Salat et al. 2004 ) and are described here only briefly. The Freesurfer software used to perform the analyses and visualization employed in this study, along with complete documentation, is freely available via the internet at  http://surfer.nmr.mgh.harvard.edu .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18550622">PMID:18550622</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Registration of FA and T1 volumes: FA volumes were registered to the high-resolution structural (T1) volumes for each participant using the T2 volume as an intermediary. This avoids the potential bias that might arise from using the FA volume in the registration, since FA is the variable of interest. In addition, the gray/white boundary in the T1 volume has better correspondence with the T2 volume than the FA volume, allowing for a more accurate registration. The T2 volume was taken from the  b  = 0 image in the DTI acquisition and was, therefore, in register with the FA volume. Each participant's averaged T2 volume was manually registered to the T1 volume with Tkregister2 ( http://surfer.nmr.mgh.harvard.edu ) using 9 df including translation, rotation and scaling. Manual registration was used to maximize anatomic agreement along the medial wall and because echo planar imaging susceptibility distortions in DTI can confound global registration procedures. The resulting spatial transformation was applied to the FA volume, thus bringing the FA and T1 volumes into register.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18079129">PMID:18079129</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The reconstructed histological volumes were used to generate surface models of the gray/white interface. This was accomplished in several steps. First, a set of  “ control points ”  were manually added to the body of the white matter to guide an intensity normalization step that resulted in the white matter across most of the volume being close to a prespecified value ( Dale et al. 1999 ). This volume was then thresholded and manually edited to separate white matter from other tissue classes. The resulting binary segmentation was used to generate topologically correct and geometrically accurate surface models of the cerebral cortex ( Dale et al. 1999 ;  Fischl, Sereno, Dale 1999 ;  Fischl et al. 2001 ) using a freely available suite of tools ( http://surfer.nmr.mgh.harvard.edu/fswiki ). An example of the results of this procedure together with the locations of the manually selected control points is given in  Figure 1 , which shows coronal (top), sagittal (middle), and axial (bottom) slices of a typical volume with the reconstructed gray/white surface shown in yellow. Note that small errors in surface positioning, which would be critical, for example, in a study of cortical thickness, are mostly irrelevant in this study in which we are more concerned with the large-scale geometry of the surface models. The 8 labeled BA maps (areas 2, 4a, 4p, 6, 44, 45, 17, and 18) were sampled onto surface models for each hemisphere, and errors in this sampling were manually corrected (e.g., when a label was erroneously assigned to both banks of a sulcus). A morphological  close  was then performed on each label. A close of a binary label is a  dilation , in which each point that is 0 and neighbors a point that is 1 is set to 1, followed by an  erosion , in which each point that is 1 and neighbors a 0 is set to 0. The close was used to remove small holes that arise due to sampling artifacts without distorting the boundary of each label.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24376684">PMID:24376684</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface models were created using Freesurfer version 4.5 [ 26 ] from the simulated T1 image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24273408">PMID:24273408</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain volumes for each gray and white matter region of interest were derived using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24260229">PMID:24260229</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer 5.10 software package was utilized for cortical reconstruction and volumetric segmentation (FreeSurfer v5.10,  http://surfer.nmr.mgh.harvard.edu/fswiki ). For processing, the original MRI data were first subjected to a series of preprocessing steps, including motion correction, T1-weighted image averaging, registration of the volume to Talairach space and stripping the skull with a deformable template model ( Figure 1A ). By encoding the shape of the corpus callosum and pons in the Talairach space and following the intensity gradients from the white matter to the cerebrospinal fluid, the white surface and the pial surface were generated for each hemisphere ( Figure 1B ). Once these surfaces were known, a cortical surface-based atlas was mapped to a sphere aligning the cortical folding patterns, which provided accurate matching of the morphologically homologous cortical locations across subjects. The average shortest distance between white and pial surfaces denoted the cortical thickness at each vertex of the cortex. Surface area was calculated by computing the area of every triangle in a standardized spherical surface tessellation. The local curvature was computed using the registration surface based on the folding patterns. The folding index over the whole cortical surface was measured using the method developed by Schaer. In the present study, the FreeSurfer pipeline was used to automatically generate the five basic cortical features. Each basic feature was divided into 68 components based on brain segments, which comprise a total of 340 cortical features for each subject ( Figure 1C ). The indexes of 340 cortical features are briefly presented in  Table 1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24260210">PMID:24260210</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Participants over age 70 years were recruited through newspaper articles, visits to senior centers and assisted living facilities, and word of mouth. Inclusion criteria included age 70 – 100 years and presenting as CH, having mild cognitive impairment (MCI), or having AD (defined below). Exclusion criteria included taking strong anticoagulants and contraindications for MRI or lumbar puncture. Consistent with the Uniform Data Set  [27] , the following data were obtained from each individual within one month of informed consent: structured clinical interview; complete prescription and over-the-counter medication and nutritional supplements history; and physical examination with a focus on neurological and cardiovascular systems. Participants self-scored physical and intellectual activity (separately) over the past year on a 1 to 4 scale, where 1 ? = ? minimal, 2 ? = ? moderate, 3 ? = ? active, and 4 ? = ? very active. Body mass index (BMI) was calculated from measured height and weight. A physician or nurse, blinded to diagnostic classification, measured diastolic and systolic blood pressures (mm Hg) with study participants seated and rested on two occasions during the workup, and the pressures were averaged. Laboratory workup to exclude conditions that might affect cognition included blood CBC, ANA, ANCA, phosphorus, vitamin B12, folate, Lyme antibody detection by EIA, HIV, syphilis, comprehensive metabolic panel, and thyroid function panel; and urinalysis screened for protein, blood, leukocytes, nitrite, glucose, ketone, pH, specific gravity, bilirubin and urobilinogen. Fasting blood glucose and lipids (mg/dl) were determined by routine methods immediately before CSF collection (below). MRI was performed on a 1.5 T GE scanner to exclude significant vascular or neoplastic disorders. To assess small vessel disease (SVD)  [28] , two qualified, blinded readers performed an analysis based on that recently proposed by Wardlaw et al  [28]  in which 6 items contribute to the MRI diagnosis of SVD. In the absence of an assessment for  “ cerebral microbleeds ”  (no susceptibility weighted imaging was performed), we investigated 5 contributing SVD features:  “ recent small subcortical infarcts ” ,  “ lacunes of presumed vascular origin ” ,  “ white matter hyperintensities of presumed vascular origin ”  (uncorrected for brain volume),  “ perivascular spaces ” , and  “ brain atrophy ” . We assigned 0 if absent or 1 if present for each feature, for a maximum score of 5. Hippocampal volumes relative to whole brain volume were derived using FreeSurfer software version 5.0 ( www.surfer.nmr.mgh.harvard.edu ). Proton magnetic resonance spectroscopy (MRS) was performed on posterior cingulate grey (PCG) and left parietal white matter (PROBE-P TE ? = ? 35 milliseconds, TR ? = ? 1.5 seconds, 8 mL voxel, 128 averages). Results of PCG-MRS were expressed as NAA/mI following observer independent spectral analysis using SAGE (GE Healthcare) and compared with an age-appropriate data base of normative results in  > 100 subjects [noting that N-acetyl aspartate (NAA), a putative neuronal biomarker, is reduced by dementing diseases, including MCI and AD, and that myo-inositol (mI), a likely biomarker of glia, is correspondingly increased  [29] – [31] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24204657">PMID:24204657</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Regional cortical thickness was calculated using custom developed software utilizing FreeSurfer segmentation data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24040196">PMID:24040196</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Of those subjects , 103 subjects were excluded from the FreeSurfer dataset ( see below ) for technical reasons , such as major hardware upgrades during the study ( at two sites ) , miscalibration of image resolution , excess movement , or failure of one or more automatic processing methods .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23967170">PMID:23967170</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Although the AAL template is widely used to define the nodes of brain networks [ 8 ] , [ 42 ] , many other templates have also been used , including the Brodmann atlas [ 9 ] , the ANIMAL ( automated nonlinear image matching and anatomical labeling ) atlas [ 83 ] , the LPBA40-atlas ( LONI Probabilistic Brain Atlas ) [ 16 ] , the Harvard – Oxford Atlas [ 16 ] , and the parcellation obtained using Freesurfer [ 45 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23936292">PMID:23936292</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Hence , we eroded the scalp layer by 3 single-voxel steps using Freesurfer 's mri_binarize tool , resulting in a mean scalp thickness of 6.9 ± 3.6 mm which is more in line with prior findings .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23922939">PMID:23922939</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1-weighted structural scans were used for skull-stripping, and the brain images were then co-registered into Talairach space  [34] . The CN was automatically segmented using Freesurfer  [35] ;  https://surfer.nmr.mgh.harvard.edu/ . The CN tail was not included in the segmentation due to the small cross-section area and the associated partial volume effects in diffusion imaging. The 21 images without diffusion weighting (seven b0 images for each of the three repetitions) distributed in the whole sequence were used to estimate motion correction parameters using rigid-body transformations  [36] , implemented in FSL ( http://www.fmrib.ox.ac.uk/fsl ). Motion correction for the 180 diffusion-weighted images was combined with a global registration to the T1 anatomy computed with the same method. The gradient direction for each volume was corrected using the rotation parameters. The registered images were linearly interpolated to the new reference frame with an isotropic voxel resolution of 1 mm and the three corresponding acquisitions and gradient directions were averaged. For each voxel, a diffusion tensor was computed from the images (DTI). Finally, local fiber orientation density functions (fODF) were computed using spherical deconvolution implemented in MRtrix  http://www.brain.org.au/software/mrtrix ;  [37] . Additionally, all individual dMRI images were non-linearly registered  [38]  to the most typical brain within the group of participants. The most typical brain was defined as the one that required the least cumulative deformation when non-linearly registered to all other brains  [39] . From this, group average diffusion tensor images were computed.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23887643">PMID:23887643</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The maps were displayed on inflated cortical surfaces reconstructed using FreeSurfer , and the boundaries between different visual areas were delineated manually according to the mirror reversals in the maps .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23826273">PMID:23826273</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Subjects We utilized data obtained from the ADNI database ( www.loni.ucla.edu/ADNI ) , including all MCI subjects for whom baseline MRI data ( 1.5T ) , at least moderately confident diagnoses ( i.e. confidence > 2 ) , hippocampus volumes ( i.e. volumes of left and right hippocampus , calculated by FreeSurfer Version 4.3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23785397">PMID:23785397</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/fswiki ), which includes a set of automated tools, to reconstruct the brain cortical surface from the T1-weighted MRIs  [36] . This method involved intensity normalization and used a skull-stripping algorithm to remove the extracerebral tissues as well as a connected component algorithm for image segmentation. The output at this stage consists of a single-filled white matter to estimate the gray – white matter interface. The gray – white matter assessment was used as the starting point of a deformable surface algorithm to examine the pial surface. The surface area was obtained according to the shortest distance between equivalent vertices in the pail and gray – white matter surfaces  [37]  by using a Gaussian smoothing kernel with a full width at half maximum of 10 mm to smooth the surface.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23776631">PMID:23776631</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The remaining 145 sections – 83 in GM and 62 in WM – were mapped onto the diffusion space by rigidly registering the T1 image with the T2 image , non-linearly registering the T2 image with a b = 0 image , and using the composite transformation to co-register the FreeSurfer parcellation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23626743">PMID:23626743</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI-based volume measurements of the hippocampus , amygdala , thalamus , caudate , putamen and pallidium were generated using an automated brain reconstruction method ( FreeSurfer ) for 101 unrelated ‘ sporadic ’ MTLE patients [ 70 with hippocampal sclerosis ( MTLE+HS ) , 31 with MRI-negative TLE ] , 83 unaffected full siblings of patients and 86 healthy control subjects .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23613741">PMID:23613741</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal volumes ( HV ) were derived from volumetric segmentation of subject MR scans , which were performed with the Freesurfer image analysis suit .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23162188">PMID:23162188</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Intracranial volume was automatically measured by using the FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23087654">PMID:23087654</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>The brain tissue itself was further segmented automatically using FreeSurfer 5.0.0 ( Cambridge , MA , USA ) software into white matter and gray matter ; and then subclassified into many cortical and deep brain structures .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23029115">PMID:23029115</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These anatomical images were re-sampled to 1 × 1 × 1 mm resolution , averaged , and then inflated to the cortical surface using FreeSurfer [ 32 ] , [ 33 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23006986">PMID:23006986</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical and subcortical volumes were extracted using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22962490">PMID:22962490</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>The goal is to discover a compact set of phenotypic imaging markers that are closely related to AD-sensitive genotypes encoded by SNPs . 4.1 Improved prediction of SNPs from longitudinal phenotypic imaging markers We first evaluate the proposed method by applying it to the ADNI cohort to predict the SNPs of the participants from each of their two types of imaging phenotypes , i.e. VBM markers and FreeSurfer markers , tracked over four different time points , including BL and M06/M12/M24 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22826609">PMID:22826609</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>However , this will require extending the current FreeSurfer pial-surface-finding algorithm , which is finely adapted to T 1 w image intensity relations among tissue types .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22692142">PMID:22692142</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>We studied 21 schizophrenic patients ( mean age=34 , 2 ± 8.2 years ; five female ; number of episodes=2.5 ± 2.0 ; age of onset=26.4 ± 7.8 years ; 19 medicated with atypicals ) giving written informed consent with the Positive and Negative Syndrome Scale ( PANSS ) . 4 Hippocampal subfields volume was assessed fully automatic with FreeSurfer 5 on the average of 2 – 4 ( mean=3.4 ± 0.9 ) T1-weighted magnetic resonance images ( 3 T Siemens Trio ; MPRAGE , resolution 1 × 1 × 1 mm 3 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22481909">PMID:22481909</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The geometric accuracy of our method was evaluated by cross-comparison with FreeSurfer as follows .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22319495">PMID:22319495</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1w image volume and surface processing has been previously described in humans ( Glasser and Van Essen , 2011 ) , and surfaces were made using FreeSurfer 5.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22127933">PMID:22127933</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical accuracy of each FreeSurfer ROI was visually inspected and those with unacceptable parcellation were excluded from all analyses ( no scans were excluded ) . Figure   1 displays the anatomical ROI for the current study , derived from the standard FreeSurfer atlas .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21977015">PMID:21977015</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>For presentation purposes , cortical surfaces were rendered on basis of the T1-weighted images by using Freesurfer ( Dale et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21897815">PMID:21897815</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>It implements typical fMRI preprocessing with smoothing ( SPM ) , motion correction ( SPM ) , artifact detection ( Nipype ) , and coregistration ( FreeSurfer ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21647211">PMID:21647211</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical extent of ROIs was determined automatically using the cortical map provided by Freesurfer ( Dale et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21483430">PMID:21483430</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Neuroanatomical images for 387 individuals have been collected using a Siemens 3T MRI scanner ( Siemens , New York , NY ) , linearly aligned and averaged ( 32 ) and analyzed using FreeSurfer software ( 33 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21430796">PMID:21430796</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The left and right cerebral white matter differed by 16.3 mL ( 7.6 % ) and 14 mL ( 6.5 % ) , respectively . Four of the regions ( all small regions with differences of 0.3 mL or less ) showed the FreeSurfer estimate to have larger volume than the CMA estimate .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20969964">PMID:20969964</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to create the source models , a cortical surface was extracted from a subject 's anatomical MRI using Freesurfer ( Dale et al . , 1999 ; Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20935035">PMID:20935035</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>All images were corrected for image distortion due to gradient non-linearity using ‘ GradWarp ’ ( Jovicich et al . , 2006 ) , correction for B1 non-uniformity as necessary ( Jack et al . , 2008 ) and for residual inhomogeneity using ‘ N3 ’ ( Sled et al . , 1998 ) with a software pipeline running at the Mayo Clinic Rochester . Hippocampal volumes and total intracranial volumes were measured at Mayo Clinic using FreeSurfer software ( version 4.5.0 ) ( Fischl et al . , 2002 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20829245">PMID:20829245</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Probabilistic mapping was determined by comparing the overlap between the projected activations on the cortical surface in FreeSurfer and the probabilistic cytoarchitectonic atlases provided in the FreeSurfer distribution ( Dale et al . 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20727757">PMID:20727757</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical Thickness Reconstruction of the pial surface and GM/WM boundary was performed for T1-weighted magnetic resonance ( MR ) images via the fully automated procedure implemented in FreeSurfer software [ 34 , 35 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20725521">PMID:20725521</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In a more comprehensive report we describe similar approaches for mapping FMA entities to FreeSurfer , AAL , and NeuroLex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20410712">PMID:20410712</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A further analysis using FreeSurfer provided measures of cortical thickness for each lobe , and for 13 regions in the frontal lobe .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19698138">PMID:19698138</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain morphometric analysis was performed with the automated segmentation work-flow implemented in FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17341309">PMID:17341309</a></td>
		<td>rdw</td>
		<td align=right>0.95</td>
		<td colspan=92 style='mso-ignore:colspan'>We used 2 independent methods for determining GMV : FreeSurfer ( FS ) and Volumetric Brain Morphology ( VBM ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25674013">PMID:25674013</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The volume of the hippocampus was estimated using the software FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25619658">PMID:25619658</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To improve the reliability of the measure , we used different experimental paradigms , where we delineated early visual cortices retinotopically according to the phase-encoded map ( Sereno et   al . , 1995 ) , retinotopically according to the population-receptive-field map ( Dumoulin and Wandell , 2008 ) , or morphologically according to the cortical folding patterns ( Desikan et   al . , 2006 ) , acquired the structural data from in   vivo T1-weighted MRI imaging , in   vivo quantitative-T1 MRI imaging , or in   vitro histology sectioning , and conducted the surface-based analysis in SPM ( Ashburner , 2012 ) , FSL ( Jenkinson et   al . , 2012 ) , Freesurfer ( Fischl , 2012 ) , or MPAV CBS ( Bazin et   al . , 2013 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25565989">PMID:25565989</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface models were created from the anatomical scan of each subject using Freesurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25565939">PMID:25565939</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The maps are displayed on an inflated surface brain of the ANTS-derived atlas created using Freesurfer ( Dale et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25431294">PMID:25431294</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The adolescent group comes from a common pool of subjects used as control group in previous publications [ 34 – 37 ] and their corresponding MR images which met the technical criteria to be processed in FreeSurfer were selected .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25414626">PMID:25414626</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The segmentation and volumetric studies were processed using FreeSurfer 4.4.0 ( Dale et al . , 1999 ; Fischl et al . , 1999 ; Fischl and Dale , 2000 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25379448">PMID:25379448</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Morphometric procedures using FreeSurfer have been shown to demonstrate good test – retest reliability ( Han , 2006 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25368566">PMID:25368566</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness was visualized by mapping vertex-wise thickness values on to individual cortical surfaces , with group comparison performed with a processing stream implicit in FREESURFER .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25309312">PMID:25309312</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A probabilistic map of the left STS was created by averaging two T1-weighted MR images from each of 61 brains , in which the STS had been demarcated using Freesurfer ( Dale et al . , 1999 ) for automatic parcellation of sulci and gyri ( Destrieux et al . , 2010 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25229673">PMID:25229673</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In processing the brain MRI data , we used FreeSurfer [ 34 ] for cortical segmentations , such as gray matter , white matter , ventricles , and cerebellum .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25098835">PMID:25098835</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The phenotypic data were processed with the FreeSurfer image analysis suite [ 16 ] by UCSF researchers [ 17 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24971513">PMID:24971513</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>One new method ( MSM-sulc ) achieves comparably good areal alignment the current standard method ( FreeSurfer ) but achieved this with much lower spatial distortion .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24879508">PMID:24879508</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The imaging parameters recommended by FreeSurfer [ 16 ] are : TR/TE = 2530/3.37 ms , 1200 ms , echo spacing time 7.9 ms , bandwidth 200 Hz/pixel , total scan time 5 minute and 41 second .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24817850">PMID:24817850</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This is accomplished through the use of the canvas feature of HTML5 . The atlas itself is based on FreeSurfer segmented structural MRI scan of a typically developing 15-year-old female subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24706010">PMID:24706010</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal volume at baseline was measured with FreeSurfer ( v5.1 ) and total intracranial volume ( TIV ) was measured using an in-house algorithm whereby a TIV mask is propagated from template space to each subject 's MPRAGE image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24675546">PMID:24675546</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As described in [ 39 ] , each of the 998 regions used in the parcellation is affiliated with one of 66 anatomical parcels defined based on surface reconstruction performed in Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24659952">PMID:24659952</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A total of eight subjects were examined , for seven of these the functional scan was followed by an anatomical reference scan using an MP2RAGE sequence ( Marques et al . , 2010 ) which had identical orientation , slice thickness and in-plane resolution as the functional scan , but 80 slices to facilitate analysis in FreeSurfer ( Dale et al . , 1999 ; Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24627820">PMID:24627820</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 images of each subject were processed to extract the cortical surface using the FreeSurfer pipeline [ 14 , 15 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24431986">PMID:24431986</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Left image was produced with FreeSurfer tksurfer tool and the right one with PySurfer ( http : //pysurfer . github . io ) which internally depends on Mayavi ( Ramachandran and Varoquaux , 2010 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24409136">PMID:24409136</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This surface model was generated by a fully automated cortical surface reconstruction procedure using FreeSurfer software ( Dale et al . , 1999 ) for producing a detailed geometric description ( regular tessellation of the cortical surface consisting of equilateral triangles known as vertices ) of the gray-white matter boundary of the neocortical mantle and the mesial temporal lobe .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23639259">PMID:23639259</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The imaging data sets were pre-processed and the cortical surface was segmented using FreeSurfer software ( Dale et al . , 1999 ; Fischl , 2012 ; Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23476025">PMID:23476025</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>An overview of the RSFC-Boundary Mapping method we employed is depicted in Figure   8 . Following the volumetric registration procedures , the atlas-registered MPRAGE anatomical image for each subject was processed through the Freesurfer analysis pipeline ( Freesurfer 4.5 ; Dale and Sereno 1993 ; Dale et al . 1999 ; Fischl , Sereno , et al . 1999 ; S é gonne et al . 2004 , 2005 ) to generate white matter , pial , and spherical surfaces for each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26053998">PMID:26053998</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26053355">PMID:26053355</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26045942">PMID:26045942</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26043845">PMID:26043845</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26040979">PMID:26040979</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26040977">PMID:26040977</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26040277">PMID:26040277</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26022748">PMID:26022748</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26022354">PMID:26022354</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26016622">PMID:26016622</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26014110">PMID:26014110</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25986304">PMID:25986304</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25984242">PMID:25984242</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25981522">PMID:25981522</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25978801">PMID:25978801</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25977761">PMID:25977761</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25971306">PMID:25971306</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25967601">PMID:25967601</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25966366">PMID:25966366</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25957598">PMID:25957598</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25953057">PMID:25953057</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25944410">PMID:25944410</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25940845">PMID:25940845</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25928394">PMID:25928394</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25927716">PMID:25927716</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25924202">PMID:25924202</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25921321">PMID:25921321</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25921032">PMID:25921032</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25904238">PMID:25904238</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25900078">PMID:25900078</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25897648">PMID:25897648</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25895453">PMID:25895453</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25880400">PMID:25880400</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25875287">PMID:25875287</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25864197">PMID:25864197</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25862483">PMID:25862483</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25859933">PMID:25859933</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25856168">PMID:25856168</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25850861">PMID:25850861</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25850640">PMID:25850640</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25847757">PMID:25847757</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was measured automatically using FreeSurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu/ , Dale et al.  1999 ; Fischl and Dale  2000 ). Details of the surface-based cortical reconstruction procedures have been extensively documented previously (Dale et al.  1999 ; Fischl and Dale  2000 ; Fischl et al.  2004 ; Segonne et al.  2004 ). Briefly, the FreeSurfer pipeline performs motion correction on the T1-images, automatically removes non-brain tissues (Segonne et al.  2004 ), transforms volumetric data to a common atlas, performs intensity normalization and topology correction (Fischl et al.  2004 ; Segonne et al.  2007 ) and defines the boundaries of the gray/white matter and pial surface (Dale et al.  1999 ; Fischl and Dale  2000 ). For the purposes of the current study, automated image surfaces and segmentations were inspected and screened for quality control but were not manually edited, in order to maintain the objectivity of results. Intracranial volume was determined by a validated automated method known to be equivalent to manual intracranial volume estimation (Buckner et al.  2004 ). Three subjects (Validation sample) were excluded due to segmentation failures in FreeSurfer resulting in 301 subjects suitable for analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25829144">PMID:25829144</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25824621">PMID:25824621</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25824306">PMID:25824306</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25801924">PMID:25801924</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>We performed vertex-wise exploratory analyses of the association between aggression and cortical thickness, cortical surface area, and cortical gyrification across the entire cortex. Furthermore, we examined whether gender moderated the association between cortical thickness, surface area, or gyrification and aggression. For this purpose, FreeSurfer ’ s QDEC was used ( www.surfer.nmr.mgh.harvard.edu ). Regions for which cortical thickness, surface area, or gyrification were significantly associated with aggressive behavior as well as regions in which gender moderated this association were determined using general linear models (GLMs) with age, gender, and IQ as covariates. For use in QDEC, missing values on intelligence were mean imputed. To correct for the effect of multiple comparisons, a cluster-based Monte Carlo simulation was performed using 10,000 iterations and  p ? ? ? .05.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25794861">PMID:25794861</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25773062">PMID:25773062</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25759285">PMID:25759285</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25749985">PMID:25749985</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25749984">PMID:25749984</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25725332">PMID:25725332</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25685703">PMID:25685703</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25680742">PMID:25680742</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25662837">PMID:25662837</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25653850">PMID:25653850</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25647329">PMID:25647329</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25645374">PMID:25645374</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25613588">PMID:25613588</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25604494">PMID:25604494</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25598428">PMID:25598428</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25592183">PMID:25592183</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25567423">PMID:25567423</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25560246">PMID:25560246</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25555998">PMID:25555998</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25522694">PMID:25522694</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25488025">PMID:25488025</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25485031">PMID:25485031</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25484927">PMID:25484927</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25481508">PMID:25481508</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25477275">PMID:25477275</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25396416">PMID:25396416</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25380682">PMID:25380682</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25370452">PMID:25370452</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25339897">PMID:25339897</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25333183">PMID:25333183</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25313508">PMID:25313508</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25309437">PMID:25309437</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25304292">PMID:25304292</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25288137">PMID:25288137</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25273619">PMID:25273619</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25260207">PMID:25260207</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25257747">PMID:25257747</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25256265">PMID:25256265</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25227744">PMID:25227744</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25223991">PMID:25223991</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25221289">PMID:25221289</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25199919">PMID:25199919</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25177284">PMID:25177284</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25165444">PMID:25165444</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25165386">PMID:25165386</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25155256">PMID:25155256</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25128453">PMID:25128453</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25124520">PMID:25124520</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25115748">PMID:25115748</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25100674">PMID:25100674</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25089021">PMID:25089021</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25079289">PMID:25079289</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25074636">PMID:25074636</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25058901">PMID:25058901</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25051543">PMID:25051543</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25050427">PMID:25050427</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25044977">PMID:25044977</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25037858">PMID:25037858</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25033473">PMID:25033473</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25032642">PMID:25032642</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25005936">PMID:25005936</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25003032">PMID:25003032</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer (v4.2.0;  http://surfer.nmr.mgh.harvard.edu/ ) was used to parcellate each participant's T1-weighted MRI into 34 neocortical regions ( Fischl et al., 2004 ). Following an automated and manually-validated skull-stripping procedure, the cortical surface of each participant's MRI was inflated into a sphere for the purpose of registering it to an atlas. These inflated representations were then segmented into 34 neocortical regions. Each participant's FreeSurfer segmentation was co-registered to their DTI scan using both an affine registration and a non-linear deformation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24998995">PMID:24998995</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24994509">PMID:24994509</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24993606">PMID:24993606</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24982615">PMID:24982615</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24976634">PMID:24976634</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24973130">PMID:24973130</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24965253">PMID:24965253</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24965103">PMID:24965103</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24961871">PMID:24961871</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24958596">PMID:24958596</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24957866">PMID:24957866</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24956065">PMID:24956065</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24951640">PMID:24951640</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24939340">PMID:24939340</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24930680">PMID:24930680</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24923523">PMID:24923523</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24904400">PMID:24904400</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24892191">PMID:24892191</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24886777">PMID:24886777</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24885889">PMID:24885889</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24882679">PMID:24882679</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24880728">PMID:24880728</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24879923">PMID:24879923</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24878607">PMID:24878607</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24862391">PMID:24862391</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24852682">PMID:24852682</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24845162">PMID:24845162</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24828364">PMID:24828364</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24827550">PMID:24827550</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24816714">PMID:24816714</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24814212">PMID:24814212</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24808018">PMID:24808018</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24795619">PMID:24795619</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24789776">PMID:24789776</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24789100">PMID:24789100</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer pipeline essentially consists of two processing streams , a volumetric stream and a surface-based stream .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24788131">PMID:24788131</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24784800">PMID:24784800</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24779869">PMID:24779869</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24777562">PMID:24777562</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24775377">PMID:24775377</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24760901">PMID:24760901</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24760032">PMID:24760032</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Measures of cortical morphology ( thickness , area , volume , curvature ) and subcortical and cerebellar volumes were compared between the groups using the program FreeSurfer 5.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24755274">PMID:24755274</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24752753">PMID:24752753</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24748210">PMID:24748210</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24742925">PMID:24742925</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24736175">PMID:24736175</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24734025">PMID:24734025</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24734019">PMID:24734019</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24731528">PMID:24731528</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24723882">PMID:24723882</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24718100">PMID:24718100</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24710997">PMID:24710997</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24705201">PMID:24705201</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24699871">PMID:24699871</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Magnetic resonance imaging ( MRI ) was performed , and volumes of basal ganglia nuclei in both hemispheres were calculated using FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24688465">PMID:24688465</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24683984">PMID:24683984</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24676464">PMID:24676464</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24671392">PMID:24671392</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24670399">PMID:24670399</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24658709">PMID:24658709</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24655655">PMID:24655655</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24650599">PMID:24650599</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24650452">PMID:24650452</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24638919">PMID:24638919</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24636843">PMID:24636843</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24632141">PMID:24632141</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24630049">PMID:24630049</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24619555">PMID:24619555</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24617977">PMID:24617977</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24609465">PMID:24609465</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24607258">PMID:24607258</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24604460">PMID:24604460</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24587566">PMID:24587566</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24584328">PMID:24584328</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24567717">PMID:24567717</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24553878">PMID:24553878</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24532701">PMID:24532701</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24532539">PMID:24532539</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24523693">PMID:24523693</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24521851">PMID:24521851</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24518149">PMID:24518149</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24514566">PMID:24514566</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Full details of the magnetic resonance imaging acquisition protocols and quality checks have been described previously. 21  Brain images were segmented with the FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ) and the entire cortex of each individual was inspected for inaccuracies. Individuals with major malformations of the cerebral cortex were excluded from further analysis. Out of 1909 images, 1584 passed these quality control checks. In addition to global mean thickness of the left and right cerebral hemispheres, neuroimaging measures included cortical thickness for 33 individual regions per hemisphere. These were combined to produce weighted average thickness (weighted for surface at each region) for the four cerebral lobes (that is, frontal, temporal, parietal and occipital). The effect of magnetic resonance imaging site was controlled by adding it as a nuisance covariate in all statistical analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24503371">PMID:24503371</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24493370">PMID:24493370</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24486655">PMID:24486655</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24475058">PMID:24475058</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24473187">PMID:24473187</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24464044">PMID:24464044</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24459035">PMID:24459035</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24430512">PMID:24430512</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24422975">PMID:24422975</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24421287">PMID:24421287</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24409141">PMID:24409141</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24395657">PMID:24395657</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24369266">PMID:24369266</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24362760">PMID:24362760</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24352368">PMID:24352368</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24345388">PMID:24345388</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24341200">PMID:24341200</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24339261">PMID:24339261</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24319420">PMID:24319420</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24319419">PMID:24319419</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24309162">PMID:24309162</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24304824">PMID:24304824</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24302911">PMID:24302911</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24302902">PMID:24302902</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24298266">PMID:24298266</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>To increase the specificity of the visual MRI analysis and to exclude the individuals with subtle HS signs , we also performed automatic hippocampal volume measurements in all patients and in 82 healthy subjects using FreeSurfer software ( version 5.1.0 1 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24297733">PMID:24297733</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24285948">PMID:24285948</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24282397">PMID:24282397</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24269113">PMID:24269113</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24256621">PMID:24256621</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24253730">PMID:24253730</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24250789">PMID:24250789</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24246495">PMID:24246495</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24239903">PMID:24239903</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24223551">PMID:24223551</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24223109">PMID:24223109</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer 5.0 was used to automatically segment the brain into various regions for G1 (as defined in the wmparc.mgz file); FreeSurfer 5.1 was used for brain segmentation for G2. Visual inspection of the automated segmentation results was performed for quality assurance purposes in all datasets. Correction was done when necessary according to the FreeSurfer manual ( http://surfer.nmr.mgh.harvard.edu/fswiki/ ). Corresponding regions from the left and right hemispheres of the brain were combined to form a single ROI, e.g., the Left-Cerebellum-Cortex and the Right-Cerebellum-Cortex were combined to form a single ROI for quantitative analysis. The procedures used to compute BP ND  values from FreeSurfer-defined and manually traced ROIs were otherwise identical.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24220510">PMID:24220510</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24211008">PMID:24211008</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24210948">PMID:24210948</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24210630">PMID:24210630</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24194710">PMID:24194710</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24187540">PMID:24187540</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24185694">PMID:24185694</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179862">PMID:24179862</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179857">PMID:24179857</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179836">PMID:24179836</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179817">PMID:24179817</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179805">PMID:24179805</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179801">PMID:24179801</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179797">PMID:24179797</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179765">PMID:24179765</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements were performed using the FreeSurfer pipeline as previously described ( http://surfer.nmr.mgh.harvard.edu/fswiki ,  Dale et al., 1999; Fischl et al., 1999 ). All our analyses were performed using FreeSurfer v5.1.0 on a Linux platform. In brief, the pipeline has a volumetric and a surface-based stream. All image volumes were first registered to the template space and intensity normalized. The images were then skull-stripped based on a combination of the watershed algorithm and deformable template model ( S é gonne et al., 2004 ). The output brain mask was then labeled using a probabilistic atlas.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24179732">PMID:24179732</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24174905">PMID:24174905</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24171045">PMID:24171045</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>For retinotopic mapping analysis, FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) was used to reconstruct a T1-weighted anatomical image. The realigned retinotopic mapping images were registered to the anatomical image to obtain the registration file. The following retinotopic analysis was consistent with the one performed in [ 20 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24170157">PMID:24170157</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24161847">PMID:24161847</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24157735">PMID:24157735</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24155718">PMID:24155718</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24151465">PMID:24151465</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24151460">PMID:24151460</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24144441">PMID:24144441</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24141834">PMID:24141834</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24140937">PMID:24140937</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24130528">PMID:24130528</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24124469">PMID:24124469</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24124417">PMID:24124417</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24120430">PMID:24120430</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24114885">PMID:24114885</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24100952">PMID:24100952</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24096296">PMID:24096296</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24077565">PMID:24077565</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24077065">PMID:24077065</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24069223">PMID:24069223</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24066153">PMID:24066153</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24058640">PMID:24058640</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24058331">PMID:24058331</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24057158">PMID:24057158</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24048830">PMID:24048830</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24046733">PMID:24046733</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24041490">PMID:24041490</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24039712">PMID:24039712</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24039028">PMID:24039028</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24037033">PMID:24037033</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24035178">PMID:24035178</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24032010">PMID:24032010</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24023538">PMID:24023538</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24012876">PMID:24012876</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23996156">PMID:23996156</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23996101">PMID:23996101</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23986698">PMID:23986698</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23986653">PMID:23986653</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23986266">PMID:23986266</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23980618">PMID:23980618</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23979107">PMID:23979107</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23978342">PMID:23978342</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23977281">PMID:23977281</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23964234">PMID:23964234</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23964224">PMID:23964224</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23962297">PMID:23962297</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23959201">PMID:23959201</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23958958">PMID:23958958</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23951142">PMID:23951142</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23950872">PMID:23950872</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23947582">PMID:23947582</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23947395">PMID:23947395</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23935975">PMID:23935975</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23935801">PMID:23935801</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23929545">PMID:23929545</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23929204">PMID:23929204</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23926114">PMID:23926114</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23914170">PMID:23914170</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23897577">PMID:23897577</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23894288">PMID:23894288</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23892552">PMID:23892552</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23888326">PMID:23888326</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23882237">PMID:23882237</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23880336">PMID:23880336</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23877833">PMID:23877833</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23876117">PMID:23876117</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874491">PMID:23874491</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23874279">PMID:23874279</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23868151">PMID:23868151</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23866739">PMID:23866739</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23861758">PMID:23861758</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23857518">PMID:23857518</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23856297">PMID:23856297</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23853714">PMID:23853714</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23847528">PMID:23847528</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23847503">PMID:23847503</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23847497">PMID:23847497</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23847496">PMID:23847496</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23845237">PMID:23845237</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23844066">PMID:23844066</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23843934">PMID:23843934</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23843881">PMID:23843881</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23842600">PMID:23842600</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23836390">PMID:23836390</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23830856">PMID:23830856</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23827332">PMID:23827332</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23822854">PMID:23822854</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23817547">PMID:23817547</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23816189">PMID:23816189</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23803260">PMID:23803260</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer software (version 4.5.0) ( http://surfer.nmr.mgh.harvard.edu ) was used to obtain\nhippocampal, amygdala, ventricular and intracranial volumes (ICVs) from T1-weighted images \n(Fischl,  2012 ). Briefly, the subcortical\nsegmentation algorithm combines information on image intensity, probabilistic atlas \nlocation, and the local spatial relationships between structures to automatically assign a\nneuroanatomical label to each voxel in the MRI volume (Fischl  et al. \n 2002 ). The reliability of the automatic volume\nmeasurements has been tested against manual tracing, and the agreement between the \nautomatically obtained volumes and manual tracings was comparable with the agreement\nbetween manual tracings obtained by two different experts (Fischl  et al. \n 2002 ). The procedures were fully automated\nwithout manual editing. All scans were visually inspected following standard \nprocedures.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23795679">PMID:23795679</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23785657">PMID:23785657</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>This procedure was performed to allow a quantitative analysis of the DAT levels in the caudate and putamen of PD patients as described in Quantitative DAT imaging of the striatum section. Cortical and subcortical reconstruction and volumetric segmentation of each patient's structural T1-weighted structural scan were performed with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) ( Fischl 2012 ). Briefly, this includes removal of nonbrain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white-matter and deep gray-matter volumetric structures, intensity normalization, tessellation of the gray-matter – white-matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23785485">PMID:23785485</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23785311">PMID:23785311</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23774715">PMID:23774715</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23770774">PMID:23770774</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23770018">PMID:23770018</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23764385">PMID:23764385</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23762488">PMID:23762488</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23762274">PMID:23762274</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23754695">PMID:23754695</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23750128">PMID:23750128</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23744687">PMID:23744687</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23740104">PMID:23740104</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23739259">PMID:23739259</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23732850">PMID:23732850</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23723706">PMID:23723706</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications. 38 – 47  Briefly, this processing includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure, 48  automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles), 42 , 45  intensity normalization, 49  tessellation of the gray matter white matter boundary, automated topology correction, 41 , 50  and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class. 38 – 40  Once the cortical models are complete, a number of deformable procedures can be performed for further data processing and analysis including surface inflation, 44  registration to a spherical atlas which utilized individual cortical folding patterns to match cortical geometry across subjects, 51  parcellation of the cerebral cortex into units based on gyral and sulcal structure, 43 , 52  and creation of a variety of surface-based data including maps of curvature and sulcal depth. This method uses both intensity and continuity information from the entire three-dimensional MRI volume in segmentation and deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/cerebral spinal fluid (CSF) boundary at each vertex on the tessellated surface. 40  The maps are created using spatial intensity gradients across tissue classes and are therefore not simply reliant on absolute signal intensity. The maps produced are not restricted to the voxel resolution of the original data; thus, they are capable of detecting submillimeter differences between groups. Procedures for the measurement of cortical thickness have been validated against histological analysis 53  and manual measurements. 54 , 55  Freesurfer morphometric procedures have been demonstrated to show good test – retest reliability across scanner manufacturers and across field strengths. 46 , 56  The numbers used to refer to the amygdala and hippocampus are the result of the correction of the measure in millimeters by the intracranial volume.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23721966">PMID:23721966</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23715094">PMID:23715094</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23702413">PMID:23702413</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23691102">PMID:23691102</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23690757">PMID:23690757</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23686576">PMID:23686576</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23668971">PMID:23668971</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23664450">PMID:23664450</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23660871">PMID:23660871</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23658813">PMID:23658813</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23641233">PMID:23641233</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23636003">PMID:23636003</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23630578">PMID:23630578</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23630517">PMID:23630517</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23630491">PMID:23630491</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23622854">PMID:23622854</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23616155">PMID:23616155</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23615718">PMID:23615718</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23608917">PMID:23608917</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23594250">PMID:23594250</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23585869">PMID:23585869</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23577105">PMID:23577105</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23577096">PMID:23577096</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23571160">PMID:23571160</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23568433">PMID:23568433</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23558337">PMID:23558337</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI data were processed with the fully automated segmentation procedure in Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ; see the online data supplement). We focused on examining ventral striatal volume, including the putamen and nucleus accumbens, which have been shown to be contiguous ventral striatal regions in humans ( 30 ). Following subcortical segmentation, we compared the volumes of these regions between groups using ANOVA and pairwise comparisons, and we performed correlation analyses to examine relationships with peak ventral striatal activity (as for the above correlation).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23554933">PMID:23554933</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23554593">PMID:23554593</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23541334">PMID:23541334</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23536915">PMID:23536915</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23523802">PMID:23523802</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23512004">PMID:23512004</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23509639">PMID:23509639</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23497383">PMID:23497383</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23493892">PMID:23493892</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23485592">PMID:23485592</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23482650">PMID:23482650</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23469101">PMID:23469101</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23468177">PMID:23468177</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23467031">PMID:23467031</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23463873">PMID:23463873</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23462677">PMID:23462677</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23459778">PMID:23459778</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23451272">PMID:23451272</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23449653">PMID:23449653</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23439846">PMID:23439846</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23424625">PMID:23424625</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23417665">PMID:23417665</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23415471">PMID:23415471</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23407860">PMID:23407860</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23405083">PMID:23405083</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23405066">PMID:23405066</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23398027">PMID:23398027</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23382713">PMID:23382713</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23377127">PMID:23377127</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23366607">PMID:23366607</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23366192">PMID:23366192</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23358957">PMID:23358957</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23358158">PMID:23358158</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Data analysis was performed using three freely available software packages: ExploreDTI( www.exploredti.com ), FreeSurfer( www.surfer.nmr.mgh.harvard.edu ) and TRACULA, diffusion toolbox of Functional MRI tool of Brain Software Library(FSL;  www.fmrib.ox.ac.uk/fsl ). ExploreDTI is an advanced quality assurance tool for diffusion images, and can examine individual images to check for gross artifacts, such as signal dropouts and interleave artifacts. Details on preprocessing are in  Supplemental material . After tensor estimation, the probability distributions of multiple fiber directions of our a priori WM tracts was based on the Bayesian framework for global tractography proposed in TRACULA 39 , that uses reproducible tracking protocols 24 , validated on a set of training subjects, and is therefore suitable for the study of well-characterized WM tracts 40 . Measures of interest (FA, RD and L1) were then extracted for each reconstructed pathway in each participant. Structural images were processed in FreeSurfer 41  to automatically parcellate the cortex and segment subcortical regions for each subject, in order for TRACUA to derive the  end  regions for the automated tractography.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23355801">PMID:23355801</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23355112">PMID:23355112</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Next, we co-registered the other three subjects MR images to the MR image of subject S1 using FreeSurfer volume registration ( www.surfer.nmr.mgh.harvard.edu ). Localization error magnitude and orientation maps were interpolated to a 1-mm 3-D grid of source locations from the 8-mm spaced source location grids. These error maps were then transformed to the S1 coordinate system to obtain mean error maps across the four subjects using the same volume registration for the source space. As the subject head shapes were quite different, in some brain regions the co-registered error directions differed, thus partly cancelling each other in the mean error images. Figure   9  shows the mean dipole source localization error directions (arrows) and magnitudes (colors), and Table   2  the median localization error magnitudes (in mm). Fig.   9 Mean dipole source localization error directions ( arrows ) and magnitudes ( colors ) for four subjects using spherical and MNI template-based head models to localize equivalent dipole sources simulated in a subject-specific four-layer realistic BEM head model. Other details as in Fig.   3</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23354984">PMID:23354984</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23350832">PMID:23350832</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23343471">PMID:23343471</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23342130">PMID:23342130</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23338839">PMID:23338839</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23329358">PMID:23329358</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23322167">PMID:23322167</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23317454">PMID:23317454</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23308272">PMID:23308272</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23300699">PMID:23300699</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23293524">PMID:23293524</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23288436">PMID:23288436</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23285601">PMID:23285601</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23284610">PMID:23284610</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23273486">PMID:23273486</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23266197">PMID:23266197</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23266013">PMID:23266013</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23261638">PMID:23261638</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23259063">PMID:23259063</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23250887">PMID:23250887</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23250414">PMID:23250414</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23230398">PMID:23230398</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23228141">PMID:23228141</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23227153">PMID:23227153</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23226429">PMID:23226429</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23224109">PMID:23224109</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23192171">PMID:23192171</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23186987">PMID:23186987</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23177922">PMID:23177922</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23177479">PMID:23177479</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23162440">PMID:23162440</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23154096">PMID:23154096</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23154095">PMID:23154095</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23153869">PMID:23153869</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23150470">PMID:23150470</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23149031">PMID:23149031</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23149030">PMID:23149030</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23144995">PMID:23144995</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23142193">PMID:23142193</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23139080">PMID:23139080</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23128363">PMID:23128363</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23124622">PMID:23124622</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23123680">PMID:23123680</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23119088">PMID:23119088</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23110888">PMID:23110888</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23109807">PMID:23109807</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23104186">PMID:23104186</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23095216">PMID:23095216</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23087638">PMID:23087638</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23087608">PMID:23087608</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23086519">PMID:23086519</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23071497">PMID:23071497</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23067727">PMID:23067727</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23055966">PMID:23055966</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23055044">PMID:23055044</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23054230">PMID:23054230</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23049910">PMID:23049910</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23049751">PMID:23049751</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23038403">PMID:23038403</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23028771">PMID:23028771</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>For gray matter segmentation we used (1)  “ Segment ”  (provided by SPM8 (r4290):  www.fil.ion.ucl.ac.uk/spm/software/spm8/ ), (2)  “ New Segment ”  (provided by SPM8), (3)  “ VBM8 ”  (provided by the Voxel-based morphometry toolbox for SPM8 (r413):  http://dbm.neuro.uni-jena.de/vbm/ ), (4)  “ FAST ”  (version 4.1, provided by FSL (version 4.1.6):  www.rmrib.ox.ac.uk/fsl/index.html ), and (5)  “ FreeSurfer ”  (provided by FreeSurfer (version 4.5.0): ( http://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferWiki ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23028511">PMID:23028511</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23021621">PMID:23021621</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23013554">PMID:23013554</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22993282">PMID:22993282</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22984010">PMID:22984010</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22973224">PMID:22973224</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22973200">PMID:22973200</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22969709">PMID:22969709</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22969416">PMID:22969416</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22960087">PMID:22960087</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22952926">PMID:22952926</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22948482">PMID:22948482</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22942008">PMID:22942008</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22941717">PMID:22941717</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22937153">PMID:22937153</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22936906">PMID:22936906</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22912605">PMID:22912605</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22910323">PMID:22910323</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22905918">PMID:22905918</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22892286">PMID:22892286</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22890700">PMID:22890700</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>We utilized the Freesurfer pipeline version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ), which includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure (Segonne et al.  2004 ), automated Talairach transformation, segmentation of the subcortical white matter and deep grey matter volumetric structures (Fischl et al.  2002 ; Fischl et al.  2004a ; Segonne et al.  2004 ) intensity normalization (Sled et al.  1998 ), tessellation of the grey matter white matter boundary, automated topology correction (Fischl et al.  2001 ; Segonne et al.  2007 ), and surface deformation following intensity gradients to optimally place the grey/white and grey/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Dale et al.  1999 ; Dale and Sereno  1993 ; Fischl and Dale  2000 ). Once the cortical models are complete, registration to a spherical atlas takes place which utilizes individual cortical folding patterns to match cortical geometry across subjects (Fischl et al.  1999 ). This is followed by parcellation of the cerebral cortex into units based on gyral and sulcal structure (Desikan et al.  2006 ; Fischl et al.  2004b ). The pipeline generated 68 cortical thickness, cortical volume, surface area, mean curvature, gaussian curvature, folding index and curvature index measures (34 from each hemisphere) and 46 regional subcortical volumes. Volumes of white matter hypointensities, optic chiasm, right and left vessel, and left and right choroid plexus were excluded from further analysis. Cortical thickness and volumetric measures from the right and left side were averaged (Fjell et al.  2009 ; Walhovd et al.  2011 ). In total 259 variables obtained from the pipeline were used as input variables for the OPLS classification, 34 cortical regions (7 types of measures) and 21 regional volumes (Table   2 ). Figure   1  illustrates the location of both the cortical and subcortical regions. This segmentation approach has been used for multivariate classification of Alzheimer ’ s disease and healthy controls (Westman et al.  2011d ), neuropsychological-image analysis (Liu et al.  2010c ,  2011 ), imaging-genetic analysis (Liu et al.  2010a ,  b ) and biomarker discovery (Thambisetty et al.  2010 ). Table   2 Variable included in OPLS analysis</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22890099">PMID:22890099</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22875857">PMID:22875857</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22868048">PMID:22868048</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22858417">PMID:22858417</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22832815">PMID:22832815</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22832611">PMID:22832611</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22824414">PMID:22824414</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22807270">PMID:22807270</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22792385">PMID:22792385</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22792301">PMID:22792301</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22792291">PMID:22792291</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22790248">PMID:22790248</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22770508">PMID:22770508</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22768163">PMID:22768163</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22763378">PMID:22763378</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22743195">PMID:22743195</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22742954">PMID:22742954</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22737162">PMID:22737162</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22732324">PMID:22732324</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22722624">PMID:22722624</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22720021">PMID:22720021</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22711160">PMID:22711160</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22707934">PMID:22707934</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22698767">PMID:22698767</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22693341">PMID:22693341</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22689752">PMID:22689752</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22675003">PMID:22675003</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22672878">PMID:22672878</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22665261">PMID:22665261</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22660480">PMID:22660480</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22654752">PMID:22654752</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22642651">PMID:22642651</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22637582">PMID:22637582</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22616011">PMID:22616011</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22613821">PMID:22613821</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22610393">PMID:22610393</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22608156">PMID:22608156</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22590626">PMID:22590626</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22586381">PMID:22586381</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22580170">PMID:22580170</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22562047">PMID:22562047</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22558222">PMID:22558222</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22551659">PMID:22551659</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22536182">PMID:22536182</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22536181">PMID:22536181</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22535514">PMID:22535514</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22528474">PMID:22528474</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22527992">PMID:22527992</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22521631">PMID:22521631</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22514646">PMID:22514646</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22510097">PMID:22510097</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22504421">PMID:22504421</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22504417">PMID:22504417</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22503937">PMID:22503937</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22503890">PMID:22503890</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22500923">PMID:22500923</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22493576">PMID:22493576</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22484411">PMID:22484411</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22482070">PMID:22482070</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22481382">PMID:22481382</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22471353">PMID:22471353</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22454243">PMID:22454243</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22450298">PMID:22450298</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22440648">PMID:22440648</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22438191">PMID:22438191</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22434839">PMID:22434839</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22426232">PMID:22426232</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22416228">PMID:22416228</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22412922">PMID:22412922</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22406550">PMID:22406550</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22405042">PMID:22405042</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22402338">PMID:22402338</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22363313">PMID:22363313</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22360759">PMID:22360759</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22348041">PMID:22348041</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22345371">PMID:22345371</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22331913">PMID:22331913</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22319489">PMID:22319489</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22303368">PMID:22303368</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22297204">PMID:22297204</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22293721">PMID:22293721</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22291030">PMID:22291030</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22289804">PMID:22289804</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22275896">PMID:22275896</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22265363">PMID:22265363</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22248573">PMID:22248573</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22246242">PMID:22246242</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22230945">PMID:22230945</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22227886">PMID:22227886</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22227717">PMID:22227717</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22226742">PMID:22226742</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22217209">PMID:22217209</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22217183">PMID:22217183</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22209534">PMID:22209534</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22200883">PMID:22200883</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22197830">PMID:22197830</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22189342">PMID:22189342</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22174743">PMID:22174743</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22164139">PMID:22164139</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22162471">PMID:22162471</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22155377">PMID:22155377</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22141926">PMID:22141926</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22125519">PMID:22125519</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22120144">PMID:22120144</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22116979">PMID:22116979</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22102649">PMID:22102649</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22095921">PMID:22095921</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22089322">PMID:22089322</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22064977">PMID:22064977</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22048467">PMID:22048467</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22047963">PMID:22047963</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22037958">PMID:22037958</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22002080">PMID:22002080</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21994500">PMID:21994500</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21994062">PMID:21994062</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21990125">PMID:21990125</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21987595">PMID:21987595</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21980405">PMID:21980405</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21980019">PMID:21980019</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21979818">PMID:21979818</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21978603">PMID:21978603</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21970560">PMID:21970560</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21967959">PMID:21967959</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21965438">PMID:21965438</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21965122">PMID:21965122</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21956440">PMID:21956440</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21952669">PMID:21952669</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21931990">PMID:21931990</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21907880">PMID:21907880</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21904527">PMID:21904527</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21898679">PMID:21898679</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21886615">PMID:21886615</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21880566">PMID:21880566</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21867750">PMID:21867750</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21864688">PMID:21864688</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21863138">PMID:21863138</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21852978">PMID:21852978</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21841261">PMID:21841261</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21840400">PMID:21840400</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21835253">PMID:21835253</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21818285">PMID:21818285</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21803164">PMID:21803164</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21799248">PMID:21799248</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21799209">PMID:21799209</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21794103">PMID:21794103</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21791102">PMID:21791102</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21769991">PMID:21769991</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21765181">PMID:21765181</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21763442">PMID:21763442</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21761694">PMID:21761694</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21761660">PMID:21761660</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21760872">PMID:21760872</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21760582">PMID:21760582</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21753165">PMID:21753165</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21749926">PMID:21749926</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21743807">PMID:21743807</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21734511">PMID:21734511</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21717131">PMID:21717131</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21713110">PMID:21713110</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21705088">PMID:21705088</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21702065">PMID:21702065</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21698393">PMID:21698393</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21693632">PMID:21693632</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21687590">PMID:21687590</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21660881">PMID:21660881</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21658457">PMID:21658457</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21640841">PMID:21640841</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21618338">PMID:21618338</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21616622">PMID:21616622</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21613600">PMID:21613600</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21575012">PMID:21575012</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21574716">PMID:21574716</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21573953">PMID:21573953</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21563239">PMID:21563239</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21551226">PMID:21551226</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21550668">PMID:21550668</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21541252">PMID:21541252</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21525610">PMID:21525610</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21523563">PMID:21523563</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21518999">PMID:21518999</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21514321">PMID:21514321</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21512741">PMID:21512741</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21508934">PMID:21508934</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21505883">PMID:21505883</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21505765">PMID:21505765</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21497489">PMID:21497489</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21493908">PMID:21493908</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21483719">PMID:21483719</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional and anatomical data were preprocessed and analyzed using FreeSurfer and FS-FAST</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21479735">PMID:21479735</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21457744">PMID:21457744</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21453353">PMID:21453353</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21437929">PMID:21437929</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21412836">PMID:21412836</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21412462">PMID:21412462</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21397703">PMID:21397703</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21381829">PMID:21381829</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21376812">PMID:21376812</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21375447">PMID:21375447</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21371528">PMID:21371528</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21368088">PMID:21368088</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21365601">PMID:21365601</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21336694">PMID:21336694</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21326631">PMID:21326631</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21326599">PMID:21326599</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21310594">PMID:21310594</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21310189">PMID:21310189</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21304991">PMID:21304991</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21293123">PMID:21293123</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21289873">PMID:21289873</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21289538">PMID:21289538</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21283683">PMID:21283683</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21266047">PMID:21266047</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21253357">PMID:21253357</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21245087">PMID:21245087</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21237273">PMID:21237273</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21236720">PMID:21236720</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21236646">PMID:21236646</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21212942">PMID:21212942</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21195778">PMID:21195778</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21191807">PMID:21191807</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21175010">PMID:21175010</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21160558">PMID:21160558</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21160549">PMID:21160549</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21145214">PMID:21145214</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21130424">PMID:21130424</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21125003">PMID:21125003</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21120141">PMID:21120141</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21113405">PMID:21113405</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21102491">PMID:21102491</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21088707">PMID:21088707</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21079730">PMID:21079730</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/21029454">PMID:21029454</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20980455">PMID:20980455</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20951979">PMID:20951979</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20932348">PMID:20932348</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20879394">PMID:20879394</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20879304">PMID:20879304</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20875635">PMID:20875635</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20869085">PMID:20869085</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20850277">PMID:20850277</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20844604">PMID:20844604</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20839293">PMID:20839293</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20800097">PMID:20800097</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20799340">PMID:20799340</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20740072">PMID:20740072</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20688164">PMID:20688164</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20674750">PMID:20674750</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20670981">PMID:20670981</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20631844">PMID:20631844</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20628643">PMID:20628643</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20621657">PMID:20621657</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20598216">PMID:20598216</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20592949">PMID:20592949</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20582489">PMID:20582489</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20570398">PMID:20570398</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20567938">PMID:20567938</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20562040">PMID:20562040</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20553893">PMID:20553893</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20550017">PMID:20550017</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20547229">PMID:20547229</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20542120">PMID:20542120</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20541772">PMID:20541772</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20538772">PMID:20538772</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20510424">PMID:20510424</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20480153">PMID:20480153</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20461147">PMID:20461147</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20454594">PMID:20454594</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20452574">PMID:20452574</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20446135">PMID:20446135</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20381626">PMID:20381626</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20373444">PMID:20373444</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20362681">PMID:20362681</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20353827">PMID:20353827</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20339478">PMID:20339478</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20338250">PMID:20338250</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20226665">PMID:20226665</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20195255">PMID:20195255</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20185465">PMID:20185465</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20182057">PMID:20182057</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20176931">PMID:20176931</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20162602">PMID:20162602</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20153146">PMID:20153146</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20131046">PMID:20131046</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20130756">PMID:20130756</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20108369">PMID:20108369</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20101038">PMID:20101038</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20100581">PMID:20100581</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20090673">PMID:20090673</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20077162">PMID:20077162</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20060915">PMID:20060915</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20029407">PMID:20029407</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/20002143">PMID:20002143</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19996042">PMID:19996042</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19965849">PMID:19965849</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19949458">PMID:19949458</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19943100">PMID:19943100</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19940480">PMID:19940480</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19926451">PMID:19926451</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19889405">PMID:19889405</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19884612">PMID:19884612</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19876635">PMID:19876635</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19874902">PMID:19874902</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19847314">PMID:19847314</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19840895">PMID:19840895</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19826470">PMID:19826470</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19796696">PMID:19796696</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19796388">PMID:19796388</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19781814">PMID:19781814</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19778621">PMID:19778621</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19722171">PMID:19722171</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19709963">PMID:19709963</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19689234">PMID:19689234</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19682030">PMID:19682030</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19668704">PMID:19668704</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19616634">PMID:19616634</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19587284">PMID:19587284</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19560122">PMID:19560122</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19559796">PMID:19559796</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19521503">PMID:19521503</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>using Freesurfer ( surfer . nmr . mgh . harvard.edu ) and 998 regions of interest ( ROIs ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19502476">PMID:19502476</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19473666">PMID:19473666</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19457748">PMID:19457748</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19414722">PMID:19414722</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19409230">PMID:19409230</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19405146">PMID:19405146</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19405129">PMID:19405129</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19370155">PMID:19370155</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19365552">PMID:19365552</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19349603">PMID:19349603</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19344771">PMID:19344771</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19342539">PMID:19342539</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19264138">PMID:19264138</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19249372">PMID:19249372</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19247451">PMID:19247451</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19236922">PMID:19236922</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19233293">PMID:19233293</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19201003">PMID:19201003</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19184562">PMID:19184562</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19162198">PMID:19162198</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19150922">PMID:19150922</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19139874">PMID:19139874</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19110391">PMID:19110391</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19104666">PMID:19104666</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19061377">PMID:19061377</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/19015086">PMID:19015086</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18999128">PMID:18999128</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18979813">PMID:18979813</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18979748">PMID:18979748</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18958220">PMID:18958220</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18946504">PMID:18946504</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18923647">PMID:18923647</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18829629">PMID:18829629</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18695738">PMID:18695738</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18675553">PMID:18675553</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18667352">PMID:18667352</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18628679">PMID:18628679</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18597554">PMID:18597554</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18537111">PMID:18537111</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18494772">PMID:18494772</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18474111">PMID:18474111</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18455931">PMID:18455931</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18431486">PMID:18431486</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortex segmentation was done on the T1 image using the Freesurfer software package ( http : //surfer . nmr . mgh . harvard.edu/ ) . Segmentation maps were normalized and resampled to a 4 × 4 × 4 mm resolution to spatially overlap the filtered time-series .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18414838">PMID:18414838</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18359198">PMID:18359198</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18242102">PMID:18242102</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18193041">PMID:18193041</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18051085">PMID:18051085</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18044566">PMID:18044566</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17999200">PMID:17999200</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17937818">PMID:17937818</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17848191">PMID:17848191</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17710494">PMID:17710494</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17609681">PMID:17609681</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17593973">PMID:17593973</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17490861">PMID:17490861</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17428750">PMID:17428750</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17353226">PMID:17353226</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17297549">PMID:17297549</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17295313">PMID:17295313</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17048979">PMID:17048979</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/17011792">PMID:17011792</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/16998482">PMID:16998482</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/16920366">PMID:16920366</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/16557292">PMID:16557292</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/16503170">PMID:16503170</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/16411982">PMID:16411982</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/16112878">PMID:16112878</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/15986433">PMID:15986433</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/15850732">PMID:15850732</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/15488412">PMID:15488412</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/15156150">PMID:15156150</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/15077112">PMID:15077112</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/12925854">PMID:12925854</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/12803969">PMID:12803969</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/11941373">PMID:11941373</a></td>
		<td>rdw</td>
		<td align=right>0.20</td>
		<td colspan=92 style='mso-ignore:colspan'></td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26509112">PMID:26509112</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted scans were processed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) including 3D surface reconstruction and full brain segmentation ( Fischl et al., 2002 ). The segmented volume was used in order to produce high quality brain masks for co-registration.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26503036">PMID:26503036</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We employ MRI data resulting from a study by the Nathan Kline Institute (NKI) 28  which is freely available via the following public online database:  http://fcon_1000.projects.nitrc.org/indi/pro/nki.html . In total we included 121 participants aged between 4 and 40 years. Cortical reconstruction was performed using the Freesurfer image analysis suite, which is documented and freely available online ( http://surfer.nmr.mgh.harvard.edu/ ); the algorithms employed for this construction are discussed elsewhere 33 34 . In brief: a single filled volume was generated for each hemisphere onto which a triangular surface tessellation is fitted, resulting in a mesh or lattice of spatial coordinates defining the grey matter surface (also termed pial surface) of approximately 150,000 vertices (the smallest lattice obtained from the NKI dataset comprises 115,390 vertices, the largest, 187,126). No manual edits were necessary.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26500508">PMID:26500508</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26485710">PMID:26485710</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All data processing was performed off-line on a Mac workstation. For all subjects, brain cortical thickness (CTh) and deep gray nuclei volume was measured from the MPRAGE images using Freesurfer software, version 3.0.5 ( http://surfer.nmr.mgh.harvard.edu ). This process consists of several stages, detailed elsewhere [ 33 ]. Briefly, the procedure included removal of non-brain tissue, Talairach segmentation of white matter, deep gray matter structures and cerebellar structures, automatic correction of topological defects and registration into an average spherical surface template. All cortical segmentations were manually checked and re-run after manual editing if errors had occurred. Thickness measurements (in mm) were obtained by calculating the closest distance between the cortical surface and the gray/white matter border. Parcellation of the brain was done using the Desikan — Killiany atlas which consists of thirty-four cortical structures per hemisphere [ 34 ]. For further analysis, we combined each area of the two hemispheres (correlation p  <  0.01) and composed seven cortical areas based on a division described in Desikan et al [ 34 ]. The composed areas for analysis were the temporal lobe — medial aspect, temporal lobe — lateral aspect, frontal lobe, parietal lobe, occipital lobe, cingulate cortex and insula. Furthermore, we obtained volumetric measures (in mm 3 ) from the cerebellum, thalamus, caudate, putamen, pallidum, hippocampus and amygdala.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26484048">PMID:26484048</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional data were sampled onto the cortical surface and projected onto an average spherical representation using the FreeSurfer analysis stream ( http://surfer.nmr.mgh.harvard.edu ). The two contrasts of interest (Approaching Faces vs. Withdrawing Faces, and Approaching Cars vs. Withdrawing Cars) were examined using a t-statistic at each vertex on the spherical surface using random effects. Only four runs of the data were included for each subject, in order to permit the exclusion of runs confounded by poor performance on the attentional task or head motion, while maintaining approximately the same number of runs per subject (and per group) in the analyses. Thus, the first four runs that met the behavioral and motion inclusion criteria (see below) for each subject were included in the analyses. One subject (a schizophrenic patient) had only three runs that met these criteria. Of note, this procedure did not result in between-group differences in the runs included. Runs 1 – 4, or runs 1 – 3 plus one other run, were used for the 14 controls and for 12 of the 15 schizophrenic patients; run 1 plus later runs (runs 5+) were used in 3 patients.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26444271">PMID:26444271</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution T1-weighted brain images were used to measure the regional cortical thicknesses in all subjects by applying cross sectional pipeline of FreeSurfer (v 5.3.0), as described in detail elsewhere [ 27 ]. Briefly, high-resolution T1-weighted brain images volumes from each subject were converted into FreeSurfer data format. Data processing included removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, intensity normalization, segmentation of sub-cortical white and deep gray matter tissue types, tessellation of the gray and white matter boundaries, automated topology correction, and surface deformation ( http://surfer.nmr.mgh.harvard.edu/ ). To measure changes in the cortical thickness between baseline and follow-up scans of ACLF patients, images were processed using the FreeSurfer longitudinal pipeline [ 31 ] ( http://surfer.nmr.mgh.harvard.edu/fswiki/LongitudinalProcessing ). All subjects ’  processed data were manually evaluated to ensure no brain areas were excluded. Similarly, gray, white, and pial boundaries were visually assessed, and if needed, edits were made to correct misidentified regions. In most subjects, only minor edits were required to remove non-brain areas after automatically detected skull strip procedures.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26442748">PMID:26442748</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image preprocessing was performed as described previously (Menzler et   al.  2011 ). Diffusion volumes were motion-corrected and averaged using FLIRT with mutual information cost function to register each direction to the minimally eddy current distorted T2-weighted b0 DTI volume that had no diffusion weighting. Eigenvalues ( λ 1 ,  λ 2 ,  λ 3 ) and eigenvectors of the diffusion tensor matrix for each voxel were computed from the DTI volumes for each subject on a voxel-by-voxel basis using conventional reconstruction methods. These tools are included in the FreeSurfer package (FreeSurfer version 4.2.0;  http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26441498">PMID:26441498</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To reduce the complexity of the Allen data, we have summarized the data into the Desikan – Killiany cortical atlas built into the FreeSurfer software for automatic labeling of regions of interest (Desikan et al.,  2006 ). FreeSurfer allows one to segment magnetic resonance images (MRIs) into 68 cortical regions and to estimate their cortical thickness, surface area and volume ( http://surfer.nmr.mgh.harvard.edu/ ). These cortical measurements are stable and agree with past histological studies (Fischl and Dale,  2000 ; Han et al.,  2006 ; Scholtens et al.,  2015 ). FreeSurfer has been used widely, with over 600 reports covering a broad range of topics that include brain development, aging and a variety of brain disorders (PubMed, May 2015). Adding the perspective of gene expression could facilitate interpretation of these reports. For example, we recently found that the magnitude of group differences in cortical thickness between cannabis users and non-users correlates with regional variations in expression levels of cannabis receptor 1 ( CNR1 ) (French et al.,  2015 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26388497">PMID:26388497</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation of the MRI images was performed in a standard manner using Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ). The processing included motion correction, removal of non-brain tissue using a hybrid watershed/surface deformation procedure 26 , automated Talairach transformation, and segmentation of the subcortical white matter and deep gray matter volumetric structures 27 28 . We analyzed the volumes of 29 subcortical structures ( Table 1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26379232">PMID:26379232</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>When realizing M/EEG source connectivity, the cortical surface file and its corresponding scout file are required. The surface file contains the cortical mesh and the scout file contains the labels of the brain regions in case of using specific atlas. The cortical parcellation provides the ROIs that are used as network nodes in EEGNET. The surface files store geometric information about the morphology of the cortex. It can be created using the open source imaging analysis tool FreeSurfer  http://surfer.nmr.mgh.harvard.edu/ . The surface file can be also checked using brainstorm  http://neuroimage.usc.edu/brainstorm/ . The identification of the ROIs in the matrices is determined by a scout file which is a.mat file (can be generated also using Brainstorm). This file contains the labels of all the ROIs based on the already used atlas for segmentation such as Desikan [ 45 ] and Destrieux [ 46 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26346648">PMID:26346648</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>It is important to notice that diagnostic and prognostic accuracy of imaging biomarkers is at least as dependent on how the biomarker is measured as it is on the type of biomarker itself. There is a wide range of metrics, from visual analysis to quantitative analysis based on manual segmentation or automatic segmentation. Visual reading or manual regions of interest (ROIs) tracing induces interobserver variability. FreeSurfer is a free automated structural MRI image processing software providing many anatomical measures, including gray matter volume ( http://surfer.nmr.mgh.harvard.edu/ ) [ 8 ], and already used to measure atrophy in patients with AD or mild cognitive impairment (MCI) [ 9 ,  10 ]. It has also been used to analyze and quantify PET data [ 11 ,  12 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26341662">PMID:26341662</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Based on the initial positive findings from our dorsal surface analysis, we completed a more comprehensive cortical reconstruction and volumetric segmentation of the MRI images using the semi-automated FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ) [Reviewed in  20 ]. Measurements were made of parcellated volumes of more than 360 cortical and subcortical brain structures, including cortical gray matter, white matter, gyri and sulci. To control for overall brain shrinkage generally observed in brains of subjects with AUDs, volumes were normalized relative to the whole brain volume. All volumetric comparisons made in this study were between normalized whole brain ratios of specific regions.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26332788">PMID:26332788</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray matter ROIs were obtained from an automatic cortical and subcortical segmentation of the T1-weighted images employing FreeSurfer [ 16 ,  17 ] ( http://surfer.nmr.mgh.harvard.edu/ ) with parcellation into anatomical ROIs according to the Lausanne anatomical atlas, distributed as part of the Connectome Mapping Toolkit ( http://www.connectome.ch ), yielding 83 ROIs in the subjects ’  native T1-weighted space (a total of 42 regions with all regions existing in both hemispheres except for brainstem (1 ROI). Essentially 41 regions are in both hemispheres.) ( S1 Table ). The ROIS were transformed from the native T1 space into each subject ’ s DTI space using an affine transformation obtained with FSL ’ s FLIRT.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26331844">PMID:26331844</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation of each subject ’ s T1 image to produce an ROI atlas was performed using the FreeSurfer image analysis suite [ 17 ], available online at  http://surfer.nmr.mgh.harvard.edu . The atlas was registered with 12 degrees of freedom to both the DWI ’ s b0 image and a mean fMRI, also using FSL ’ s FLIRT algorithm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26301716">PMID:26301716</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The longitudinal 3D approaches include the adaptation of popular 3D/cross-sectional methodswith some longitudinal constraints or longitudinal pre-processing. For instance, in the context of clinical evaluation over a few years where anatomical changes are small and continuous, the use of 3D individual template targets have been proposed to perform non-linear registration [ 15 – 17 ] or tensor-based analyses (TBM) [ 18 ]. Indeed, to compare anatomical differences, 3D population templates have proven their importance for different applications such as mapping function, structure, or vasculature [ 19 ] and group comparisons [ 20 ]. While different techniques exist to create unbiased population templates for multi-subject cross-sectional studies [ 21 ,  22 ], few of these techniques have been developed for the creation of an individual 3D subject template. More recently, Reuter et al. [ 16 ] created a subject-specific 3D template for longitudinal analysis by computing the median image of the linearly registered images of the same subject from different time-points and this method is implemented within the longitudinal version of FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) [ 16 ]. In the continuity of their work on voxel-based morphometry (VBM) [ 23 – 25 ], Ashburner et al. [ 17 ] presented an unbiased  “ group-wise intra-subject ”  template with an iterative longitudinal non-uniformity correction, linear and non-linear diffeormorphic registration that is implemented in the Statistical Parametric Mapping 12 (SPM12) ( http://www.fil.ion.ucl.ac.uk/spm ). Aubert-Broche et al. [ 26 ] also proposed to use robust non-linear individual templates to perform tissue classification and segmentation of pediatric images.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26295459">PMID:26295459</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We use a patient ’ s brain MRI to construct the corresponding brain surface. First, we obtain clinical neuroimaging Digital Imaging and Communications in Medicine (DICOM) images using K-Pacs ( http://www.k-pacs.net/ ), a DICOM visualization and downloading software ( Table 1 : step A1). Often, DICOMs can be obtained directly from the radiology department. We preview the images to ensure quality and resolution of the necessary sequences (T1-weighted 1 mm isotropic MPRAGE images) ( Fig 1A ). We convert the DICOM images into a model of the brain's pial surface using the recon-all function in FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ , free and open-source) (step A2). This function generates a pial surface for each hemisphere automatically and takes 3 – 5 hours (mean 3.57 hr, standard deviation 1.24 hr, n = 32 subjects) on a desktop computer (Dell XPS 8700, CPU Core i7-4770 @ 3.4 GHz, 12 GB RAM, GeForce GTX 645, running Debian Jessie [ 21 ]) using the GPU assisted version of recon-all [ 22 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26286324">PMID:26286324</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation will be performed using the MPRAGE images with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ). Briefly, non-brain tissue will be removed using a hybrid watershed/surface deformation procedure [ 157 ]. Subcortical white matter and deep grey matter volumetric structures will be segmented automatically and intensity inhomogeneity of the images will be corrected [ 158 ,  159 ]. The cerebral cortex will be parcellated into 34 units per hemisphere based on gyral and sulcal structure [ 160 ,  161 ]. In addition to cortical regions, the left and right thalamus, left and right cerebellum and brain stem will be extracted. The accuracy of the cortical and subcortical parcellation will be assessed visually. An axial slice will be manually defined below the inferior most slice on which the pons was visible to include only the most caudal part of the brain stem.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26280173">PMID:26280173</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The original scans from both time points were visually inspected to assure good quality before the segmentation. For volumetric analyses, the baseline images were reprocessed, so that both baseline and follow-up images of patients and controls were processed with the same software version, Freesurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu ). To extract reliable volume estimates, images were automatically processed with the longitudinal stream in Freesurfer [ 22 ]. The processing steps included registration of the scans to a common atlas, ensuring that minor differences in head positioning in the scanner would not affect the results. An unbiased within-subject template was created using robust, inverse consistent registration [ 23 ]. Several processing steps, such as skull stripping, Talairach transformations, atlas registration, spherical surface maps and segmentations were then initialized with common information from the within-subject template, in order to increase reliability and statistical power [ 22 ]. After segmentation, one scan was discarded because of obvious segmentation mistakes. Data from the further scans were transferred to SPSS for statistical analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26267665">PMID:26267665</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed at T0 and at T5 on a volumetric T1-weighted data set by means of the  longitudinal stream  included in the  Freesurfer image analysis suite  (release v5.3.0), available online ( http : //surfer . nmr . mgh . harvard . edu/ ) . The technical details of these procedures have been described previously [ 23 ]. Topological defects in cortical surfaces due to white matter hypointensities were detected and manually corrected to have an accurate cortical segmentation. Since no significant differences were observed between right and left hemisphere, we decided to average the measures from both hemispheres [ 4 , 12 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26267069">PMID:26267069</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We measured the connectivity patterns of the regional mean time series in the resting-state fMRI data using six different seed regions-of-interests (ROI) located in the STG. Freesurfer image analysis ( http://surfer.nmr.mgh.harvard.edu/ ) was used to parcellate the STG into six different ROIs in both hemispheres: HG, PT, and PP ( S1 Fig ). The six different ROIs of the STG were defined using the Destrieux atlas from the Freesurfer image analysis suite; thus, the seed locations were independent of our data [ 17 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26252714">PMID:26252714</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>WML were outlined on a computer using a custom written IDL program (Exelis Visual Information Solutions, USA; see  Fig 1 ). Lesion areas were segmented by combined region growing and local thresholding following manual selection by a single instructed rater [ 52 ]. To counteract potential inter-rater variability, 15 cases from the study cohort were randomly chosen and re-rated by the instructor, a neurologist who is highly experienced in identification, volumetry and rating of WML. The neurologist was blinded to previous results and outlined WML on FLAIR scans of the 15 cases using the same procedure as the main rater. WML volumes of both raters were entered into SPSS. Subsequent intraclass correlation analysis yielded an intraclass correlation coefficient (ICC (2, 1)) of 0.938 (95%CI 0.570 – 0.984; p  <  .001). An ICC of 0.938 indicates high agreement and supports the reliability of WML volumetry between the two raters. Lesion volume in mm3 was calculated using the program FSLMATHS (FSL, Oxford,  www.fmrib.ox.ac.uk ) by multiplying the lesion area with the slice thickness and normalized by total intracranial volume (TIV). Segmentation of TIV and cortex volume from the T1-weighted high resolution MPRAGE scans was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ) [ 53 ,  54 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26236626">PMID:26236626</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>84 cortical and subcortical regions with anatomical landmarks were mapped from MPRAGE image using Freesurfer 5.0 software ( http://surfer.nmr.mgh.harvard.edu ). These regions of interest (ROIs) are then co-registered to the diffusion image space using a nonlinear registration tool of FSL (FNIRT) ( http://fsl.fmrib.ox.ac.uk ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25895507">PMID:25895507</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Thirteen patients with PCA and seven patients with typical Alzheimer ’ s disease who completed the eye-tracking assessments also had volumetric MRI available. Scans from 25 healthy controls were used for comparison (these are separate from the eye tracking controls, as those participants did not receive scans). T 1 -weighted volumetric MRI brain scans were acquired on a 3.0 T Siemens TIM Trio scanner using a magnetization prepared rapid gradient echo sequence with a 28.2 cm field-of-view to provide 208 contiguous 1.1 mm thick slices. Cortical thickness measurements were made using FreeSurfer version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ). The detailed procedure has been described and validated in previous publications ( Dale  et al. , 1999 ;  Fischl and Dale, 2000 ). Two modifications to the standard FreeSurfer processing stream were made: a locally generated brain mask was used to improve skull stripping, and FreeSurfer ventricular segmentations were added to the white matter mask to improve cortical segmentation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25893216">PMID:25893216</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To determine whether patients differed from controls in cortical thickness, SBA was performed. The cortical thickness measure was obtained from T1-weighted images for each participant using FreeSurfer (version 5.3; http://surfer.nmr.mgh.harvard.edu) ( Dale et al., 1999 ;  Fischl et al., 1999 ;  Fischl and Dale, 2000 ). The processing pipeline consisted of affine registration of the T1-weighted volume to Talairach space, skull stripping, followed by WM segmentation and tessellation of the gray/white matter boundary. At each step, visual inspection and manual correction of topological errors was performed. Following cortical reconstruction, brains were inflated, averaged across participants, and smoothed using a 10 mm FWHM Gaussian kernel. The cortical surface was then parcellated into 34 distinct regions (Desikan − Killiany atlas) for each hemisphere from which mean cortical thickness was derived ( Desikan et al., 2006 ). A direct measure of cortical thickness was calculated by FreeSurfer using the distance (mm) between the pial and gray − white matter boundary at each vertex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25829180">PMID:25829180</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the FreeSurfer image analysis suite (version 4.5,  http://surfer.nmr.mgh.harvard.edu ) to derive measures of cortical thickness ( Fischl and Dale, 2000 ), subcortical volume ( Fischl et al., 2002 ), and total intracranial volume ( Buckner et al., 2004 ). The technical details of these processing pipelines have been previously described in detail ( Dale et al., 1999 ;  Fischl et al., 1999 ). The cortical reconstructions for each participant were visually inspected for inaccuracies in segmentation and manually corrected as necessary by a single rater (J.L.P.) blind to subject identity, diagnostic group, and time point. To measure changes in cortical thickness and volume in patients between baseline and follow-up scans, images were processed using the FreeSurfer longitudinal pipeline ( Reuter et al., 2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25480358">PMID:25480358</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional and anatomical data from both human and monkey subjects were preprocessed and analyzed using FreeSurfer and FS-FAST (version 5.3;  http://surfer.nmr.mgh.harvard.edu/  ( Fischl 2012 ). For each human and monkey subject, we reconstructed the cortical surface based on the high-resolution anatomical data ( Fischl et al. 1999 ). All functional images were corrected for motion artifact and then spatially smoothed using a 3D Gaussian kernel (2.5-mm HWHM in humans and 1-mm HWHM in monkeys), and normalized across scans. To estimate the intensity of the hemodynamic response, a model based on a  γ  function was fit to the fMRI signal, and then, the average signal intensity maps were calculated for each condition ( Friston et al. 1999 ). Voxel-wise statistical tests were conducted by computing contrasts based on a univariate general linear model. Finally, the significant levels were projected onto the inflated/flattened cortex after a rigid co-registration of functional and anatomical volumes.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25452574">PMID:25452574</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The location of the cortical current sources cannot be precisely determined using the measured magnetic fields from outside the head. Here, we estimate the location of these sources using the neuroanatomically constrained minimum norm estimate (MNE) procedure, based on distributed source modeling rather than equivalent current dipoles ( H ä m ä l ä inen et al. 1993 ). Since most of the electromagnetic signals originate from postsynaptic currents in the apical dendrites of cortical pyramidal cells in the cortex, the orientation of these currents is tangential to the cortical mantle. MNE therefore computes the inverse solution taking into account the individual anatomical information provided by a structural magnetic resonance imaging (MRI) scan of each participant, using the boundary between gray and white matter for its algorithm. A complete overview of the MNE suite for EMEG source estimation can be found in  Gramfort et al. (2014) . For each participant, MRI images were obtained using a GRAPPA 3D magnetization prepared rapid acquisition gradient echo (MPRAGE) T1-weighted scans (time repetition = 2250 ms; time echo = 2.99 ms; flip angle = 9 ° ; acceleration factor = 2) on a 3-T Trio (Siemens, Erlangen, Germany) with 1 mm isotropic voxels. From the MRI data, a representation of the cerebral cortex was constructed using the FreeSurfer program ( http://surfer.nmr.mgh.harvard.edu/ ) to separate the scalp, skull, and brain. The forward model was calculated with a three-layer boundary element model using the outer surface of the scalp as well as the outer and inner surfaces of the skull identified in the anatomical MRI with different electrical conductivities for each compartment ( Gramfort et al. 2014 ). The interface between the gray and white matter representation was downsampled to yield a source space of 10 242 vertices per hemisphere that was used as the location of the dipoles (average spacing of 3.1 mm between dipoles). Anatomically constrained activation movies were created by combining MRI, MEG, and EEG data, providing a better source localization than MEG or EEG does independently ( Liu et al. 2002 ). Fusing all modalities increases the conditional precision of the underlying source estimates relative to that obtained by inverting magnetometers, gradiometers, or EEG alone ( Henson et al. 2009 ). To visualize activation across subjects, the cortical surface representations of individual subjects were aligned using a spherical morphing technique ( Fischl, Sereno, Tootell, et al. 1999 ) and inflated ( Fischl, Sereno, Dale 1999 ). We employed depth weighting to correct for MNE's bias toward attributing signals to superficial sources and a loose-orientation constraint (0.2), as recommended by  Lin et al. (2006 ), to improve the spatial accuracy of localization. Sensitivity to neural sources was improved by calculating a noise covariance matrix based on the 100-ms prestimulus period. The activations at each location of the cortical surface were estimated over 1 ms windows, resulting in spatiotemporal brain activation movies.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24927720">PMID:24927720</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The bilateral BA10 were determined using a cortical surface parcellation atlas included in the FreeSurfer software package version 5.0.0 ( http://surfer.nmr.mgh.harvard.edu/ ). We used the PALS ( http://brainvis.wustl.edu/wiki/index.php/Caret:Atlases ; Van Essen  2005 ) Brodmann area parcellation atlas to define a mask for BA10 for each hemisphere in FreeSurfer standard space (fsaverage). Importantly, the masks were defined on the surface of white matter, i.e. on the border of white and grey matter in the brain (see Fig.   1 ). These masks (one for each hemisphere) were then transformed to FMRIB Software Library (FSL version 4.1.2;  http://www.fmrib.ox.ac.uk/fsl/ ; Smith et al.  2004 ) standard space (MNI152) for analysis (see Fig.   1 ). The BA10 mask was transformed to individual space using the linear registration tool (FLIRT) implemented in FSL, using 12 degrees of freedom (Jenkinson et al.  2002 ), and visually checked for aberrant registration. All registrations were satisfactory upon visual inspection.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26606247">PMID:26606247</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We analyzed GM by looking at the cortical volume/thickness as extracted from FreeSurfer (v. 5.3) [ 24 ] ( http://surfer.nmr.mgh.harvard.edu/ ). We extracted 34 suspected and primarily frontal and temporal ROIs per hemisphere, from the Desikan-Killiany atlas [ 25 ] from all native space T1-weighted structural MRI scans ( Table 1 ). For each cortical label, or ROI, we extracted the GM volumes as output by FreeSurfer. We manually adjusted the WM/GM boundaries to obtain most reliable GM volume/thickness measures. We also computed the overall brain volume (i.e., brain size) for each subject and used it as a covariate in the statistical analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26605231">PMID:26605231</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI scans were processed using the Free-Surfer software package, available at  http://surfer.nmr.mgh.harvard.edu . Multiple MP-RAGE MRI acquisitions for each participant were motion corrected, averaged and normalized to create a single image volume with relatively high contrast to noise. This averaged volume was used to locate the grey/white matter boundary (white matter surface) and this, in turn, was then used to locate the grey/cerebrospinal fluid boundary (grey matter surface).[ 26 ]</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26590556">PMID:26590556</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and subcortical volumetric segmentation were performed using the FreeSurfer imaging analysis suite (v5.1.0,  http://surfer.nmr.mgh.harvard.edu/ ) [ 39 ]. Details of this pipeline are fully described on its webpage. Briefly, the T1-weighted images were firstly registered to the Talairach space of each participant ’ s brain with the skulls stripped. Images were then segmented into white/grey matter (WM/GM) tissues based on local and neighbouring intensities. The cortical surface of each hemisphere was inflated to an average spherical surface to locate both the grey matter (pial) surface and the WM/GM boundary. Preprocessed images were visually inspected (by GF and YD) to ensure the reconstruction and segmentation qualities. Any topological defects were excluded from the subsequent analyses but no data had to be excluded at this point. At the cortical level, cortical thickness was measured as the shortest distance between the pial surface and the GM/WM boundary at each point across the cortical mantle. Surface area was measured as the area of a vertex on the pial surface, calculated as the average of the tessellation areas touching that vertex. In addition, the cerebral cortex of each participant was automatically parcellated into 70 regions according to the Desikan-Killiany cortical atlas [ 40 ], with their mean cortical thickness and surface area calculated for the ROI analysis. Before group-level statistical analyses, individual cortical surface maps were smoothed with a Gaussian 25   mm full-width-at-half-maximum (FWHM) kernel when accounting for the sample size. At the subcortical level, volumes of a series of subcortical structures were extracted using the automated segmentation function in FreeSurfer [ 41 ] for the ROI analysis. Total intracranial volume (ICV) of each participant was also extracted.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26573408">PMID:26573408</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Measures of cortical folding across the brain were obtained by calculating the local gyrification index for each MRI structural scan using the method described by Schaer  et al . 30 . The gyrification index gives a measure of cortical folding by comparing the amount of cortex buried within sulcal folds at the grey/white matter interface with the amount of visible or surface cortex at each vertex of the reconstructed brain surface, based on 3D spheres of radius 25   mm. Calculations made at each vertex can then be averaged to give a global gyrification index for each hemisphere, or for 34 individual brain regions as defined by an automated parcellation procedure 31 . Calculation was undertaken using FreeSurfer Software version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ), which also provided measures of intracranial volume for each subject's scan. For statistical analysis, individual gyrification maps were registered to a template constructed by averaging the patient structural scans included in the study, and smoothed with a 5   mm full-width, half-maximum isotropic Gaussian kernel. Group differences in local gyrification were analysed by fitting a general linear model at each vertex on the surface. Age was included as a covariate, and non-parametric cluster-wise correction for multiple comparisons was performed using Monte Carlo simulation 32 , with a threshold of  P < 0.05, corrected for multiple comparisons across each hemisphere. Two-sample  t -tests were used to identify region-specific differences averaged across  a priori  mPFC regions of interest in the vicinity of the PCS, the frontopolar, medial orbitofrontal, superior frontal and paracentral cortices 31 , at a threshold of  P < 0.05. Regional differences outside the region of interest were reported if they exceeded a threshold of  P < 0.05, corrected for multiple comparisons across the 34 parcellated brain regions.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26573340">PMID:26573340</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For each animal we performed a first-level, fixed-effects General Linear Model analysis (FEAT, FSL) contrasting BOLD responses to violation versus consistent sequences. This involved motion correction and registration of the functional images to the macaque's own high resolution anatomical scan, via intermediate scans including a lower resolution anatomical scan that was in register with the functional scans. The results of the first-level GLM analyses were then registered to a standard macaque template brain 69 , which is in register with an accepted macaque atlas in stereotactic coordinates, to determine the anatomical fields within which the significant activity clusters occurred 40 . The results were then combined into higher level analyses, evaluated at the cluster corrected ( P < 0.05) level (2   mm smoothing full-width half maximum, FWHM). The results were then registered to a FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) surface-based representation of a standard template monkey brain 69 . For display purposes, a majority consensus map was used to summarize the macaque whole-brain results ( Fig. 2d ). This was created by identifying common voxels which were significantly activated ( P < 0.05 cluster corrected) in at least two of the three animals and plotting the average  z -score values on the rendered surfaces.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26557052">PMID:26557052</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>These measurements yield three thickness parameters as shown in Figure  1 : the total cortical thickness,  T , the thickness of the unmyelinated layers,  G , and the thickness of the myelinated layers,  M . One may also calculate the proportional myelinated thickness of the cortex as:\n (1) P = M T \nwhich summarizes the proportion of the cortical depth that is myelinated. The approach is similar to that used to measure total cortical thickness in standard software packages such as Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ), except that it is considering three tissue classes in the cerebrum (GM, mGM, and WM), rather than two (GM and WM). This is only appropriate in the case that the input MRI image has enough intracortical contrast to allow segmentation into three classes.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26539075">PMID:26539075</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The preprocessed images were imported into the automated brain segmentation software FreeSurfer (FS) installed on a Dell Optiplex machine using the Ubuntu 12.04 operating system. In addition to resampling of the image into 0.9375 mm 3  using a cubic spline transformation during preprocessing as described above, the FS segmentation process resampled the images into 1 mm 3  as part of its motion correction step. Cortical reconstruction and volumetric segmentation was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl et al.,  1999a , b ,  2001 ,  2002 ,  2004a , b ; Fischl and Dale,  2000 ; S é gonne et al.,  2004 ; Han et al.,  2006 ; Jovicich et al.,  2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26206860">PMID:26206860</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural magnetic resonance imaging scans obtained from the participants were processed using Freesurfer (5.1.0) ( http://surfer.nmr.mgh.harvard.edu/ ) as previously described. 10  Reconstructed surfaces were inspected for topological defects and edited in accordance with our previous work 11  by a single rater (L.P.) masked to the severity status at the time of surface editing. Cortical folding was measured using local gyrification index proposed by Schaer  et al . 12  Cortical thickness was estimated using the standard procedures described by Fischl  &  Dale. 13  The reconstructed brain surfaces were parcellated using the Destrieux atlas to provide 148 brain regions based on sulcogyral boundaries described by Duvernoy. 14  For each metric, these 148 values were used as features in the classifier.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25821911">PMID:25821911</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each site administered a standardized structural MRI protocol. (See  Supplementary Table 4  for scan parameters) Pre- and post-processing techniques have been described previously 45 . Briefly, high-resolution structural MRI included a 3-D T1-weighted scan, a T2-weighted volume, and diffusion-weighted scans with multiple b-values and 30-directions. Image analyses were performed using a modified Freesurfer software suite ( http://surfer.nmr.mgh.harvard.edu/ ) to obtain measures of cortical and subcortical volume regions of interest (ROIs), and vertex-wise cortical thickness and surface area 46 . All neuroimaging data passed a standardized quality-image check.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26702432">PMID:26702432</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>With the FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) and MNE software packages, the MRI data were used to construct individualized, cortically constrained source spaces and boundary element models (BEMs). The high-resolution MPRAGE data were used to create three-dimensional (3D) reconstructions of the white and pial surface of each participant using FreeSurfer ( 70 ,  71 ). The resulting white matter surface was decimated (to about 4100 dipole locations per hemisphere) to create the MEG source space. The resulting dipoles represent, on average, a cortical surface area of about 24 mm 2 , with an intersource spacing of about 5 mm. The FLASH images were used to produce high-resolution inner-skull, outer-skull, and scalp surfaces. These, in turn, were decimated and used to construct participant-specific three-layer BEMs. The MEG and MRI coordinate frames were aligned by manually demarking the fiducial points on the high-resolution scalp surface. Refinement of this initial 3D affine registration was carried out using the iterative closest-point algorithm ( 72 ) as implemented by the MNE suite.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26696839">PMID:26696839</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26693400">PMID:26693400</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image files from all sites were transferred to a Linux workstation for morphometric analysis. Images from the primary sample were further processed with the FreeSurfer5.1 software package ( http://surfer.nmr.mgh.harvard.edu ). Images from the secondary multi-site sample were processed with FreeSurfer 5.3. Processing steps include correction for spatial distortion due to gradient nonlinearity ( Jovicich et al., 2006 ) and B1 field inhomogeneity ( Sled et al., 1998 ). Assignment of a neuroanatomical label to each voxel is based on a manually labeled training set and Bayesian prior information ( Fischl et al., 2002 ). Labeling is performed by rigid-body alignment of the subject's brain to the probabilistic atlas, followed by non-linear morphing to the atlas. Labels are generated based on the prior probability of a given tissue class occurring at a specific atlas location, the likelihood of the image intensity given that tissue class, and the probability of the local spatial configuration of labels given the tissue class. For the WMH measure, the spatial prior is relaxed to allow for detection of darker voxels anywhere in the white matter. These darker voxels are labeled WMH and summed to quantify total WMH volume. Total supratentorial brain volume (TSBV) is calculated by summation of all surface-based volumes and labeled voxels inside the pial surface plus any structures that might fall partially or totally outside of the pial surface (e.g., hippocampus, amygdala, corpus callosum, ventral DC, thalamus, ventricles, and choroid plexus), excluding the cerebellum and brain stem.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26650042">PMID:26650042</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To facilitate more regionalized analysis of total brain, WM and GM volumes, individual lobes have been segmented. This is achieved by non-rigidly registering a template image in which the lobes have been manually outlined [ 35 – 37 ]. Finally, T1-weighted MR images have been used to calculate subcortical structures and thickness of the cerebral cortex using a model-based automated procedure using Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) [ 38 ,  39 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26618360">PMID:26618360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Baseline T1-weighted images obtained using a standardized 1.5 Tesla MRI protocol ( http://adni.loni.usc.edu/methods/mri-analysis/mri-acquisition/ ) of the ADNI-1 study were used. Details of the MRI acquisition and processing method have been previously described [ 31 ]. All T1-weighted MR images used in this study were automatically segmented using FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) to perform regional volumetric measurements on the hippocampus, intracranium and cerebral cortices based on the Desikan-Killiany atlas [ 32 ]. After visual inspection and manual correction of segmentation errors, 5 CN and 8 AD cases were excluded because of poor-quality of segmentation. As such, 222 CN and 163 AD subjects were included in the present study.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26173861">PMID:26173861</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation of MRI data was performed using the Freesurfer 5.3 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) using standard methods ( Fischl  et al. , 1999 ;  Fischl and Dale, 2000 ). The initial processing of T 1  MRI images, for each subject and each time point (baseline and follow-up), includes the following steps: removal of non-brain tissue, automated Talairach transformation, segmentation of the subcortical white matter and deep grey matter volumetric structures, intensity normalization, tessellation of the grey matter/white matter boundary, automated topology correction and surface deformation to optimally place the grey matter/white matter and grey matter/CSF boundaries. The cortical thickness was calculated as the closest distance from the grey/white matter boundary to the grey/CSF boundary at each vertex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26834623">PMID:26834623</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer version 5.3 was used to extract surface area and cortical thickness. This image processing suite provides automated parcellation of both cortical and subcortical brain structures. Details of the procedure are documented elsewhere (Fischl et al.,  2002 ) and are freely available for download ( http://surfer.nmr.mgh.harvard.edu/ ). In brief, processing included motion correction and averaging of T1 weighted images (Reuter et al.,  2010 ), skull stripping using a hybrid watershed/surface deformation procedure (S é gonne et al.,  2004 ), automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (Fischl et al.,  2002 ,  2004a ), intensity inhomogeneity correction (Sled et al.,  1998 ), gray/white matter boundary tessellation, topology correction (Fischl et al.,  2001 ; S é gonne et al.,  2007 ), and surface deformation following intensity gradients to ensure optimal gray/white and gray/cerebrospinal fluid boarder (Dale and Sereno,  1993 ; Sled et al.,  1998 ; Dale et al.,  1999 ; Fischl and Dale,  2000 ; Fischl et al.,  2004a ; S é gonne et al.,  2004 ,  2007 ). Cortical regions were parcellated into specific regions with respect to sulcal and gyral structures (Fischl et al.,  2004b ; Desikan et al.,  2006 ), with manual inspection for errors in the automatic program by one of two trained raters.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26816612">PMID:26816612</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We employed a standard processing pipeline of the widely-employed, freely-available FreeSurfer image analysis suite [ 35 ] ( http://surfer.nmr.mgh.harvard.edu/ ), which uses surface-based morphometry (SBM), as opposed to voxel-based morphometry (VBM), that reconstructs and characterizes key information about brain structure. It provides better alignment of cortical landmarks than volume-based registration and does not result in an age-associated bias between older and younger children when registering children ’ s brains to a common space [ 36 ]. These qualities make FreeSurfer an ideal tool for accurately comparing the brain anatomy of pre-adolescent children.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26789126">PMID:26789126</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>SBM analyses of cortical thickness, mean curvature, surface area and grey matter volume were performed using the Freesurfer image analysis suite (version 5.3.0, available at:  http://surfer.nmr.mgh.harvard.edu/ ). The processing includes removal of non-brain tissue,[ 18 ] automated Talairach transformation, intensity normalization,[ 19 ] tessellation of the grey/white and grey/cerebrospinal fluid boundaries and automatic correction of topologic inaccuracies,[ 20 , 21 ] surface deformation and inflation,[ 22 , 23 ] registration to a spherical atlas[ 24 ] and automatic parcellation of the cortex surface based on gyral and sulcal structures.[ 25 , 26 ] The reconstruction process resulted in of a variety of surface-based data, such as cortical thickness, mean curvature, surface area measurement and grey matter volume.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26766957">PMID:26766957</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness, defined as the shortest distance between the gray/white matter boundary and the spiral surface at each point across the cortical mantle, was automatically estimated by FreeSurfer (software version 5.0,  http://surfer.nmr.mgh.harvard.edu ). 19  The technical details of these procedures have been described elsewhere. 20  Briefly, the process consists of correction for motion artifact, averaging over multiple T1 images, removal of non-brain tissue, Talairach transformation, white matter and subcortical gray matter segmentation, tessellation of the gray matter-white matter boundary, automatic correction for topological defects, intensity normalization, surface deformation, and parcellation of the cerebral cortex into anatomical regions. Total intracranial volume (TIV) was also automatically calculated using FreeSurfer software. 21  Through careful inspection of all raw images at segmented and inflated stages, we confirmed that none of them exhibited substantial defects. For further analysis, cortical maps were smoothed using a Gaussian kernel with a full width at a half maximum of 10 mm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26759784">PMID:26759784</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness evaluation was performed with the Freesurfer image analysis suite, which is documented and freely available online ( http://surfer.nmr.mgh.harvard.edu ). It is a set of automated processing procedures, requiring 3D MPRAGE MRI images and comprising skull-stripping, registration, intensity normalisation, Talairach transformation, tissue segmentation and surface parcellation. Further information and technical details of the procedures are described in prior publications ( Fischl et al., 1999 ). To detect possible misclassifications of white and grey matter, e.g. due to multiple sclerosis lesions, all images were visually inspected after the white/grey matter segmentation. In two patients, a semi-automated correction of topological defects was required by manually adding control points.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26759553">PMID:26759553</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In this paper we introduce the MRBrainS challenge evaluation framework ( http://mrbrains13.isi.uu.nl/ ), an online framework to evaluate automatic and semiautomatic algorithms that segment gray matter (GM), white matter (WM), and cerebrospinal fluid (CSF) in 3T brain MRI scans of older (mean age 71) subjects with varying degrees of atrophy and white matter lesions. This framework has three main advantages. Firstly, researchers apply their own segmentation algorithms to the provided data. Parameters are optimally tuned to achieve the best possible performance. Secondly, all algorithms are applied to the exact same data and the reference standard of the test data is unknown to the participating researchers. Thirdly, the evaluation algorithm and measures are the same for all evaluated algorithms, enabling direct comparison of the various algorithms. The framework was launched at the MRBrainS13 challenge workshop at the Medical Image Computing and Computer Assisted Intervention (MICCAI) conference on September 26th in 2013. Eleven teams participated in the challenge workshop with a wide variety of segmentation algorithms, the results for which are presented in this paper and provide a benchmark for the proposed evaluation framework. In addition, we evaluated three commonly used freeware packages on the evaluation framework: FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) [ 23 ,  24 ], FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ) [ 25 ], and SPM ( http://www.fil.ion.ucl.ac.uk/spm/ ) [ 26 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26743811">PMID:26743811</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We compared the vertex-based cortical thickness to implement the surface-based analysis. The cortical thickness was calculated by FreeSurfer software version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ). The image process included non-uniform intensity correction, skull-strip of the brain 20 , construction of the white surface and the pial surface 21  and cortical thickness calculation 22 . Finally, the vertex-based thickness of each subject was normalized to a template provided by FreeSurfer. We subsequently conducted two sample  t -tests to assess the thickness differences between two groups with age and gender as covariates (FWE corrected,  ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26656996">PMID:26656996</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each subject participated in 3 sets of scans across multiple sessions to collect anatomical scans, functional task data, and visuotopic mapping data. A high-resolution (1.0  ×  1.0  ×  1.3 mm) magnetization-prepared rapid gradient-echo sampling structural scan was acquired for each subject to computationally reconstruct the cortical surface of each hemisphere using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). For the functional auditory task data,  T 2 *-weighted gradient-echo, echo-planar images were collected using forty-two 3-mm slices (no skip), oriented axially (time echo 30 ms, time repetition 2600 ms, in-plane resolution 3.125  ×  3.125 mm). Imaging for all auditory data was performed at the Center for Brain Science at Harvard University on a 3-T Siemens Tim Trio scanner with a 32-channel matrix coil. For 6 subjects, structural and visuotopic mapping scans were collected on an identically equipped scanner (i.e., same model and same pulse sequences) at the Martinos Center for Biomedical Imaging at Massachusetts General Hospital.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26477650">PMID:26477650</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Fischl et al., 2002 ,  Fischl et al., 2004  and  Han and Fischl (2007)  take a different approach in the method that is used in the FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ). A probabilistic atlas, which is derived from a number of manual segmentations, is still used, but in this case it is used as a prior on a Markov random field (MRF). The spatial constraints imposed by the atlas allow the MRF to segment the image into the large number of classes needed to segment the subcortical structures. This idea is combined with a multi-atlas approach by  van der Lijn et al. (2008)  and  Wolz and Aljabar (2009) . A mesh-based spatial prior can also be used ( Puonti et al., 2013 ).  Khan et al. (2008)  propose to use the segmentation output from the method by  Fischl et al. (2002)  to drive registration of a secondary manually labelled template.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26397226">PMID:26397226</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction of the structural images was performed with the FreeSurfer software package, version 5.1 ( http://surfer.nmr.mgh.harvard.edu ). The procedures have been fully described elsewhere [ 38 ]. Estimated surfaces were inspected to detect errors in the automatic segmentation procedure. Fifty-seven of the 225 N3 processed MRI analyzed were excluded because of segmentation errors and 168 were included in the analyses. A Gaussian kernel of 15 mm full-width at half maximum was applied. To avoid false positives, we tested Monte Carlo simulation with 10,000 repeats in Qdec (family-wise error [FWE] ,  p   <  0.05). Only regions that survived FWE are presented in the figures.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25319752">PMID:25319752</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing steps were identically performed for all subjects with AFNI ( http://afni.nimh.nih.gov/afni/ ) by applying standard procedures that have been executed within a R software framework ( http://cran-r-project.or/ ) for automation purposes ( Boubela et al. 2012 ). Preprocessing included reconstruction, slice-timing correction, rigid-body motion correction, and alignment to the individual anatomical brain using a 12-point affine transformation. The first five volumes were removed to ensure that magnetization equilibrium was reached. For statistical reasons, the last seven volumes of adolescents ’  data were also removed to achieve identical trial length (175 TRs) in both samples. ANATICOR artifact regression analysis was applied to resting state time series to control for nuisance signals and localized transient hardware artifacts (see  http://afni.nimh.nih.gov/sscc/hjj/anaticor/ ) ( Jo et al. 2010 ). Motion parameters have been generated by the alignment procedure, whereas additional nuisance variables have been estimated from eroded white matter (WM) and cerebrospinal fluid (CSF) masks provided by FreeSurfer anatomical segmentation (processed using FreeSurfer software version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ) on a Linux system (Red Hat Enterprise Linux 5, x86_64 architecture) as described elsewhere ( Rabl et al. 2014 ). For temporal filtering a broad frequency band (0.008 – 0.15 Hz) was used, which has recently been found to yield the highest reliability in resting state fMRI analysis ( Braun et al. 2012 ). Moreover, studies demonstrated that higher frequencies contain meaningful information when proper noise regression is used ( Boubela et al. 2013 ). Eventually, data underwent a spatial Gaussian blur (full width at half maximum (FWHM) = 6 mm) followed by warping to Talairach – Tournoux stereotactic space and final calculation of functional connectivity maps.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26888383">PMID:26888383</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural imaging was obtained at a 3 Tesla Siemens scanner using a 32-channel head coil to obtain a T1 5 echo sagittal MPRAGE sequence [TE   =   1.64   ms; 3.5   ms; 5.36   ms; 7.22   ms; 9.08   ms; TR   =   2530   ms; voxel size   =   1.0   ×   1.0   ×   1.0   mm3; FOV   =   256   mm; slices   =   192; acquisition time   =   6:03]. For all scans, each T1 was reviewed for image quality. Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer-v5.3.0 image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The methodology for FreeSurfer is described in full in several papers, and summarized by Reuter 60 . Briefly, this process includes motion correction and averaging of volumetric T1 weighted images, removal of non-brain tissue, automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures, intensity normalization, tessellation of the gray matter, white matter boundary identification, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders. Segmented data were then parceled into units based on gyral and sulcal structure, resulting in values for cortical thickness, surface area, and volume. The results of the automatic segmentations were quality-controlled, and any errors were manually corrected. FreeSurfer morphometric procedures have been demonstrated to show good test-retest reliability across scanner manufacturers and across field strengths 60 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26885628">PMID:26885628</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu ) was used to perform gray and white matter segmentation and extraction of the midgray cortical surface that was used for source locations. We made separate models for each observer using the boundary element method, with boundary surfaces derived from the T1- and T2-weighted MRI scans of each observer (Jenkinson, Pechaud,  &  Smith,  2005 ; Smith,  2002 ) using the FMRIB Software Library ( http://fsl.fmrib.ox.ac.uk/fsl/ ). Forward and inverse electrical models were made with MNE ( http://martinos.org/mne/stable/index.html ) using a standard set of conductances (0.33, 0.025, and 0.33 S/m for the scalp, skull, and brain compartment, respectively). Retinotopic regions of interest (ROI) corresponding to areas in early visual cortical areas V1, V2v, V2d, V3v, V3d, V3a, and hV4 in each hemisphere (Engel, Glover,  &  Wandell,  1997 ; Huk  &  Heeger,  2002 ; Press, Brewer, Dougherty, Wade,  &  Wandell,  2001 ; Sereno et al.,  1995 ; Tootell et al.,  1997 ; Wade, Brewer, Rieger,  &  Wandell,  2002 ) were mapped using rotating wedges and expanding rings of contrast-reversing checkerboards (Brewer, Liu, Wade,  &  Wandell,  2005 ) created with VISTADISP ( http://web.stanford.edu/group/vista/cgi-bin/wiki/index.php/Software#Vistadisp ). Stimuli were presented on a liquid crystal display behind the bore, viewed through a mirror at a distance of 122 cm, providing a 14 °  diameter field of view. ROI corresponding to hMT+ were identified using low-contrast motion stimuli similar to those described by Huk and Heeger ( 2002 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26885405">PMID:26885405</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the Freesurfer Image Analysis Suite ( http://surfer.nmr.mgh.harvard.edu/ ) to parcellate the regions of interest and automatically calculate cortical thickness across the brain. The details of this procedure are described in previous publications. In brief, the steps include removal of nonbrain tissue [ 42 ], intensity normalization [ 43 ], tessellation of grey and white matter borders, automated topology correction [ 44 ,  45 ], surface deformation [ 46 ,  47 ], surface inflation [ 48 ], and registration to a spherical atlas [ 49 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26881120">PMID:26881120</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In each individual subject, a brain mask including the whole GM and WM of the brain was used as seed and target mask for the tractography in native space. The brain mask was obtained by performing a cortical reconstruction process of the two averaged T1-weighted images within each subject (FreeSurfer v. 5.3.0,  http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26876488">PMID:26876488</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness was defined as the shortest distance between gray/white matter boundary and the pial surface at each point across the cortical mantle. Cortical thickness was automatically estimated using FreeSurfer (software version 5.0,  http://surfer.nmr.mgh.harvard.edu ). The technical details of measuring cortical thickness and TIV using FreeSurfer have been described elsewhere 61 . We carefully inspected all raw images at segmented and inflated stages, and we confirmed that no images had substantial defects. For further analysis, cortical maps were smoothed using a Gaussian kernel with a full width at half maximum of 10   mm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26870631">PMID:26870631</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We generated individual structural brain networks for each of the forty-six subjects using a pipeline reported previously [ 14 ]. First, diffusion-weighted (DW) images were eddy current corrected using the automatic image registration (AIR) tool embedded in DtiStudio software ( http://www.mristudio.org ), by registering all DW images to their corresponding B 0  images with 12-parameter affine transformations. This was followed by computation of diffusion tensors and deterministic tractography using fiber assignment by a continuous tracking algorithm [ 15 ]. T1-weighted images were used to generate label maps using the Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26869913">PMID:26869913</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical geometry we used in this study has been reconstructed from a MRI scan with FreeSurfer image analysis suite, which is documented and freely available for download online at  http://surfer.nmr.mgh.harvard.edu/ . This processing includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure (S é gonne et al.,  2004 ), automated Talairach transformation, intensity normalization, tessellation of the gray/white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class. For further details, see Fischl ( 2012 ) and references therein.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26869873">PMID:26869873</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The whole analyses pipeline is shown in Figure  1  (lower row). The construction of the cortical surface was based on 3D-T1 images using Freesurfer version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu ). The detailed procedure for surface reconstruction has been described and validated in a previous study (Hanganu et al.,  2015 ). For connectivity matrix reconstruction we applied the same node definition as above. We extracted the individual cortical thickness values for each ROI from the AAL-Atlas. This data was used as above described for FA values for the construction of the structural correlation matrices.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26706997">PMID:26706997</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For each participant, the brain's cortical surface was reconstructed from structural MRI images processed using automated segmentation algorithms of the FreeSurfer software (Version 4.3;  http://surfer.nmr.mgh.harvard.edu/ ). The functional data analysis was accomplished using MNE software (Version 2.6;  http://www.nmr.mgh.harvard.edu ). The cortical surface was down-sampled to about 10,000 vertices, with an average distance between vertices of 5   mm. EEG and MEG sensor configurations and MRI images were coregistered based on the matching of scalp surface reconstructed from the MRI images, and about 50 – 100 digitized locations on the scalp surface prior to the recording session. Forward solutions for combined EEG and MEG data were computed using a boundary element model (BEM) based on a three-layer segmentation of the MRI image (inner skull, outer skull and skin surfaces, 5120 triangles), created using a watershed algorithm. The noise covariance matrices for each data set were calculated for baseline intervals of 200   msec before the beginning a trial. For regularization, the default method in MNE was used, i.e., a SNR value of 3 was specified.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26577652">PMID:26577652</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High resolution T1-weighted anatomic images were acquired on a GE 3T Excite MRI system. Automated reconstruction and volumetric segmentation of images were conducted using FreeSurfer 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ), which is widely used and provides accurate renderings of regional volumes without rater bias. We used an imaging protocol that maximizes differentiation of the white and gray matter boundary (3D spoiled gradient-recalled at steady state, TE 3.0 ms, TR 7.8 ms, TI 600, 8 °  flip angle, NEX 1, 1-mm axial slice thickness, FOV 25.6 cm, 256  ×  256 matrix). MRIs for each participant were motion corrected, normalized for intensity in homogeneities, and transformed to Talairach space. Non-brain tissue was removed using a hybrid watershed/surface deformation procedure, subcortical structures were segmented, and further intensity normalization was conducted. This was followed by white-matter segmentation, tessellation of the gray-white matter boundary, and automated topology correction. Finally, surface deformation following intensity gradients optimally placed the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class. The cerebral cortex was parcellated into 34 gyral-based regions in each hemisphere based on the Desikan-Killiany atlas [ 33 ]. Subcortical structures were delineated by an algorithm that examines variations in voxel intensities and spatial relationships. Anatomical accuracy of the segmentations was manually verified. To account for differences in head size, volumes for each region were divided by intracranial volume (ICV).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26452176">PMID:26452176</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical scan was used to reconstruct the cortical surface of each hemisphere using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). Regions of interest (ROIs) were selected on the basis of previous findings reported in the MI literature (Ehrsson et al.,  2003 ; Gr è zes and Decety,  2001 ; Heed et al.,  2011 ; Jeannerod,  2001 ] and defined anatomically on an individual basis using the FreeSurfer parcellation algorithm [Destrieux et al.,  2010 ]. We defined eight ROIs per hemisphere as follows (cf. Fig.  2 a):</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27014392">PMID:27014392</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) was used for surface reconstructions. Pre-processing of functional images was completed in SPM8 ( http://www.fil.ion.ucl.ac.uk/spm/ ). Software built in house was used in order to complete the pRF mapping analysis ( http://dx.doi.org/10.6084/m9.figshare.1344765 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27014013">PMID:27014013</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical surfaces were reconstructed and the anatomical features were measured for each participant with the FreeSurfer 5.3.0 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ , (Dale et al.,  1999 ; Fischl et al.,  1999a , b ,  2001 ,  2002 ,  2004 ; Fischl and Dale,  2000 ; Fischl,  2012 ). The FreeSurfer analysis stream includes intensity bias field removal optimized for the present 3T acquisitions and skull stripping. In the native anatomical space of the subject, FreeSurfer constructs models of the cortical surface that can be represented as a mesh of triangles and tessellated into over 160,000 vertices for each hemisphere. The vertex positions are adjusted such that the surface follows the T1 intensity gradient between the cortical WM and GM. This highly folded surface, called  “ white surface, ”  can be inflated to see buried surfaces within the sulci. A second surface, called the  “ pial surface, ”  is fit to the outer GM surface of the brain. The hemispheres are processed separately through a precise estimation of the inter-hemispheric fissure plane.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27013993">PMID:27013993</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data were analyzed with FS-FAST and FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) software. For image preprocessing, all functional images were head motion corrected, slice-time corrected, spatially smoothed with a Gaussian kernel of 8.0 mm (FWHM). The mean intensity for the entire functional volume was computed for each scan. The global mean of the entire brain was rescaled so that the same mean was set across scans. All functional data were registered to the individual anatomically reconstructed brain.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26985236">PMID:26985236</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI was performed on the same 1.5 Tesla Siemens Symphony Sonata (Siemens AG, Erlangen, Germany) at St Olav ’ s University Hospital (Trondheim, Norway) at 15 and 19   years of age with Quantum gradients (30 mT/m) and a quadrature head coil. A structural T1-weighted magnetization prepared rapid acquisition gradient echo (MPRAGE) sequence was acquired with the following specifications: TR   =   7.1   ms, TE   =   3.45   ms, TI   =   1000   ms, flip angle 7 o , FOV 256   ×   256, slab thickness 170   mm, slice thickness 1.33   mm, acquisition matrix 256   ×   192   ×   128, reconstructed to 256   ×   256   ×   128, giving a reconstructed voxel resolution of 1   ×   1   ×   1.33   mm, and acquisition duration of 8.5   min. Two MPRAGE sequences acquired at each time point (15 and 19   years) were registered to correct for head motion and averaged into a single image. FreeSurfer software package 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ) was used for the volumetric segmentation. This is an automated method of labeling human structures to extract GM and WM volumes for each participant ’ s entire brain [ 30 ,  31 ], and parcellating of the cortex of each participant [ 32 ,  33 ]. All processed images were visually inspected for accuracy of segmentation. Structures with obvious segmentation errors were rejected and no manual editing was performed to avoid introducing bias and increasing variances into the data set of MRI images. All images were processed with the longitudinal stream in FreeSurfer to enable longitudinal analyses [ 34 – 36 ] and to account for unbalanced time points [ 37 ]. For each participant, mean cerebellar volumes of GM and WM and estimated intracranial volume (eICV) were extracted and used in further analyses. The eICV volume is an indirect measure of the whole volume inside the human cranium except cerebellum, brain stem and ventricles.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26976307">PMID:26976307</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness analyses were carried out using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ), as previously described [ 15 ]. Briefly, T1 images were intensity-normalized, after which a skull-stripping procedure was applied. Images were segmented using an estimation of the structure of the grey-white interface. Each scan was covered with a triangular tessellation and inflated (for a smooth spherical representation of the grey-white interface and pial surface). Inflated scans were aligned to FreeSurfer ’ s default reference template via a 2D warp based on cortical folding patterns. Once smoothed (circularly symmetric Gaussian kernel), sulci and gyri curvature patterns were aligned and average cortical thickness was measured at each surface point. A uniform surface-based spherical coordinate system was created by transforming the reconstructed surfaces into parameterizable surfaces. An averaging procedure (50 iterations) was applied to smooth the surface and the reconstructed pial surface was refined with a deformable surface algorithm. Data was again aligned on a common spherical coordinate system. Cortical thickness was determined by measuring and averaging the distance between the grey/white matter boundary and pial surfaces. The following regions were assessed: a) cingulate regions (rostral anterior cingulate, caudal anterior cingulate, posterior cingulate and isthmus cingulate in both hemispheres), b) PFC/orbital regions (frontal pole, lateral and medial OBF as well as parsorbitalis cortices in both hemispheres), c) fronto-lateral regions (superior frontal, rostral middle frontal, parsopercularis, parstriangularis and caudal middle frontal cortices in both hemispheres), d) parahippocampal regions (entorhinal and parahippocampal cortices in both hemispheres) and e) the insula (in both hemispheres).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26973497">PMID:26973497</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with FreeSurfer (version 5.3.0,  http://surfer.nmr.mgh.harvard.edu ). The automated processing stream mainly included removal of non-brain tissue (Segonne et al.,  2004 ), Talairach transformation, segmentation of gray/white matter tissue (Fischl et al.,  2004 ), intensity normalization, topological correction of the cortical surface (Saur et al.,  2008 ), and surface deformation to optimally place the tissue borders (Fischl and Dale,  2000 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26963246">PMID:26963246</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each electrode contact was localized in 3-dimensional anatomical space in a custom Python-based visualization environment using the electrode contact artifacts in post-operative volumetric CT scans co-registered with the pre-operative MRI scans. Since the SEEG electrodes are fabricated from polyurethane tubing and the electrode contacts span 60 mm, the shaft of the electrodes can follow a curved course between superficial and deep targets. Thus, each contact was individually represented as a sphere and labeled by color according to the nearest neural structure. Using the T1-weighted image, FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) [ 28 ] parcellated cortical regions of interest including the motor, premotor, primary sensory, and insular cortices [ 29 ] and to create 3D surface views of each participant ’ s brain. The supplementary motor area (SMA) was manually extracted from Freesurfer ’ s definition of the pre-frontal cortical region using FSLView ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ) [ 30 ]. We defined the SMA as the region on the medial surface of both hemispheres, dorsal to the cingulate sulcus that was posterior to the anterior commissure and anterior to the posterior commissure [ 31 ]. For more detailed information on the imaging pipeline please see  S1 Text . We have made these Python scripts available for download ( https://github.com/mcintyrelab ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26958615">PMID:26958615</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Fig.1  illustrates the processing pipeline used and identifies the data provided with respect to this pipeline. For each subject, three to four individual T1-weighted magnetization-prepared rapid gradient echo (MP-RAGE) scans were obtained on a 1.5   T Vision system (Siemens, Erlangen, Germany) with the following protocol: in-plane resolution=256 × 256 (1   mm × 1   mm), slice thickness=1.25   mm, TR=9.7   ms, TE=4   ms, flip angle=10   u, TI=20   ms, TD=200   ms. Images were motion corrected and averaged to create a single image with a high contrast-to-noise ratio (Marcus et al., 2007)  [9] . Anatomical MR images were processed using Freesurfer 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ), in order to extract the inner cortical surface mesh (white), surface area and intracranial volume, and obtain spherical inter-individual correspondences through the Freesurfer sphere.reg mesh  [10] ,  [11] . The accuracy of the cortical surface extraction was visually controlled for each individual. We then defined two groups of 68 and 69 young adult healthy subjects (aged 18 – 34   yr) matched in age, gender, cortical surface area and intracranial volume (see  Table 1 ). These two groups can be used to evaluate the consistency of the sulcal pits extraction tool across two well controlled populations.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26955515">PMID:26955515</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The extraction of the whole brain structural connectivity matrices (connectomes) for each subject was performed using the Connectome Mapping Toolkit, a python-based open-source software that implements a full diffusion MRI processing pipeline, from raw diffusion/T1/T2 data to multi-resolution connection matrices ( www.cmtk.org , ( Daducci et al., 2012 )). The high-resolution T1w volume is first registered to the diffusion space using the  bbregister  tool from Freesurfer software ( http://surfer.nmr.mgh.harvard.edu /). From there, the whole processing pipeline can be sequentially divided in (i) WM-GM surface extraction and cortical and subcortical parcellation (again, using Freesurfer), (ii) streamline tractography (done by an in-house developed method implemented in the  cmtk  pipeline and (iii) connectome construction, by estimating the connection density between each pair of regions of interest (ROI). The connection density between two cortical or subcortical ROIs is calculated as the sum of all streamlines connecting these two ROIs divided by the length of the streamlines. This value is normalized by the size of the ROIs.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26941619">PMID:26941619</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All structural and functional data were analyzed with FreeSurfer and FS-FAST packages ( http://surfer.nmr.mgh.harvard.edu/ ). Using standard FreeSurfer methods (Fischl et al.,  1999 ; Greve and Fischl,  2009 ), the surface of each cerebral cortical hemisphere was extracted via image processing based on the anatomical gray-white matter boundary and computationally reconstructed as a 3D mesh of vertices. In order to permit visualization of multiple visual cortical areas, each cortical hemisphere was computationally inflated to reveal buried sulcal regions and an occipital lobe  “ flat patch ”  was created for each subject hemisphere by making cuts in the vertex mesh (along the calcarine sulcus and the anterior extent of the occipital lobe) and unfurling this mesh into a 2D representation (see Figure  2A  for further details). Each functional run was rigidly registered to the anatomical data using gray-white matter boundary based registration (Greve and Fischl,  2009 ). Functional data was motion corrected and spatially smoothed using a 3D Gaussian kernel (fwhm = 3.0), separately by run and by hemisphere. Voxel-wise statistical tests were based on a univariate general linear model (GLM), and the significance levels (inverse log  p -value) were visualized on the inflated and flattened cortical surfaces.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26919667">PMID:26919667</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution structural MRI scans were obtained at the VUmc, using a GE Signa HDxt 3.0-Tesla MRI-scanner (General Electric, Milwaukee, Wisconsin, USA) with an 8-channel head coil. We acquired structural MRI data using a sagittal 3-dimensional gradient-echo T1-weighted sequence (256 x 256 matrix; field of view = 25cm; slice thickness = 1mm; voxel size = 1 x 0.98 x 0.98 mm; TR = 7.8 ms; TE = 3.0 ms; view angle = 12 ° ). Image analysis was carried out with the stable version (v.5.3.0) of the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ) [ 23 – 25 ]. In short, the procedure included: motion correction, intensity normalization, Talairach registration, skull stripping, segmentation of subcortical white matter, tessellation of the GM/white matter (WM) boundary, automated topology correction, and surface deformation. We used a 10 mm (full-width at half-maximum) Gaussian kernel to smooth maps. Finally, FreeSurfer created a surface 3D model of the cortex using intensity and continuity information.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26901338">PMID:26901338</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Subjects received high resolution T1-weighted MRI scans of the brain at 1.5 Tesla acquired using a variety of scanners (General Electric, Philips, or Siemens) and a standardized protocol [ 39 ]. Each MRI dataset was post-processed  using FreeSurfer v5.0.0 ( http://surfer.nmr.mgh.harvard.edu ) [ 40 – 43 ], an image processing software tool for (a) automated model-based reconstruction and segmentation of the brain's cortical surface and subcortical structures and (b) morphometric analysis. Finally, a variety of morphometric measures were computed across 180 anatomically-defined brain regions as MRI features for classification, including cortical and subcortical volumes, mean cortical thickness (and its standard deviation), surface area, and curvature. FreeSurfer-derived morphometric MRI measures have been validated in studies of normal aging, MCI, and AD [ 44 – 46 ]. See section 1.3 in  S1 File  for details on MRI acquisition and processing.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26894510">PMID:26894510</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Inflation and flattening of the anatomical images was carried out using the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu ). The skull-stripping step was done manually. The activation maps from the meridian mapping were overlaid on the flattened occipital patches in order to delineate the V1/V2 borders (the upper or lower vertical meridian contrasted against the horizontal meridian) as well as the V2/V3 borders (the horizontal meridian contrasted against the upper and lower vertical meridians). The activations from the position localizer scan (each position in the Ternus-Pikler display contrasted against its within-hemisphere neighbor) were overlaid on the same flattened patch in order to identify the cortical representations of positions 1 – 4. The regions of interest (ROIs) were then identified in the three-dimensional data and extracted using the MarsBaR toolbox ( http://marsbar.sourceforge.net ). Thresholds were chosen individually to ensure consistent ROI sizes ( t  values between two and 11 were used).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26551663">PMID:26551663</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Two regions of interest - the medial temporal lobe (MTL; incorporating the hippocampus and parahippocampus) and the middle frontal gyrus - were identified using a mask from the Automated Anatomical Labeling (AAL) library. The middle frontal gyrus volume was extracted as clear anatomically delimited templates exist, and this region incorporates a large portion of the dorsolateral prefrontal cortex (DLPFC; which has no clear anatomical boundaries). Grey matter volumes for the regions of interest were extracted from the DARTEL flow output using MarsBar software. In addition, global measures of ventricular volume (sum of 3rd, 4th left and right lateral and inferior lateral ventricles), total grey matter and intracranial volume were generated using Freesurfer software (version 4.5.0;  http://surfer.nmr.mgh.harvard.edu/ ) according to a standard pipeline. The resultant extracted volumes were used in linear regression (generalised estimating equation) and paired t tests performed in STATA11.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26116964">PMID:26116964</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cluster analysis was performed on the location of individual peaks found in the t-maps for the vocal   >   non-vocal contrast across a group of 218 subjects, in order to determine if they could be grouped into distinct clusters. Analysis was performed using the following steps for each individual subject: (1) Pial and white/gray matter surfaces were extracted with freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al., 1999 ), subsampled to 81,924 vertices and all subjects brought into anatomical correspondence by spherical co-registration to the average template ( Fischl et al., 1999 ). Then the midgray surface was calculated by averaging the coordinates from corresponding vertices of the pial and white/gray matter surfaces; (3) the anatomical T1 was then co-registered with the mean functional EPI image, and the co-registration transformation parameters were used to warp the midgray surface into the fMRI native space; (4) the unsmoothed fMRI time series (after slice timing and, movement correction in SPM8) were sampled (with spatial interpolation) on all surface vertices; (5) the time series on the surface were high pass filtered with a 128   s time constant; (6) a GLM using the localizer design matrix (vocal and non-vocal regressors plus realignment parameters and the session constant term) was used to obtain betas on the surface. The contrast vocal   >   non-vocal was calculated at each vertex; (7) a threshold was obtained for these individual t-maps using a false discovery rate with  q   =   0.05 and the default cluster forming threshold  p   =   0.001 ( Chumbley et al., 2010 ); (8) the 10 largest peaks in each individual were identified in each hemisphere; this choice in the number of peak was arbitrary, but sufficiently large to reveal clustering of activity; (9) disks of 3   mm were created around each peak using functions from the Fast marching Toolbox ( http://www.mathworks.com/matlabcentral/fileexchange/file_infos/6110-toolbox-fast-marching ) and the Surfing Toolbox ( http://sourceforge.net/projects/surfing/ ) and a weight of 1; (10) the 10 disks were summed, resulting in a density map for each subject; (11) a permutation procedure was used in which the peaks were then randomly positioned within the surface (excluding the medial wall) 1000 times, and the max of the density was measured. The histogram of this max over all the permutations was used to determine the  p   <   0.01 threshold; (12) connected clusters in the thresholded density map were found with the clustermeshmap.m function from the Neuroelf toolbox ( http://neuroelf.net/ ). We report here the maxima for each cluster.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26033243">PMID:26033243</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Left and right lateral ventricle, thalamus, caudate, putamen, pallidum, accumbens, hippocampus and amygdala volumes, as well as ICVs, were obtained with FreeSurfer 22 ,  23  ( http://surfer.nmr.mgh.harvard.edu ) from high-resolution T 1 -weighted structural brain scans ( Supplementary Tables S4 and S4a and o ). Details on study type (single site or multisite), scanner vendor/strength/sequence, acquisition parameters and FreeSurfer versions used are provided in  Supplementary Table S3b . For quality control, all regions of interest (ROIs) with a volume larger or  < 1.5 times the interquartile range were identified and visually inspected by overlaying their segmentations on the subjects' anatomical images. Only ROI data for which segmentation was judged accurate upon visual inspection were subjected to statistical analyses (see  Supplementary Figure S2  for ROI volume histograms).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27117083">PMID:27117083</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>ADNI MRIs were acquired at multiple sites with a GE Healthcare (Buckinghamshire, England), Siemens Medical Solutions USA (Atlanta, Georgia), or Philips Electronics 3.0 T system (Philips Electronics North America; Sunnyvale, California) 40 . These analyses utilized the dataset of UCSF FreeSurfer to conduct association test of  PICALM  genotypes with brain structure. We processed the cerebral image segmentation and analysis using the FreeSurfer version 5.1.0 software package ( http://surfer.nmr.mgh.harvard.edu/ ) based on the 2010 Desikan-Killany atlas 41 . The main work contained that motion correction and averaging of multiple volumetric T1-weighted images (when more than one is available) 42 , removal of non-brain tissue using a hybrid watershed/deformable surface algorithm 43 , automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles) 44 45 , intensity normalization 46 , tessellation of the gray matter white matter boundary, automated topology correction 47 , and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid (CSF) borders at the location where the greatest shift in intensity defines the transition to the other tissue class 48 . More detailed technical procedures were available in previous study 48 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27115029">PMID:27115029</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The data and supplementary information provided in this article relate to our research article  “ Task complexity and location specific changes of cortical thickness in executive and salience networks after working memory training ”  (Metzler-Baddeley et al., 2016)  [1] . We provide cortical thickness and subcortical volume data derived from parieto-frontal cortical regions and the basal ganglia with the FreeSurfer longitudinal analyses stream ( http://surfer.nmr.mgh.harvard.edu  [2]) before and after Cogmed working memory training (Cogmed and Cogmed Working Memory Training, 2012) [3] . This article also provides supplementary information to the research article,  i.e. , within-group comparisons between baseline and outcome cortical thickness and subcortical volume measures, between-group tests of performance changes in cognitive benchmark tests ( www.cambridgebrainsciences.com  [4]), correlation analyses between performance changes in benchmark tests and training-related structural changes, correlation analyses between the time spent training and structural changes, a scatterplot of the relationship between cortical thickness measures derived from the occipital lobe as control region and the chronological order of the MRI sessions to assess potential scanner drift effects and a  post-hoc  vertex-wise whole brain analysis with FreeSurfer Qdec ( https://surfer.nmr.mgh.harvard.edu/fswiki/Qdec  [5]).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27110218">PMID:27110218</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/ .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27096608">PMID:27096608</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A second ROI labeling analysis was performed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ). The automatic processing of each individual brain scan included the removal of non-brain tissues, a Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (aseg atlas [ 31 ]), intensity normalization, tessellation of the gray matter white matter boundary, a topology correction and a surface deformation step to end up with the reconstruction of the cortical surface and the volumetric segmentation into white matter, gray matter and cerebrospinal fluid. Subsequently, a surface inflation step and a registration to a spherical atlas step were performed. Finally, each cerebral cortex was parcellated into ROIs according to the cortical atlases provided by the FreeSurfer software (the Desikan-Killiany atlas [ 32 ], the Desikan-Killiany-Tourville (DKT 40) atlas [ 33 ] and some labels from the Talairach atlas). All 169 ROIs were used in the statistical analyses (see the  results  table provided in  S3 File  for a full list of anatomical labels included).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27065855">PMID:27065855</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed by Freesurfer 5.3 image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications, we made no changes to this pipeline. Briefly, image processing includes motion correction (Reuter et al.,  2010 ), removal of non-brain tissue using a hybrid watershed/surface deformation procedure (Segonne et al.,  2004 ), automated Talairach transformation, segmentation of the subcortical white matter, and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles) (Fischl et al.,  2004 ) intensity normalization, tessellation of the gray matter white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Dale et al.,  1999 ; Fischl and Dale,  2000 ). Once the cortical models were completed, Freesurfer performed a number of deformable procedures for in further data processing and analysis. Steps included surface inflation (Fischl et al.,  1999a ), registration to a spherical atlas which utilized individual cortical folding patterns to match cortical geometry across subjects (Fischl et al.,  1999b ), parcellation of the cerebral cortex into units based on gyral and sulcal structure (Fischl et al.,  2004 ), and creation of a variety of surface based data including maps of curvature and sulcal depth. Finally cortical models and the results of segmentation were quality checked and manually corrected on each subject, however correction showed no significant changes to the results.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27065831">PMID:27065831</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction is described in the Supplementary Materials and was performed with the FreeSurfer image analysis suite (version 5.3.0), which is documented and freely available online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications (Dale et al.,  1999 ; Fischl et al.,  1999a , b ,  2001 ,  2002 ,  2004a , b ; Fischl and Dale,  2000 ). Reconstructed surface models were visually checked for accuracy by an experienced imaging analyst (J.H.) and no manual interventions were necessary.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27064591">PMID:27064591</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>2 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27057360">PMID:27057360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Both the cortical reconstruction and morphological features extraction were obtained by using the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) with a standard cortical automatic handling protocol. First, the data were normalized to a standard anatomical template [ 27 ] and corrected for bias-field inhomogeneity. Then the images were skull-stripped using a watershed algorithm [ 28 ] and subsequently segmented into subcortical white matter and deep gray matter volumetric structures [ 29 ,  30 ]. The initial tessellation was formed by reconstructing the gray matter/white matter boundary (white surface) and the outer cortical surface (pial surface) [ 31 ,  32 ]. Subsequently, a series of deformable procedures were performed, including surface inflation [ 31 ], registration to a spherical atlas [ 33 ], and parcellation of the cerebral cortex into units based on gyral and sulcal structures [ 30 ]. All reconstructed surfaces were visually inspected for gross-anatomical topological defects. Finally, a variety of morphological features at each vertex on the pial surface were computed, including volumetric (cortical thickness, surface area, and GM volume) and geometric (sulcal depth, metric distortion, and mean curvature) measures, more details seen in this paper [ 34 ]. The thickness maps of both NC and aMCI groups are shown in  Figure 1(a) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27047376">PMID:27047376</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>3 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27047361">PMID:27047361</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27047360">PMID:27047360</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data processing and analysis was completed within FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). Both T1-weighted images were parcellated into cortical and segmented into subcortical gray matter, white matter, and CSF for each subject. These surface reconstructions were combined and then used for inter-subject alignment and for seed placement. Of the three BOLD scans obtained at the subject level, the two with the least movement were used. Time points with more than 0.4 mm of motion were removed from the analysis. These scans were motion corrected, smoothed on the surface at 15 m.m., and then concatenated. Next, the scans were band-pass filtered between 0.01 and 0.1 Hz., and the global signal, white matter signal, and ventricle CSF, and the motion time course were globally regressed out. Behavioral regressors (covariates) included frequency of classification of doctrinal and experiential phrases, which were paired with bilateral seeds from the nucleus accumbens for analysis of the resting state data within the general linear model (GLM) in FreeSurfer. The seeds were defined using the entire structure as described in Fischl et al. ( 2002 ). The time course for the seeds was averaged across the two hemispheres into a single time course for analysis. We then corrected for multiple comparisons using a Monte Carlo simulation within the GLM in FreeSurfer. Cluster locations were identified based on FreeSurfer parcellation and the FreeSurfer surface atlas aparc.annot (Desikan-Killiany Atlas). Brodmann areas were derived from MNI305, as were reported coordinates for the greatest correlation/significance within a cluster. All results were obtained using one group one covariate (OGOC) intercept/offset difference analysis where a group average of network maps from bilateral nucleus accumbens were derived and correlated with behavioral regressors (covariates) within the GLM in FreeSurfer.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27035571">PMID:27035571</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>ROIs (left and right putamen and caudate nucleus) were anatomically delineated using a semi-automatic approach on the3D T1-weighted images. The initial step was an automatic brain segmentation with Freesurfer software ( http://surfer.nmr.mgh.harvard.edu/ ). In order to rule out segmentation errors, ROIs were visually inspected and (if required) manually corrected by an investigator blinded to the participant's condition. T2* maps were generated with the relaxometry routine in the Philips Research Imaging Development Environment (PRIDE ® , Philips Medical Imaging). A mono-exponential trend-line was fitted with this equation: y = Kexp( – TE/T2*), where y is the MR signal intensity, K is a constant and TE is the echo time. Head motion correction was performed by rigid registration (3 translations and 3 rotations) of echoes. The first echo was used as reference.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26992858">PMID:26992858</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI acquisitions were carried out on a 3T Siemens Trio MRI system at the Centre for Translational Imaging, Northwestern University of Chicago. T1-weighted magnetization-prepared rapid gradient echo (MPRAGE) sequences were acquired with the following parameters: repetition time 2,300 ms; echo time 2.86 ms; flip angle, 9; field of view, 256 mm; 60 slices; slice thickness 1.0 mm, as previously described. 6  FreeSurfer image analysis suite (version 4.5.0) ( http://surfer.nmr.mgh.harvard.edu/ ) was used to measure cortical thickness on T1-weighted MPRAGE images. Measures of cortical thickness were obtained by estimating the closest distance between the gray/white matter boundary and the gray matter/CSF boundary at each vertex of the tessellated surface. 6 , 16  Differences in cortical thickness between patients with PPA and healthy controls were shown on the entire surface area of the neocortex using an FDR of 0.001. 6 , 17  Cortical thickness was measured from a 20-mm region of interest (ROI) placed in the most atrophic anterior temporal and lateral orbitofrontal cortex, within the cortical projections areas of the uncinate fasciculus.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26806288">PMID:26806288</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The longitudinal stream of FreeSurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu ) was used to improve the robustness and sensitivity of the longitudinal analyses of the cortical thickness of brain regions of using a semi-automated approach described in detail elsewhere ( Fischl, 2012 ,  Jovicich et al., 2009 ), with the use of additional computing resources from the high performance computing TIER1 cluster at the University of Gent ( http://www.ugent.be/hpc/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26778365">PMID:26778365</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 images were processed using the volume-based stream in FreeSurfer version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ), described in detail by  Fischl et al. (2002) . In short, intensity normalisation was conducted, followed by removal of non-brain tissue using a hybrid watershed/surface deformation procedure. Next, neuroanatomical labels were assigned to each voxel in an MRI volume based on probabilistic information derived from a manually labelled training set. The accuracy of the final subcortical brain segmentations was visually inspected and edited where needed according to the standard FreeSurfer guidelines ( http://surfer.nmr.mgh.harvard.edu/fswiki/Edits ) by a single rater, blind to diagnostic status (MP).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26190555">PMID:26190555</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surfaces were reconstructed using FreeSurfer version 5.0 ( Fischl et al., 1999 ) in accordance with the description available on-line ( http://surfer.nmr.mgh.harvard.edu/ ). After completion of skull-stripping and intensity correction, the grey – white matter boundary for each cortical hemisphere was determined using tissue intensity and neighbourhood constraints. With the use of a deformable surface algorithm guided by the grey-cerebrospinal fluid (CSF) intensity gradient, the resulting grey-white interface was expanded to create the pial surface, followed by a spherical morphing procedure and registration based on sulcogyral alignment. The surface boundary was tessellated to generate multiple vertices (and triangles) across the whole brain. All surfaces were visually inspected following an automated topology fixation procedure, and remaining minor defects were manually corrected as recommended by the software guidelines. Automated subcortical segmentation using probabilistic information regarding the location of subcortical structures was carried out using Freesurfer v5.0, with the investigator being blind to the diagnostic group ( Fischl et al., 2002 ). Following this procedure, the right and left thalamic volumes were extracted. Thalamic volume estimation using Freesurfer has been shown to have an impressive consistency with stereological measurement ( Keller et al., 2012 ), while ensuring observer blindness during measurement. We chose whole thalamic volume instead of parcellating the subdivisions of thalamus, as anatomical segmentation of thalamus using structural (T1) MRI scans alone often differs from connectivity-based identification of thalamic subdivisions obtained using white matter tractography or functional connectivity ( Traynor et al., 2010 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25673447">PMID:25673447</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject, volume estimation for seven bilateral subcortical structures, including the nucleus accumbens, amygdala, caudate, hippocampus, pallidum, putamen and thalamus, was performed using the Freesurfer image analysis suite (version 5.3.0, Charlestown, MA, USA;  http://surfer.nmr.mgh.harvard.edu ). Automatic subcortical segmentation of a brain volume is based upon the existence of an atlas containing probabilistic information on the location of structures. The technical details of these procedures are described in the previous publication.[ 18 ] Finally, two expert radiologists verified the accuracy of each participant's native-space SDGM segments, and no errors were identified.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25627477">PMID:25627477</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We performed rigorous region ‐ of ‐ interest (ROI) manual volumetry of the left and right hippocampus and entorhinal cortex, which are brain structures considered to be of crucial importance for the generation of mesial temporal lobe seizures. We chose to use rater ‐ dependent manual techniques given their increased reliability over automated techniques generally, and also given the particular difficulties in reliable automated segmentation of the mesial temporal lobe into its constituent regions, particularly the entorhinal cortex. We quantified hippocampal volume using our previous described and well ‐ validated stereological approach. 22  Entorhinal volume was estimated based on published anatomical boundaries 23  using 3D Slicer software ( http://www.slicer.org ). All volumetric data were acquired blind to participant ID. We also determined whole (global) cortical and white matter volumes, which were automatically generated using FreeSurfer software (v5.3.0;  http://surfer.nmr.mgh.harvard.edu ). All volumetric data were generated in participant native space, and analyzed with other extracted ROI data (see statistical analysis below).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27253393">PMID:27253393</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The raw data were reviewed for quality, and automatically corrected for spatial distortion due to gradient nonlinearity [ 43 ] and B 1  field inhomogeneity [ 44 ]. For all participants, the two image volumes collected at each time point were co-registered, averaged to improve the signal-to-noise ratio (SNR), and resampled to isotropic 1-mm voxels. Images were first automatically processed cross-sectionally (independently) for each time point with the FreeSurfer software package (version 5.1.0; Athinoula A. Martinos Center for Biomedical Imaging, Boston, MA), which is documented and freely available online ( http://surfer.nmr.mgh.harvard.edu/ ). This processing includes motion correction, removal of non-brain tissue, automated Talairach transformation, intensity correction, volumetric segmentation [ 45 ], and cortical surface reconstruction [ 46 – 48 ] and parcellation [ 49 ,  50 ]. All volumes were inspected for accuracy and minor manual edits were performed where needed by a trained operator, usually restricted to removal of nonbrain tissue included within the cortical boundary. To extract reliable longitudinal cortical volume, thickness and area change estimates, the cross-sectionally processed images were subsequently run through the longitudinal stream in FreeSurfer [ 51 ]. Here, an unbiased within-subject template volume based on the two cross-sectional images is created for each participant, and processing of both time points are then initialized using common information from this template. This increases sensitivity and robustness of the longitudinal analysis [ 51 ]. The registration method used to generate the within-subject template from Tp1 and Tp2 ensures inverse consistency [ 52 ], meaning that the transform computed from Tp2 to Tp1 is the inverse of that computed from Tp1 to Tp2 [ 53 ], which is critical in longitudinal analyses [ 54 ]. In addition, new probabilistic methods (temporal fusion) were applied to further reduce the variability across time points. Surface maps were resampled, mapped to a common surface, smoothed using a circularly symmetric Gaussian kernel with a full-width half-maximum of 15 mm [ 55 ] and submitted to statistical analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27252649">PMID:27252649</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27252636">PMID:27252636</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was performed with the Freesurfer image analysis suite (Version 5.3.0), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). In brief, the processing stream includes a Talairach transform of each participant's native brain, removal of non-brain tissue, and segmentation of gray matter (GM)/white matter (WM) tissue. The GM/WM boundary was tessellated to generate multiple vertices across the whole brain. The cortical surface of each hemisphere was inflated to a sphere to locate the pial surface and the GM/WM boundary. After the creation of cortical representations, all vertices were assigned neuroanatomical labels on a cortical surface model based on the automated labeling system, and the entire cortex of each hemisphere was parcellated into 34 brain regions based on Desikan-Killiany atlas (Desikan et al.,  2006 ). The inner surface area of a region was computed by summing up the area of the vertices in that region. After calculating the cortical surface area for each brain region and each hemisphere, the lateralization index was computed as a standardized asymmetry index (Left region  –  Right region)/(Left region + Right region).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27244227">PMID:27244227</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The preprocessing pipeline for our cortical thickness analysis was performed using FreeSurfer (version 5.3.0) ( http://surfer.nmr.mgh.harvard.edu ), a semi-automated toolbox for reconstruction and visualization of the cortical surface [ 29 – 31 ]. The pipeline consisted of affine registration of the T1-weighted volume to Talairach space, skull stripping, white matter (WM) segmentation and tessellation of the gray/white matter boundary. At each step, visual inspection and manual correction of topological errors were performed. Following reconstruction of the cortical surface, brains were inflated, averaged across participants to produce a study-specific brain, and then smoothed using a 10 mm full-width at half maximum Gaussian kernel. Smoothing was performed on the surface tessellation using an iterative nearest-neighbor procedure, avoiding averaging of data across sulci or outside gray matter (GM). Each hemisphere was then parcellated into 34 distinct regions using the Desikan-Killany atlas [ 32 ]. A direct measure of cortical thickness was calculated using the shortest distance (mm) between the pial surface and gray-white matter boundary at each point or vertex of the cortical mantle, using spatial intensity gradients without limitation of individual voxel intensities, and therefore allowing for subvoxel resolution.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27242521">PMID:27242521</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>4 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27242478">PMID:27242478</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27239530">PMID:27239530</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Details of the MRI methodology have previously been described  [20] . Cross-sectional regional measures of brain volume for the hippocampus, entorhinal cortex, middle temporal gyrus, fusiform, ventricles, and whole brain, as well as total intracranial volume, were collected on a 1.5-T scanner using a standardized back-to-back 3D magnetization prepared rapid gradient echo (MP-RAGE) protocol: sagittal plane, TR/TE/TI, 2400/3/1000   ms, flip angle 8 ° , 24-cm FOV, 192  ×  192 in-plane matrix, 1.2-mm slice thickness. All regional volumes were normalized by dividing by total intracranial volume for each subject and calculated at baseline using FreeSurfer version 4.3 ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27231702">PMID:27231702</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MR images were transferred in DICOM format to a dedicated Linux workstation (Centos 5). Cortical reconstruction and volumetric segmentation were performed using surface ‐ based morphometry (SBM) with the Freesurfer v5.1 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of SBM are described and referenced on the Freesurfer website ( http://surfer.nmr.mgh.harvard.edu/ ). Processing includes several stages: removal of non ‐ brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white matter, deep gray and white matter volumetric structures, intensity normalization, tessellation of the gray and white matter boundaries, automated topology correction, and finally surface deformation using intensity gradients to place the gray/white and gray/cerebrospinal fluid (CSF) borders optimally at the location where the greatest shift in intensity defines the transition to the other tissue class. In this process, voxels are classified as white matter or something other than white matter based on intensity and neighbor constraints. An initial surface is then generated for each hemisphere by tiling the outside of the white matter mass for that hemisphere. This initial surface is then refined to follow the intensity gradients between the white and gray matter (this is referred to as the white surface). The white surface is then nudged to follow the intensity gradients between the gray matter and CSF (this is the pial surface). The distance between the white and the pial surfaces equates to the thickness at each location of the cortex. Surface models of the cortical surface are generated, consisting of a mesh of triangles, and the location of the mesh is controlled by adjusting the location of the vertices. A vertex is the place where the points of neighboring triangles meet and are typically about 1 mm apart. The vertex positions are adjusted such that the surface follows the contour of the maximum T1 intensity gradient between cortical white matter (WM), and cortical gray matter (GM), and between the cortical gray matter and pia. All surfaces are constructed in native anatomical space.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27229352">PMID:27229352</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Our study used UCSF FreeSurfer datasets to conduct association test of  CLU  genotypes with brain structure. The cerebral image segmentation and analysis were performed with the FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) based on the 2010 Desikan-Killany atlas 71 . We obtained data from motion correction and averaging of multiple volumetric T1 weighted images (when more than one is available), removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles) 72 , intensity normalization, tessellation of the gray matter white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white as well as gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class. The technical details of these procedures are described in prior publications 73 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199740">PMID:27199740</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted MRIs were automatically segmented and volumes were calculated using FreeSurfer software, version 5.3.0, available at  http://surfer.nmr.mgh.harvard.edu/  ( Fischl et al., 2002 ). Following preprocessing, all results underwent quality control to confirm correct detection of gray and white matter. Any errors in segmentation were corrected manually and the T1 images were re-processed through FreeSurfer. Seventy one percentages of cases had some form of manual edit. These consisted of adding control points to extend the white matter boundary (55%), removing voxels from the brain mask (29%), and editing the white matter mask (6%). Some subjects had edits from more than one category, such as both control points and brain mask edits. Previous research has shown that this semi-automated procedure yields accurate and reliable results when compared to manual segmentation ( Fischl et al., 2002 ;  Jovicich et al., 2009 ;  Morey et al., 2009 ) and histological measures ( Morey et al., 2009 ). Automatically parcellated FreeSurfer gray matter regions of interest (ROIs) were based on the Desikan-Killiany atlas. Intracranial volume (ICV) was calculated based on the talairach transform ( Buckner et al., 2004 ). ROIs for the left and right hemisphere were summed and corrected for ICV (by dividing ROI volume by ICV and multiplying by 100) to create bilateral, normalized ROIs. Some of these were then summed to create volumes for the four major lobes of the brain. Specifically, the frontal ROI consisted of the caudal and rostral anterior cingulate cortices, the caudal and rostral middle frontal cortices, the lateral and medial orbitofrontal cortices, the pars orbitalis, the superior frontal cortex, and the frontal pole; the parietal ROI consisted of the precuneus, the inferior and superior parietal lobules, and the supramarginal gyrus; the temporal ROI consisted of the entorhinal cortex, the inferior, middle, and superior temporal regions, the transverse temporal region, and the temporal pole; and the occipital ROI consisted of the lateral occipital region, the lingual and fusiform gyri, the pericalcarine region, and the cuneus. All four lobes were added together to create a measure of total cortical gray matter volume. Subcortical ROIs were chosen based on their relevance to cognitive and behavioral functioning and included the thalamus, caudate, hippocampus, and amygdala.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199638">PMID:27199638</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We constructed a cortical surface model comprised of current dipoles equidistantly distributed on and perpendicular to the cortical surface, which consisted of sulci and gyri. We also constructed a three-layer model consisting of boundary information for scalp, skull, and cerebrospinal fluid. The T1-weighted sagittal image was used for creating the two models. A bias corrected image and a gray-matter volumetric image of the sagittal image were obtained using the  “ Segment ”  function in SPM8 (Wellcome Department of Cognitive Neurology, UK; available at  http://www.fil.ion.ucl.ac.uk/spm ). The bias corrected image was used for creating a polygon model of the cortical surface in FreeSurfer (Martinos Center software,  http://surfer.nmr.mgh.harvard.edu/ ), followed by the cortical surface model. The gray-matter volumetric image was used in conjunction with the output files of FreeSurfer to create the three-layer model.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27190672">PMID:27190672</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Some current neuroimaging software programs have the potential to be used for automatic integration in a pipeline and assessment of gray matter measurements on an individual level. These include Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) [ 24 ], voxel-based morphometry (VBM) analyses using the FMRIB Software Library (FSL) ( http://www.fmrib.ox.ac.uk/fsl/fslvbm ) [ 25 ], Statistical Parametric Mapping (SPM) analyses in MATLAB (MathWorks, Inc., Natick, MA, USA) using the VBM toolbox ( http://www.fil.ion.ucl.ac.uk/spm/ ) [ 26 ], and the SurfStat toolbox ( http://www.math.mcgill.ca/keith/surfstat/ ) [ 27 ] in MATLAB. This brings the possibility for defining brain structural markers for certain pathologies and for measuring the cortical parameters on an individual level, thus opening the opportunity for a quick estimation of the risk for developing the pathology and for assessing its progression over time. One additional approach to do this on individual level is to use automatic learning algorithms such as Support Vector Machines, which allow both the definitions of new patients' groups as well as single-patient classifications into those groups [ 28 ]. Considering that in a clinical setting the costs for performing a structural T1 magnetic resonance imaging (MRI) sequence are lower compared to a diffusion weighted image (DWI) and that analyzing the gray matter parameters of cortical thickness and cortical surface area deformation can be easily performed without manual interventions compared with analyses of white matter parameters which need manual adjustments at more stages, currently it seems more feasible to implement an automated pipeline based on structural gray matter changes. Thus, the present review will focus mainly on the brain structural markers of cognitive decline in patients with Parkinson's disease with emphasis on the gray matter markers.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27158911">PMID:27158911</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI Image Processing: Post-acquisition processing of the MPRAGE images was performed using FreeSurfer v.5.3 software ( http://surfer.nmr.mgh.harvard.edu/ ), using published methods [ 13 – 15 ]. Briefly, local cortical thickness measurements are based on differences between the position of equivalent vertices in pial and gray-white matter surfaces. Steps include white matter segmentation from T1-weighted images, estimation of gray-white matter interfaces, followed by examination for any geometric inaccuracies or topological defects remedied by automatic and manual corrections. Following this, deformable procedures lead to surface inflation of gray-white matter borders, spherical atlas registration, cortical gyral and sulcal structure parcellation, and creation of surface-based data maps [ 16 ]. Differences between subjects in gyral and sulcal depths are normalized; reconstructed brains of each subject are deformed and registered to an average spherical surface. Cortical and subcortical volumes were obtained from the FreeSurfer ver. 5.3 default subcortical segmentation routine Aseg [ 17 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27154626">PMID:27154626</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional statistical maps from each microstimulated afferent and the traveling-wave localizer were rendered onto flattened representations of the central sulcus obtained using the mrFlatMesh algorithm (VISTA software,  http://white.stanford.edu/software/ ) based on cortical segmentations from the whole head T 1 -weighted anatomical data obtained using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Having aligned functional data to the participant ’ s whole head T 1 -weighted anatomical reference volume (see  Alignment of functional data ), statistical maps were transformed to flattened space using linear interpolation and displayed at the central cortical depth.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27148392">PMID:27148392</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer, version 5.1 (a free available ( http://surfer.nmr.mgh.harvard.edu/ ), widely used and extensively validated brain MRI analysis software package), to process the structural brain MRI scans and compute the morphological measurements [ 33 ]. FreeSurfer morphometric procedures have been demonstrated to show good test-retest reliability across scanner manufacturers and across field strengths [ 34 ]. The FreeSurfer pipeline is fully automatic and includes the next procedures: a watershed-based skull stripping [ 35 ], a transformation to the Talairach, an intensity normalization and bias field correction [ 36 ], tessellation of the gray/white matter boundary, topology correction [ 37 ], and a surface deformation [ 38 ]. Consequently, a representation of the cortical surface between white and gray matters, of the pial surface, and segmentation of white matter from the rest of the brain are obtained. FreeSurfer computes structure-specific volume, area, and thickness measurements. Cortical Volumes and Subcortical Volumes are normalized to each subject's Total Intracranial Volume (eTIV) [ 39 ].   Table 2  summarizes the five feature sets extracted for each subject, which are concatenated into the feature matrix  X  with dimensions  N  = 1993 and  D  = 324.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27148109">PMID:27148109</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical imaging was obtained using a 3 Tesla Siemens scanner using a 32-channel head coil. We obtained a T1 5 echo sagittal MPRAGE sequence (TE = 16.4; 3.5; 5.36; 7.22; 9.08 ms; TR = 2530 ms; voxel size = 1.0  ×  1.0  ×  1.0 mm3; slices = 192; acquisition time = 6:03). Methods for cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) and are described in detail elsewhere (Fischl et al.,  2002 ,  2004 ; Han and Fischl,  2007 ). Thickness measurements were obtained by reconstructing representations of the Gray Matter/White Matter boundary and the pial surface and then calculating the distance between those surfaces at each point across the cortical mantle (Dale et al.,  1999 ). The results of the automatic segmentations were quality controlled and any errors were manually corrected. Volume measures are a combination of thickness (a one-dimensional measure) and area (a two dimensional measure) across 33 measures per hemisphere (i.e., 66 across the surface of the brain) as well as seven subcortical volumes per hemisphere (i.e., 14 across the brain) including bilateral caudate, putamen, globus pallidus, nucleus accumbens, thalamus, amygdala, and hippocampus (Fischl et al.,  2002 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27144832">PMID:27144832</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were processed with the FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ). Technical details of the development and validation of FreeSurfer have been previously described[ 24 – 26 ]. Briefly, skull stripping is achieved using a hybrid watershed/deformation procedure followed by automated Talairach transformation and segmentation of subcortical white and gray matter volumetric structures[ 26 ,  27 ]. FLAIR images were used for pial surface refinement. Hippocampal subfield volumes were measured based on methods described in Van Leemput and colleagues[ 28 ] that use Bayesian inference and a statistical model of the medial temporal lobe. The current analyses focused on four hippocampal subfields of interest: CA1, CA2/CA3, CA4/Dentate Gyrus (CA4/DG), and subiculum. The anatomical borders used to delineate these subdivisions are described fully in Van Leemput et al [ 28 ]. Segmentation results were visually inspected for errors in all datasets and manual edits were performed as needed. Entorhinal cortex (EC) and total hippocampal volume (HIP) were also obtained as primary MRI-derived variables of interest. The entorhinal cortex has been shown to be involved in the pathophysiology of Alzheimer ’ s disease[ 29 ,  30 ]. Whole brain (WB) volume was obtained as a control that was not thought to be associated with measures of perceived stress. Additionally, we controlled for differences in head size using estimated total intracranial volume (TICV) that was calculated based on procedures described in[ 31 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27144122">PMID:27144122</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional magnetic resonance imaging data were analyzed using FSL ( http://fsl.fmrib.ox.ac.uk ), including standard preprocessing and general linear model analysis. Voxels exhibiting significant positive/negative BOLD signal changes that correlated with the 15-s SWD event were identified by cluster thresholding (cluster p   <   0.05, voxel z   >   2.3). Freesurfer ( http://surfer.nmr.mgh.harvard.edu /) was used for tissue segmentation and cortical reconstruction from MP-RAGE images.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27130663">PMID:27130663</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ) was used to derive models of the cortical surface for each  T 1 -weighted image. These well-validated and fully automated procedures have been extensively described elsewhere (e.g.,  Dale et al. 1999 ). The resulting surface models were visually inspected for reconstruction errors, and surface reconstructions with visible inaccuracies were further excluded from the statistical analysis (dropout  <  10%) prior to sample generation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27093070">PMID:27093070</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer image analysis suite (version 5.3) was used for structural data processing. This program is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). Briefly, this processing included skull stripping, automated Talairach transformation, subcortical structure segmentation, intensity inhomogeneity correction, gray/white matter boundary tessellation, topology correction and surface deformation following intensity gradients to ensure optimal gray/white and gray/cerebrospinal fluid border. 41 ,  42 ,  43 ,  44 ,  45 ,  46 ,  47  The cerebral cortex was parcellated into specific regions with respect to sulcal and gyral structures. Each image was also manually inspected for errors in the automatic program by one of two raters. An interclass correlation coefficient was calculated for volume adjustments using a two-way mixed effects model. Interclass correlation coefficient between raters was extremely high (0.99), likely reflecting the minimal manual adjustments needed following the automatic processing. Gray matter volumes for the ACC, PCC and isthmus of the cingulate were extracted for each hemisphere of the brain. Statistical analyses were conducted using SAS version 9.4 (Cary, NC, USA). Separate regression analyses were conducted for each region with CES-D subscale scores as the predictors and the proportion of regional volume to estimated total intracranial volume as the outcome, controlling for sex and age. The depressed mood, somatic symptoms and lack of positive affect subscales of the CES-D were entered simultaneously in the same regression models to examine the association of each subscale while controlling for other scores. We applied a statistical significance threshold of  α ⩽ 0.05. Effect sizes are reported as omega squared ( ω 2 ), for which 0.0196 indicates a small effect, 0.059 a medium effect and 0.138 a large effect. 48  Follow-up analyses were conducted to examine the impact of including a vascular risk covariate in the models (coded 0 – 2 based on the presence of hypertension, high cholesterol or both) and after applying a square root transformation to address positive skewness of the CES-D subscale scores.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27070408">PMID:27070408</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstructions were generated using the software FreeSurfer 5.2 (freely available from  http://surfer.nmr.mgh.harvard.edu/ ). 23 ,  28 ,  29  In brief, raw image data voxels were sub-sampled to voxels of side 1   mm 3 . The data were then normalized for intensity, radio-frequency-bias field inhomogeneities were modeled and removed, followed by skull-stripping. The cerebral white matter was subsequently identified after which the hemispheres were separated, tessellated and deformed to produce an accurate and smooth representation of the gray – white interface. These surface reconstruction processes were conducted in native space. To correct for minor inaccuracies, the reconstructions were manually edited. Eight scans (three patients and five controls) were omitted from further analyses due to large errors or artefacts. Mean curvature was measured to divide the cortex into gyri and sulci. Gyri have a negative mean curvature; sulci have a positive mean curvature.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27070405">PMID:27070405</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The software package PMOD (v3.3, PMOD Technologies, Zurich, Switzerland) was used in the initial PET image processing and kinetic analysis. Inter-frame motion correction and PET-MRI co-registration were completed as previously described. 27  Based on the T1-weighted MR images, automated cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26479587">PMID:26479587</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image preprocessing was performed using the Automatic Analysis Toolbox 23 , which uses custom scripts combined with core functions from SPM8 ( www.fil.ion.ucl.ac.uk/spm ), FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) and FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ). The structural images were bias-corrected and de-noised using an optimized non-local means filter to improve image quality 24 . The unified realign and unwarp procedure, as implemented in SPM8 25 , was used to correct for head motion and voxel displacement due to magnetic-field inhomogeneity. Co-registration of the functional images with the structural image was performed with the following procedure: the structural image was co-registered to the T1 template, and the mean EPI was co-registered to the EPI template. This co-registered mean EPI was then co-registered to the structural image. The co-registration parameters of the mean EPI were applied to all functional volumes. Functional images were corrected for physiological noise with RETROICOR 26 . RETROICOR uses 10 cardiac phase regressors (5 th  order fourier set), 10 respiratory phase regressors (5 th  order fourier set), and 6 other nuisance regressors (heart rate frequency, heart rate variability, raw respiration data, respiratory amplitude, respiratory frequency, and frequency times amplitude of respiration). The FSL brain extraction toolbox was used to create a skull-stripped structural image. This structural image was segmented into grey matter (GM), white matter (WM) and cerebro-spinal fluid (CSF) with SPM8 27 . Mean-intensity values at each time point were extracted for WM and CSF and used as nuisance regressors in the GLM analysis (see below).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26301809">PMID:26301809</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All scans were anonymized at the time of acquisition. Image preprocessing followed a set protocol, and the analysis protocol was strictly adhered to regardless of the genetic or clinical status of the participant. Structural images were processed using FreeSurfer v5.3 ( http://surfer.nmr.mgh.harvard.edu/ ). Volumetric segmentation, cortical surface reconstruction, and cortical parcellation were used to quantify the thickness and surface areas of cortical anatomical regions. Cortical anatomical regions were defined by the Desikan-Killiany atlas ( 47 ). Subcortical volumes were defined as by Fischl and colleagues ( 48 ). Cortical surface area and the subcortical volumes were normalized by the total intracranial volume for each subject ( 49 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26161436">PMID:26161436</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Thickness of cortical structures and volume of subcortical structures were obtained for select regions of interest (ROIs) using the FreeSurfer image analysis suite version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ )  [8] ,  [9] . Briefly, the T1 MRI images were skull stripped and transformed into Talairach space. Then, using a standard atlas, the images are segmented, and volumes of subcortical regions, e.g., the hippocampus and amygdala, were obtained via this segmentation. Next, cortical surface meshes were created, defined as the gray/white matter boundary (the white matter surface) and the gray/CSF boundary (the pial surface). After topology correction and deformation of the surface meshes, the surfaces were parcellated based on a template atlas. Cortical thickness measurements were obtained by calculating the distance along a normal vector from each vertex in the white matter surface to the pial surface. The thickness values at each vertex within an ROI were averaged to obtain the ROI measurements. We focused on eight medial temporal and posteromedial cortex ROIs in this study. These ROIs were chosen because of their involvement in the AD cascade. Cortical thickness measurements were obtained from the entorhinal cortex, parahippocampal gyrus, fusiform gyrus, cingulate isthmus, posterior cingulate, and the precuneus, whereas volumetric measurements were obtained from the hippocampus and amygdala. Measurements were averaged across hemispheres to obtain a single value for each ROI.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25938132">PMID:25938132</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was done using the FreeSurfer software package, version 5.1.0 (available at:  http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures have been previously described  [23] ,  [24] ,  [25] ,  [26] . In brief, the T 1 -weighted spoiled gradient-recalled echo acquisitions were skull stripped and transformed into Talairach space. Next, surface meshes were created, which are defined by the gray/white matter boundary (the white matter surface) and the gray/cerebrospinal fluid boundary (the pial surface)  [27] . Parcellation and segmentation were completed using a predefined atlas  [28] . Subcortical volume measures were acquired from the segmentations, and cortical thickness measurements were obtained by calculating the distance along a normal vector from each vertex in the white matter surface to the pial surface. The thickness values at each vertex within a region of interest (ROI) were averaged to obtain the thickness of the ROI. A summary measure for each ROI was then derived by averaging the values from the right and left hemispheres. Each FreeSurfer output was visually inspected to ensure that the cortical reconstruction was accurate and without topologic defects. Our analyses focused on select ROIs known to be affected early in the AD cascade, such as the hippocampus and posterior cingulate ( Table   2 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25893207">PMID:25893207</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The goal of this study was to compare the shape of the   hippocampal surface, including its subfields, between   PPA   subjects and matched controls. We predicted   a spectrum   of hippocampal deformity in PPA based on the fact that some might have AD neuropathology, whereas others might not. We also predicted a greater degree   of deformity in the left hippocampus, reflective of the left   hemispheric focus of PPA. Study methodology involved   a new multiatlas FreeSurfer-initiated Large-Deformation Diffeomorphic Metric Mapping (ma-FSLDDMM) procedure for the mapping of the hippocampal structure, and the   assessment of nonverbal memory to correlate with hippocampal morphology. ma-FSLDDMM was based on our previously published single-atlas technique (sa-FSLDDMM)  [21] ,  [22] , for automated brain segmentation in high-resolution structural scans which combined initial FreeSurfer segmentation of gray and white matter structures ( http://surfer.nmr.mgh.harvard.edu ) with a smoothed approximation via Large-Deformation Diffeomorphic Metric Mapping (LDDMM)  [23] . Compared with single-atlas methods, approaches that combine maps from multiple atlases that best match any individual subject's scan features have been shown to improve segmentation accuracy and reduce biases  [24] . We chose the nonverbal Faces subtest in the Wechsler Memory Scale III (WMS-III)  [25]  to assess episodic memory. Although even an apparently nonverbal memory test can elicit internal verbalization  [26] , this choice avoided the pitfalls of using story or word-list recall to test episodic memory in patients with PPA  [19] ,  [27] . Finally, the  ε 4 allele of the apolipoprotein  ε  ( ApoE ) gene has been revealed as a risk factor for amnestic but not aphasic dementias  [28] . Because the genotype does not predict AD pathology in PPA, we hypothesized that the presence of   the  ApoE   ε 4 allele would not influence a patient's hippocampal shape or memory performance.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25833685">PMID:25833685</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were automatically segmented to generate total gray matter and total WMH volume data. First, the two copies of  T 1 -weighted MPRAGE data collected on each participant were spatially co-registered and averaged. Gray matter was automatically segmented using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ), as recently described elsewhere (Fleischman et al.  2014 ). The total gray matter volume was then measured for each participant and normalized by the corresponding intracranial volume (ICV) generated by FreeSurfer, by dividing by the corresponding ICV, as previously described (Fleischman et al.  2014 ). The total gray matter volume variable used in the analyses included cortical, subcortical, and other gray matter regions (including the cerebellum).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25721800">PMID:25721800</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical gray matter was assessed with cortical thickness analysis (CTA) using Freesurfer software v.5 ( http://surfer.nmr.mgh.harvard.edu/ ). The details of these methods have been described previously (Dale et al.  1999 ; Fischl et al.  1999 ; Fischl and Dale  2000 ). Pre-processing of T1-weighted scans included transformation to Talairach space, intensity normalization, hemispheric separation, and tissue segmentation. The gray matter/WM and gray matter/CSF border were identified and modelled as surfaces. Thickness was calculated by the software as the distance between the two borders along each point of the cortex in each hemisphere. Each subject ’ s cortex was anatomically parcellated and each sulcus and gyrus was labelled and aligned to Freesurfer ’ s average surface map. A 6-mm full-width half-maximum (FWHM) Gaussian spatial smoothing kernel was applied to the dataset, and a corrected threshold of  p   <   0.05 was used based on Monte Carlo permutations with 5000 iterations using AlphaSim ( http://afni.nimh.nih.gov/afni/ ) as previously used by our group (Moayedi et al.  2011 ; Erpelding et al.  2012 ). A general linear model (GLM) was used to assess group differences, with age included in the model as a variable of no interest. We also included impulsivity, aggression and mania measures as regressors of interest to determine the correlations between gray matter and behaviour. An ROI approach based on anatomical landmarks connecting the UF restricted the analysis to a mask consisting of the OFC areas (Brodmann areas 10, 11, 47) and temporal pole (Brodmann area 38) obtained from the cortical parcellation atlas (Brodmann) in Freesurfer.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27389402">PMID:27389402</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All patients were scanned on a 1.5-T Magnetom Avanto MRI scanner (Siemens, Erlangen, Germany) as previously described [ 22 ]. Full brain and skull coverage was obtained, and quality control was carried out on all images according to previously published criteria [ 30 ]. Data processing was performed with FreeSurfer 5.1.0 software ( http://surfer.nmr.mgh.harvard.edu/ ) as detailed elsewhere [ 22 ]. Visual quality control was performed, and an index of global brain atrophy was calculated as follows: brain volume (BV)/CSF index   =   (total grey matter volume   +   total white matter volume)/total CSF volume. Lower values of the BV/CSF index represent greater brain atrophy.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27375408">PMID:27375408</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>3 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27336300">PMID:27336300</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The amygdalae were segmented from the T1-weighted images using a rater-independent well-validated automatic segmentation method implemented in the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu )[ 39 ,  40 ]. This segmentation method consists of several steps, in which T1-weighted images are first processed by performing motion correction and intensity inhomogeneity correction. After removing non-brain tissue, gray matter and white matter are segmented. Then, segmentation of subcortical structures including the amygdalae was performed using an atlas-based approach by assigning each voxel of the preprocessed volume to the corresponding probabilistic information on the amygdalae based on manually labelled training sets [ 41 ,  42 ]. An expert visually inspected the final segmented images to assure the appropriateness of the results. Manual editing was performed to correct mislabeled voxels or prominent anatomical errors if necessary. Anatomical location for the bilateral amygdalae are presented in  Fig 1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27330968">PMID:27330968</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and cortical thickness measurement were performed with the FreeSurfer 4.4.0 image analysis suite, and a detailed description of preprocessing and processing is available at  http://surfer.nmr.mgh.harvard.edu /. Concisely, after skull-stripping and intensity correction, the gray-white matter boundary for each cortical hemisphere (with tissue intensity and neighborhood constraints) was identified. Then the surface boundary was tessellated to generate multiple vertices across the whole brain, and inflated. The expansion of the resulting gray matter surface created the pial surface with a point-to-point correspondence, and spherical morphing and spherical registration to an atlas were performed. Cortical thickness was computed by using the methods developed by Fishl and Dale ( Fischl and Dale, 2000 ). Cortical maps were finally smoothed with a 10-mm FWHM Gaussian kernel.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27325442">PMID:27325442</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We preprocessed MRI data using the Automatic Analysis framework ( https://github.com/rhodricusack/automaticanalysis/wiki ), which combines tools from SMP8 ( http://www.fil.ion.ucl.ac.uk/spm/software/spm8/ ), FreeSurfer (v5.1,  http://surfer.nmr.mgh.harvard.edu/ ) and the FMRIB Software Library (v5.0,  http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ), complemented by custom scripts. The preprocessing pipeline consisted of the following steps: we removed biases resulting from field inhomogeneities from the native structural images using the SPM8 new segment option. Furthermore, we denoised the structural images using an Adaptive Optimized Nonlocal Means filter (MRI denoising software package) 66 . Next, we performed a premasking procedure to exclude the neck from the structural image using a template image and ran a Freesurfer brain extraction and SPM segmentation procedure to obtain segmentation masks for grey matter, white matter, cerebrospinal fluid and out-of-brain voxels. Furthermore, we realigned and unwarped the functional images using the fieldmap images. In addition, we employed a spike-detection algorithm to record and later model signal spike events as nuisance variables. Functional and structural images were coregistered to a functional template (mean EPI) and a structural template respectively, after which the functional images were registered to structural space. We extracted the signal time course from white matter, cerebrospinal fluid and out-of-brain voxels and included these as nuisance variables. Field bias was removed from the mean EPI after which we performed a Freesurfer brain extraction procedure to obtain a brain mask. To account for inter-subject differences in brain morphology, we constructed a group structural template using the Advanced Normalization Tools toolbox (v1.9,  http://stnava.github.io/ANTs/ ). Subsequently, we used the parameters obtained via this procedure to later normalize our single-subject statistical maps to an intermediate common space as a final step, before transforming to MNI space using FMRIB's Linear Image Registration Tool and performing the group-level statistical analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27320849">PMID:27320849</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each scan was processed using  FreeSurfer  (  http://surfer.nmr.mgh.harvard.edu/ ) to obtain the LGI. Briefly, the LGI map can be obtained in four steps 8 . First, the pial surface is reconstructed in three-dimensional space. Second, an outer surface can be obtained from the outer hull which tightly wraps the pial surface. Third, the LGI was calculated for each vertex on the outer surface, as a ratio of areas of circular region centered on this vertex and the area of the corresponding region on the pial surface. Hence, LGI is able to quantify the amount of cortical surface invaginated in the sulci and measure the spatial frequency of cortical gyrification and the depth of the sulci. Fourth, the LGI map is obtained by propagating LGI values from the outer surface to the pial surface. For comparison, all of the individual reconstructed cortical surfaces were aligned to an average template by using a surface-based registration algorithm 25 . Then the LGI maps were resampled and smoothed with a heat kernel of 10   mm width.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27303358">PMID:27303358</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27298761">PMID:27298761</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al., 1999 ,  Fischl and Dale, 2000 ). Using this approach, the grey and white matter surfaces were defined by an automated brain segmentation process. An experienced investigator, who was blinded with respect to whether subjects were patients or controls, then manually corrected the automated segmentation. Cortical thickness was estimated at each point across the cortex by calculating the distance between the grey/white matter boundary and the cortical surface. Individual whole brain surface maps were then registered to a common FreeSurfer template surface (fsaverage) by the FreeSurfer spherical registration system ( Fischl et al., 1999 ) and smoothed with a 10   mm 2D Gaussian smoothing kernel ( Fischl et al., 1999 ). We compared a) patients with vs. patients without sensory aura and b) patients vs. controls in a vertex-wise GLM using an unpaired design while applying cluster-wise correction for multiple comparisons using a permutation-based non-parametric analysis (cluster-wise P   <   0.05). Age, gender, disease duration, and attack frequency were included in the model as nuisance variables. Additionally, we measured the volume of the thalamus, caudate, putamen, amygdala and hippocampus bilaterally for each subject, as well as the total intracranial volume, total grey matter volume, total cortical volume, and total subcortical grey matter volume using FreeSurfer's automated segmentation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27298741">PMID:27298741</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical Parcellation . Cortical reconstruction and volumetric segmentation were performed on the  T 1 -weighted 3D high-resolution anatomical images using the FreeSurfer analysis tools ( http://surfer.nmr.mgh.harvard.edu/ ). Briefly, this processing includes brain extraction, automated Talairach transformation, and segmentation of the subcortical WM and deep GM volumetric structures (including hippocampus, amygdala, caudate, putamen, and ventricles) [ 34 ]. Mean values of thickness (mm) and area (mm 2 ) were measured in 35 cortical areas based on the Desikan-Killiany Atlas [ 33 ,  34 ]. The children with hemiplegia included in this study had varying degrees of brain injury that in some cases prevented accurate alignment to a standard space. In these cases ( n  = 4) the anatomical images were modified by replacing the affected hemisphere with the CLH (just for the purpose of registration).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27271861">PMID:27271861</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer was used to obtain surface meshes of the boundary between the gray and white matter from T1 anatomical brain images ( http://surfer.nmr.mgh.harvard.edu ). After registering surface meshes into the DTI space, volumetric regions of interests (ROIs) were generated based on the Desikan atlas, which includes gray matter voxels of 34 anatomical regions of cortices 41 ,  42  and seven subcortical regions 41  for each hemisphere. Thus, we obtained 82 ROIs as nodes of the brain networks.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27271858">PMID:27271858</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural images were processed using the FreeSurfer's image analysis suite (5.3.0,  http://surfer.nmr.mgh.harvard.edu/ ). Processing steps included correction for magnetic field inhomogeneities, removal of non-brain tissue, polygonal tessellation of the white/gray matter and pial cortical surfaces and parcellation of the cortical mantle into distinct cortical units leveraging local curvature and other contextual information. 42 ,  43  For within-subject analysis, the FreeSurfer's longitudinal processing stream created an unbiased within-subject template across time points using robust, inverse consistent registration, 44  from which surface tessellation and parcellations were initialized. Each image was visually inspected, and topographical errors in surface extraction were corrected. Cortical thickness was measured using the shortest distance between the white – gray and pial interface at each vertex and projected to the tessellated pial surface smoothed using a 10-mm full-width at half maximum kernel. Methods for cortical thickness calculation have been validated against histological analysis 45  and manual measurements. 46  Notably, the use of volumetric navigators for prospective motion correction 41  and the longitudinal pipeline for cortical thickness analysis served to reduce the potential for motion-related artifacts.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27134768">PMID:27134768</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements across the whole cortex were obtained using FREESURFER tools ( http://surfer.nmr.mgh.harvard.edu/ ). Processing steps included removal of nonbrain tissue, segmentation of the subcortical white matter and deep gray matter volumetric structures, tessellation of the gray and white matter boundary, registration to a spherical atlas which is based on the individual cortical folding patterns to match cortical geometry across subjects, and creation of a variety of surface ‐ based data. Cortical thickness is calculated as the closest distance from the gray/white boundary to the gray/CSF boundary at each vertex on the tessellated surface (Fischl et   al.  1999 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27105060">PMID:27105060</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To identify visual field maps in individual participants, the representations of polar angle and eccentricity were visualized on surface reconstructions of both hemispheres and inspected. Surface reconstructions of the gray and white matter boundary of individual participant hemispheres were made using the Freesurfer4 autorecon script  http://surfer.nmr.mgh.harvard.edu/ ). Retinotopically organized maps were visible and present in all tested hemispheres. Notwithstanding subtle interparticipant variability, the main features of the maps, in particular the reversals in visual field representation at the vertical and horizontal meridians, were consistent across participants. In accordance with previous reports (DeYoe et al.,  1996 ; Engel et al.,  1994 ; Larsson  &  Heeger,  2006 ; Sereno et al.,  1995 ; Wandell et al.,  2007 ), retinotopic visual field maps were delineated using the following criteria: (a) the polar angle representations displayed reversals. That is, the representations of polar angle in neighboring visual areas were mirror reversals of one another, with a reversal in the representation along their shared boundary; (b) the polar angle and eccentricity components within each visual area were organized largely orthogonal to one another. The following visual field maps were identified in each hemisphere and participant (V1, V2d, V2v, V3d, V3v, hV4, V3A, V3B, V7 (IPS0), LO1, and LO2). In order to demonstrate the visual field coverage in our retinotopic maps, we utilized data from seven participants who also completed independent checkerboard mapping sessions. Visual field map boundary definitions, using the criteria described above, were made on the basis of this checkerboard mapping data. The resulting retinotopic boundaries were then applied to data obtained during our primary pRF sessions in the same participants. This independent sampling avoids circular analyses, which can arise when one defines the boundaries of visual field maps and subsequently plots visual field coverage from the same region and data — the extent of visual field coverage will reflect directly where the boundaries of a given visual area are drawn in that specific dataset.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27104668">PMID:27104668</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was measured automatically using FreeSurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu ). Structural brain maturation of TP1 was reported earlier using a prior version of FreeSurfer ( Koolschijn et al., 2014 ), therefore we reconstructed and reanalyzed all anatomical scans from the first wave using version 5.3. Details of the surface-based cortical reconstruction and subcortical volumetric segmentation procedures have been extensively documented previously ( Dale et al., 1999 ,  Fischl and Dale, 2000 ,  Fischl et al., 2004 ,  S é gonne et al., 2004 ). To extract reliable volume and thickness estimates, images where automatically processed with the longitudinal stream in FreeSurfer ( Reuter et al., 2012 ). Specifically, an unbiased within-subject template space and image (Reuter and Fischl, 2011) is created using robust, inverse consistent registration ( Reuter et al., 2012 ). Several processing steps, such as skull stripping, Talairach transforms, atlas registration as well as spherical surface maps and parcellations were then initialized with common information from the within-subject template, significantly increasing reliability and statistical power ( Reuter et al., 2012 ). To extract average cortical thickness from the FSL anatomical ROIs, we performed the following steps: (1) Each anatomical ROI (DLPFC, SMA, ACC and SPC) was registered automatically to the FreeSurfer  “ fsaverage ”  template with normalized mutual information and inspected for accuracy of registration. Of note, as FreeSurfer calculates cortical thickness per hemisphere, the ROIs were split into a left and right structural ROI. (2) Individual cortical thickness data was mapped to the  “ fsaverage ”  template. (3) Average cortical thickness in mm was extracted for each ROI and individual separately.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26876452">PMID:26876452</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For surface ‐ based analysis all synthetic MPRAGE T1w images were processed independently with the same default settings to measure cortical thickness using the open source FreeSurfer package ( http://surfer.nmr.mgh.harvard.edu /). Briefly, image processing included removal of non ‐ brain tissue using a hybrid watershed/surface deformation procedure [S é gonne et al.,  2004 ], automated Talairach transformation, extraction of the subcortical WM and deep GM structures (including hippocampus, amygdala, caudate, putamen, ventricles) [Fischl et al.,  2004 ], intensity normalisation [Sled et al.,  1998 ] and tessellation of the GM/WM boundary. The GM/WM and GM/CSF borders were placed where the highest intensity gradients defined the transition to the other tissue class [Fischl and Dale,  2000 ; S é gonne et al.,  2007 ]. Cortical thickness was calculated as the shortest distance from the GM/WM boundary to the GM/CSF boundary at each vertex on the tessellated surface [Fischl and Dale,  2000 ]. The cortical thickness maps were warped to standardised space and smoothed by convolution with an isotropic Gaussian kernel of 15 mm FWHM.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26401709">PMID:26401709</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness analyses were performed on T1-weighted 3D images using Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ). Briefly, T1-weighted 3D images were preprocessed with intensity variations correction, normalization, affine registration to the Talairach atlas, skull stripping, and segmentation of grey and white matter. The pipeline for longitudinal processing has been used that includes the creation of an unbiased within-subject template using robust, inverse consistent registration [ 9 ]. For cortical thickness, we used surface-based analysis of thickness values at each vertex. Surface-based analyses of cortical thickness were performed using Surfstat software ( http://www.math.mcgill.ca/keith/surfstat/ ) following the methodology previously used [ 10 ]. Cortical thickness maps were smoothed using a 20   mm surface-based kernel. The comparison of baseline cortical thickness between groups was carried out using a two-sample  t -test at each vertex. For longitudinal analyses, a paired  t -test was used. Statistics were corrected for multiple comparisons using the random field theory for non-isotropic images [ 11 ]. A statistical threshold of  p   <     0.005 was first applied (height threshold). An extent threshold of  p   <     0.05 corrected for multiple comparisons was then applied at the cluster level.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26063438">PMID:26063438</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging data were obtained at the Academic Medical Center in Amsterdam, using a 3.0   T Intera full-body MRI scanner (Philips Medical Systems, Best, The Netherlands) with a phased array SENSE 8-channel receiver head coil. A sagittal MP-RAGE (magnetization prepared rapid gradient echo (T1W, TR/TE   =   6.8/3.1   ms, FOV 252   ×   270   mm, 170 slices, voxel size 1.0   ×   1.1   ×   1.2   mm) was acquired for anatomical reference and volume analysis. Volumes were calculated automatically using the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ), as previously described (Fischl et al.  2002 ). Then, a sagittal 3D FLAIR scan (TR/TE/TI 4,800/355/1,650   ms, FOV 250  ×  250   mm, 321 slices, voxel size 1.1   ×   1.1   ×   0.56   mm, slice gap  − 0.56   mm) was acquired to score white matter abnormalities with the visual rating scale of Fazekas (range 0 – 3) (Fazekas et al.  1987 ). All ratings were performed by a neuroradiologist (L. R.) blind to the clinical data. A single voxel 1H-MRS of 12.0   ml was acquired using a fully automated point resolved spectroscopy (PRESS) sequence positioned in the bilateral medial frontal cortex directly superior to the corpus callosum, TE 2,000   ms, TE of 36   ms, NEX   =   64. Following, a DTI sequence was assessed along 16 nonlinear and 16 antipodal directions, as we described elsewhere. (de Ruiter et al.  2012 ). Pseudo-continuous ASL (pCASL) with background suppression was performed to assess CBF. The applied parameters are described elsewhere (Gevers et al.  2011 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25698109">PMID:25698109</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data preprocessing was performed using the FreeSurfer software package v. 4.5.0 ( http://surfer.nmr.mgh.harvard.edu ) ( Dale et al., 1999 ;  Fischl et al., 1999 ).  Fischl et al. (2002 ,  2004 ) have documented the processing steps in detail. In brief, all T1 images were first skull stripped using a surface deformation procedure to remove non-brain tissue, and transformed into Talairach space. Then, these images underwent segmentation of subcortical white matter (WM) and grey matter (GM) structures, and intensity normalization to correct for MR intensity non-uniformity. For surface-based morphometry, preprocessing involved tessellation of the GM/WM boundaries, automated topology correction, surface deformation to generate the GM/WM and GM/CSF boundaries based on signal-intensity gradients. The generated pial and WM surfaces were visually inspected for accuracy (by W. Li) and edited manually if necessary. These images were then inflated and registered to a spherical atlas in the MNI (Montreal Neurological Institute) 152-space and parceled on the basis of gyral and sulcal information derived from the brain atlas, resulting in values for volume, cortical thickness, and surface area ( Desikan et al., 2006 ). Finally, all these measurements were calculated in the native space by an inverse transformation. In the present study, we used grey matter volume as the structural attribute for construction of the brain connectivity matrix. Based on the clinical expertise of the clinicians (J. Li  &  J. Xian) and the existed literatures ( Beason-Held et al., 1998 ;  Burton, 2001 ;  Cavanna  &  Trimble, 2006 ;  Jeannerod, 2004 ;  Kawashima et al., 1995 ;  Lauter et al., 1985 ;  Nixon et al., 2004 ;  Stoeckel et al., 2009 ), fourteen ROIs, including the transverse temporal gyrus, superior temporal gyurs, middle temporal gyrus, inferior temporal gyrus, pars opercularis, pars triangularis, pars orbitalis, supramarginalgyrus, cuneus, fusiform gyrus, lateral occipital cortex, lingual gyrus and precuneus, were selected and categorized as auditory, language and visual regions, respectively (see  Table   2 ). To attenuate the variations brought by brain asymmetries, we considered the left and right sides of these regions together, calculating the average volume over the bilateral regions.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27635121">PMID:27635121</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject, we performed image preprocessing and computed cortical thickness using FreeSurfer v 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ) (Step A in Figure  1 ). Outer and inner cortical surface meshes were first constructed from T1-weighted MR data. The inner surface represented the boundary between white matter and cortical gray matter, and the outer surface was defined as the exterior of the cortical gray matter. As the outer surface was constructed by deforming the inner surface, the two surface meshes are isomorphic, with the same number of vertices and edge connectivity. Due to inter-subject variability of brain shapes, we resampled the surfaces with 40,962 vertices for each hemisphere using our in-house software (Cho et al.,  2012 ). The image preprocessing and cortical thickness computation process were manually checked and corrected by an expert neuroanatomist.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27633944">PMID:27633944</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer (version 5.1.0;  http://surfer.nmr.mgh.harvard.edu/ ) was used for cortical: thickness, surface area, and curvature analysis. The processing stream consisted of the removal of non-brain tissue, mapping to Talairach-like space, and segmentation of the gray-white matter and pial boundaries. These maps of measurements were obtained by reconstructing representations of the GM/WM boundary and the WM boundary to the GM/cerebrospinal fluid boundary, and then calculating the closest distance from those surfaces to each vertex on the tessellated surfaces. All subjects ’  data were resampled to the FreeSurfer default common surface template using a high-resolution surface-based averaging technique that aligned cortical folding patterns. Finally, the surface data were spatially smoothed using a Gaussian kernel of 10   mm full-width at half-maximum. Regional variations of the cortical thickness, cortical surface area, and cortical curvature were compared using paired samples t-test. The statistical parametric map was generated at  p   <   0.05 (FDR corrected for multiple comparisons).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27625311">PMID:27625311</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical images were co-registered with a mean functional image and used for surface reconstruction, which was performed with the Freesurfer image analysis software ( http://surfer.nmr.mgh.harvard.edu/ ). Data from the retinotopic mapping was used to define the borders between areas V1, V2 and V3/VP in each hemisphere 39 . Data from the localizer scans were used to define the ROIs within each visual area that responded more strongly to each stimulus location (see  Supplementary Material ). Functional data from both scans were projected onto flattened representations of each participant ’ s visual cortex. ROIs corresponding to the search grating locations or checkerboard distractor locations in each visual area were drawn in Freesurfer and the ROIs created were then imported into SPM8.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27622135">PMID:27622135</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) was used to process the T1-weighted images, including surface reconstruction and full brain segmentation ( Fischl et al., 2002 ) to obtain precise brain extracted volumes for registration.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27610082">PMID:27610082</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27599373">PMID:27599373</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For visualization purposes, group-level findings in MNI space were projected onto a flattened representation of the cortex for each of the brain hemispheres using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) and its average subject  “ fsaverage. ”</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27597821">PMID:27597821</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27588001">PMID:27588001</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27581062">PMID:27581062</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We calculated the volume of the corpus callosum (CC) and its subregions, and hippocampal volume using the FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ , v.5.3) following previous approach 72 . The automated procedures for subcortical volume measurements of different brain structures have been described previously 73  (see details in Supplementary Methods). This FreeSurfer-based automatic procedure is comparable with manual segmentation 73 . The volume of each brain structure was then calculated by summing up all voxels within the structure.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27570433">PMID:27570433</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In this sample of 681 participants with useable MRI data at a mean age of 72.64 years (SD = 0.72), we used Freesurfer v.5.3 ( http://surfer.nmr.mgh.harvard.edu/ ) and the Desikan-Killiany atlas (Desikan et   al.,  2006 ) to derive a measure of each participant's frontal lobe volume (summing the volumes of the following regions: superior frontal, middle frontal, rostral middle frontal, middle orbitofrontal, lateral orbitofrontal, frontal pole, rostral and caudal anterior cingulate and the inferior frontal pars opercularis, pars triangularis and pars orbitalis). We corrected the measure for intracranial volume (maximum healthy brain size in younger adulthood) to produce a proxy measure of frontal lobe atrophy.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27567641">PMID:27567641</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Three methods based on a three-dimensional, T1-weighted sequence have been applied to ALS: VBM, volume analysis and the measurement of cortical thickness. As a volume-based analysis, VBM involves the automatic segmentation of grey matter and inter-subject comparison of the local grey matter density [ 16 ,  17 ]. After all the data have been transformed in the same stereotaxic space, the images are partitioned and corrected for the separation between white matter, grey matter and CSF as a function of voxel intensity [ 17 ]. Measurement of cortical thickness is a surface-based technique analysis in which the white matter and pial surfaces are automatically extracted by applying image analysis packages such as Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27564416">PMID:27564416</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In both the sOPM and sSQUID simulations, simulated dipoles were located on the cortical surface, which was extracted from the anatomical MRI using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ; [ 51 ]). This meant that 21,401 individual dipoles could be located at the vertices of the triangular cortical mesh, with each dipole oriented perpendicular to the cortical surface. In order to simulate magnetic fields at the individual sensor locations, a multiple local sphere head model was used [ 52 ] in conjunction with the forward model derived by Sarvas [ 53 ]. Gradiometers were formed via a subtraction of the radial field components through each gradiometer loop (in the case of the sSQUID) or through two OPMs in the case of sOPM. Third-order gradiometers for the sSQUID were constructed using weighting coefficients from the MEG recording.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27552827">PMID:27552827</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data will be processed using a combination of in-house image processing tools developed in MATrix LABoratory(MATLAB) (Mathworks, Natick, USA) and tools available as part of Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) and FMRIB ’ s Software Library (FSL   FSL   version   5.0,  http://www.fmrib.ox.ac.uk/fsl ). The diffusion dataset will be corrected to get potential 3D head motion and eddy current distortion using FSL eddy correct. The toolbox implemented in MATLAB will be used to deal with diffusional kurtosis tensors [ 24 ]. Coefficients from these tensors will be estimated for each voxel using a previously described homogenous polynomials approach, in which positive diffusivity function, positive-definite estimated diffusion tensor, and constrained estimated apparent kurtosis will be done [ 24 ,  25 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27552587">PMID:27552587</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The MRI data were analyzed by using FreeSurfer software version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu ). This software package is almost completely automated and reliably computes cortical thickness. The data were motion corrected and intensity normalized. We performed segmentation of white matter and tessellation of the gray – white matter junction. Topological defects in the gray – white estimate were fixed. Then a deformable surface algorithm was applied to find the pial surface. We visually inspected the entire cortex in each subject and corrected any inaccuracies in segmentation manually. The reconstructed cortical surfaces were inflated to normalize interindividual differences in gyral or sulcal depth. Each reconstructed brain was morphed and registered to an average spherical surface representation so that sulci and gyri were optimally aligned and cortical thickness difference maps could be constructed on a common spherical coordinate system.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27552103">PMID:27552103</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For all research scans in this study, volumetric analysis was performed by trained research assistants using Freesurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ). Freesurfer is a semi-automated parcellation program in which cortical and subcortical volumes can be computed. Skull stripping, Tailarach transforms, atlas registration, spherical surface maps and parcellations were initialized with common information from the within – subject template [ 21 ]. The cortex was segmented using the Fischl and Desikan-Killiany atlases into 10 separate regional volumes of interest [ 21 ,  22 ]. Manual data quality checks were performed by trained researchers for each MR image to confirm proper segmentation, with manual corrections as deemed necessary by the technician.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27546195">PMID:27546195</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cerebral MRI data were preprocessed using Freesurfer v5.3 ( http://surfer.nmr.mgh.harvard.edu/ ). Removal of non-brain tissues was manually performed on a slice-by-slice basis in each participant to increase the accuracy of segmentation. Volumetric measures (mm 3 ) were obtained for left and right sides of the hippocampus and amygdala, two MTL regions that have shown reduced volume in aMCI patients that progressed to AD at 1-year follow-up 29 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27541829">PMID:27541829</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstructions were performed using previously published methods [ 29 , 30 ]. In brief, the reconstructions were generated as follows: Preoperative MRI was coregistered with postoperative CT imagery using the Statistical Parametric Mapping software package [ 31 ]. Reconstructions of the pial surface were then generated from the preoperative MRI using Freesurfer (freely available for download at  http://surfer.nmr.mgh.harvard.edu/ ) and custom Matlab (The Mathworks, Natick, MA) code. Electrode positions were then estimated in the postoperative CT and projected onto the reconstructed pial surface following previously described methods [ 30 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27535542">PMID:27535542</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T 1 -weighted brain MRI scans at baseline were acquired using a sagittal 3D MP-RAGE sequence following the ADNI MRI protocol [ 21 ]. As detailed in previous studies [ 22 ], a widely employed automated MRI analysis technique was used to process MRI scans: FreeSurfer V5.1 software ( http://surfer.nmr.mgh.harvard.edu/ ). FreeSurfer was used to process and extract brain-wide target MRI imaging phenotypes (region volume and cortical thickness) by automated segmentation and parcellation. The cortical surface was reconstructed to measure thickness at each vertex on surface. The cortical thickness was calculated by taking the Euclidean distance between the grey/white boundary and the grey/cerebrospinal fluid (CSF) boundary at each vertex on surface. For surface-based comparison of the cortical thickness, all individual cortical surfaces were registered to a common surface template, which was an average created from all cognitively normal control subjects. The cortical thickness was smoothed with 10   mm FWHM Gaussian kernel to improve the signal-to-noise ratio and statistical power.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27507940">PMID:27507940</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed on co-registered T1 images with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). Briefly, this processing involves motion correction and averaging (Reuter et al.,  2010 ) of T1 weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure (Segonne et al.,  2004 ), segmentation of the subcortical white matter and deep gray matter volumetric structures (Fischl et al.,  2004 ), tessellation of the gray matter white matter boundary, automated topology correction (S é gonne et al.,  2007 ), and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Fischl and Dale,  2000 ). Once the cortical models were completed, parcellation of the cerebral cortex into units with respect to gyral and sulcal structure (Desikan et al.,  2006 ) was performed. Successively, the obtained parcellation and segmentation of each individual were visually inspected and, if needed, manually edited by one of the authors.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27489874">PMID:27489874</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Magnetic resonance images (MRIs) were acquired on a GE X750 Discovery 3.0-T scanner with an eight-channel phased array head coil (General Electric, Waukesha, WI). Three-dimensional T1-weighted inversion recovery prepared spoiled gradient echo (SPGR) anatomic sequences were collected using the   following parameters: TI/TE/TR   =   450   ms/3.2   ms/8.2   ms, flip angle   =   12 ° , slice thickness   =   1   mm no gap, field of view (FOV)   =   256, matrix size   =   256   ×   256, yielding a voxel resolution of 1   mm   ×   1   mm   ×   1   mm. Temporal lobe regions of interest (ROIs) included superior, middle, inferior, temporal pole, hippocampus, parahippocampal, and entorhinal. These were derived from the T1 images using the Freesurfer image analysis suite, version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu/ ), with automated volumetric segmentation. Further technical processing details are described in previous publications  [15] ,  [16] . All images were visually inspected and edited (if necessary) by trained personnel to ensure that they were accurately reconstructed and without topologic defects. A summary measure for each region of interest was derived by averaging the values from the right and left hemispheres and then expressed as a percentage of intracranial volume (ICV) to account for differences in overall head size.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27487011">PMID:27487011</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Most diffusion analyses provide metrics that can be used to make inferences about the brain ’ s underlying microstructure. When studying motor function, such approaches require robust delineation of cortical motor regions and/or subregions (such as the hand knob). Cortical parcellation performs this role reliably in healthy individuals, but problems often emerge when applying analyses to individuals with brain pathology. These problems are twofold. Firstly, tools such as Freesurfer ( http://surfer.nmr.mgh.harvard.edu ) often fail to delineate the cortical surface when pathology or abnormal morphology is present. For example, in our experience, Freesurfer fails to delineate this surface acceptably in around 30% of children and adolescents with mild-to-moderate cerebral palsy, even after repeated manual adjustment. Consequently, the majority of studies measuring dMRI metrics in cerebral palsy (and similar conditions) so far have relied on manual-parcellation [ 8 ]. Unfortunately, manual parcellation is prone to bias or human error and is extremely time consuming, especially when pathology is present. The second issue is that cortical parcellation, both automated and manual, may be invalidated by functional reorganization that occurs in response to pathology ( Fig 1 ). Using coarser regions of interest (ROIs; such as the entire sensorimotor cortex) might improve confidence that tissue that is functionally-relevant to the hypothesis is included, but will likely also include tissue that is functionally irrelevant to the hypothesis at hand. Tracking towards the surface using multiple hand-drawn  ‘ include ’  ROIs is an alternative approach. This can often identify the corticomotor tracts reasonably, but is unable to reliably locate specific subregions of interest (such as those innervating the hand knob) and in a clinical situation often results in tracks that fail to reach the cerebral surface.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27462214">PMID:27462214</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27453930">PMID:27453930</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Magnetic resonance imaging (MRI) scans were obtained on a 3.0-Tesla Siemens (Siemens, Iselin, NJ) TIM Trio scanner equipped with a 12-channel head coil located at the UCSF Neuroscience Imaging Center and were available for a subset of participants (n   =   55). Whole brain images were acquired using volumetric MPRAGE sequence (repetition time, echo time, inversion time [TR/TE/TI]   =   2300/2.98/900   ms,  α   =   9 ° ). The field of view was 240   ×   256   mm, with 1   ×   1   mm in-plane resolution and 1-mm slice thickness. The T1 MPRAGE structural MR images were analyzed using the FreeSurfer 5.1 image analysis suite, which is documented and freely available for download online at:  http://surfer.nmr.mgh.harvard.edu . Previous publications have provided detailed descriptions and validation of the software  [21] ,  [22] ,  [23] . FreeSurfer is a surface-based structural MRI analysis tool that segments white matter and tessellates both gray and white matter surfaces  [24] . For the purposes of this study, the T1 image for each subject was processed through FreeSurfer, version 5.1 and then individually quality checked for anatomic accuracy of white or gray matter segmentation. Common geometric inaccuracies in white matter and pial surfaces were manually corrected using the built in editing packages of FreeSurfer. Medial temporal lobe volumes served as our primary region of interest (ROI; divided into left and right hemisphere) and included the entorhinal cortex, parahippocampal gyrus, and hippocampus from the Desikan atlas  [25] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27453817">PMID:27453817</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>EEG source imaging entails using the scalp's electrical data to estimate the locations of active sources in the brain, and has been recently reviewed by Michel and colleagues ( Michel et al. 2004 ). Source imaging of high-density EEG data (43) is increasingly used as a brain imaging method for research purposes ( Michel and Murray 2012 ) and has also been incorporated into clinical practice ( Brodbeck et al. 2011 ). The cortical generators of the average EPs were estimated for each participant and stimulus type using the Brainstorm toolbox ( Tadel et al. 2011 ). This toolbox was also used for analysis and visualization of the source reconstructed brain activation maps. The conductive head volume was modeled according to a 3-sphere method ( Berg and Scherg 1994 ). A standard set of electrode positions was used for the construction of the forward model. For all participants, for the head model calculation, source imaging and projection, we used the default anatomy in Brainstorm which is based on the Colin27 MRI volume provided by the Montreal Neurological Institute (MNI) ( Holmes et al. 1998 ) as an MRI was not available for our healthy participants. While using this template may have some limitations in children, and some pediatric MRI templates are now available, we opted for the default template which would allow others to consider using our approach more readily in healthy participants. For all brain-injured patients, as they underwent MRI, we also extracted the segmented brain surfaces from their T1-weighted MRI images using Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu /) or Brainsuite ( Shattuck and Leahy 2002 ) and used them to recalculate the head model and the source imaging. These were then used to compare the source activation results to the ones generated by using a canonical head model. Comparison of the source activation using the standard canonical MRI to that generated by using the individual participant MRI, when available, did not show qualitative differences, as can be seen in  Fig. 6  and as examined quantitatively in  Fig. 7 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445914">PMID:27445914</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional data were analyzed using AFNI ( Cox, 1996 ); program names appearing in parentheses below are part of the AFNI suite. Cortical surface models were created with FreeSurfer (available at:  http://surfer.nmr.mgh.harvard.edu/ ), and functional data projected into anatomical surface space using SUMA (AFNI and SUMA are available at  http://afni.nimh.nih.gov/afni ,  Saad et al., 2004 ;  Argall et al., 2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445798">PMID:27445798</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Semiautomatic software quantified ventricular and subcortical volumes, as well as total intracranial volume, total gray matter and white matter hypointensities (Freesurfer Image Analysis Suite,  http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl and Dale,  2000 ; Fischl et al.,  2001 ,  2002 ,  2004 ). Briefly, this processing included motion correction and averaging (Reuter et al.,  2012 ) of T1-weighted images, removal of non-brain tissue (S é gonne et al.,  2004 ), automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures of interest (including hippocampus, amygdala, anterior cingulate cortex, lateral ventricles) (Fischl et al.,  2002 ,  2004 ), intensity normalization (Sled et al.,  1998 ), delineation of the gray and white matter boundary, automated topology correction (Fischl et al.,  2004 ; S é gonne et al.,  2004 ) and surface deformation following intensity gradients for optimal placement of gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl and Dale,  2000 ). Measurement of ventricular volume is particularly amenable to segmentation due to the high signal intensity contrast between cerebrospinal fluid and surrounding brain tissue in T 1 -weighted MRI. Of note, ventricle volume segmentation included only the lateral ventricles. Adjusting regional volumes to intracranial volume accounted for the potential impact of sex and height on structural outcomes. Subcortical results were corrected for multiple comparisons at FDR,  p   <  0.05.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445764">PMID:27445764</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27429670">PMID:27429670</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The second technique  –  volumetric measurement  –  was performed using FreeSurfer (FS) software  –  version 5.3. ( http://surfer.nmr.mgh.harvard.edu/ ), which is widely available. It is composed of a set of automated tools (algorithms), which allow to obtain data about the brain and its structures, including their volume. The software performs, among other things, a complete and automatic segmentation of the brain cortex and subcortical structures using so-called neuroanatomical labels assigned to each voxel in the given volume ( Figure 2A, 2B ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27408803">PMID:27408803</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We created 3D cortical brain models for each subject by submitting preoperative high-resolution magnetic resonance imaging (MRI) scans to Freesurfer software ( http://surfer.nmr.mgh.harvard.edu / ). We coregistered MRI scans with postoperative CT images using SPM software ( http://www.fil.ion.ucl.ac.uk/spm/ ) and identified the stereotactic coordinates of each grid electrode using custom MATLAB scripts (The MathWorks Inc., Natick, MA). Finally, we visualized the cortical surface of each subject and ECoG grid locations using NeuralAct software  [29] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27408802">PMID:27408802</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We created a three-dimensional patient-specific cortical surface brain model by submitting the preoperative high resolution MRI scans to Freesurfer ( http://surfer.nmr.mgh.harvard.edu ). We identified the stereotactic coordinates of the standard subdural grid using SPM8 software ( http://www.fil.ion.ucl.ac.uk/spm/ ) and custom MATLAB scripts (The MathWorks Inc., Natick, MA), which coregistered the MRI scans with the postoperative CT scans. The high-density subdural grid contacts were coregistered with those of the standard subdural grid using scalp fiducial markers, an intraoperative neuronavigation system (BrainLab AG, Feldkirchen, Germany), and novel custom software  [32] . The electrode locations were then projected onto a three-dimensional brain model and custom NeuralAct  [33]  software ( Fig. 4 , right) to render activation maps of corresponding ECoG activity.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27408796">PMID:27408796</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and estimation of cortical thickness was performed on the coronal SPGR scans using FreeSurfer (version 5.1), an automated image processing pipeline that is documented and freely available online ( http://surfer.nmr.mgh.harvard.edu /). The technical details of these methods have been described in prior publications (e.g.,  Dale et al., 1999 ,  Fischl and Dale, 2000 ,  Fischl et al., 1999a ,  Fischl et al., 1999b ,  Fischl et al., 2004b ). Briefly, processing includes removal of non-brain tissue ( Segonne et al., 2004 ), segmentation of the subcortical white matter and deep gray matter structures ( Fischl et al., 2002 ,  Fischl et al., 2004a ), tessellation of the gray matter-white matter boundary, and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders ( Dale et al., 1999 ,  Dale and Sereno, 1993 ,  Fischl and Dale, 2000 ). Parcellation of the cerebral cortex into units based on gyral and sulcal structure ( Desikan et al., 2006 ,  Fischl et al., 2004b ) allows for representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/CSF boundary at each vertex ( Fischl and Dale, 2000 ). The procedures for the measurement of cortical thickness have been validated against histological analysis ( Rosas et al., 2002 ) and manual measurements ( Kuperberg et al., 2003 ,  Salat et al., 2004 ), and FreeSurfer morphometric procedures have been demonstrated to show good reliability and validity ( Desikan et al., 2006 ,  Han et al., 2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27362265">PMID:27362265</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional MR images were preprocessed using SPM8 (Wellcome Trust Centre for Neuroimaging, University College London). The T1 structural scan was segmented and underwent cortical reconstruction [ 23 ,  24 ] using Freesurfer (version 5.0.0,  http://surfer.nmr.mgh.harvard.edu ). Any further analyses were performed using software developed in-house based in MATLAB ( http://dx.doi.org/10.6084/m9.figshare.1344765 ). We projected functional data to the cortical reconstruction by identifying the voxel within the functional images that corresponded to the median location between the pial and white matter surface for each vertex of the cortical mesh. For each participant, we used a forward mapping approach to estimate pRF parameters for each vertex: center position in visual space (x,y), size of the center ( σ 1 ) and surround ( σ 2 ) component, the amplitude ratio of center and surround ( δ ), and an overall scale factor ( β ) [ 19 ,  20 ,  25 ,  26 ]. The predicted neural response was then convolved with the participant ’ s HRF fitted based on data from the HRF run. pRF parameters were then fitted to the time series for each vertex using a coarse-to-fine fitting approach in which we first performed an extensive grid search on heavily smoothed data followed by an optimization procedure applied to unsmoothed data. The final parameter maps were smoothed (FWHM 5mm on the spherical model) to reduce high frequency variability in the parameter estimates.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27355840">PMID:27355840</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 MPRAGE structural MRI scans were processed using FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) to delineate anatomical regions of interest (ROI;  S1 Table ) masks for multiple brain subregions on the MRI (coregistered to the THK5351 PET scan), in each participant ’ s native space. This was the manner in which we derived the gray matter cerebellum mask used as a reference region for DVR and SUVR calculations. An eroded white matter mask was created by smoothing the binary FreeSurfer white matter mask by an 8mm 3D Gaussian kernel, then thresholding and masking at a white matter probability of  p   >  .7.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27340852">PMID:27340852</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>GM volumes of the left and right hippocampus were extracted automatically using FreeSurfer 5.3 [ 41 ] ( http://surfer.nmr.mgh.harvard.edu ) followed by visual quality control. FreeSurfer analyses were manually checked by two raters and compared to an anatomical atlas. An agreement between raters was sought on all cases with low segmentation quality. We did not perform a formal inter-rater reliability analysis because ratings were not done completely independently. The hippocampus is strongly associated with changes in episodic memory performance and atrophies at an annual rate of approximately 3% [ 42 ]. The normalized sum of left and right hippocampal volume was calculated for each individual, using residuals from a linear regression between hippocampal volume (y) and total intracranial volume (x) as described by Jack et al. [ 43 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27325682">PMID:27325682</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>At the second level, the data were modeled with 6 conditions (GND8, GND4, GND2, FIG2, FIG4, and FIG8) with a design matrix including a subject-specific regressor and correcting for heteroscedasticity across conditions. We sought to identify brain areas whose activity increased parametrically with corresponding changes in coherence (i.e., over and above the changes in power associated with adding components). For this purpose, a parametric contrast [ − 8  − 4  − 2 +2 +4+8]/14 was used. Effectively, the contrast can be expressed as: 2  ×  (FIG2-GND2) + 4  ×  (FIG4-GND4) + 8  ×  (FIG8-GND8), thus targeting brain regions whose activity is parametrically modulated by rising temporal coherence (2  <  4  <  8) while controlling (by subtracting activity of matched GND signals) for the increase in power associated with the added figure components. We also used a simple  “ Figure versus Ground ”  contrast: [ − 1  − 1  − 1 1 1 1]/3. Statistical maps were initially thresholded at a level of  P   <  0.001 uncorrected, and peaks were considered significant only if they survived family-wise error (FWE) correction at  P   <  0.05 across the whole-brain volume. Because we had prior hypotheses regarding PT and IPS based on our fMRI results ( Teki et al. 2011 ), FWE correction was also applied using small volume correction (SVC;  Frackowiak et al. 2003 ) within these regions. Due to limitations inherent to the resolution of our source analysis, a (conservative) compound mask over PT and the adjacent Heschl's Gyrus (HG) was used. The corresponding cortical masks were determined by reference to the Juelich histologic atlas (for IPS;  Eickhoff et al. 2005 ;  Choi et al. 2006 ;  Scheperjans et al. 2008 ), and the Harvard – Oxford Structural atlases (for Heschl's gyrus and PT;  Desikan et al. 2006 ), available in FSLview ( http://surfer.nmr.mgh.harvard.edu/ ), thresholded at 10% probability. SVC-corrected regions are indicated by asterisks in Tables   1  and  2 .\n Table   1 MEG sources whose activity increased with coherence for the basic SFG stimulus</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27318138">PMID:27318138</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurements were made using the freely available software Freesurfer, version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ). The detailed procedure for the surface construction has been described and validated in previous publications ( Dale et al., 1999 ;  Fischl and Dale, 2000 ). Briefly, the image processing included intensity normalization, removal of nonbrain tissue, segmentation, surface inflation, and topological correction. Cortical thickness was then calculated as the closest distance from the grey/white boundary to the grey/CSF boundary at each vertex on the surface. Cortical thickness was smoothed with a 20-mm full-width at half height Gaussian kernel to reduce local variations in the measurements for further analysis. Surfaces were checked and manual edits were performed in cases of gross inaccuracies using the Freesurfer editing tools. Values for estimated total intracranial volume were also obtained from Freesurfer.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27306512">PMID:27306512</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Quantification of cortical thickness values was performed using FreeSurfer v.5.3.0 ( http://surfer.nmr.mgh.harvard.edu ) (Dale, Fischl,  &  Sereno,  1999 ; Fischl  &  Dale,  2000 ). This method involves vertex ‐ wise reconstruction of the white matter and pial surface and parcellation of the cortex into 34 regions of interest (ROIs) per hemisphere according to the Desikan – Killiany atlas (Desikan et   al.,  2006 ). An average cortical thickness value was then extracted for each ROI by reconstructing the gray – white matter boundary and the cortical surface; the distance between these surfaces was calculated individually at each point/vertex across the cortical mantle (see Supporting information for further details).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27230217">PMID:27230217</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural brain imaging was done within 285 days from the self-rated scales at T5 and within 46 days at T6. Structural brain imaging was performed on a 3-T General Electric scanner with a 32-channel head coil. High-resolution T1-weighted images were collected with a 3D fast spoiled gradient echo sequence using the following parameters: 180 slices with 1 mm thickness, TR 8.2 ms, TE 3.2 ms, flip angle 12 ° , field of view 25 cm  ×  25 cm. For segmentation and parcellation of cortical and subcortical structures the FreeSurfer version 5.3 tool was used  http://surfer.nmr.mgh.harvard.edu/ . In brief, this processing includes motion correction and normalization of the structural T1 weighted images. A hybrid surface deformation procedure removes nonbrain tissue and transforms the images to Talairach space. Tesselation of gray and white matter and intensity gradients places the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to other tissues. For further description, see  Dale et al. (1999 ). Images were automatically processed with the longitudinal stream in FreeSurfer ( Reuter et al. 2012 ) and brain volumes were adjusted for total intracranial volume ( Raz et al. 2005 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27177894">PMID:27177894</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was measured automatically using FreeSurfer5.3 ( http://surfer.nmr.mgh.harvard.edu/  (Dale et al.  1999 ; Fischl and Dale  2000 ). Details of the surface-based cortical reconstruction and subcortical volumetric segmentation procedures have been extensively documented previously (Dale et al.  1999 ; Fischl and Dale  2000 ; Fischl et al.  2004 ; Segonne et al.  2004 ). In short, the FreeSurfer pipeline performs motion correction on the T1-images, automatically removes non-brain tissues (Segonne et al.  2004 ), transforms volumetric data to a common atlas, performs intensity normalization, topology correction (Fischl et al.  2004 ; Segonne et al.  2007 ) and defines the boundaries of the gray/white matter and pial surface (Dale et al.  1999 ; Fischl and Dale  2000 ). The volumetric subcortical segmentation procedure automatically classifies brain tissue into multiple distinct structures such as cerebral and cerebellar gray and white matter, and subcortical structures (Fischl et al.  2002 ,  2004 ). For the purposes of the current study, automated image surfaces and segmentations were inspected and screened for quality control but were not manually edited, in order to maintain the objectivity of results. No large deformities such as failure to segment were found in this study. Lobar segmentations were automatically computed by FreeSurfer using the  “ mri_annotation2label with  – lobesStrict ” -command. Intracranial volume (ICV) was determined by a validated automated method known to be equivalent to manual intracranial volume estimation (Buckner et al.  2004 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27043685">PMID:27043685</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness and cortical volume measurements were calculated using the freely available software FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Using a surface-based approach, FreeSurfer can automatically segment the brain into different cortical regions of interest, and calculate average thickness in the defined regions. In brief, images underwent preprocessing including intensity normalization and skull stripping, which was followed by labeling of cortical and subcortical regions. FreeSurfer's main cortical reconstruction pipeline begins with registration of the structural volume with the Talairach atlas. 23  After bias field estimations and the removal of bias, the skull is stripped and subcortical white and gray matter structures were segmented. 24  Next, tessellation, automated topology correction, and surface deformation routines create white/gray (white) and gray/cerebrospinal fluid (pial) surface models. 25  These surface models were then inflated, registered to a spherical atlas, and used to parcellate the cortical mantle according to gyral and sulcal curvature. 26  The closest distance from the white surface to the pial surface at each surface's vertex was defined as the thickness. 26  Average cortical thickness, surface area, and total volume statistics corresponding to each parcellated region could then be computed. The accuracy of FreeSurfer's results was then assessed visually for the different subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26996778">PMID:26996778</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The results of the voxel-based comparisons were projected onto either the individual or the group average cortical surfaces using Freesurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26950278">PMID:26950278</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>UCSF FreeSurfer datasets were used to conduct association test of  HMGCR  genotypes with brain structure and the cerebral image segmentation and analysis were performed with the FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) based on the 2010 Desikan-Killany atlas [ 21 ]. This process mainly contained motion correction and averaging of multiple volumetric T1 weighted images (when more than one is available), removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles) [ 22 ], intensity normalization, tessellation of the gray matter white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally place the gray/white borders at the location where the greatest shift in intensity defines the transition to the other tissue class. The technical details of these procedures are described in prior publications [ 23 ]. we selected regions have been reported to link to AD closely, such as hippocampus, parahippocampus, middle temporal, posterior cingulate, precuneus, and entorhinal cortex as our regions of interest (ROI) to analyze their associations with  HMGCR  genotypes [ 24 - 26 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26538289">PMID:26538289</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were visually inspected and those with movement artefacts were removed from the data set. This included six subjects with TS and two control subjects, all of whom were under the age of thirteen. This leads to a sample size of 29 subjects with TS. We only included the control subjects that matched each subject with TS one ‐ for ‐ one, leading to 29 matched ‐ control subjects. Images were transformed into nifti format and then analysed using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) to determine cortical thickness measures. FreeSurfer can be used to pre ‐ process the image data by correcting for motion, extracting brain tissue, transforming into standard Talairach space, and performing intensity normalization. The normalized images are segmented into anatomical structures (e.g., amygdala, hippocampus, caudate, ventricles, and putamen). Then, surface deformation is performed by following intensity gradients to closely estimate the position of the grey matter – white matter and the grey matter – cerebrospinal fluid boundaries, which are defined by identifying the location of the greatest shift in intensity (for more technical details on FreeSurfer software, please refer to the following publications; Dale, Fischl,  &  Sereno,  1999 ; Fischl, Sereno,  &  Dale,  1999 ). Grey matter thickness was measured as the closest distance between these two boundaries (Fischl  &  Dale,  2000 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25414004">PMID:25414004</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) 22  was used on the T1 images to produce subcortical segmentation, cortical parcellation, and tissue-type masks (WM, GM, and other tissue), which were checked and manually edited for misclassification due to WM hyperintensities and common temporal region errors. T2 images were linearly coregistered to the T1 and FreeSurfer parcellations. The T2FLAIR images were masked with the WM segmentation and subcortical mask and thresholded to create the WM abnormality masks. The resulting WM hyperintensity masks were then overlaid on T2 and T2 FLAIR images and manually edited (W.V., S.A.G., and E.M.), after which a trained neurologist (W.V.) gave a final approval. T1 images were also acquired on 15 age-matched healthy volunteers and processed with the same pipeline for calculating atrophy ( Table ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27683563">PMID:27683563</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 scan was processed using FreeSurfer (available at  http://surfer.nmr.mgh.harvard.edu ), a freely available and extensively validated automated parcellation software program. The FreeSurfer analysis was completed with a single version (5.3.0) and a single OS (Redhat Enterprise Linus 6). Technical details on the FreeSurfer analysis are described elsewhere [e.g., Ref. ( 46 ,  47 )]. Neuroanatomical labels were automatically assigned to the volumes. This automatic labeling was based on probabilistic information from a manually labeled set and was previously shown to give similar results as when manually labeled ( 48 ). A standardized protocol (available at  http://enigma.ini.usc.edu/protocols/imaging-protocols/ ) was used to check the quality of the segmentations before the group analyses were performed. The volumes of the left and right hippocampus and the vmPFC [a combination of the by FreeSurfer defined medial orbitofrontal cortex and lateral orbitofrontal cortex ( 49 )] were used for the group analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27683010">PMID:27683010</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the FreeSurfer software package, version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu ) for image processing and for thickness and surface area analyses, using our previously reported process 59 . In brief, the stripped, intensity-corrected, subdivided volume was segmented to classify the white matter and to approximate the gray-white matter boundary for each cortical hemisphere. From this, a topologically correct, gray-white matter boundary (white) surface triangulation was generated 60 61 . Subsequently, a pial surface was generated using a deformable surface algorithm. The surface reconstruction results for each subject were visually checked before further analysis. We then obtained a vertex-wise cortical thickness map by calculating the shortest distance between the pial and white surfaces 62 63 . Vertexwise estimates of the surface area were calculated by assigning one-third of the area of each triangle to each of its vertices. For comparison, the reconstructed cortical surfaces for each individual were aligned to an average template by using a surface-based registration algorithm. The thickness and area maps were resampled and smoothed with a Gaussian kernel of full width at half maximum of 20   mm.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27644039">PMID:27644039</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each site administered a standardized structural MRI protocol (see  S2 Table  for scanner models and parameters). Image acquisition and processing techniques have been described previously [ 44 ,  45 ]. Briefly, high-resolution structural MRI included a three-dimensional T1-weighted scan, a T2-weighted volume, and diffusion-weighted scans with multiple b values and 30 directions. In this paper, we focus on the T1-weighted images. All neuroimaging data passed a standardized quality control procedure. Image processing and analyses were performed using a modified FreeSurfer software suite ( http://surfer.nmr.mgh.harvard.edu/ ) to obtain measures of cortical and subcortical volume and vertex-wise CT and SA [ 46 ]. Thirty-four cortical regions in each hemisphere were automatically parcellated by FreeSurfer using the Desikan-Killian Atlas [ 47 ], including the STG and fusiform gyrus. The IFG included the following Desikan-Killian parcellations: pars triangularis, pars orbitalis, and pars opercularis. CT of the IFG was computed by averaging CT for these parcellations, whereas SA of the IFG was computed by summing SA for these parcellations.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27608359">PMID:27608359</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We specified seven ROIs implicated in facial preference [ 14 – 20 ] according to anatomical data for each subject: the cingulate cortex (CC), lateral prefrontal cortex (LPFC), orbitofrontal cortex (OFC), occipitotemporal cortex, insular cortex, basal ganglia, and amygdala. LPFC was defined as the middle frontal gyrus plus the inferior frontal sulcus. OFC was defined as the orbital gyrus plus the orbital sulci. The occipitotemporal cortex was defined as the lateral occipitotemporal gyrus, the medial occipitotemporal gyrus, plus the occipitotemporal sulcus. The basal ganglia were defined as the caudate, the pallidum, the putamen, plus the nucleus accumbens. Voxels from the left and right hemispheres were merged for each ROI. The cortical regions were specified using an atlas on the BrainVoyager QX software [ 36 ]. A cortical surface for each subject was spatially normalized into a standard cortical surface using a cortex-based alignment method [ 37 ]. Then, the specified regions were projected into a native space for each subject. The subcortical regions were specified for each subject using an automated brain parcellation method [ 38 ] on the Freesurfer software ( http://surfer.nmr.mgh.harvard.edu ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26782053">PMID:26782053</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The procedures described in this section are summarized in  Figure 1 . In the first quality control step, original T1-weighted images were checked by visual inspection. We excluded images with insufficient brain coverage (field-of-view problem), those with low signal-to-noise ratios or any artifacts (for example, motion artifacts and magnetic susceptibility artifacts) and those with any abnormal organic findings (for example, large cerebellar cysts and cavum septum pellucidum). Through this process, 306 healthy subjects and 167 patients with schizophrenia were excluded. Next, T1-weighted imaging data that had passed the first quality control step were processed with FreeSurfer software version 5.3 ( http://surfer.nmr.mgh.harvard.edu ), as described previously. 44  Through this procedure, we obtained images of subcortical segmentation and regional volumes (for the bilateral lateral ventricles, thalamus, caudate, putamen, globus pallidus, hippocampus, amygdala, accumbens and the ICV). Fourteen healthy controls and three patients with schizophrenia were rejected owing to the failure of FreeSurfer processing. After that, two independent researchers visually inspected each segmentation image to exclude images with poor parcellation. One patient with schizophrenia was excluded by this approach. A total of 14 healthy controls and four patients with schizophrenia were excluded in the second quality control step (the quality control of FreeSurfer analysis). After the two quality control steps (inspection of original MRI data and screening for successful FreeSurfer processing), 1771 healthy subjects and 946 patients with schizophrenia remained for possible analysis in the study (available data remained as shown in  Supplementary Table 2 ). Among potential candidates, protocols with less than 50 total participants and those with less than 10 participants in either diagnostic group were excluded before the following meta-analysis, to minimize undue effects of very small groups. Finally, 1680 healthy individuals and 884 patients with schizophrenia from 15 protocols remained for inclusion ( Table 1 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24872554">PMID:24872554</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26306865">PMID:26306865</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25048627">PMID:25048627</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26156987">PMID:26156987</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27257515">PMID:27257515</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25705174">PMID:25705174</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26381159">PMID:26381159</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25449739">PMID:25449739</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26032888">PMID:26032888</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25687594">PMID:25687594</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26509362">PMID:26509362</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27069771">PMID:27069771</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26516605">PMID:26516605</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27895573">PMID:27895573</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>4 surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27874098">PMID:27874098</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The data from the moving bar runs were used to estimate the population receptive field (pRF) of each voxel in the functional volumes, using MrVista ( http://white.stanford.edu/software/ ). In this analysis, a predicted BOLD signal is calculated from the known stimulus parameters and a model of the underlying neuronal population. The model of the neuronal population consisted of a two-dimensional Gaussian pRF, with parameters  x0, y0 , and  σ , where  x0  and  y0  are the coordinates of the centre of the receptive field, and  σ  indicates its spread (standard deviation), or size. All parameters were stimulus-referred, and their units were degrees of visual angle. These parameters were adjusted to obtain the best possible fit of the predicted to the actual BOLD signal. For details of this procedure, see Dumoulin and Wandell 18 . By means of this method, we were able to estimate the coordinates of the receptive field centre, as well as the receptive field size, of each voxel. Once estimated,  x0  and  y0  were converted to eccentricity and polar-angle measures, which were overlayed on inflated cortical surfaces using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ) to identify the boundaries of retinotopic areas in early visual cortex 19 20 . In this way, we identified areas V1, V2 and V3 for the left and right hemisphere separately. For all analyses discussed below, only voxels for which the pRF estimation provided a good fit (i.e. variance explained  > 20%) were included.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27872811">PMID:27872811</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Analyses were conducted using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) and FreeSurfer Functional Analysis Stream (FS-FAST) software. Functional images were intensity-normalized, smoothed using a 3D 8-mm FWHM Gaussian kernel and aligned to the MPRAGE scan for each participant. MPRAGE scans were used to reconstruct inflated (2D) models of individual cortical surfaces using FreeSurfer segmentation, surface reconstruction, and inflation algorithms ( Dale et al., 1999 ,  Fischl et al., 1999 ). For group-level analysis, inflated cortical surfaces were registered to a template brain consisting of the averaged cortical surface of an independent sample of 40 adults from the Buckner laboratory at Washington University (St. Louis, MO) using the Freesurfer surface-based spherical coordinate system. Cortical activation was localized using automated surface-based parcellation software ( Fischl et al., 2004 ). We defined approximate anatomical boundaries for the dACC by dividing the ACC labels into dorsal and rostral segments by drawing a line perpendicular to the intercommissural plane at the anterior boundary of the genu of the corpus callosum ( Devinsky et al., 1995 ); VLPFC was defined by combining the pars opercularis and pars triangularis labels and DLPFC was defined as the rostral middle frontal label.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27872810">PMID:27872810</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Pial and WM surfaces reconstruction was performed using FreeSurfer, version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu /), according to a multi-step procedure that calculates the grey matter (GM)/WM border and the cerebrospinal fluid (CSF)/GM (pial) border in the 3D MEMPR volume ( Dale et al., 1999 ), currently the recommended anatomical sequence for FreeSurfer pipeline ( Fischl and Dale, 2000 ). We applied the FreeSurfer pipeline on 3   T MEMPR sequences since the pipeline for surface reconstruction from 7   T is not optimal due to large B1 inhomogeneities ( Fujimoto et al., 2014 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27843501">PMID:27843501</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Neuroimaging data processing was conducted using the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ). Three-dimensional cortical mesh models were reconstructed to obtain the interface between white and gray matter (white surface) and between gray matter and cephalospinal fluid (pial surface), and cortical thickness was measured as the distance between the two [ 53 ]. All surfaces were inspected and, if necessary, corrected by experienced users according to the gold-standard procedures. Cortical surfaces of each subject were resampled to an average spherical surface using an optimal registration algorithm based on the alignment of cortical folding patterns [ 54 ] and smoothed using a fill-width at half-maximum (FWHM) of 10   mm. Vertex-wise correlations were then computed between cortical thickness or local cortical volume and visual memory measures. Vertex-wise correlations were corrected for multiple comparisons using Monte Carlo permutation [ 55 ] at the threshold  p   <   0.05. In addition to these vertex-wise correlations, we also extracted cortical volume, total area, and average cortical thickness in specific regions of interest (ROI) corresponding to brain areas involved in the dorsal and ventral visual streams [ 13 ,  15 ]. For that purpose, we used the well-validated parcellation methodology by Desikan et al. [ 56 ] and selected regions of the superior parietal cortex as well as inferior and ventral (fusiform) temporal cortex (Fig.   1 ), which are involved in dorsal and ventral cognitive functions. Given the non-normal distribution of the BVRT error scores, we computed Spearman ’ s correlation coefficients between average cortical thickness, volume, and surface area in the superior parietal cortex and both BVRT spatial memory errors and CMS dorsal subtests scores. We further correlated the same morphometric measures in the inferior and ventral temporal cortex with the BVRT object memory errors and the CMS ventral subtests scores. Correlation analyses were conduced using age, gender, and scan type as covariates. As our aim was to associate the memory impairments observed in patients with 22q11DS with potential alterations in the ventral and dorsal streams, TD participants were not analyzed. Fig. 1 Regions of interest belonging to the dorsal (superior parietal cortex) and the ventral visual streams (inferior and ventral temporal cortex)</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27841320">PMID:27841320</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomic reconstruction of the cortical surfaces was performed with the FreeSurfer image analysis suite (version stable 5.3.0;  http://surfer.nmr.mgh.harvard.edu ) as described previously 61 62 . Visual inspection by raters unaware of the clinical diagnosis was performed to correct errors due to miss-classification manually. An automatic parcellation technique was used to subdivide each hemisphere into 34 labeled gyral components 63 , (node labels see  supplemental Table S2 ), with each representing a node in the network. After reconstructing the gray matter surfaces, we computed the cortical thickness of each node in all subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27830093">PMID:27830093</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to delineate the subcortical structures from high-resolution T1 scans, we used public domain software Freesurfer V5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) to segment the structural data. Each voxel in the T1 volume was assigned one of the 45 labels (for a complete list, refer to  http://freesurfer.net/fswiki/SubcorticalSegmentation ). These label maps were then projected back to fMRI data using the obtained coregistration information in preprocessing to define ROIs to extract the voxel data from corresponding FC maps. Labels denoting the basal ganglia (caudate, putamen, and pallidum) and thalamus were selected as ROIs for the analysis. The cerebellum was divided bilaterally and analyzed accordingly based on the AAL atlas [ 14 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27812129">PMID:27812129</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer (v5.1) toolkit ( http://surfer.nmr.mgh.harvard.edu/ ) [ 31 – 34 ] was used to process anatomical MPRAGE images from the subjects to construct white matter, pial, inflated, and spherical brain surfaces. Briefly, this processing includes spatial inhomogeneity correct ion, non-linear noise-reduction, skull-stripping [ 35 ], subcortical segmentation [ 36 ,  37 ], intensity normalization [ 38 ], surface generation [ 31 ,  32 ,  39 ], topology correction [ 40 ,  41 ], surface inflation [ 32 ], registration to a spherical atlas [ 33 ] and thickness calculation [ 34 ]. This approach matches morphologically homologous cortical areas based on the cortical folding patterns with minimal metric distortion and allows sampling at subvoxel resolution and detection of cortical thickness differences at the sub-millimeter level. Cortical thickness was estimated at each point across the cortical mantle by calculating the distance between the gray/white matter boundary and the cortical surface.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27803816">PMID:27803816</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted structural MRI images for each subject underwent full reconstruction procedure using the Freesurfer image analysis suite (publicly available at:  http://surfer.nmr.mgh.harvard.edu/ ), resulting in generation of a high-definition cortical layer and brain-skull-skin layer. The high-density cortical layer mesh was downsampled to ~10,000 vertices per hemisphere and used as the source space, where each vertex location corresponds to a dipole source oriented perpendicular to the surface. A lead field matrix  G  was computed via a forward calculation using the cortical source space, a 3-layer head model, and 64-channel electrode locations coregistered to the model.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27798694">PMID:27798694</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer (Martinos Center for Biomedical Imaging, Harvard-MIT, Boston; available at  http://surfer.nmr.mgh.harvard.edu/ ) consists of surface based analysis [ 13 ] and volumetric segmentation. [ 10 , 14 ] It involves intensity non-uniformity correction [ 15 ], affine transformation to a MNI305 template, intensity normalization, removal of non-brain tissue [ 16 ], linear and non-linear transformations to a probabilistic brain atlas and labeling of cortical and subcortical structures. [ 10 , 14 ] It uses a Markov Random Field model for each structure for each point in space. Spatial localization priors are used in determining the right label per voxel. [ 17 ] Since Freesurfer version 5.2, surface-based calculations are used to calculate various brain volumes to get better accuracy. In our study, segmentation was performed using default settings (i.e. using the command:  ‘ recon-all ’ ). For our study, we used the compartment measurements reported by Freesurfer. All volumes were extracted from the stat files that Freesurfer produces using the  ‘ asegstats2table ’  command. Since Freesurfer estimates ICV and does not perform segmentation of extracerebral CSF, we obtained the CSF volume by subtracting TBV from the estimated ICV.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27776388">PMID:27776388</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The MRI data were preprocessed with the FMRIB Software Library (FSL,  www.fmrib.ox.ac.uk/fsl ) version 5.0, Analysis of Functional NeuroImages ( http://afni.nimh.nih.gov/afni/ ), and the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) version 5.3. Because our participants had separate T1 and resting functional magnetic resonance (fMR) images before and after the EAA/T, the data from each time point were separately processed in the following steps. First, a high-resolution T1 image of each participant was reconstructed to the cortical surface using methods based on the FreeSurfer processing pipeline: registration to the Talairach atlas, bias field correction, skull stripping, intensity normalization, surface modeling, and spherical mapping and registration. 23 , 26 – 28)  Second, slice timing correction, deobliquing, and motion correction were sequentially applied to the fMR data in each participant ’ s space. The whole image was normalized by mean intensity and scaled by a factor of 10,000, and linear and quadratic trends from each signal were removed. Next, the each fMR image was co-registered with the T1 images using a transformation matrix created from boundary-based registration. Finally, 5 major components from time course of white matter and cerebrospinal fluid were regressed out from the fMRI data using principal component analysis. This processing method can significantly reduce physiological noise as well as noise from other sources. 29 , 30)</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27766076">PMID:27766076</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27757129">PMID:27757129</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All images were processed using FreeSurfer (v. 5.3.0) with the default settings ( http://surfer.nmr.mgh.harvard.edu ) on a Mac Pro 64-bit OS X platform. The methods of surface-based analysis using FreeSurfer are described previously. 21  In brief, a T1-weighted magnetization-prepared rapid gradient-echo (MPRAGE) scan was acquired from each participant and was aligned. Processing involved a surface-based method that calculated the distance between gray and white matter outlines after segmentation of gray matter, white matter, and cerebrospinal fluid (CSF) space. All images were aligned to a common surface template using a surface-based averaging technique that aligned cortical folding patterns. 22 23</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27755551">PMID:27755551</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We created one seed mask and nine target masks for each hemisphere from the automated anatomical labeling template [ 26 ] and previous studies [ 27 ,  28 ]. These masks included the following regions (the numbers in parentheses refer to AAL map values and correspond to the left hemisphere; subtract 1 for the right hemisphere; also note that some masks were combined): the striatum (72, 74), medial orbitofrontal cortex (mOFC;28, 6, 26), lateral orbitofrontal cortex (lOFC;10, 16), ventrolateral prefrontal cortex (vlPFC;14), dorsolateral prefrontal cortex (dlPFC; 14), posterior cingulate cortex/retrosplenial cortex (PCC; 68), rostral cingulate cortex (rostral CC, 32), dorsal cingulate cortex (dorsal CC,34), hippocampus (38), and amygdala (42).  S1 Fig  (see  S1 Dataset  for the brain masks) shows the anatomical locations of these brain masks. Due to the fact that probabilistic tractography must be conducted in individual participants ’  diffusion space using the FMRIB Diffusion Toolbox, all masks in the MNI standard space were transformed to individuals ’  diffusion space by using transformation matrix and warp field produced in the previous step and voxels within masks were assigned the value of 1 and outside voxels were assigned the value of 0. Finally, volumes of these ten masks in individuals ’  diffusion space were obtained. The total intracranial volume (ICV) was obtained from high-resolution T1-weighted anatomical image by using the Freesurfer segmentation software package ( http://surfer.nmr.mgh.harvard.edu/ ) [ 41 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27752514">PMID:27752514</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>WMH and total gray matter volumes were obtained, as previously described. 10  Briefly, the two copies of T 1 ‐ weighted MPRAGE data collected on each participant were spatially coregistered and averaged. The result was then registered to the T 2 ‐ weighted FLAIR data using affine registration (FLIRT, FMRIB [Oxford Centre for Functional Magnetic Resonance Imaging of the Brain], University of Oxford, UK). The brain was extracted from the coregistered MPRAGE and FLAIR image volumes (BET, FMRIB, University of Oxford). WMH were automatically segmented for each participant using a support vector machine classifier using T 1 ‐ weighted MPRAGE and T 2 ‐ weighted FLAIR information (WMLS, SBIA [Section of Biomedical Image Analysis], University of Pennsylvania). The total WMH volume for each participant was normalized by the corresponding intracranial volume (ICV) generated by FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) based on the averaged raw T 1 ‐ weighted MPRAGE data. The measure was then logarithm transformed to reduce skewness. Gray matter was automatically segmented for each participant using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) and the total gray matter volume also was normalized by the corresponding ICV generated by FreeSurfer.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27752275">PMID:27752275</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness analyses were performed on the three-dimensional model of cortical surface reconstructions computed from T1 images using the FreeSurfer 5.0 software package (Massachusetts General Hospital, Boston, U.S.,  http://surfer.nmr.mgh.harvard.edu ). The details of technical aspects in these procedures have been described in previous publications [ 26 – 30 ]. Briefly, the implanted processing stream involved motion correction of volumetric T1-weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure, automated Talairach transformation of each subject ’ s native brain, segmentation of the gray matter – white matter volumetric structures, inflation of cortical surface to an average spherical surface to locate both the pial surface and the gray matter – white matter boundary, intensity normalization, and automated topology correction. Transition of gray/white matter and pial boundary was indicated by detecting the greatest shift in intensity through surface deformation. The cortex of each subject was then visually inspected. The entire cerebral cortex was parcelated into units based on gyral and sulcal structure. The computed cortical thickness was defined as the shortest distance between the pial surface and the gray matter – white matter boundary at each given point across the cortex. The cortical maps were generated by computing mean cortical thickness for each subject at each vertex, right and left hemispheres separately, and mapping these data to the surface of an average brain template enabling visualization of data across the entire cortical surface. Smoothing with a Gaussian kernel of 10   mm full width at half-maximum was performed on the cortical maps of each subject for the entire cortex analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27747153">PMID:27747153</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For the current analysis, T1-weighted images were processed using the FreeSurfer v5.3.0 ( http://surfer.nmr.mgh.harvard.edu / ) recon-all pipeline. Detailed description of FreeSurfer algorithms are available from the published literature ( Dale et al., 1999 ,  Fischl, 2012 ,  Fischl and Dale, 2000 ,  Fischl et al., 2004 ). In summary, after correction for magnetic field inhomogeneities, skull stripping and intensity normalisation, surface reconstruction is achieved through segmentation of the boundary between subcortical white matter and grey matter based on intensity differences. Next, a triangular mesh is generated to construct a three dimensional representations of the cortical sheath. Defects in brain mask, GM or WM volumes were manually corrected if necessary and the surface generation steps were repeated on the corrected volumes. All surface reconstructions were visually inspected and incorrect GM/WM segmentation was corrected if necessary following the FreeSurfer guidelines ( https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/TopologicalDefect_freeview ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27736900">PMID:27736900</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Twenty one subjects were scanned for anatomical MRI, DWI, and resting-state fMRI. Data were preprocessed using a standard approach (see  S1 Text ,  § 1). The brain was parcellated using Freesurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) [ 76 ] and the procedure described in [ 77 ], with a parcellation of 160 regions. These regions were further subdivided into two sets of smaller compact regions, yielding two finer partitions of the cerebral cortex into 461 and 825 regions (about 6 and 3 cm 2  each respectively). A probabilistic white matter fiber tracking method [ 78 ] implemented in FSL was used to track all possible connections between all pairs of regions. An index of structural connectivity between two regions was then defined as the proportion of fiber samples connecting these two regions per unit surface. This index was further divided by the average fiber length to reduce bias towards longer fibers. This structural connectivity index allowed to build an individual structural connectivity matrix  D  for each subject,  D rs  being the structural connectivity index from regions  r  to region  s , with no self-connections (i.e.,  D rr  = 0).  D  was then thresholded at 0.001; supra-threshold values were kept as such. In fMRI, the time series of all voxels within a given region were spatially averaged to form the representative signal of that region. Data were considered with and without regression of the global signal.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27733896">PMID:27733896</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical data that provided a canonical volume were acquired with different procedures for the fMRI study and for the follow-up TMS study. For the fMRI study, a 3D-Fast Spoiled Gradient-Recalled Echo (FSPGR) sequence was used to acquire multi-average, whole-head T1-weighted anatomical volumes for each participant (repetition time [TR]   =   7.8   ms, time to echo [TE]   =   3   ms, TI   =   450   ms, field of view [FOV]   =   290   ×   290   ×   176, 256   ×   256   ×   176 matrix, flip angle   =   20 ° , 1.13   ×   1.13   ×   1.0   mm 3 ). Data were obtained with an eight-channel head coil. For the TMS study, we adopted a revised protocol that we now routinely use to increase tissue contrast for automated segmentation: Three whole-head T1-weighted anatomical volumes were acquired for each subject (TR   =   7.8   ms, TE   =   2.7   ms, TI   =   600   ms, flip angle   =   12 ° , FOV   =   256   ×   256   ×   176, 256   ×   256   ×   176 matrix, 1   ×   1   ×   1 mm 3 ) using a 16-channel (half-head coil) and averaged. One T2*-weighted fast gradient recalled echo scan was also acquired (TR   =   400   ms, TE   =   4.3   ms, flip angle   =   25 ° , field of view   =   290   ×   290   ×   176, 256   ×   256   ×   88 matrix, 1.13   ×   1.13   ×   2 mm 3 ) using a 16-channel head coil. Average T1 data were divided by the T2* data in order to correct for signal gradient resulting from the signal dropout of the 16-channel coil and to improve white/gray matter contrast. One whole-head eight-channel T1-weighted volume was acquired for each subject for use in coregistration for TMS (TR   =   7.8   ms, TE   =   2.9   ms, TI   =   450   ms, flip angle   =   20 ° , FOV   =   290   ×   290   ×   176, 256   ×   256   ×   176 matrix, 1.13   ×   1.1.13   ×   1   mm 3 ). For both methods of acquiring anatomical data, the average T1-weighted anatomical volume was segmented into white and gray matter for each hemisphere using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). The subsequent gray-white matter segmentation was hand edited and checked for topology errors using itkGray ( http://white.stanford.edu ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27733825">PMID:27733825</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27733562">PMID:27733562</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We measured cortical thickness across the cortical mantle at approximately 300,000 vertices using FreeSurfer v5.30's ( surfer.nmr.mgh.harvard.edu ) cross-sectional pipeline. The procedure used by FreeSurfer for the surface construction has been described and validated previously. 12 , 13  Two modifications to the standard FreeSurfer processing stream were undertaken: a locally generated brain mask was used for skull stripping and ventricular segmentations were added to the white matter mask to improve cortical segmentation. Cortical thickness was smoothed with a 20-mm full-width at half-maximum kernel. All cortical segmentations were visually inspected. Mean cortical thickness values were calculated for 34 parcellated cortical regions bilaterally.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27723823">PMID:27723823</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Modeling and volumetric estimations of cortical brain regions were performed on structural T1 images using the Freesufer image analysis package version 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ). This automated tool performs imaging intensity normalization, removal of non-brain tissues, segmentation of cortical and subcortical brain regions into white and grey matter, spherical surface-based intersubject registration, which is based on the cortical surface curvature (sulci and gyri), and, finally, an automated parcellation of the cortical surface. Quality control of the Freesurfer output was done by visual inspection. When errors were noted the first approach was to edit these errors and, if errors remained after rerun of the edited output, the images were discarded from further analysis. The Query Design Estimate Contrast (QDEC) tool was used to do a general linear model (GLM) analysis at each vertex of the cortical surface. The dependent variable was cortical thickness. In the group comparisons, no nuisance variables were entered in to the model as groups were balanced with regards to gender and age. The results of the GLM analysis were corrected for multiple comparisons at the cluster level using the Monte Carlo method for p-cluster at  p   <  0.01 (z-vertex 2.0). In the correlation analysis between cortical thickness and clinical and neuropsychological data, age was entered as a nuisance variable and corrections for multiple comparisons were at the  p   <  0.01 (z-vertex 1.3) level.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27721908">PMID:27721908</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each study participant underwent brain MRI at a 3.0-Tesla Philips Medical Systems Achieva scanner equipped with an 8-channel head coil (InVision Imaging, Honolulu). A high-resolution anatomical volume was acquired with a sagittal T1-weighted 3D turbo field echo (T1W 3D TFE) sequence (echo time TE/repetition time TR=3.2 ms/6.9 ms; flip angle 8 ° ; slice thickness 1.2 mm with no gap; in-plane resolution 1.0 mm 2 ; field of view 256  ×  256 mm 2 ). To obtain regional brain volumes, MRI data were processed with FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ , version 5.0) [ 24   –   26 ] in a procedure that includes intensity normalization [ 27 ], Talairach transformation, subcortical white matter (WM) and deep gray matter (GM) segmentation [ 25 ], and cortical gray/white matter boundary and pial surface reconstruction [ 24 ]. Quality assurance of processed data was performed visually. Cortical surfaces and subcortical segmentations were checked prior to statistical analysis. FreeSurfer ’ s estimate of intracranial volume (ICV) can be reliably used to normalize regional brain volumes [ 28 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27721750">PMID:27721750</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>One seed mask and nine target masks, including the striatum, mOFC, lOFC, dlPFC, vlPFC, rostral CC, dorsal CC, PCC, hippocampus, and amygdala, were created for each hemisphere based on the automated anatomical labeling template (Tzourio-Mazoyer et al.,  2002 ) and previous studies (Cohen et al.,  2009 ; Lei et al.,  2014 ). Detailed information of these masks was presented in our previous study (Lei et al.,  2014 ). Figure  S1  shows the anatomical locations of these brain masks for one hemisphere. All masks in the MNI standard space were transformed to individual diffusion space by using transformation matrix and warp field produced in the previous step and binarized. Volumes of these ten masks in individuals' diffusion space were obtained. The total intracranial volume (ICV) was obtained from high resolution T1-weighted anatomical image by using the Freesurfer segmentation software package ( http://surfer.nmr.mgh.harvard.edu/ ; Fischl et al.,  2002 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27721745">PMID:27721745</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Brodmann areas and anatomical segmentation were both used to perform the cortical parcellation. DSI Studio uses linear transformation to register Talairach atlas on subject's diffusion space. The Talairach coordinate system provides anatomical and functional information (Nowinski,  2005 ). For comparison, FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) was used to segment cortical gyral ROIs based on previous brain atlases using each participant's T 1 -weighted magnetization prepared rapid axial gradient echo (MPRAGE) image (Desikan et al.,  2006 ). For the template, ICBM-152 T 1  template image was performed in FreeSurfer (Fonov et al.,  2011 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27721203">PMID:27721203</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation of MRI data were performed using the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) ( Fischl and Dale, 2000 ,  Fischl et   al., 1999 ). The initial processing of T1 MRI images, for each subject and each time point (baseline and follow-up), includes the following steps: removal of nonbrain tissue, automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures, intensity normalization, tessellation of the gray matter/white matter boundary, automated topology correction and surface deformation to optimally place the gray matter/white matter and gray matter/CSF boundaries. The cortical thickness was calculated as the closest distance from the gray/white matter boundary to the gray/CSF boundary at each vertex. The cortical thickness maps were smoothed using a 10-mm full width half maximum Gaussian. All surface models in our study were visually inspected for accuracy, and manual corrections were performed in the event of tissue misclassification/white matter errors while blinded to diagnostic group information. Ten subjects (5   controls and 5 LLD subjects) who had excessive pial or white matter surface segmentation errors after the manual correction were excluded from the analyses. Thus, the final sample was 25 controls and 33 LLD. To define the nodes for subsequent network analyses, the cortical thickness map of the cerebral cortex was parcellated using the Desikan-Killiany atlas, resulting in 34 regions of interest (ROI) for each hemisphere, each corresponding to the average cortical thicknesses of a gray matter region ( Desikan et   al., 2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27708577">PMID:27708577</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>2 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27703766">PMID:27703766</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To obtain whole hippocampal volumes,  T 1  images were automatically processed in FreeSurfer v5.3 ( http://surfer.nmr.mgh.harvard.edu ) using the hippocampal-subfields 22  and longitudinal 23 , 24  streams. The  T 1  images were reprocessed using an independent pipeline, the MAGeT-Brain (Multiple Automatically Generated Templates) algorithm, 25  based on subfield definitions derived from the Chakravarty Laboratory; 26  see  supplementary data  material for a brief summary of this technique. The hippocampal labels derived from MAGeT-Brain were found to strongly overlap with labels manually derived as well as with those automatically derived using both FreeSurfer and FSL. 27  Subfield volumes derived from each technique were summed to obtain left- and right-side hippocampal volumes; values from Scan 1 were subtracted from Scan 2 to present volumes as a function of change over time. Finally, total intracranial volume (TIV) was estimated using FreeSurfer for patients and controls.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27699206">PMID:27699206</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>All subjects were scanned in a horizontal 3 T scanner (Magnetom Trio Tim System, Siemens). Gradient EPI sequences were used to acquire functional images (TR, 2000 ms; TE, 30 ms; flip angle, 90 ° ; 3.0 mm isotropic voxels; and 33 axial slices). For functional connectivity scans, each TR was 3000 ms. Forty-seven axial slices were acquired in each TR. In the fMRI scans, the field of view included the whole brain, for all subjects. A 3D MP-RAGE sequence (1.0 mm isotropic) was also used for high-resolution anatomical imaging from the same subjects. Functional and anatomical data were preprocessed and analyzed using FreeSurfer and FS-FAST (version 5.3;  http://surfer.nmr.mgh.harvard.edu/ ;  Fischl, 2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27676441">PMID:27676441</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The 2011 MRI was obtained on a 3-Tesla Siemens Verio MR system (Siemens Medical Solutions, Erlangen, Germany) at a Quest Imaging Center. Sequences included high-resolution T1-weighted MPRAGE images (TR=2300   ms, TE=3.1   ms, 160 sagittal slices, 1.2   mm slice thickness, voxel-size of 1.33  ×  1.33  ×  1.2   mm 3 , flip angle=9 ° ). The 2015 MRI was obtained on a 3-Tesla Siemens Biograph MR system at the Icahn School of Medicine at Mount Sinai (ISMMS) Hess Center for Science and Medicine. Comparable high-resolution T1-weighted MPRAGE images were acquired (TR=2300   ms, TE=2.98   ms, 176 sagittal slices, 1.2   mm slice thickness, voxel-size of 1  ×  1  ×  1.2   mm 3 , flip angle=9 ° ). The duration of time between the two scans was 3.9 years. Longitudinal analysis of cortical thickness and subcortical volume was performed using the FreeSurfer reconstruction pipeline ( http://surfer.nmr.mgh.harvard.edu , version 5.3.0). Processing of individual time points included removal of non-brain tissue, automated Talairach transformation and intensity normalization, and detection of gray matter/white matter and gray matter/cerebrospinal fluid boundaries. Cortical thickness was calculated as the distance between these tissue boundaries (white matter and pial surfaces). The reconstructed surface models produced by the pipeline were inspected visually for accuracy and corrected manually using FreeSurfer's FreeView software ( http://surfer.nmr.mgh.harvard.edu , version 5.3.0) as needed. Longitudinal processing included creating an unbiased within-subject base template from both time points and then using common information from the template to perform atlas registration, spherical surface map creation, and subcortical parcellation to extract reliable volume and thickness estimates. 31 ,  32</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27669146">PMID:27669146</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Source reconstruction was performed using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) for the volumetric segmentation of the MRI data, surface reconstruction, flattening, cortical parcellation, and neuroanatomical labeling with the Destrieux atlas ( Dale et al., 1999 ;  Destrieux et al., 2010 ;  Fischl et al., 2002 , see  Figure 2 — source data 1  for a list of parcels).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27658309">PMID:27658309</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 Atlas  ‘ aparc.a2005s.annot ’  was chosen from FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ), containing cortical anatomical label information of the whole brain with a total of 162 segmented regions. The group ICA maps and the segmented cortical regions from T1 Atlas  ‘ aparc.a2005s.annot ’  were registered and normalized into the Montreal Neurological Institute (MNI) structural template ( http://imaging.mrc-cbu.cam.ac.uk/imaging/Templates ) and overlapped to register the anatomical location of the activated regions during the resting-state. Overall, 120 of 162 cortical ROIs were selected from the union of ICA activation maps of the young and elderly groups. Removing limbic ROIs yielded 112 ROIs of the four cortical lobes. The TCs of the 112 ROIs for the young and elderly groups were used as the FDA input. (The ROIs in the T1 Atlas  ‘ aparc.a2005s.annot ’ were listed in  S2 File .)</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27658292">PMID:27658292</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and thickness estimation (CT) was performed using the Freesurfer software package (Mac version 5.1.0 available at ( www.surfer.nmr.mgh.harvard.edu ), following standard procedures and rigorous visual quality control. Briefly the processing involves removal of non-brain tissue, transformation into Talairach space, intensity normalization, tessellation of the grey matter white matter boundary, automated topology correction, and surface deformation is then performed to indicate the grey/white and grey/cerebrospinal fluid borders based on detection of greatest shift in intensity [ 36 ]. The method uses both intensity and continuity information from the entire three dimensional MR volume in deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the grey/white boundary to the grey/CSF boundary at each vertex on the tessellated surface. The reconstruction output is then visually inspected and any parcellation inaccuracies were manually corrected as per Freesurfer manual. This process involved all subjects to ensure that the automated parcellations follow the actual surface of the cortex. In all subjects some degree of manual correction was required ranging from as minimal as 2 consecutive brain slices (1 slice = 1mm) to a maximum of 16 non-consecutive slices. This was performed and checked in coronal, sagittal and transverse views to ensure accuracy by one trained researcher while blinded to the clinical details (HA) supervised by an experienced neuroradiologist (RAD). The maps produced are not restricted to the voxel resolution of the original data thus are capable of detecting changes at sub-millimetre levels.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27655165">PMID:27655165</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction was performed with the Freesurfer image analysis suite (Version 5.3.0,  http://surfer.nmr.mgh.harvard.edu ). The technical details of these procedures are described in previous publications [ 25 – 28 ]. Briefly, this processing includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure [ 28 ], automated Talairach transformation, intensity normalization [ 29 ], tessellation of the gray matter/ white matter boundary, automated topology correction [ 30 ,  31 ], and surface deformation following intensity gradients for optimal placement of the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class [ 25 ]. Once the cortical models are complete, a number of deformable procedures can be performed for further data processing and analysis including surface inflation [ 26 ], registration to a spherical atlas which utilizes individual cortical folding patterns to match cortical geometry across subjects [ 27 ], parcellation of the cerebral cortex into units based on gyral and sulcal structure, and creation of a variety of surface based data including maps of curvature and sulcal depth. This method uses both intensity and continuity information from the entire three dimensional MR volume in segmentation and deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/cerebrospinal fluid boundary at each vertex on the tessellated surface [ 32 ]. The maps are created using spatial intensity gradients across tissue classes and are therefore not simply reliant on absolute signal intensity. In addition, the maps produced are not restricted to the voxel resolution of the original data and thus are capable of detecting submillimeter differences between groups. This means that if the pial boundary is between a gray matter and a cerebrospinal fluid voxel, some gray matter change might cause the boundary to move into the voxel [ 25 ,  26 ,  32 ]. Because of smoothness constraints implemented in Freesurfer, the surface placement and the thickness estimation might reach subvoxel accuracy [ 25 ,  26 ,  32 ]. This automated approach and the subvoxel accuracy of cortical thickness and cortical area measurements were validated against manual measurements on high-resolution MRI images as well as against neuropathological postmortem analysis [ 33 ,  34 ]: Cortical thickness measurements obtained from these two methods (MRI images vs. neuropathology) agreed within 0.2mm, with a mean difference of 0.077mm [ 34 ]. Before statistical analysis, cortical thickness and area maps were smoothed with a 10mm Full Width at Half Maximum (FWHM) Gaussian kernel.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27598966">PMID:27598966</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Based on the high-resolution T1-weighted structural image, the cortical and subcortical structures as well as the brain stem were parcellated using Freesurfer (V5.1.,  http://surfer.nmr.mgh.harvard.edu/ ). Processing included automatic segmentation into gray and white matter tissue compartments followed by parcellation of the gray matter mask into distinct brain regions based on a normalized template. The resulting parcellation consisted of a total of 83 distinct brain regions of which 68 were cortical (34 per hemisphere), 14 subcortical (7 per hemisphere: thalamus, caudate, putamen, pallidum, hippocampus, amygdala, nucleus accumbens) and 1 represented the brainstem 29 ,  30 ,  31  (see  Supplementary Figure 1  for illustration of nodes). This parcellation scheme comprises several nodes that are thought to play an important role in the pathophysiology of OCD. Of special interest are several subcortical (for example, caudate, putamen, nucleus accumbens, amygdala, thalamus), as well as cortical regions (rostral middle frontal, medial orbital frontal, insula). In addition, this scheme is in line with studies examining other psychiatric diseases 8 ,  32  and may thus facilitate comparison of results across diagnostic categories.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27479794">PMID:27479794</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 ‐ weighted images were segmented using FreeSurfer (FS) version 5.0.7 ( http://surfer.nmr.mgh.harvard.edu /). We used the automated reconstruction protocol  “ recon ‐ all ”  which has been described previously (Fischl et al.,  2004 ). Briefly, this pipeline includes motion correction, removal of nonbrain tissues, affine registration into Talairach space and segmentation of the subcortical white ‐  and grey ‐ matter structures. This automated segmentation procedure assigns anatomical labels to each voxel using probabilistic information derived from a manually labeled training set, which constrains the location of structures in relation to each other. The accuracy of this segmentation is illustrated for each patient in Supporting Information, Figure 5. Regional volumes were extracted for subcortical regions of interest (ROIs).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27473521">PMID:27473521</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the Freesurfer image analysis suite (v5.3;  www.surfer.nmr.mgh.harvard.edu ). Briefly, this processing includes motion correction and averaging (Reuter, Rosas,  &  Fischl,  2010 ) of volumetric T1-weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure (S é gonne et al.,  2004 ), automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (Fischl et al.,  2002 ), intensity normalization (Sled, Zijdenbos,  &  Evans,  1998 ), tessellation of the gray matter and white matter boundary, automated topology correction (Fischl, Liu,  &  Dale  2001 ), and surface deformation following intensity gradients to locate the gray/white and gray/cerebrospinal fluid borders (Fischl  &  Dale,  2000 ). After the cortical models, the surface was inflated (Fischl, Sereno, Tootell,  &  Dale,  1999 ), registered to a spherical atlas which utilized individual cortical folding patterns to match cortical geometry across subjects (Fischl, Sereno,  &  Dale,  1999 ), parcellation of the cerebral cortex into units based on gyral and sulcal structure (Desikan et al.,  2006 ), and creation of a variety of surface-based data including maps of curvature and sulcal depth.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27453157">PMID:27453157</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Details regarding image acquisition at each site are described in Supplemental material. Two samples (CPB and NCD) were scanned using 1.5-T MRI machines, and two samples (Pittsburgh and Braintime) were scanned using 3-T MRI machines. MRI processing was performed with the FreeSurfer 5.3 image analysis suite, which is documented and freely available online ( http://surfer.nmr.mgh.harvard.edu /). The technical details of these procedures are described in detail in seminal publications ( Dale et al., 1999 ,  Fischl et al., 1999 ,  Fischl et al., 2002 ). This processing stream includes motion correction ( Reuter et al., 2010 ), removal of non-brain tissue using a hybrid watershed/surface deformation procedure ( S é gonne et al., 2004 ), automated Talairach transformation, non-parametric non-uniform intensity normalization ( Sled et al., 1998 ), tessellation of the grey/white matter boundary, automated topology correction ( Fischl et al., 2001 ,  S é gonne et al., 2007 ), and surface deformation following intensity gradients to optimally place the grey/white and grey/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class ( Dale et al., 1999 ,  Dale and Sereno, 1993 ,  Fischl and Dale, 2000 ). Each cortical model was registered to a spherical atlas using individual cortical folding patterns to match cortical geometry across participants ( Dale et al., 1999 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27405617">PMID:27405617</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed using FreeSurfer image analysis suite (version 5.3; Martinos Center for Biomedical Imaging, Harvard-MIT, Boston, MA;  http://surfer.nmr.mgh.harvard.edu/ ). FreeSurfer includes motion correction and averaging, Talairach transformation, removal of non-brain tissue, segmentation of subcortical structures and cortical regions, intensity normalization and cortical reconstruction. For quality assessment, a visual inspection of all subcortical structures and cortical regions was performed, using a protocol developed by the ENIGMA consortium ( http://enigma.ini.usc.edu/protocols/imaging-protocols/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27399108">PMID:27399108</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Diffusion-weighted image data were pre-processed with FMRIB's Diffusion Toolbox ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslOverview/ ). Preprocessing consisted of Eddy current correction, motion correction, and the removal of non – brain tissue using the robust Brain Extraction Tool. [ 23 ]  DTI were created using the weighed least squares fitting method. We derived images of fractional anisotrophy and mean diffusivity from the DTI. To calculate the specific fractional anisotrophy and mean diffusivity values in the subcortical regions, we first obtained the subcortical volume measurements from the 3D T1 MRI images using the freely available software FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ). FreeSurfer is a popular and publicly available software package for studying cortical and subcortical anatomy. Briefly, images underwent preprocessing including intensity normalization and skull stripping, which was followed by labeling of cortical and subcortical regions. FreeSurfer's main cortical reconstruction pipeline begins with registration of the structural volume with the Talairach atlas. After bias field estimations and the removal of bias, the skull is stripped and subcortical white and gray matter structures are segmented. [ 24 ]  Then, the fractional anisotrophy and mean diffusivity maps were resampled by means of a rigid-body transformation from the diffusion space to the freesurfer's structural space. After that, the fractional anisotrophy and mean diffusivity values from the Freesurfer's subcortical regions were computed.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27177090">PMID:27177090</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the high quality data of structural volumetric MRI, which were available in the ADNI data archive provided by the University of California, San Francisco (UCSF) medical center, to conduct association test of SORL1 genotypes with brain structure. The raw Digital Imaging and Communications in Medicine images were downloaded from the public ADNI site ( http://www.loni.ucla.edu/ADNI/Data/index.shtml ) and parameter values were available at  http://www.loni.ucla.edu/ADNI/Research/Cores/ . All MRIs were processed using the FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/ ) based on the 2010 Desikan-Killany atlas [ 53 ,  54 ]. The procedure included averaging of multiple volumetric T1 weighted images, intensity normalization, removal of non-brain tissue, segmentation of the subcortical white matter and deep gray matter volumetric structures [ 55 ], automated Talairach transformation, tessellation of the gray matter white matter boundary, automated topology correction, and surface deformation following intensity gradients to optimally locate the gray-white and gray-cerebrospinal fluid boundary where the greatest shift in intensity defines the transition to the other tissue class [ 56 ]. In addition, cortical thickness measurements were then obtained by calculation of the distance between the white and grey matter surfaces at each point across the entire cortical surface (per hemisphere) [ 57 ]. We used the ROIs strategies in MRI analysis to assess the relationship between SORL1 and AD and focused on the discriminant brain regions that had been found to be strongly associated with AD by previous studies: hippocampus, parahippocampal gyrus, middle temporal gyrus, posterior cingulate and the entorhinal cortex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27163203">PMID:27163203</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The MRI scans were analysed using the FreeSurfer software package v5.1\n( http://surfer.nmr.mgh.harvard.edu/ ). Regional estimates of\nsubcortical volumes were assessed with a specialized surface-based \nreconstruction and automatic labelling tool, which is described in detail\nelsewhere. 29  In summary,\nFreeSurfer processing includes motion correction, skull-stripping, Talairach \ntransformation, segmentation of subcortical white matter and deep grey\nmatter volumetric structures, intensity normalization, tessellation of the \ngrey matter/white matter boundary, automated topology correction and\nsurface deformation.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27144904">PMID:27144904</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The structural scans of each subject were segmented with in FreeSurfer toolkit version 5.1 ( http://surfer.nmr.mgh.harvard.edu ), which implements a semi ‐ automated segmentation workflow. FreeSurfer enables the segmentation of both GM and WM as well as subcortical segmentation. The stages of processing implemented in this pipeline are fully described elsewhere [Desikan et al.,  2006 ; Destrieux et al.,  2010 ; Fischl et al.,  2002 ,  2004 ]. Validation against manual segmentations has also been described [Fischl et al.,  2002 ] and its results are considered robust across sessions, scanner platforms, updates, and field strengths [Jovicich et al.,  2009 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27065363">PMID:27065363</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal subfields, including CA2 – CA3 – DG, CA1, and the subiculum, were defined in the TSE images using the automatic segmentation of hippocampal subfields (ASHS) machine learning toolbox 18  and a database of manual medial temporal lobe (MTL) segmentations from a separate set of 24 participants 19 . Manual segmentations were based on anatomical landmarks used in prior studies 6 , 19 – 21 . Consistent with these studies, CA2, CA3 and DG were combined into a single ROI because these subfields cannot be distinguished at our functional resolution (1.5 mm isotropic). The inclusion of DG could in principle be problematic for observing pattern completion because it is often linked to the opposite effect of pattern separation 22 . However, DG may also support pattern completion, as suggested by recent computational models 23  and neurophysiological findings 24 . We had reason to believe that CA1 might show pattern completion, as it receives input from CA3 via Schaffer collaterals, and is believed to translate completed representations such that they can be reactivated in cortex 11 . Indeed, functional connectivity between CA3 and CA1 is enhanced during retrieval 25  and evidence of pattern completion has been observed in CA1. Not much is known about the role of the subiculum in pattern completion and this region is left out of most hippocampal models 9  and theories 11 . We included it as a control region where pattern completion might not be observed, as well as for the sake of completeness and to mirror prior high-resolution studies of human hippocampus 6 , 19 , 21 , 25 , 26 . Finally, in visual cortex, we focused on V1 and V2 because they can be precisely segmented anatomically within individual participants 27 , 28 . These ROIs were automatically defined in each participant ’ s T1-weighted anatomical scan with FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) 29 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27009679">PMID:27009679</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>For anatomical data, the images were processed using FREESURFER software ( http://surfer.nmr.mgh.harvard.edu , ver. 5.1.0) to generate anatomical ROIs, including the hippocampus. After automatic segmentation using FREESURFER (Fischl et al.,  2002 ; Desikan et al.,  2006 ), the hippocampal volume was manually edited according to previously established protocols (Pruessner et al.,  2000 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26954295">PMID:26954295</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with Freesurfer software (version 5.3;  http://surfer.nmr.mgh.harvard.edu ) using a standard processing pipeline. The technical details of these procedures have been previously described in detail. 22 ,  23  In short, reconstruction procedures comprised intensity normalization to Montreal Neurological Institute (MNI) ‐ space, skull stripping, filtering, segmentation, and surface deformation. The quality of the skull stripping and accuracy of the gray and white matter outer boundaries were reviewed by a trained researcher (S.L.), who was blinded to the grouping of the patients. To extract the volumetric data of the predefined ROIs, we used specialized Freesurfer tools for automated parcellation of gray and white matter. 24  As in the resting ‐ state fMRI analysis, ROIs comprised bilateral SM 1  (BA 1 ‐ 4), SMA (bilateral BA 6), rIFG (BA 44 and 45), and bilateral putamen. The extracted volume measures were transformed into  z  scores for further analyses.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26645627">PMID:26645627</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain images were automatically segmented using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ) and the resulting outputs quality controlled following a publically available protocol from ENIGMA ( http://enigma.ini.usc.edu/ ). Whenever a region-of-interest (ROI) was detected as inadequately segmented, its metric was declared missing and excluded from the analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26503586">PMID:26503586</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To take into account our a priori hypotheses about the relation between neural activity and between ‐ subject variability, anatomical regions of interest (ROI) were defined for regions consistently implicated in action observation (the AON) [Grafton,  2009 ] and found to be activated during the observation of dynamic whole ‐ body movements in a recent ALE meta analysis [Grosbras et al.,  2012 ; see also Caspers et al.,  2010 ]. These included the lateral occipito ‐ temporal cortex (corresponding to visual area MT/V5), the pSTS, the anterior inferior parietal lobule (IPL; corresponding to BA40), the inferior frontal gyrus (IFG; corresponding to BA44/BA45), and the lateral and medial PMC (corresponding to BA6). Based on the results of our previous study, we additionally checked whether activity in the caudate is related to the sensitivity in action prediction [cf., Diersch et al.,  2013 ]. The single ROIs were created based on each participant's T1 structural scan using a semiautomated anatomic reconstruction and labeling procedure as implemented in FreeSurfer v5.3.0, which has been shown to be comparable in terms of accuracy to manual labeling techniques ( http://surfer.nmr.mgh.harvard.edu ) [Dale et al.,  1999 ; Fischl et al.,  1999 ]. In each hemisphere, labels corresponding to the IFG (BA44/45), PMC (BA6), anterior IPL (supramarginal gyrus as defined in the Desikan ‐ Killiany Atlas), pSTS (banks of the superior temporal sulcus as defined in the Desikan ‐ Killiany Atlas), and MT produced by the automatic cortical parcellation as well as the caudate from the subcortical segmentation were extracted [Desikan et al.,  2006 ; Fischl et al.,  2002 ]. This procedure has the advantage of accounting for age ‐ related structural changes in the brain since it does not involve the mapping of different brains to a common atlas that refers to a young brain. Each ROI was thresholded at 0.5 and transformed to MNI space for the usage in SPM8. Finally, cluster ROIs representing the whole sample were built based on each of the individual ROIs, with the constraint that adjacent cortical ROIs such as the PMC and IFG, did not contain any overlapping voxels.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27685352">PMID:27685352</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27707978">PMID:27707978</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27688947">PMID:27688947</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27688941">PMID:27688941</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27843694">PMID:27843694</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27418273">PMID:27418273</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28058272">PMID:28058272</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28132187">PMID:28132187</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28210220">PMID:28210220</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28203528">PMID:28203528</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional MRI data were analyzed using Freesurfer FS-FAST software (version 5.3;  http://surfer.nmr.mgh.harvard.edu ). Functional imaging data were motion corrected to the middle time point of each BOLD run, and inspected for gross motion. Slices were excluded if motion was  >   1   mm. In addition, BOLD data were intensity normalized and spatially smoothed (full-width half-maximum   =   4   mm) using a 3D Gaussian filter. The first three volumes in each run were discarded to allow for T2* equilibration. Following preprocessing, functional images for each participant were registered to that participant's 3D MPRAGE image.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28191353">PMID:28191353</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution T1WI analysis was performed using the FreeSurfer package ( http://surfer.nmr.mgh.harvard.edu/ ). Cortical thickness was measured as the difference between the position of equivalent vertices in the pial and GM/WM surfaces, by utilizing both intensity and continuity information from the entire 3-dimensional MR volume in the segmentation and deformation procedures. The reliability of the method was previously validated against histological analysis on postmortem brains and manual measurements, and the test-retest reliability is high [ 16 – 19 ]. Briefly, the main steps included motion correction using FSL FLIRT and automated registration to Talairach space, segmentation of the subcortical GM and WM structures, intensity normalization and removal of nonbrain voxels, and tessellation of GM and WM boundaries and automated topology correction and surface deformation to optimally place the GM/WM and GM/CSF borders defined at the location with the greatest shift in signal intensity [ 16 ,  20 – 24 ]. Following registration of all subjects' cortical reconstructions to a common average surface, the surface maps were interpolated and analyzed. This procedure is capable of detecting submillimeter differences between groups [ 16 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28180083">PMID:28180083</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer ( http://surfer.nmr.mgh.harvard.edu /) was used to analyze cortical thickness. This method extracts the pial and white matter surfaces on T1-weighted structural scans and computes the distance between them in order to measure the thickness of the cortical ribbon in between ( Fischl and Dale, 2000 ). Initial outputs from the standard Freesurfer cortical reconstruction processing stream were inspected and the ventricles manually segmented to correct for reconstruction errors due to enlarged ventricles, before completing reconstruction. To investigate the visual areas most affected by gray matter loss, three particular regions of interest were chosen for further analysis in the visual cortex: V1, V2, and hMT+. Labels for each of these areas were used from the automatic brain segmentation in Freesurfer, using the Brodmann atlas. These cytoarchitectonically defined regions are identified on the basis of cortical folding patterns ( Fischl et al., 2008 ). Average cortical thickness measures were extracted for each of these areas for each subject.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28180074">PMID:28180074</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Each individual's T1-weighted MRI was processed using FreeSurfer's main recon-all processing stream with a recently described extension for the segmentation of WMSAs ( Lindemer et al., 2015 ). Cortical reconstruction and volumetric segmentation was performed using FreeSurfer ( surfer.nmr.mgh.harvard.edu /, version 5.1). The technical details of these procedures are described in prior publications ( Dale and Sereno, 1993 ,  Dale et al., 1999 ,  Fischl and Dale, 2000 ,  Fischl et al., 1999 ,  Fischl et al., 2001 ,  Fischl et al., 2002 ,  Fischl et al., 2004a ,  Fischl et al., 2004b ,  Han et al., 2006 ,  Jovicich et al., 2006 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28178980">PMID:28178980</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Written informed consent was obtained from each participant. The study was approved by the ethics committee at the Institute of Physiology and Pathology of Hearing and performed according to the Declaration of Helsinki. The patients underwent thorough clinical evaluation. Functional impairment was assessed according to spinocerebellar degeneration functional score (SDFS) [ 20 ]. In brain and cervical spine MRI (3T Siemens Magnetom Trio, 12-channel Head Matrix Coil) T1-weighted, T2-weighted and diffusion-weighted images were acquired. In addition, high-resolution 3D structural T1-weighted volumes were acquired using an MPRAGE sequence with 208 sagittal slices and an isotropic resolution 0.9   ×   0.9   ×   0.9   mm. Sequence parameters were: TR   =   1900   ms, TE   =   2.21   ms, TI   =   900   ms, FA   =   9, FOV   =   26   ×   28.8   cm, matrix   =   320   ×   290, Pixel bandwidth   =   200   Hz/pix, iPAT   =   2, TA   =   5   min. Brain structures were segmented by Freesurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ). The diameter of the vestibulocochlear nerve was evaluated directly and compared with the neighboring facial nerve used as an internal reference. Assessment of the cochlear and vestibular components was based on their visual comparison and with reference to the facial nerve. For comparisons of the cerebrum and cerebellum volumes and their white and gray matters, age- and sex-matched controls [ 21 – 23 ] from the Internet Brain Volume Database funded by The Human Brain Project ( http://ibvd.virtualbrain.org/ ) were used. A difference above 2.3 of the standard deviation was considered as statistically significant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28167897">PMID:28167897</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual structural MRI T1 images were collected for each participant using the 1.5-T Philips Intera scanner. Reconstruction of the cortical surfaces was performed using the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu ). The resulting cortical surface meshes were imported into Brainstorm (Tadel et al.,  2011 ) and down-sampled to 8000 vertices for further processing. Forward modeling was performed using the overlapping spheres method (Huang et al.,  1999 ) as implemented in the Brainstorm software. Due to the unavailability of individual structural MRIs, the default MNI anatomy with 1 mm resolution was used for two participants.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28149941">PMID:28149941</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The volumetric analysis was performed using the FreeSurfer software package (version 4.5.0) ( http://surfer.nmr.mgh.harvard.edu/ ). The brain imaging preprocessing steps include motion correction, affine transformation to Talairach image space, nonuniform intensity normalization for intensity inhomogeneity correction, and removal of nonbrain tissues  [11] ,  [12] ,  [13] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28143492">PMID:28143492</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ) was used for the volumetric parcellation and segmentation. This is an automated method of labeling human structures to extract GM and WM volumes for each participant ’ s entire brain [ 49 ,  50 ], and parcellating of the cortex of each participant as well as extracting segmentations of subcortical structures [ 51 ,  52 ]. Parcellations of the cortex are automatically corrected for total brain volume differences [ 51 ,  52 ]. In order to avoid segmentation errors, all images were inspected manually and structures with obvious segmentation errors were rejected. No manual adjustments were made to avoid introducing bias and increasing variances into the data set of MRI images.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28129351">PMID:28129351</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Also, for each hemisphere, we determined the average sulcal width for each of the five sulci: the superior frontal, intra-parietal, superior temporal, central, and Sylvian fissure. Sulcal width was defined as the average 3D distance between opposing gyral banks along the normal projections to the medial sulcal mesh [ 41 , 42 ]. The five sulci investigated in the present study were chosen because they are present in all individuals, large and relatively easy to identify after facilitating error detection and correction, and they are located on different cerebral lobes. We also calculated cortical thickness, cortical volume and subcortical volume. This was done using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) based on the atlas including 34 cortical regions of interest in each hemisphere [ 43 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28119599">PMID:28119599</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28116240">PMID:28116240</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed using FreeSurfer 6.0 image analysis suit ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details have been described previously ( Fischl and Dale, 2000 ). In summary, image processing procedures include the following steps: motion correction and average of multiple volumetric T1-weighted images, removal of non-brain tissue, automated Talairach transformation, intensity normalization, segmentation of the subcortical white matter and deep grey matter volumetric structures, tessellation of the grey matter and white matter boundary, automated topology correction, and surface deformation to optimally place the grey/white and grey/cerebrospinal fluid boundaries ( Reuter et al., 2012 ). All surface models were inspected for accuracy and manual corrections were performed in the event of tissue misclassification and/or white matter errors while blinded to the diagnostic information.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28116232">PMID:28116232</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The following steps were performed to bring the Craddock200-based parcellations in the MNI space into the subject-specific DTI space: 1- The T1-weighted volume was intensity-normalized using the 3dUnifize command in AFNI, and then skull-stripped in FreeSurfer ( http://surfer.nmr.mgh.harvard.edu /). The skull-stripped T1-weighted volume was then co-registered to the MNI_caez_N27 template in AFNI using the auto_warp.py nonlinear registration function.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28107346">PMID:28107346</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging data were coded and catalogued before being transferred to the Psychotic Disorders Program of the University of Massachusetts Medical School (UMMS) for blinded analysis. AFNI [ 24 ] was used for image preprocessing ( http://www.afni.nimh.gov/afni ), performing slice timing correction for interleaved acquisition (using Fourier interpolation), motion correction (by aligning each volume to a  “ base ”  image [middle volume] using Fourier interpolation) and de-spiking (detection and reduction of extreme time series outliers using a hyperbolic tangent function). All other data processing was carried out using FMRIB Software Library (FSL) ( http://www.fmrib.ox.ac.uk ), including spatial smoothing (FWHM = 6 mm), mean-based intensity normalization of all volumes, temporal bandpass filtering (highpass temporal filtering: Gaussian-weighted least-squares straight line fitting, with sigma = 100.0 s; Gaussian lowpass temporal filtering HWHM 2.8 s), and pre-whitening. Each individual ’ s time series were spatially normalized by registration to the MNI152 (Montreal Neurological Institute) template with 2mm 3  resolution, using a 12 degree of freedom affine transformation. Nine nuisance covariates (time series for global signal intensity, white matter, cerebrospinal fluid, and six motion parameters) were regressed out of the data to minimize the contributions of artifactual physiological signals (e.g., cardiac and respiratory cycles) using the general linear model implemented in FSL program FEAT. The FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) was used to segment T1-weighted SPGR images into cortical and subcortical gray and white matter regions, as well as total intracranial volume, for each subject[ 24 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28097205">PMID:28097205</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess AD ‐ related differences in tau deposition and glucose hypometabolism, we computed voxel ‐ wise mean ( μ ) and standard deviation ( σ ) of SUVR values for healthy controls. The healthy control groups were provided by AVID Radiopharmaceuticals ([ 18 F]AV ‐ 1451;  N  = 15, Age Mean  = 64.4, MMSE=29.5), from the Alzheimer's Disease Imaging database ( http://adni.loni.usc.edu ; [ 18 F]FDG PET  N  = 18, Age Mean  = 65.8, MMSE = 29.1*; *MMSE scores for three healthy controls were not available from the ADNI database), and 15 healthy controls (Age Mean  = 64.0, MMSE = 29.2) who underwent a [ 11 C]PiB PET imaging protocol. 8  We then computed  z ‐ score deviations for every voxel ( x ) using the formula  z = x − μ σ . The deviation images were overlaid on the lateral surface of the left and the right hemisphere using to Freesurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28056889">PMID:28056889</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A total of 77 patients were identified from the Alzheimer ’ s Disease Neuroimaging Initiative (ADNI) database (adni.loni.usc.edu), of which 52 had been catalogued as having AD while the remainder were normal healthy controls. The ADNI was launched in 2003 as a public-private partnership, led by Principal Investigator Michael W. Weiner, MD. The primary goal of ADNI has been to test whether serial magnetic resonance imaging (MRI), positron emission tomography (PET), other biological markers, and clinical and neuropsychological assessment can be combined to measure the progression of mild cognitive impairment (MCI) and early Alzheimer ’ s disease (AD). For up-to-date information, see www.adni-info.org. Patients were included in  S \n 1  based on having both (a) a structural 1.5 T T1w MRI scan acquired per the standardized ADNI protocol, and (b) plasma proteomics data, for which detailed collection and transportation protocols are described on the ADNI website ( http://www.adni-info.org/Scientists/ADNIScientistsHome.aspx ). The brain regions known to be most affected by AD had been segmented and quantified via the widely used FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu/ ), that was run on each T1w MRI scan, yielding a total of 327 features (that were available for download). Similarly, plasma proteomics had been extracted through a multiplex immunoassay panel of blood samples to yield a protein expression vector (that was available for download). These features are summarized in the  Appendix  (Table 5), and described in more detail on the ADNI webpage ( http://adni.loni.ucla.edu/ ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28052106">PMID:28052106</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal atrophy and other MRI signs of HS were assessed by visual analyses by two epilepsy imaging experts (CLY and FC) and the images were classified as having normal findings or signs of HS. Classic signs of HS were as follows: reduction of volume and/or abnormal shape observed on T1-weighted images, with or without increased signal observed in T2-weighted and FLAIR images [ 25 ]. Special attention was given to rule out subtle signs of other lesions, such as focal cortical dysplasia, tumours or gliotic scars. Patients with dual pathology or other MRI abnormalities were not included. In addition, hippocampal volumetry was performed in all patients and compared to a group of 79 healthy controls (with similar age and sex distribution) with FreeSurfer software v.5.1.0 ( http://surfer.nmr.mgh.harvard.edu ), as previously described [ 24 ]. Therefore, the presence of MRI findings of HS or normal MRI was defined by results of visual analyses and automatic hippocampal volumetry.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28045900">PMID:28045900</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Scans were processed and analyzed using Freesurfer stable release v5.3.0,  http://surfer.nmr.mgh.harvard.edu  [ 55 – 61 ]. To measure CT, the cerebral cortex was parcellated into units based on gyral and sulcal structure [ 62 ,  63 ]. Furthermore, a variety of surface-based data was created including maps of curvature and sulcal depth. This method used both intensity and continuity information from the entire three dimensional MR volume in segmentation and deformation procedures to produce representations of CT, calculated as the closest distance from the grey/white matter boundary to the grey matter/CSF boundary at each vertex on the tessellated surface [ 56 ]. The maps were created using spatial intensity gradients across tissue classes and were not restricted to the voxel resolution of the original data, thus were capable of detecting sub-millimeter differences between groups. CT measurement procedures have been validated against histological analysis [ 64 ] and manual measurements [ 18 ,  65 ]. Per hemisphere there are 34 regions (68 in total) of interest (adapted from the Desikant atlas) [ 62 ]; for each of these regions, CT measurements were derived by FreeSurfer and exported to Stata and R version 3.2.1. Thus, every individual had 68 CT measurements over the predefined ROIs. These procedures are described in detail in a previous paper [ 29 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28032004">PMID:28032004</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain volume and cortical thickness were calculated using the FreeSurfer software environment (version 4.4), which is free to use and available to download online ( http://surfer.nmr.mgh.harvard.edu/ ). The process used to analyze each subject's brain images is available on the FreeSurfer website. Scans were processed by a blinded technician (JBK). This included motion correction, extraction of brain tissue, transformation to Talairach space, segmentation and parcellation into regions of interest as specified by Deiskan et   al. ( 2006 ). After processing, scans were inspected by an experienced technician to ensure segmentation and parcellation were performed correctly. Scans were also examined for the presence of artifacts; no artifacts were found. Uncorrected OFC cortical volumes (mm 3 ) were divided by the total segmented brain volume (mm 3 ) and then multiplied by 1000, which resulted in the corrected cortical volumes. Four regions of interest were investigated: left LOFC, right LOFC, left MOFC, and right MOFC. Each region of interest was studied using two morphometric measures: cortical volume, corrected for total brain volume, and cortical thickness. Means and standard deviations of these measures can be seen in Table   3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28031997">PMID:28031997</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A standard FreeSurfer pipeline ( http://surfer.nmr.mgh.harvard.edu/ , v5.1.0) was applied to all the T1 ‐ weighted MRI images to intensity normalize, skull ‐ strip, and segment brain tissue (Fischl  &  Dale,  2000 ). The choice of imaging parameters closely matched the FreeSurfer recommended parameters. Han et   al. ( 2006 ) describe that segmentation inaccuracies are minimal with FreeSurfer ‐ recommended imaging parameters, and do not affect study outcome; thus, reducing the need to verify segmentation accuracy for each subject. In order to ensure accuracy of segmentation for this analysis, we randomly selected and checked 15% of processed images. Each subject's data were registered to the standard FreeSurfer brain (fsaverage). The Destrieux atlas was used to identify cortical brain regions and evaluate cortical thickness per region (Destrieux, Fischl, Dale,  &  Halgren,  2010 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28003960">PMID:28003960</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1-weighted images were processed using the well-validated open access software package FreeSurfer v5.1.0 ( Fischl, 2012 ) (available at  http://surfer.nmr.mgh.harvard.edu /). This software package involves an automated procedure that allows quantitative assessment of brain anatomy including volumetry of subcortical structures and cortical morphometry, with accuracy comparable to manual methods ( Fischl et al., 2002 ). The procedures have been described previously ( Dale et al., 1999 ,  Fischl and Dale, 2000 ,  Fischl et al., 2004a ,  Fischl et al., 2004b ,  Fischl et al., 2002 ,  Fischl et al., 1999 ). In short, brain segmentation consists of removing non-brain tissue using a deformable template model and neuroanatomical labeling based on both voxel intensity values and a probabilistic atlas ( Fischl et al., 2002 ). Next the cortical surface is reconstructed where the segmentation of white matter is used to derive a surface representing the grey-white matter boundary ( Dale et al., 1999 ). Finally, by incorporating both geometric information derived from the cortical model and standard neuroanatomical conventions ( Desikan et al., 2006 ,  Destrieux et al., 2010 ), the procedure automatically assigns a neuroanatomical label to each location on the cortical surface. All scans were processed on the same hardware using the same software to avoid potential confounds ( Gronenschild et al., 2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28003958">PMID:28003958</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Nodes in the network were defined for each patient according to parcellation of whole-brain gray matter on the structural images. The processing pipeline was implemented using MATLAB scripts (version 7.13, MathWorks, Inc.) in which adapter functions were embedded to execute FreeSurfer reconstruction (version 5.3.0;  http://surfer.nmr.mgh.harvard.edu ) and several FMRIB Software Library (FSL) suite tools ( Smith et al., 2004 ). First, FreeSurfer reconstruction of cerebral cortical surfaces was performed on the T1 structural image. This processing stream includes motion correction, skull stripping, intensity normalization, segmentation of white matter and gray matter structures, parcellation of the gray matter and white matter boundary, and surface deformation following intensity gradients which optimally place the gray matter/white matter and gray matter/cerebrospinal fluid borders ( Fischl et al., 2001 ,  Fischl et al., 2004 ). The pial and gray white surfaces were visually inspected using the Freeview software for accurate placement.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27999541">PMID:27999541</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric analysis was performed on data from a 3D high-resolution (0.9 mm isotropic) T1-weighted scan using MPRAGE acquisition. Cortical reconstruction was performed with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl et al.,  1999a , b ,  2001 ,  2002 ,  2004 ; Fischl and Dale,  2000 ; Fischl,  2004 ; S é gonne et al.,  2004 ; Han et al.,  2006 ; Jovicich et al.,  2006 ; Reuter et al.,  2010 ,  2012 ). All cortical reconstructions were manually checked for accuracy, as recommended by the software developers. The volumetric analyses focused on gray matter volume in the temporal cortex provided by Freesurfer parcellation. Regions of interest included the superior temporal cortex, middle temporal cortex, inferior temporal cortex, banks of the superior temporal sulcus, fusiform cortex, transverse temporal cortex, entorhinal cortex, temporal pole, and parahippocampal cortex.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27999528">PMID:27999528</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated segmentation of subcortical structures (aseg) was performed with the FreeSurfer image analysis pipeline (Version 5.1), which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The processing includes removal of non-brain tissue, automated Talairach transformation and segmentation of the subcortical WM and deep GM volumetric structures. The last step is completed by automatically assigning one of 37 neuroanatomical labels to each voxel in the MRI volume based on probabilistic information from a manually labeled training set (Fischl et al.,  2002 ). This method has been shown to be comparable to much slower, labor-intensive manual labeling methods (Fischl et al.,  2002 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27995069">PMID:27995069</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation was performed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu /). The technical details of these procedures have been described and validated in prior publications ( Fischl et al., 1999 ,  Fischl et al., 2002 ,  Fischl et al., 2004 ). Briefly, the processing included skull stripping, Talairach transformation, optimization of the grey matter-white matter and grey matter-CSF boundaries, segmentation, and tessellation ( Dale et al., 1999 ). Images were visually inspected and scans with errors in segmentation of cortical or CSF surfaces and grey-white junctions were manually corrected. Atrophied brains were carefully inspected for errors. The tessellated surfaces were then inflated and registered to the Desikan-Killiany atlas which parcellates the cerebral cortex into 34 regions in each hemisphere based on gyral and sulcal structures ( Desikan et al., 2006 ). The FreeSurfer longitudinal stream, which creates an unbiased within-subject template for measuring changes in cortical thickness over time ( http://surfer.nmr.mgh.harvard.edu/fswiki/LongitudinalProcessing ) ( Reuter et al., 2012 ) was used. Measures of regional cortical thickness were compared between subgroups and longitudinally. The FreeSurfer QDEC tool was used to carry out whole-brain vertex-wise comparisons between the baseline scans of symptomatic C9   + subjects and healthy controls, and between the C9   + ALS/ALS-FTD patients and sALS patients, including gender as a covariate. Data were smoothed at 10   mm, and p   <   0.05 with FDR correction was used as criteria for significance.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27992533">PMID:27992533</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were acquired at a 3 Tesla MRI scanner (Achieva; Philips, Best, the Netherlands) equipped with an 8-channel head coil. A T1-weighted echo-planar imaging sequence (1 mm slice thickness, 208  ×  208 matrix, TE = 3.6 ms, TR = 7.8 ms, flip angle = 8 degrees) was used. Cortical reconstruction and volumetric segmentation was performed with FreeSurfer 5.3 image analysis suite (Massachusetts General Hospital, Harvard Medical School;  http://surfer.nmr.mgh.harvard.edu ). Briefly, this included motion correction and averaging of multiple volumetric T1 weighted images, removal of non-brain tissue using a hybrid surface deformation procedure[ 17 ], automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures[ 18 ], intensity normalization[ 19 ], tessellation of the gray/white matter boundary, automated topology correction[ 20 ], and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders[ 21 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27976739">PMID:27976739</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The fMRI signals in native space were preprocessed using custom software (mrVista software package for MATLAB, freely available at  http://vistalab.stanford.edu/software/ ). The mrVista package uses functions and algorithms from the SPM suite (freely available at  http://www.fil.ion.ucl.ac.uk/spm/ ). All the functional images underwent three-dimensional motion correction. For retinotopic scans, we applied slice-timing correction, averaged runs across stimuli groups and computed a coherence analysis. No spatial or temporal smoothing was applied. The rigid-body transformations were performed to align the functional images to the structural image for each subject. A grey-matter mask was used to extract fMRI data only from grey-matter voxels for further analyses. The boundaries between the retinotopic areas V1, V2, V3A and the sub-region that corresponded to the reference stimulus within V1/V2, were identified using the standard visual field mapping procedure 62 , and a motion localizer was used to functionally define hMT. Other regions of interest (ROIs), such as the fusiform gyrus and parts of the lateral prefrontal cortex (lateral PFC) and parietal areas, were anatomically defined through cortical reconstruction and volumetric segmentation using the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). More specifically, the prefrontal ROIs of the lateral PFC were the IFS, the MFS and MFG. Once the ROIs were identified, time courses of BOLD signal intensities were extracted from each voxel in each ROI and shifted by 6   s to account for the hemodynamic delay using the Matlab software. A linear trend was removed from the time course, and the time course was  z -score normalized for each voxel in each run to minimize the baseline differences across runs. The data samples for computing the MVPA were created by averaging the BOLD signal intensities of each voxel for three volumes, corresponding to the 6   s from stimulus onset to response onset.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27959330">PMID:27959330</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted three-dimensional MPRAGE data were analysed using FreeSurfer version 5.1.0 ( http://surfer.nmr.mgh.harvard.edu ) with technical details previously described. 31 ,  32 ,  33  Automated anatomic segmentation procedure was used to measure volume of T1 white matter hypointensities, 32 ,  34  whereas regional cortical thickness measures were obtained from the automated anatomic parcellation procedure 34  for each participant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27930678">PMID:27930678</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to corroborate the findings of the VBM 8 analysis and to obtain a more fine-grained understanding of the underlying structural differences driving the relationship between insular gray matter volume and Observer Justice Sensitivity, we additionally performed a surface-based structural analysis with the Freesurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications [ 13 ,  46 – 50 ]. Briefly, T1-weighted MRI images volumes were processed in a fully automated fashion using a cortical surface-based reconstruction that ultimately provides measurement of cortical thickness and surface area throughout the cortical mantle for each individual participant. In order to examine whether thickness or surface area, or a combination of both, drives the relationship between Observer Injustice Sensitivity and insular gray matter volume, we extracted the average thickness and surface area in this brain region. We then conducted linear regression analyses with the Observer Justice Sensitivity score as the independent variable and either the cortical thickness or surface area as the dependent variable, controlling for the same co-variates used in the VBM 8 analysis (age, total intracranial brain volume). Notably, if we perform an exploratory statistical analysis of the whole cortical mantle using cortical thickness or surface area as measurement of interests (controlling for all covariates), no other brain area showed an association with Observer Injustice Sensitivity that survived whole brain false discovery rate (FDR) correction. This finding further corroborates the results from the VBM 8 analysis.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27917152">PMID:27917152</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A recently developed in-house object-based supervised machine-learning algorithm was used for automated segmentation and quantification of WMHs based on FLAIR image intensity and masks of tissue types ( 36 ). All segmentations were manually inspected for accuracy and edited when necessary. For each subject, the number of non-interconnected hyperintense areas was further determined, as well as periventricular WMH volume ( ≤ 10   mm from the ventricles). The reconstruction and segmentation of WM was performed with the FreeSurfer image analysis suite version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ). NAWM was defined by subtracting WMH from the total WM. FDG-PET measurements in NAWM were extracted using FSL.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27917112">PMID:27917112</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27884107">PMID:27884107</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cerebral MRI will be performed at baseline, at 3   months, and 1   year postoperatively on the same day as the neuropsychological assessment. MRI analysis will be carried out by the Division of Neuroradiology at the University Hospital Basel. High-resolution anatomic and diffusion cerebral MRI will be performed using the hospital ’ s 3-Tesla MAGNETOM Prisma ™  MRI scanner (Siemens, Zurich, Switzerland). For MRI analysis, we will assume that the same regions as those described in mild cognitive impairment and Alzheimer ’ s disease [ 29 ] are relevant to POCD. We will perform a region of interest (ROI) analysis (hippocampus, lateral ventricle, total grey matter volume, regional cortical thickness). Cortical reconstruction and volumetric segmentation will be performed with the FreeSurfer software suite [ 30 ], which is freely available for download online ( http://surfer.nmr.mgh.harvard.edu ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27802229">PMID:27802229</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>A network is given by a set of nodes connected by edges that can be undirected or directed and weighted or unweighted. It can be represented by the adjacency matrix A, in which each column/row is associated with a node, and the element A ij   > 0, if there exists an edge between node i and node j. In our analysis, the weighted individual networks representing brain structural connectivity were constructed as follows: 1. The automated parcellation of the T1-weighted images was performed in FreeSurfer (Athinoula A. Martinos Center for Biomedical Imaging, Harvard-MIT, Boston [ http://surfer.nmr.mgh.harvard.edu ]). The Destrieux atlas was used giving 148 cortical structures that were then used as regions of interest for fiber tracking [ 33 ]. Labels and names of the regions of interest can be found in  Supplementary Table   1 . Subsequently, T1-weighted images were co-registered to the first b0 image by means of the between modality coregistration methodology using information theory, and finally re-sampled to the b0 images space. The T1-b0 transformation was also applied to atlas image using nearest neighbor interpolation. The Normalized Mutual Information cost function was employed to estimate a 12-parameter (degree of freedom) affine transformation matrix to transform voxels from MRI to b0 space. SPM5 tools were used to perform non-linear registration.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27801645">PMID:27801645</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Segmentation was performed in a semi-automated fashion using the FreeSurfer image analysis suite, which is documented and available online (version 5.1;  http://surfer.nmr.mgh.harvard.edu ) with details described previously (e.g.  Fischl et al., 2004 ;  Poppenk   and   Norman, 2014 ). Briefly, this processing includes removal of non-brain tissue using a hybrid watershed/surface deformation procedure ( S é gonne et al., 2004 ), automated Talairach transformation, intensity normalization ( Sled et al., 1998 ), tessellation of the grey matter / white matter boundary, automated topology correction ( Fischl et al., 2001 ;  Segonne et al., 2007 ), surface deformation following intensity gradients ( Fischl and Dale, 2000 ), parcellation of cortex into units based on gyral and sulcal structure ( Desikan et al., 2006 ;  Fischl et al., 2004 ), and creation of a variety of surface-based data, including maps of curvature and sulcal depth.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27756805">PMID:27756805</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were preprocessed using Freesurfer V.5.3 ( http://surfer.nmr.mgh.harvard.edu/ ). The cerebral cortex was parcellated into 68 regions 15  and segmented into 24 subcortical structures (deep grey matter, ventricles and cerebellum). 16</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27695405">PMID:27695405</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27695331">PMID:27695331</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the automated volumetric output of the FreeSurfer image analysis suite, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). FreeSurfer analysis was completed using Version 4.3. The volumetric and cortical thickness measures were corrected for the total intracranial volume. The technical details of these procedures have been described in prior publications. 6 , 7</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27688965">PMID:27688965</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were acquired on a 3T scanner including a magnetization prepared rapid acquisition gradient-echo (MPRAGE) T1-weighted sequence (T1, pre-post Gadolinium(Gd)) and a PD-T2-weighted sequence (T2). The software JIM was used to semi-automatically mask lesions in T2 (T2-hyperintense lesions), T1 (T1-hypointense lesions) and T1Gd sequence. Two raters counted lesions and evaluated new lesions. Regions of Interest (ROI) were semi-automatically placed around single lesions in the PD/T2, T1 and T1Gd sequence. The number of lesions was determined manually while volumes were calculated automatically with the ROI-analysis function. Two raters evaluated the number of new T2/BH lesions. Afterwards, all images were processed with the FSL-toolbox ( Smith et al., 2004 ). Brain tissue volume, normalized for subject head size (NBV = normalized brain volume, NGM = normalized grey matter, NWM = normalized white matter), was estimated with SIENAX ( Smith et al., 2002 ) and lesion volume was normalized based on the SIENAX results. To reduce the risk of false tissue assignment in lesions, lesion masks were dilated and filled with normal appearing white matter contrast. Brain masks were manually corrected to minimize false tissue assignment by the FSL-segmentation. Longitudinal atrophy was assessed with SIENA ( Smith et al., 2002 ) and results were corrected for the individual duration between the two baseline scans to calculate an annualized Percentage Brain Volume Change (aPBVC). In addition, we used FreeSurfer (Version 5.2.0,  http://surfer.nmr.mgh.harvard.edu/ ). To extract reliable and comparable volume estimates from both baseline MRI scans, images were processed with the FreeSurfer longitudinal stream ( Reuter et al., 2012 ). We extracted volumes of the grey and white matter. Brain masks and white/grey matter segmentation were also manually corrected if needed.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27668178">PMID:27668178</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness of the brain was evaluated based on the T1-weighted magnetic resonance images (MRI), acquired on a 1.5T Philips healthcare scanner and processed using the FreeSurfer software (version 5.3.0;  http://www.surfer.nmr.mgh.harvard.edu ). The processing steps were the following. The intensities of the T1-weighted images were bias-corrected using the nonparametric nonuniform intensity normalization implemented in the N3 v1.10 package along with their default parameters. Then, skull stripping was performed using a hybrid watershed algorithm. The cortical regions of the brain were extracted, and the surface of each hemisphere was registered to the surface of the FreeSurfer's atlas. This registration generated the same number of vertices in the two hemispheres. Finally, the cortical thickness of the left hemisphere (LH) was subtracted from the cortical thickness of the right hemisphere (RH) point-by-point. The results are graphically displayed on the LH using a color-coding scheme. A smoothed version of the  “ LH – RH ”  map, spatially smoothed using a 5-mm FHWM Gaussian kernel, is also shown on the patient's LH surface, using the same thresholding and color-coding scheme. Left lateral orbitofrontal and medial orbitofrontal (marked in red color) regions showed thicker cortex ( Fig. A .3).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27667972">PMID:27667972</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27509850">PMID:27509850</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Image post-processing and analysis were performed using FreeSurfer software suite ( http://surfer.nmr.mgh.harvard.edu/ ). Surface-based cortical reconstruction and subcortical volumetric segmentation procedures have been shown elsewhere 53 – 59 . Briefly, three-dimensional model of the cortical surface was generated using MRI scans with 4 attributes: white matter segmentation; tessellation of the gray/white matter boundary; inflation of the folded, tessellated surface; and correction of topological defects 54 ,  55 . Cortical thickness was measured using the distances from each point on the white matter surface to the pial surface 58 . Cortical surface area was measured at the pial surface for the entire cerebrum and for each parcel of the Desikan and Destrieux atlases  54 ,  55 ,  59 ,  60 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27500407">PMID:27500407</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI data were processed using a semiautomated processing stream using the default parameters in FreeSurfer v5.2.0 35 , 36 , 37 , 38 ;  http://surfer.nmr.mgh.harvard.edu/ ), which includes motion and intensity correction, surface coregistration, spatial smoothing, subcortical segmentation, and cortical parcellation based on spherical template registration. The resulting cortical parcellation and subcortical segmentations were individually edited and reviewed for quality control and were used as seeds and targets for DWI tractography (below).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27157680">PMID:27157680</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal volume, ERC, PHG, PCG thickness, and intracranial volume (ICV) were determined through analysis with FreeSurfer ( surfer.nmr.mgh.harvard.edu ), which has been validated as a suitable method for hippocampal segmentation in large samples ( 40 ). The resulting output was quality controlled following a publically available protocol from ENIGMA ( http://enigma.ini.usc.edu/ ) ( 41 ). Whenever a region of interest was detected as inadequately segmented by XC or SFF, its metric was declared missing and excluded from analysis. The numbers of brain regions included in the final analysis were hippocampus right and left ( n  = 270), entorhinal cortex left ( n  = 257) and right ( n  = 268), posterior cingulate gyrus left ( n  = 272) and right ( n  = 271), and parahippocampal gyrus left ( n  = 259) and right ( n  = 271).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26971253">PMID:26971253</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The DMN structural analysis based on segmentation of brain cortical and subcortical structures from T1 high-resolution anatomical data was performed using the Freesurfer toolkit version 5.1 ( http://surfer.nmr.mgh.harvard.edu ). Intracranial volume (ICV) was used to correct the regional volumes and the processing pipeline was the same as previously described (Soares et al.  2012 ). DMN was defined by the summed volume of the angular gyrus of the inferior parietal lobe, the posterior cingulate, the precuneus and the frontopolar region (Raichle et al.  2001 ; Buckner et al.  2008 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26911621">PMID:26911621</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness analysis was conducted using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ). Briefly, brain images were corrected for intensity and contrast irregularities due to magnetic field inhomogeneities [ 36 ]. After an automated skull-stripping procedure, images were manually edited to remove remaining extra-cerebral voxels. In the segmentation step, the gray and white matter interface was determined based on intensity and geometric information [ 36 ]. A triangular surface tessellation was then applied to each hemisphere, and smoothed using a deformable surface algorithm [ 36 ]. The surface of each image was inflated to better visualize cortical folding patterns and sulci [ 37 ]. The inflated cortical surface was then transformed into a parameterizable surface [ 37 ] and aligned with a reference brain template [ 38 ]. Cortical thickness was calculated by taking the smallest distance between the pial surface and gray-white matter interface. FreeSurfer parcellates each hemisphere into 34 regions and derives an average thickness (mm) for each of these [ 39 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26909332">PMID:26909332</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer imaging analysis environment (v5.3.0), which is documented and freely available for download ( http://surfer.nmr.mgh.harvard.edu /), was used to assess the integrated rectified mean curvature of the cortex and as well as brain volumes. A detailed description of the morphometric procedures used in the FreeSurfer pipeline can be found on the FreeSurfer website. Briefly, images were motion corrected, followed by the removal of non-brain tissue using a hybrid watershed/surface deformation procedure ( Segonne et al., 2004 ), automated transformation to Talairach space, deep gray matter and subcortical white matter segmentations ( Fischl et al., 2002 ,  Fischl et al., 2004a ), intensity normalization ( Sled et al., 1998 ), tessellation of the white and gray matter boundary, automated topology correction ( Fischl et al., 2001 ,  Segonne et al., 2007 ), and surface deformation following intensity gradients leading to border placement of gray/cerebrospinal fluid and gray/white at the location where there is the greatest intensity shift thereby defining tissue class transition ( Dale and Sereno, 1993 ,  Dale et al., 1999 ,  Fischl and Dale, 2000 ). Quality control was performed by a trained operator (JBK) throughout MRI processing within the FreeSurfer environment via manual visual inspection of the output for each subject to ensure proper integrity. No subjects included in this study required any manual edits to surface topology.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26903271">PMID:26903271</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing of the structural images collected at both sites was performed using the open-source FreeSurfer pipeline (version 5.3,  http://surfer.nmr.mgh.harvard.edu ). 40 – 43  The longitudinal data was additionally processed using a specialized longitudinal processing stream where unbiased subject-specific templates were created, and then within-subject images for each time-point reprocessed with common information from the template. 44  This has been demonstrated to reduce the variability in within-subject morphological measures, and hence result in greater statistical power and the ability to detect small changes. 44</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26640762">PMID:26640762</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed on T1 images with the Freesurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ). This involves removal of non-brain tissue followed by automated segmentation of cortical and subcortical structures ( Fischl et al., 2002 ). Manual assessment was performed at each step and subjects were excluded if parcellation was either deemed to be inadequate or was unable to be processed. This resulted in exclusion of one subject (52 total included as well as 17 controls).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26512024">PMID:26512024</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite, version 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ), optimized for longitudinal processing and SA accuracy ( 36 , 37 ). The FreeSurfer software pipeline calculates the cortical surface and sulci of every subject using a surface registration method ( 38 ), segments the volumes of subcortical regions ( 28 ), and calculates their volumes in native space without any geometric deformations. The gray-white and pial surfaces were visually inspected, and, where needed, the appropriate manual corrections were performed as per the FreeSurfer tutorial for the longitudinal pipeline ( http://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/ ). All segmentations were visually examined by a trained reviewer blinded to participant group, with particular attention to the consistency of segmentations across time points for each participant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26052283">PMID:26052283</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>1 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26052275">PMID:26052275</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer reconstruction pipeline ( http://surfer.nmr.mgh.harvard.edu , version 5.3) was used for CT analyses. The processing of T1-weighted images included removal of non-brain tissue (Segonne et al.,  2004 ), automated Talairach transformation and intensity normalization (Sled et al.,  1998 ), and surface deformation to detect gray matter/white matter and gray matter/cerebrospinal fluid boundaries (Fischl et al.,  2001 ). The resulting representation of CT was calculated as the distance between tissue boundaries (gray matter/white matter and gray matter/cerebrospinal fluid) (Fischl and Dale,  2000 ). The surface models of each subject were inspected visually for accuracy.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25879588">PMID:25879588</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical volume and thickness measures were estimated using the openware, Freesurfer ( http://surfer.nmr.mgh.harvard.edu/ ). Our MR data were of high quality, and appropriate checks and edits were performed throughout the entire Freesurfer workflow (both authors evaluated/checked all the steps especially, brain segmentation, registration to Talaraich space, pial surface and GM-WM boundaries extraction results in each subject). Standard image-processing steps were adopted including: 1) correct for motion artifacts and strip skull based on a hybrid watershed/surface deformation procedure [ 17 ], 2) register images to a Talairach brain template and segment for subcortical WM and GM structures [ 18 ], 3) estimate the GM-WM boundary via a tessellation step, and subsequently perform automated topology correction, 4) optimally place GM-WM and GM-cerebrospinal fluid (GM-CSF) boundaries using surface normalization and intensity gradients, 6) after cortical models are complete, perform deformable procedures for further processing and analysis, such as surface inflation and registration to a spherical atlas [ 19 ]. Both intensity and continuity information from the entire 3D  MR volume are used to produce representations of cortical thickness, where thickness is measured as the closest distance from GM-WM to GM-CSF boundary at each vertex on the tessellated surface [ 20 ].</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25477721">PMID:25477721</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The MPRAGE data was processed using the Freesurfer\n5.3.0 ( http://surfer.nmr.mgh.harvard.edu ) for\nMac OS. The data processing took about 30 hours \nwith over 30 steps including stripping, normalization,\nremoval of non-brain structure, flattening and image \nreconstruction. [24]  Upon completion of these data\nprocessing steps,the volumes of the left and right \nhippocampus were obtained.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22399092">PMID:22399092</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal and amygdalar volumes were determined by manually tracing the periphery of the Region of Interest (ROI) on each slice of a T1-weighted scan in coronal orientation using Analyze 5.0 (Brain Imaging Resource, Mayo Clinic, Rochester, MI;  Fig. 1 ) by the same experienced tracers. The outlining of the hippocampus and amygdala always proceeded from anterior to posterior and was traced according to the protocol outlined by Watson and colleagues ( Watson et al. 1997 ) with a modification suggested by  Brierly et al. (2002) . Volume estimations were repeated on 10 randomly selected scans and interclass correlations between raters ranged from 0.948 to 0.989 and 0.981 to 0.993 for the right and left hippocampus, and from 0.975 to 0.989 and 0.995 to 0.996 for the right and left amygdala, respectively (averaged 0.982 and 0.996) ( Maller et al. 2007 ). Intracranial volume (ICV) was computed with the Freesurfer/FSL package ( http://surfer.nmr.mgh.harvard.edu/ ;  Fischl et al. 2002 ,  2004 ) for wave 1 and wave 2 images.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/18666833">PMID:18666833</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Data analysis was performed using FS-FAST ( http://surfer.nmr.mgh.harvard.edu ), fROI ( http://froi.sourceforge.net ), and custom-written software. Before statistical analysis, images were motion corrected  [ 41 ] , and the data from the blocked localizer scans (not the event-related scans) were smoothed (3 mm full width at half maximum Gaussian kernel).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28210542">PMID:28210542</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Finally , a field map was acquired using a spoiled gradient echo sequence to correct for field inhomogeneities ( TR = 500 ms , TE1 = 6.5 ms , TE2 = 8.5 ms , flip angle 45° , scan time = 1:10 min ) . 2.4 Neuroimaging data processing 2.4.1 T1-weighted anatomical image processing T1 anatomical images were reconstructed and parcellated into regions of interest using FreeSurfer software ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28203530">PMID:28203530</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Groups were compared using a t -test and results were corrected for multiple comparisons using false discovery rate ( FDR ) ( p = 0.05 ) and cluster size > 30 voxels . 2.4.1.2 Longitudinal 2.4.1.2.1 FreeSurfer We employed the longitudinal FreeSurfer pipeline to evaluate longitudinal CT change .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28180080">PMID:28180080</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All ROIs were automatically segmented on T1-weighted images ( Freesurfer software ( ) and FSL ( v5.0 ) ( ) ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28180075">PMID:28180075</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Bilateral hippocampal volumes were extracted from volumetric segmentations of T1 weighted images by FreeSurfer v5.3 ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28168210">PMID:28168210</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data analysis Structural MRI data Cortical and subcortical regions of interest ( ROIs ) were generated by segmenting a T1‐weighted image , using Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28158201">PMID:28158201</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical surface of each subject was reconstructed from the corresponding magnetic resonance ( MR ) images with the Freesurfer software [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28150723">PMID:28150723</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volume segmentation of structural T1 MRI was performed with Freesurfer , fMRI data were analyzed using mrVista to obtain population receptive field maps , using methods described in Dumoulin and Wandell and Clavagnier et al . .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28149277">PMID:28149277</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain segmentation Freesurfer ( Fischl et al . , ) , version 6-beta ( 20151015 ) longitudinal stream ( Reuter et al . , ) , was used to segment the brain into known anatomical structures .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28148276">PMID:28148276</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Region of interest ( ROI ) analysis was also performed , and segmentation of cortical and subcortical structures was automated with FreeSurfer 5.3 ( https://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28138428">PMID:28138428</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We determined the factors influencing probabilistic tract reconstruction and investigated alterations of DTI scalar metrics along white matter tracts with respect to hippocampal volume , which was automatically estimated using Freesurfer 's morphometric pipelines .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28123950">PMID:28123950</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The lesion masks were then registered onto the cortical surface reconstructions . 2.5 Measures of morphological/intensity features FreeSurfer was used to calculate the established measures : cortical thickness , grey-white matter intensity contrast , curvature , sulcal depth and FLAIR intensity at each vertex of the 3D cortical reconstruction .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28123789">PMID:28123789</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Concerning the neuroimaging analysis , the Qdec interface in Freesurfer software was used : a vertex-by-vertex analysis was carried out performing a general linear model to analyse whether any difference in mean cortical thickness existed between groups ( low : α3/α2 < 1 μV2 ; middle : 1 < α3/α2 < 1.16 μV2 ; high : α3/α2 > 1.17 μV2 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28121613">PMID:28121613</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The preoperative MRI scan was used to generate a 3D rendering of the brain using FreeSurfer ( ; ) ; this rendering was placed in a 3D coordinate system ( RAS ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28119818">PMID:28119818</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This was motivated by the fact that hippocampal volume is the most widely used MRI biomarker of AD ( ) , and since FreeSurfer segments many ROIs simultaneously in one objective function , it is not specific to a certain ROI .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28119593">PMID:28119593</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Reconstruction of the anatomical locations of the implanted electrodes and their mapping onto a standardized set of coordinates across subjects was performed using FreeSurfer image analysis suite and in-house software , as described in detail in Nourski et al . ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28116559">PMID:28116559</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data processing MR structural images were processed using Freesurfer software ( version 4.3.0 ) ( http://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurfer ) , which was run using the Linux 2.6.15-2.5 operating system .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28116233">PMID:28116233</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the vertex-wise calculation of the annual percentage change of cortical T1 and of the cortical thickness , data were further processed using the longitudinal stream implemented in FreeSurfer ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28111547">PMID:28111547</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Several data analysis tools and processing pipelines are currently deployed in these clusters ( CIVET , FreeSurfer , FSL , etc .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28105013">PMID:28105013</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This has been achieved by using the TRACULA tool ( Yendiki et al . , ) ( https://surfer.nmr.mgh.harvard.edu/fswiki/Tracula ) implemented in the FreeSurfer software suite ( http://surfer.nmr.mgh.harvard.edu/fswiki ) . ( 4 ) Diffusion gradients were adjusted for rotations introduced by the eddy current and head movement corrections . ( 5 ) The preprocessed DTI data were then subjected to the DTK to compute voxel-wise diffusion tensors and to construct the ( principal ) eigenvector and eigenvalue maps as well as a map of fractional anisotropy ( FA ) . ( 6 ) Deterministic tractography was conducted in TrackVis using the “brute force” approach with an interpolated streamline tracking algorithm .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28101053">PMID:28101053</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1 -Weighted Neuroimaging Processing Cortical reconstruction and volumetric segmentation was performed with FreeSurfer version 5.3 image analysis suite .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28098162">PMID:28098162</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging analysis and quality control Hippocampal volumes were estimated using the automated and previously validated segmentation algorithms , FSL FIRST from the FMRIB Software Library ( FSL ) and FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28095856">PMID:28095856</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The global cortical thickness ( gCTh ) was analyzed by means of Freesurfer on 3D-T1 sequences .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28095463">PMID:28095463</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>However , since Freesurfer is a well-known segmentation method the studies were presented in and included in the method specific calculations of the regression of BPF in relation to age and the estimated marginal means .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28082873">PMID:28082873</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject the structural scan was processed in FreeSurfer using the default processing settings .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28074926">PMID:28074926</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To obtain precise brain masks for fMRI co-registration , automated whole-brain segmentation was performed on the T1-weighted anatomical references in FreeSurfer ; the surface reconstructions were visually inspected and manually edited when necessary .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28072399">PMID:28072399</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data selection Data were processed primarily using custom scripts , with tools from the FMRIB Software Library ( FSL ) version 4.1.8 and Freesurfer additionally used for registration and motion correction .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28070484">PMID:28070484</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumes of the 14 subcortical areas and the brainstem were computed with FreeSurfer 's automated procedure for volumetric measurements ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28054634">PMID:28054634</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Downsampling has the benefit of making the subsequent network analysis less computationally intensive whilst still preserving key aspects of cortical gyrification and ensuring all triangles are more similarly sized , rather than vast differences in surface area as in the original FreeSurfer generated surface mesh which may bias connectivity to be higher in larger areas .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28050344">PMID:28050344</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to assess the reproducibility of the tractography framework , 5 HV underwent a second MRI acquisition on the same day ( intra-session reproducibility ) and 10 HV on different days ( inter-session reproducibility ) . 2.4 Reconstruction of structural networks 2.4.1 Anatomical parcellation Cortex was parcellated in 3D-structural image with Freesurfer ( FS ) software V5.3.0 ( http://freesurfer.net/ ) in 34 regions per hemisphere with the Desikan-Killiany atlas ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28050342">PMID:28050342</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For methods based on FreeSurfer , we directly used the volume and thickness values provided by FreeSurfer , which are calculated using FreeSurfer 's native Desikan-Killiany atlas ( ) . 3 Experiments This section provides descriptions of the experiments performed using each dataset .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28035900">PMID:28035900</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We have clarified this as follows in the Methods section : Methods : Phase-encoding fMRI analysis : inter-digit overlap “Cortical surface reconstructions were derived from MEMPRAGE images using FreeSurfer ( Marcus et al . , 2011 ) . [ … ] Where A and B are the area of the two digit representations , the Dice Coefficient is expressed as : ” Did the binary maps obtained through different thresholds show the same patterns of overlap across conditions ( i.e. , those reported in ) ?</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28031219">PMID:28031219</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis Automatic segmentation and volume estimation of hippocampal and extrahippocampal subcortical structures was performed using Freesurfer software ( ) applied to the T1 -weighted images , as previously described ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28009020">PMID:28009020</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Segmentation and registration The T1w images were segmented using Freesurfer version 5.1.0 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28008314">PMID:28008314</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Neuroimaging Processing T 1 -weighted MRI scans were processed with the software FreeSurfer version 5.3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28003961">PMID:28003961</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to avoid data irregularities stemming from the use of multiple workstations , all subjects were post-processed on a single Macintosh computer ( osX : 10.10.5 , Model : MacPro 6.1 , Processor : 3.7 Quad-Core Intel Xeon E5 , and RAM : 16GB 1866 MHz DDR3 ECC ) in accordance with the manual guidelines used by FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27981028">PMID:27981028</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For this study T1 images were used only for definition of the cerebellar gray matter reference region in native space ( using FreeSurfer v 5.1 software ) , and for tissue segmentation and spatial warping to the Montreal Neurological Institute template ( using statistical parametric mapping , SPM8 ; http://www.fil.ion.ucl.ac.uk/spm ) . 2.3 PET acquisition and pre-processing All subjects underwent PET imaging with [ 11 C ] PiB at Lawrence Berkeley National Laboratory on a Siemens ECAT EXACT HR camera ( N = 66 ) or a Siemens Biograph PET/CT scanner ( N = 1 ) as described in a previous study ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27976715">PMID:27976715</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing The T1-weighted MRI scans were processed using FreeSurfer ( version 5.1 ) to obtain segmentations and volumetric summaries of the following seven subcortical structures for each hemisphere : nucleus accumbens , amygdala , caudate , hippocampus , pallidum , putamen and thalamus ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27976682">PMID:27976682</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gradient distortion correction was applied using tools developed by the Freesurfer and Human Connectome Project groups , available at https://github.com/Washington-University/Pipelines .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27942448">PMID:27942448</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We extracted brain masks from the T1-weighted volumes using Freesurfer ( ) , used for registration to standard space using FLIRT ( ) with boundary-based registration ( BBR , ( ) ) and FNIRT ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27941946">PMID:27941946</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI processing FreeSurfer pipeline ( v5.0 ) was applied to the MRI scans to estimate the cortical thickness and anatomical structure volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27932963">PMID:27932963</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Using each participant’s MPRAGE scan , individual segmented and inflated cortical surface models were created in Freesurfer ( ; ) and then fMRI data were projected onto those surface models for analysis and visualization .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27925613">PMID:27925613</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>ROIs are displayed on the right hemisphere of the Freesurfer average cortical surface in the Figure , but were defined bilaterally in each subject 's native space for all analyses .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27917950">PMID:27917950</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Quantitative assessments FreeSurfer Freesurfer v5.3.0 was used for image processing .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27917121">PMID:27917121</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Desikan-Killiany cortical and FreeSurfer subcortical segmentation atlas was used to estimate volume in regions of interest ( ROIs ; Fischl et al . , , ) including the bilateral hippocampus and dACC .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27915027">PMID:27915027</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each participant a representation of their cerebral cortex was constructed using FreeSurfer ( Freesurfer 5.3 , Martinos Center for Biomedical Imaging , Boston , Massachusetts ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27911322">PMID:27911322</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer outputs a 3D segmentation of 115 cortical and subcortical brain structures according to the Desikan-Killiany Atlas [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27911297">PMID:27911297</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Recently , FreeSurfer has been shown to have similar reproducibility for atrophic measurements as manual hippocampal segmentation [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27901100">PMID:27901100</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All pre-processing steps were performed using CMTK , FSL , FreeSurfer , MIAMI fuse and in-house MATLAB code .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27897244">PMID:27897244</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All data analyses were performed using MATLAB , Brainstorm , and FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27896064">PMID:27896064</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional task data were processed using a pipeline developed using software from FSL ( ) , FreeSurfer ( ) , and AFNI ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27891073">PMID:27891073</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer . Neuroimage 62 , 774 –781 . 10.1016/j.neuroimage.2012.01.021 22248573 Fischl B .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27883082">PMID:27883082</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI images were processed with Freesurfer software ( version 5.1.0 ) and its specialized tool for automated cortical and subcortical segmentation in order to obtain the volume of several brain areas .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27881948">PMID:27881948</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer . Neuroimage 62 , 774 –781 . 10.1016/j.neuroimage.2012.01.021 22248573 Jenkinson M .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27872808">PMID:27872808</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>At each voxel , MTR was calculated from the Sat and NoSat image intensities as 100 × (NoSat − Sat )  / NoSat . 2.3 Image processing Cortical surface meshes along the white matter and pial surfaces were first generated based on the T1 -weighted image volumes of each patient at each time point using the standard FreeSurfer analysis pipeline ( , , ) , version 5.1.0 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27861502">PMID:27861502</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Projecting resting-state data to the cortex The preprocessed fMRI data was projected onto the cortical surface using Freesurfer tools .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27855061">PMID:27855061</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surfaces were computed with FreeSurfer ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27853245">PMID:27853245</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The surface-based parcellation ( Destrieux parcellation scheme ) is obtained on HCP subjects via the method in the FreeSurfer toolkit .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27852296">PMID:27852296</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>BMI varied from 13.5 to 20.7 within the patient group , and 11 patients had a BMI > 17.5 . FreeSurfer was used to estimate brain volumes and regional cortical thickness .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27845344">PMID:27845344</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To visualize shape variation along the first PC of the shape descriptor for a given structure , we first aligned the structures from all subjects to a template , fsaverage , which is a population average distributed with FreeSurfer , using a seven-parameter ( global scaling plus six-parameter rigid body transformation ) registration with linear interpolation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27843692">PMID:27843692</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer ( Fischl et al . , ) for automated brain segmentation of the T1‐weighted data to obtain brain masks used for co‐registration to a standard coordinate system using FLIRT ( Jenkinson & Smith , ) , optimized using boundary‐based registration ( BBR ; Greve & Fischl , ) and FNIRT ( Andersson , Jenkinson , & Smith , , ) . 2.6 Voxel‐wise GLMs In the first‐level general linear model ( GLM ) , the onset and duration of the PV and tracking blocks ( L1 and L2 ) were modeled with the fixation blocks as implicit baseline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27837005">PMID:27837005</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based cortical thickness comparison between ALS and HC was performed in Qdec V1.5 , a statistical tool part of Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27830169">PMID:27830169</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The impact of the hardware setup on the volumes generated by FreeSurfer was tested by dependent one-sample t -tests with Bonferroni correction .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27830114">PMID:27830114</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were processed with the FreeSurfer image analysis suite ( http://freesurfer.net ) , a set of software tools for the study of cortical and subcortical anatomy .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27828960">PMID:27828960</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer uses a surface based imaging processing pipeline to reconstruct the brain’s cortical surface from structural MRI data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27826238">PMID:27826238</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Repetition time ( TR ) = 2250 ms ; echo time ( TE ) = 2.6 ms ; inversion time ( T1 ) = 900 ms ; field of view ( FOV [ ap , fh , rl ] ) = 240 mm × 256 mm × 176 mm ; slice gap , 0 ; 1.0 mm × 1.0 mm × 1.1 mm voxel size ; flip angle ( FA ) = 9 . Anatomical MRI Processing Image processing ( registration , segmentation and surface reconstruction ) and CT estimation were performed using Freesurfer ( version 5.3.0 ) , a surface based analysis tool that estimates CT by measuring the distance between the boundary of gray/white mater and the pial surface ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27826224">PMID:27826224</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Graphs generation Cortical and sub-cortical gray matter ( GM ) segmentation was performed on 3D T1-weighted images using the Freesurfer image analysis suite ( Reuter et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27809791">PMID:27809791</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric assessment of T1-weighted images will be performed using FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27807349">PMID:27807349</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All decoding analyses were performed with MATLAB R2013a using radial basis function kernel SVM. Cortical current estimation by VBMEG A polygonal model of the cortical surface was constructed based on structural MRI ( T1-weighted ; Signa HDxt Excite 3.0 T ; GE Healthcare UK Ltd , Buckinghamshire , UK ) using the Freesurfer software ( Martinos Center Software ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27807186">PMID:27807186</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>HVs were analyzed from 3T MRIs using FreeSurfer ( 5.3 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27803665">PMID:27803665</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As the processing time is much shorter than that of the Freesurfer pipelines which are widely used in scientific researches , this method has the potential to be adopted in routine clinic .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27803645">PMID:27803645</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To visualize fMRI results in three-dimensional ( 3-D ) space , the T1-weighted anatomical data were segmented and surface-rendered using Freesurfer freeware ( version : 5.3.0 ) ( Dale et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27797832">PMID:27797832</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In humans , we registered the FreeSurfer segmentation of the hippocampus , parahippocampal cortex and perirhinal/entorhinal cortex to a downsampled MNI space at 2 × 2 × 2 mm3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27792012">PMID:27792012</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Participants underwent a 60-min dynamic scan with 11 [ C ] PiB , and PET data was processed using regions of interest ( ROIs ) derived from structural MRIs using Freesurfer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27790095">PMID:27790095</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical and subcortical segmentations based on the T1 images were obtained with FreeSurfer segmentation software ( ) . FIGURE 1 Flow diagram of the SAGIT group tractography framework .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27790082">PMID:27790082</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These MRI images were processed with Freesurfer software ( version 5.1.0 ) and its specialized tool for automated cortical and subcortical segmentation ( Fischl et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27779618">PMID:27779618</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The primary areas like V1 and V2 are also provided as labels with the Freesurfer segmentation using the recon-all pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27769199">PMID:27769199</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the manual procedure of control points , which is implemented in the Freesurfer software package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27747596">PMID:27747596</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Schmitter et al . [ . 99 . ] . AD versus NC . FreeSurfer .   . + .   . SVM . MRI . ADNI . NaN . 82.0 . 88.0 . MorphoBox .   . + .   . SVM . MRI . ADNI . NaN . 86.0 . 91.0 . MCI versus NC . FreeSurfer .   . + .   . SVM . MRI . ADNI . NaN . 66.0 . 80.0 . MorphoBox .   . + .   . SVM . MRI . ADNI . NaN . 69.0 . 83.0 . 2013 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27746713">PMID:27746713</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The effects of FreeSurfer version , workstation type , and Macintosh operating system version on anatomical volume and cortical thickness measurements .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27722088">PMID:27722088</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric tissue segmentation and inference of cortical structure were obtained from each patient 's MPRAGE ( neurological and radiological orientation , independently ) using the Freesurfer software ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27708745">PMID:27708745</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Desikan-Killiany atlas of FreeSurfer was adopted to apply anatomical labels .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27695402">PMID:27695402</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A morphing map to optimally align the cortical surface of each participant to an average cortical representation was computed in FreeSurfer ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27694257">PMID:27694257</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used for quantification of volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27686458">PMID:27686458</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Longitudinal imaging changes were assessed using cortical volumetric reconstruction and segmentation of white and deep grey matter structures with Freesurfer [ 2 ] ; cortical sulci were automatically reconstructed and identified with Brainvisa software , and a voxel-based DTI analysis was performed with Comasoft .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27672558">PMID:27672558</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to reduce within-subject variability between scan sessions , the longitudinal analysis processing pipeline of FreeSurfer was used for subjects with more than one scan ( , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27660604">PMID:27660604</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>DICOM files were converted to nifti volumes ( as required by SPM ) using the mri_convert function of FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/fswiki ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27656132">PMID:27656132</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For comparison , FreeSurfer was used to segment cortical gyral ROIs using each participant’s T1-weighted MPRAGE image ( Desikan et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27656121">PMID:27656121</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Manual validation of FreeSurfer 's automated hippocampal segmentation in normal aging , mild cognitive impairment , and Alzheimer Disease subjects .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27647705">PMID:27647705</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer 's volume‐based stream is designed to preprocess MRI volumes and label subcortical tissue classes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27639350">PMID:27639350</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the FC and FDC analysis , we included intra-cranial volume ( as computed using FreeSurfer from T1-weighted images ( ) ) as a nuisance covariate .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27634551">PMID:27634551</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Looking forward , we suspect that when automated and semi‐automated methods of morphometry ( e.g. , ANTs , FSL , FreeSurfer ) are used to compare populations where movement differences are more prominent or samples are smaller , the effects of head motion could be exacerbated and may incorrectly influence the conclusions drawn from the data [ see Ducharme et al . , ] . Moreover , it seems likely that motion‐related bias will limit other T1w image processing steps that rely on accurate brain anatomy ; for instance , our preliminary observations suggest that T1w scans flagged according to F D ¯ a l l − t a s k and QC ratings exhibit reduced precision of within‐modality registration ( see Supporting Information ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27625975">PMID:27625975</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Subcortical structures , caudate , putamen , pallidum and thalamus were specified by segmentation results from the registered T1-weighted images using Freesurfer ( Fischl ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27622139">PMID:27622139</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.3 Image processing In order to obtain measures of cortical thickness , raw data were imported into FreeSurfer version 5.0 ( http://surfer.mgh.harvard.edu ) . FreeSurfer 's surface-based image processing pipeline is described extensively elsewhere ( , , , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27620984">PMID:27620984</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Next , the data were projected onto a 3D reconstruction of the grey-white matter boundary obtained using FreeSurfer ( , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27611328">PMID:27611328</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical scans were segmented using FreeSurfer [ ] and parcellated according to the Lausanne 2008 atlas included in the connectome mapping toolkit [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27610248">PMID:27610248</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data Processing Data were analyzed using Matlab R2010b ( MathWorks GmbH , Ismaning , Germany ) with a custom-built code , the Toolbox SPM8 ( Wellcome Trust Centre for Neuroimaging , London , UK ) , the FreeSurfer Software Suite ( Martinos Centre for Biomedical Imaging , Charlestown , USA ) , and SPSS V21 ( IBM GmbH , Ehningen , Germany ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27600846">PMID:27600846</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image Preprocessing All data were processed using the Connectome Computation System ( CCS ) ( https://github.com/zuoxinian/CCS ) , which provides a platform for multimodal image analysis by combining components of AFNI ( ) , FSL ( ; ) , FreeSurfer ( ; Fischl et al . 1999a ) , and SPM ( http://www.fil.ion.ucl.ac.uk/spm ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27598250">PMID:27598250</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 weighted MP-RAGE scan served to segment the bilateral caudate nucleus , globus pallidus , putamen , and thalamus in an automated manner using FreeSurfer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27595976">PMID:27595976</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The volumes of subcortical structures in both left and right hemispheres including thalamus , caudate , putamen , pallidum , hippocampus , amygdala , and nucleus accumbens , and ICV were measured also using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27582714">PMID:27582714</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The phase lags ( i.e. , the resulting polar angle and eccentricity maps ) were then overlaid onto the reconstructed , inflated cortical surface ( obtained via FreeSurfer , Martinos Center for Biomedical Imaging , Boston , MA , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27578494">PMID:27578494</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were analyzed with FreeSurfer ( www.surfer.nmr.mgh.harvard.edu , version 6.0 beta ) ( ; , ) in order to perform subvoxel reconstruction of the pial surface and gray matter/white matter boundary ( i.e. , the white matter surface ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27562529">PMID:27562529</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A high resolution 3D T1-weighted structural image was acquired using a magnetization prepared rapid gradient echo sequence with the following parameters : repetition time = 2250 milliseconds ; echo time = 2.99 milliseconds ; inversion time = 900 milliseconds ; flip angle = 9° ; field of view = 256 mm × 240 mm × 192 mm ; voxel size = 1 mm isotropic ; GRAPPA ( generalized autocalibrating partially parallel acquisitions ) acceleration factor = 2 ; acquisition time of 4 minutes and 32 seconds . 2.2.2 Cortical reconstruction and structural measures Cortical reconstructions were generated using the software FreeSurfer ( , , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27555964">PMID:27555964</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Primary Visual Cortex Thickness T1-weighted volumetric images were processed using FreeSurfer 5.3.0 ( http://freesurfer.net/ ) using the standard processing pipeline [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27552053">PMID:27552053</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Spatial correspondence was calculated in S1 ( as defined by FreeSurfer ) between each possible digit pair across the split-halves of the unimanual digit maps ( , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27551668">PMID:27551668</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.5 Cortical and subcortical Freesurfer analyses Scans were analysed using standardized image toolbox ( Freesurfer , version 5.0 ) ( ) , quality assurance ( outlier detection based on inter quartile of 1.5 standard deviations along with visual inspection of segmentations ) , and statistical methods .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27547729">PMID:27547729</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Full details of the acquisition protocols can be found at http://www.humanconnectome.org/documentation/Q1/imaging-protocols.html . Subject characteristics are included in . 2.2 Image processing high resolution pipeline FreeSurfer recon-all was used to generate the cortical surface mesh of NKI data from the T1 image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27547184">PMID:27547184</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To isolate and examine hippocampal rCBF , we used FreeSurfer 's ( version 5.3.0 ) automated subcortical processing stream ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27536233">PMID:27536233</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The joint fusion algorithm was applied on our HD MRI atlas candidates by using 20 T1-w MRI with whole brain label atlases that were already available from Neuromorphometrics Inc . Note that we further incorporated white matter sub-parcellation by using FreeSurfer automatic segmentation so that our output incorporates both white and gray matter sub-parcellation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27534871">PMID:27534871</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated volume measures were performed with FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27527227">PMID:27527227</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The individual subject’s MRIs were converted and processed using the Freesurfer software to derive the boundary element models . ( 2 ) MRI-MEG co-registration .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27524960">PMID:27524960</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>It was used for definition of TMS targets and registration with MEG imaging using FreeSurfer ( Dale et al . , ; Martinos Center for Biomedical Imaging ) and Brainstorm ( Tadel et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27512372">PMID:27512372</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Table 2 Median of PVD . Structure . volBrain . Freesurfer . FIRST . Lat . Ventricles . 4.95 [ 7.42 ] . 5.06 [ 4.85 ] . — . Caudate . 0.53 [ 1.22 ] . 1.39 [ 2.47 ] . 1.19 [ 1.20 ] . Putamen . 0.69 [ 1.76 ] . * . 2.23 [ 2.90 ] . 1.20 [ 1.23 ] . Thalamus . 0.82 [ 1.04 ] . 0.93 [ 0.66 ] . 0.98 [ 1.67 ] . Pallidum . 1.60 [ 1.92 ] . 3.18 [ 3.72 ] . 0.92 [ 0.96 ] . † . Hippocampus . 1.41 [ 2.97 ] . 1.73 [ 1.74 ] . 2.15 [ 3.20 ] . Amygdala . 3.38 [ 2.06 ] . 4.13 [ 5.23 ] . 3.94 [ 3.63 ] . Accumbens . 2.65 [ 22.81 ] . 2.68 [ 2.48 ] . 4.26 [ 6.92 ] . Overall . 1.59 . 2.33 . 2.09 volBrain was compared to the other two methods ( corresponding interquartile interval is shown in brackets ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27508254">PMID:27508254</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The 3D anatomical mesh of the conventional T1-w was created in FreeSurfer and imported in mrVista ( https://github.com/vistalab/vistasoft ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27500640">PMID:27500640</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All of the structural MRI images were preprocessed using the fully automated pipeline of FreeSurfer 5.3.0 [ – ] for volumetric segmentation and cortical reconstruction .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27499204">PMID:27499204</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The white asterisk depicts the interaction between the processing of lip-related sounds with the lip-stimulation , due to the recruitment of overlapping neural resources ( brain surface reconstructed from the MRI of one author using Freesurfer image analysis suite , version 5.3.0 , site URL http://freesurfer.net ) . Figure 2 ERD/ERS pattern over left electrodes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27489875">PMID:27489875</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Only subjects with Freesurfer cortical and hippocampal segmentations judged acceptable by the structural MR processing core were included .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27489767">PMID:27489767</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition , experienced neuroradiologists visually confirmed the good or fair parcellation in each volumetric analysis . 2.3 FreeSurfer volumetry of hippocampal subfields FreeSurfer software ver.5.3 was used for calculation of hippocampal subregions volumes using 3D T1-weighted images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27488504">PMID:27488504</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical MRI ( T1 ) for each subject was treated with the FreeSurfer software to obtain automatic volumetric segmentation and cortical parcellation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27486400">PMID:27486400</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For a more detailed description of the Freesurfer processing methods used by our group see Szymkowicz et al . ( ) . Brain volumetric indices including regional and global volumes of cortical and subcortical structures as well as cortical thickness were generated .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27484739">PMID:27484739</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Images were analyzed with the FreeSurfer ( http://freesurfer.net ) semi-automated processing pipeline [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27484312">PMID:27484312</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data acquisition and analyses In the present study , brain images were collected using 2 different MRI scanners : ( 1 ) 1.5T MR scanner ( Siemens Symphony , Erlangen , Germany ) , equipped with a circularly polarized transmit–receive coil to acquire 3D magnetization-prepared rapid gradient echo ( MPRAGE ) T1-weighted scan ( repetition time [ TR ] 2.010 milliseconds , echo time [ TE ] 3.93 milliseconds , matrix 1 × 1 × 1 , in-plane field of view [ FOV ] 250 3 250 mm2 , slice thickness 1 mm , flip angle 15° ) ; and ( 2 ) 1.5T MR scanner ( Siemens Avanto ) to acquire 3D MPRAGE T1-weighted scan ( TR 2.050 milliseconds , TE 2.56 milliseconds , matrix 1 × 1 × 1 , in-plane FOV 256 mm2 , slice thickness 1 mm , flip angle 15° ) . T1-weighted MRI were analyzed with FreeSurfer version 5.0 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27483187">PMID:27483187</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To display statistics at the group level , the statistic of interest was projected on the parcellated ( aparc 2009 ( ref . ) cortical inflated map associated with the Freesurfer average template ( ‘fsaverage ' ) using SUMA . MVPA was performed within anatomical ROIs rather than a moving ‘searchlight' because we wished to preserve borders between spatially adjacent areas ( for example , IFG and STG ) that exhibited differential phoneme specificity at noisy conditions .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27482534">PMID:27482534</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Minimal manual edits were completed when necessary by a single individual ( T.M.H . ) . Intracranial volume ( ICV ) estimates from FreeSurfer were used to normalize hippocampal thickness estimates from baseline scans for baseline-only analyses .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27478822">PMID:27478822</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Employing FLIRT’s transformation matrix output , the FreeSurfer derived nodes were then registered to DWI using FSL’s ApplyXFM .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27473322">PMID:27473322</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing Structural MRI Cortical reconstruction and volumetric segmentation of 14 subcortical areas ( Supplementary Material 1 , left and right : GP , thalamus , caudate nucleus , putamen , hippocampus , amygdala , and NAcc ) was performed with FreeSurfer image analysis software v4.4.0 , which is documented and freely available for download on-line ( surfer . nmr . mgh . harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27472278">PMID:27472278</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The segmentation and meshing for Head 2 was done using the same pipeline as for Head 1 , with small modifications : First , the FreeSurfer [ – ] reconstruction was done using the T1-weighted data only as this was observed to result in slightly smoother boundary surface between the gray and white matter .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27466220">PMID:27466220</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Then , all nonbrain voxels were excluded using the Freesurfer software ( relevant functions : ‘fslmerge’ to create a mean brain mask and ‘fslstats’ to remove nonbrain voxels ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27464262">PMID:27464262</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The masked T1‐weighted images were used to segment gray matter and white matter structures for volumetric analyses ( FreeSurfer ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27462146">PMID:27462146</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the neuroimaging analysis , the Qdec interface in FreeSurfer application was used ; a vertex-by-vertex evaluation was carried out by a general linear model to analyze whether any change in mean cortical thickness existed among the groups ( low : α3/α2<1 μV2 ; center : 1< α3/α2<1.16 μV2 ; and high : α3/α2>1.17 μV2 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27459196">PMID:27459196</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The imaging data were analyzed using FreeSurfer software and the cortical surface was reconstructed to measure surface areas at 160k surface locations for each hemisphere .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445740">PMID:27445740</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Every volume for each participant was corrected for head size with the following formula proposed by : V adj =V nat -b ( eTIV fs - eTIV ¯ fs ) where Vadj is the covariance-corrected ( adjusted ) volume , Vnat is the original volume calculated by FreeSurfer in native space , b is the slope of the regression of Vnat over eTIVfs , which is the participant’s eTIV produced by FreeSurfer , and is the average eTIV of the group .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445715">PMID:27445715</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data Analysis fMRI Data were registered to the anatomical T1 volume using the bbregister ( FreeSurfer ) program after which registration was verified by eye and was adjusted by hand when needed .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445675">PMID:27445675</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data preprocessing and feature extraction The preprocessing of the entire data set of structural MRI included the volumetric segmentation and cortical reconstruction by the Freesurfer image analysis suite version 5.1.0 , documented and freely available online ( http://freesurfer.net/ ; Fischl et al . , , ; Klein and Tourville , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27445670">PMID:27445670</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Intervals with excessive motion were excluded , data were sampled to and smoothed on the surface , and each brain was warped to a surface-based template using FreeSurfer ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27437579">PMID:27437579</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The mean ( across space ) of the absolute value of the areal distortion averaged across subjects from both registration stages was 30 % less than the standard FreeSurfer folding-based registration and the maximum ( across space ) of this measure was 54 % less . Despite less overall distortion , the areal-feature-based registration delivers substantially more accurate registration of cortical areas than does FreeSurfer folding-based registration as judged by cross-subject task fMRI statistics , an areal feature that was not used to drive the registration .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27433401">PMID:27433401</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Shaw Nicolas Cherbuin 487 504 Diagnosis 1 2015 5 Structural neuroimaging in the detection and prognosis of pre-clinical and early AD Behav Neurol Fennema-Notestine C McEvoy LK Hagler DJ Jr Jacobson MW Dale AM The Alzheimer 's Disease Neuroimaging Initiative 3 12 21 2009 19847040 6 Comparison of different MRI brain atrophy rate measures with clinical disease progression in AD Neurology Jack CR Jr Shiung MM Gunter JL O ' Brien PC Weigand SD Knopman DS Boeve BF Ivnik RJ Smith GE Cha RH Tangalos EG Petersen RC 591 600 62 2004 14981176 7 MRI measures of entorhinal cortex vs hippocampus in preclinical AD Neurology Killiany RJ Hyman BT Gomez-Isla T Moss MB Kikinis R Jolesz F Tanzi R Jones K Albert MS 1188 1196 58 2002 11971085 8 Minimal atrophy of the entorhinal cortex and hippocampus : progression of cognitive impairment Dement Geriatr Cogn Disord Varon D Loewenstein DA Potter E Greig MT Agron J Shen Q Zhao W Celeste Ramirez M Santos I Barker W Potter H Duara R 276 283 31 2001 21494034 9 Alzheimer disease : quantitative structural neuroimaging for detection and prediction of clinical and structural changes in mild cognitive impairment Radiology McEvoy LK Fennema-Notestine C Roddey JC Hagler DJ Jr Holland D Karow DS Pung CJ Brewer JB Dale AM ; Alzheimer 's Disease Neuroimaging Initiative 195 205 251 2009 19201945 10 Brain atrophy associated with baseline and longitudinal measures of cognition Neurobiol Aging Cardenas VA Chao LL Studholme C Yaffe K Miller BL Madison C Buckley ST Mungas D Schuff N Weiner MW 572 580 32 2011 19446370 11 Multivariate analysis of MRI data for Alzheimer 's disease , mild cognitive impairment and healthy controls Neuroimage Westman E Simmons A Zhang Y Muehlboeck JS Tunnard C Liu Y Collins L Evans A Mecocci P Vellas B Tsolaki M Kłoszewska I Soininen H Lovestone S Spenger C Wahlund LO ; AddNeuroMed consortium 1178 1187 54 2011 20800095 12 Analysis of regional MRI volumes and thicknesses as predictors of conversion from mild cognitive impairment to Alzheimer 's disease Neurobiol Aging Liu Y Paajanen T Zhang Y Westman E Wahlund LO Simmons A Tunnard C Sobow T Mecocci P Tsolaki M Vellas B Muehlboeck S Evans A Spenger C Lovestone S Soininen H ; AddNeuroMed Consortium 1375 1385 31 2010 20447732 13 AddNeuroMed and ADNI : similar patterns of Alzheimer 's atrophy and automated MRI classification accuracy in Europe and North America Neuroimage Westman E Simmons A Muehlboeck JS Mecocci P Vellas B Tsolaki M Kłoszewska I Soininen H Weiner MW Lovestone S Spenger C Wahlund LO ; AddNeuroMed consortium ; Alzheimer 's Disease Neuroimaging Initiative 818 828 58 2011 21763442 14 Toward defining the preclinical stages of Alzheimer 's disease : recommendations from the National Institute on Aging-Alzheimer 's Association workgroups on diagnostic guidelines for Alzheimer 's disease Alzheimers Dement Sperling RA Aisen PS Beckett LA Bennett DA Craft S Fagan AM Iwatsubo T Jack CR Jr Kaye J Montine TJ Park DC Reiman EM Rowe CC Siemers E Stern Y Yaffe K Carrillo MC Thies B Morrison-Bogorad M Wagster MV Phelps CH 280 292 7 2011 21514248 15 Science to practice : translating automated brain MRI volumetry in Alzheimer 's disease from research to routine diagnostic use in the work-up of dementia Front Neurol Rathakrishnan BG Doraiswamy PM Petrella JR 216 4 2014 24409168 16 Atrophy patterns in Alzheimer 's disease and semantic dementia : a comparison of FreeSurfer and manual volumetric measurements Neuroimage Lehmann M1 Douiri A Kim LG Modat M Chan D Ourselin S Barnes J Fox NC 2264 2274 49 2010 19874902 17 High-throughput , fully automated volumetry for prediction of MMSE and CDR decline in mild cognitive impairment Alzheimer Dis Assoc Disord Kovacevic S Rafii MS Brewer JB ; Alzheimer 's Disease Neuroimaging Initiative 139 145 23 2009 19474571 18 Validation of a fully automated 3D hippocampal segmentation method using subjects with Alzheimer 's disease mild cognitive impairment , and elderly controls Neuroimage Morra JH Tu Z Apostolova LG Green AE Avedissian C Madsen SK Parikshak N Hua X Toga AW Jack CR Jr Weiner MW Thompson PM ; Alzheimer 's Disease Neuroimaging Initiative 59 68 43 2008 18675918 19 Fully-automated quantification of regional brain volumes for improved detection of focal atrophy in Alzheimer disease AJNR Am J Neuroradiol Brewer JB Magda S Airriess C Smith ME 578 580 30 2009 19112065 20 Influence of age , disease onset and ApoE4 on visual medial temporal lobe atrophy cut-offs J Intern Med Pereira JB Cavallin L Spulber G Aguilar C Mecocci P Vellas B Tsolaki M Kłoszewska I Soininen H Spenger C Aarsland D Lovestone S Simmons A Wahlund LO Westman E ; AddNeuroMed consortium and for the Alzheimer 's Disease Neuroimaging Initiative 317 330 275 2014 24118559 21 Qualitative assessment of cerebral atrophy on MRI : inter- and intra-observer reproducibility in dementia and normal aging Eur Neurol Scheltens P Pasquier F Weerts JG Barkhof F Leys D 95 99 37 1997 9058064 22 Visual assessment of medial temporal lobe atrophy on magnetic resonance imaging : interobserver reliability J Neurol Scheltens P Launer LJ Barkhof F Weinstein HC van Gool WA 557 560 242 1995 8551316 23 The interuncal distance : a new MR measurement for the hippocampal atrophy of Alzheimer disease AJNR Am J Neuroradiol Dahlbeck JW McCluney KW Yeakley JW Fenstermacher MJ Bonmati C Van Horn G 3rd Aldag J 931 932 12 1991 http://www.ajnr.org/content/12/5/931.short 1950924 24 Temporal lobe atrophy on magnetic resonance imaging in the diagnosis of early Alzheimer 's disease Arch Neurol Erkinjuntti T Lee DH Gao F Steenhuis R Eliasziw M Fry R Merskey H Hachinski VC 305 310 50 1993 8442711 25 Radial width of the temporal horn : a sensitive measure in Alzheimer disease AJNR Am J Neuroradiol Frisoni GB Geroldi C Beltramello A Bianchetti A Binetti G Bordiga G DeCarli C Laakso MP Soininen H Testa C Zanetti O Trabucchi M 35 47 23 2002 http://www.ajnr.org/content/23/1/35.full 11827874 26 MTA index : a simple 2D-method for assessing atrophy of the medial temporal lobe using clinically available neuroimaging Front Aging Neurosci Menéndez-González M López-Muñiz A Vega JA Salas-Pacheco JM Arias-Carrión O 23 6 2014 24715861 27 Feasibility of the medial temporal lobe atrophy index ( MTAi ) and derived methods for measuring atrophy of the medial temporal lobe Front Aging Neurosci Conejo Bayón F Maese J Fernandez Oliveira A Mesas T Herrera de la Llave E Álvarez Avellón T Menéndez-González M 305 6 2014 25414666 28 Planimetry of the medial temporal lobe : a feasible method for supporting the diagnosis of Alzheimer’s disease in clinical practice Neurology and Neuroscience Menéndez-González M Fernández Oliveira A Conejo Bayón F Maese J Mesas Uzal T Herrera de la Llave E Álvarez Avellón T 8 and 1 2015 29 The use of MRI , CT and lumbar puncture in dementia diagnostics : data from the SveDem Registry Dement Geriatr Cogn Disord Falahati F Fereshtehnejad SM Religa D Wahlund LO Westman E Eriksdotter M 81 91 39 2015 25358376 30 Development of an MRI rating scale for multiple brain regions : comparison with volumetrics and with voxel-based morphometry Neuroradiology Davies RR Scahill VL Graham A Williams GB Graham KS Hodges JR 491 503 51 2009 19308367 31 Visual assessment of medial temporal atrophy on MR films in Alzheimer 's disease : comparison with volumetry Aging Clin Exp Res Bresciani L Rossi R Testa C Geroldi C Galluzzi S Laakso MP Beltramello A Soininen H Frisoni GB 8 13 17 2005 15847116 32 Visual rating and volumetry of the medial temporal lobe on magnetic resonance imaging in dementia : a comparative study J Neurol Neurosurg Psychiatry Wahlund LO Julin P Johansson SE Scheltens P 630 635 69 2000 http://jnnp.bmj.com/cgi/pmidlookup ? view=long&pmid=11032615 11032615 33 Visual rating of hippocampal atrophy : correlation with volumetry J Neurol Neurosurg Psychiatry Vermersch P Leys D Scheltens P Barkhof F 1015 57 1994 8057102 34 Combination analysis of neuropsychological tests and structural MRI measures in differentiating AD , MCI and control groups--the AddNeuroMed study Neurobiol Aging Liu Y Paajanen T Zhang Y Westman E Wahlund LO Simmons A Tunnard C Sobow T Mecocci P Tsolaki M Vellas B Muehlboeck S Evans A Spenger C Lovestone S Soininen H ; AddNeuroMed Consortium 1198 1206 Lovestone S 32 2011 19683363 35 Biomarkers for disease modification trials--the innovative medicines initiative and AddNeuroMed J Nutr Health Aging Lovestone S Francis P Strandgaard K 359 361 11 2007 17653500 36 The AddNeuroMed framework for multi-centre MRI assessment of Alzheimer 's disease : experience from the first 24 months Int J Geriatr Psychiatry Simmons A Westman E Muehlboeck S Mecocci P Vellas B Tsolaki M Kłoszewska I Wahlund LO Soininen H Lovestone S Evans A Spenger C 75 82 C .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27415826">PMID:27415826</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer ( version 5.1 ) [ ] was used to calculate regional cortical volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27415431">PMID:27415431</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI analysis Automated methods to measure cortical volume were performed using FreeSurfer ( version 4.5.1 ) [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27408790">PMID:27408790</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These data were downloaded in table format from https://db.humanconnectome.org/ and were the result of the HCP FreeSurfer preprocessing pipeline ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27386561">PMID:27386561</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical MRI analysis The T1 anatomical MRIs were analyzed in FreeSurfer ( FS ) version 5.3 ( surfer . nmr . mgh . harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27383146">PMID:27383146</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Thalamic masks and occipital lobe masks were obtained using FSL FIRST [ ] and FreeSurfer ( http://freesurfer.net/ ) , respectively . Thalamic masks were further modified by constraining them to voxels whose FA was in the range [ 0.1 , 0.5 ] in order to remove parts of the cortico-spinal tract ( CST ) that were incorrectly included in the original segmentation , as proposed by Mang et al . [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27378875">PMID:27378875</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing and reconstruction of the T1 anatomical data was done using the FreeSurfer longitudinal stream Software package version 5.3 , as this procedure is unbiased by the number of time points and more robust when multiple time points are available ( Reuter et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27378836">PMID:27378836</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Developers of the SimNIBS software pipeline recommend MPRAGE acquisitions with selective water excitation for fat suppression for FreeSurfer tools to work well ( version2 : mri_sequences | SimNIBS ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27375417">PMID:27375417</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Inter-subject cortical surface registration for group analysis A morphing map to optimally align the cortical surface of each participant to an average cortical representation ( FsAverage in FreeSurfer ) was computed in FreeSurfer ( Fischl et al . , ) . Data analysis Phase locking Inter Trial Phase Locking ( PL ) is a method to quantify phase synchrony across multiple trials .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27371764">PMID:27371764</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Scans of the same type were averaged ; the mean image of the T 1 w MPRAGE scans was then divided by the mean image of the T 2 w TSE scans to create a T 1 w/T 2 w image , after the methods of . Structural Analysis : 7T MP2RAGE For each of the 10 individuals scanned at 7T , the structural MP2RAGE scan was reconstructed using the Freesurfer recon-all pipeline to create a cortical surface .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27366994">PMID:27366994</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray/white matter segmentation , cortical surface reconstruction , and surface inflation and flattening were completed using FreeSurfer ( Dale , Fischl , & Sereno , ; Fischl , Sereno , & Dale , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27364184">PMID:27364184</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For illustration , individual maps were morphed by linear mapping to a common template ( “fsaverage” in FreeSurfer package ) and averaged across subjects .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27357864">PMID:27357864</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer ( https://surfer.nmr.mgh.harvard.edu/fswiki ) for automatic segmentation and reconstruction to create a three-dimensional inflated model of the cortical surfaces for the grey–white matter boundary and the pial surface , respectively .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27357347">PMID:27357347</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the analysis of correlations between regional 18 F-FDG SUVRs and 18 F-AV-1451 SUVRs in Patients A and B ( ) , the MRI was segmented into grey and white matter and regions of interest were defined using FreeSurfer ( version 5.3 and its Desikan-Killiany Atlas ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27352783">PMID:27352783</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>( A ) Surface reconstruction was carried out using Freesurfer software ; Local Gyrification Indices were estimated using Schaer’s procedure ; ( B ) Desikan atlas was used for parcellating the cortical surface to 68 regions ( 34 on each hemisphere ) ; ( C ) Association matrices were obtained by calculating the correlations between regional gyrification across subjects within each group separately ; ( D ) Binary adjacency matrices were derived from thresholding at minimum density for fully connected graphs in all groups .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27351196">PMID:27351196</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing Scans were processed using FreeSurfer 5.3 [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27347263">PMID:27347263</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric Analysis Cortical reconstruction and volumetric segmentation was performed through the Freesurfer image analysis suite , which is documented and freely available for download online at ( surfer . nmr . mgh . harvard.edu/ ) . Freesurfer volumetric analysis has been widely used across past studies addressing various issues [ - ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27336309">PMID:27336309</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The segmented gray matter was partitioned into 144 anatomic regions of interest ( ROIs ) to represent nodes of the individual brain networks using Destrieux atlas of Freesurfer [ ] . 10.1371/journal.pone.0157268.g001 Fig 1 Diagram of the brain network analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27336050">PMID:27336050</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For statistical analysis of imaging data , following segmentation that included use of a multi-atlas technique ( ) hippocampal volume data were analyzed with linear mixed effects Models in the Freesurfer package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27336028">PMID:27336028</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1-weighted scans were used to extract anatomically delineated regions of interest ( ROIs ) of the caudate nucleus , putamen , nucleus accumbens , DLPFC ( consisting of the rostral middle frontal gyrus ) , mOFC , and IFG , consisting of the pars opercularis , pars orbitalis , and pars triangularis ; , ) in each hemisphere using the FreeSurfer 5.1.0 structural imaging pipeline.A set of two transverse DWI scans were acquired ( 30 diffusion-weighted volumes with different non-collinear diffusion directions with b-factor=1,000 s/mm2 and eight diffusion-unweighted volumes with b-factor=0 s/mm2 ; parallel imaging SENSE factor=2.5 ; flip angle=90° ; 60 slices of 2.5 mm ; no slice gap ; 96×96 acquisition matrix ; reconstruction matrix 128×128 ; FOV=240 mm ; TE=88 ms ; TR=9,822 ms ; no cardiac gating ; and total scan duration=296 s ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27330979">PMID:27330979</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain segmentation and volumetric information was obtained from MPRAGE images using FreeSurfer ( Martinos Center for Biomedical Imaging , MGH/HST , US ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27330836">PMID:27330836</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Han et al . describe that segmentation inaccuracies are minimal with FreeSurfer recommended imaging parameters and do not affect study outcome , thus reducing the need to verify segmentation accuracy for each subject [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27327256">PMID:27327256</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Second , voxel dimension drift was corrected using linear registration procedures used by the longitudinal processing stream in FreeSurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu/fswiki/LongitudinalProcessing ) , which involves the creation of an unbiased within-subject template space and average image using robust , inverse consistent registration .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27326542">PMID:27326542</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Feedback was solicited over Twitter , by presenting BIDS and distributing informational pamphlets at conferences ( INCF , SfN ) , as well as by sending emails to SPM , FSL , Freesurfer , MRTrix , Slicer , Nipy , and HCP mailing lists ( reaching over 5,000 researchers ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27322475">PMID:27322475</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The intracranial volume ( ICV ) calculated by FreeSurfer did not differ significantly between groups ( t -test , P = .514 ) and Student 's t -test was done on the ICV normalized data ( i.e. volume/ICV ) to assess between-group differences ( Supplementary Table S2 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27317925">PMID:27317925</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based analyses were performed using FreeSurfer ( FreeSurfer ; Martinos Center for Biomedical Imaging , Boston , MA , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27317630">PMID:27317630</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data Analysis Plan T1-Weighted 3D Anatomical Images Anatomical images will be pre-processed using the FreeSurfer imaging analysis suite version 5.3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27302746">PMID:27302746</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In the present study , a seed-based functional connectivity analysis was applied along the cortical surfaces , which were reconstructed using the Freesurfer software package ( MGH , Harvard Medical School , Boston , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27302485">PMID:27302485</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and measures Surface-based cortical reconstruction was performed in FreeSurfer ( Version 5.3 , https://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27298764">PMID:27298764</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition , whole brain diffusion-weighted images were collected ( twice refocused PGSE EPI ; 60 diffusion-weighted images ; b-factor 1000 s/mm2 ; 5 non-diffusion-weighted images ; 60 slices interleaved ; TE/TR = 97/8500 ms ; GRAPPA-acceleration 2 ; voxel size 2.0 × 2.0 × 2.2 mm ) . 2.4 MRI processing 2.4.1 Measures of gray matter structure 2.4.1.1 Cortical thickness and areal expansion We extracted cortical thickness and areal expansion estimates using Freesurfer v5.3 software ( http://surfer.nmr.mgh.harvard.edu/ ; ( , ) ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27293419">PMID:27293419</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer We tested FreeSurfer on both the BrainWeb and the IBSR datasets in a fully automatic mode without any user intervention .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27282108">PMID:27282108</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical MRI analysis We reconstructed the cortical surface of each participant using Freesurfer on the basis of the T1 structural scan .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27280153">PMID:27280153</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The mask was constructed from the standard surface cvs_avg35_inMNI152 in Freesurfer , using mri_label2vol to combine multiple labels from the aparc.a2009s atlas into one mask ( G_pariet_inf-Supramar , G_pariet_inf-Angular , G_temp_sup-Plan_tempo , G_temp_sup-Lateral , G_temp_sup-G_T_transv , S_interm_prim-Jensen , S_temporal_sup , S_temporal_transverse ) and trimming temporal cortex voxels anterior to the postcentral sulcus .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27273817">PMID:27273817</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomically realistic full head model was constructed using well-known tools : FreeSurfer , FMRIB FSL , and Seg3D ( refer to Kim et al . for details ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27273250">PMID:27273250</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging biomarkers Imaging data in this study is obtained from dataset UCSF—Cross-Sectional FreeSurfer ( FreeSurfer Version 4.3 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27272936">PMID:27272936</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface models ( ) were reconstructed using FreeSurfer software ( v5.1 ) [ ] , and imported to SUMA for visualization [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27263124">PMID:27263124</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer brain template ( FSAverage ; see ) was used as the brain model .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27252872">PMID:27252872</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing For all participants , images were first preprocessed with the FreeSurfer software package , V.5.3.0 , including removal of non-brain tissue and intensity correction .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27250872">PMID:27250872</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical current estimation using a hierarchical Bayesian method To map the current dipoles on the cortical surface , we constructed a polygon model of the cortical surface based on MR structure images using FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27242453">PMID:27242453</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Eye motion was additionally measured with an Eye Link SR Research Eye Tracker . Preprocessing Data were analyzed with MATLAB version R2013a and the open source MATLAB toolbox FieldTrip for analysis of neuroimaging data ( Oostenveld et al . , ) , as well as FreeSurfer ( Version 5.3.0 ; Fischl , ) and MNE-Suite ( Version 2.7.0 ; http://www.martinos.org/mne/stable/index.html ) for some steps of source localization ( see below ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27239527">PMID:27239527</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We investigated the impact of segmentation methodologies ( manual vs. automated ) through meta-regression analyses and found that manual segmentation of the hippocampus resulted in larger atrophy rate estimates compared with automatic segmentation using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27239489">PMID:27239489</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The whole hippocampus was initially segmented by completing the FreeSurfer image analysis pipeline ( Martinos Center for Biomedical Imaging , Boston , MA , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27231194">PMID:27231194</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This pre-processing pipeline incorporated functions in FSL and FreeSurfer , including gradient distortion correction , motion correction , field map pre-processing , distortion correction , EPI to T1 registration , resampling original EPI frames to atlas space , intensity normalization , and ribbon-based native-to-standard space registration .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27230218">PMID:27230218</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer DK atlas produced the initial parcellations based on gyri and sulci .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27226895">PMID:27226895</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer default templates , Desikan-Killiany atlas [ ] for cortical regions and the aseg atlas [ ] for subcortical regions , were used for segmentation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27217080">PMID:27217080</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual single-layer boundary-element models ( BEMs ) were created for each participant using watershed segmentation algorithms ( FreeSurfer 4.3 software , Martinos Center for Biomedical Imaging ) to reconstruct the brain’s cortical grey matter surface as a high-resolution triangularised mesh with 10424 vertices in each hemisphere .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27214389">PMID:27214389</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface thickness was estimated in 3D‐T1W images using FreeSurfer ( General Hospital Corporation , Boston , MA ) ; tissue segmentation and cortical parcellation were performed with the Desikan‐‐Killiany cortical atlas .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27213683">PMID:27213683</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To maintain consistency amongst the expert raters , the final segmentations in all scans was checked by a single expert rater , J.Y . W . Segmentation correction We supplied three sets of images to the corrective learning tool , SegAdapter ( http://www.nitrc.org/projects/segadapter/ ) : ( 1 ) segmentation from the automated FreeSurfer process , ( 2 ) manually edited segmentation and ( 3 ) MPRAGE scans , with signal intensity squared to improve contrast .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27207916">PMID:27207916</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To exploit novel methods of automated hippocampal subfield segmentation , we used Freesurfer software to generate volumetric data in subject groups for the hippocampal subiculum and its major striatal efferent target , the nucleus accumbens .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27207489">PMID:27207489</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Next , using the individual T1 image ( Q3 , voxel size : 0.7 mm isotropic ) , tissue classification of cortical gray matter , white matter , and CSF was performed using FreeSurfer [ Fischl and Dale , ] and the cortical mantle was parcellated in 114 cortical regions using the described DK‐57 atlas .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27200264">PMID:27200264</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.2 Network construction Gray matter and white matter tissues were classified and the gray matter further parcellated into 86 anatomical regions of interest using the semi-automated FreeSurfer software ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199720">PMID:27199720</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Recording sites assigned to posteromedial and anterolateral ROIs as described in Section “Materials and Methods” are shown in orange and purple , respectively . ( B ) Locations of all posteromedial and anterolateral HG sites ( orange and purple symbols , respectively ) plotted in MNI coordinate space and projected onto the FreeSurfer average template brain .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199712">PMID:27199712</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Diffusion-weighted images were processed with TRACULA ( Yendiki et al . , ) , which applies the ball-and-stick model ( FSL’s bedpostx ) for probabilistic tractography of known white matter pathways using anatomically constrained priors from FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199710">PMID:27199710</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortex was segmented from the brain using FreeSurfer software ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199650">PMID:27199650</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Finally , we extracted rs-fMRI signals from the gray matter by using a binary gray matter mask obtained from the tissue segmentation in FreeSurfer ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27199645">PMID:27199645</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each participant a representation of their cerebral cortex was constructed using FreeSurfer ( Freesurfer 5.3 , Martinos Center for Biomedical Imaging , Boston , Massachusetts ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27178195">PMID:27178195</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image Preprocessing All T 1 -weighted images were preprocessed using the FreeSurfer software , version 5.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27171035">PMID:27171035</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As negative controls for regional segmentations , recon-all generated gross brain volume measurements for larger-scale regions ( i.e. , total number of voxels per region ) : total grey matter , subcortical grey matter , brain mask volume ( BrainMaskVol , the residual volume following skull stripping ) , and estimated total intracranial volume ( eTIV , the Freesurfer approximation of total intracranial volume , extrapolated from the transform of each brain from native to standard space ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27170866">PMID:27170866</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer Moreover , the Bland-Altman analysis in demonstrates an overestimation bias for OLM-ACM_Joint method , while the Multi-atlas_Joint method presents ( similarly with the OASIS results ) a larger underestimation bias .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27160707">PMID:27160707</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The qT1 images were mapped onto cortical surfaces generated by FreeSurfer [ Fischl , ] at various depths ( i.e. , 25 , 50 , and 75 % of cortical thicknesses , measured from the interface between gray and white matter ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27158586">PMID:27158586</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To investigate the effect of PE direction on FA-values , mean FA values were extracted from parallel processed ( corrected ) AP and PA images for 70 cortical white matter regions and 27 subcortical regions as derived with FreeSurfer ( , ) ( see ) . 2.3.1 Illustration of correction for susceptibility artifacts presents an overview of the procedure to correct for susceptibility-induced artifacts .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27148045">PMID:27148045</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The atlas used in FreeSurfer included 34 cortical ROIs per hemisphere ( Table ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27148002">PMID:27148002</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>White matter segmentation in this T1 scan was conducted using FreeSurfer ( Martinos Center for Biomedical Imaging , Massachusetts General Hospital ) and gray matter was generated with the mrGray function in the mrVista software package ( Stanford Vision and Imaging Science and Technology ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27147956">PMID:27147956</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In fact the 1-fiber tracts only trace the medial portion of the lateral occipital cortex , whereas the 2-fiber NODDI tracts cover most of the lateral-occipital cortex as labeled by Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27144219">PMID:27144219</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the 2 most established and most commonly applied tools to analyze subcortical volumes to account for possible methodologic differences between previous studies , i.e. , FSL FIRST ( v5.0.0 ) and Freesurfer ( v5.1 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27118379">PMID:27118379</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>15 , e245 , 10.2196/jmir.2930 ( 2013 ) .24220210 Fischl B . FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27104135">PMID:27104135</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To reduce bias from a specific time point , a template volume using all time points from each subject was generated and run through FreeSurfer and was later used as an initial estimate for surface reconstruction and cortical segmentation for each time-point in each subject ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27102654">PMID:27102654</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural Connectivity ( SC ) The cortex was parcellated into 114 distinct regions using a subdivision of FreeSurfer 's Desikan–Killiany atlas ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27095660">PMID:27095660</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The geometry of the cortical mantle was extracted from the MRI data of each individual subject using the Freesurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27094093">PMID:27094093</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Detailed descriptions of FreeSurfer procedures are in Supplementary Methods 2 . Cluster Analysis MATLAB ( release 2014b , The MathWorks , Inc . , Natick , Massachusetts ) was used to perform an agglomerative hierarchical cluster analysis using whole‐brain cortical thickness vertex information for each of the 88 PD patients .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27092179">PMID:27092179</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Our sources consisted of current dipoles placed at the vertices of a tessellated MNI-Colin 27 cortical surface , which was segmented and tessellated using FreeSurfer [ ] and downsampled to 15028 vertices .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27089992">PMID:27089992</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI processing Cortical reconstructions were generated using FreeSurfer , which was specifically developed for this purpose .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27078862">PMID:27078862</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Briefly , the two T1-weighted images obtained per subject were processed with FreeSurfer to obtain a grey matter ( GM ) parcellation into 68 cortical regions [ ] . The first five volumes of the rs-fMRI were removed to avoid T1 effects and images were preprocessed with FSL [ ] . fMRI time-series were averaged within each GM region and the precision matrix was used to estimate fMRI functional connectomes [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27065101">PMID:27065101</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Recon-all is a fully automated workflow that performs all the FreeSurfer cortical reconstruction and sub-cortical segmentation steps in a unified pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27064767">PMID:27064767</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing included non-local means denoising using DIPY ( ) , computing subject-specific gray matter masks and CT measures using FreeSurfer ( ) as well as registration to the ICBM 2009c nonlinear asymmetric template in MNI space ( ) using ANTs ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27064239">PMID:27064239</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition , cortical reconstruction and volumetric segmentation were performed with FreeSurfer software version 5.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27061771">PMID:27061771</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This was performed by projecting each participant 's complex‐valued phase‐encoded map to the FreeSurfer spherical atlas ( using FreeSurfer mri_surf2surf ) , only performing ten steps of surface‐based smoothing ( ∼3mm FWHM in 2D ) before vector averaging across subjects at each vertex in the common surface coordinate system .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27051589">PMID:27051589</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.6 Cortical thickness estimation steps Cortical thickness measurements for 74 MCI patients were made using a fully automated magnetic resonance imaging-based analysis technique : FreeSurfer , a set of software tools for the study of cortical and subcortical anatomy .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27049765">PMID:27049765</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer Analyses MRI scans were reconstructed to measure cortical thickness using FreeSurfer software version 5.3.0 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27047366">PMID:27047366</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These dipole sources were distributed at each node ( i.e. , vertex ) of a tessellated cortical mesh template surface ( brain model ) derived from the FreeSurfer brain template ( FSAverage ) provided in Brainstorm ( see ) . Dipole orientations were constrained to be normal to the surface of the cortex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27039086">PMID:27039086</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The entire imaging session procedure is summarized in Fig .   . Fig . 2 Functional magnetic resonance imaging experiment paradigm ( REST , meal-loaded steady-state resting run ; CONT , continuous MA steady-state run ; TASKT1/2 , block design MA stimulus run ; T1 , structural image acquisition ; ACUP , acupuncture treatment ; MA , modified pressure algometer ) fMRI data processing Functional MRI data will be processed using the FSL ( FMRIB Software Library ; http://fsl.fmrib.ox.ac.uk/fsl ) , AFNI ( Analysis of Functional NeuroImages ; https://afni.nimh.nih.gov/afni ) , and FreeSurfer ( http://freesurfer.net ) software packages .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27034889">PMID:27034889</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The registered T1-weighted image was segmented into white matter , grey matter , and cerebrospinal fluid using FreeSurfer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27026657">PMID:27026657</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These ROIs were defined in each participant 's native space using FreeSurfer and the average parameter estimate from the contrast of interest in the functional data was extracted for each participant within the structurally defined region .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27014049">PMID:27014049</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A standard FreeSurfer pipeline for surface and volume based reconstruction is used for preprocessing the data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27012442">PMID:27012442</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Each person 's cortical surface was reconstructed from these structural images with FreeSurfer ( Dale et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27010959">PMID:27010959</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In summary , FreeSurfer v5.1 ( Martinos Center for Biomedical Imaging , Charlestown , Massachusetts , USA , https://surfer.nmr.mgh.harvard.edu/fswiki ) was used to automatically segment the brain using T1-weighted MR acquired in corresponding visit as the PET scan .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27009536">PMID:27009536</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical Reconstruction Cortical thickness and grey matter volume were calculated using Freesurfer ( version 5.3.0 ) –a surface based analysis tool that calculates the distance between the grey/white matter boundary and the pial surface .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27009409">PMID:27009409</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Using the scalp surfaces created by Freesurfer , we remodelled the volume contained within the entire head using MNE in the first five subjects , and inspected the FFR>baseline difference map overlaid on the T1 anatomical map .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27001615">PMID:27001615</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>( 1 ) Expert hippocampal manual tracing was performed using our previous published protocol ; , ( 2 ) Functional MRI of the brain ( FMRIB )  's Integrated Registration and Segmentation Tool ( FIRST v5.0.0 http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIRST ) in FMRIB 's Software Library ( FSL , v5.0 , http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ) was used to measure hippocampal volume and morphometry ; ( 3 ) the longitudinal pipeline in FreeSurfer ( v5.1.0 , http://freesurfer.net ) was applied to measure the longitudinal training effect on cortical thickness on a vertex basis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26981099">PMID:26981099</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The following preprocessing steps were performed using FreeSurfer ( Martinos Center for Biomedical Imaging located at Charlestown , Massachusetts , USA ) and FSL software ( Oxford Centre for Functional MRI of the Brain , Oxford , UK ) ( Fischl , 2012 ; Jenkinson et al . , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26980615">PMID:26980615</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Between-Session and Anatomical Alignment of MRI Data fMRI data were preprocessed using AFNI software ( ; http://afni.nimh.nih.gov ) and programs for defining and manipulating cortical surfaces using FreeSurfer ( ; http://freesurfer.net ) and PyMVPA ( ; http://www.pymvpa.org ) , including definition of cortical surface searchlights ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26979435">PMID:26979435</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>An automatic segmentation procedure ( Freesurfer ASEG ) ( Fischl et al .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26977400">PMID:26977400</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer output was visually inspected on a slice-by-slice basis and manually corrected when deemed necessary .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26977397">PMID:26977397</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The initial subcortical segmentations for the hippocampus , amygdala , basal ganglia and thalamus were obtained from FreeSurfer version 5.3.0 , followed by image registration with LDDMM that produced smooth transformations for each region of interest ( ROI ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26973588">PMID:26973588</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For ease of visualization , the significantly activated clusters were rendered on an average cortical surface ‘fsaverage’ provided in the FreeSurfer software ( ; ) . Region of Interest Localization We performed ROI localization using SPM8 in MATLAB ( The MathWorks , Natick , MA , USA ) and the MarsBaR MATLAB toolbox ( ) for the nine subjects who participated in the localizer experiments .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26967228">PMID:26967228</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Total computer time per day ( in minutes ) was averaged over a one-month period surrounding the participant’s MRI . MRI acquisition and analysis MRI data were obtained using a 3.0T Siemens MRI scanner and included : T1 -weighted magnetization prepared rapid gradient echo : repetition time = 2300 ms , echo time = 3.4 ms , inversion time = 1200 ms , spatial resolution = 1 mm isotropic and fluid attenuated inversion recovery : TR = 9000 ms , TE = 87 ms , TI = 2500 ms , Axial , FOV = 248 mm , slice thickness = 2 mm , number of slices = 95 . T1 -weighted images were segmented into white matter , gray matter ( GM ) , hippocampal , and ventricular cerebrospinal fluid ( vCSF ) masks using FreeSurfer ( v.5.1 ) , and manually edited for tissue misclassification .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26958465">PMID:26958465</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Because it includes both a 3D and 4D implementation , a natural comparison that we focus on is the combination of temporal filtering and 3D Freesurfer against 4D Freesurfer ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26941710">PMID:26941710</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A supplementary series of simple correlations was conducted between trimester-specific maternal TSH values and individually extracted CT values at the peak vertex for each of the significant clusters listed in Table ; values were derived from the FreeSurfer analysis conducted with all HYPO participants included .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26937385">PMID:26937385</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Maxima locations were derived from the Talairach Daemon ( , ) . 2.4.2 White matter VBM White matter VBM was implemented on the preprocessed images as described above ( following Matlab denoising and FreeSurfer brain extraction ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26936941">PMID:26936941</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated volume measures were performed with FreeSurfer version 5.1 ( http://surfer.nmr.mgh.harvard.edu/fswiki ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26936939">PMID:26936939</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To further improve the segmentations , each baseline T1 -weighted image was processed using the recon-all pipeline in Freesurfer ( version 5.0 ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26930054">PMID:26930054</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>CGM and DGM were parcellated in 70 cortical and 12 subcortical regions using Freesurfer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26928125">PMID:26928125</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Total brain volume ( excluding brainstem , cerebellum and cerebrospinal fluid ) and the volume of the frontal , temporal , parietal and occipital lobes were calculated with Freesurfer v5.1 , averaging left and right hemispheres .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26928024">PMID:26928024</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal volume ( and change in volume over time ) will be measured using FreeSurfer ( FreeSurfer V.1.0 ( program ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26922372">PMID:26922372</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>From an initial recruitment of 137 participants , a total of 15 participants ( MT = 8 , non-MT = 7 ) were excluded from the analysis due to concerns over image quality , resulting in a final sample of 122 ( Table  ) . Cortical reconstruction was performed with the FreeSurfer image analysis suite ( Dale et al .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26919234">PMID:26919234</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Whole-brain analyses Whole-brain CT measures were obtained from FreeSurfer routines calculating the closest distance from the gray/white boundary to the gray/CSF boundary at each vertex on the tessellated surface [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26911640">PMID:26911640</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All participants were scanned on 3T MRI scanners and a 3D volumetric T1- magnetization-prepared rapid gradient echo ( MPRAGE ) was acquired and preprocessed as described previously . Freesurfer software ( version 5.3 ) with a longitudinal processing pipeline was used to obtain total hippocampal volumes on serial MRI scans .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26904967">PMID:26904967</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Analysis of Offline Data Cortical current estimation by VBMEG A reconstruction of the cortical surface was constructed based on MR structural images using FreeSurfer image analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26903272">PMID:26903272</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Diffusion Imaging ( DI ) Methodology DI data were collected in the same scanning session as fMRI data at the above three sites , and were processed using ExploreDTI , Freesurfer and Tracts Constrained by Underlying Anatomy ( Tracula ) software. ( ) White matter tracts were automatically reconstructed using probabilistic tractography accounting for anatomy ( see ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26900568">PMID:26900568</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1-weighted image was also used to calculate hippocampal volumes using FreeSurfer for later use as an established measure of disease severity ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26900567">PMID:26900567</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Whole brain T1-weighted magnetization-prepared rapid gradient-echo ( MP RAGE ) structural scans were acquired with a FOV of 256 × 256 mm , TE of 3.48 ms , flip angle of 90° , and TR of 2530 ms , resulting in 1 mm isotropic voxels . 2.7 fMRI preprocessing and analyses Preprocessing and data analyses were performed using the following software packages : Nipype ( ) and standard preprocessing pipelines from BIPs , Nipy ( ) , FSL v5.0 ( ) , Analysis of Functional NeuroImages ( AFNI ) ( ) , FreeSurfer ( ) , Advanced Normalization Tools ( ANTS ) ( ) , and artifact detection toolbox ( ART — as implemented in Nipype ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26891986">PMID:26891986</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image Preprocessing Image processing steps were performed using the Connectome Computation System ( CCS , ; https://github.com/zuoxinian/CCS ) , which integrates the functionalities of AFNI ( ) , FSL ( ) , and FreeSurfer ( ) with shell and MATLAB scripts .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26888576">PMID:26888576</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A priori cortical regions For surface-based cortical metrics , cortical maps were generated in Freesurfer by identifying the gray/white matter border and pial surfaces in each individual and then applying a triangular tessellation to the cortical surface .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26888070">PMID:26888070</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject , the brain surface was reconstructed from the MRI and then assigned to one of 75 regions by Freesurfer . Depth electrodes were assigned to either a subcortical structure or to gyri/sulci .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26885824">PMID:26885824</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>TIV was computed using the Freesurfer pipeline [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26880358">PMID:26880358</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The NAcc mask was extracted from a FreeSurfer 5.3 ( http://freesurfer.net/ ) segmentation map for the MNI template brain .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26876151">PMID:26876151</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>First , the entire hippocampal formation was segmented using the routine volumetric FreeSurfer pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26872350">PMID:26872350</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer LGI computation creates a triangulated mesh of the outer hull of the brain , calculates the LGI ratio between the pial surface and outer hull at the center of a 25 mm spherical region of interest , and propagates these values onto the pial surface , assigning LGI values to each vertex on the mesh surface [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26870660">PMID:26870660</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was applied to registered T1-weighted volumes to remove non-brain tissue and segment remaining tissue into gray and white matter ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26870632">PMID:26870632</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Plasma luminex . 0.801 . 0.705 . 0.744 . Freesurfer . 0.664 . 0.632 . 0.644 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26866917">PMID:26866917</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The fMRI data were analyzed with SPM8 Matlab toolbox [ ] and Freesurfer [ ] software packages .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26859312">PMID:26859312</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These steps were primarily completed using FreeSurfer and included ( 1 ) spatial noises removal with a non‐local means filter [ Zuo and Xing , ] ; ( 2 ) brain extraction or skull stripping ; ( 3 ) visual inspection and manual intervention of brain tissues by removing the residual non‐brain tissues or patching the excluded brain tissues ; ( 4 ) tissue segmentation and surface reconstruction by identifying the boundary between white and gray matter ; ( 5 ) subcortical segmentation by labeling each voxel to one of 40 subcortical regions based on an existing atlas containing probabilistic location information ; ( 6 ) b0 image realignment by a 6‐parameter rigid transformation to the mean rfMRI image and boundary‐based registration ( BBR ) between individual structural image and the realigned b0 image [ Greve and Fischl , ] ; and ( 7 ) the generation of regions of interests ( ROIs ) for individual left and right amygdala areas in functional space by transferring structurally segmented amygdala areas to their ROIs in individual functional spaces with BBR .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26845558">PMID:26845558</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Processing methods for HCP data has been described elsewhere [ , ] . Selection and evaluation of white-matter connectomes Candidate connectome generation The total white-matter volume was initially identified from the tissue type segmentation using FreeSurfer [ ] , edited manually ( [ ] http://www.itksnap.org/pmwiki/pmwiki.php ) , and finally resampled at the resolution of the diffusion data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26843604">PMID:26843604</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>If needed to better determine the type of area in which an event occurred , the coordinates of these points were converted to Talairach coordinates using the FreeSurfer image analysis suite ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26839485">PMID:26839485</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Briefly , the subject’s first florbetapir image was coregistered to their MR image and segmented into cortical regions ( frontal , anterior/posterior cingulate , lateral parietal , and lateral temporal ) using FreeSurfer ( version 4.5.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26834544">PMID:26834544</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Analysis of axonal activation Cortical and subcortical gray matter structures were segmented from the reference brain structural MRI ( Table ) using Freesurfer ( Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26824348">PMID:26824348</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing Cortical reconstruction was performed using the longitudinal stream of FreeSurfer version 5.3 [ ] , which creates a within-unbiased subject template space and average image from both time points using robust , inverse consistent registration [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26812647">PMID:26812647</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://freesurfer.net , Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26792267">PMID:26792267</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For quantitative evaluation of the resting state networks ( RSNs ) we use the parcellation of the cerebral cortex into seven networks ( default mode , control , dorsal attention , ventral attention , visual , limbic and somato-motor networks ) identified from 1,000 subjects' intrinsic functional connectivity data , available on http://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation_Yeo2011 . Workflow From T1-weighted MRI data of each subject ( resolution 0.7 mm ) , we reconstruct the cortical surfaces separating the white and grey matter , referred to as the white matter surface in the rest of the manuscript , as well as grey matter and the cerebrospinal fluid , referred to as the grey matter surface in the rest of the manuscript , separately for each hemisphere using the Freesurfer Software http://freesurfer.net .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26784744">PMID:26784744</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Final volumes were also segmented with FSL/FIRST [ ] subcortical segmentation tool for comparison , with intraclass coefficient values between FreeSurfer and FIRST values ≥ 0.85 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26767944">PMID:26767944</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each of the 148 regions automatically parcelated by FreeSurfer , we contrasted the mean cortical thickness of the blind and sighted groups using unpaired T-tests , and controlling for multiple comparisons using FDR correction . 2.4 Partial least squares analysis for group-linked CT covariance To evaluate whether there is a network-level covariance pattern that discriminates the two groups , we used a partial least squares regression ( PLSR ) approach ( , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26754953">PMID:26754953</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumes of gray matter , white matter and CSF segments in this volume of interest were then generated using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26749003">PMID:26749003</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The total cortical area generated using the FreeSurfer model was consistent with the surface area derived from postmortem studies and has been validated on several brain phantoms and compared with other surface-based analysis packages ( Lee et al . ; Makris et al . ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26745873">PMID:26745873</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Vertex-wise FreeSurfer analysis was used to create a study-specific template , and vertex-wise comparisons were performed after smoothing using a 10-mm FWHM Gaussian kernel , as frequently applied in literature .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26744699">PMID:26744699</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The individual scalp and cortical surfaces of the patient were extracted from the MRI volume data using Freesurfer , with default parameter settings .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26742566">PMID:26742566</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cereb Cortex . 25 ( 9 ) : 2793 –2805 .24794920 Fischl B 2012 FreeSurfer . NeuroImage . 62 ( 2 ) : 774 –781 .22248573 Fischl B , van der Kouwe A , Destrieux C , Halgren E , Segonne F , Salat DH , Busa E , Seidman LJ , Goldstein J , Kennedy D et al 2004 Automatically parcellating the human cerebral cortex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26740912">PMID:26740912</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Further , in previous manuscripts ( ) we also validated hippocampal segmentations against commonly used methods in the field , including : FreeSurfer ( ) , FSL FIRST ( ) , MAPER ( ; ) , and SNT ( Medtronic Surgical Navigation Technologies , Louisville , CO , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26733601">PMID:26733601</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Furthermore , vertex-wise CT was compared pairwise between the three groups using mri_glmfit , part of the FreeSurfer toolkit .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26715588">PMID:26715588</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness estimation steps Cortical thickness measurements for 74 MCI patients were made using a fully automated magnetic resonance imaging-based analysis technique : FreeSurfer v5.1.0 , a set of software tools for the study of cortical and subcortical anatomy .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26714192">PMID:26714192</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To account for heterogeneous changes in regional volume , the amount of regional volume change required to warp a subject into the standard surface , fsaverage , is measured using the vertex-wise covariate derived from the Jacobian determinant of the spherical transform in FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26713863">PMID:26713863</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Computations and Visualizations AFNI [ ] and Freesurfer [ ] were used to process MRI images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26696852">PMID:26696852</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Figure 1 Graphical representation of the Freesurfer hippocampal region of interest ( ROI ) overlaid on a T1w/T2w intensity map .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26691722">PMID:26691722</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The phase information at the frequency of stimulation ( 0.033Hz ) was extracted , using its amplitude as threshold ( ≥2 SNR , calculated as in the analysis of SWIFT data explained above ) and overlaid them on each participant 's cortical surface reconstruction obtained using Freesurfer [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26689815">PMID:26689815</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All these proceeding steps used scripts provided by the 1000 Functional Connectomes Project ( http://www.nitrc.org/projects/fcon_1000 ) with both FSL and AFNI software ( http://www.afni.nim.nih.gov/ ) . Definition of MTG seed masksThe bilateral MTG masks were defined using Freesurfer ( http://surfer.nmr.mgh.harvard.edu/fswiki ) and transformed from surface space to volume space .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26677082">PMID:26677082</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer ( http://surfer.nmr/mgh.harvard.edu/fswiki ) to accomplish this computationally .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26657566">PMID:26657566</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Firstly FreeSurfer [ ] was used to obtain surface meshes of gray and white matter boundries from the MRI data , and to parcellate the brain into regions of interest ( ROI ) based on the Desikan atlas [ , ] . There were 82 ROI included , spanning cortical and subcortical regions ( subcortical regions included the Nucleus accumbens , Amygdala , Caudate , Hippocampus , Pallidum , Putamen and the Thalamus ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26650926">PMID:26650926</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Regional and whole brain volumes were determined using FreeSurfer Suite 5.0 ( Martinos Center for Biomedical Imaging , Massachusetts General Hospital , Harvard Medical School , Cambridge , MA , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26648521">PMID:26648521</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>First , following volumetric registration , anatomical surfaces were generated from the subject 's MP-RAGE image using FreeSurfer 's default recon-all processing pipeline ( version 5.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26643541">PMID:26643541</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI data were processed using the standard longitudinal processing stream in FreeSurfer 5.3 ( Massachusetts General Hospital , Boston , MA , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26640768">PMID:26640768</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted MP-RAGE sequences ( 240 × 256 matrix ; FOV = 256 mm ; 160 slices ; voxel size = 1.0 × 1.0 × 1.0 mm3 ; TI = 900 ms ; TR = 2300 ms ; TE = 2.98 ms ; flip angle = 9° ) . 2.3 Morphological descriptors Previously validated FreeSurfer ( ) workflows , including non-brain tissue removal , intensity normalization and automated volumetric parcellation based on probabilistic information from manually labeled training sets , were used to segment the bilateral thalamus , putamen , pallidum , amygdala , accumbens , caudate and hippocampus from the raw MRIs .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26629847">PMID:26629847</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>fMRI Preprocessing Preprocessing of functional data was performed using AFNI [ ] and Freesurfer [ ] , and resting-state and task-state data were preprocessed identically in order to facilitate their comparison .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26627483">PMID:26627483</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition , we used a 32 k surface created using FreeSurfer on the T1-weighted image ( acquired using an MPRAGE sequence at 0.7 mm isotropic resolution ) and aligned to the diffusion space as part of the HCP’s minimum preprocessing pipeline ( Glasser et al . ) to display the data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26619203">PMID:26619203</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric analysisAutomatic segmentation of the hippocampus , striatum ( caudate and putamen ) , and vmPFC was performed using Freesurfer ( v 5.3 ; details about the subcortical segmentation process have been described in [ ] ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26613098">PMID:26613098</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Striatal target structures , consisting of the left and right putamen and caudate , were segmented from the MPRAGE using the FreeSurfer suite of automated tools ( see ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26604762">PMID:26604762</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Report of the MCI Working Group of the European Consortium on Alzheimer’s diseaseJ Neurol Neurosurg Psychiatry2006777147181654941246LezakMHowiesonDLoringDWNeuropsychological Assessment4th edOxfordUniversity Press200447RadloffLSThe CES-D scale : a self-report depression scale for research in the general populationAppl Psychol Meas1977138540148MorettiDVMild Cognitive Impairment : Structural , Metabolical , and Neurophysiological Evidence of a Novel EEG BiomarkerFront Neurol201576615210.3389/fneur.2015.001522621729949KlimeschWEEG-alpha rhythms and memory processesInt J Psychophysiol199726319340920301250KlimeschWEEG alpha and theta oscillations reflect cognitive and memory performance : a review and analysisBrain Res Rev1999291691951020923151KaplanAYThe problem of segmental description of human electroencephalogramHuman Physiol19992510711452CohenBASancesAStationarity of the human electroencephalogramMed Biol Eng Comput19771551351819980553KawabataNTest of statistical stability of the electroencephalogramBiol Cybern19762223523895308154McEwenJAAndersonGBModeling the stationarity and gaussianity of spontaneous electroencephalographic activityIEEE Trans Biomed Eng197522361369119362255KipińskiLKönigRSielużyckiCKordeckiWApplication of modern tests for stationarity to single-trial MEG data : transferring powerful statistical tools from econometrics to neuroscienceBiol Cybern20111053–41831952209517356MorettiDVFracassiCPievaniMIncrease of theta/gamma ratio is associated with memory impairmentClin Neurophysiol200912022953031912160257MorettiDVPievaniMFracassiCIncrease of theta/gamma and alpha3/alpha2 ratio is associated with amygdalo-hippocampal complex atrophyJ Alzheimers Dis20091723493571936326358MorettiDVPievaniMGeroldiCIncreasing of hippocampal atrophy and cerebrovascular damage is differently associated with functional cortical coupling in MCI patientsAlzheimer Dis Assoc Disord20092343233321956144259CabezaRHemispheric asymmetry reduction in older adults : the HAROLD modelPsychol Aging200217185100Review1193129060BalstersJHO’ConnellRGGalliAChanges in resting connectivity with age : a simultaneous electroencephalogram and functional magnetic resonance imaging investigationNeurobiol Aging2013349219422072360811361WatsonPConroyAMoranGDuncanSRetrospective study of sensitivity and specificity of EEG in the elderly compared with younger age groupsEpilepsy Behav20122534084112311097162TenkeCEKayserJMillerLNeuronal generators of posterior EEG alpha reflect individual differences in prioritizing personal spiritualityBiol Psychol20139424264322399899663GrandyTHWerkle-BergnerMChicherioCSchmiedekFLövdénMLindenbergerUPeak individual alpha frequency qualifies as a stable neurophysiological trait marker in healthy younger and older adultsPsychophysiology20135065705822355108264GrandyTHWerkle-BergnerMChicherioCLövdénMSchmiedekFLindenbergerUIndividual alpha peak frequency is related to latent factors of general cognitive abilitiesNeuroimage20137910182362449065BekhterevaVSanderCForschackNOlbrichSHegerlUMüllerMMEffects of EEG-vigilance regulation patterns on early perceptual processes in human visual cortexClin Neurophysiol20141251981072387117866SégonneFDaleAMBusaEA hybrid approach to the skull stripping problem in MRINeuroimage2004223106010751521957867FischlBDaleAMMeasuring the thickness of the human cerebral cortex using magnetic resonance imagesProc Natl Acad Sci U S A20009711044110491099546068HanXJovicichJSalatDReliability of MRI-derived measurements of human cerebral cortical thickness : the effects of field strength , scanner upgrade and manufacturerNeuroimage20063211801941665100869GronenschildEHHabetsPJacobsHIThe effects of FreeSurfer version , workstation type , and Macintosh operating system version on anatomical volume and cortical thickness measurementsPLoS One201276e382342267552770DeCarliCFletcherERameyVHarveyDJagustWJAnatomical mapping of white matter hyperintensities ( WMH ) : exploring the relationships between periventricular WMH , deep WMH , and total WMH burdenStroke20053650551557665271PennanenCTestaCLaaskoMPA voxel based morphometry study on mild cognitive impairmentJ Neurol Neurosurg Psychiatry20057611141560798872MarkesberyWRSchmittRAKryscioRJDavisDSmithCWeksteinDNeuropathologic substrate of mild cognitive impairmentArch Neurol20066338461640173573McKhannGMKnopmanDSChertkowHThe diagnosis of dementia due to Alzheimer’s disease : recommendations from the national institute on aging-Alzheimer’s association workgroups on diagnostic guidelines for Alzheimer’s diseaseAlzheimers Dement201172632692151425074SperlingRAAisenPSBeckettLAToward defining the preclinical stages of Alzheimer’s disease : recommendations from the National Institute on Aging-Alzheimer’s Association workgroups on diagnostic guidelines for Alzheimer’s diseaseAlzheimers Dement2011732802922151424875CaroliATestaCGeroldiCBrain perfusion correlates of medial temporal lobe atrophy and white matter hyperintensities in mild cognitive impairmentJ Neurol20072548100010081737526076MatsudaHThe role of neuroimaging in mild cognitive impairmentNeuropathology2007275705771802137977PetrellaJRWangLKrishnanSCortical deactivation in mild cognitive impairment : high-field-strength functional MR imagingRadiology20072452242351788519078PihlajamakiMJauhiainenAMSoininenHStructural and functional MRI in mild cognitive impairmentCurr Alzheimer Res200961791851935585379DickersonBCSperlingRALarge-scale functional brain network abnormalities in Alzheimer’s disease : insights from functional neuroimagingBehav Neurol20092163751984704680RyuSYKwonMJLeeSBMeasurement of precuneal and hippocampal volumes using magnetic resonance volumetry in Alzheimer’s diseaseJ Clin Neurol201061962032126420081SperlingRADickersonBCPihlajamakiMFunctional alterations in memory networks in early Alzheimer’s diseaseNeuromolecular Med20101227432006939282de HaanWMottKvan StraatenECScheltensPStamCJActivity dependent degeneration explains hub vulnerability in Alzheimer’s diseasePLoS Comput Biol201288e10025822291599683PievaniMde HaanWWuTSeeleyWWFrisoniGBFunctional network disruption in the degenerative dementiasLancet Neurol2011108298432177811684ChatwalJPSperlingRAFunctional MRI of mnemonic networks across the spectrum of normal aging , mild cognitive impairment , and Alzheimer’s diseaseJ Alzheimers Dis2012310S155S1672289009885JonesDTMachuldaMMVemuriPAge-related changes in the default mode network are more advanced in Alzheimer diseaseNeurology20117716152415312197520286BrierMRThomasJBSnyderAZLoss of intranetwork and internetwork resting state functional connections with Alzheimer’s disease progressionJ Neurosci20123226889088992274549087PalopJJMuckeLSynaptic depression and aberrant excitatory network activity in Alzheimer’s disease : two faces of the same coin ?</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26597325">PMID:26597325</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical surface was parcellated using Freesurfer ( version 5.3 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26595654">PMID:26595654</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the superior IPS localizer , each volume contained 24 slices ( 5mm thick , 3.75 x 3.75mm in plane , no skip ) parallel to the AC-PC line ( TR = 1.5s , TE = 30ms , flip angle = 90º ) . fMRI data were analyzed using the Freesurfer software package – .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26594620">PMID:26594620</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This analysis method provides whole brain WM voxel-based MWF quantification . 2.4 Patient MWF registration and lesion identification Subject 's T1W images where automatically processed using FreeSurfer 's longitudinal stream enabling us to register all sequences and all time points together for each patient ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26594378">PMID:26594378</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Similar processing can also be achieved using tools provided by other freely available packages , such as FreeSurfer ( http://freesurfer.net ) or FSL ( http://fsl.fmrib.ox.ac.uk/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26594185">PMID:26594185</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used for reconstruction of cortical surfaces and coregistration ( version 5.3.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26594164">PMID:26594164</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface flattening and visualization were performed with Freesurfer and custom python code ( Gao et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26587289">PMID:26587289</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Moreover , the GM volume in each hemisphere and total intracranial volume ( TIV ) was also calculated from the FreeSurfer processing stream .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26582986">PMID:26582986</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We reconstructed the cortical surface using FreeSurfer image analysis ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26579071">PMID:26579071</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The brain was segmented into 83 cortical and subcortical regions using the Lausanne 2008 ( Freesurfer ) parcelation scheme within the Connectome Mapper Toolkit ( , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26578952">PMID:26578952</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>WM mask was created by extracting the cerebral WM , cerebellar WM , ventral diencephalon , and brainstem from the individual’s FreeSurfer cortical parcelation and subcortical segmentation ( obtained by recon-all command line ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26572733">PMID:26572733</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical-level visualization was performed using FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26569619">PMID:26569619</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1 image of each participant was segmented into 87 anatomical regions of interest ( ROIs ) of the Desikan atlas [ ] using Freesurfer [ ] . FSL’s probtrackx was used to perform tractography seeded from each of the 87 ROIs and going to the others [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26551545">PMID:26551545</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Visualization and Statistics The iterative parcellation was performed on the FreeSurfer fsaverage4 template and the resulting network labels were upsampled to each individual subject’s own cortical surface using the mri_surf2surf function .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26539100">PMID:26539100</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As detailed below , analyses were performed using FreeSurfer 5.1.0 ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26536598">PMID:26536598</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>However , this was short-term reliability , and further studies are needed to examine long-term reliability for this method [ ] . Finally , although the local GM volume was used to construct individual brain networks in the present study , other morphological ( e.g. , cortical thickness and area from Freesurfer , tissue density , and cell type ) and non-morphological features ( e.g. , statistical parametric mapping in task fMRI ) may also be appropriate .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26536135">PMID:26536135</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image pre-processing : DTIThe HCP database provides pre-processed DTI and rs-fMRI data for many commonly performed pre-processing steps using FSL and FreeSurfer software [ , , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26528121">PMID:26528121</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and coregistration were conducted using Freesurfer ( version 5.3 , www.freesurfer.net ) . Functional data were motion-corrected , resampled to the standard cortical surface , and surface-smoothed ( 10 mm FWHM ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26503502">PMID:26503502</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These ROIs were defined in the individual 's T1 volume , using Freesurfer software ( version 5.1.0 ) and its cortical parcellation [ Desikan et al . , ] ( see supplementary Figure and Supporting Information Table I for a description of the cortical regions and abbreviations ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26500525">PMID:26500525</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Therefore , DLPFC volumes were taken from FreeSurfer 5.3 parcellations ( Fischl et al . , , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26499814">PMID:26499814</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>At each stage in the Freesurfer processing pipeline , the MP-RAGE T1 -weighted images were visually inspected and , if needed , manually edited and corrected .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26497266">PMID:26497266</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The analysis procedure and results are provided in Supplementary Material 2 . Region-of-interest analysis To obtain the ROI masks , the NIH 4.5- to 8.5-year-old T1 template was segmented and parcellated using the software package FreeSurfer ( available at https://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26466114">PMID:26466114</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The CoV was significantly tighter for FreeSurfer than for Manual ( t 11.48 = 3.18 , P = 0.007 ) and showed a trend when compared with Stereology ( t 10.68 = 2.09 , P = 0.061 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26464989">PMID:26464989</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A high-resolution MNI152 template ( ) was used to reconstruct and inflate the cortical sheet , separately for the left and right hemisphere , using the FreeSurfer Toolbox ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26461053">PMID:26461053</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Estimated total intracranial volume ( eTIV ) was measured using Freesurfer [ Buckner et al . , ] . Hippocampal Shape Analysis The hippocampal regions generated by manual segmentation were used to analyze differences in shape between the subject groups .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26457551">PMID:26457551</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Comparisons of identification accuracy between the Shen and FreeSurfer and Yeo schemes are shown in .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26448913">PMID:26448913</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Each T1-weighted was processed using the recon-all pipeline in Freesurfer ( version 5.0 , ( ; http://freesurfer.net ) ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26441551">PMID:26441551</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional connectivity and cortical response during the task Preprocessing for fMRI data analyses For both resting-state and task-related fMRI data analyses , surface-based analysis was conducted using FreeSurfer Functional Analysis Stream ( FS-FAST ) ( http://surfer.nmr.mgh.harvard.edu/fswiki/FsFast , http://surfer.nmr.mgh.harvard.edu/fswiki/FsFastFunctionalConnectivityWalkthrough ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26440276">PMID:26440276</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>ROI 's were generated by an automatic segmentation that was applied to all subjects anatomical image to delineate the cortical areas using Freesurfer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26439117">PMID:26439117</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All participants’ FreeSurfer data were manually inspected and edited by the second author ( EAO ) who previously had attended an official training workshop given by the FreeSurfer developers at Massachusetts General Hospital . 10.1371/journal.pone.0139807.g001 Fig 1 Dorsal anterior cingulate cortex ( dACC ) region of interest , obtained from the Desikan-Killiani FreeSurfer atlas .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26436901">PMID:26436901</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For illustrations of across-observer contrasts ( , ) each observer 's per-scan FEAT results were entered into an across-scan fixed effects analysis using FSL 's FEAT tool , and the resulting per-observer data were then converted to an average surface for a random-effects analysis across observers using Freesurfer 's mri_glmfit .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26435832">PMID:26435832</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Seven bilateral subcortical ROIs were automatically segmented by FreeSurfer ( thalamus , caudate , putamen , pallidum , hippocampus , amygdala , and accumbens ) , as well as 34 cortical ROIs per hemisphere based on a standard atlas [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26428666">PMID:26428666</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We summed right and left hippocampal volumes from Freesurfer ( v 5.3 ) , and adjusted them for total intracranial volume ( TIV ) by calculating the residual from a linear regression of hippocampal volume ( y ) versus intracranial volume ( x ) among 133 cognitively normal subjects aged 30 to 59 ( described in ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26423806">PMID:26423806</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26423222">PMID:26423222</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As with the DSI data , anatomical scans were segmented using FreeSurfer and parcellated according to the Lausanne 2008 atlas included in the connectome mapping toolkit .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26422790">PMID:26422790</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Morphometric analyses of the corpus callosum Volumetric whole brain segmentation was performed using the FreeSurfer image analysis suite ( stable Linux version 5.3.0 released on 15th of May 2013 ) , which is available online at https://surfer.nmr.mgh.harvard.edu/ .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26414616">PMID:26414616</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>191 supplied measures from the T1-weighted structural brain analysis using FreeSurfer ( including volumes of subcortical structures , and average thickness and surface area of many cortical regions ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26413474">PMID:26413474</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer , was used to derive measurements of individual brain regions .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26413471">PMID:26413471</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The starting and target regions of interest were generated with the white matter parcellation provided by Freesurfer applied to the structural imaging ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26404239">PMID:26404239</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Due to the congruent results between FSL-FIRST and FreeSurfer , we decided to present only the obtained by FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26404042">PMID:26404042</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer . NeuroImage 62 , 774–781 ( 2012 ) .22248573JenkinsonM. , BannisterP . , BradyM. & SmithS .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26401514">PMID:26401514</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Subsequently , the Freesurfer processing pipeline was resumed , and after completion results were again visually inspected for accuracy of outlining of the WM and pial borders .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26401443">PMID:26401443</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Shimming ( green ) and REST slabs ( blue ) are also shown above. ( B ) , ( C ) Different VOIs for NAA ( red ) and mI ( yellow ) co-registered in FreeSurfer native space .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26388756">PMID:26388756</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray and white matter were segmented using Freesurfer , ITK Gray , 1.6.0.1 . , and mrGray ( VISTA ) and were manually edited to minimize segmentation errors .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26384928">PMID:26384928</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing Cortical and subcortical regions of interest were generated by segmenting a T1 -weighted image using Freesurfer ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26376179">PMID:26376179</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The parcellation scheme from Freesurfer ( FS ) software [ ] was used as a first approximation to outline the thalamus and visual cortex that were going to serve as the seed and target masks , respectively .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26368278">PMID:26368278</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For visualization , results were registered to a standard template in Freesurfer 5.3 ( fsaverage ) [ , ] . Contrasts of parameter estimates were extracted for each subject at both time points from the clusters identified from the whole-brain analyses .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26354982">PMID:26354982</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal volume was analyzed with FreeSurfer version 5.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26350516">PMID:26350516</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>When cortical areas were included in the VOI , both the cortical ribbon and subcortical white matter ( within 5 mm from the cortical ribbon ) were included in the definition of the VOI . Pre-surgical volumes of these two VOIs were calculated by summing the individual volumes extracted from the pre-surgical MR label volume ( Fig .   ) . Calculations of the post-surgical volumes of VOIs in the resected temporal lobe were performed with a different method because many assumptions of Freesurfer algorithms would be violated by resected MR images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26349802">PMID:26349802</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Intracranial volume ( ICV ) was computed with the Freesurfer 5.3 package [ ] for wave 1 and wave 2 images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26348613">PMID:26348613</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Analyses Data were analyzed in Freesurfer ( v5.1 , https://surfer.nmr.mgh.harvard.edu ) and Matlab ( R2011b , The Mathworks ) using custom code .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26347632">PMID:26347632</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Post-processing and image analysis SA expansion and CT mapping were conducted using methods based on the publicly available FreeSurfer software package ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26346238">PMID:26346238</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As depicted in panel a of , our results show subjects 18 and 20 ( repository ID : 0028214 and 0028216 ) in the chess group have suboptimal GM SNR , WM SNR and WM/GM contrasts , which are consistent with results from the FreeSurfer QA tools .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26339175">PMID:26339175</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Variations in regional cortical thickness and volume between DIP patients and the control group were estimated using a general linear model ( GLM ) at each vertex across the cortical surface , with group ( DIP , control ) as the dependent variable and age as a nuisance variable , using Qdec implemented in FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26330834">PMID:26330834</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Table 1Subject informationImage pre-processing and connectivity analysisImaging data were pre-processed using FSL ( Functional Magnetic Resonance Imaging of the Brain Software Library ) and Freesurfer software ( Fischl , 2012 ; Jenkinson et al . , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26325185">PMID:26325185</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Parcellations of Resting-State fMRI Data The parcellation procedure relied on a recently published structural segmentation procedure using the Desikan labeled mesh in freesurfer [ ] , [ ] . More specifically , the Lausance 2008 parcellation within the Connectome viewer toolkit ( http://www.cmtk.org ) was used to create the 5 embedded hierarchical cortical parcellations within Freesurfer [ , , , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26324104">PMID:26324104</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The standard Freesurfer v5.1 morphometric pipeline was computed , including reconstruction of the cortical mantle and spatial smoothing of 20mm FWHM .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26303198">PMID:26303198</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A T2-weighted structural image in the same prescription as the functional images ( that is , coplanar image ; TR=15,780 ms , TE=82 ms , 512 × 512 matrix , 0.4 × 0.4 mm in-plane resolution ) and a T1-weighted 3D magnetization-prepared rapid gradient echo ( MPRAGE ) volume ( 256 × 256 × 192 matrix , 1 mm isotropic voxels ) were also collected to facilitate image co-registration , intracranial volume estimation using Freesurfer , and spatial normalization to the MNI template brain.fMRI preprocessingData were preprocessed and analysed using FSL version 5.0 ( FMRIB 's Software Library , http://www.fmrib.ox.ac.uk/fsl ) and Advanced Normalization Tools ( ANTS ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26288753">PMID:26288753</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Nevertheless , a cross validation of results between different scanners was performed ( section ) .2.3Neuroimaging analyses2.3.1MRI — cortical thickness asymmetriesThe rationale for considering cortical thickness as a potential predictor of SOZ laterality followed the reports of several groups who found more widespread atrophy in LTLE than in RTLE ( ; ; ) . Two popular software were used to estimate cortical thickness in order to cross-validate potential findings and to investigate differences arising from estimation methods : Freesurfer v.5.3.0 ( http://freesurfer.net ; ) and Advanced Normalization Tools v.2.1.0 ( ANTs , http://stnava.github.io/ANTs ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26288751">PMID:26288751</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted IR-SPGR data were automatically segmented using Freesurfer ( https://surfer.nmr.mgh.harvard.edu/ ) in order to calculate white matter hypointensity volume ( WMHV ) and hippocampal volume .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26288143">PMID:26288143</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray matter thickness and volume were estimated using a well-validated method[ ] implemented in FreeSurfer 4.0[ ] ( available free to the public , http://surfer.nmr.mgh.harvard.edu/fswiki and http://ucla.in/xSQPqT ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26284520">PMID:26284520</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Full brain and skull coverage was required for all MR images according to previously published quality control criteria [ , ] . Image analysis was carried out using the Freesurfer image analysis pipeline ( version 5.1.0 ) to produce 34 regional cortical thickness and 23 subcortical volumetric measures as previously described [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26283974">PMID:26283974</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Group Comparison on Brain Measures FreeSurfer processing of the brain images generated volumes of cortical and subcortical ROIs for each individual subjects .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26267897">PMID:26267897</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Talairach transforms , skull stripping , gray–white tissue segmentation , and surface reconstructions were visually checked for errors ( and , as noted , four children were excluded from analysis due to inaccurate gray-white tissue segmentation and motion noise in the reconstructed structural image ) .10.1371/journal.pone.0134115.g001Fig 1Cortical thickness regions of interest via Freesurfer ( adapted from 43 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26236628">PMID:26236628</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>White matter hypointesities were quantified on MPRAGE images using Freesurfer software ( https://surfer.nmr.mgh.harvard.edu/ ) .2.6Volumetric assessmentVolumetric information was computed for all the 7 ROIs mentioned above and for WM and cGM of each lobe ( frontal , parietal , temporal and occipital ) on MPRAGE and MP2RAGE images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26235800">PMID:26235800</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Two high-resolution structural images were acquired in the beginning of the first measurement session using an MPRAGE sequence , from which the white and gray matter borders were segmented and reconstructed using Freesurfer software package ( , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26230626">PMID:26230626</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image segmentation All image segmentation was performed using the automatic brain segmentation software tool FreeSurfer version 4.0.4 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26230510">PMID:26230510</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The maps were then hand-edited to ensure that the grey matter of the posterior vermis was separated from the abutting hemispheres of lobule HIX . Surface reconstruction We reconstructed the outer grey-matter surface ( ) and the inner white-matter surface ( ) using the Freesurfer packages [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26229673">PMID:26229673</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain extraction was employed before segmentation using the FSL ( FreeSurfer Library ) BET ( Brain Extraction Tool ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26218835">PMID:26218835</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A 3D surface representation was reconstructed from the MEMPRAGE image using FreeSurfer , and was used to visualize the SNR data on the cortical surface .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26217299">PMID:26217299</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness estimation steps Cortical thickness measurements for 74 MCI patients were made using a fully automated MRI-based analysis technique : FreeSurfer v5.1.0 , a set of software tools for the study of the cortical and subcortical anatomy .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26208170">PMID:26208170</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Journal of the International Neuropsychological Society . 2010 ; 16 : 846 –855 . doi : 10.1017/S135561771000072X 20937164 31 Fischl B . FreeSurfer . NeuroImage . 2012 ; 62 : 774 –781 . doi : 10.1016/j.neuroimage.2012.01.021 22248573 32 Vogt BA , Absher JR , Bush G .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26196438">PMID:26196438</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer v5.0.0 computer code is available at https://surfer.nmr.mgh.harvard.edu/fswiki/ReadOnlyCVS .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26194183">PMID:26194183</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI image processing was performed on a Mac OSX 10.8 workstation using FreeSurfer v5.1.0 ( Desikan Atlas ) to calculate volumes for specific brain regions of interest .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26193290">PMID:26193290</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Regional cerebral blood flow images were then corrected for partial volume effects and co-aligned with FreeSurfer v5.1 segmented and parcellated volumes [ , ] to yield subject-specific perfusion averages ( given in institutional units ) for each FreeSurfer cortical and subcortical anatomical label .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26192187">PMID:26192187</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>ROIs including the dorsal striatum ( putamen , caudate ) , and ventral striatum ( nucleus accumbens ( NAc ) ) were selected a priori and identified using FreeSurfer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26190405">PMID:26190405</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We also briefly compared our EM algorithm to the openly available mass-univariate mixed-effects algorithms from Freesurfer ( https://surfer.nmr.mgh.harvard.edu/fswiki/LinearMixedEffectsModels ) ( ) ( not shown in results ) using synthetic longitudinal data ( from the section on ) with linear models and balanced data with Gaussian errors .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26189703">PMID:26189703</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI processing MRI scans were analysed with software developed at the University of California , San Diego , Multi-Modal Imaging Laboratory based on the freely available FreeSurfer software package ( http://freesurfer-software.org/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26180782">PMID:26180782</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This system was developed based on FCP scripts for providing a computational platform for multimodal brain connectome analysis by integrating AFNI [ , ] , FSL [ ] , and FreeSurfer [ ] and has been used in our previous studies [ , ] . All structural MRI images were processed with the CCS structural pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26163805">PMID:26163805</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>An initial 3-D functional analysis , contrasting all sound versus all silent blocks , was performed in FEAT to verify data quality and coregister functional volumes to anatomical images processed by Freesurfer 4.1 ( Martinos Center for Biomedical Imaging , MGH , Boston ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26161125">PMID:26161125</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical images were reconstructed using Freesurfer recon-all and then tilt correction was done for functional and anatomical images using 3drefit and fslorient .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26149490">PMID:26149490</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical thickness measurement and cortical parcellation Images were processed and cortical thickness was measured using the longitudinal stream in FreeSurfer version 5.2.0 ( http://surfer.nmr.mgh.harvard.edu/fswiki/recon-all/ ) , which has been validated in previous research [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26137443">PMID:26137443</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All MRI scans were acquired on a Siemens 3 T Tim Trio scanner at Massachusetts General Hospital . From the T1-weighted structural scan , ventricular size was estimated from a volumetric segmentation using Freesurfer ( v 5.1.0 , e.g. ) . Ventricle volume was normalized by intracranial volume following the methods described in , and used to control for the effects of subcortical atrophy in the association between functional connectivity and striatal DVR . Pre-processing of the fMRI data followed the procedures of prior studies in this field ( ; ) and included slice-timing correction , motion-correction , registration to standard MNI152 space , spatial smoothing , temporal filtering , and intensity normalization .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26132764">PMID:26132764</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Connectome Mapper Toolkit [ ] default partitioning was used ( computed by the FreeSurfer , and based on the Desikan-Killiany anatomical atlas ) into 83 , 129 , 234 , 463 and 1015 cortical and sub-cortical structures ( as the brainstem and deep-grey nuclei ) , referred to as “Regions of Interest” , ROIs , ( see in [ ] ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26131378">PMID:26131378</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The motion-corrected functional data were then projected onto the cortical surface by averaging between the white and pial boundaries ( identified with FreeSurfer ) using AFNI/SUMA .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26122586">PMID:26122586</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Moderator analyses with meta-regression We tested whether mean age of each sample ( ) , magnetic resonance field strength , FreeSurfer version used for image processing ( ) , percentage of patients acutely depressed , percentage of patients with a co-occurring anxiety disorder , percentage of patients taking antidepressants and the percentage of patients taking antipsychotics ( ) explained a significant proportion of the variance in effect sizes across sites in the meta-analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26115451">PMID:26115451</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Intracranial volumes ( ICV ) , total grey matter volume , and cortical white matter volume were obtained from each subject 's 3D T1 images using FreeSurfer 4.5.0 [ ] .2.4 Diffusion Image Processing2.4.1 DTI pre-processingFMRIB 's Linear Image Registration Tool ( FLIRT ; www.fmrib.ox.ac.uk/fsl/flirt ) was used to register all diffusion-weighted volumes to their corresponding b = 0 s/mm2 volume , and to correct for motion and eddy currents [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26110431">PMID:26110431</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The MPRAGE volumes were used to reconstruct cortical surfaces using FreeSurfer version 4.5.0 [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26110112">PMID:26110112</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Quantitative regional brain volumes were obtained using the parcellation method incorporated in Freesurfer 5.1 software ( http://freesurfer.net/fswiki ) using the Desikan atlas ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106620">PMID:26106620</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The MRI analyses from which information was acquired were the stroke summary analysis , reporting the number and location of strokes , and the white matter hyperintensity volume of the whole brain ; the University of Arizona Gene Alexander Laboratory statistic parametric mapping voxel based morphometry analysis , reporting the mean gray matter value from 90 regions of interest ( ROI ) ; the University of California at San Diego Anders Dale Laboratory derived volumes analysis , reporting the volumes of 15 ROI ; and the University of California at San Francisco FreeSurfer analysis , reporting the volume , surface area , and cortical thickness of 139 ROI [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106566">PMID:26106566</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The MPRAGE structural scan was segmented using FreeSurfer software ; white matter regressors were used to remove unwanted physiological components .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106548">PMID:26106548</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Binding potentials were calculated for multiple regions of interest ( ROIs ) derived from Freesurfer using a cerebellar reference for regions-of-interest .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106547">PMID:26106547</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The other was anatomical , using each ROI 's center MNI coordinate and a surface based atlas ( ) included in FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106546">PMID:26106546</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.3 MRI processing For each dataset , the T1 -weighted image was segmented into WM , gray matter ( GM ) , and ventricle masks using FreeSurfer ( v.5.1 ) , and manually corrected for any tissue misclassification .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106538">PMID:26106538</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Furthermore , we also segmented T1-weighted images using Freesurfer to obtain subcortical GM volumes , to conduct ancillary region of interest analyses .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106535">PMID:26106535</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>ROI-based comparison was computed through FreeSurfer version 5.3 tractography software TRACULA providing both measures of average weighted fractional anisotropy ( FA ) and mean diffusivity ( MD ) ( ) for the following tracts : the corpus callosum forceps minor ( fMinor ) , and bilateral : anterior thalamic radiation ( ATR ) , and uncinate fasciculus ( UNC ) ( see ) .2.8Data analysisAll analyses were conducted using SPSS .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106529">PMID:26106529</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>DTI acquisitions were processed using tools available through FreeSurfer and FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSL ) image analyses software ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106320">PMID:26106320</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>; Norman-Haignere et al . , ) . Regions of Interest Definition V1 eccentricity sectors were defined manually on the Freesurfer fsaverage brain using the Freesurfer fsaverage V1 label file as a boundary .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26106263">PMID:26106263</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Segmentation and regions-of-interest ( ROIs ) Segmentation accuracies of the packages FreeSurfer , SPM , and FSL have been previously compared , and for WM , a higher volumetric segmentation accuracy was found with the FSL package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26101520">PMID:26101520</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric Correction FreeSurfer pipeline ( version 5.1.0 ) was applied to the MRI scans to produce 115 cortical and subcortical volumetric variables .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26098331">PMID:26098331</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer ( surfer . nmr . mgh . harvard.edu/ ) was used to inflate the cortical surface of each participant’s T1-weighted structural image and to analyze the functional data from the retinotopy session .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26085965">PMID:26085965</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric analyses of lateral ventricles We performed volumetric segmentation of 3D T1-weighted brain MRI using FreeSurfer software ( version 5.3.0 , A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26074200">PMID:26074200</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The imaging protocol used for the structural magnetic resonance scans , as well as the steps necessary to construct the surface representation of the cortical mantle , have been described extensively in ( see also the references therein ) ; FreeSurfer ( Martinos Center for Biomedical Imaging , Massachusetts General Hospital , Boston , ma , usa ) was used to generate the surfaces and to obtain cortical thickness measurements ( , , ) ; image registration was performed using the Multimodal Surface Matching ( msm ) framework ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26061305">PMID:26061305</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Quantitative MRI data were obtained by automatic segmentation at the Department of Medical Informatics , Erasmus University Medical Center , The Netherlands . For each participant , the following MRI markers were computed : Cortical thickness was calculated using a model-based automated procedure ( FreeSurfer , v.5.1.0 ) on T1-weighted images ( TR = 7.2 ms , TE = 3.3 ms , matrix = 256 × 256 × 180 mm3 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26057524">PMID:26057524</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We segmented WM , GM and the cerebellum based on FreeSurfer [ , ] , and used FMRIB FSL [ ] to extract CSF , ventricles , skull and scalp .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26052748">PMID:26052748</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer 's cortical reconstruction algorithm ( ‘recon-all’ ) was used on a T1 weighted MNI template image to generate an ROI mask of the EC with a probability threshold of 90 % that was binarized and converted to NiFTI file format . Seed-based connectivity analysisFor the quantification of the model-based analyses of connectivity between the AT and PM networks and the manually segmented medial and lateral or anterior and posterior EC , singular-value decomposition ( SVD ) and subtraction of the mean was performed on the voxel-wise time-series within each ROI .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26048954">PMID:26048954</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional images were registered to anatomical images using a rigid-body transformation determined by Freesurfer 's bbregister ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26042016">PMID:26042016</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>) , FreeSurfer ( Fischl et al . , ) , or Caret ( Van Essen et al . , ) rely on the same generic steps , we may suppose that , similarly , differences would be observable in the majority of subjects .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26029159">PMID:26029159</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal volumes acquired for patients were subsequently transformed into ARs according to the formula [ ( left volume − right volume ) /(left volume + right volume ) ] . FreeSurfer ( Martino Center for biomedical Imaging , Harvard-MIT , Boston , MA , USA ) was used to find cortical thickness for the parahippocampal gyrus .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26017779">PMID:26017779</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each participant , cortical thickness was estimated using the FreeSurfer software v5.1.0 ( 13,14 ) at each vertex over the entire cortical surface , a process that has been validated against postmortem histological analysis [ ] and manual measurements [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26009613">PMID:26009613</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To identify these thalamic subdivisions in our patients and controls based on anatomical connections with the cortex , we , therefore , first created anatomical masks of the cortical lobes by combining individual cortical parcellations obtained using FreeSurfer ( v5.2 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26000325">PMID:26000325</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The population in this study included ADNI-1 and ADNI-2 participants who were classified as cognitively normal controls at screening , were tested for CSF biomarkers and presence of the APOE -ε 4 allele , had successful baseline FreeSurfer , The General Hospital Corporation , Boston MA , USA processing of MR images , and completed a battery of neuropsychological exams .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25997414">PMID:25997414</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each patient , the realistic cortical surface and three layers ( inner skull , outer skull and skin ) were reconstructed from the anatomical MRI images using the Freesurfer software ( Compumedics , Charlotte , NC ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25997100">PMID:25997100</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We first parceled the cortex into 68 regions with FreeSurfer ( Desikan–Killiany atlas ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25977936">PMID:25977936</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Freesurfer subcortical segmentation requires these steps : ( 1 ) optimal linear transform maximizing the likelihood of the input image , given an atlas constructed from manually labeled images ; ( 2 ) nonlinear transform initialized with the linear one , in order to allow the image to further deform and better match the atlas ; and ( 3 ) Bayesian segmentation procedure and maximum a posteriori estimate of the labeling .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25977810">PMID:25977810</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A comparison using open access brain data with publicly available brain extraction algorithms including AFNI 's 3dIntracranial , FSL 's BET2 ( ref . ) , Freesurfer 's mri_watershed , and BrainSuite demonstrated that our combined registration/segmentation approach performs at the top level alongside BrainSuite ( tuned ) and FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25977800">PMID:25977800</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Neuroimage 62 , 743 –747 ( 2012 ) .21889996 Fischl B . FreeSurfer . . Neuroimage 62 , 774 –781 ( 2012 ) .22248573 Software registered at the Neuroimaging Informatics Tools and Resources Clearinghouse ( NITRC ) http://www.nitrc.org/projects/gretna/ ( 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25972798">PMID:25972798</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For participants with MRIs , cortical reconstructions were generated using FreeSurfer resulting in a source space of 5124 vertices ( CorTechs Labs Inc . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25964757">PMID:25964757</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cluster A . Cluster B . Applications . Freesurfer 5.3.0 , build . 1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25961571">PMID:25961571</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For MNE analysis , the cortical surface of each participant was reconstructed from their individual MR images with the Freesurfer software [ , ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25954161">PMID:25954161</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Tractography and connectome measurement White matter maps were obtained from segmentation of T1-weighted images using FreeSurfer ( Martinos Center for Biomedical Imaging , Harvard-MIT , Boston USA ) ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25953026">PMID:25953026</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In phase I , Reference manual segmentation of ICV is measured by Operator 1 and the corresponding error of the ICV estimated through automatic software is determined , ΔFS1 for Freesurfer , ΔFSL1 for FSL and ΔSPM1for SPM .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25945925">PMID:25945925</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The PostFreeSurfer pipeline converts the FreeSurfer cortical parcellations to native-space GIFTI surface meshes , and then registers the meshes to the Conte69 population average surface .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25945121">PMID:25945121</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Bayesian classifiers are used in the expectation-maximization ( EM ) segmentation methods which have been successfully implemented in several software packages used in the medical imaging community : SPM [ ] , FAST [ ] , FreeSurfer [ ] , and 3DSlicer [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25944614">PMID:25944614</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface projections of cortical T1 and MWF The higher resolution anatomical image for each child was intensity normalized using the N3 algorithm ( ) and the Freesurfer analysis pipeline ( ) was used to segment the cortical ribbon and delineate the cortex into 34 regions per hemisphere .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25926791">PMID:25926791</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We admit there are other ways to define ROIs , such as FreeSurfer parcellation or using non-linear registration method with other atlases .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25926789">PMID:25926789</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The authors reported significant differences between FreeSurfer version ( average : 2.8–3 % ) and a smaller differences between workstation and operating system version .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25926483">PMID:25926483</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Masks for these regions were created by using Freesurfer to automatically segment the MNI-space cohort-specific T1-weighted template .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25915906">PMID:25915906</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based morphometry data For the SBM data , a vertex-wise general linear model based on parametric statistics was applied within FreeSurfer ( MRI_GLMFIT tool ) without using the threshold free cluster enhancement technique .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25914640">PMID:25914640</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Each of the Freesurfer cortical regions was arbitrarily subdivided into a set of small and compact parcels of about 1.5 cm2 ( Hagmann et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25884012">PMID:25884012</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>After lesion segmentation and inpainting , 1 mm3 -isometric T1-weighted images were submitted to FreeSurfer ( v5.2 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25883851">PMID:25883851</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>( a ) Original image. ( b ) Color-coded anatomical segmentation generated by FreeSurfer overlaid on the original imageAll linear measurements were calculated independently by a medical student , a mid-level resident in neurosurgery , a neuroradiology fellow , a board-certified neurosurgeon , and a neuroradiologist .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25879933">PMID:25879933</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>An initial segmentation was performed using Freesurfer [ ] and additional manual editing of the segmentation was then performed using itkgray [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25879535">PMID:25879535</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The details of ADNI standard image acquisition and processing by Freesurfer are described in detail in [ , ] , but we note that the acquisition consisted of T1-weighted MRI scans with an acquisition matrix size of 256 x 256 x 166 in the x- , y- and z-dimensions with a nominal voxel size of 0.94 x 0.94 x 1.2 mm using a sagittal volumetric magnetization prepared rapid gradient echo ( MP-RAGE ) sequence , with an echo time ( TE ) of 4 ms , repetition time ( TR ) of 9 ms , flip angle of 8° .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25875816">PMID:25875816</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data preprocessing and brain network construction FreeSurfer was employed to parcellate the cortex in the subject’s native brain space based on surface-based registration and anatomical prior [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25875758">PMID:25875758</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Amygdala volume comparisons We defined the amygdala ( and all other target regions ) by using Freesurfer ( described above ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25874163">PMID:25874163</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer software ( Fischl et al . ) was utilized to segment the cranial volume ( for a single-compartment boundary element model ) and the cortical mantle from a structural 3D head MRI of the subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25873854">PMID:25873854</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The optimal mapping ϕ0 , 1 is used in further steps to transform the images from the source space to the target image space.2.2.2 . FS + LDDMM segmentationThe FS + LDDMM segmentation steps ( Khan et al . , ) combine the probabilistic labels obtained from the FreeSurfer ( FS ) program along with the LDDMM registration with ground truth template data , to create accurate labels for the subcortical ROIs in the target MRI data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25870760">PMID:25870760</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>An example of this is the Freesurfer recon_all workflow [ ] that segments brain MRI images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25858359">PMID:25858359</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We chose the fsaverage_sym template distributed with the FreeSurfer package ( see Greve et al . , for details of the atlas creation ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25849581">PMID:25849581</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In summary , before PET data analysis using PUP , automatic brain segmentation and parcellation of the MPRAGE data was performed using FreeSurfer v5.1 ( Martinos Center for Biomedical Imaging , Charlestown , Massachusetts , USA ) for each participant .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25844306">PMID:25844306</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Of the 36 , two articles were excluded as they used non-VBM modalities ( i.e. DTI , Freesurfer ) , another two were excluded due to region of interest ( ROI ) analysis as opposed to whole brain analyses , and another eleven were excluded as those studies had subjects with comorbid conditions in the comparison group ( i.e. Intellectual disability , ADHD , Fragile X ) , or in which participants had IQs < 70 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25808370">PMID:25808370</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Briefly , structural T1 images were segmented into 165 cortical and subcortical grey matter masks according to the Desikan-Kilaney grey matter atlas , using Freesurfer ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25803861">PMID:25803861</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The publicly available FreeSurfer ( v5.1 ) volumetric segmentation and cortical surface reconstruction methods were used to obtain regional measures of cortical volume , surface area , and thickness .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25802212">PMID:25802212</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were segmented using FreeSurfer in order to create masks defining 1 , cerebellar cortex , 2 , cerebellar white matter , 3 , brain stem , 4 , cerebral white matter , 5 , basal ganglia ( including the caudate nucleus , the putamen and the globus pallidus ) , 6 , thalamus , 7 , choroid plexus , and the optic chiasm ( not shown ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25798111">PMID:25798111</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical seed regions for tractography were obtained from an automatic segmentation process employing FreeSurfer ( ) applied to T1-weighted images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25798103">PMID:25798103</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRI data can be used to distinguish different tissue types with sufficient spatial resolution , and various open-source and commercial software tools exist for automatic MRI tissue segmentation and mesh generation , including FreeSurfer ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25798082">PMID:25798082</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The skull was removed from the functional images with the brain extraction tool ( BET ) and from the structural images using FreeSurfer ( https://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25793383">PMID:25793383</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image processing and statistical analyses Statistical analyses were carried out in R and FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25784862">PMID:25784862</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Anatomical data were first aligned to the EPI data and then FreeSurfer ( Fischl et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25781983">PMID:25781983</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A ) Absolute difference maps ( mm ) in Freesurfer and Civet .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25781010">PMID:25781010</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal subfields volumes were assessed automatically with the program FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25762941">PMID:25762941</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used to segment the T1-weighted images according to the Desikan–Killiany method ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25750620">PMID:25750620</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>High-resolution anatomical images were processed with FreeSurfer ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25750526">PMID:25750526</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Estimation of cortical thickness Cortical thickness in the 74 MCI patients was measured using FreeSurfer version 5.1.0 , a fully automated MRI-based analysis technique comprising a set of software tools for the study of cortical and subcortical anatomy , as described elsewhere.–Statistical analysis Differences in sociodemographic and neuropsychological characteristics between the groups were tested for using Statistical Package for the Social Sciences version 13.0 software ( SPSS Inc , Chicago , IL , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25745395">PMID:25745395</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Dual-tasking regions of interest ( ROIs ) were then drawn manually using Freesurfer software to cover areas showing overlap between these two contrasts .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25741260">PMID:25741260</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI Parameters . Geometry . Timing . X . Machine . 0.14 . F . 192 ; 100 % ; 0\\192\\192\\0 ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; ( 1.74 , 3.60 , 5.46 , 7.32 ) ms ; 1260.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 0.29 . F . 192 ; 100 % ; 0\\192\\192\\0 ; 120 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2300 ms ; ( 1.74 , 3.60 , 5.46 , 7.32 ) ms ; 1400.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 0.40 . F . – . ; . – . ; . – . \\200\\200\\ . – . ; 176 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 3.39 ms ; 1100.00 ms ; 1 ; 195 . 7 . ° . 3T TimTrio . 0.60 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 176 ; ( 1.1458 , 1.1458 ) ; 1.00 ; PIL . 2530 ms ; 1.66 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 10.60 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 14.40 . F . – . ; . – . ; . – . \\256\\256\\ . – . ; 144 ; ( 0.7812 , 0.7812 ) ; 0.90 ; PIL . 2000 ms ; 2.58 ms ; 900.00 ms ; 1 ; 180 . 9 . ° . 3T Skyra . 15.70 . F . 192 ; 100 % ; 0\\192\\192\\0 ; 128 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; ( 1.74 , 3.60 , 5.46 , 7.32 ) ms ; 1400.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 17.90 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 21.00 . F . 192 ; 100 % ; 0\\256\\256\\0 ; 160 ; ( 0.7656 , 0.7656 ) ; 0.90 ; PIL . 2000 ms ; 2.59 ms ; 900.00 ms ; 1 ; 180 . 9 . ° . 3T Skyra . 25.10 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 128 ; ( 0.9375 , 0.9375 ) ; 1.00 ; PIL . 2530 ms ; 2.24 ms ; 1100.00 ms ; 1 ; 199 . 7 . ° . 3T TimTrio . 26.00 . F . – . ; . – . ; . – . \\448\\512\\ . – . ; 176 ; ( 0.5 , 0.5 ) ; 1.00 ; PIL . 1950 ms ; 2.26 ms ; 900.00 ms ; 1 ; 200 . 9 . ° . 3T TimTrio . 33.90 . M . – . ; . – . ; . – . \\220\\220\\ . – . ; 176 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 3.39 ms ; 1100.00 ms ; 1 ; 195 . 7 . ° . 3T TimTrio . 35.40 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 41.00 . M . – . ; . – . ; . – . \\448\\512\\ . – . ; 352 ; ( 0.5 , 0.5 ) ; 0.5 ; PIL . 1950 ms ; 2.26 ms ; 900.00 ms ; 1 ; 200 . 9 . ° . 3T TimTrio . 45.30 . M . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 47.40 . M . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 55.40 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 55.70 . M . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 63.30 . M . – . ; . – . ; . – . \\192\\192\\ . – . ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; 1.74 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 68.60 . M . 192 ; 100 % ; 0\\192\\192\\0 ; 160 ; ( 1.00 , 1.00 ) ; 1.00 ; PIL . 2530 ms ; ( 1.74 , 3.60 , 5.46 , 7.32 ) ms ; 1400.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 73.30 . F . – . ; . – . ; . – . \\192\\192\\ . – . ; 176 ; ( 1.1458 , 1.1458 ) ; 1.00 ; PIL . 2530 ms ; 1.66 ms ; . − . 1.00 ms ; 4 ; 651 . 7 . ° . 3T TimTrio . 80.00 . F . – . ; . – . ; . – . \\256\\256\\ . – . ; 192 ; ( 0.8594 , 0.8594 ) ; 0.90 ; PIL . 2000 ms ; 2.52 ms ; 900.00 ms ; 1 ; 180 . 9 . ° . 3T Skyra . 80.10 . M . – . ; . – . ; . – . \\448\\512\\ . – . ; 288 ; ( 0.5 , 0.5 ) ; 0.5 ; PIL . 1950 ms ; 2.26 ms ; 900.00 ms ; 1 ; 200 . 9 . ° . 3T TimTrio Missing information is indicated with “– . ” MRI pre-processing Images were de-identified and tissue segmentation of DICOM images was carried out using FreeView , the visualization and editing tool of the FreeSurfer package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25737959">PMID:25737959</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Both protocols had a TR = 9400 ms , TE = 83 ms , and field of view ( FOV )  = 256 mm with an acquisition matrix of 128 × 128 , and a slice thickness of 2 mm to produce 2 × 2 × 2 mm3 isotropic voxels . 2.4 fMRI processing 2.4.1 Structural ( gray-matter ) segmentation and parcellation T1-image segmentation and regional parcellation were conducted using FreeSurfer ( ; , ) following the nomenclature described in .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25725468">PMID:25725468</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The human visual cortex was parcellated with a matching functional hierarchy , adapted from using regions found in the PALS-B12 atlas and V1 as identified by FreeSurfer ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25713530">PMID:25713530</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each participant a representation of their cerebral cortex was constructed using FreeSurfer ( Freesurfer 4.3 , Martinos Center for Biomedical Imaging , Boston , Massachusetts ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25713510">PMID:25713510</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Seventh , multiple cortical regions ( Table ) were segmented from the structural imaging , using Freesurfer and based on the Desikan atlas , to quantify the active connections associated with each of the four models ( Fischl et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25710328">PMID:25710328</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The preferred phase lags for each voxel were projected on the reconstructed , inflated cortical surface using Freesurfer 5.1.0 [ ] . Visual regions V1–V3 , V3AB , and IPS0-IPS4 were defined as phase reversal in angular retinotopic maps . IPS0–4 were defined as contiguous , approximately rectangular regions based on phase reversals along the anatomical IPS [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25710122">PMID:25710122</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In contrast , Freesurfer registers each vertices at individual gyrus/sulcus level to template but ultimately produce individualized measurement of volume , area and surface based on personally modeled brain morphometry and gray matter/white matter boundary contours .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25698957">PMID:25698957</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This assignment of sources to BAs was then compared to an annotation file ( lh.aparc.a2009s . annot ) from Fs average data of FreeSurfer ( CorTechs Labs Inc . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25693169">PMID:25693169</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Statistical maps presented onto inflated cortical surfaces were constructed using the FreeSurfer package ( version 5.0.0 , http://surfer.nmr.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25688202">PMID:25688202</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer software was used together with functional localizer data in order to isolate voxels belonging to visual area V1 and V2 using established methods for retinotopy ( Sereno et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25687773">PMID:25687773</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We assessed longitudinal sub-regional change in gray matter volume ( atrophy ) on serial 2471 T1 -weighted MRI scans using a modified version of the FreeSurfer software package ( for additional details see ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25687738">PMID:25687738</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Briefly , the Freesurfer process involves the following steps . 1 ) Brain scan motion correction , non-uniform intensity normalization ( ) , non-brain tissue ( e.g. skull and neck ) removal and transformation of structural scans into the Talairach space. 2 ) Segmentation of subcortical white matter and gray matter anatomical volumes ( ) . 3 ) Generation of volumetric and surface-based morphometric data ( e.g. volume , cortical thickness ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25685708">PMID:25685708</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Subsequently , a new attenuation map was created based on the registration of the native MP-RAGE with the SUV60–90 min image obtained in this first reconstruction using FreeSurfer 's spmregister .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25658484">PMID:25658484</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For Freesurfer and Spineseg analyses , we used volumetric T1 images of the brain and spinal cord acquired using a standard 8 channel head coil : sagittal orientation , voxel matrix 240x240x180 , voxel size 1x1x1mm3 , TR/TE 7/ 3.201ms , flip angle 8° .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25658118">PMID:25658118</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer definition of the hippocampal subfields includes the dentate gyrus ( CA4-DG ) , the Ammon’s horn subfields ( CA1 , CA2-3 ) , the subiculum and presubiculum [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25653598">PMID:25653598</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject , the 3D brain surface was reconstructed and then an automatic parcellation was performed using Freesurfer ( Destrieux et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25642430">PMID:25642430</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer , was used to derive measurements of individual brain regions .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25635390">PMID:25635390</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical surface was reconstructed using Freesurfer version 5.0 [ ] and network nodes were defined according to the atlas provided in [ ] ( see ) in combination with subcortical segmentation using FIRST [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25633832">PMID:25633832</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumes of the hippocampus and amygdala and subcortical grey matter including thalamus , caudate and putamen were calculated using the automated full brain segmentation process in Freesurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25628562">PMID:25628562</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data can be in one of several supported binary or text formats commonly used in neuroimaging research : MNI OBJ , Freesurfer binary and Freesurfer ASC .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610795">PMID:25610795</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.4.2 Cortical thickness analysis We analyzed the cortical thickness and subcortical volumes of lesion-filled 3D-MPRAGE images using FreeSurfer 5.10 as described by Fischl et al . ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610792">PMID:25610792</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Ventricle and white matter time series were derived for each subject by segmenting the T1 weighted scan into gray , white , and CSF compartments using FreeSurfer ( ) , and these masks were resampled to EPI resolution and eroded by 1 voxel to prevent partial volume effects with gray matter .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25610784">PMID:25610784</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All scanning acquisition was completed using a 3.0 T Philips 3T ‘Achieva’ MRI scanner ( Best , The Netherlands ) . MPRAGE images ( Slice thickness = 1.0 mm ; FOV = 26 cm ; Matrix size : 256 × 256 ) were checked for motion and only images with minimal motion were used for FreeSurfer processing .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25569227">PMID:25569227</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image Analysis Structural MRI acquisition and pre-processing Structural scans were intensity-corrected , skull-stripped , normalized , segmented and parcellated ( as described below ) using Freesurfer v.5.0.0 cortical reconstruction all with default settings , accessed via the Connectome Mapping Toolkit v.1.2.0 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25540017">PMID:25540017</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Unlike SPM , FSL and our method , FreeSurfer is designed to segment between the brain structures ( both cortical and sub-cortical ) such as thalamus , hippocampus , etc .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25536493">PMID:25536493</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Participants' structural scans were processed via FreeSurfer ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25505411">PMID:25505411</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural analysis based on segmentation of brain cortical and subcortical structures from T1 high-resolution anatomical data was performed using the Freesurfer toolkit version 5.1 ( https://surfer.nmr.mgh.harvard.edu ) running on an Ubuntu 12.04 LTS system .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25493427">PMID:25493427</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Oc VOI included cuneus , lingual , lateraloccipital , and pericalcarine lobules given by Freesurfer , and served as reference region , after confirming that these regions showed similarly low radioactivity .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25484929">PMID:25484929</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer cortical thickness and volume calculations The FreeSurfer cortical parcellation procedure was subsequently applied to the fully segmented Lesion Explorer scans .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25484928">PMID:25484928</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Partial correlations were calculated between the 45 Freesurfer subcortical atlas ROI volumes and Aβln values and also between the 68 Freesurfer cortical atlas ROI volumes and thickness average values and Aβln values .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25479554">PMID:25479554</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer allows an automated parcellation of the cortical surfaces into 34 anatomical regions of interest ( ROI ) for each hemisphere , segmentation of subcortical ROIs ( thalamus , caudate , putamen , hippocampus , amygdala ) and cerebellum , see , for technical details .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25463817">PMID:25463817</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The resulting clusters with an average tool picture preference ( red ) and an average animal picture preference ( blue ) for groups of 7- to 8-year-olds , 9- to 10-year-olds and adults are displayed on the standard Freesurfer surface in ( top ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25452742">PMID:25452742</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The GM and WM mask were obtained using Freesurfer from a T1-weighted image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25429357">PMID:25429357</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>On an up-to-date single-processor PC , the FreeSurfer pipeline typically takes several hours to run for a single scan while SPM takes minutes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25429309">PMID:25429309</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer [ ] ( version 5.1 [ ] ) was used to align cortical folding patterns [ ] and to parcellate the cortical surface and segment the subcortical grey matter into predefined regions of interest ( ROIs ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25413681">PMID:25413681</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Visualization of all-to-all Connectivity and Graph-Theoretical Network AnalysisGroup-level clusters with significant task-dependent modulation of interareal coupling were visualized on an atlas brain using FreeSurfer [ Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25405780">PMID:25405780</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Therefore , we created a study-specific hippocampal mask from the Freesurfer volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25400541">PMID:25400541</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray and white matter were automatically segmented using Freesurfer and hand edited in ITKGray to minimize segmentation errors ( Teo et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25392196">PMID:25392196</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface and labelling of cortical gyri were obtained using Freesurfer v5.0 ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25379457">PMID:25379457</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Conventional autopsy was performed according to the guidelines set down by the Royal College of Pathologists ( UK ) ( ) , during which the cerebrum weight was measured. 2.4 Image preprocessing All images underwent bias field correction using the N3 algorithm of FreeSurfer ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25379441">PMID:25379441</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Moreover , in the subset being studied , the number of subjects in each diagnostic group belonging to the two scanners is evenly distributed across the two scanners ( see ) , indicating that the chances of biases are reduced. 2.4 Thickness measurement and processing Initial cortical reconstruction and volumetric segmentation of the whole brain were performed with the Freesurfer image analysis suite ( ) to obtain pial and WM/GM surfaces .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25379433">PMID:25379433</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Source locations were defined in the subject 's space using the cortical segmentation produced by Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25370484">PMID:25370484</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Hippocampal Subfield Segmentation Image analysis was carried out using the Freesurfer image analysis pipeline ( version 5.1.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25367029">PMID:25367029</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Masks of whole brain grey and white matter were extracted and the Desikan-Killiany Atlas in Freesurfer was used to obtain 34 grey matter volumes of interest per hemisphere in the native coordinate system of each subject ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25356977">PMID:25356977</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to increase the signal to noise ratio of the structural image , the two acquired T1-weighted images were registered and averaged in Freesurfer v5.1.0 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25356425">PMID:25356425</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MRI images were segmented and parcellated into individual cortical regions with FreeSurfer , and used to extract mean florbetapir uptake ( standardized uptake value ratio , SUVr ) from gray matter within lateral and medial frontal anterior , posterior cingulate , lateral parietal , and lateral temporal regions relative to uptake in the whole cerebellum ( white and gray matter ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25356420">PMID:25356420</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis Data were analysed using Freesurfer , an automated segmentation and cortical reconstruction tool .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25346721">PMID:25346721</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>fMRI preprocessing Structural data were preprocessed by Freesurfer software ( v.5.1 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25346672">PMID:25346672</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Each participant’s cortical surface was reconstructed from high-resolution T1-weighted MR images using FreeSurfer software ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25346670">PMID:25346670</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MEG source analysis The structural MR images were processed using FreeSurfer ( CorTechs Labs , La Jolla , CA and MGH/HMS/MIT Athinoula A . Martinos Center for Biomedical Imaging , Charleston , MA ) to create a cortical reconstruction of each brain .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25337918">PMID:25337918</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These scans were then used to create a cortical surface model using FreeSurfer , for visualization in SUMA .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25337075">PMID:25337075</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cross-sectional image processing was performed using FreeSurfer version 5.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25333997">PMID:25333997</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>NeuroImage 30 : 1010 –1020 .16300967 27 Lehmann M , Douiri A , Kim LG , Modat M , Chan D , et al ( 2010 ) Atrophy patterns in Alzheimer 's disease and semantic dementia : a comparison of FreeSurfer and manual volumetric measurements .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25309436">PMID:25309436</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer to obtain total cerebral gray and white matter volumes and total cerebellar gray and white matter volumes ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25309398">PMID:25309398</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Source localization analysis An MRI-derived surface model of each participant’s brain was initially estimated from the T1-weighted structural volumetric images by using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25255942">PMID:25255942</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Here , we use FreeSurfer 5.3.0 ( the latest stable release as of April 2014 ) , running “recon-all -autorecon1” and obtaining the ICV using “mri_segstats --etiv-only” .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25249977">PMID:25249977</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Statistical analysis of MRI and PiB-PET data T1 MPRAGE 3D volumes were postprocessed using an automated subcortical parcellation algorithm ( Freesurfer image analysis suite ; Fischl et al . , ) for definition and volumetry of 29 cerebral anatomical structures included in the standard lookup table ( FreeSurferColorLUT ) , as performed in earlier projects of our group ( Unschuld et al . , , , ; Steininger et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25247680">PMID:25247680</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface pial anatomy was extracted using Freesurfer 5 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25228868">PMID:25228868</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>; Jenkinson et al . , ) and FreeSurfer ( Fischl et al . , ) softwares .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25228353">PMID:25228353</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>None L Left , R Right , ACC Anterior Cingulate Cortex , DLPFC Dorsolateral Prefrontal Cortex . Figure 2 Summary of fMRI results rendered onto an averaged brain by FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25221467">PMID:25221467</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing T1-weighted images were processed with Freesurfer to obtain gray matter ( GM ) 68 cortical regions and 14 subcortical regions ( Desikan et al . , ) ( Table ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25221393">PMID:25221393</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Pardoe and Jackson[ ] conducted a study comparing manual segmentation and automated methods for HV assessment , in reply to the study conducted by Coan et al . [ ] They used the Freesurfer software for automated assessment .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25217827">PMID:25217827</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>PIB distribution volume ratio ( DVR ) images were created using Logan graphical analysis with frames corresponding to 35–90 min post-injection and a gray matter ( GM ) cerebellum reference region defined using FreeSurfer v5.1 software– .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25205048">PMID:25205048</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Objectives : We aimed to evaluate if cortical thickness and subcortical volumes obtained with FreeSurfer software could be reliably measured in 2D T1-weighted images as compared to the same measures obtained with 3D T1 sequences .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25194848">PMID:25194848</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Segmenting the connections terminating in specific regions of interestTo identify connections between different brain regions ( hMT+ , superior parietal gyrus , and primary visual cortex , Fig . 8 and 9 ) , we : ( 1 ) performed a whole-brain automatic parcellation using FreeSurfer . ( 2 ) We used the cortical regions for primary visual cortex ( V1 ) and hMT+ using the cortical regions provided by FreeSurfer by transforming them to the space of the diffusion data . ( 2 ) We expanded these regions to cover portions of the white matter adjacent to each cortical area by applying a 3 dimensional Gaussian smoothing with a spatial kernel of 3 mm3 . ( 3 ) We identified the fascicles in the whole-brain connectome with termination ( endpoints ) inside these expanded ROIs . ( 4 ) The anterior commissure was identified manually in each subject from the high-resolution anatomical image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25193945">PMID:25193945</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric measures of the same brain images made in Freesurfer and in SPM can differ by as much as 20 % ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25174814">PMID:25174814</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These time series data were preprocessed using tools from FSL and FreeSurfer to implement gradient unwarping , motion correction , fieldmap-based EPI distortion correction , brain-boundary-based registration of EPI to structural T1-weighted scan , non-linear ( FNIRT ) registration into MNI152 space , and grand-mean intensity normalization .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25161817">PMID:25161817</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Normalized amygdala and hippocampal values were calculated as follows : [ raw data volume/ICV ] *1000 . For each of these ROIs the right- and left-hemisphere measurements were pooled together . Figure 1 Cortical parcellation units ( PUs ) involved with a priori hypothesis as obtained by Freesurfer 's segmentation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25153800">PMID:25153800</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Non-uniformity correction using N3 MINC B0 tool ( part of Freesurfer , http//www.nitrc.org/projects/freesurfer ) was applied on the FLAIR images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25152729">PMID:25152729</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Voxels with differences at p < 0.05 were considered statistically significant , and were superimposed on the template of the inflated cortical surface brain extracted by FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25121624">PMID:25121624</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging preprocessing The fMRI data were preprocessed and analyzed using combined scripts of three software , AFNI ( Medical College of Wisconsin , Milwaukee , Wisconsin , USA ) , FSL ( Analysis Group , FMRIB , Oxford , UK ) , and Freesurfer ( Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25117541">PMID:25117541</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The surfaces were aligned via spherical registration to the Freesurfer average atlas ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25111472">PMID:25111472</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were fed to Freesurfer ( version 5.1.0 ) in order to segment each participant’s cortex into sixty-six anatomical cortical regions ( ) ( see Supplementary Table 1 for region labels ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25101235">PMID:25101235</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Each peak is represented as a 1 cm diameter sphere projected on the cortical surface using Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25092675">PMID:25092675</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the four cortical ROIs ( i.e. , rACC , dACC , DLPFC , and lOFC ) , we used the FreeSurfer package ( https://surfer.nmr.mgh.harvard.edu/ ) to parcellate these structures and evaluated MTR within cortical white matter to minimize partial volume effects from the cerebrospinal fluid .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25075609">PMID:25075609</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Eroded thalamus masks refer to an eroded version of the Freesurfer thalamus which was transformed using the fslmaths erode filtering operation with a box kernel of width 9 , a step taken to prevent the thalamic mask from overlapping the corpus callosum and resulting in spurious interhemispheric streamlines .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25071554">PMID:25071554</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Regional volume segmentation and cortical thickness parcellation FreeSurfer ( version 4.5.0 ) was used for image processing and analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25071521">PMID:25071521</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each participant , FreeSurfer was used to create a surface representation of the participant 's MRI ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25047083">PMID:25047083</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This approach has been previously validated by direct comparison to a reference , automated method based on co-registration of anatomical parcellation maps ( obtained by applying the FreeSurfer analysis pipeline to T1-weighted images ) to the DTI images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25038441">PMID:25038441</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer surface extraction FreeSurfer ( ) was used to perform extraction of cortical surfaces from the anatomical MRI of the individual subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25018690">PMID:25018690</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As described in section 2.7.3 , MEG source localization was performed based on anatomical information from MRI data , which was processed and segmented using the FreeSurfer software ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25013396">PMID:25013396</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To study the effects of all common variation in FOXP2 on candidate brain regions from prior studies of rare variation , volumes of the caudate nucleus , cerebellum and inferior frontal cortex were segmented using FreeSurfer version 5.1 using labels “caudate right” + “caudate left” as caudate nucleus volume ; “cerebellum cortex right” + “cerebellum cortex left” as cerebellum volume ; and “left and right pars orbitalis” + “left and right pars triangularis” + “left and right pars opercularis” as inferior frontal cortex volume .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25012867">PMID:25012867</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis was carried out using the FreeSurfer image analysis pipeline ( version 5.1.0 ) to produce regional cortical thickness and subcortical volumetric measures as previously described .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25005037">PMID:25005037</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MEG sensor configurations and MRI images obtained for each participant were co-registered based on the matching of 1 ) digitized locations on the scalp surface ( varying in number between ∼50 and 100 across participants ) with 2 ) the reconstructed scalp surface from the FreeSurfer software ( see below ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24994981">PMID:24994981</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To map current dipoles on the cortical surface , a polygon model of the cortical surface was constructed based on T1-weighted MRI data using FreeSurfer software ( Martinos Center software ) ( Dale et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24985056">PMID:24985056</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer software carries out sulcal alignment and cortical thickness analyses in the spherical space following linear alignment to the Talairach atlas [ Talairach and Tournoux , 1988 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24982647">PMID:24982647</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All individual ROI masks were generated using FreeSurfer 's automatic anatomical parcellations of both cortical gray-matter and subcortical regions ( Destrieux et al . , ) , allowing us to perform the analyses within each subject 's own anatomical space and to avoid the loss of statistical power and normalization inaccuracies associated with group-level analyses done in a standard space .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24961428">PMID:24961428</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These results are in keeping with hypotheses and results found in [ , , ] . Software for our fMRI image analysis includes FSL [ ] , SPM , FreeSurfer [ ] , and fScan [ , , ] . Longitudinal experiments provide a unique window into individual as well as group differences and similarities in second language acquisition .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24961262">PMID:24961262</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This T1 pulse sequence was specifically selected to optimize processing using the FreeSurfer suite , which aligns sequential images based on patterns of cortical folds [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24945305">PMID:24945305</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>By combining q-ball tractography and Freesurfer parcellation we provided on a large group of individuals a consistent demonstration of Sarubbo et al . post mortem findings , by using a similar description of the different subcomponents of the IFOF ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24942135">PMID:24942135</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Data analyses Anatomical image processing FreeSurfer was used for surface reconstruction ( Dale et al .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24924673">PMID:24924673</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Three-dimensional T1-weighted volumetric images are analyzed using FreeSurfer software ( Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24922325">PMID:24922325</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetry analysis Anatomical data were processed to derive the total supratentorial volume , regional subcortical volumes ( 45 regions ) , white matter ( WM ) volumes ( 70 regions ) , and cortical gray matter ( GM ) volumes ( 148 regions ) using Freesurfer software ( version 5.1.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24917840">PMID:24917840</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Additionally , single-subject inflated cortical surfaces were constructed using Freesurfer software ( ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24907249">PMID:24907249</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Note : in all figures , overall thresholded activation maps ( Z > 3.1 , whole-brain cluster-corrected level of P < 0.05 ) for each group are registered to and displayed on an adult 's cortical surface in FreeSurfer ( normalized to the MNI-152 template ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24904520">PMID:24904520</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The dimension reduction method subdivides the cortex by partitioning each Freesurfer label ( such as posterior cingulate cortex ) into 10 smaller patches using the spatial clustering of vertices using k -means method .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24904377">PMID:24904377</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The subject’s MRI was processed using FreeSurfer ; cortical and head surfaces were extracted .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24903826">PMID:24903826</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional imaging analysis T1-weighted anatomical images were processed with FreeSurfer ( Dale et al .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24896833">PMID:24896833</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To this end , a regional parcellation was imposed on T2 images using the labeled Freesurfer atlas using a recently developed pipeline in our laboratory .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24889999">PMID:24889999</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The all group included the FreeSurfer segmented regions excluding left-vessel , right vessel and 5th ventricle because these regions were not segmented by FreeSurfer in all subjects .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24886034">PMID:24886034</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Automated morphometry to evaluate cortical thickness and surface area , total gray and white matter volumes , and deep nuclei volumes will be performed using the freely available FreeSurfer software program , v5.1 [ ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24879410">PMID:24879410</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject , a polygon model of the cortical surface was constructed based on MR structural images using FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24860545">PMID:24860545</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In the automated FS + LDDMM pipeline , FreeSurfer ( FS ) [ ( ) , using version 4.5.0 ] first provides rough , initial segmentation of 37 brain regions including the hippocampus and the ventricles in each hemisphere .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24860494">PMID:24860494</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The main steps of structural and functional preprocessing in CCS included ( 1 ) brain tissue segmentation and cortical surface reconstruction , ( 2 ) removal of the first 5 EPI volumes ( 10 s ) from each scan to allow for signal equilibration , ( 3 ) slice timing correction , ( 4 ) 3D motion correction , ( 5 ) 4D global mean-based intensity ( 10,000 ) normalization , ( 6 ) removal of 26 nuisance covariates of WM/CSF signals and 24 motion parameters estimated with the Friston-24 model ( ; ) , ( 7 ) band-pass temporal filtering ( 0.01–0.1 Hz ) , ( 8 ) removal of linear and quadratic trends , ( 9 ) co-registration between individual functional and anatomical images using a rigid boundary-based transformation ( BBR ; ) , and ( 10 ) projection of functional images onto the standard cortical surface ( subject fsaverage5 in Freesurfer 5.1 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24860458">PMID:24860458</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Preprocessing The preprocessing of resting-state fMRI data was carried out using FSL ( Jenkinson et al . , ) , AFNI ( Cox , ) , and FreeSurfer ( Dale et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24859328">PMID:24859328</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The analysis was performed with SPM8 , FreeSurfer , Artifact Rejection Toolbox ( ART ) , and Advanced Normalization Tools ( ANTS ) , using Nipype and bash scripts for workflow design and execution .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24834062">PMID:24834062</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Since individual MRI-images were not available for six subjects , a FreeSurfer average brain was applied as a surrogate in these subjects ( by aligning the individual fiducial points to the fiducial points of the average head ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24824880">PMID:24824880</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>MR data processing Preprocessing Cortical surface reconstruction was performed on the T1-weighted scans using FreeSurfer 5.1 – followed by iterative manual inspection and correction of the cortical segmentations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24823365">PMID:24823365</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the Freesurfer software package – and 3dSlicer to create the surface models of the cortex , amygdala , and hippocampus .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24802625">PMID:24802625</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis Cortical reconstruction and volumetric segmentation was performed with the FreeSurfer v5.1.0 imaging analysis suite ( https://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24795624">PMID:24795624</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric measurements were obtained from parcellation of anatomical T1 images at baseline and at 18-months using the longitudinal method of FreeSurfer software ( v . 5.1 ) ( Fischl et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24782699">PMID:24782699</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To generate surface-based data , data are projected onto a cortical fold that is also shipped in the package and we resorted to an external projection tool , developed in Operto et al . ( ) but others are available such as Freesurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24778617">PMID:24778617</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer allows for automated segmentation of cortical and sub-cortical volumes ( Fischl et al . , , , ; Desikan et al . , ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24778370">PMID:24778370</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>( iii ) Source reconstruction Individual MRIs were segmented using Freesurfer software ( Martinos Center for Biomedical Imaging , MA , USA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24771247">PMID:24771247</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gyrification analysisCortical surfaces were reconstructed using FreeSurfer version 5.1.0 , employing standard preprocessing procedures as described by Dale et al . ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24763508">PMID:24763508</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Using 87 regions of interest ( ROIs ) defined by the Freesurfer Desikan atlas , functional connectivity measures were derived using pairwise BOLD signal averages correlations after Fisher 's r-to-z transformations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24729983">PMID:24729983</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRIs were segmented into cortical regions using FreeSurfer version 4.5.0 ( surfer . nmr . mgh . harvard.edu/ ) . These cortical regions of interest ( ROIs ) were used to extract 18 F-AV45-PET uptake from gray matter ( GM ) in lateral and medial frontal , anterior , and posterior cingulate , lateral parietal , and lateral temporal cortex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24715863">PMID:24715863</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>f PredictAD tool : including demographic data , APOE , MMSE , ADAS-Cog , neuropsychological battery , Aβ42 , T-tau , and several features derived from MRI using FreeSurfer , manifold learning , hippocampal volume , and Tensor-Based Morphometry .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24676149">PMID:24676149</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FMRIB Software Library ( FSL ) toolkit ( http://www.fmrib.ox.ac.uk/fsl/ ) was used to process anatomical images which were then reconstructed and inflated using FreeSurfer ( v5.1 ) ( https://surfer.nmr.mgh.harvard.edu/ ) – .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24672483">PMID:24672483</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Post acquisition image analysis and quantification of functional connectivity Determination of anatomical structures and estimation of volumes was performed by applying the automated image-quantification functions provided by the FreeSurfer image analysis suite ( Fischl et al . , ) , using standard operations for construction of cortical models and further data analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24625753">PMID:24625753</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The hippocampal volume was determined using Freesurfer software ( Athinoula A Martinos Center for Biomedical Imaging , Charlestown , MA ) ( ) and manually edited according to the T1-weighted structural MRI data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24614676">PMID:24614676</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In brief , cortical grey matter regions of interest ( frontal , anterior/posterior cingulated , lateral parietal , lateral temporal ) and the reference region ( whole cerebellum ) were defined with a native-space magnetic resonance imaging ( MRI ) scan for each subject using Freesurfer ( version 4.5.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24608869">PMID:24608869</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To investigate the cortical depth dependence of the diffusion anisotropy , 13 surfaces evenly spaced along the cortical depth in increments of 10 % of the cortical thickness were generated from the anatomical images with FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24608106">PMID:24608106</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In order to assess artifacts and distortions , images of all patients were inspected and CAT/TSE volumes of one representative patient were registered to each other using the inversely consistent , rigid body registration with 6 degrees of freedom ( DoF ) provided by mri_robust_register ( part of FreeSurfer v . 5.3.0 , http://surfer.nmr.mgh.harvard.edu/fswiki ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24600380">PMID:24600380</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gray- and white-matter surfaces were generated from these images using FreeSurfer ( Dale et al . , ) , from which flattened representations of the cortical surface could be constructed for visualization of data .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24594874">PMID:24594874</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Network construction Anatomical parcellation Freesurfer ( Martinos Center of Biomedical Imaging , Boston , US ) software was used to segment the T1 images of each subject into 82 cortical and subcortical regions .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24574993">PMID:24574993</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( FS ) software was used to perform segmentation and regional parcellation ( Fischl et al . , , ) according to methodologies described by Destrieux et al . ( ) . MRI and DTI volumes were co-registered before further analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24558410">PMID:24558410</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the reconstruction of the evoked activity we estimated the minimum-norm solution of a distributed source model , based on the individual cortical sheet , reconstructed using Freesurfer and downsampled to 8196 dipole locations using MNE-suite ( Hämäläinen , Martinos Center for Biomedical Imaging , Massachusetts General Hospital , MA ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24498072">PMID:24498072</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The average cPMC thickness , calculated as the average distance between gray matter/white matter boundary and gray matter/cerebrospinal fluid boundary within the cPMC , was obtained from 3D T1 -weighted MP-RAGE images using FreeSurfer ( v5.1.0 ) , a set of tools that automatically segments , labels and quantifies brain tissue volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24489665">PMID:24489665</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Fiber-projected cortical thickness analyses Individual T1 MRI images were processed using the FreeSurfer software package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24478813">PMID:24478813</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In a different session , two sets of structural images ( FSPGR ; FOV = 25.6 cm ; voxel size = 1×1×1 mm3 ; matrix = 256×256 ; 160–170 axial slices ) were acquired and averaged to make cortical surface reconstructions using FreeSurfer [ , ] . Data Analysis For each session , functional images were motion-corrected and registered to a single functional reference volume using the 3dvolreg tool of AFNI package ( available at http://afni.nimh.nih.gov/afni ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24475148">PMID:24475148</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>implemented in FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24466084">PMID:24466084</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis T1-weighted images were processed using the FreeSurfer image analysis pipeline ( version 5.3.0 ) , which has been described in detail elsewhere , .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24448846">PMID:24448846</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Moreover , WM ROIs based on the FreeSurfer WM parcellations were extracted for FA and DR—the FSL FMRIB FA template ( to which every participant 's FA volume initially was registered ) was co-registered to the standard space T1 volume MNI152 , which subsequently went through the FreeSurfer processing stream to create a volume with WM parcellations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24447099">PMID:24447099</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Thalamocortical segment , hippocampal , and entorhinal volumes were analyzed , both corrected and uncorrected , for total intracranial volume ( ICV ) ; ICV was automatically obtained from Freesurfer segmentations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24429520">PMID:24429520</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A three-dimensional reconstruction of each patient’s brain was computed using FreeSurfer to generate , .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24416229">PMID:24416229</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>BA1 processes tactile shape recognition and is included in Freesurfer as an anatomical mask .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24409145">PMID:24409145</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For volumetric analysis , Freesurfer software package ( version 5.1.0 ) and its automated sub-cortical segmentation tool ( Fischl et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24392092">PMID:24392092</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>First we defined an anatomical search mask of the MTL memory system using a study-specific anatomical probabilistic atlas based on FreeSurfer segmentations of individual T1 images ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24391569">PMID:24391569</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Figure 3 Uninflated surface of the right ( A ) and left hemispheres ( B ) ( FreeSurfer reconstruction ) of the patient with transcortical motor aphasia ( RTP ) showing gyri cultured in green with sulci shown in red .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24371794">PMID:24371794</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For all analyses , subjects more than 2.5 standard deviations were excluded ( typically 1 to 2 subjects per analysis ) . 2.7 Cortical thickness measures FreeSurfer uses intensity and continuity sMRI information in segmentation and deformation procedures to obtain cortical thickness ( CT ) measures , calculated as the closest distance from the gray/white to gray/cerebral-spinal-fluid boundary at each vertex on the tessellated surface ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24367327">PMID:24367327</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All scans performed on a 3T Siemens Trio . Brain masks were created using MBWSS , FreeSurfer version 5.3.0 's preprocessing pipeline ( autorecon1 ) , the graph cut method in VBM8 ( Dahnke et al . , ) , standard BET ( Smith , ) , BET with robust brain center estimation ( BET -R ) and BET with bias field and neck cleanup ( BET -S ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24363990">PMID:24363990</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The large sample size of 325 subjects included in this study should outweigh any concerns about how FreeSurfer calculates TIV ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24363966">PMID:24363966</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Morphometric analyses For morphometric analyses , T1 volumes were segmented into gray , white , cerebrospinal fluid , and nonbrain tissues using the FreeSurfer software package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24351977">PMID:24351977</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used a combination of FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/fswiki ) and Matlab-based software from the Heeger Lab ( New York University ) ( http://www.cns.nyu.edu/heegerlab/ ) to convert the T 1 -weighted and segmented Freesurfer image into SurfRelax ( http://www.pc.rhul.ac.uk/staff/J.Larsson/software.html ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24349453">PMID:24349453</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The intracranial volume ( ICV ) calculated by the FreeSurfer did not differ significantly between groups ( t-test , P=.514 ) and a one-way analysis of covariance ( ANCOVA ) was conducted ( age as a covariance ) on the ICV normalised data ( i.e. volume/ICV ) to assess between-group differences ( ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24348363">PMID:24348363</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used Freesurfer 's automated segmentation program ( Fischl et al . ,</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24324679">PMID:24324679</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For visualization , statistical corrected p value maps ( p FWE <0.05 ) are displayed on the pial cortical surface of the FreeSurfer brain ( fsaverage ) template ( htttp://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24298172">PMID:24298172</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Freesurfer software was applied to the anatomical T1 scan , to parcellate the cortical surface into 68 distinct regions ( ie , 34 cortical regions per hemisphere ) for each individual subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24264234">PMID:24264234</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The surface anatomy used in this procedure was extracted using Freesurfer 5 ; the segmented pial surface was filled and then blurred modestly ( 2 mm ) such that electrodes arrive at a location reflecting the smoothed convexity of the brain .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/24133482">PMID:24133482</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Quantitative evaluation of LDDMM , FreeSurfer , and CARET for cortical surface mapping .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23824159">PMID:23824159</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Previous publications such as have reported Dice values such as 0.73 and 0.76 for the hippocampus , and has reported Dice overlaps on the order of 0.75 for the amygdala using either FreeSurfer and FSL .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23748435">PMID:23748435</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For the source estimations , activity in the left ( top ) and right ( middle ) hemispheres has been pooled for both subject groups in each time-window MRI Acquisition and EMEG Source Reconstruction In order to explore the neuronal generators underlying electrophysiological and neuromagnetic activity , L2 minimum norm source estimations ( Hämäläinen and Ilmoniemi ) for combined EEG/MEG data were computed using MNE and Freesurfer 4.3 software ( Martinos Centre for Biomedical Imaging ) in conjunction with individual subject structural MRI images used to model cortical grey matter surface .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23608060">PMID:23608060</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As detailed below , analyses were performed using FreeSurfer 5.1.0 ( ; ) , FSL 4.1.6 ( ) , and AFNI/SUMA ( 2012/11/23 ; ; ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23542881">PMID:23542881</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We subsequently carried out a more detailed analysis of the pattern of lGI and intrinsic curvature by dividing each individuals cortical reconstruction into the 6 lobes of the brain ( cingulate , frontal , insula , parietal , occipital , and temporal ) , which were defined using the Desikan-Killiany atlas ( ) which is part of the FreeSurfer 5.1 distribution .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23355665">PMID:23355665</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The effects of FreeSurfer Version , Workstation Type , and Macintosh Operating System Version on anatomical volume and cortical thickness measurements .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/23272041">PMID:23272041</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Main processing stages Every processing stage uses a mixture of state-of-the-art neuroimaging tools , such as Fsl , Freesurfer and Diffusion Toolkit , and in-house developed scripts .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/22384048">PMID:22384048</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We would also like to thank all who responded with feedback on the FSL and Freesurfer email lists while we were addressing the methodological complexities of our approach .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28190642">PMID:28190642</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28229041">PMID:28229041</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstructions and volumetric segmentation were generated with Freesurfer image analysis suite V5.3, which is documented and freely available for download online ( http://surfer.nmr.mgh.harvard.edu/ ). The technical details of these procedures are described in prior publications ( Dale et al., 1999 ,  Fischl and Dale, 2000 ,  Fischl et al., 2001 ,  Reuter et al., 2010 ,  Reuter et al., 2012 ,  S é gonne et al., 2004 ). Briefly, this processing included motion correction, removal of non-brain tissue using a hybrid watershed/surface deformation procedure ( S é gonne et al., 2004 ), automated Talairach transformation, segmentation of the subcortical white matter and deep gray matter volumetric structures (including hippocampus, amygdala, caudate, putamen, ventricles; ( Fischl et al., 2002 ,  Fischl et al., 2004 ), intensity normalization ( Sled et al., 1998 ), tessellation of the gray matter white matter boundary, automated topology correction ( Fischl et al., 2001 ,  S é gonne et al., 2007 ), and surface deformation following intensity gradients to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to another tissue class ( Dale et al., 1999 ,  Fischl and Dale, 2000 ). Freesurfer morphometric procedures have good test-retest reliability across scanner manufacturers and field strengths ( Han et al., 2006 ,  Reuter et al., 2012 ). All Freesurfer analyses were performed on the same workstation, using the same software version ( Gronenschild et al., 2012 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28223932">PMID:28223932</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28209179">PMID:28209179</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ) to obtain regions of interest via an automated approach, which parcellated the cortical and white matter regions. We applied FreeSurfer software to segment T1-weighted SPGR images into 34 bilateral, cortical and white matter regions for each participant [ 68 ]. The label map with FreeSurfer-generated regions of interest was registered to the DWI space by first diffeomorphically registering a T2 image in the same space as the SPGR image to the baseline DWI image of the same participant using the FLIRT algorithm of the FSL software [ 69 ], and then applying this diffeomorphism to register the FreeSurfer-generated label map to the DWI space for the same participant. We performed this transformation of the FreeSurfer label map to DWI space for each participant.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28198448">PMID:28198448</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>The cortical and subcortical gray-matter volumes and cortical thickness were calculated using the automated procedure for volumetric measures of whole brain structures implemented in the FreeSurfer 5.3 development version (Massachusetts General Hospital, Boston, U.S.,  http://surfer.nmr.mgh.harvard.edu ) from T1 imaging. We used the demeaned value for the total intracranial cavity volume (TICV) measured manually 62  to normalize the regional brain volume.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5318367">PMID:5318367</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used GEE models with cortical volumes from the FreeSurfer parcellation to confirm VBM results .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28420972">PMID:28420972</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer v5.3.0 image analysis suite, which is documented and freely available for download online at  http://surfer.nmr.mgh.harvard.edu/ . The technical details of these procedures have been described elsewhere (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl et al.,  1999a , b ,  2001 ,  2002 ,  2004a , b ; Fischl and Dale,  2000 ; S é gonne et al.,  2004 ; Han et al.,  2006 ; Jovicich et al.,  2006 ; Reuter et al.,  2010 ,  2012 ). Briefly, the preprocessing procedure includes motion correction of volumetric T1-weighted images, removal of non-brain tissue using a hybrid watershed/surface deformation procedure (S é gonne et al.,  2004 ), and automated Talairach transformation. Moreover, the procedure includes the segmentation of the subcortical white matter and deep gray matter volumetric structures including hippocampus, amygdala, caudate, putamen, ventricles (Fischl et al.,  2002 ,  2004a ), intensity normalization (Sled et al.,  1998 ), tessellation of the gray matter-white matter boundary, and automated topology correction (Fischl et al.,  2001 ; S é gonne et al.,  2007 ). In addition, surface deformation following intensity gradients was performed to optimally place the gray/white and gray/cerebrospinal fluid borders at the location where the greatest shift in intensity defines the transition to the other tissue class (Dale and Sereno,  1993 ; Dale et al.,  1999 ; Fischl and Dale,  2000 ). Once the cortical models are complete, a number of deformable procedures can be performed for further data processing and analysis. These procedures include surface inflation (Fischl et al.,  1999a ), registration to a spherical atlas that is based on individual cortical folding patterns to match the cortical geometry across subjects (Fischl et al.,  1999b ). Moreover, parcellation of the cerebral cortex into units with respect to gyral and sulcal structure (Fischl et al.,  2004b ; Desikan et al.,  2006 ) and creation of a variety of surface based data including maps of curvature and sulcal depth are part of the procedure. This method uses both intensity and continuity information from the entire three-dimensional MR volume in segmentation and deformation procedures to produce representations of cortical thickness, calculated as the closest distance from the gray/white boundary to the gray/CSF boundary at each vertex on the tessellated surface (Fischl and Dale,  2000 ). The maps are created using spatial intensity gradients across tissue classes rather than simply relying on absolute signal intensity. The maps produced are not restricted to the voxel resolution of the original data, and are thus capable of detecting submillimeter differences between groups. Procedures for the measurement of cortical thickness have been validated against histological analysis (Rosas et al.,  2002 ) and manual measurements (Kuperberg et al.,  2003 ; Salat et al.,  2004 ). FreeSurfer morphometric procedures have been demonstrated to show good test-retest reliability across scanner manufacturers and across field strengths (Han et al.,  2006 ; Reuter et al.,  2012 ). A cortical surface-based Desikan-Killiany-Tourville (DKT) atlas Klein and Tourville,  2012 ) was mapped to a sphere aligning the cortical folding patterns, which provided accurate matching of the morphologically homologous cortical locations across subjects. For each of the DKT31 protocol-based segments, FreeSurfer calculated the nine different measures, including number of vertices, surface area, gray matter volume, average cortical thickness, cortical thickness standard deviation, cortical mean curvature, cortical Gaussian curvature, cortical folding index, and cortical curvature indices (Colby et al.,  2012 ). For the subcortical regions, FreeSurfer calculated the area and volume of the whole segment, white matter volume, intensity and overall volume of the whole brain divisions including cerebrospinal fluid (CSF), intracranial volume (ICV), gray matter (GM), and white matter (WM). Two of the selected measures are the most common features in the structural studies (Arbabshirani et al.,  2017 ). The surface area was calculated by computing the area of every triangle in a standardized spherical surface tessellation. The local curvature was computed using the registration surface based on the folding patterns (Qureshi et al.,  2016 ).</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28320844">PMID:28320844</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28317116">PMID:28317116</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28523229">PMID:28523229</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28213439">PMID:28213439</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28383284">PMID:28383284</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28611613">PMID:28611613</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>2 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28596727">PMID:28596727</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>3 http://surfer.nmr.mgh.harvard.edu/</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28079061">PMID:28079061</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Standard preprocessing of the structural images, followed by longitudinal preprocessing steps that further reduced within-subject variability, were performed using FreeSurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu ) ( Fischl  et al , 2002 ,  2004 ;  Reuter  et al , 2012 ). The segmentation of the hippocampus into its subfields was performed using a recently validated segmentation algorithm that was based on an atlas constructed from ultra-high-resolution  ex vivo  hippocampal tissue ( Iglesias  et al , 2015 ). Details of these methods can be found in  Ho  et al  (2016) . Visual quality control of the image processing that included checking for excessive motion was performed ( Supplementary Methods S3 ). The proportion of UHR subjects excluded because of motion was not different from healthy subjects. Further, data set from an UHR-NR subject was removed from the analysis because of poor longitudinal image reconstruction. Hippocampal segmentation results were visually checked for each subject; there were no instances of incomplete labeling of hippocampal subregions or mislabeling of extrahippocampal regions in the remaining 147 subjects.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5447679">PMID:5447679</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The individual cortical surfaces were imported from FreeSurfer and tessellated with 15,000 nodes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5437892">PMID:5437892</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Standard preprocessing of the structural images , followed by longitudinal preprocessing steps that further reduced within-subject variability , were performed using FreeSurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu ) ( Fischl et al , 2002 , 2004 ; Reuter et al , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5405083">PMID:5405083</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The first volume-based pipeline removes spatial distortions , realigns volumes using FSL 's FLIRT-based motion correction , normalizes the intensity of 4D images to a global mean , registers data into MNI space and finally masks the data with the final brain mask derived from FreeSurfer segmentation , while the second surface-based pipeline aims at transforming the time series from volume space to CIFTI gray-ordinates standard space with 2 mm average surface vertex and subcortical volume spacing .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28209734">PMID:28209734</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28179553">PMID:28179553</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28671063">PMID:28671063</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28242797">PMID:28242797</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28676749">PMID:28676749</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and thickness segmentation were performed with the FreeSurfer image analysis suite (FreeSurfer 5.1,  http://surfer.nmr.mgh.harvard.edu ). To extract reliable volume and thickness estimates, images were automatically processed with the longitudinal stream in FreeSurfer (Reuter et al.,  2012 ). The process includes three different steps: Firstly, all scans of different time points are processed independently in a cross-sectional analysis. Secondly, a base image of the person's brain anatomy is created. This is done by using all cross-sectional scans to create an unbiased within-subject template space and image using robust, inverse consistent registration (Reuter et al.,  2010 ). More specifically, a median image is conducted which functions as an average image across all time points. Using this average image, all subsequent segmentations can be done on the approximated subject's anatomy of the brain. Thirdly, the longitudinal analyses take place where each time point is processed with respect to the within-subject template. This means, the difference to the template is observed and analyzed. The use of such a matching between average images and individual time point provides the advantage to automatically correct for parameters, which are the same in all time points for the subject. Several processing steps, such as skull stripping, Talairach transforms, atlas registration as well as spherical surface maps and parcellations are then initialized with common information from the within-subject template, significantly increasing reliability and statistical power (Reuter et al.,  2012 ). Continuative statistical analysis was based on the resulting difference variable and done with mri_glmfit.</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5524936">PMID:5524936</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The 221 individual brain scans were processed with the FreeSurfer version 5.3 pipeline ( http://surfer.nmr.mgh.harvard.edu ) to produce surface-based maps of sulcal depth and local gyrification index for each individual .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5524625">PMID:5524625</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The procedure including motion correction , intensity normalization , automated topology corrections and automatic segmentations of cortical and subcortical regions is documented elsewhere. 37 – 39 A novel automated algorithm from FreeSurfer that will be released in the next version was used to segment the hippocampal subfields .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5516607">PMID:5516607</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The authors showed that this method is able to predict cognitive outcomes of ADNI subjects using FreeSurfer ‐ based baseline MRI features , MMSE score demographic information and ApoE status .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5516586">PMID:5516586</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ) as previously described. 12 This includes segmentation of the subcortical white matter and deep gray matter volumetric structures , 47 and parcellation of the cortical surface according to a previously published parcellation scheme .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5515845">PMID:5515845</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The AlphaSim program in the rest software ( http://restfmri.net ) was used for the VBM analysis and a precomputed Monte Carlo simulation implemented by FreeSurfer was used for the thickness analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5504280">PMID:5504280</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The processing steps to obtain an accurate mask of the whole thalamus are summarized in Fig .   1 . First , we performed cortical and subcortical parcellation of the T1-weighted images with the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5500013">PMID:5500013</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based analyses in FreeSurfer involves the removal of non-brain tissue using a hybrid watershed algorithm , automated Talairach transformation , segmentation of subcortical white matter and cortical gray matter , intensity normalization , tessellation of gray/white-matter boundary , automated correction of topological defects and surface deformation to form the gray- and white matter boundary [ 16 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5498510">PMID:5498510</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Two of the parcellations commonly used in the FreeSurfer software package ( Dale et al . , 1999 ; Fischl et al . , 1999 , 2004 ) are the Desikan-Killiany Atlas ( Desikan et al . , 2006 ) and the Destrieux Atlas ( Destrieux et al . , 2010 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5491557">PMID:5491557</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The CCS designed by Zuo et al . ( 2013 ) provides a computational platform for multimodal neuroimaging brain connectomics computations by integrating the functionalities of AFNI , FSL and FreeSurfer ( Zuo et al . , 2013 ; Betzel et al . , 2014 ; Cao et al . , 2014 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5491489">PMID:5491489</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.6 Each participant 's cortical surface was determined using the FreeSurfer image analysis software ( http://surfer.nmr.mgh.harvard.edu ) from T1-weighted MR volumetric images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5491456">PMID:5491456</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.3 T 1 -weighted scans were processed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) to estimate vertex-wise cortical thickness and surface arealization ( Dale et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5490683">PMID:5490683</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The parcellations were obtained in FreeSurfer surface space from http://www.freesurfer.net/fswiki/CorticalParcellation_ Yeo2011 . After calculating the time series for each of these networks , we performed a functional connectivity analysis separately for each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5490676">PMID:5490676</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Because processing pipelines and ROI parcellations differ between image processing software programs , and because studies have reported that results may vary as a function of program ( Rajagopalan and Pioro , 2015 ) , we computed volumes of the prefrontal cortex ROIs ( mOFC , lateral orbitofrontal cortex , inferior frontal gyrus , MFG , SFG , and ACC ) from two separate programs ( FreeSurfer and SPM ) and performed separate analyses with both sets of values ( see Supplementary Table S1 for correlations between FreeSurfer and SPM for the average volume of each prefrontal cortex subregion ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5479971">PMID:5479971</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We obtained cortical morphology ( thickness , and surface area ) and brain volume ( global , subcortical , and regional ) measurements using FreeSurfer ( version 5.3.0 , https://surfer.nmr.mgh.harvard.edu ) and compared measurements of children with epilepsy and SCN1A gene mutation ( n   =   21 ) with those of age and gender matched healthy controls ( n   =   42 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5476738">PMID:5476738</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To extract reliable volume and thickness estimates , images were automatically processed with the longitudinal stream in FreeSurfer ( Reuter et al . , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5476728">PMID:5476728</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer analysis suite ( v5.3.0 release 1 ) was used to derive models of cortical surface in each T1-weighted image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5474556">PMID:5474556</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain volumes were derived using the automated FreeSurfer 5.3 Pipeline 2 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5474436">PMID:5474436</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Then we inputted them into a FreeSurfer software package ( version 5.3.0 , http://surfer.nmr.mgh.harvard.edu ) , a widely documented and automated program for reconstructing brain cortical surfaces and calculating CT ( Fischl , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5472153">PMID:5472153</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These images were processed using the Tracts Constrained by Underlying Anatomy pipeline ( Yendiki et al . , 2011 ) , which combines processing steps from FDT ( FSL software for diffusion tensor analysis ; Behrens et al . , 2003 ) with the surface , volume , and automatic segmentation derived from the FreeSurfer software package ( Fischl et al . , 2002 , Fischl and Dale 2000 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5465275">PMID:5465275</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In case of inaccuracies , manual editing was performed according to the FreeSurfer editing manual , 4 either by adding control points to help FreeSurfer identify the WM voxels or removing the skull and dura in case they were considered to be parts of the brain .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5465264">PMID:5465264</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>First , pre-processing co-registers T1 and the brain tissue segmentation from FreeSurfer to FLAIR space before applying inhomogeneity correction and noise removal .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5463038">PMID:5463038</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used to generate anatomical regions of interest ( ROIs ) for the CBF data and , for secondary analyses , cortical thickness and volume and data for these ROIs .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5460987">PMID:5460987</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu ) was used to perform gray and white matter segmentation and extraction of the mid-gray cortical surface used for source locations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5460049">PMID:5460049</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with FreeSurfer 5.3 software which procedure is documented online ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5459352">PMID:5459352</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Using the FreeSurfer BrainSegNotVent variable , we compared brain size between the two groups with age and gender as covariates .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5457495">PMID:5457495</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>V1 was determined using the automatic cortical parcellation provided by FreeSurfer 44 based on individual T1 images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5456080">PMID:5456080</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Global probabilistic tractography and diffusion parameter analysis were performed using TRACULA [ 17 ] , which is part of the FreeSurfer 5.3 [ 18 ] software package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5451775">PMID:5451775</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Amygdala volumes were analysed with FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5449740">PMID:5449740</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer version 5.3 3 was used to calculate the cortical thickness from T1-structural images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5445177">PMID:5445177</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://freesurfer.net ) was used to segment T1 images into GM and WM masks , which were visually checked and manually edited for misclassification due to WM T1-hypointensities associated with lesions .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5444023">PMID:5444023</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural T1-weighted MRI brain scans were acquired at each site and analysed locally using harmonized analysis and quality-control protocols from the ENIGMA consortium ; in this case , all cortical parcellations were performed with the freely available and validated segmentation software FreeSurfer ( versions 5.1 and 5.3 ) . 20 Image acquisition parameters and software descriptions are given in Supplementary Table S4 . Segmentations of 68 ( 34 left and 34 right ) cortical gray matter regions based on the Desikan – Killiany atlas 21 and two whole-hemisphere measures were visually inspected and statistically evaluated for outliers following standardized ENIGMA protocols ( http://enigma.ini.usc.edu/protocols/imaging-protocols ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5443828">PMID:5443828</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In the FreeSurfer analysis suite , there were 34 predefined regions of interest in each cerebral hemisphere 15 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5438403">PMID:5438403</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Similarly , the sgACC ROI was identified using FreeSurfer ’ s Destrieux anatomical parcellation ( aparc.a2009s + aseg . mgz values 11132 and 12132 for left and right , respectively ) 97 , 100 . Given the considerable variability across individual sgACC parcellations in the inclusion of lateral aspects of subcallosal sulcus and gyrus , we defined the sgACC from the FSL MNI template brain rather from individual participant parcellations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5435825">PMID:5435825</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The images were processed and analyzed by the software FreeSurfer version 5.0.0 ( http://surfer.nmr.mgh.harvard.edu/ ) , which provides a set of tools that can be used for the analysis and visualization of structural and functional brain images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5434146">PMID:5434146</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Details about FreeSurfer parcellation can be found at https://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5430685">PMID:5430685</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer volumetry analysis of the striatum was conducted using a multivariate ANCOVA with age and eTIV as covariates .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5430150">PMID:5430150</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The first of these is the segmentation algorithm ( Fischl et al . , 2002 ) included in the well-known software FreeSurfer 2 ( Fischl , 2012 ) , which follows a learning-based strategy .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5429235">PMID:5429235</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging analysis included cortical thickness measurements with Advanced Normalization Tools ( ANTS ) and FreeSurfer , voxel based morphometry ( VBM ) , and Tract Based Spatial Statistics ( TBSS ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5426045">PMID:5426045</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As FreeSurfer is not designed to cope with cerebral lesions , manual editing of the patient data sets was performed to achieve a good anatomical accuracy of the surfaces ( see https://surfer.nmr.mgh.harvard.edu/fswiki/Edits ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5416711">PMID:5416711</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used to label each voxel in the T 1 -weighted image as gray matter , or white matter , or cerebrospinal fluid , or subcortical structures ( for example , hippocampus , amygdala , thalamus , caudate , putamen and globus pallidus ) . 51 FreeSurfer employed a Markov random field model that requests for a prior probability obtained from a training data set with T 1 -weighted images and their manual structural labels .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5416698">PMID:5416698</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Three-dimensional T1-weighted magnetic resonance images were analyzed with the program FreeSurfer v.5.1.0 ( The General Hospital , Boston , MA , USA ) to obtain volumes for structures , automatically and non-interactively , in right and left hemispheres .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5414038">PMID:5414038</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were processed using the publicly available FreeSurfer software ( v4.5 ) ( refs 31 , 32 , 33 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5413592">PMID:5413592</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Fractal dimensionality was quantified using the calcFD toolbox ( http://cmadan.github.io/calcFD/ ) , which we previously developed and distribute freely [ 21 , 24 ] . calcFD is a MATLAB toolbox that calculates the fractal dimensionality of 3D structures and was developed to work with intermediate files from the standard FreeSurfer pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5410600">PMID:5410600</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Multiple viewers can quickly be constructed on viewerjs ( for example a FreeSurfer surface viewer , a tractography viewer , etc ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5407357">PMID:5407357</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.9 Subsequently , a priori defined language-related ROIs chosen on the basis of adult language studies discussed in the introduction ( left BA 45 , left BA 44 , and posterior portion of the left STG ) were obtained using the segmentation tool recon on a standard T1-template in FreeSurfer ( https://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5405238">PMID:5405238</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All electrode sites on each individual ’ s FreeSurfer brain surface were transformed into Talairach coordinates , and finally plotted on the averaged FreeSurfer pial surface image , referred to as ‘ fsaverage ’ ( Desikan et al . , 2006 ; Chan et al . , 2011 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5403938">PMID:5403938</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Studies have assessed FreeSurfer 's performance with images acquired from various MRI scanners or sequences and received reliable results for cortical and subcortical measurements ( Khan et al . , 2008 ; Shi et al . , 2013 ; Mulder et al . , 2014 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5403893">PMID:5403893</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer software ( version 5.1.0 ; Fischl et al . , 2002 ) was used to obtain the hippocampal volumes , which were normalized with the overall intracranial volume ( ICV ) of each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5402578">PMID:5402578</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The high-resolution T2 structure images were introduced into FreeSurfer 5.0 ( http://surfer.nmr.mgh.harvard.edu/fswiki/Download ) for automatic anatomical segmentation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5398548">PMID:5398548</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Once acquired , information contained in the T1 images was summarized in the following data features ( see also Fig 1 ) : Cortical thickness of left and right hemispheres : sMRI data were analyzed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) . Briefly , the pre-processing included removal of non-brain tissue , automated Talairach transformation , tessellation of the grey and white matter boundaries and surface deformation [ 18 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5397412">PMID:5397412</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Local gyrification index ( lGI ) quantifies GI at each vertex on the surface and is computed in a 3D fashion by using a region of interest around each vertex within the FreeSurfer software ( Schaer et al . , 2008 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5394684">PMID:5394684</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analyses based on the vertex across the cortical mantle were carried out using FreeSurfer software as detailed elsewhere 48 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5390726">PMID:5390726</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These pipelines use freely available software from FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ) , FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) , and Connectome Workbench image analysis suites that are discussed in detail in Glasser et al . ( 2013 ) and in the HCP Reference Manual , Chapter 4 : HCP Processing Pipeline ( Jenkinson et al . , 2012 ; Glasser et al . , 2013 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5390400">PMID:5390400</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For coregistration of the resting-state data , the low-resolution T 1 w images were preprocessed using FreeSurfer 's standard recon-all pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5389043">PMID:5389043</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The volume of the PRC , ERC , and hippocampus , and the cortical thickness of the PRC and ERC were calculated using FreeSurfer ( version 5.3.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5387713">PMID:5387713</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each brain hemisphere , a total of 74 cortical structures ( gyri and sulci ) were identified and parceled in FreeSurfer 6.0 using a probabilistic atlas , as described elsewhere 45 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5387160">PMID:5387160</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer gave a cortical mesh of the left hemisphere which we used directly , and a hippocampal volume file which we converted into a tessellated surface mesh .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5385530">PMID:5385530</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Voxels with differences at p < 0.01 corrected by FWER were considered statistically significant and were superimposed on the template of the inflated cortical surface of the brain extracted by FreeSurfer ( http://freesurfer.net/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5385465">PMID:5385465</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted 3 T MRI data were processed in FreeSurfer , yielding accurate cortical reconstructions for surface-based analyses of cortical thickness , area , and volume , and brain segmentation for subcortical and global brain volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5385345">PMID:5385345</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Desikan-Killiany atlas included in the FreeSurfer image analysis suite ( Fischl et al . , 2004 , available at http://surfer.Nmr.mgh.harvard.edu/ ) was employed to obtain volumetric segmentation of the amygdala and entorhinal cortex based on the co-registered T1 images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5381748">PMID:5381748</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer software to conduct a GMV analysis of the subregions ( version 5.0.0 ; http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5380432">PMID:5380432</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We constructed a model of each subject 's cortical surface from the high-resolution anatomical scans using FreeSurfer ( Fischl , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5377433">PMID:5377433</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.4 All cortical and subcortical volumes were extracted from the T 1 -weighted images via the FreeSurfer standard processing pipeline ( Fischl et al . , 2002 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5376622">PMID:5376622</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition to the functionally defined ROIs , we also used an anatomically defined ROI , which covered the intraparietal and transverse parietal sulci ( label = “ S_intrapariet_and_P_trans ; ” Destrieux et al . , 2010 ) defined using an automatic parcellation technique implemented in FreeSurfer ( Fischl , 2004 ; Destrieux et al . , 2010 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5374525">PMID:5374525</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual T1 scans were reconstructed into a surface representation , and functional data were co-registered using the reconstructed cortical surface 49 using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5374503">PMID:5374503</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted MRI images from this database were preprocessed using the recon-all pipeline of the FreeSurfer 5.1 tool ( http://surfer.nmr.mgh.harvard.edu/ ) 63 , which involves the following six steps : ( 1 ) small-motion correction by averaging the available volumes of subjects , ( 2 ) ( non-uniform ) intensity normalization , ( 3 ) affine registration of volumes to the MNI305 atlas , ( 4 ) skull-stripping , ( 5 ) non-linear registration and further normalization using the Gaussian Classifier Atlas ( GCA ) , and ( 6 ) brain parcellation and subcortical region labelling using the GCA .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5373815">PMID:5373815</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual CT values for each predefined region of interest ( hereafter : ROI ; adapted from the Desikan atlas [ 16 ] , 34 ROIs per hemisphere ) were derived by FreeSurfer and exported to R version 3.2.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5373428">PMID:5373428</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Briefly , segmentation of white matter ( WM ) , gray matter ( GM ) , cerebrospinal fluid ( CSF ) , skull and skin was based on FreeSurfer v5.3.0 34 , 35 , FSL v5.0.0 36 and MeshFix v2.0 37 , as shown in Figure 1(a ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5368610">PMID:5368610</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In this step , the binary WM and ventricular cerebrospinal fluid ( CSF ) masks generated with FreeSurfer ( version 5.1 , http://surfer.nmr.harvard.edu ) were linearly registered to native functional space using FSL ( release 5.0.9 , http://www.fmrib.ox.ac.uk/fsl ) , and subsequently eroded by applying a threshold of 0.9 ( WM mask ) and 0.3 ( CSF ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5366904">PMID:5366904</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical T1-weighted images were preprocessed using recon-all pipeline ( FreeSurfer 115 ) to retrieve anatomical parcellation and localize the regions of interest ( ROIs ) in the amygdale of both hemispheres in each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5366815">PMID:5366815</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The target masks were obtained through a subject-based cortical segmentation with FreeSurfer software v . 5.3.0 [ 27 , 28 ] based on the averaged T1-weighted volumes [ 29 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5363996">PMID:5363996</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This can be useful if you want to do a group level analysis on a subset of all participants in your dataset Example : . /my_pipeline /data/my_dataset /scratch/outputs group The script can also accept other arguments specific to your pipeline ( see for example --template_name in FreeSurfer App ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5362049">PMID:5362049</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A second level of analysis was performed by fitting a general linear model ( GLM ) in FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5358981">PMID:5358981</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We also created an anatomical mask of VTC by computing the union of the fusiform and inferiortemporal parcels provided by the FreeSurfer Desikan-Killiany atlas ( Desikan et al . , 2006 ) and trimming the anterior extent of the result to include only visually responsive cortex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5357686">PMID:5357686</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.3 Cortical thickness was estimated with FreeSurfer package ( version 5.3.0 , http://surfer.nmr.mgh.harvard.edu ) , a set of automated programs used for cortical surface reconstruction and thickness measurement ( Dale et al . , 1999 , Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5355434">PMID:5355434</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In FreeSurfer v6.0 , a newly developed version of the hippocampal segmentation tool has been implemented ( 38 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5352661">PMID:5352661</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Figure 3 The collaborative quality control web service of a FreeSurfer segmentation element of one subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5352012">PMID:5352012</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional data was registered to the individual high-resolution anatomical T1 images using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5347012">PMID:5347012</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer 5.1.0 software package ( http://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5344896">PMID:5344896</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Remaining inaccuracies in either gray and white matter segmentation or pial boundaries and residual skull fragments were manually corrected with FreeSurfer ’ s built-in editing software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5343676">PMID:5343676</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer extracts various surfaces from T1-weighted images , including the pial surfaces ( outer boundary of the gray matter ) , white matter surfaces ( boundary between gray and white matter ) , and spherically deformed surfaces which are registered to the FreeSurfer ’ s standard subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5339600">PMID:5339600</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In the present work , the FreeSurfer pipeline ( version 5.1.0 ) was used to calculate total intracranial volume ( eTIV ) representing an estimate for ICV as described in Buckner et al . ( 2004 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5339339">PMID:5339339</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For quality control , the pial and white matter surfaces were then visually inspected for errors , and edited when necessary according to the FreeSurfer editing manual ( https://surfer.nmr.mgh.harvard.edu/fswiki/FreeviewGuide/FreeviewWorkingWithData/FreeviewEditingaRecon ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5339301">PMID:5339301</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image preprocessing and analyses were done using the software package FSL 4.1 ( http://www.fmrib.ox.ac.uk/fsl ) , AFNI 2011 ( http://afni.nimh.nih.gov/afni ) , and FreeSurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ) , as indicated below .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5338815">PMID:5338815</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For up-to-date information , see www.adni-info.org . Data from a total of 427 subjects was retrieved from the ADNI database for whom preprocessed images and FreeSurfer post-processed images were available .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5338621">PMID:5338621</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The segmentation was conducted manually in FreeView , a volume and surface visualization tool within the FreeSurfer software application ( Dale et al . 1999 ) , resulting in segmented skull , scalp , cerebrospinal fluid , grey and white matter , and anterior fontanel compartments .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5337928">PMID:5337928</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The construction of cortical surface was based on 3D T1-images using FreeSurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5337523">PMID:5337523</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis to determine the cortical thickness , gray matter volume and white matter hypodensity were performed using the FreeSurfer software package ( version 5 ; Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5334444">PMID:5334444</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess the association of ROBO1 SNPs with thickness of cortex , the cortical thickness of the structural MRI data was computed using automatic longitudinal stream in FreeSurfer ( Reuter et al . 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5328749">PMID:5328749</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Then the individual brain MRI is processed using automated FreeSurfer preprocessing steps ( version 5.3 ; surfer . nmr . mgh . harvard.edu / ) , which includes segmentation of subcortical white matter , deep gray matter structures and automated parcellation of the cerebral cortex ( Dale and Sereno , 1993 , Desikan et al . , 2006 , Fischl et al . , 2002 , Fischl et al . , 2004 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5325620">PMID:5325620</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>PIB DVR images were created using Logan graphical analysis with frames corresponding to 35 – 90 min after injection and a cerebellar gray matter reference region defined using FreeSurfer version 5.3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5325189">PMID:5325189</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer 5.3 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ; e.g. , [ 61 – 65 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5318543">PMID:5318543</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T 2 values were extracted from normal-appearing white and gray matter brain regions using both expert manual regions-of-interest and user-independent FreeSurfer segmentation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5318380">PMID:5318380</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Specifically , clusters of significant voxels ( corrected for multiple comparison using False Discovery Rate thresholding at q < 0.05 ) were used to functionally define ROIs for each participant separately within left hemisphere using FreeSurfer ( Dale et al . , 1999 ) and AFNI ’ s Surface Mapper ( SUMA ; Saad and Reynolds , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5309810">PMID:5309810</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A three-dimensional model of cortical surface reconstructions computed from T1 images was used in the FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5309571">PMID:5309571</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction was performed using FreeSurfer 5.1 ( https://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5306388">PMID:5306388</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Specifically , the FreeSurfer gray matter parcellations ( wmparc.2 . nii ) were used as ROIs to compute the averages of the temporal mean and dispersion values of ALFF ( t ) and l FCD ( t ) within 34 cortical and 9 subcortical gray matter regions in each brain hemisphere .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5304232">PMID:5304232</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.4 FreeSurfer ( v . 5.3.0 ) was used to quantify cortical thickness and subcortical volumes using high-resolution T 1 -weighted images in all subjects as described in detail elsewhere ( Dale et al . , 1999 , Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5302035">PMID:5302035</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation of the hippocampus was performed using FreeSurfer ( 5.1.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5293760">PMID:5293760</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The thickness value of the entorhinal cortex ( ERC ) was calculated using a method based on ultra-high resolution ex vivo applied to in vivo MRI , as implemented in FreeSurfer ( Fischl et al . , 2009 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/5267531">PMID:5267531</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer segmentation ( 44 ) ( http://freesurfer.net/ ) was used as the basis of the quantitative analysis to obtain regional SUVR with cerebellar gray matter as the reference region .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/4883013">PMID:4883013</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were processed using FreeSurfer ( FS ) version 5.3.0 using the software package ’ s default , automated reconstruction protocol described by Anders M. Dale , Bruce Fischl and colleagues ( ‘ recon-all ’ – see Dale et al . , 1999 ; Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28729929">PMID:28729929</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28757305">PMID:28757305</a></td>
		<td>user</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'> </td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28740249">PMID:28740249</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The 221 individual brain scans were processed with the FreeSurfer version 5.3 pipeline ( http://surfer.nmr.mgh.harvard.edu ) to produce surface-based maps of sulcal depth and local gyrification index for each individual .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28729939">PMID:28729939</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The authors showed that this method is able to predict cognitive outcomes of ADNI subjects using FreeSurfer ‐ based baseline MRI features , MMSE score demographic information and ApoE status .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28725691">PMID:28725691</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer image analysis suite version 5.3.0 ( http://surfer.nmr.mgh.harvard.edu/ ) as previously described. 12 This includes segmentation of the subcortical white matter and deep gray matter volumetric structures , 47 and parcellation of the cortical surface according to a previously published parcellation scheme .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28720860">PMID:28720860</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The AlphaSim program in the rest software ( http://restfmri.net ) was used for the VBM analysis and a precomputed Monte Carlo simulation implemented by FreeSurfer was used for the thickness analysis .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28713261">PMID:28713261</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The CCS designed by Zuo et al . ( 2013 ) provides a computational platform for multimodal neuroimaging brain connectomics computations by integrating the functionalities of AFNI , FSL and FreeSurfer ( Zuo et al . , 2013 ; Betzel et al . , 2014 ; Cao et al . , 2014 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28702351">PMID:28702351</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.6 Each participant 's cortical surface was determined using the FreeSurfer image analysis software ( http://surfer.nmr.mgh.harvard.edu ) from T1-weighted MR volumetric images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28702349">PMID:28702349</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.3 T 1 -weighted scans were processed using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) to estimate vertex-wise cortical thickness and surface arealization ( Dale et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28683072">PMID:28683072</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Surface-based analyses in FreeSurfer involves the removal of non-brain tissue using a hybrid watershed algorithm , automated Talairach transformation , segmentation of subcortical white matter and cortical gray matter , intensity normalization , tessellation of gray/white-matter boundary , automated correction of topological defects and surface deformation to form the gray- and white matter boundary [ 16 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28676757">PMID:28676757</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer analysis suite ( v5.3.0 release 1 ) was used to derive models of cortical surface in each T1-weighted image .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28674492">PMID:28674492</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Brain volumes were derived using the automated FreeSurfer 5.3 Pipeline 2 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28664031">PMID:28664031</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We obtained cortical morphology ( thickness , and surface area ) and brain volume ( global , subcortical , and regional ) measurements using FreeSurfer ( version 5.3.0 , https://surfer.nmr.mgh.harvard.edu ) and compared measurements of children with epilepsy and SCN1A gene mutation ( n   =   21 ) with those of age and gender matched healthy controls ( n   =   42 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28649229">PMID:28649229</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In case of inaccuracies , manual editing was performed according to the FreeSurfer editing manual , 4 either by adding control points to help FreeSurfer identify the WM voxels or removing the skull and dura in case they were considered to be parts of the brain .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28649200">PMID:28649200</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>First , pre-processing co-registers T1 and the brain tissue segmentation from FreeSurfer to FLAIR space before applying inhomogeneity correction and noise removal .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28642699">PMID:28642699</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used to generate anatomical regions of interest ( ROIs ) for the CBF data and , for secondary analyses , cortical thickness and volume and data for these ROIs .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28620297">PMID:28620297</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer version 5.3 3 was used to calculate the cortical thickness from T1-structural images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28616384">PMID:28616384</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Using the FreeSurfer BrainSegNotVent variable , we compared brain size between the two groups with age and gender as covariates .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28607927">PMID:28607927</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Amygdala volumes were analysed with FreeSurfer software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28603479">PMID:28603479</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer ( http://freesurfer.net ) was used to segment T1 images into GM and WM masks , which were visually checked and manually edited for misclassification due to WM T1-hypointensities associated with lesions .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28586896">PMID:28586896</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer software package ( http://surfer.nmr.mgh.harvard.edu ) was used to perform gray and white matter segmentation and extraction of the mid-gray cortical surface used for source locations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28575122">PMID:28575122</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Global probabilistic tractography and diffusion parameter analysis were performed using TRACULA [ 17 ] , which is part of the FreeSurfer 5.3 [ 18 ] software package .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28572766">PMID:28572766</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The images were processed and analyzed by the software FreeSurfer version 5.0.0 ( http://surfer.nmr.mgh.harvard.edu/ ) , which provides a set of tools that can be used for the analysis and visualization of structural and functional brain images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28567001">PMID:28567001</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Details about FreeSurfer parcellation can be found at https://surfer.nmr.mgh.harvard.edu/fswiki/CorticalParcellation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28540179">PMID:28540179</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The first of these is the segmentation algorithm ( Fischl et al . , 2002 ) included in the well-known software FreeSurfer 2 ( Fischl , 2012 ) , which follows a learning-based strategy .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28539636">PMID:28539636</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In the FreeSurfer analysis suite , there were 34 predefined regions of interest in each cerebral hemisphere 15 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28534870">PMID:28534870</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>V1 was determined using the automatic cortical parcellation provided by FreeSurfer 44 based on individual T1 images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28529875">PMID:28529875</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging analysis included cortical thickness measurements with Advanced Normalization Tools ( ANTS ) and FreeSurfer , voxel based morphometry ( VBM ) , and Tract Based Spatial Statistics ( TBSS ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28526817">PMID:28526817</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Similarly , the sgACC ROI was identified using FreeSurfer ’ s Destrieux anatomical parcellation ( aparc.a2009s + aseg . mgz values 11132 and 12132 for left and right , respectively ) 97 , 100 . Given the considerable variability across individual sgACC parcellations in the inclusion of lateral aspects of subcallosal sulcus and gyrus , we defined the sgACC from the FSL MNI template brain rather from individual participant parcellations .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28516033">PMID:28516033</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>As FreeSurfer is not designed to cope with cerebral lesions , manual editing of the patient data sets was performed to achieve a good anatomical accuracy of the surfaces ( see https://surfer.nmr.mgh.harvard.edu/fswiki/Edits ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28507515">PMID:28507515</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Multiple viewers can quickly be constructed on viewerjs ( for example a FreeSurfer surface viewer , a tractography viewer , etc ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28487647">PMID:28487647</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer software ( version 5.1.0 ; Fischl et al . , 2002 ) was used to obtain the hippocampal volumes , which were normalized with the overall intracranial volume ( ICV ) of each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28487638">PMID:28487638</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Studies have assessed FreeSurfer 's performance with images acquired from various MRI scanners or sequences and received reliable results for cortical and subcortical measurements ( Khan et al . , 2008 ; Shi et al . , 2013 ; Mulder et al . , 2014 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28473750">PMID:28473750</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Local gyrification index ( lGI ) quantifies GI at each vertex on the surface and is computed in a 3D fashion by using a region of interest around each vertex within the FreeSurfer software ( Schaer et al . , 2008 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28455501">PMID:28455501</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The FreeSurfer volumetry analysis of the striatum was conducted using a multivariate ANCOVA with age and eTIV as covariates .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28443631">PMID:28443631</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were processed using the publicly available FreeSurfer software ( v4.5 ) ( refs 31 , 32 , 33 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28443058">PMID:28443058</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted 3 T MRI data were processed in FreeSurfer , yielding accurate cortical reconstructions for surface-based analyses of cortical thickness , area , and volume , and brain segmentation for subcortical and global brain volumes .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28443000">PMID:28443000</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The Desikan-Killiany atlas included in the FreeSurfer image analysis suite ( Fischl et al . , 2004 , available at http://surfer.Nmr.mgh.harvard.edu/ ) was employed to obtain volumetric segmentation of the amygdala and entorhinal cortex based on the co-registered T1 images .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28440816">PMID:28440816</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer was used to label each voxel in the T 1 -weighted image as gray matter , or white matter , or cerebrospinal fluid , or subcortical structures ( for example , hippocampus , amygdala , thalamus , caudate , putamen and globus pallidus ) . 51 FreeSurfer employed a Markov random field model that requests for a prior probability obtained from a training data set with T 1 -weighted images and their manual structural labels .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28426817">PMID:28426817</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Once acquired , information contained in the T1 images was summarized in the following data features ( see also Fig 1 ) : Cortical thickness of left and right hemispheres : sMRI data were analyzed with the FreeSurfer image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ) . Briefly , the pre-processing included removal of non-brain tissue , automated Talairach transformation , tessellation of the grey and white matter boundaries and surface deformation [ 18 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28420952">PMID:28420952</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition to the functionally defined ROIs , we also used an anatomically defined ROI , which covered the intraparietal and transverse parietal sulci ( label = “ S_intrapariet_and_P_trans ; ” Destrieux et al . , 2010 ) defined using an automatic parcellation technique implemented in FreeSurfer ( Fischl , 2004 ; Destrieux et al . , 2010 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28417965">PMID:28417965</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analyses based on the vertex across the cortical mantle were carried out using FreeSurfer software as detailed elsewhere 48 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28408973">PMID:28408973</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Briefly , segmentation of white matter ( WM ) , gray matter ( GM ) , cerebrospinal fluid ( CSF ) , skull and skin was based on FreeSurfer v5.3.0 34 , 35 , FSL v5.0.0 36 and MeshFix v2.0 37 , as shown in Figure 1(a ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28402565">PMID:28402565</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Because processing pipelines and ROI parcellations differ between image processing software programs , and because studies have reported that results may vary as a function of program ( Rajagopalan and Pioro , 2015 ) , we computed volumes of the prefrontal cortex ROIs ( mOFC , lateral orbitofrontal cortex , inferior frontal gyrus , MFG , SFG , and ACC ) from two separate programs ( FreeSurfer and SPM ) and performed separate analyses with both sets of values ( see Supplementary Table S1 for correlations between FreeSurfer and SPM for the average volume of each prefrontal cortex subregion ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28402561">PMID:28402561</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The parcellations were obtained in FreeSurfer surface space from http://www.freesurfer.net/fswiki/CorticalParcellation_ Yeo2011 . After calculating the time series for each of these networks , we performed a functional connectivity analysis separately for each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28398341">PMID:28398341</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Three-dimensional T1-weighted magnetic resonance images were analyzed with the program FreeSurfer v.5.1.0 ( The General Hospital , Boston , MA , USA ) to obtain volumes for structures , automatically and non-interactively , in right and left hemispheres .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28397802">PMID:28397802</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each brain hemisphere , a total of 74 cortical structures ( gyri and sulci ) were identified and parceled in FreeSurfer 6.0 using a probabilistic atlas , as described elsewhere 45 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28393878">PMID:28393878</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Voxels with differences at p < 0.01 corrected by FWER were considered statistically significant and were superimposed on the template of the inflated cortical surface of the brain extracted by FreeSurfer ( http://freesurfer.net/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28393012">PMID:28393012</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.4 All cortical and subcortical volumes were extracted from the T 1 -weighted images via the FreeSurfer standard processing pipeline ( Fischl et al . , 2002 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28386486">PMID:28386486</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The target masks were obtained through a subject-based cortical segmentation with FreeSurfer software v . 5.3.0 [ 27 , 28 ] based on the averaged T1-weighted volumes [ 29 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28375078">PMID:28375078</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We constructed a model of each subject 's cortical surface from the high-resolution anatomical scans using FreeSurfer ( Fischl , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28367136">PMID:28367136</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In FreeSurfer v6.0 , a newly developed version of the hippocampal segmentation tool has been implemented ( 38 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28361948">PMID:28361948</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual T1 scans were reconstructed into a surface representation , and functional data were co-registered using the reconstructed cortical surface 49 using FreeSurfer ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28361913">PMID:28361913</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted MRI images from this database were preprocessed using the recon-all pipeline of the FreeSurfer 5.1 tool ( http://surfer.nmr.mgh.harvard.edu/ ) 63 , which involves the following six steps : ( 1 ) small-motion correction by averaging the available volumes of subjects , ( 2 ) ( non-uniform ) intensity normalization , ( 3 ) affine registration of volumes to the MNI305 atlas , ( 4 ) skull-stripping , ( 5 ) non-linear registration and further normalization using the Gaussian Classifier Atlas ( GCA ) , and ( 6 ) brain parcellation and subcortical region labelling using the GCA .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28360851">PMID:28360851</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Figure 3 The collaborative quality control web service of a FreeSurfer segmentation element of one subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28358925">PMID:28358925</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Individual CT values for each predefined region of interest ( hereafter : ROI ; adapted from the Desikan atlas [ 16 ] , 34 ROIs per hemisphere ) were derived by FreeSurfer and exported to R version 3.2.1 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28349948">PMID:28349948</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In this step , the binary WM and ventricular cerebrospinal fluid ( CSF ) masks generated with FreeSurfer ( version 5.1 , http://surfer.nmr.harvard.edu ) were linearly registered to native functional space using FSL ( release 5.0.9 , http://www.fmrib.ox.ac.uk/fsl ) , and subsequently eroded by applying a threshold of 0.9 ( WM mask ) and 0.3 ( CSF ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28348952">PMID:28348952</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.3 Cortical thickness was estimated with FreeSurfer package ( version 5.3.0 , http://surfer.nmr.mgh.harvard.edu ) , a set of automated programs used for cortical surface reconstruction and thickness measurement ( Dale et al . , 1999 , Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28345642">PMID:28345642</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The anatomical T1-weighted images were preprocessed using recon-all pipeline ( FreeSurfer 115 ) to retrieve anatomical parcellation and localize the regions of interest ( ROIs ) in the amygdale of both hemispheres in each subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28344553">PMID:28344553</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Remaining inaccuracies in either gray and white matter segmentation or pial boundaries and residual skull fragments were manually corrected with FreeSurfer ’ s built-in editing software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28338882">PMID:28338882</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation was performed with FreeSurfer 5.3 software which procedure is documented online ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28338785">PMID:28338785</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These images were processed using the Tracts Constrained by Underlying Anatomy pipeline ( Yendiki et al . , 2011 ) , which combines processing steps from FDT ( FSL software for diffusion tensor analysis ; Behrens et al . , 2003 ) with the surface , volume , and automatic segmentation derived from the FreeSurfer software package ( Fischl et al . , 2002 , Fischl and Dale 2000 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28334963">PMID:28334963</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>All electrode sites on each individual ’ s FreeSurfer brain surface were transformed into Talairach coordinates , and finally plotted on the averaged FreeSurfer pial surface image , referred to as ‘ fsaverage ’ ( Desikan et al . , 2006 ; Chan et al . , 2011 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28328939">PMID:28328939</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A second level of analysis was performed by fitting a general linear model ( GLM ) in FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28326033">PMID:28326033</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In the present work , the FreeSurfer pipeline ( version 5.1.0 ) was used to calculate total intracranial volume ( eTIV ) representing an estimate for ICV as described in Buckner et al . ( 2004 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28326021">PMID:28326021</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For quality control , the pial and white matter surfaces were then visually inspected for errors , and edited when necessary according to the FreeSurfer editing manual ( https://surfer.nmr.mgh.harvard.edu/fswiki/FreeviewGuide/FreeviewWorkingWithData/FreeviewEditingaRecon ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28326010">PMID:28326010</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image preprocessing and analyses were done using the software package FSL 4.1 ( http://www.fmrib.ox.ac.uk/fsl ) , AFNI 2011 ( http://afni.nimh.nih.gov/afni ) , and FreeSurfer 5.3 ( http://surfer.nmr.mgh.harvard.edu/ ) , as indicated below .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28321189">PMID:28321189</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Image analysis to determine the cortical thickness , gray matter volume and white matter hypodensity were performed using the FreeSurfer software package ( version 5 ; Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28304286">PMID:28304286</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The volume of the PRC , ERC , and hippocampus , and the cortical thickness of the PRC and ERC were calculated using FreeSurfer ( version 5.3.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28287184">PMID:28287184</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer 5.1.0 software package ( http://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28278228">PMID:28278228</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>This can be useful if you want to do a group level analysis on a subset of all participants in your dataset Example : . /my_pipeline /data/my_dataset /scratch/outputs group The script can also accept other arguments specific to your pipeline ( see for example --template_name in FreeSurfer App ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28276464">PMID:28276464</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer extracts various surfaces from T1-weighted images , including the pial surfaces ( outer boundary of the gray matter ) , white matter surfaces ( boundary between gray and white matter ) , and spherically deformed surfaces which are registered to the FreeSurfer ’ s standard subject .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28275543">PMID:28275543</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Then the individual brain MRI is processed using automated FreeSurfer preprocessing steps ( version 5.3 ; surfer . nmr . mgh . harvard.edu / ) , which includes segmentation of subcortical white matter , deep gray matter structures and automated parcellation of the cerebral cortex ( Dale and Sereno , 1993 , Desikan et al . , 2006 , Fischl et al . , 2002 , Fischl et al . , 2004 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28270763">PMID:28270763</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Specifically , clusters of significant voxels ( corrected for multiple comparison using False Discovery Rate thresholding at q < 0.05 ) were used to functionally define ROIs for each participant separately within left hemisphere using FreeSurfer ( Dale et al . , 1999 ) and AFNI ’ s Surface Mapper ( SUMA ; Saad and Reynolds , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28264071">PMID:28264071</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For up-to-date information , see www.adni-info.org . Data from a total of 427 subjects was retrieved from the ADNI database for whom preprocessed images and FreeSurfer post-processed images were available .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28262813">PMID:28262813</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The construction of cortical surface was based on 3D T1-images using FreeSurfer version 5.3 ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28261049">PMID:28261049</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Specifically , the FreeSurfer gray matter parcellations ( wmparc.2 . nii ) were used as ROIs to compute the averages of the temporal mean and dispersion values of ALFF ( t ) and l FCD ( t ) within 34 cortical and 9 subcortical gray matter regions in each brain hemisphere .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28248957">PMID:28248957</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Functional data was registered to the individual high-resolution anatomical T1 images using FreeSurfer .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28239545">PMID:28239545</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T 2 values were extracted from normal-appearing white and gray matter brain regions using both expert manual regions-of-interest and user-independent FreeSurfer segmentation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28239524">PMID:28239524</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used GEE models with cortical volumes from the FreeSurfer parcellation to confirm VBM results .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28234912">PMID:28234912</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical reconstruction and volumetric segmentation were performed with the FreeSurfer 5.3 image analysis suite ( http://surfer.nmr.mgh.harvard.edu/ ; e.g. , [ 61 – 65 ] .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28226243">PMID:28226243</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We also created an anatomical mask of VTC by computing the union of the fusiform and inferiortemporal parcels provided by the FreeSurfer Desikan-Killiany atlas ( Desikan et al . , 2006 ) and trimming the anterior extent of the result to include only visually responsive cortex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28224079">PMID:28224079</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.4 FreeSurfer ( v . 5.3.0 ) was used to quantify cortical thickness and subcortical volumes using high-resolution T 1 -weighted images in all subjects as described in detail elsewhere ( Dale et al . , 1999 , Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28184415">PMID:28184415</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For coregistration of the resting-state data , the low-resolution T 1 w images were preprocessed using FreeSurfer 's standard recon-all pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28177283">PMID:28177283</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>PIB DVR images were created using Logan graphical analysis with frames corresponding to 35 – 90 min after injection and a cerebellar gray matter reference region defined using FreeSurfer version 5.3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28157104">PMID:28157104</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Volumetric segmentation of the hippocampus was performed using FreeSurfer ( 5.1.0 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28131892">PMID:28131892</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer gave a cortical mesh of the left hemisphere which we used directly , and a hippocampal volume file which we converted into a tessellated surface mesh .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28122961">PMID:28122961</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These pipelines use freely available software from FSL ( http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ ) , FreeSurfer ( http://surfer.nmr.mgh.harvard.edu/ ) , and Connectome Workbench image analysis suites that are discussed in detail in Glasser et al . ( 2013 ) and in the HCP Reference Manual , Chapter 4 : HCP Processing Pipeline ( Jenkinson et al . , 2012 ; Glasser et al . , 2013 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28115740">PMID:28115740</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The procedure including motion correction , intensity normalization , automated topology corrections and automatic segmentations of cortical and subcortical regions is documented elsewhere. 37 – 39 A novel automated algorithm from FreeSurfer that will be released in the next version was used to segment the hippocampal subfields .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28109831">PMID:28109831</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Then we inputted them into a FreeSurfer software package ( version 5.3.0 , http://surfer.nmr.mgh.harvard.edu ) , a widely documented and automated program for reconstructing brain cortical surfaces and calculating CT ( Fischl , 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28054317">PMID:28054317</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Fractal dimensionality was quantified using the calcFD toolbox ( http://cmadan.github.io/calcFD/ ) , which we previously developed and distribute freely [ 21 , 24 ] . calcFD is a MATLAB toolbox that calculates the fractal dimensionality of 3D structures and was developed to work with intermediate files from the standard FreeSurfer pipeline .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27888345">PMID:27888345</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The processing steps to obtain an accurate mask of the whole thalamus are summarized in Fig .   1 . First , we performed cortical and subcortical parcellation of the T1-weighted images with the FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27852731">PMID:27852731</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The segmentation was conducted manually in FreeView , a volume and surface visualization tool within the FreeSurfer software application ( Dale et al . 1999 ) , resulting in segmented skull , scalp , cerebrospinal fluid , grey and white matter , and anterior fontanel compartments .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27631235">PMID:27631235</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The high-resolution T2 structure images were introduced into FreeSurfer 5.0 ( http://surfer.nmr.mgh.harvard.edu/fswiki/Download ) for automatic anatomical segmentation .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27542319">PMID:27542319</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.9 Subsequently , a priori defined language-related ROIs chosen on the basis of adult language studies discussed in the introduction ( left BA 45 , left BA 44 , and posterior portion of the left STG ) were obtained using the segmentation tool recon on a standard T1-template in FreeSurfer ( https://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27240594">PMID:27240594</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>To assess the association of ROBO1 SNPs with thickness of cortex , the cortical thickness of the structural MRI data was computed using automatic longitudinal stream in FreeSurfer ( Reuter et al . 2012 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27169802">PMID:27169802</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer segmentation ( 44 ) ( http://freesurfer.net/ ) was used as the basis of the quantitative analysis to obtain regional SUVR with cerebellar gray matter as the reference region .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27137745">PMID:27137745</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural T1-weighted MRI brain scans were acquired at each site and analysed locally using harmonized analysis and quality-control protocols from the ENIGMA consortium ; in this case , all cortical parcellations were performed with the freely available and validated segmentation software FreeSurfer ( versions 5.1 and 5.3 ) . 20 Image acquisition parameters and software descriptions are given in Supplementary Table S4 . Segmentations of 68 ( 34 left and 34 right ) cortical gray matter regions based on the Desikan – Killiany atlas 21 and two whole-hemisphere measures were visually inspected and statistically evaluated for outliers following standardized ENIGMA protocols ( http://enigma.ini.usc.edu/protocols/imaging-protocols ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/26747746">PMID:26747746</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>T1-weighted images were processed using FreeSurfer ( FS ) version 5.3.0 using the software package ’ s default , automated reconstruction protocol described by Anders M. Dale , Bruce Fischl and colleagues ( ‘ recon-all ’ – see Dale et al . , 1999 ; Fischl et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25880157">PMID:25880157</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer software to conduct a GMV analysis of the subregions ( version 5.0.0 ; http://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28824404">PMID:28824404</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>2 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28775685">PMID:28775685</a></td>
		<td>rdw</td>
		<td align=right>1.00</td>
		<td colspan=92 style='mso-ignore:colspan'>2 http://surfer.nmr.mgh.harvard.edu</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28852030">PMID:28852030</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The DTIs of the participants were processed using the probabilistic tractography functions in TRACULA ( Tracts Constrained by UnderLying Anatomy ) implemented in the FreeSurfer 5.3 development version software package ( Massachusetts General Hospital , Boston , U.S. , http://surfer.nmr.mgh.harvard.edu ) 43 . TRACULA determines 18 major WM tracts using an automated method that reconstructs probabilistic distributions of WM pathways from the native DTIs of each participant .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28822101">PMID:28822101</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Afterwards , the MR images were coregistered to a summation PET image , using a six degrees of freedom , rigid body , affine transformation implemented in SPM5 ( Wellcome Department of Cognitive Neurology , University College , London , UK ) . FreeSurfer v5.0.0 [ 8 , 9 ] was used for definition of ROIs for caudate , putamen , pallidum , accumbens , and cerebellar cortex .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28819123">PMID:28819123</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural MRIs were used to create high-resolution surface reconstructions 24 using FreeSurfer ( https://surfer.nmr.mgh.harvard.edu ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28819107">PMID:28819107</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For completeness , we also performed the same mediation analysis with independently defined DMPFC ROIs based on Desikan-Killiany Atlas 67 provided by FreeSurfer v.5.1.0 . software ( http://surfer.nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28817577">PMID:28817577</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The calculated normalization parameters were then applied to all diffusion parameter maps ( FA , NDI , etc. ) . T1 data were processed using recon — all in FreeSurfer , and then input to TRACULA [ 23 ] along with bedpostx data to automatically segment the following white matter tracts : anterior thalamic radiation ( ATR ) , cingulum , cortical spinal tract ( CST ) , forceps major & minor , inferior longitudinal ( ILF ) , superior longitudinal ( SLF ) , and uncinate fasciculi .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28795120">PMID:28795120</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>2.5 Automated brain tissue segmentation and reconstruction of the T1-weighted structural MRI images were performed using the standard recon-all processing pipeline in FreeSurfer , version 5.2.0 ( Released May , 2013 ; http://surfer-nmr.mgh.harvard.edu/ ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28794970">PMID:28794970</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Whole-brain and gray matter ( GM ) volume were measured with the cross-sectional pipeline of MSmetrix and compared to the outcomes of FreeSurfer ( cross-sectional pipeline ) , SIENAX and SPM .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28794427">PMID:28794427</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Imaging data were preprocessed using FSL and FreeSurfer software 14 , 44 . T1- and T2-weighted structural MRI data were processed as follows .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28792148">PMID:28792148</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used the FreeSurfer software package for estimating the cortical thickness of regions of interest , including the temporal pole , insula , and pars triangularis ( mid-ventrolateral prefrontal cortex ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28791334">PMID:28791334</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The segmented white matter mask from FreeSurfer ( 1 for WM ; 0 elsewhere ) was smoothed to PET resolution , and then a threshold of 0.90 applied to the mask , yielding a volume of interest where the partial volume contribution from non-white matter regions to the mask was < 10 % for every voxel .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28784996">PMID:28784996</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Gini Coefficient calculations for binned top coded income distributions were conducted using the “ binequality ” package in R reference manual available on CRAN ( https://cran.r-project.org/web/packages/binequality/index.html ) . Gene expression ( NR3C1 and AR ) available through Allan Brain Atlas ( http://human.brain-map.org ) . Acquisition of gene expression for FreeSurfer cortical regions using R ( with available code ) provided by French and Paus ( 2015 ) ( https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4584957/ ) 45 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28769095">PMID:28769095</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>The T1-weighted MRI data were pre-processed using FreeSurfer ( Athinoula A .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28765594">PMID:28765594</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>FreeSurfer 48 – 50 was used to segment the MRIs .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28763447">PMID:28763447</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>These regional values can be obtained using the Statistical Parametric Mapping ( SPM ; http://www.fil.ion.ucl.ac.uk/spm/ ) , FMRI Software Library ( FSL ; https://fsl.fmrib.ox.ac.uk/fsl/fslwiki ) , FreeSurfer ( https://surfer.nmr.mgh.harvard.edu/ ) or any other image preprocessing software .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28761095">PMID:28761095</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>For each subject , the intracranial volume ( ICV ) and left and right hemisphere volumes of the basal ganglia nuclei ( caudate , putamen , pallidum ) , thalamus , hippocampus , amygdala and nucleus accumbens were measured using the FreeSurfer 5.1.0 structural imaging pipeline http://surfer.nmr.mgh.harvard.edu 63 , 64 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28747717">PMID:28747717</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In addition to standard fMRI preprocessing using FSL and FreeSurfer 73 , 74 , data were denoised to minimize the impact of subject movement on connectivity estimates .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28728026">PMID:28728026</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>REAGENT or RESOURCE SOURCE IDENTIFIER Software and Algorithms FreeSurfer Fischl , 2012 http://surfer.nmr.mgh.harvard.edu MATLAB MathWorks www.mathworks.com FSL Smith et   al . , 2004 https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/ Connectome Workbench Marcus et   al . , 2011 www.humanconnectome.org Contact for Reagent and Resource Sharing .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28556829">PMID:28556829</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>In contrast to FreeSurfer , the standard method of VBM is performed to determine group differences in regional GM volume by measuring tissue composition ( GM density ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28549795">PMID:28549795</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface reconstruction and regional parcellation was conducted using FreeSurfer v5.3 ( https://surfer.nmr.mgh.harvard.edu ) according to the Desikan atlas ( Desikan et al . , 2006 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28463239">PMID:28463239</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Structural T1-weighted magnetic resonance imaging brain scans were acquired at each site and analysed locally using the fully automated and validated segmentation software FreeSurfer 36 ( freely available at http://surfer.nmr.mgh.harvard.edu/ ) . Image acquisition parameters and software descriptions for each sample are given in Supplementary Table 3 .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28432784">PMID:28432784</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Experiments were also replicated using an alternate implementation of the spm_coreg algorithm included with FreeSurfer version 6 ( see Strengths and Limitations of Current Study section ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/28108742">PMID:28108742</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>A 3D reconstruction of brain structure and brain region segmentation was performed using FreeSurfer software ( http://surfer.nmr.mgh.harvard.edu/ ) on the Linux system .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/27307220">PMID:27307220</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>We used FreeSurfer for segmentation of striatal regions ( 19 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25069046">PMID:25069046</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface meshes were generated from the anatomical images using the standard processing pipeline of the FreeSurfer v5.1.0 software package ( Martinos Center for Biomedical Imaging , http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al . , 1999 ) .</td>
	</tr>	<tr height=21 style='height:16.0pt'>
		<td height=21 align=right style='height:16.0pt'>1847	</td>
		<td><a class="external" target="_blank"href="https://www.ncbi.nlm.nih.gov/pubmed/25067814">PMID:25067814</a></td>
		<td>rdw</td>
		<td align=right>0.80</td>
		<td colspan=92 style='mso-ignore:colspan'>Cortical surface meshes were generated from the anatomical images using the standard processing pipeline of the FreeSurfer v5.1.0 software package ( Martinos Center for Biomedical Imaging , http://surfer.nmr.mgh.harvard.edu/ ) ( Dale et al . , 1999 ) .</td>
	</tr>
</table>

<hr>
<address>UC San Diego, La Jolla, CA</address>
<!-- hhmts start -->Last modified: Sun Oct 23 00:17:08 PDT 2016 <!-- hhmts end -->
</body> </html>
